{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "gDvK9xHtcjTr"
      },
      "source": [
        "#Lab.05 / IBM3202 ‚Äì¬†Membrane Protein Modelling using PyRosetta"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "CTFZC7Mafuzu"
      },
      "source": [
        "##Theoretical Aspects"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VWTkCQifjWXY"
      },
      "source": [
        "As stated by Alford et al. 2020 [Biophys J 2020 118(8), 2042-2055], membrane proteins partner with the surrounding lipid environment to perform essential life processes. They constitute 30% of all proteins and are targets for over 60% of pharmaceuticals. However, experimental difficulties have limited our insights into their molecular mechanisms of function."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "0nPD5ByvctQy"
      },
      "source": [
        "<figure>\n",
        "<center>\n",
        "<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_01.jpg'/>\n",
        "<figcaption>FIGURE 1. The challenge of solving the structure of membrane proteins. As shown in this graph, the number of membrane proteins solved before the extensive use of cryoEM was (and still is) significantly inferior to the number of soluble proteins.\n",
        "<br>Bijelic A & Rompel A (2015) <i>Coord Chem Rev 299, 22-38.\n",
        "</i></figcaption></center>\n",
        "</figure>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "iHpJP9wCdgqp"
      },
      "source": [
        "Therefore, membrane protein modelling is ideal for determining the structure of these proteins and for developing new therapeutics. In this tutorial we will learn to model membrane proteins in PyRosetta and provide examples of the types of problems one can solve with this framework."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wr-vcvaVTKTf"
      },
      "source": [
        "##Overview"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "62vLxb1sSVVf"
      },
      "source": [
        "In this tutorial, we will model different homologs with varying sequence identities of the membrane protein Glycophorin A, also known as GYPA. This is a pore-forming protein that, in the case of the human ortholog, has been broadly studied using different structural biology methods such as NMR and X-Ray crystallography.\n",
        "\n",
        "This overview and tutorial include information from this [tutorial](http://www.meilerlab.org/jobs/downloadfile/name/May2018_rosetta_cm_presentation.pdf) by Nica Marlow and Georg Kuenze"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jmsmt7SGbTmQ"
      },
      "source": [
        "**The Algorithm**\n",
        "\n",
        "At a high-level, the algorithm consists of a long Monte Carlo trajectory starting from a randomly-chosen template. The MC trajectory employs the following moves: \n",
        "\n",
        "* Fragment insertion in unaligned regions\n",
        "\n",
        "* Replacement of a randomly-chosen segment with that from a different template structure\n",
        "\n",
        "* Cartesian-space minimization using a smooth (differentiable) version of the Rosetta centroid energy function. Finally, this is followed by all-atom optimization. \n",
        "\n",
        "See more at the [official docs](https://www.rosettacommons.org/docs/latest/application_documentation/structure_prediction/RosettaCM)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "_rm1_EjsTMWF"
      },
      "source": [
        "**Differences between Single and Multiple Template modeling**\n",
        "\n",
        "Single Template Modeling:\n",
        "- Single template as input\n",
        "- Uses sequence and template derived\n",
        "fragments\n",
        "- Used when available templates have very\n",
        "high identity (>60%)\n",
        "\n",
        "Multiple Template Modeling:\n",
        "- Multiple templates as input\n",
        "- Combine sections of multiple threaded\n",
        "models and sequence derived fragments\n",
        "- Used when available templates have low\n",
        "identity (30-50%)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eiINEFYXfajF"
      },
      "source": [
        "**The protocol**\n",
        "\n",
        "Rosetta comparative modelling (**RosettaCM**) performs a complex series of steps that are illustrated in the scheme below, but that can be summarize in three main steps:\n",
        "\n",
        "1. Align target sequence to template sequences\n",
        "\n",
        "2. Perform a partial-thread of the target sequence onto template structures\n",
        "\n",
        "3. Combine pieces from different templates using RosettaCM hybridize"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NK5aEeL_e2w_"
      },
      "source": [
        "<figure>\n",
        "<center>\n",
        "<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_02.jpg'/>\n",
        "<figcaption>FIGURE 2. A simple scheme of the RosettaCM protocol\n",
        "<br>Song Y et al (2013) <i>Structure 21(10), 1735-1742</i></figcaption></center>\n",
        "</figure>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eTZzTm6SUOIH"
      },
      "source": [
        "**Input Files for RosettaCM**\n",
        "\n",
        "Bare minimum:\n",
        "\n",
        "* Partial-threaded structures (‚úÖ)\n",
        "\n",
        "* Mover definition and options (‚úÖ)\n",
        "\n",
        "Specific to membrane proteins (not needed if modeling\n",
        "soluble proteins):\n",
        "\n",
        "* Membrane spanning regions (span file) (üôã)\n",
        "\n",
        "* Membrane weight patches  (‚úÖ)\n",
        "\n",
        "Optional files based on available information:\n",
        "\n",
        "* Constraint information (eg. atom pair connectivity) \n",
        "\n",
        "* Disulfide Connectivity"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "SKd7eVaiRunj"
      },
      "source": [
        "**For a deep dive into membrane protein modelling using Rosetta, we suggest to check the following articles:**\n",
        "1. Alford RF, Leman JK et al., ‚ÄúAn Integrated Framework Advancing Membrane Protein Modeling and Design,‚Äù PLOS Computational Biology. 2015;11(9).\n",
        "2. Alford RF, Fleming PJ, Fleming KG, Gray JJ (2020) ‚ÄúProtein structure prediction and design in a biologically-realistic implicit membrane‚Äù BioRxiv\n",
        "3. Koehler Leman J, Muller BK, Gray JJ (2017) ‚ÄúExpanding the toolkit for membrane protein modeling in Rosetta‚Äù Bioinformatics 33(5):754-756.\n",
        "4. Koehler Leman J & Gray JJ (2015) ‚ÄúComputational modeling of membrane proteins‚Äù Proteins Structure, Function, and Bioinformatics 83(1):1-24."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "U2vWpWg8klhM"
      },
      "source": [
        "#Part I ‚Äì Exploring the sequences of GYPA orthologs"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "D-u6sgce8Zxm"
      },
      "source": [
        "1. The sequences that we are going to be working with today  correspond to three ortologs of the Glycophorin A membrane protein. In our particular case we are going to model the following ortologs using PyRosetta:\n",
        "\n",
        "  *   [Marmota flaviventris](https://www.ncbi.nlm.nih.gov/protein/XP_027782564.1?report=fasta&log$=seqview)\n",
        "\n",
        "  *   [Gorilla gorilla](https://www.ncbi.nlm.nih.gov/protein/XP_007666620.2?report=fasta&log$=seqview)\n",
        "  \n",
        "  *   [Ornithorhynchus anatinus](https://www.uniprot.org/uniprot/O18798)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xHLARFm0-8ae"
      },
      "source": [
        "<figure>\n",
        "<center>\n",
        "<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_03.png'/>\n",
        "</figcaption></center>\n",
        "</figure>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ESDFFpOd-pYU"
      },
      "source": [
        "2. For the sake of time we are going to provide the already aligned and trimmed sequences in a Grishin format file (yet another alignment format), the partial-threaded structures required to run the modelling protocol and the fragment files:\n",
        "\n",
        "  You will find these files inside its corresponding folder inside input_files. If you want to pursue the generation of these input files on your own, check **Appendix A**.\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ImD_xOcegVuQ",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "05ae5e88-82a3-4565-9861-5434e08c1b31"
      },
      "source": [
        "#Here we download the input files from Github\n",
        "!wget https://raw.githubusercontent.com/pb3lab/ibm3202/master/input_files/input_files_mp.zip\n",
        "#This remove command is just in case you had some previous input files\n",
        "!rm -rf input_files\n",
        "#Here, we unzip the downloaded files\n",
        "!unzip input_files_mp.zip"
      ],
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2022-09-26 17:07:45--  https://raw.githubusercontent.com/pb3lab/ibm3202/master/input_files/input_files_mp.zip\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.108.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 6407895 (6.1M) [application/zip]\n",
            "Saving to: ‚Äòinput_files_mp.zip‚Äô\n",
            "\n",
            "input_files_mp.zip  100%[===================>]   6.11M  --.-KB/s    in 0.08s   \n",
            "\n",
            "2022-09-26 17:07:45 (74.7 MB/s) - ‚Äòinput_files_mp.zip‚Äô saved [6407895/6407895]\n",
            "\n",
            "Archive:  input_files_mp.zip\n",
            "   creating: input_files/\n",
            "   creating: input_files/grishin/\n",
            "  inflating: __MACOSX/input_files/._grishin  \n",
            "  inflating: input_files/.DS_Store   \n",
            "  inflating: __MACOSX/input_files/._.DS_Store  \n",
            "   creating: input_files/cleaned/\n",
            "  inflating: __MACOSX/input_files/._cleaned  \n",
            "   creating: input_files/fasta/\n",
            "  inflating: __MACOSX/input_files/._fasta  \n",
            "  inflating: input_files/gcm.xml     \n",
            "   creating: input_files/threaded_pdbs/\n",
            "  inflating: __MACOSX/input_files/._threaded_pdbs  \n",
            "  inflating: input_files/gcm.options  \n",
            "   creating: input_files/pdbs/\n",
            "  inflating: __MACOSX/input_files/._pdbs  \n",
            "   creating: input_files/octopus/\n",
            "  inflating: __MACOSX/input_files/._octopus  \n",
            "   creating: input_files/wts/\n",
            "  inflating: __MACOSX/input_files/._wts  \n",
            "   creating: input_files/frags/\n",
            "  inflating: __MACOSX/input_files/._frags  \n",
            "   creating: input_files/clustal/\n",
            "  inflating: __MACOSX/input_files/._clustal  \n",
            "  inflating: input_files/grishin/.DS_Store  \n",
            "  inflating: input_files/grishin/Icon  \n",
            "  inflating: __MACOSX/input_files/grishin/._Icon  \n",
            "  inflating: input_files/grishin/5eh6_on_humanGYPA.grishin  \n",
            "  inflating: input_files/grishin/5eh6_on_ornithorhynchusGYPA.grishin  \n",
            "  inflating: input_files/grishin/5eh6_on_gorillaGYPA.grishin  \n",
            "  inflating: input_files/grishin/5eh6_on_marmotaGYPA.grishin  \n",
            "  inflating: input_files/cleaned/.DS_Store  \n",
            "  inflating: input_files/cleaned/Icon  \n",
            "  inflating: __MACOSX/input_files/cleaned/._Icon  \n",
            "  inflating: input_files/cleaned/5eh6_A.pdb  \n",
            "  inflating: input_files/cleaned/5eh6_A.fasta  \n",
            "  inflating: __MACOSX/input_files/cleaned/._5eh6_A.fasta  \n",
            "  inflating: input_files/fasta/.DS_Store  \n",
            "  inflating: __MACOSX/input_files/fasta/._.DS_Store  \n",
            "  inflating: input_files/fasta/Icon  \n",
            "  inflating: __MACOSX/input_files/fasta/._Icon  \n",
            "  inflating: input_files/fasta/gorillaGYPA.fasta  \n",
            "  inflating: __MACOSX/input_files/fasta/._gorillaGYPA.fasta  \n",
            "  inflating: input_files/fasta/ornithorhynchusGYPA.fasta  \n",
            "  inflating: __MACOSX/input_files/fasta/._ornithorhynchusGYPA.fasta  \n",
            "  inflating: input_files/fasta/marmotaGYPA.fasta  \n",
            "  inflating: __MACOSX/input_files/fasta/._marmotaGYPA.fasta  \n",
            "  inflating: input_files/fasta/GYPA.fasta  \n",
            "  inflating: __MACOSX/input_files/fasta/._GYPA.fasta  \n",
            "  inflating: input_files/threaded_pdbs/.DS_Store  \n",
            "  inflating: input_files/threaded_pdbs/5eh6_A.human.pdb  \n",
            "  inflating: input_files/threaded_pdbs/Icon  \n",
            "  inflating: __MACOSX/input_files/threaded_pdbs/._Icon  \n",
            "  inflating: input_files/threaded_pdbs/5eh6_A.marmota.pdb  \n",
            "  inflating: input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb  \n",
            "  inflating: input_files/threaded_pdbs/5eh6_A.gorilla.pdb  \n",
            "  inflating: input_files/pdbs/5eh6.pdb  \n",
            "  inflating: __MACOSX/input_files/pdbs/._5eh6.pdb  \n",
            "  inflating: input_files/pdbs/.DS_Store  \n",
            "  inflating: input_files/pdbs/Icon   \n",
            "  inflating: __MACOSX/input_files/pdbs/._Icon  \n",
            "  inflating: input_files/octopus/.DS_Store  \n",
            "  inflating: __MACOSX/input_files/octopus/._.DS_Store  \n",
            "  inflating: input_files/octopus/Icon  \n",
            "  inflating: __MACOSX/input_files/octopus/._Icon  \n",
            "  inflating: input_files/octopus/ornithorhynchus.topo.txt  \n",
            "  inflating: __MACOSX/input_files/octopus/._ornithorhynchus.topo.txt  \n",
            "  inflating: input_files/octopus/marmota.span  \n",
            "  inflating: input_files/octopus/octopus2span.pl  \n",
            "  inflating: input_files/octopus/gorilla.topo.txt  \n",
            "  inflating: __MACOSX/input_files/octopus/._gorilla.topo.txt  \n",
            "  inflating: input_files/octopus/gorilla.span  \n",
            "  inflating: input_files/octopus/marmota.topo.txt  \n",
            "  inflating: __MACOSX/input_files/octopus/._marmota.topo.txt  \n",
            "  inflating: input_files/octopus/GYPA.topo.txt  \n",
            "  inflating: __MACOSX/input_files/octopus/._GYPA.topo.txt  \n",
            "  inflating: input_files/octopus/ornithorhynchus.span  \n",
            "  inflating: input_files/octopus/GYPA.span  \n",
            "  inflating: input_files/wts/stage1_membrane.wts  \n",
            "  inflating: input_files/wts/rosetta_cm_3.8.xml  \n",
            "  inflating: input_files/wts/.DS_Store  \n",
            "  inflating: input_files/wts/Icon    \n",
            "  inflating: __MACOSX/input_files/wts/._Icon  \n",
            "  inflating: input_files/wts/ref2015.wts  \n",
            "  inflating: input_files/wts/stage3_rlx_membrane.wts  \n",
            "  inflating: input_files/wts/stage2_membrane.wts  \n",
            "  inflating: input_files/frags/GYPA.9mers  \n",
            "  inflating: __MACOSX/input_files/frags/._GYPA.9mers  \n",
            "  inflating: input_files/frags/.DS_Store  \n",
            "  inflating: input_files/frags/marmotaGYPA.9mers  \n",
            "  inflating: __MACOSX/input_files/frags/._marmotaGYPA.9mers  \n",
            "  inflating: input_files/frags/Icon  \n",
            "  inflating: __MACOSX/input_files/frags/._Icon  \n",
            "  inflating: input_files/frags/ornithorhynchusGYPA.3mers  \n",
            "  inflating: __MACOSX/input_files/frags/._ornithorhynchusGYPA.3mers  \n",
            "  inflating: input_files/frags/gorillaGYPA.3mers  \n",
            "  inflating: __MACOSX/input_files/frags/._gorillaGYPA.3mers  \n",
            "  inflating: input_files/frags/gorillaGYPA.9mers  \n",
            "  inflating: __MACOSX/input_files/frags/._gorillaGYPA.9mers  \n",
            "  inflating: input_files/frags/ornithorhynchusGYPA.9mers  \n",
            "  inflating: __MACOSX/input_files/frags/._ornithorhynchusGYPA.9mers  \n",
            "  inflating: input_files/frags/GYPA.3mers  \n",
            "  inflating: __MACOSX/input_files/frags/._GYPA.3mers  \n",
            "  inflating: input_files/frags/marmotaGYPA.3mers  \n",
            "  inflating: __MACOSX/input_files/frags/._marmotaGYPA.3mers  \n",
            "  inflating: input_files/clustal/clustalo-I20201006-164940-0747-5352144-p1m.trimmed.aln  \n",
            "  inflating: __MACOSX/input_files/clustal/._clustalo-I20201006-164940-0747-5352144-p1m.trimmed.aln  \n",
            "  inflating: input_files/clustal/.DS_Store  \n",
            "  inflating: input_files/clustal/.crtz_promals3d_full_edit_alt.fasta.swp  \n",
            "  inflating: input_files/clustal/Icon  \n",
            "  inflating: __MACOSX/input_files/clustal/._Icon  \n",
            "  inflating: input_files/clustal/clustalo-I20201006-164940-0747-5352144-p1m.full.aln  \n",
            "  inflating: __MACOSX/input_files/clustal/._clustalo-I20201006-164940-0747-5352144-p1m.full.aln  \n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "agqXm_0dmf8k"
      },
      "source": [
        "#Part II ‚Äì Determining the membrane spanning regions (span file) using OCTOPUS server"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "a0Qhw-MSmtjk"
      },
      "source": [
        "As we briefly noted at the begining of this tutorial (Overview), we need to give Rosetta a way of knowing to which residues a specific and implicit \"membrane\" potential needs to be applied. These instructions are generated based on the transmembrane (TM) prediction server [**OCTOPUS**](https://octopus.cbr.su.se)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ZIycpMeaCwI4"
      },
      "source": [
        "1. In [**OCTOPUS**](https://octopus.cbr.su.se) you need to input the target protein sequence for the TM prediction. You can find these protein sequences inside the FASTA folder within the downloaded input files."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jNyDLK3CCShq"
      },
      "source": [
        "2. The, the topology files must be processed to convert them into span files as shown in the code cell below:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "u_pxQELHbPX7"
      },
      "source": [
        "%%bash\n",
        "cd /content/input_files/octopus\n",
        "perl octopus2span.pl GYPA.topo.txt > GYPA.span\n",
        "perl octopus2span.pl marmota.topo.txt > marmota.span\n",
        "perl octopus2span.pl ornithorhynchus.topo.txt > ornithorhynchus.span\n",
        "perl octopus2span.pl gorilla.topo.txt > gorilla.span"
      ],
      "execution_count": 2,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "zjUSFLdJCY7f"
      },
      "source": [
        "3. Once you obtained all the span files from **OCTOPUS**, remember to place them inside the following folder\n",
        "\n",
        "```\n",
        "/content/input_files/octopus #Sweet trick! You can open the folder clicking this path\n",
        "```\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "o3hcWpFQXoUx"
      },
      "source": [
        "# Part III ‚Äì Setting Up PyRosetta and Executing the Membrane Protein Modelling protocols\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "i4mcFtHR6u_-"
      },
      "source": [
        "1. We start by setting up PyRosetta in Google Colab, based on your previously compiled and installed PyRosetta readily available in your Google Drive\n",
        "\n",
        "**‚ö†Ô∏èWARNING:** Remember that you MUST have compiled and installed PyRosetta before this tutorial!"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SePtDf1JST8a",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9fe5f6c6-1cd7-4a49-90a4-73c55fcbde72"
      },
      "source": [
        "!pip install pyrosettacolabsetup\n",
        "import pyrosettacolabsetup\n",
        "pyrosettacolabsetup.install_pyrosetta_on_google_drive()"
      ],
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting pyrosettacolabsetup\n",
            "  Downloading pyrosettacolabsetup-1.0.6-py3-none-any.whl (4.7 kB)\n",
            "Installing collected packages: pyrosettacolabsetup\n",
            "Successfully installed pyrosettacolabsetup-1.0.6\n",
            "Mounted at /content/google_drive\n",
            "To obtain PyRosetta license please visit https://www.rosettacommons.org/software/license-and-download\n",
            "Please enter you RC license login:levinthal\n",
            "Please enter you RC license password:paradox\n",
            "Downloading PyRosetta package...\n",
            "--2022-09-26 17:10:18--  https://west.rosettacommons.org/pyrosetta/release/release/PyRosetta4.MinSizeRel.python37.ubuntu.wheel/.latest\n",
            "Resolving west.rosettacommons.org (west.rosettacommons.org)... 128.95.160.153\n",
            "\n",
            "HTTP request sent, awaiting response... 401 Unauthorized\n",
            "Authentication selected: Basic realm=\"Rosetta Repo\"\n",
            "Connecting to west.rosettacommons.org (west.rosettacommons.org)|128.95.160.153|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://west.rosettacommons.org/pyrosetta/release/release/PyRosetta4.MinSizeRel.python37.ubuntu.wheel/pyrosetta-2022.34+release.9ec33c9fe00-cp37-cp37m-linux_x86_64.whl [following]\n",
            "\n",
            "Connecting to west.rosettacommons.org (west.rosettacommons.org)|128.95.160.153|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 1567051012 (1.5G)\n",
            "--2022-09-26 17:10:19--  https://west.rosettacommons.org/pyrosetta/release/release/PyRosetta4.MinSizeRel.python37.ubuntu.wheel/pyrosetta-2022.34+release.9ec33c9fe00-cp37-cp37m-linux_x86_64.whl\n",
            "Reusing existing connection to west.rosettacommons.org:443.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 1567051012 (1.5G)\n",
            "Saving to: ‚Äò/content/google_drive/MyDrive/PyRosetta/colab.bin/tmp6erciih5/pyrosetta-2022.34+release.9ec33c9fe00-cp37-cp37m-linux_x86_64.whl‚Äô\n",
            "\n",
            "pyrosetta-2022.34+r 100%[===================>]   1.46G  38.4MB/s    in 40s     \n",
            "\n",
            "2022-09-26 17:10:59 (37.2 MB/s) - ‚Äò/content/google_drive/MyDrive/PyRosetta/colab.bin/tmp6erciih5/pyrosetta-2022.34+release.9ec33c9fe00-cp37-cp37m-linux_x86_64.whl‚Äô saved [1567051012/1567051012]\n",
            "Moving wheel file pyrosetta-2022.34+release.9ec33c9fe00-cp37-cp37m-linux_x86_64.whl to target dir /content/google_drive/MyDrive/PyRosetta/colab.bin/wheels...\n",
            "Installing PyRosetta wheel '/content/google_drive/MyDrive/PyRosetta/colab.bin/wheels/pyrosetta-2022.34+release.9ec33c9fe00-cp37-cp37m-linux_x86_64.whl'...\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "\n",
            "\n",
            "PyRosetta installed at 'PyRosetta/colab.bin'... Please click \"Runtime ‚Üí Restart runtime\" before using it.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MglIIrmAhWZ0"
      },
      "source": [
        "2. In the following cell code, we import the PyRosetta modules that we are going to use"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "e02ruAYR076e",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "710cc936-4f5d-4751-9ab8-0fc38984f109"
      },
      "source": [
        "#Here we import the PyRosetta modules that we are going to be using\n",
        "%%time\n",
        "import pandas as pd\n",
        "from pyrosetta import *\n",
        "from pyrosetta.rosetta.protocols.rosetta_scripts import *\n",
        "from pyrosetta import (\n",
        "    init, pose_from_sequence, pose_from_file, Pose, MoveMap, create_score_function, get_fa_scorefxn,\n",
        "    MonteCarlo, TrialMover, SwitchResidueTypeSetMover, PyJobDistributor,\n",
        ")\n"
      ],
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "CPU times: user 279 ms, sys: 25.1 ms, total: 304 ms\n",
            "Wall time: 307 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "4vFqicSeD-NQ"
      },
      "source": [
        "3. Next, we performing the Membrane Protein Modelling protocol. This is briefly described for the GYPA from _Gorilla_, whereas the modelling of GYPA for the other orthologs is just a modification of this protocol."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wlmDdcxmXZ9T"
      },
      "source": [
        "#### Gorilla ü¶ç"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "cQc4_jK8Oug2"
      },
      "source": [
        "1. In the following cell code we give PyRosetta all the needed **Flags**. These flags are equivalent to command-line instructions in order to set the path for files and set the value of different (and numerous) options. We pass them into PyRosetta, thus initializing it\n",
        "\n",
        "\n",
        "```\n",
        "pyrosetta.init(flags)\n",
        "```\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "vG9dJLT4bZkk",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "ee5db7db-fcac-433d-80ea-7c31d0613e29"
      },
      "source": [
        "#Flags are equivalent to command-line instructions in order to set the path for files and set the value of different options \n",
        "flags = f\"\"\"\n",
        "-no_fconfig\n",
        "-in:file:fasta ./input_files/fasta/gorillaGYPA.fasta\n",
        "-out:prefix GYPA_gorilla_\n",
        "-out:path:all ./output_files\n",
        "-in:file:spanfile ./input_files/octopus/gorilla.span\t\t\t\n",
        "-membrane:no_interpolate_Mpair\n",
        "-membrane:Menv_penalties\n",
        "-rg_reweight .1\n",
        "-relax:fast\n",
        "-relax:minimize_bond_angles\n",
        "-relax:minimize_bond_lengths\n",
        "-relax:jump_move true\n",
        "-default_max_cycles 2\n",
        "-relax:min_type lbfgs_armijo_nonmonotone\n",
        "-score:weights input_files/wts/stage3_rlx_membrane.wts\t\t\n",
        "-use_bicubic_interpolation\n",
        "-hybridize:stage1_probability 1.0\n",
        "-sog_upper_bound 15\n",
        "-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n",
        "-analytic_etable_evaluation true\n",
        "-linmem_ig 10\n",
        "-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n",
        "\"\"\"\n",
        "pyrosetta.init(flags)"
      ],
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PyRosetta-4 2022 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2022.34+release.9ec33c9fe00427e5b1b0393e84cd8062c07f104c 2022-08-24T09:49:27] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r328 2022.34+release.9ec33c9fe00 9ec33c9fe00427e5b1b0393e84cd8062c07f104c http://www.pyrosetta.org 2022-08-24T09:49:27\n",
            "core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/gorillaGYPA.fasta -out:prefix GYPA_gorilla_ -out:path:all ./output_files -in:file:spanfile ./input_files/octopus/gorilla.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /content/google_drive/MyDrive/PyRosetta/colab.bin/lib/python3.7/site-packages/pyrosetta/database\n",
            "basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-1128725562 seed_offset=0 real_seed=-1128725562\n",
            "basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-1128725562 RG_type=mt19937\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "3EwFD1YdO-xV"
      },
      "source": [
        "2. One of the crucial concepts in PyRosetta are **poses**. They contain the information, sequence or structure, to which the protocol will be applied.\n",
        "\n",
        "  With this in mind, here we create a **pose object** from a sequence, which in this case is the sequence of the GYPA from **_Gorilla_**. Also, we specify a **res_type** which corresponds to the **fullatom_standard**."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "MQRxSQh4bZku",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e4192969-4ac3-443e-803b-8fdfa700ffd2"
      },
      "source": [
        "#Here we create a pose object from a sequence\n",
        "#Also, we specify a res_type which corresponds to the fullatom_standard  \n",
        "%%time\n",
        "pose = pose_from_sequence('DNGEWIQLVPRFSGPEITLIIFGVMAGIIGTILLISYSIRRL', 'fa_standard')"
      ],
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "CPU times: user 1.49 s, sys: 640 ms, total: 2.13 s\n",
            "Wall time: 4.46 s\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "40X9c4WjPXeq"
      },
      "source": [
        "3. While there are multiple ways of creating a protocol in Rosetta, in our case we are going to create a protocol from a **XML object**.\n",
        "\n",
        "**NOTE:** In the **MOVERS** tag we specify the **fragment files** and the **threaded_pdb**!!"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "pLfWzrs0bZky",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0e525a9f-7398-4dc4-e54b-e3ff7459f5cd"
      },
      "source": [
        "#Creating a Rosetta protocol from a XML object\n",
        "#Note that in the <MOVERS> we specify the fragment files and the threaded_pdb!!\n",
        "\n",
        "%%time\n",
        "xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<ROSETTASCRIPTS>\n",
        "    <TASKOPERATIONS>\n",
        "    </TASKOPERATIONS>\n",
        "    <RESIDUE_SELECTORS>\n",
        "    </RESIDUE_SELECTORS>\n",
        "    <SCOREFXNS>\n",
        "      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
        "        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
        "\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "\t</ScoreFunction>\n",
        "    </SCOREFXNS>\n",
        "    <FILTERS>\n",
        "    </FILTERS>\n",
        "    <MOVERS>\n",
        "        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n",
        "       \t\t\t<Fragments three_mers=\"/content/input_files/frags/gorillaGYPA.9mers\" nine_mers=\"/content/input_files/frags/gorillaGYPA.9mers\" />\n",
        "\t          \t<Template pdb=\"/content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n",
        "\t       </Hybridize>\n",
        "    </MOVERS>\n",
        "    <APPLY_TO_POSE>\n",
        "    </APPLY_TO_POSE>\n",
        "    <PROTOCOLS>\t\n",
        "      Add mover=\"add_memb\"\n",
        "\t    <Add mover=\"hybridize\"/>\n",
        "    </PROTOCOLS>\n",
        "<OUTPUT scorefxn=\"membrane\"/>\n",
        "</ROSETTASCRIPTS>\n",
        "    \"\"\").get_mover(\"ParsedProtocol\") "
      ],
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "protocols.rosetta_scripts.RosettaScriptsSchemaValidator: Generating XML Schema for rosetta_scripts...\n",
            "protocols.rosetta_scripts.RosettaScriptsSchemaValidator: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsSchemaValidator: Initializing schema validator...\n",
            "protocols.rosetta_scripts.RosettaScriptsSchemaValidator: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<TASKOPERATIONS/>\n",
            "\t<RESIDUE_SELECTORS/>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n",
            "\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
            "\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t</SCOREFXNS>\n",
            "\t<FILTERS/>\n",
            "\t<MOVERS>\n",
            "\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n",
            "\t\t\t<Fragments nine_mers=\"/content/input_files/frags/gorillaGYPA.9mers\" three_mers=\"/content/input_files/frags/gorillaGYPA.9mers\"/>\n",
            "\t\t\t<Template cst_file=\"AUTO\" pdb=\"/content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\" weight=\"1.000\"/>\n",
            "\t\t</Hybridize>\n",
            "\t</MOVERS>\n",
            "\t<APPLY_TO_POSE/>\n",
            "\t<PROTOCOLS>\n",
            "\t\t<Add mover=\"hybridize\"/>\n",
            "\t</PROTOCOLS>\n",
            "\t<OUTPUT scorefxn=\"membrane\"/>\n",
            "</ROSETTASCRIPTS>\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.scoring.etable: Starting energy table calculation\n",
            "core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "core.scoring.etable: Finished calculating energy tables.\n",
            "core.scoring.P_AA: shapovalov_lib::shap_p_aa_pp_smooth_level of 1( aka low_smooth ) got activated.\n",
            "core.scoring.ramachandran: shapovalov_lib::shap_rama_smooth_level of 4( aka highest_smooth ) got activated.\n",
            "core.energy_methods.CartesianBondedEnergy: Initializing IdealParametersDatabase with default Ks=300 , 80 , 80 , 10 , 80\n",
            "core.energy_methods.CartesianBondedEnergy: Read 759 bb-independent lengths.\n",
            "core.energy_methods.CartesianBondedEnergy: Read 1434 bb-independent angles.\n",
            "core.energy_methods.CartesianBondedEnergy: Read 1 bb-independent torsions.\n",
            "core.energy_methods.CartesianBondedEnergy: Read 529 bb-independent improper tors.\n",
            "core.scoring.etable: Starting energy table calculation\n",
            "core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "core.scoring.etable: Finished calculating energy tables.\n",
            "core.scoring.etable: Using alternate parameters: LK_DGFREE in MembEtable construction.\n",
            "core.scoring.etable: Using alternate parameters: MEMB_LK_DGFREE in MembEtable construction.\n",
            "core.scoring.etable: Using alternate parameters: LK_DGREFCE in MembEtable construction.\n",
            "core.scoring.etable: Using alternate parameters: MEMB_LK_DGREFCE in MembEtable construction.\n",
            "core.scoring.etable: Starting membrane specific energy table calculation\n",
            "core.scoring.etable: Starting energy table calculation\n",
            "core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "core.scoring.etable: Finished calculating energy tables.\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.io.fragments: reading fragments from file: /content/input_files/frags/gorillaGYPA.9mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file /content/input_files/frags/gorillaGYPA.9mers\n",
            "core.import_pose.import_pose: File '/content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb' automatically determined to be of type PDB\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n",
            "protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n",
            "CPU times: user 11.9 s, sys: 165 ms, total: 12.1 s\n",
            "Wall time: 12.6 s\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Dn9Gu8TkjmVU"
      },
      "source": [
        "4. Once our protocol is set up, in the following code cells:\n",
        "  * We create a membrane score function \n",
        "  * Then, we call the PyJobDistrubutor and pass three arguments\n",
        "    * A name prefix to output our models as PDB files\n",
        "    * The number of models to generate\n",
        "    * The score function to employ\n",
        "  * We pass our pose to the job distributor that we just created\n",
        "  * Then, we create a Pandas dataframe to store the score results\n",
        "  * Finally, we loop while the jobs are not completed"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "flBA8JKLbZk6",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "47f1ee53-790c-4eeb-db20-d6b42796eb16"
      },
      "source": [
        "%%time\n",
        "#Here we create a membrane score function\n",
        "membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n",
        "#Here we create a job distributor object and specify the prefix, number of output models or poses and the scorefunction\n",
        "jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_gorilla_\",nstruct=2,scorefxn=membrane)\n",
        "#Here we pass our previously created pose to the Job Distributor\n",
        "jd.native_pose = pose\n",
        "#Here we create a panda dataframe in order to store our score results as we are going to see later\n",
        "gdf = pd.DataFrame()\n",
        "#Here we loop until al n jobs are completed\n",
        "while not jd.job_complete:\n",
        "    test_pose = pose.clone()\n",
        "    xml.apply(test_pose)\n",
        "    test_gdf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n",
        "    gdf = gdf.append(test_gdf)\n",
        "    jd.output_decoy(test_pose)"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working on decoy: GYPA_gorilla__1.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: Found 1 domains using template /content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\n",
            "protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 42  cut: 0  size: 42  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n",
            "core.fragment: compute strand/loop fractions for 42 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLEEEEELLLLLLLEEEEEHHHHHHHHHHHHHHHHHHHHL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 37 1  EDGE 37 1 -1  EDGE 37 42 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 2 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.0098294 0.0098294\n",
            "protocols.moves.TrialCounter:      RandomMover trials=      2;  accepts= 1.0000;  energy_drop/trial=  -1.99151\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.60473 0.459663\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4860;  energy_drop/trial=  -0.00192\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.89403 0.310858\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4344;  energy_drop/trial=  -0.00036\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.86479 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 4.8979 0.310858\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.50005\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    418;  accepts= 0.4761;  energy_drop/trial=   0.00978\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.79795 0.310858\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=  -0.03830\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5025;  energy_drop/trial=   0.00973\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.65678 0.310858\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.3333;  energy_drop/trial=  -0.31937\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.4869;  energy_drop/trial=   0.00731\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.49924 0.203915\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4291;  energy_drop/trial=   0.00220\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.63464 -0.712728\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   0.17630\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.3818;  energy_drop/trial=   0.00208\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.2833 -0.712728\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.19979\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4589;  energy_drop/trial=   0.01142\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.83485 -0.712728\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.20099\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4323;  energy_drop/trial=   0.01118\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 11.0293 -0.712728\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=  -0.39943\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4799;  energy_drop/trial=   0.01339\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.67442 -0.712728\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4244;  energy_drop/trial=   0.00739\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -14.0328 -14.0328\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1300;  energy_drop/trial=  -0.10560\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -12.3161 -12.6829\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.0603;  energy_drop/trial=  -0.01414\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -2.28429 -9.68363\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.1263;  energy_drop/trial=  -1.00521\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -33.0944 -36.2482\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     75;  accepts= 0.0267;  energy_drop/trial=   0.08100\n",
            "protocols.moves.TrialCounter:           picker trials=     10;  accepts= 0.2000;  energy_drop/trial=  -0.57919\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     15;  accepts= 0.2667;  energy_drop/trial=  -0.61032\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -35.0098 -36.3662\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    137;  accepts= 0.0146;  energy_drop/trial=   0.04434\n",
            "protocols.moves.TrialCounter:           picker trials=     38;  accepts= 0.2105;  energy_drop/trial=  -0.23105\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     25;  accepts= 0.3200;  energy_drop/trial=  -0.32328\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -34.3901 -36.246\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     68 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     14 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.1667;  energy_drop/trial=  -0.15385\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -24.2088 -39.0012\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    140;  accepts= 0.0143;  energy_drop/trial=   0.07327\n",
            "protocols.moves.TrialCounter:           picker trials=     29;  accepts= 0.0345;  energy_drop/trial=   0.03007\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.2258;  energy_drop/trial=  -0.11992\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.8216 -37.8216\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     79 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     10 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     11 NO ACCEPTS.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -41.6073 -43.2023\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    146 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     27;  accepts= 0.0370;  energy_drop/trial=  -0.17788\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     27;  accepts= 0.1852;  energy_drop/trial=   0.03766\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -26.9285 -27.144\n",
            "protocols.moves.TrialCounter:           picker trials=     49;  accepts= 0.0816;  energy_drop/trial=   0.09711\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     51;  accepts= 0.0980;  energy_drop/trial=  -0.09385\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -128.191 Finite Diff= 9.23886e+06\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.5019e-09 Deriv= -38.2961 Finite Diff= 7.35921e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 6.7684e-09 Deriv= -39.582 Finite Diff= 6.77314e+09\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "protocols.relax.RelaxScriptManager: Reading relax scripts list from database.\n",
            "protocols.relax.RelaxScriptManager: Looking for no_cst_ramping.txt\n",
            "protocols.relax.RelaxScriptManager: ================== Reading script file: /content/google_drive/MyDrive/PyRosetta/colab.bin/lib/python3.7/site-packages/pyrosetta/database/sampling/relax_scripts/no_cst_ramping.txt ==================\n",
            "protocols.relax.RelaxScriptManager: repeat %%nrepeats%%\n",
            "protocols.relax.RelaxScriptManager: ramp_repack_min 0.02  0.01     1.0\n",
            "protocols.relax.RelaxScriptManager: ramp_repack_min 0.250 0.01     1.0\n",
            "protocols.relax.RelaxScriptManager: ramp_repack_min 0.550 0.01     1.0\n",
            "protocols.relax.RelaxScriptManager: ramp_repack_min 1     0.00001  1.0\n",
            "protocols.relax.RelaxScriptManager: accept_to_best\n",
            "protocols.relax.RelaxScriptManager: endrepeat\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.relax.FastRelax: CMD: repeat  14958.5  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -22.0406  0.0965299  0.0965299  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  11.4139  0.311132  0.311132  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  22.0743  0.213589  0.213589  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  43.2324  0.258036  0.258036  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  43.2324  43.2324  0.258036  0.258036\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  43.2324  0.258036  0.258036  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  43.2324  0.258036  0.258036  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -16.6939  0.151462  0.151462  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  12.5923  0.320267  0.320267  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  21.9592  0.143144  0.143144  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  40.774  0.211868  0.211868  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  40.774  40.774  0.211868  0.211868\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  40.774  0.211868  0.211868  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  40.774  0.211868  0.211868  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -18.7515  0.13225  0.13225  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  16.6627  0.359457  0.359457  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  19.8968  0.252444  0.252444  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  39.7715  0.282396  0.282396  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  39.7715  39.7715  0.282396  0.282396\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  39.7715  0.282396  0.282396  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  39.7715  0.282396  0.282396  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -18.0604  0.155654  0.155654  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  24.0428  0.446814  0.446814  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  26.1945  0.238215  0.238215  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  40.2702  0.28194  0.28194  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  40.2702  39.7715  0.282396  0.282396\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  40.2702  0.28194  0.28194  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  40.2702  0.28194  0.28194  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -21.2622  0.199254  0.199254  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  20.546  0.411558  0.411558  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  21.0707  0.326829  0.326829  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  38.8688  0.336472  0.336472  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  38.8688  38.8688  0.336472  0.336472\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  38.8688  0.336472  0.336472  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  38.8688  0.336472  0.336472  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "Working on decoy: GYPA_gorilla__0.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n",
            "core.fragment: compute strand/loop fractions for 42 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLEEEEELLLLLLLEEEEEHHHHHHHHHHHHHHHHHHHHL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 32 1  EDGE 32 1 -1  EDGE 32 42 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 11 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.118974 0\n",
            "protocols.moves.TrialCounter:      RandomMover trials=     11;  accepts= 0.8182;  energy_drop/trial=  -0.35217\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.392 0.460038\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.26756\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4669;  energy_drop/trial=  -0.00097\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.26244 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.15036 0.586114\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    839;  accepts= 0.4768;  energy_drop/trial=   0.00961\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.1826 0.437684\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4214;  energy_drop/trial=   0.01161\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.10108 0.416995\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.08082\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4849;  energy_drop/trial=   0.00034\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.66504 0.416995\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.3333;  energy_drop/trial=  -0.09862\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.4829;  energy_drop/trial=   0.00889\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.8687 0.175325\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=   0.45919\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4835;  energy_drop/trial=   0.00910\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.76574 -0.00969461\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4760;  energy_drop/trial=   0.00961\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.5799 -0.00969461\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 1.0000;  energy_drop/trial=   0.62313\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5150;  energy_drop/trial=   0.00034\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.52327 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.81853 -0.00969461\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    796;  accepts= 0.5025;  energy_drop/trial=   0.01109\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.84743 -0.00969461\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      8;  accepts= 0.1250;  energy_drop/trial=   0.02096\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    992;  accepts= 0.4183;  energy_drop/trial=   0.00876\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.48923 -0.00969461\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.26545\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4754;  energy_drop/trial=   0.00520\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -2.78989 -5.42477\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.0854;  energy_drop/trial=  -1.01147\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -14.0804 -14.0804\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.0450;  energy_drop/trial=  -0.08155\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 3.46688 -10.4217\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.0850;  energy_drop/trial=   0.04565\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.8434 -39.3755\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     68;  accepts= 0.4265;  energy_drop/trial=  -0.03470\n",
            "protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.0667;  energy_drop/trial=  -0.17053\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     17;  accepts= 0.2941;  energy_drop/trial=   0.11680\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -29.7939 -39.3755\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    130;  accepts= 0.3308;  energy_drop/trial=   0.11769\n",
            "protocols.moves.TrialCounter:           picker trials=     34;  accepts= 0.1765;  energy_drop/trial=  -0.21805\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     36;  accepts= 0.4444;  energy_drop/trial=  -0.07693\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -44.0269 -44.0269\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     69;  accepts= 0.3768;  energy_drop/trial=  -0.22090\n",
            "protocols.moves.TrialCounter:           picker trials=     13;  accepts= 0.0769;  energy_drop/trial=   0.01934\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.2778;  energy_drop/trial=   0.07932\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.7789 -44.0269\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    145;  accepts= 0.3517;  energy_drop/trial=  -0.02696\n",
            "protocols.moves.TrialCounter:           picker trials=     24;  accepts= 0.0833;  energy_drop/trial=  -0.03874\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.2258;  energy_drop/trial=  -0.07988\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -35.8027 -42.5978\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     65;  accepts= 0.2308;  energy_drop/trial=   0.07145\n",
            "protocols.moves.TrialCounter:           picker trials=     22;  accepts= 0.1364;  energy_drop/trial=  -0.15549\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     13;  accepts= 0.2308;  energy_drop/trial=   0.29373\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -34.4781 -42.5978\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    135;  accepts= 0.3333;  energy_drop/trial=   0.03216\n",
            "protocols.moves.TrialCounter:           picker trials=     34;  accepts= 0.0882;  energy_drop/trial=  -0.10061\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.1613;  energy_drop/trial=   0.17568\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -28.1858 -32.6002\n",
            "protocols.moves.TrialCounter:           picker trials=     51;  accepts= 0.0784;  energy_drop/trial=  -0.02397\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     49;  accepts= 0.2857;  energy_drop/trial=  -0.07588\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -14.7963 Finite Diff= 1.32371e+08\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.relax.FastRelax: CMD: repeat  13468.9  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  65.667  0.069236  0.069236  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  131.719  0.38468  0.38468  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  152.889  0.299374  0.299374  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  187.644  0.44157  0.44157  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  187.644  187.644  0.44157  0.44157\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  187.644  0.44157  0.44157  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  187.644  0.44157  0.44157  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  75.9381  0.222731  0.222731  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  114.374  0.196257  0.196257  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  155.469  0.431659  0.431659  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  183.719  0.407947  0.407947  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  183.719  183.719  0.407947  0.407947\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  183.719  0.407947  0.407947  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  183.719  0.407947  0.407947  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  57.0869  0.11315  0.11315  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  113.973  0.254695  0.254695  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  147.159  0.267477  0.267477  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  183.574  0.33843  0.33843  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  183.574  183.574  0.33843  0.33843\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  183.574  0.33843  0.33843  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  183.574  0.33843  0.33843  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  62.7238  0.154044  0.154044  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  111.532  0.155992  0.155992  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  152.374  0.403286  0.403286  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  178.225  0.357647  0.357647  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  178.225  178.225  0.357647  0.357647\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  178.225  0.357647  0.357647  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  178.225  0.357647  0.357647  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  54.5267  0.142401  0.142401  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  111.192  0.225451  0.225451  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  143.033  0.237753  0.237753  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  178.522  0.306214  0.306214  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  178.522  178.225  0.357647  0.357647\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  178.522  0.306214  0.306214  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  178.522  0.306214  0.306214  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "CPU times: user 1min 47s, sys: 1.21 s, total: 1min 49s\n",
            "Wall time: 1min 51s\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Mh_oZyC4XgY6"
      },
      "source": [
        "#### Ornitorrinchus ü•ö"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "PmDCmCogbbja",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a74e6670-88a0-44ba-c209-86d621954180"
      },
      "source": [
        "flags = f\"\"\"\n",
        "-no_fconfig\n",
        "-in:file:fasta ./input_files/fasta/ornithorhynchusGYPA.fasta\n",
        "-out:prefix GYPA_ornithorhynchus_\n",
        "-in:file:spanfile ./input_files/octopus/ornithorhynchus.span\n",
        "-membrane:no_interpolate_Mpair\n",
        "-membrane:Menv_penalties\n",
        "-rg_reweight .1\n",
        "-relax:fast\n",
        "-relax:minimize_bond_angles\n",
        "-relax:minimize_bond_lengths\n",
        "-relax:jump_move true\n",
        "-default_max_cycles 2\n",
        "-relax:min_type lbfgs_armijo_nonmonotone\n",
        "-score:weights input_files/wts/stage3_rlx_membrane.wts\n",
        "-use_bicubic_interpolation\n",
        "-hybridize:stage1_probability 1.0\n",
        "-sog_upper_bound 15\n",
        "-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n",
        "-analytic_etable_evaluation true\n",
        "-linmem_ig 10\n",
        "-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n",
        "\"\"\"\n",
        "pyrosetta.init(flags)"
      ],
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PyRosetta-4 2022 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2022.34+release.9ec33c9fe00427e5b1b0393e84cd8062c07f104c 2022-08-24T09:49:27] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r328 2022.34+release.9ec33c9fe00 9ec33c9fe00427e5b1b0393e84cd8062c07f104c http://www.pyrosetta.org 2022-08-24T09:49:27\n",
            "core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/ornithorhynchusGYPA.fasta -out:prefix GYPA_ornithorhynchus_ -in:file:spanfile ./input_files/octopus/ornithorhynchus.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /content/google_drive/MyDrive/PyRosetta/colab.bin/lib/python3.7/site-packages/pyrosetta/database\n",
            "basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=579149100 seed_offset=0 real_seed=579149100\n",
            "basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=579149100 RG_type=mt19937\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NxxxpSsKbbjd",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "eff6d54f-57a4-4c76-f213-b28ab360c2bf"
      },
      "source": [
        "%%time\n",
        "pose = pose_from_sequence('IDPEIEHIFTGPVAAVIIYAVVCGVIGTILFIALVIKVV', 'fa_standard')"
      ],
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "CPU times: user 44 ms, sys: 0 ns, total: 44 ms\n",
            "Wall time: 43.8 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "XKkrl3q8bbjf",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a000a355-f6b3-401b-beab-c3d217b61dcc"
      },
      "source": [
        "%%time\n",
        "xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<ROSETTASCRIPTS>\n",
        "    <TASKOPERATIONS>\n",
        "    </TASKOPERATIONS>\n",
        "    <RESIDUE_SELECTORS>\n",
        "    </RESIDUE_SELECTORS>\n",
        "    <SCOREFXNS>\n",
        "      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
        "        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
        "\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "\t</ScoreFunction>\n",
        "    </SCOREFXNS>\n",
        "    <FILTERS>\n",
        "    </FILTERS>\n",
        "    <MOVERS>\n",
        "          AddMembraneMover name=\"add_memb\" \n",
        "        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n",
        "       \t\t\t<Fragments three_mers=\"/content/input_files/frags/ornithorhynchusGYPA.3mers\" nine_mers=\"/content/input_files/frags/ornithorhynchusGYPA.9mers\" />\n",
        "\t          \t<Template pdb=\"/content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n",
        "\t       </Hybridize>\n",
        "    </MOVERS>\n",
        "    <APPLY_TO_POSE>\n",
        "    </APPLY_TO_POSE>\n",
        "    <PROTOCOLS>\t\n",
        "      Add mover=\"add_memb\"\n",
        "\t    <Add mover=\"hybridize\"/>\n",
        "    </PROTOCOLS>\n",
        "<OUTPUT scorefxn=\"membrane\"/>\n",
        "</ROSETTASCRIPTS>\n",
        "    \"\"\").get_mover(\"ParsedProtocol\") "
      ],
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<TASKOPERATIONS/>\n",
            "\t<RESIDUE_SELECTORS/>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n",
            "\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
            "\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t</SCOREFXNS>\n",
            "\t<FILTERS/>\n",
            "\t<MOVERS>\n",
            "\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n",
            "\t\t\t<Fragments nine_mers=\"/content/input_files/frags/ornithorhynchusGYPA.9mers\" three_mers=\"/content/input_files/frags/ornithorhynchusGYPA.3mers\"/>\n",
            "\t\t\t<Template cst_file=\"AUTO\" pdb=\"/content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\" weight=\"1.000\"/>\n",
            "\t\t</Hybridize>\n",
            "\t</MOVERS>\n",
            "\t<APPLY_TO_POSE/>\n",
            "\t<PROTOCOLS>\n",
            "\t\t<Add mover=\"hybridize\"/>\n",
            "\t</PROTOCOLS>\n",
            "\t<OUTPUT scorefxn=\"membrane\"/>\n",
            "</ROSETTASCRIPTS>\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.io.fragments: reading fragments from file: /content/input_files/frags/ornithorhynchusGYPA.3mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 3mer fragments from file /content/input_files/frags/ornithorhynchusGYPA.3mers\n",
            "core.io.fragments: reading fragments from file: /content/input_files/frags/ornithorhynchusGYPA.9mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file /content/input_files/frags/ornithorhynchusGYPA.9mers\n",
            "core.import_pose.import_pose: File '/content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb' automatically determined to be of type PDB\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n",
            "protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n",
            "CPU times: user 562 ms, sys: 28.3 ms, total: 590 ms\n",
            "Wall time: 616 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_a7SPqrMbbji",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7096f5b5-c7a0-4ae2-a96d-014602f4bd65"
      },
      "source": [
        "%%time\n",
        "membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n",
        "jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_ornitorrinchus\",nstruct=2,scorefxn=membrane)\n",
        "jd.native_pose = pose\n",
        "odf = pd.DataFrame()\n",
        "while not jd.job_complete:\n",
        "    test_pose = pose.clone()\n",
        "    xml.apply(test_pose)\n",
        "    test_odf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n",
        "    odf = odf.append(test_odf)\n",
        "    jd.output_decoy(test_pose)"
      ],
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working on decoy: GYPA_ornitorrinchus_0.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: Found 1 domains using template /content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\n",
            "protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 39  cut: 0  size: 39  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n",
            "core.fragment: compute strand/loop fractions for 39 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLHEELLLHHHHHHHHHHHHHHHHHHHHHHEEEEL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 39 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 40 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 40 30 1  EDGE 30 1 -1  EDGE 30 39 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99638 275\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00362 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 2 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n",
            "protocols.moves.TrialCounter:      RandomMover trials=      2;  accepts= 1.0000;  energy_drop/trial= -14.93090\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 13.0943 6.74537\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.15088\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5216;  energy_drop/trial=   0.00183\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.96403 6.48293\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   0.19230\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4749;  energy_drop/trial=  -0.00083\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.77418 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 15.0887 6.48293\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    411;  accepts= 0.4866;  energy_drop/trial=   0.02094\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.9132 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.39339 6.46528\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.5000;  energy_drop/trial=   0.10643\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    841;  accepts= 0.4732;  energy_drop/trial=   0.00058\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.65939 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.37302 6.46528\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.42524\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    420;  accepts= 0.4738;  energy_drop/trial=   0.00115\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.70571 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.86132 6.34599\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    431;  accepts= 0.4640;  energy_drop/trial=   0.00788\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.99738 6.00862\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.05792\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4990;  energy_drop/trial=   0.00042\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.39535 6.00862\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.5000;  energy_drop/trial=   0.51638\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4809;  energy_drop/trial=  -0.00068\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.43175 6.00862\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4910;  energy_drop/trial=   0.00143\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.64542 6.00862\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.08511\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5005;  energy_drop/trial=   0.00122\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.41351 6.00862\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 1.0000;  energy_drop/trial=  -0.02830\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4609;  energy_drop/trial=   0.00247\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.6065 11.76\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.2677;  energy_drop/trial=  -1.04782\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 14.8433 9.42503\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1700;  energy_drop/trial=   0.02180\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 18.6397 11.5346\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.3367;  energy_drop/trial=   0.03087\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=39   prot_res=39\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -26.8975 -27.9715\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     69 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     12;  accepts= 0.1667;  energy_drop/trial=  -0.30732\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     19;  accepts= 0.4211;  energy_drop/trial=  -0.84356\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -29.7491 -29.7491\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    133 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     32;  accepts= 0.0938;  energy_drop/trial=  -0.20435\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     35;  accepts= 0.2286;  energy_drop/trial=  -0.45793\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -29.3761 -31.3418\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     68 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     10 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     22;  accepts= 0.3636;  energy_drop/trial=  -0.19640\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -23.0729 -31.3418\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    134 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     21;  accepts= 0.0476;  energy_drop/trial=   0.16162\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     45;  accepts= 0.3111;  energy_drop/trial=  -0.03137\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -32.0175 -32.0175\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     71 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     10;  accepts= 0.2000;  energy_drop/trial=  -0.36734\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     19;  accepts= 0.3684;  energy_drop/trial=  -0.05985\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -31.6146 -32.603\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    140 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     25;  accepts= 0.1200;  energy_drop/trial=  -0.16515\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     35;  accepts= 0.2857;  energy_drop/trial=  -0.00797\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -30.1411 -30.4861\n",
            "protocols.moves.TrialCounter:           picker trials=     48;  accepts= 0.2292;  energy_drop/trial=  -0.16005\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     52;  accepts= 0.2885;  energy_drop/trial=   0.01905\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 9.53674e-09 Deriv= -160.105 Finite Diff= 1.50753e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.41204e-09 Deriv= -154.355 Finite Diff= 3.35105e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.54352e-09 Deriv= -188.888 Finite Diff= 3.46135e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 1.82059e-08 Deriv= -189.794 Finite Diff= 4.68148e+06\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 2.54058e-09 Deriv= -5.69468 Finite Diff= 4.52487e+08\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n",
            "protocols.relax.FastRelax: CMD: repeat  10263  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -72.6519  0.178141  0.178141  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -51.2179  0.254619  0.254619  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -35.6678  0.377333  0.377333  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -17.1521  0.473466  0.473466  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  -17.1521  -17.1521  0.473466  0.473466\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -17.1521  0.473466  0.473466  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -17.1521  0.473466  0.473466  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -68.7699  0.199888  0.199888  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -52.4179  0.234161  0.234161  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -36.2668  0.346316  0.346316  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -17.6825  0.483129  0.483129  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  -17.6825  -17.6825  0.483129  0.483129\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -17.6825  0.483129  0.483129  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -17.6825  0.483129  0.483129  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -67.9807  0.18342  0.18342  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -51.0573  0.249536  0.249536  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -36.5464  0.346117  0.346117  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -17.8365  0.477437  0.477437  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  -17.8365  -17.8365  0.477437  0.477437\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -17.8365  0.477437  0.477437  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -17.8365  0.477437  0.477437  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -67.2102  0.168972  0.168972  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -53.0964  0.19269  0.19269  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -34.2125  0.455577  0.455577  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -19.4501  0.409326  0.409326  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  -19.4501  -19.4501  0.409326  0.409326\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -19.4501  0.409326  0.409326  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -19.4501  0.409326  0.409326  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -66.8238  0.157918  0.157918  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -50.5772  0.312691  0.312691  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -37.6685  0.297547  0.297547  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -18.5312  0.489596  0.489596  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  -18.5312  -19.4501  0.409326  0.409326\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -18.5312  0.489596  0.489596  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -18.5312  0.489596  0.489596  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "Working on decoy: GYPA_ornitorrinchus_1.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n",
            "core.fragment: compute strand/loop fractions for 39 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLHEELLLHHHHHHHHHHHHHHHHHHHHHHEEEEL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 39 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 40 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 40 32 1  EDGE 32 1 -1  EDGE 32 39 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99638 275\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00362 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 2 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.104486 0.104486\n",
            "protocols.moves.TrialCounter:      RandomMover trials=      2;  accepts= 1.0000;  energy_drop/trial= -14.87866\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.17754 6.74537\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.4000;  energy_drop/trial=  -0.14863\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4844;  energy_drop/trial=  -0.00436\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.43913 5.91962\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      7;  accepts= 0.2857;  energy_drop/trial=   0.10088\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    993;  accepts= 0.4824;  energy_drop/trial=  -0.00071\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 0.949828 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.49225 5.91962\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    422;  accepts= 0.4739;  energy_drop/trial=   0.00373\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.71051 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.7333 5.91962\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    366;  accepts= 0.5464;  energy_drop/trial=   0.00496\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 0.6326 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 11.4946 5.91962\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.7500;  energy_drop/trial=   0.07063\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    355;  accepts= 0.5549;  energy_drop/trial=   0.01491\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.89117 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.07618 5.91962\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    431;  accepts= 0.4640;  energy_drop/trial=   0.00500\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.42258 5.91962\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=   1000;  accepts= 0.4710;  energy_drop/trial=   0.00150\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 11.2131 5.91962\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.13784\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4604;  energy_drop/trial=   0.00572\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.85909 5.49457\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.5000;  energy_drop/trial=   0.04903\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4930;  energy_drop/trial=   0.00075\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.71532 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.32448 5.49457\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 1.0000;  energy_drop/trial=  -0.85790\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    426;  accepts= 0.4648;  energy_drop/trial=   0.00832\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.47136 5.49457\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.4000;  energy_drop/trial=  -0.94429\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5025;  energy_drop/trial=   0.00673\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.0081 10.1415\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.2350;  energy_drop/trial=  -0.99405\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 14.7222 7.27201\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.1869;  energy_drop/trial=  -0.03002\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 11.7333 5.81776\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1650;  energy_drop/trial=   0.00130\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=39   prot_res=39\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.1309 -42.6009\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     66 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     18;  accepts= 0.4444;  energy_drop/trial=  -1.09142\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     16;  accepts= 0.3125;  energy_drop/trial=  -0.26611\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -41.8449 -42.6009\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    140 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     27;  accepts= 0.3333;  energy_drop/trial=  -0.77835\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     33;  accepts= 0.2727;  energy_drop/trial=  -0.23036\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -36.3798 -43.7608\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     67 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.2000;  energy_drop/trial=   0.49207\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18 NO ACCEPTS.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -40.3929 -43.7608\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    140 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     31;  accepts= 0.2581;  energy_drop/trial=   0.68174\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     29;  accepts= 0.1379;  energy_drop/trial=  -0.61262\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -45.1789 -45.3219\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     68 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     14;  accepts= 0.3571;  energy_drop/trial=   0.13499\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.3889;  energy_drop/trial=  -0.44740\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -38.5575 -45.3219\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    146 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     29;  accepts= 0.3448;  energy_drop/trial=   0.17624\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     25;  accepts= 0.3200;  energy_drop/trial=  -0.18611\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -34.7624 -40.6716\n",
            "protocols.moves.TrialCounter:           picker trials=     48;  accepts= 0.1250;  energy_drop/trial=   0.05278\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     52;  accepts= 0.3077;  energy_drop/trial=   0.02466\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -151.968 Finite Diff= 1.01157e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 5.38142e-09 Deriv= -111.527 Finite Diff= 3.97211e+08\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n",
            "protocols.relax.FastRelax: CMD: repeat  12102.8  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -91.9869  0.0708871  0.0708871  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -54.5343  0.293931  0.293931  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -47.4069  0.233325  0.233325  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -33.9541  0.294208  0.294208  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  -33.9541  -33.9541  0.294208  0.294208\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -33.9541  0.294208  0.294208  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -33.9541  0.294208  0.294208  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -86.8568  0.0871266  0.0871266  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -58.543  0.255834  0.255834  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -49.5965  0.24396  0.24396  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -35.2043  0.309289  0.309289  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  -35.2043  -35.2043  0.309289  0.309289\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -35.2043  0.309289  0.309289  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -35.2043  0.309289  0.309289  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -91.0486  0.117285  0.117285  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -60.1934  0.210799  0.210799  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -50.9911  0.275682  0.275682  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -37.7478  0.323456  0.323456  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  -37.7478  -37.7478  0.323456  0.323456\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -37.7478  0.323456  0.323456  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -37.7478  0.323456  0.323456  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -91.5941  0.115627  0.115627  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -63.2369  0.195167  0.195167  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -52.8327  0.278684  0.278684  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -38.3568  0.309415  0.309415  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  -38.3568  -38.3568  0.309415  0.309415\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -38.3568  0.309415  0.309415  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -38.3568  0.309415  0.309415  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -91.5138  0.107031  0.107031  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -62.756  0.207986  0.207986  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -51.9452  0.264523  0.264523  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -38.9752  0.302348  0.302348  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  -38.9752  -38.9752  0.302348  0.302348\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -38.9752  0.302348  0.302348  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -38.9752  0.302348  0.302348  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "CPU times: user 1min 28s, sys: 950 ms, total: 1min 29s\n",
            "Wall time: 1min 30s\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "feibleVGXjZX"
      },
      "source": [
        "####Marmota ü¶°"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "sh2p_Qq0bee4",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0890299b-602d-45b9-b476-f5be77784744"
      },
      "source": [
        "%%time\n",
        "flags = f\"\"\"\n",
        "-no_fconfig\n",
        "-in:file:fasta ./input_files/fasta/marmotaGYPA.fasta\n",
        "-out:prefix GYPA_marmota_\n",
        "-out:path:all ./output_files\n",
        "-in:file:spanfile ./input_files/octopus/marmota.span\t\t\t\n",
        "-membrane:no_interpolate_Mpair\n",
        "-membrane:Menv_penalties\n",
        "-rg_reweight .1\n",
        "-relax:fast\n",
        "-relax:minimize_bond_angles\n",
        "-relax:minimize_bond_lengths\n",
        "-relax:jump_move true\n",
        "-default_max_cycles 2\n",
        "-relax:min_type lbfgs_armijo_nonmonotone\n",
        "-score:weights input_files/wts/stage3_rlx_membrane.wts\t\t\n",
        "-use_bicubic_interpolation\n",
        "-hybridize:stage1_probability 1.0\n",
        "-sog_upper_bound 15\n",
        "-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n",
        "-analytic_etable_evaluation true\n",
        "-linmem_ig 10\n",
        "-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n",
        "\"\"\"\n",
        "pyrosetta.init(flags)"
      ],
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PyRosetta-4 2022 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2022.34+release.9ec33c9fe00427e5b1b0393e84cd8062c07f104c 2022-08-24T09:49:27] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r328 2022.34+release.9ec33c9fe00 9ec33c9fe00427e5b1b0393e84cd8062c07f104c http://www.pyrosetta.org 2022-08-24T09:49:27\n",
            "core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/marmotaGYPA.fasta -out:prefix GYPA_marmota_ -out:path:all ./output_files -in:file:spanfile ./input_files/octopus/marmota.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /content/google_drive/MyDrive/PyRosetta/colab.bin/lib/python3.7/site-packages/pyrosetta/database\n",
            "basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=322640306 seed_offset=0 real_seed=322640306\n",
            "basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=322640306 RG_type=mt19937\n",
            "CPU times: user 47.5 ms, sys: 1.01 ms, total: 48.5 ms\n",
            "Wall time: 53.8 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "GahGjthKbee7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "cc52e7ab-6c72-476f-c824-517132ac5c5e"
      },
      "source": [
        "%%time\n",
        "pose = pose_from_sequence('KQGRNQIIHPFSEPVIILIIFAVMFGIIGTILLISFCVRRL', 'fa_standard')"
      ],
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "CPU times: user 52 ms, sys: 979 ¬µs, total: 52.9 ms\n",
            "Wall time: 52.7 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "BhLwUqtBbee9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3ae135a2-017e-484d-969f-9083b8486074"
      },
      "source": [
        "%%time\n",
        "xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<ROSETTASCRIPTS>\n",
        "    <TASKOPERATIONS>\n",
        "    </TASKOPERATIONS>\n",
        "    <RESIDUE_SELECTORS>\n",
        "    </RESIDUE_SELECTORS>\n",
        "    <SCOREFXNS>\n",
        "      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
        "        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
        "\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "\t</ScoreFunction>\n",
        "    </SCOREFXNS>\n",
        "    <FILTERS>\n",
        "    </FILTERS>\n",
        "    <MOVERS>\n",
        "          AddMembraneMover name=\"add_memb\" \n",
        "        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n",
        "       \t\t\t<Fragments three_mers=\"./input_files/frags/marmotaGYPA.3mers\" nine_mers=\"./input_files/frags/marmotaGYPA.9mers\" />\n",
        "\t          \t<Template pdb=\"./input_files/threaded_pdbs/5eh6_A.marmota.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n",
        "\t       </Hybridize>\n",
        "    </MOVERS>\n",
        "    <APPLY_TO_POSE>\n",
        "    </APPLY_TO_POSE>\n",
        "    <PROTOCOLS>\t\n",
        "      Add mover=\"add_memb\"\n",
        "\t    <Add mover=\"hybridize\"/>\n",
        "    </PROTOCOLS>\n",
        "<OUTPUT scorefxn=\"membrane\"/>\n",
        "</ROSETTASCRIPTS>\n",
        "    \"\"\").get_mover(\"ParsedProtocol\") "
      ],
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<TASKOPERATIONS/>\n",
            "\t<RESIDUE_SELECTORS/>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n",
            "\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
            "\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t</SCOREFXNS>\n",
            "\t<FILTERS/>\n",
            "\t<MOVERS>\n",
            "\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n",
            "\t\t\t<Fragments nine_mers=\"./input_files/frags/marmotaGYPA.9mers\" three_mers=\"./input_files/frags/marmotaGYPA.3mers\"/>\n",
            "\t\t\t<Template cst_file=\"AUTO\" pdb=\"./input_files/threaded_pdbs/5eh6_A.marmota.pdb\" weight=\"1.000\"/>\n",
            "\t\t</Hybridize>\n",
            "\t</MOVERS>\n",
            "\t<APPLY_TO_POSE/>\n",
            "\t<PROTOCOLS>\n",
            "\t\t<Add mover=\"hybridize\"/>\n",
            "\t</PROTOCOLS>\n",
            "\t<OUTPUT scorefxn=\"membrane\"/>\n",
            "</ROSETTASCRIPTS>\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.io.fragments: reading fragments from file: ./input_files/frags/marmotaGYPA.3mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 3mer fragments from file ./input_files/frags/marmotaGYPA.3mers\n",
            "core.io.fragments: reading fragments from file: ./input_files/frags/marmotaGYPA.9mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file ./input_files/frags/marmotaGYPA.9mers\n",
            "core.import_pose.import_pose: File './input_files/threaded_pdbs/5eh6_A.marmota.pdb' automatically determined to be of type PDB\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n",
            "protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n",
            "CPU times: user 615 ms, sys: 70 ms, total: 685 ms\n",
            "Wall time: 726 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "l2l4g1onbefA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e413a956-4093-4a44-b74e-06dc80c3b979"
      },
      "source": [
        "membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n",
        "jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_marmota_\",nstruct=2,scorefxn=membrane)\n",
        "jd.native_pose = pose\n",
        "mdf = pd.DataFrame()\n",
        "while not jd.job_complete:\n",
        "    test_pose = pose.clone()\n",
        "    xml.apply(test_pose)\n",
        "    test_mdf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n",
        "    mdf = mdf.append(test_mdf)\n",
        "    jd.output_decoy(test_pose)"
      ],
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working on decoy: GYPA_marmota__0.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: Found 1 domains using template ./input_files/threaded_pdbs/5eh6_A.marmota.pdb\n",
            "protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 41  cut: 0  size: 41  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 ./input_files/threaded_pdbs/5eh6_A.marmota.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n",
            "core.fragment: compute strand/loop fractions for 41 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLLLLLLLLLEEHHHHHHHHHHHHHHHHHHHHHHHHL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 41 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 42 27 1  EDGE 27 1 -1  EDGE 27 41 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99693 325\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00307 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 35 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n",
            "protocols.moves.TrialCounter:      RandomMover trials=     35;  accepts= 0.8286;  energy_drop/trial=  -0.92778\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.63251 0.459412\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=  -0.00001\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5512;  energy_drop/trial=  -0.00333\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.07791 -0.0809074\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4975;  energy_drop/trial=   0.00002\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.745287 -0.15361\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 1.0000;  energy_drop/trial=   1.35715\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5251;  energy_drop/trial=  -0.00189\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.01413 -0.15361\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.4000;  energy_drop/trial=   0.62715\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5216;  energy_drop/trial=   0.00204\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.27248 -0.15361\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.5175;  energy_drop/trial=   0.00143\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.33776 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.42598 -0.15361\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    772;  accepts= 0.5181;  energy_drop/trial=   0.00205\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.10667 -0.15361\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.14739\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5050;  energy_drop/trial=   0.00067\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.14856 -0.15361\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.4000;  energy_drop/trial=  -0.35833\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5296;  energy_drop/trial=   0.00311\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.96826 -0.34796\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.10947\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5256;  energy_drop/trial=   0.00647\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.77552 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.4303 -0.34796\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.03112\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    387;  accepts= 0.5142;  energy_drop/trial=   0.00476\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.57243 -0.34796\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4794;  energy_drop/trial=   0.00694\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -10.0191 -10.0191\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.2060;  energy_drop/trial=  -0.04542\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -5.97796 -9.66875\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.3550;  energy_drop/trial=  -0.02649\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -4.77529 -14.525\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1550;  energy_drop/trial=   0.00176\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=41   prot_res=41\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -56.9287 -56.9287\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     77 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     10;  accepts= 0.2000;  energy_drop/trial=  -1.16447\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     13;  accepts= 0.1538;  energy_drop/trial=  -0.57091\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -53.3924 -56.9287\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    145 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     25;  accepts= 0.1200;  energy_drop/trial=  -0.32434\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     30;  accepts= 0.0667;  energy_drop/trial=  -0.24740\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -51.4015 -54.2617\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     81 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     11;  accepts= 0.1818;  energy_drop/trial=   0.17933\n",
            "protocols.moves.TrialCounter:      picker_rand trials=      8;  accepts= 0.2500;  energy_drop/trial=  -0.03147\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -42.6312 -54.9156\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    152;  accepts= 0.0395;  energy_drop/trial=   0.08348\n",
            "protocols.moves.TrialCounter:           picker trials=     28;  accepts= 0.1071;  energy_drop/trial=   0.05602\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     20;  accepts= 0.1500;  energy_drop/trial=  -0.18829\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -50.1202 -56.7462\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     72 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     13;  accepts= 0.3846;  energy_drop/trial=  -0.55106\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     15;  accepts= 0.2000;  energy_drop/trial=   0.10196\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -51.8218 -56.7462\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    142 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     34;  accepts= 0.2647;  energy_drop/trial=  -0.27323\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     24;  accepts= 0.2083;  energy_drop/trial=   0.08141\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -44.7481 -46.7332\n",
            "protocols.moves.TrialCounter:           picker trials=     43;  accepts= 0.0930;  energy_drop/trial=  -0.01862\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     57;  accepts= 0.1053;  energy_drop/trial=  -0.05199\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -120.565 Finite Diff= 1.02901e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 9.73008e-09 Deriv= -114.396 Finite Diff= 4.41811e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 5.04234e-09 Deriv= -97.4289 Finite Diff= 2.88728e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 6.51261e-07 Deriv= -137.236 Finite Diff= 2.16492e+06\n",
            "core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#15\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n",
            "protocols.relax.FastRelax: CMD: repeat  17544.1  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -114.479  0.312385  0.312385  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -58.544  0.344196  0.344196  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -61.3573  0.30314  0.30314  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -46.2037  0.369864  0.369864  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  -46.2037  -46.2037  0.369864  0.369864\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -46.2037  0.369864  0.369864  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -46.2037  0.369864  0.369864  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -113.403  0.178631  0.178631  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -65.1784  0.37148  0.37148  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -60.9369  0.312581  0.312581  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -47.3743  0.378649  0.378649  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  -47.3743  -47.3743  0.378649  0.378649\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -47.3743  0.378649  0.378649  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -47.3743  0.378649  0.378649  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -99.3572  0.252039  0.252039  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -53.8446  0.457192  0.457192  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -59.7485  0.342766  0.342766  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -47.1033  0.392026  0.392026  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  -47.1033  -47.3743  0.378649  0.378649\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -47.1033  0.392026  0.392026  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -47.1033  0.392026  0.392026  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -105.195  0.375141  0.375141  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -57.5364  0.410116  0.410116  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -59.091  0.214468  0.214468  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -49.3589  0.338775  0.338775  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  -49.3589  -49.3589  0.338775  0.338775\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -49.3589  0.338775  0.338775  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -49.3589  0.338775  0.338775  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -108.388  0.339606  0.339606  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -64.423  0.378999  0.378999  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -61.49  0.235732  0.235732  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -49.7827  0.335205  0.335205  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  -49.7827  -49.7827  0.335205  0.335205\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -49.7827  0.335205  0.335205  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -49.7827  0.335205  0.335205  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "Working on decoy: GYPA_marmota__1.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 ./input_files/threaded_pdbs/5eh6_A.marmota.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n",
            "core.fragment: compute strand/loop fractions for 41 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLLLLLLLLLEEHHHHHHHHHHHHHHHHHHHHHHHHL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 41 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 42 27 1  EDGE 27 1 -1  EDGE 27 41 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99693 325\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00307 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 7 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n",
            "protocols.moves.TrialCounter:      RandomMover trials=      7;  accepts= 0.8571;  energy_drop/trial=  -4.63892\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.58132 0.459412\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.1667;  energy_drop/trial=  -0.00326\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.5322;  energy_drop/trial=  -0.00137\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.15163 -0.388872\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.04323\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5005;  energy_drop/trial=  -0.00019\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.77937 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.14853 -0.388872\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.04640\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    775;  accepts= 0.5148;  energy_drop/trial=   0.00216\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.18815 -0.388872\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      6 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.5161;  energy_drop/trial=   0.00159\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.94616 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.07813 -0.388872\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.04232\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    794;  accepts= 0.5025;  energy_drop/trial=   0.00179\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.50428 -0.487655\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.5000;  energy_drop/trial=  -0.21871\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5371;  energy_drop/trial=   0.00278\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.918087 -0.487655\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=  -0.02014\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5191;  energy_drop/trial=   0.00149\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.90068 -0.487655\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5015;  energy_drop/trial=   0.00240\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.37589 -0.487655\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.5025;  energy_drop/trial=   0.00787\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.12532 -0.487655\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5095;  energy_drop/trial=   0.00162\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 3.38233 -0.487655\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5125;  energy_drop/trial=   0.00388\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -6.68843 -11.4501\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1150;  energy_drop/trial=  -1.06374\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -4.56877 -12.2725\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.1106;  energy_drop/trial=  -0.02361\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -5.92394 -12.2704\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.3434;  energy_drop/trial=   0.03205\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=41   prot_res=41\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -34.1892 -41.9681\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     68;  accepts= 0.2647;  energy_drop/trial=   0.08573\n",
            "protocols.moves.TrialCounter:           picker trials=     16;  accepts= 0.1250;  energy_drop/trial=  -0.09543\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     16;  accepts= 0.3125;  energy_drop/trial=  -0.88502\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -30.6995 -41.9681\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    143;  accepts= 0.1958;  energy_drop/trial=   0.17539\n",
            "protocols.moves.TrialCounter:           picker trials=     28;  accepts= 0.1071;  energy_drop/trial=  -0.04332\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     29;  accepts= 0.3103;  energy_drop/trial=  -1.04263\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -38.1299 -41.4766\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     65;  accepts= 0.0615;  energy_drop/trial=  -0.05539\n",
            "protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.0667;  energy_drop/trial=  -0.21497\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     20;  accepts= 0.0500;  energy_drop/trial=  -0.02646\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -41.0891 -41.4766\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    145;  accepts= 0.1517;  energy_drop/trial=   0.06519\n",
            "protocols.moves.TrialCounter:           picker trials=     26;  accepts= 0.0769;  energy_drop/trial=  -0.49304\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     29;  accepts= 0.0690;  energy_drop/trial=  -0.23953\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -39.8413 -44.2343\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     58 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:           picker trials=     17;  accepts= 0.0588;  energy_drop/trial=  -0.38356\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     25;  accepts= 0.0800;  energy_drop/trial=   0.17572\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -35.5795 -44.2343\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    137;  accepts= 0.0365;  energy_drop/trial=   0.02887\n",
            "protocols.moves.TrialCounter:           picker trials=     27;  accepts= 0.1111;  energy_drop/trial=  -0.30244\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     36;  accepts= 0.1111;  energy_drop/trial=   0.17626\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.924 -39.5582\n",
            "protocols.moves.TrialCounter:           picker trials=     47;  accepts= 0.0638;  energy_drop/trial=  -0.25415\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     53;  accepts= 0.1698;  energy_drop/trial=  -0.16021\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 9.53674e-09 Deriv= -81.1497 Finite Diff= 2.15768e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 3.77389e-07 Deriv= -408.697 Finite Diff= 6.96721e+06\n",
            "core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#8\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n",
            "protocols.relax.FastRelax: CMD: repeat  15056.4  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -51.8908  0.321408  0.321408  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -39.4537  0.225903  0.225903  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -33.8527  0.318766  0.318766  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -25.3096  0.469192  0.469192  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  -25.3096  -25.3096  0.469192  0.469192\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -25.3096  0.469192  0.469192  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -25.3096  0.469192  0.469192  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -54.9389  0.12786  0.12786  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -40.9137  0.360214  0.360214  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -36.5795  0.341977  0.341977  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -27.0305  0.462582  0.462582  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  -27.0305  -27.0305  0.462582  0.462582\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -27.0305  0.462582  0.462582  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -27.0305  0.462582  0.462582  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -59.1016  0.117105  0.117105  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -40.4436  0.365383  0.365383  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -35.4968  0.311706  0.311706  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -25.8999  0.492412  0.492412  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  -25.8999  -27.0305  0.462582  0.462582\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -25.8999  0.492412  0.492412  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -25.8999  0.492412  0.492412  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -59.032  0.278672  0.278672  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -44.9773  0.297289  0.297289  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -36.3705  0.380562  0.380562  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -28.2869  0.441664  0.441664  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  -28.2869  -28.2869  0.441664  0.441664\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -28.2869  0.441664  0.441664  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -28.2869  0.441664  0.441664  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -61.2064  0.271766  0.271766  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -44.4947  0.317184  0.317184  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -37.3606  0.370959  0.370959  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -28.7395  0.468224  0.468224  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  -28.7395  -28.7395  0.468224  0.468224\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -28.7395  0.468224  0.468224  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -28.7395  0.468224  0.468224  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "0ehjClu0YhXk"
      },
      "source": [
        "####Human üßë"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "slUPRFWuoOQR",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4c9e0dbc-b569-4278-d9ca-1a2b43d37628"
      },
      "source": [
        "%%time\n",
        "flags = f\"\"\"\n",
        "-no_fconfig\n",
        "-in:file:fasta ./input_files/fasta/GYPA.fasta\n",
        "-parser:protocol ./input_files/cm.xml\n",
        "-out:prefix GYPA_\n",
        "-out:path:all ./output_files\n",
        "-in:file:spanfile ./input_files/octopus/GYPA.span\t\t\t\n",
        "-membrane:no_interpolate_Mpair\n",
        "-membrane:Menv_penalties\n",
        "-rg_reweight .1\n",
        "-relax:fast\n",
        "-relax:minimize_bond_angles\n",
        "-relax:minimize_bond_lengths\n",
        "-relax:jump_move true\n",
        "-default_max_cycles 2\n",
        "-relax:min_type lbfgs_armijo_nonmonotone\n",
        "-score:weights input_files/wts/stage3_rlx_membrane.wts\t\t\n",
        "-use_bicubic_interpolation\n",
        "-hybridize:stage1_probability 1.0\n",
        "-sog_upper_bound 15\n",
        "-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n",
        "-analytic_etable_evaluation true\n",
        "-linmem_ig 10\n",
        "-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n",
        "\"\"\"\n",
        "pyrosetta.init(flags)"
      ],
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PyRosetta-4 2022 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2022.34+release.9ec33c9fe00427e5b1b0393e84cd8062c07f104c 2022-08-24T09:49:27] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r328 2022.34+release.9ec33c9fe00 9ec33c9fe00427e5b1b0393e84cd8062c07f104c http://www.pyrosetta.org 2022-08-24T09:49:27\n",
            "core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/GYPA.fasta -parser:protocol ./input_files/cm.xml -out:prefix GYPA_ -out:path:all ./output_files -in:file:spanfile ./input_files/octopus/GYPA.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /content/google_drive/MyDrive/PyRosetta/colab.bin/lib/python3.7/site-packages/pyrosetta/database\n",
            "basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-393799654 seed_offset=0 real_seed=-393799654\n",
            "basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-393799654 RG_type=mt19937\n",
            "CPU times: user 44.6 ms, sys: 3.06 ms, total: 47.7 ms\n",
            "Wall time: 53.4 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "dYwsIljin-vA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "30f7e7c8-d32e-43ea-ab9b-4da2be81b8e4"
      },
      "source": [
        "%%time\n",
        "pose = pose_from_sequence('ETGERVQLAHHFSEPEITLIIFGVMAGVIGTILLISYGIRRL', 'fa_standard')"
      ],
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "CPU times: user 52 ms, sys: 0 ns, total: 52 ms\n",
            "Wall time: 53.6 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "oYRxZf4M8_Un",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "44e9e266-4ab8-44ae-fe68-5aa307bfbfe8"
      },
      "source": [
        "%%time\n",
        "xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<ROSETTASCRIPTS>\n",
        "    <TASKOPERATIONS>\n",
        "    </TASKOPERATIONS>\n",
        "    <RESIDUE_SELECTORS>\n",
        "    </RESIDUE_SELECTORS>\n",
        "    <SCOREFXNS>\n",
        "      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "      </ScoreFunction>\n",
        "      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n",
        "        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
        "        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
        "\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
        "\t</ScoreFunction>\n",
        "    </SCOREFXNS>\n",
        "    <FILTERS>\n",
        "    </FILTERS>\n",
        "    <MOVERS>\n",
        "          AddMembraneMover name=\"add_memb\" \n",
        "        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n",
        "       \t\t\t<Fragments three_mers=\"./input_files/frags/GYPA.3mers\" nine_mers=\"./input_files/frags/GYPA.9mers\" />\n",
        "\t          \t<Template pdb=\"/content/input_files/threaded_pdbs/5eh6_A.human.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n",
        "\t       </Hybridize>\n",
        "    </MOVERS>\n",
        "    <APPLY_TO_POSE>\n",
        "    </APPLY_TO_POSE>\n",
        "    <PROTOCOLS>\t\n",
        "      Add mover=\"add_memb\"\n",
        "\t    <Add mover=\"hybridize\"/>\n",
        "    </PROTOCOLS>\n",
        "<OUTPUT scorefxn=\"membrane\"/>\n",
        "</ROSETTASCRIPTS>\n",
        "    \"\"\").get_mover(\"ParsedProtocol\") "
      ],
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<TASKOPERATIONS/>\n",
            "\t<RESIDUE_SELECTORS/>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n",
            "\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n",
            "\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n",
            "\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n",
            "\t\t</ScoreFunction>\n",
            "\t</SCOREFXNS>\n",
            "\t<FILTERS/>\n",
            "\t<MOVERS>\n",
            "\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n",
            "\t\t\t<Fragments nine_mers=\"./input_files/frags/GYPA.9mers\" three_mers=\"./input_files/frags/GYPA.3mers\"/>\n",
            "\t\t\t<Template cst_file=\"AUTO\" pdb=\"/content/input_files/threaded_pdbs/5eh6_A.human.pdb\" weight=\"1.000\"/>\n",
            "\t\t</Hybridize>\n",
            "\t</MOVERS>\n",
            "\t<APPLY_TO_POSE/>\n",
            "\t<PROTOCOLS>\n",
            "\t\t<Add mover=\"hybridize\"/>\n",
            "\t</PROTOCOLS>\n",
            "\t<OUTPUT scorefxn=\"membrane\"/>\n",
            "</ROSETTASCRIPTS>\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n",
            "core.io.fragments: reading fragments from file: ./input_files/frags/GYPA.3mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 3mer fragments from file ./input_files/frags/GYPA.3mers\n",
            "core.io.fragments: reading fragments from file: ./input_files/frags/GYPA.9mers ...\n",
            "core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n",
            "core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file ./input_files/frags/GYPA.9mers\n",
            "core.import_pose.import_pose: File '/content/input_files/threaded_pdbs/5eh6_A.human.pdb' automatically determined to be of type PDB\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n",
            "protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n",
            "CPU times: user 614 ms, sys: 32 ms, total: 646 ms\n",
            "Wall time: 675 ms\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "gvyV2upvoAW1",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b45fabae-370e-4b1d-b9f1-6d7e439fe1a3"
      },
      "source": [
        "%%time\n",
        "membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n",
        "jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_human_\",nstruct=2,scorefxn=membrane)\n",
        "jd.native_pose = pose\n",
        "#Name of the score dataframe\n",
        "hdf = pd.DataFrame()\n",
        "while not jd.job_complete:\n",
        "    test_pose = pose.clone()\n",
        "    xml.apply(test_pose)\n",
        "    test_hdf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n",
        "    hdf = hdf.append(test_hdf)\n",
        "    jd.output_decoy(test_pose)"
      ],
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working on decoy: GYPA_human__1.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: Found 1 domains using template /content/input_files/threaded_pdbs/5eh6_A.human.pdb\n",
            "protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 42  cut: 0  size: 42  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol:\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.human.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n",
            "core.fragment: compute strand/loop fractions for 42 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLHHHHLLLLLLLEEEEHHHHHHHHHHHHHHHHHHHHLL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 23 1  EDGE 23 1 -1  EDGE 23 42 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 3 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.0956383 0.0956383\n",
            "protocols.moves.TrialCounter:      RandomMover trials=      3;  accepts= 1.0000;  energy_drop/trial=  -3.25101\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.477872 0.459527\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.6116;  energy_drop/trial=  -0.01115\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.96458 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.58114 0.464753\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.07383\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    732;  accepts= 0.5451;  energy_drop/trial=   0.00185\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.37678 -0.130143\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.5000;  energy_drop/trial=   0.67410\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5763;  energy_drop/trial=   0.00323\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.850027 -0.130143\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.26624\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5581;  energy_drop/trial=   0.00152\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.11737 -0.130143\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.18536\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5928;  energy_drop/trial=   0.00181\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.87007 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.35517 -0.130143\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    336;  accepts= 0.5952;  energy_drop/trial=   0.02525\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.14713 -0.130143\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.3333;  energy_drop/trial=  -0.22452\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.5815;  energy_drop/trial=   0.00264\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 4.9265 -1.09824\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.4000;  energy_drop/trial=  -0.04188\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5377;  energy_drop/trial=   0.00529\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 11.7646 -1.09824\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5431;  energy_drop/trial=   0.01289\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.49561 -1.09824\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=   0.34455\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5838;  energy_drop/trial=   0.00758\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.12508 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.23128 -1.09824\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    377;  accepts= 0.5305;  energy_drop/trial=   0.00618\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -18.4886 -18.8918\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.0302;  energy_drop/trial=  -1.16976\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -18.3753 -20.4999\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.0354;  energy_drop/trial=  -0.05962\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -6.24952 -11.2963\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1250;  energy_drop/trial=  -0.00718\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -28.1376 -35.8029\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     69;  accepts= 0.0580;  energy_drop/trial=   0.02043\n",
            "protocols.moves.TrialCounter:           picker trials=     14 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     17;  accepts= 0.0588;  energy_drop/trial=   0.00574\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -33.8444 -35.8029\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    144;  accepts= 0.0625;  energy_drop/trial=  -0.02699\n",
            "protocols.moves.TrialCounter:           picker trials=     25 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.0645;  energy_drop/trial=  -0.01012\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -39.0912 -40.1873\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     71;  accepts= 0.0141;  energy_drop/trial=  -0.06219\n",
            "protocols.moves.TrialCounter:           picker trials=     12;  accepts= 0.0833;  energy_drop/trial=  -0.68314\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     17;  accepts= 0.0588;  energy_drop/trial=   0.06448\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -40.6002 -43.4659\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    140;  accepts= 0.0071;  energy_drop/trial=  -0.03154\n",
            "protocols.moves.TrialCounter:           picker trials=     25;  accepts= 0.0800;  energy_drop/trial=  -0.21328\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     35;  accepts= 0.0571;  energy_drop/trial=  -0.09367\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -33.1464 -33.1464\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     70;  accepts= 0.2143;  energy_drop/trial=  -3.10281\n",
            "protocols.moves.TrialCounter:           picker trials=     12;  accepts= 0.1667;  energy_drop/trial=  -0.81066\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.1111;  energy_drop/trial=  -0.00660\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -38.3404 -38.6511\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    149;  accepts= 0.1946;  energy_drop/trial=  -1.49255\n",
            "protocols.moves.TrialCounter:           picker trials=     20;  accepts= 0.1000;  energy_drop/trial=  -0.48639\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.0645;  energy_drop/trial=  -0.00383\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -7.61539 -7.61539\n",
            "protocols.moves.TrialCounter:           picker trials=     52 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     48;  accepts= 0.1667;  energy_drop/trial=  -0.55181\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -99.6495 Finite Diff= 1.18144e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 2.84276e-09 Deriv= -439.285 Finite Diff= 2.3714e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 5.6877e-09 Deriv= -59.5991 Finite Diff= 1.20493e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 3.09233e-09 Deriv= -73.8344 Finite Diff= 5.14965e+09\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 3.02566e-07 Deriv= -635.828 Finite Diff= 5.8065e+06\n",
            "core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#32\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.relax.FastRelax: CMD: repeat  16289.7  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  10.3673  0.339476  0.339476  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  88.908  1.00329  1.00329  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  81.2047  0.690927  0.690927  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  97.0643  0.869004  0.869004  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  97.0643  97.0643  0.869004  0.869004\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  97.0643  0.869004  0.869004  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  97.0643  0.869004  0.869004  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  22.3172  0.267869  0.267869  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  71.2842  0.788771  0.788771  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  71.2621  0.63329  0.63329  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  94.7719  0.739022  0.739022  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  94.7719  94.7719  0.739022  0.739022\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  94.7719  0.739022  0.739022  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  94.7719  0.739022  0.739022  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  16.6161  0.295205  0.295205  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  81.0835  0.97526  0.97526  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  72.3544  0.779542  0.779542  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  91.8068  0.809843  0.809843  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  91.8068  91.8068  0.809843  0.809843\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  91.8068  0.809843  0.809843  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  91.8068  0.809843  0.809843  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  18.4604  0.346042  0.346042  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  85.3008  0.933666  0.933666  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  75.1575  0.823382  0.823382  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  93.6686  0.763558  0.763558  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  93.6686  91.8068  0.809843  0.809843\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  93.6686  0.763558  0.763558  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  93.6686  0.763558  0.763558  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  13.7142  0.38965  0.38965  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  60.7295  0.815031  0.815031  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  66.5197  0.781761  0.781761  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  86.3587  0.840662  0.840662  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  86.3587  86.3587  0.840662  0.840662\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  86.3587  0.840662  0.840662  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  86.3587  0.840662  0.840662  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "Working on decoy: GYPA_human__0.pdb\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n",
            "protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n",
            "protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.human.pdb\n",
            "protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n",
            "protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n",
            "protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n",
            "core.fragment: compute strand/loop fractions for 42 residues...\n",
            "protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLHHHHLLLLLLLEEEEHHHHHHHHHHHHHHHHHHHHLL\n",
            "protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n",
            "protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n",
            "protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.FoldTreeHybridize:\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n",
            "protocols.hybridization.HybridizeFoldtreeDynamic:\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n",
            "protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 33 1  EDGE 33 1 -1  EDGE 33 42 -1\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n",
            "protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 1                                                         \n",
            "   Folding with score0 for max of 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 25 cycles.\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.000866687 0\n",
            "protocols.moves.TrialCounter:      RandomMover trials=     25;  accepts= 0.9200;  energy_drop/trial=  -0.39391\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 2                                                         \n",
            "   Folding with score1 for 1000\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.461976 0.459412\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=   0.12019\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.6038;  energy_drop/trial=  -0.01063\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 3                                                         \n",
            "   Folding with score2 and score5 for 1000\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.14853 0.370033\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=  -0.00002\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5608;  energy_drop/trial=  -0.00005\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.50535 0.370033\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=  -1.97590\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.6056;  energy_drop/trial=   0.01112\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.12826 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 1.0000;  energy_drop/trial=   2.40247\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4960;  energy_drop/trial=  -0.00405\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.18473 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 1.0000;  energy_drop/trial=   0.76683\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5617;  energy_drop/trial=  -0.00100\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.25607 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.3333;  energy_drop/trial=   0.07780\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.5392;  energy_drop/trial=   0.00192\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n",
            "protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.37567 converge_rms_small: 0\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.286 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    708;  accepts= 0.5650;  energy_drop/trial=   0.00198\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.10288 -0.11931\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=   1000;  accepts= 0.5510;  energy_drop/trial=   0.00122\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.65139 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      9;  accepts= 0.5556;  energy_drop/trial=   0.02113\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    991;  accepts= 0.5792;  energy_drop/trial=   0.00664\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.29349 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.5856;  energy_drop/trial=   0.00141\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.46478 -0.11931\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.14541\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5972;  energy_drop/trial=   0.00689\n",
            "protocols.hybridization.FoldTreeHybridize: \n",
            "===================================================================\n",
            "   Stage 4                                                         \n",
            "   Folding with score3 for 200\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -1.87931 -9.55317\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1900;  energy_drop/trial=  -1.13043\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -5.90109 -8.83091\n",
            "protocols.moves.TrialCounter:  ChunkTrialMover trials=      5 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    195;  accepts= 0.2718;  energy_drop/trial=  -0.01273\n",
            "protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -8.26129 -9.0217\n",
            "protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.2400;  energy_drop/trial=  -0.00369\n",
            "protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n",
            "core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n",
            "protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n",
            "protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 1\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -43.5294 -43.5294\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=     75;  accepts= 0.2533;  energy_drop/trial=  -0.13192\n",
            "protocols.moves.TrialCounter:           picker trials=     13;  accepts= 0.0769;  energy_drop/trial=   0.15124\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     12;  accepts= 0.1667;  energy_drop/trial=  -0.02855\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -36.948 -43.5294\n",
            "protocols.moves.TrialCounter:       fragNS_15+ trials=    153;  accepts= 0.1438;  energy_drop/trial=  -0.02165\n",
            "protocols.moves.TrialCounter:           picker trials=     29;  accepts= 0.0345;  energy_drop/trial=   0.06780\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.1111;  energy_drop/trial=  -0.01904\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 2\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -47.3235 -47.3235\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     71;  accepts= 0.3239;  energy_drop/trial=   0.05148\n",
            "protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.1333;  energy_drop/trial=  -0.61622\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     14;  accepts= 0.3571;  energy_drop/trial=   0.13207\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -48.4872 -48.4872\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    134;  accepts= 0.1716;  energy_drop/trial=   0.02728\n",
            "protocols.moves.TrialCounter:           picker trials=     29;  accepts= 0.1034;  energy_drop/trial=  -0.47100\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     37;  accepts= 0.1892;  energy_drop/trial=   0.13786\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 3\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -45.1001 -45.1001\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=     72;  accepts= 0.0972;  energy_drop/trial=  -3.05758\n",
            "protocols.moves.TrialCounter:           picker trials=     10 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.0556;  energy_drop/trial=  -0.13923\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -45.3773 -45.3773\n",
            "protocols.moves.TrialCounter:         frag_15+ trials=    134;  accepts= 0.0522;  energy_drop/trial=  -1.64288\n",
            "protocols.moves.TrialCounter:           picker trials=     27 NO ACCEPTS.\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     39;  accepts= 0.0513;  energy_drop/trial=  -0.07137\n",
            "protocols.hybridization.CartesianHybridize: CYCLE 4\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n",
            "protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n",
            "protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -21.5015 -21.5015\n",
            "protocols.moves.TrialCounter:           picker trials=     50;  accepts= 0.1200;  energy_drop/trial=  -2.84499\n",
            "protocols.moves.TrialCounter:      picker_rand trials=     50;  accepts= 0.2400;  energy_drop/trial=  -1.61316\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n",
            "protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -152.951 Finite Diff= 8.85145e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 7.82984e-09 Deriv= -32.1053 Finite Diff= 1.93377e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 3.42921e-09 Deriv= -53.0553 Finite Diff= 2.39083e+08\n",
            "core.optimization.LineMinimizer: Inaccurate G in step= 4.60802e-07 Deriv= -274.127 Finite Diff= 2.33557e+06\n",
            "core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#32\n",
            "protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n",
            "core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n",
            "protocols.relax.FastRelax: CMD: repeat  14462.8  0  0  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -58.4755  0.21549  0.21549  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -46.6003  0.173689  0.173689  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -38.8011  0.259909  0.259909  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -27.2064  0.313783  0.313783  0.44\n",
            "protocols.relax.FastRelax: MRP: 0  -27.2064  -27.2064  0.313783  0.313783\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -27.2064  0.313783  0.313783  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -27.2064  0.313783  0.313783  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -57.2463  0.132128  0.132128  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -46.0239  0.245404  0.245404  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -40.4349  0.220741  0.220741  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -28.7039  0.264967  0.264967  0.44\n",
            "protocols.relax.FastRelax: MRP: 1  -28.7039  -28.7039  0.264967  0.264967\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -28.7039  0.264967  0.264967  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -28.7039  0.264967  0.264967  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -64.3545  0.170607  0.170607  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -50.779  0.155869  0.155869  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -41.0654  0.243191  0.243191  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -28.2158  0.317461  0.317461  0.44\n",
            "protocols.relax.FastRelax: MRP: 2  -28.2158  -28.7039  0.264967  0.264967\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -28.2158  0.317461  0.317461  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -28.2158  0.317461  0.317461  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -59.0865  0.150946  0.150946  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -47.2142  0.19955  0.19955  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -40.9032  0.202618  0.202618  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -29.1626  0.266374  0.266374  0.44\n",
            "protocols.relax.FastRelax: MRP: 3  -29.1626  -29.1626  0.266374  0.266374\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -29.1626  0.266374  0.266374  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -29.1626  0.266374  0.266374  0.44\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -64.5928  0.159092  0.159092  0.0088\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -50.4146  0.175882  0.175882  0.11\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -41.5076  0.254111  0.254111  0.242\n",
            "core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n",
            "protocols.relax.FastRelax: CMD: ramp_repack_min  -29.7777  0.302803  0.302803  0.44\n",
            "protocols.relax.FastRelax: MRP: 4  -29.7777  -29.7777  0.302803  0.302803\n",
            "protocols.relax.FastRelax: CMD: accept_to_best  -29.7777  0.302803  0.302803  0.44\n",
            "protocols.relax.FastRelax: CMD: endrepeat  -29.7777  0.302803  0.302803  0.44\n",
            "protocols::checkpoint: Deleting checkpoints of FastRelax\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "CPU times: user 2min 11s, sys: 1.56 s, total: 2min 13s\n",
            "Wall time: 2min 15s\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1GSWo_C0qE3M"
      },
      "source": [
        "#Part IV ‚Äì Analyze and visualize the modelled structures"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eVDe0mz3QRQi"
      },
      "source": [
        "1. Now, we need to determine our best models by analyzing their score dataframes. For this, we are going to obtain the data frame for each of the models that we generated with PyRosetta.\n",
        "\n",
        "  We can visually inspect these data frames just writing their name in a code chunk."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "sD8nrCKprPpr",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 205
        },
        "outputId": "6d52a5b4-349c-4c17-e024-85c5e23122ea"
      },
      "source": [
        "#Call here the name of the different dataframes that were created\n",
        "hdf\n"
      ],
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                   Menv_smooth  atom_pair_constraint  cart_bonded  \\\n",
              "GYPA_human__1.pdb     0.970298              5.181256   281.528695   \n",
              "GYPA_human__0.pdb    -2.893909              6.601572   152.179496   \n",
              "\n",
              "                   dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n",
              "GYPA_human__1.pdb          0.0          0.0          0.0          0.0   \n",
              "GYPA_human__0.pdb          0.0          0.0          0.0          0.0   \n",
              "\n",
              "                       fa_atr     fa_dun  fa_intra_rep  ...      fa_rep  \\\n",
              "GYPA_human__1.pdb -184.297489  51.319853    135.618316  ...  107.158779   \n",
              "GYPA_human__0.pdb -165.321403  51.243187    125.653439  ...   50.120807   \n",
              "\n",
              "                   hbond_bb_sc  hbond_lr_bb  hbond_sc  hbond_sr_bb     omega  \\\n",
              "GYPA_human__1.pdb    -0.168541          0.0       0.0   -37.152029  0.071146   \n",
              "GYPA_human__0.pdb    -0.115185          0.0       0.0   -40.748802 -2.362940   \n",
              "\n",
              "                    p_aa_pp       rama   ref  total_score  \n",
              "GYPA_human__1.pdb -3.265264  -2.756347 -4.61    64.813449  \n",
              "GYPA_human__0.pdb -2.255776 -18.282052 -4.61   -30.782040  \n",
              "\n",
              "[2 rows x 23 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-749ba0b4-6814-4b60-8a44-30bd7022ec4a\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Menv_smooth</th>\n",
              "      <th>atom_pair_constraint</th>\n",
              "      <th>cart_bonded</th>\n",
              "      <th>dslf_ca_dih</th>\n",
              "      <th>dslf_cs_ang</th>\n",
              "      <th>dslf_ss_dih</th>\n",
              "      <th>dslf_ss_dst</th>\n",
              "      <th>fa_atr</th>\n",
              "      <th>fa_dun</th>\n",
              "      <th>fa_intra_rep</th>\n",
              "      <th>...</th>\n",
              "      <th>fa_rep</th>\n",
              "      <th>hbond_bb_sc</th>\n",
              "      <th>hbond_lr_bb</th>\n",
              "      <th>hbond_sc</th>\n",
              "      <th>hbond_sr_bb</th>\n",
              "      <th>omega</th>\n",
              "      <th>p_aa_pp</th>\n",
              "      <th>rama</th>\n",
              "      <th>ref</th>\n",
              "      <th>total_score</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>GYPA_human__1.pdb</th>\n",
              "      <td>0.970298</td>\n",
              "      <td>5.181256</td>\n",
              "      <td>281.528695</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-184.297489</td>\n",
              "      <td>51.319853</td>\n",
              "      <td>135.618316</td>\n",
              "      <td>...</td>\n",
              "      <td>107.158779</td>\n",
              "      <td>-0.168541</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-37.152029</td>\n",
              "      <td>0.071146</td>\n",
              "      <td>-3.265264</td>\n",
              "      <td>-2.756347</td>\n",
              "      <td>-4.61</td>\n",
              "      <td>64.813449</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>GYPA_human__0.pdb</th>\n",
              "      <td>-2.893909</td>\n",
              "      <td>6.601572</td>\n",
              "      <td>152.179496</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-165.321403</td>\n",
              "      <td>51.243187</td>\n",
              "      <td>125.653439</td>\n",
              "      <td>...</td>\n",
              "      <td>50.120807</td>\n",
              "      <td>-0.115185</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-40.748802</td>\n",
              "      <td>-2.362940</td>\n",
              "      <td>-2.255776</td>\n",
              "      <td>-18.282052</td>\n",
              "      <td>-4.61</td>\n",
              "      <td>-30.782040</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>2 rows √ó 23 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-749ba0b4-6814-4b60-8a44-30bd7022ec4a')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-749ba0b4-6814-4b60-8a44-30bd7022ec4a button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-749ba0b4-6814-4b60-8a44-30bd7022ec4a');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 21
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "GZ1NnFdOlIT0"
      },
      "source": [
        "odf\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "AixtSW1JlJXz"
      },
      "source": [
        "mdf\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "yvfyIrqClKJL"
      },
      "source": [
        "gdf"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "gKSGvDM8Okly"
      },
      "source": [
        "2. As Rosetta uses an stochastic algorithm for comparative modelling, we are only interested in the low energy structures. Thus, we can sort our models by the column total_score as in the following cell code:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0XFEzgf4Li4f",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 205
        },
        "outputId": "3b6b97bd-29d5-49f5-fe34-034baff541a2"
      },
      "source": [
        "#hdf.sort_values('total_score')\n",
        "gdf.sort_values('total_score')"
      ],
      "execution_count": 22,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                     Menv_smooth  atom_pair_constraint  cart_bonded  \\\n",
              "GYPA_gorilla__1.pdb    -4.740364              5.218839   204.952748   \n",
              "GYPA_gorilla__0.pdb   -11.885115              4.941457   276.354830   \n",
              "\n",
              "                     dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n",
              "GYPA_gorilla__1.pdb          0.0          0.0          0.0          0.0   \n",
              "GYPA_gorilla__0.pdb          0.0          0.0          0.0          0.0   \n",
              "\n",
              "                         fa_atr     fa_dun  fa_intra_rep  ...      fa_rep  \\\n",
              "GYPA_gorilla__1.pdb -158.693431  48.134211    127.950463  ...   98.714569   \n",
              "GYPA_gorilla__0.pdb -159.573828  51.272357    125.096164  ...  144.794714   \n",
              "\n",
              "                     hbond_bb_sc  hbond_lr_bb  hbond_sc  hbond_sr_bb  \\\n",
              "GYPA_gorilla__1.pdb    -0.170032     0.000000       0.0   -37.642528   \n",
              "GYPA_gorilla__0.pdb    -0.014161    -1.151943       0.0   -40.201735   \n",
              "\n",
              "                        omega   p_aa_pp       rama   ref  total_score  \n",
              "GYPA_gorilla__1.pdb -3.874564 -8.318453 -20.433496 -4.81    25.416618  \n",
              "GYPA_gorilla__0.pdb -4.235573 -0.942424  -5.158601 -4.81    71.719294  \n",
              "\n",
              "[2 rows x 23 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-fa0919f2-fc17-46d1-974b-db47dfecdebc\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Menv_smooth</th>\n",
              "      <th>atom_pair_constraint</th>\n",
              "      <th>cart_bonded</th>\n",
              "      <th>dslf_ca_dih</th>\n",
              "      <th>dslf_cs_ang</th>\n",
              "      <th>dslf_ss_dih</th>\n",
              "      <th>dslf_ss_dst</th>\n",
              "      <th>fa_atr</th>\n",
              "      <th>fa_dun</th>\n",
              "      <th>fa_intra_rep</th>\n",
              "      <th>...</th>\n",
              "      <th>fa_rep</th>\n",
              "      <th>hbond_bb_sc</th>\n",
              "      <th>hbond_lr_bb</th>\n",
              "      <th>hbond_sc</th>\n",
              "      <th>hbond_sr_bb</th>\n",
              "      <th>omega</th>\n",
              "      <th>p_aa_pp</th>\n",
              "      <th>rama</th>\n",
              "      <th>ref</th>\n",
              "      <th>total_score</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>GYPA_gorilla__1.pdb</th>\n",
              "      <td>-4.740364</td>\n",
              "      <td>5.218839</td>\n",
              "      <td>204.952748</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-158.693431</td>\n",
              "      <td>48.134211</td>\n",
              "      <td>127.950463</td>\n",
              "      <td>...</td>\n",
              "      <td>98.714569</td>\n",
              "      <td>-0.170032</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-37.642528</td>\n",
              "      <td>-3.874564</td>\n",
              "      <td>-8.318453</td>\n",
              "      <td>-20.433496</td>\n",
              "      <td>-4.81</td>\n",
              "      <td>25.416618</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>GYPA_gorilla__0.pdb</th>\n",
              "      <td>-11.885115</td>\n",
              "      <td>4.941457</td>\n",
              "      <td>276.354830</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-159.573828</td>\n",
              "      <td>51.272357</td>\n",
              "      <td>125.096164</td>\n",
              "      <td>...</td>\n",
              "      <td>144.794714</td>\n",
              "      <td>-0.014161</td>\n",
              "      <td>-1.151943</td>\n",
              "      <td>0.0</td>\n",
              "      <td>-40.201735</td>\n",
              "      <td>-4.235573</td>\n",
              "      <td>-0.942424</td>\n",
              "      <td>-5.158601</td>\n",
              "      <td>-4.81</td>\n",
              "      <td>71.719294</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>2 rows √ó 23 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-fa0919f2-fc17-46d1-974b-db47dfecdebc')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-fa0919f2-fc17-46d1-974b-db47dfecdebc button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-fa0919f2-fc17-46d1-974b-db47dfecdebc');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 22
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jE8R-lrqRdXe"
      },
      "source": [
        "3. Having too many tables or tables with too much information would be challenging to analyze our results. For simplicity, we can make a plot with the **total_score** of each PDB for each species using **matplotlib**"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WU7m7LhCq8DG",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 516
        },
        "outputId": "8b418083-232f-4c54-e232-fadd335bcaf2"
      },
      "source": [
        "import matplotlib\n",
        "import seaborn\n",
        "frames = [hdf,odf,mdf,gdf]\n",
        "results = pd.concat(frames)\n",
        "matplotlib.rcParams['figure.figsize'] = [12.0, 8.0]\n",
        "seaborn.barplot(x=\"total_score\", y=\"index\", data=results.sort_values('total_score').reset_index())"
      ],
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<matplotlib.axes._subplots.AxesSubplot at 0x7f2116f5f610>"
            ]
          },
          "metadata": {},
          "execution_count": 23
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 864x576 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAA0YAAAHhCAYAAABKoq5nAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAAgAElEQVR4nOzdfZRlVX3u+++jHYNEBMQeRI1SogQBBUKXGIQAQRPRyBERhJYTNSfjeFVCDIYTk0NUNCFKiPEgtPZVYzDQgEKuBNAr8YrQ0IpQzVvTImJoXkyitongSwSV/t0/9izclFVdL11Vu7rW9zPGHrXWXHPO9dt7A9bjXGtVqgpJkiRJ6rLHDLoASZIkSRo0g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzlsy6AIkgCc/+ck1NDQ06DIkSZK0yK1du/Y7VbV0bLvBSAvC0NAQIyMjgy5DkiRJi1ySe8Zr91I6SZIkSZ1nMJIkSZLUeQYjSZIkSZ3nPUbqvGX/6x8GXYKkebD2jNcOugRJ0gLmipEkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeq8rSYYJdk5yflJ7kqyNsmXkqxPcnpfn13a8R2SXJXkjiS3JFmTZPe+fpckuW4w72Rmkgwlec0sz/m6JHe21+u2cK67kzy51XnbbNUoSZIkzYetIhglCXAJsLqqdq2qZcBxwErgyCR7tK5nAm+vqvvb/vFVtQ/wceCMNtcOwDJg+yS7zuN7eOwWTjEEzFowSvIk4J3AC4D9gXcm2XG25pckSZK2JltFMAIOA35cVStHG6rqnqo6CzgJWJHkZcB2VbVqnPGrgWe37aOAy4AL6YWrCSU5J8mHklzXVqIOTfKxJLcnOaev34eSjLQVrHf1td+d5PQkNwLHtP33JLm59d8vyRVJ/iXJG9uYJDkjyW1J1iU5tk33XuA32tiT2srMNUlubK8XTu8j5SXA56rqP6vqu8DngMPH+QyuSnJmO+9tSfZv7Tsl+ef2nj8KpG/YkiSr2ud0cZJtp1mbJEmSNK+2lmC0F3DjeAeq6jPAd+mtCr15gvFHAOva9nLggvZaPoVz7wgcQC+AXQq8v9XzvCT7tj6nVNUwsDdwSJK9+8b/R1XtV1UXtv17q2pf4BrgHOBo4NeB0UB1FLAvsA/wYuCMJE8B/hS4pqr2rar3A98Gfquq9gOOBT4whffS72nAfX3732ht49m21fxm4GOt7Z3AtVW1F/Ap4Bl9/XcHPlhVewDfY4LvJckbWkAc2bhx4zTLlyRJkmbP1hKMHiXJinbv0A2taQVwQ1XdMabrqiQ3AwcCJyfZGdiN3i/0XwN+kuS5k5zusqoqesHqW1W1rqo2AevpXd4G8Oq2KnQTvdC0Z9/4T4yZ79L2cx3w5ar6flVtBB5ql/kdBFxQVQ9X1beAq4Hnj1PXLwAfSbIOuGjMOWfbBQBVtRp4YqvzYOC81v5peuF01H1VtaZtn0fvPf2cqvpwVQ1X1fDSpUvnrHhJkiRpMltLMFoP7De6U1UnAC8CRn+b3tReYx3fVliOrKr7gFfTWwHakORuesFmslWjh/rO8VBf+yZ6l4w9EzgZeFFV7Q18Gtimr98PpzPfJLX0Own4Fr2VpWHgcdMYC/CvwNP79n+ltY2nJtnf0v6SJEnSQG0twehKYJskb+prm8l9K8uBw6tqqKqG6D2EYbP3GU3BE+mFnwfaitRLt3C+a4Bjkzw2yVJ6KzPXA98Htuvrtz3w72316neB6T7c4Qrgt5Ps2B668NutbTzHAiQ5CHigqh6gd9/Wa1r7S+kFzlHPSHJA234NcO00a5MkSZLm1VYRjNqlbEfSu39nQ5Lr6d1T9LapzpFkCNgFeOQx3VW1gV6gecEW1HYLvUvovgqcD6zZ/IhJfQq4FbiFXiD8k6r6Zmt7uF1CeBLwQeB1SW4BnsPPr0xNVvd/An8B3NBe725tJPlokuG+7g8muYneUwB/v7W9Czg4yXp690Xd29f/DuCEJLfTC0wfmk5tkiRJ0nxLL3NI40tyFXByVY3M5XmGh4drZGROTzGhZf/rHwZyXknza+0Zrx10CZKkBSDJ2vbgtEfZKlaMJEmSJGkuTedm/0UrySnAMWOaL6qq0wZRz0wleR5w7pjmp/Pox3IDPFRVU7p8sKoOnYXSJEmSpAXNYAS0ALRVhaDxVNU6en8DSZIkSdI0eCmdJEmSpM4zGEmSJEnqPIORJEmSpM4zGEmSJEnqPIORJEmSpM4zGEmSJEnqPIORJEmSpM4zGEmSJEnqPIORJEmSpM4zGEmSJEnqPIORJEmSpM4zGEmSJEnqPIORJEmSpM5bMugCpEFbe8ZrB12CJEmSBswVI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HlLBl2ApJm5993PG3QJ0lblGe9YN+gSJEkLmCtGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8wxGkiRJkjrPYCRJkiSp8+Y0GCXZOcn5Se5KsjbJl5KsT3J6X59d2vEdklyV5I4ktyRZk2T3vn6XJLluLutt53lqkovb9r5JXjYLcw4n+cAMx56T5OgtrWGCuQ9OcmOSn27pOZIMJbmtbb8+ydmzU6UkSZI09+YsGCUJcAmwuqp2raplwHHASuDIJHu0rmcCb6+q+9v+8VW1D/Bx4Iw21w7AMmD7JLvOQm1LJjpWVf9WVaMhYV9gWsFo7NxJllTVSFX94fQrnXP3Aq8Hzh9wHZIkSdJAzeWK0WHAj6tq5WhDVd1TVWcBJwEr2mrMdlW1apzxq4Fnt+2jgMuAC+mFqwm1lYsrk9ya5PNJntHaz0myMsmXgb9u+x9I8sW2YnV03/jbkjwOeDdwbJKbkxyb5Elt5erWJNcl2buNOTXJuUnWAOeOs39oksv7+n6srY7dleQP+2p/bZv7liTn9r2tg8ep85E52/7ZSV7ftt+b5Cttrr+Z6LOqqrur6lZg0ySf6Q+SvL+t9n0+ydLWvqzVegtwwphhT2/v8c4k79zc/JIkSdKgzWUw2gu4cbwDVfUZ4Lv0VoXePMH4I4B1bXs5cEF7LZ/kvGcBH6+qvYFVQP8lbL8CvLCq3tr2nwIcBLwceO+YGn8MvAP4RFXtW1WfAN4F3NTm/t/AP/QN2RN4cVUtn2C/33OAlwD7A+9M8gtJ9gL+HDisrZi9pa//hHWOlWQn4JXAXq3Ov9xc/yn6JWCkqvYCrgZGg87fAye2esfaH3gVsDdwTJLhcWp9Q5KRJCMbN26chTIlSZKkmZm3hy8kWdFWF25oTSuAG6rqjjFdVyW5GTgQODnJzsBuwLVV9TXgJ0meu5lTHcDPLg07l16gGHVRVT3ct39JVW2qqq8AO0/hbRzU5qSqrgR2SvLEduzSqvpRX9+x+/0+XVUPVdV3gG+3cx/W6vtOm/8/Z1jnA8CDwN8lOQr4rym8r8lsAj7Rts8DDmqXN+5QVatb+7ljxnyuqv6jfQb/D4/+HgCoqg9X1XBVDS9dunQWypQkSZJmZi6D0Xpgv9GdqjoBeBEw+hvwJsa/hOv4tkJzZFXdB7wa2BHYkORuYIjJV40m8sMx+w/1bWeGc04099j9ic77MDDhPU/j9B+t86c8+vvbBqCqfkpvteZieitMn51k7pmoGfSZyhhJkiRpIOYyGF0JbJPkTX1t285gnuXA4VU1VFVD9B7CsLn7jL7Yd/x44JoZnHPU94Ht+vavaXOS5FDgO1X1vS2Yv9+V9C4526nN/6RJ+t8D7JnkF9vqzYvauCcA27fLFU8CxrvMbboeA4w+kOI19Fbv7gfuTzK6EnT8mDG/1e7JejxwJLBmFuqQJEmS5sScBaOqKnq/EB+SZEOS6+ndU/S2qc6RZAjYBXjkMd1VtQF4IMkLJhh2IvB7SW4FfpdH36szXV+gFz5uTnIscCqwrM39XuB1WzD3o1TVeuA04Or2MIO/naT/fcAngdvaz5vaoe2Ay1uN1wJvHX8GSPL8JN8AjgH+7yTr+47d3Nf1h8D+7XHch9F7KAXA79F7iMbN/PyK2/XAPwK3Av9YVSObez+SJEnSIKWXX6SJJflBVT1hLs8xPDxcIyNmp+m4993PG3QJ0lblGe9YN3knSdKil2RtVf3cg8Hm7eELkiRJkrRQTXbT/4KV5BR6l4D1u6iqThtEPQvZln5Wc71aJEmSJA3aVhuM2i/1hqAp8LOSJEmSNs9L6SRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUuctGXQBkmbmGe9YN+gSJEmSFg1XjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUuctGXQBknTgWQcOugR1wJoT1wy6BEnSAuaKkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6rxFGYyS7Jzk/CR3JVmb5EtJ1ic5va/PLu34DkmuSnJHkluSrEmye1+/S5JcN4VznpPk6Ll6T3MhyZOSfC7Jne3njlsw1+uTnN22t7rPQpIkSd226IJRkgCXAKurateqWgYcB6wEjkyyR+t6JvD2qrq/7R9fVfsAHwfOaHPtACwDtk+y63y+j3nyp8Dnq2o34PNtX5IkSeqcRReMgMOAH1fVytGGqrqnqs4CTgJWJHkZsF1VrRpn/Grg2W37KOAy4EJ64WoyByf5YluJOhogyaFJLh/tkOTsJK9v23cneU+Sm5OMJNkvyRVJ/iXJG1ufJyT5fJIbk6xL8orWPpTk9iQfaath/5zk8dP7qHgFvSBI+3nk2A6t/tVJPt1W1VYmeUw79ntJvpbkeuDAMUNf3N7T15K8fJp1SZIkSfNqMQajvYAbxztQVZ8BvksvBLx5gvFHAOva9nLggvZaPoVzPwU4CHg58N4p1ntvVe0LXAOcAxwN/Drwrnb8QeCVVbUf8JvA+9qqGMBuwIqq2gu4H3jVFM85aueq+ve2/U1g5wn67Q+cCOwJPAs4KslTWo0H0nvPe44ZM9TG/Q6wMsk2YydN8oYWnkY2btw4zdIlSZKk2bNk0AXMtSQr6P3i/uOqej6wAnh8Vd0xpuuqJD8C7gZOTLIzveBxbVVVkp8keW5V3baZ011SVZuAr7TxU3Fp+7kOeEJVfR/4fpKH2qV8PwT+KsnBwCbgafwswGyoqpvb9lp6YWRG2nusCQ5fX1V3ASS5gN7n+VPgqqra2No/Afxq35hPts/iziR3Ac8Bbu6ftKo+DHwYYHh4eKJzS5IkSXNuMa4YrQf2G92pqhOAFwFLW9Om9hrr+Krat6qOrKr7gFcDOwIbktxNL3RMtmr0UN/26KrOT3n05zx25WR0zKYx4zfRC67Ht9qXtZWlb/XN0d//YaYfdL/VVn5oP789Qb+xoWUqIWYmYyRJkqSBWIzB6EpgmyRv6mvbdgbzLAcOr6qhqhqi9xCGqdxnNNY9wJ5JfrGtAL1omuO3B75dVT9J8pvALjOoYSKXAq9r268D/mmCfvsneWa7t+hY4Frgy8AhSXZK8gvAMWPGHJPkMUmeBewKjF2hkyRJkhaMRReMqqroPUTgkCQb2oMBPg68bapzJBmiF0AeeUx3VW0AHkjygmnWcx/wSeC29vOm6YwHVgHDSdYBrwW+Os3xm/Ne4LeS3Am8uO2TZDjJR/v63QCcDdwObAA+1e5NOhX4ErCmHet3L3A98P8Cb6yqB2exbkmSJGlWpZcjpPElORQ4uarm9Mlyw8PDNTIyMpen0AJ24FljH2oozb41J64ZdAmSpAUgydqqGh7bvuhWjCRJkiRpuhb9U+lmW5JT+Pn7aS6qqtMGUc9Y7Sl8Y//v992AO8e0nVlVfz/ZfFV1FXDVrBQnSZIkLVAGo2lqAWhBhKDxtKfwSZIkSZoGL6WTJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HlLBl2AJK05cc2gS5AkSR3nipEkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeo8g5EkSZKkzjMYSZIkSeq8JYMuQJImc/XBhwy6BC0Ch6y+etAlSJIWMFeMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHWewUiSJElS5xmMJEmSJHXeVhOMkuyc5PwkdyVZm+RLSdYnOb2vzy7t+A5JrkpyR5JbkqxJsntfv0uSXDeYdzIzSYaSvGaW5/xskvuTXD4Lc12VZLht/2DLq5MkSZLmz1YRjJIEuARYXVW7VtUy4DhgJXBkkj1a1zOBt1fV/W3/+KraB/g4cEabawdgGbB9kl3n8T08dgunGAJmNRjR+0x+d5bnlCRJkrY6W0UwAg4DflxVK0cbquqeqjoLOAlYkeRlwHZVtWqc8auBZ7fto4DLgAvphasJJTknyYeSXNdWog5N8rEktyc5p6/fh5KMtBWsd/W1353k9CQ3Ase0/fckubn13y/JFUn+Jckb25gkOSPJbUnWJTm2Tfde4Dfa2JPaCtI1SW5srxdO7yOFqvo88P0pfAYrW71fS/Ly1v74JBe2z+JTwOPHjHt/+zw+n2TpdGuTJEmS5tPWEoz2Am4c70BVfQb4Lr1VoTdPMP4IYF3bXg5c0F7Lp3DuHYED6AWwS4H3t3qel2Tf1ueUqhoG9gYOSbJ33/j/qKr9qurCtn9vVe0LXAOcAxwN/DowGqiOAvYF9gFeDJyR5CnAnwLXVNW+VfV+4NvAb1XVfsCxwAem8F5magjYH/gdYGWSbYA3Af9VVXsA76S3Cjfql4CRqtoLuLod/zlJ3tAC18jGjRvnsHxJkiRp87aWYPQoSVa0e4duaE0rgBuq6o4xXVcluRk4EDg5yc7AbsC1VfU14CdJnjvJ6S6rqqIXrL5VVeuqahOwnl5gAHh1WxW6iV5o2rNv/CfGzHdp+7kO+HJVfb+qNgIPtcv8DgIuqKqHq+pb9ILF88ep6xeAjyRZB1w05pyz7ZNVtamq7gTuAp4DHAycB1BVtwK39vXfxM/e93n03tPPqaoPV9VwVQ0vXeqikiRJkgZnyaALmKL1wKtGd6rqhCRPBkZa06b2Guv4qhrtQ5IT6a0AbejdtsQT6a0anbKZcz/Ud46H+to3AUuSPBM4GXh+VX23XWK3TV+/H05nvs3UMdZJwLforSw9BnhwGmOnqybZn+54SZIkaUHZWlaMrgS2SfKmvrZtZzDPcuDwqhqqqiF6l39t9j6jKXgivfDzQFuReukWzncNcGySx7Z7cw4Grqd3L9B2ff22B/69rV79LrClD3fYnGOSPCbJs4BdgTvo3bf1GoC26tZ/+eBj6F0iSOtz7RzWJkmSJG2xrWLFqKoqyZHA+5P8CbCRXhh521TnSDIE7AI88pjuqtqQ5IEkL6iqL8+wtluS3AR8FbgPWDOTefp8it49TbfQW2n5k6r6ZpL/AB5Ocgu9e5M+CPxjktcCn+XnV6YmleQaepfFPSHJN4Dfr6orkryb3j1Co5f93UsvnD0ReGNVPZjkQ8DfJ7kduB1Y2zf1D4H9k/w5vXuhjkWSJElawNK7fUYaX7s08PKqunguzzM8PFwjIyOTd1QnXX3wIYMuQYvAIauvHnQJkqQFIMna9uC0R9laLqWTJEmSpDmzVVxKN9eSnAIcM6b5oqo6bRD1zFSS5wHnjml+Or1L/Po9VFUvmMqcVfX6WShNkiRJWtAMRkALQFtVCBpPVa2j9zeQJEmSJE2Dl9JJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOm3EwSvK42SxEkiRJkgZlSsEoyVVJhvr29wdumKOaJEmSJGleTfUPvL4H+GySDwBPA14K/N6cVSVJkiRJ82hKwaiqrkjyRuBzwHeAX6uqb85pZZIkSZI0T6Z6Kd3bgbOAg4FTgauS/M4c1iVJkiRJ82aql9LtBOxfVT8CvpTks8BHgU/PWWWSJEmSNE+mtGJUVX8EkGT3tn9PVf3WXBYmSZIkSfNlSitGSY4A/gZ4HPDMJPsC766q/zaXxUkSwCGrrx50CZIkaZGb6t8xOhXYH7gfoKpuBnado5okSZIkaV5NNRj9pKoeGNO2abaLkSRJkqRBmOrDF9YneQ3w2CS7AX8IfHHuypIkSZKk+TPVFaMTgb2Ah4ALgO8BfzRXRUmSJEnSfJrqH3j9L+CU9pIkSZKkRWWzwSjJZUBNdNyn0kmSJElaDCZbMfqb9vMo4JeB89r+cuBbc1WUJEmSJM2nzQajqroaIMn7qmq479BlSUbmtDJJkiRJmidTffjCLyV55O8WJXkm8EtzU5IkSZIkza+pPq77JOCqJHcBAXYB/q85q0qSJEmS5tFUn0r32fb3i57Tmr5aVQ/NXVmStHCd/ceXDboEzcAfvO+IQZcgSVrAprpiBLAMGGpj9klCVf3DnFQlSZIkSfNoSsEoybnAs4CbgYdbcwEGI0mSJElbvamuGA0De1bVhH/TSJIkSZK2VlN9Kt1t9P6OkSRJkiQtOlNdMXoy8JUk1wOPPHShqv7bnFQlSZIkSfNoqsHo1LksQpIkSZIGaaqP6756rguRJEmSpEHZbDBKcm1VHZTk+/SeQvfIIaCq6olzWp0kSZIkzYPNBqOqOqj93G5+ypEkSZKk+TfVp9JJkiRJ0qJlMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ03p8Eoyc5Jzk9yV5K1Sb6UZH2S0/v67NKO75DkqiR3JLklyZoku/f1uyTJdXNZbzvPU5Nc3Lb3TfKyWZhzOMkHZjj2nCRHb2kNE8z9i0k+keTrSb6cZGgL5jo0yeVt+9QkJ89WnZIkSdJcm7NglCTAJcDqqtq1qpYBxwErgSOT7NG6ngm8varub/vHV9U+wMeBM9pcOwDLgO2T7DoLtS2Z6FhV/VtVjQaRfYFpBaOxcydZUlUjVfWH0690zv0+8N2qejbwfuD0SfpLkiRJi9JcrhgdBvy4qlaONlTVPVV1FnASsKKtxmxXVavGGb8aeHbbPgq4DLiQXriaUJKhJFcmuTXJ55M8o7Wfk2Rlki8Df932P5Dki23F6ui+8bcleRzwbuDYJDcnOTbJk9rK1a1JrkuydxtzapJzk6wBzh1nf+xqysfa6thdSf6wr/bXtrlvSXJu39s6eJw6H5mz7Z+d5PVt+71JvtLm+pvNfFyvoBdAAS4GXtQC7djP86tJViW5PcnFSbZtxw5vx25s31G/fdoK4Z1J/ufmvjNJkiRp0OYyGO0F3Djegar6DPBder+Uv3mC8UcA69r2cuCC9lo+yXnPAj5eVXsDq4D+S9h+BXhhVb217T8FOAh4OfDeMTX+GHgH8Imq2reqPgG8C7ipzf2/gX/oG7In8OKqWj7Bfr/nAC8B9gfemeQXkuwF/DlwWFsxe0tf/wnrHCvJTsArgb1anX+5me5PA+5r7/enwAPATuP02x34YFXtAXwPeHOSbYCP0PuelgG/PGbM3vTC8QHAO5I8dZxa35BkJMnIxo0bN/e2JEmSpDk1bw9fSLKirYTc0JpWADdU1R1juq5KcjNwIHBykp2B3YBrq+prwE+SPHczpzoAOL9tn0svUIy6qKoe7tu/pKo2VdVXgJ2n8DYOanNSVVcCOyV5Yjt2aVX9qK/v2P1+n66qh6rqO8C327kPa/V9p83/nzOs8wHgQeDvkhwF/NcU3tdk7quqNW37PHqfw3OADVV1Z1VVa+/3T1X1o/Z+vkAvBD5KVX24qoaranjp0qWzUKYkSZI0M3MZjNYD+43uVNUJwIuA0d+AN7XXWMe3FZojq+o+4NXAjsCGJHcDQ0y+ajSRH47Zf6hvO2yZsXOP3Z/ovA8DE97zNE7/0Tp/yqO/v23gkZWf/eldGvdy4LObmfdfgafDI/dGbQ/8xzj9apL98cxkjCRJkjQQcxmMrgS2SfKmvrZtZzDPcuDwqhqqqiF6l21t7j6jL/YdPx64ZgbnHPV9YLu+/WvanCQ5FPhOVX1vC+bvdyVwTLsUjiRPmqT/PcCe7clyO9ALnSR5ArB9u1zxJGCfzcxxKfC6tn00cGVb/RnrGUkOaNuvAa4FvgoMJXlWax8bVl+RZJv2fg4FbkCSJElaoOYsGLVfsI8EDkmyIcn19O4pettU52iPj94FeOQx3VW1AXggyQsmGHYi8HtJbgV+l0ffqzNdX6AXPm5OcixwKrCszf1efhYqtlhVrQdOA65Ocgvwt5P0vw/4JHBb+3lTO7QdcHmr8VrgrePPAMDf0bsc8Out35/CI48s/0xfvzuAE5LcTkGLne4AABkdSURBVG/17kNV9SDwBuDT7eEL3x4z9630Pr/rgL+oqn/b3PuRJEmSBinjLxBIPS2cXl5Vm7uva4sNDw/XyMjIXJ5CmjVn//Flgy5BM/AH7zti0CVIkhaAJGuranhs+7w9fEGSJEmSFqrJbvpfsJKcAhwzpvmiqjptEPUsZFvyWVXV3cCcrhZJkiRJg7bVBqP2S70haAr8rCRJkqTN81I6SZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeQYjSZIkSZ1nMJIkSZLUeUsGXYAkbW3+4H1HDLoESZI0y1wxkiRJktR5BiNJkiRJnWcwkiRJktR5BiNJkiRJnWcwkiRJktR5BiNJkiRJnWcwkiRJktR5BiNJkiRJnWcwkiRJktR5BiNJkiRJnbdk0AVIkjQfTvvvRw+6BA3IKeddPOgSJG0FXDGSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdt+iDUZKdk5yf5K4ka5N8Kcn6JKf39dmlHd8hyVVJ7khyS5I1SXbv63dJkuvmqe4vtp9DSW5r24cmuXyW5t8pyReS/CDJ2bMw391JntxfryRJkrS1WNTBKEmAS4DVVbVrVS0DjgNWAkcm2aN1PRN4e1Xd3/aPr6p9gI8DZ7S5dgCWAdsn2XUOa14CUFUvnKtzNA8CbwdOnuPzSJIkSQveog5GwGHAj6tq5WhDVd1TVWcBJwErkrwM2K6qVo0zfjXw7LZ9FHAZcCG9cDWhJM9Kcl2SdUn+MskPWnuSnJHktnbs2NZ+aJJrklwKfKW1/WCSc+zfVr9uSvLF/pWtqaiqH1bVtfQC0ubOc1WSM5Pc3Orev7XvlOSf2+rbR4H0DVuSZFWS25NcnGTb6dQmSZIkzbfFHoz2Am4c70BVfQb4Lr1VoTdPMP4IYF3bXg5c0F7LJznvmcCZVfU84Bt97UcB+wL7AC8GzkjylHZsP+AtVfWrk8w96qvAb1TVrwHvAP5qiuNmYtuq2pfe5/Sx1vZO4Nqq2gv4FPCMvv67Ax+sqj2A7zHB55vkDUlGkoxs3Lhx7qqXJEmSJrHYg9GjJFnR7h26oTWtAG6oqjvGdF2V5GbgQODkJDsDu9ELAl8DfpLkuZs51QHARW37/L72g4ALqurhqvoWcDXw/Hbs+qraMI23sz1wUbuf5/30QuBcuQCgqlYDT2yXFR4MnNfaP00vZI66r6rWtO3z6L3vn1NVH66q4aoaXrp06ZwVL0mSJE1msQej9fRWYgCoqhOAFwGjv4Vvaq+xjq+qfavqyKq6D3g1sCOwIcndwBCTrxpN1w+n2f8vgC9U1XPprWxtM8v19KtJ9re0vyRJkjRQiz0YXQlsk+RNfW0zud9lOXB4VQ1V1RC9hzBs7j6j64BXte3+ftcAxyZ5bJKl9FZdrp9BPdBbMfrXtv36Gc4xVaP3Qh0EPFBVD9C7/+o1rf2l9ILjqGckOaBtvwa4do7rkyRJkrbIog5GVVXAkcAhSTYkuZ7ePUVvm+ocSYaAXeiFndF5NwAPJHnBBMP+CHhrklvpPbzhgdb+KeBW4BZ6oe1Pquqb03lPff4aeE+Sm4AlM5mgrX79LfD6JN9Ismdr/2iS4b6uD7bzrAR+v7W9Czg4yXp6907d29f/DuCEJLfTC0wfmkl9kiRJ0nxJLztoNrWnsP2oqirJccDyqnrFoOuaiSRXASdX1chcnmd4eLhGRub0FJI67rT/fvSgS9CAnHLexYMuQdICkmRtVQ2PbZ/RSoMmtQw4u/0dpfuB/zHgeiRJkiRthsFoCyQ5BThmTPNFVXUavUdyz3c9LwFOH9O8C3DPmLYNVfXKqcxZVYfOQmmSJEnSgmYw2gItAJ026DpGVdUVwBWDrkOSJEna2izqhy9IkiRJ0lQYjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUuctGXQBkiTNh1POu3jQJUiSFjBXjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUuctGXQBkiTNh9tPu3LQJUjSwO1xymGDLmHBcsVIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUucZjCRJkiR1nsFIkiRJUuctymCUZOck5ye5K8naJF9Ksj7J6X19dmnHd0hyVZI7ktySZE2S3fv6XZLkuimc85wkR8/Ve5oLSY5pn8umJMNbONehSS5v26cmOXl2qpQkSZLm3qILRkkCXAKsrqpdq2oZcBywEjgyyR6t65nA26vq/rZ/fFXtA3wcOKPNtQOwDNg+ya7z+T7myW3AUcDqQRciSZIkDdKiC0bAYcCPq2rlaENV3VNVZwEnASuSvAzYrqpWjTN+NfDstn0UcBlwIb1wNZmDk3yxrUQdDY9eSWn7Zyd5fdu+O8l7ktycZCTJfkmuSPIvSd7Y+jwhyeeT3JhkXZJXtPahJLcn+Uhb9fnnJI+fzgdVVbdX1R2b69PO89Ukq9r5Lk6ybTt2eDt2Y/us+u3TVuruTPI/p1OXJEmSNN8WYzDaC7hxvANV9Rngu/RWhd48wfgjgHVtezlwQXstn8K5nwIcBLwceO8U6723qvYFrgHOAY4Gfh14Vzv+IPDKqtoP+E3gfW1VDGA3YEVV7QXcD7xqiuecrt2BD1bVHsD3gDcn2Qb4CL3Paxnwy2PG7E0vpB4AvCPJU8dOmuQNLRCObNy4cY5KlyRJkia3GIPRoyRZ0e4duqE1rQBuGGelZFWSm4EDgZOT7EwveFxbVV8DfpLkuZOc7pKq2lRVXwF2nmKJl7af64AvV9X3q2oj8FC7lC/AXyW5Ffj/gKf1zb2hqm5u22uBoSmec7ruq6o1bfs8euHvOe38d1ZVtfZ+/1RVP6qq7wBfAPYfO2lVfbiqhqtqeOnSpXNUuiRJkjS5JYMuYA6sp2/lpKpOSPJkYKQ1bWqvsY6vqtE+JDkR2BHY0BZonkhv1eiUzZz7ob7t0VWdn/LoALrNBGM2jRm/id73czywFFhWVT9JcnffHP39HwamdSndNNQk+7M1RpIkSRqIxbhidCWwTZI39bVtO4N5lgOHV9VQVQ3Ru1xsKvcZjXUPsGeSX2wrQC+a5vjtgW+3UPSbwC4zqGFLPSPJAW37NcC1wFeBoSTPau1jLzV8RZJtkuwEHArcgCRJkrRALbpg1C7rOhI4JMmGJNfTu6fobVOdI8kQvQDyyGO6q2oD8ECSF0yznvuAT9J7AtwngZumMx5YBQwnWQe8ll4gmRVJXpnkG/TuA/p0kita+1OTfKav6x3ACUlup7eK9qGqehB4Qxt3I/DtMdPfSu8SuuuAv6iqf5utuiVJkqTZll6OkMbXQuLlVTXZ/VVbZHh4uEZGRibvKEkzdPtpVw66BEkauD1OOWzQJQxckrVV9XN/w3PRrRhJkiRJ0nQtxocvzKkkpwDHjGm+qKpOG0Q9YyVZQe/Jev12A+4c03ZmVf39ZPNV1d3AnK4WSZIkSYNmMJqmFoAWRAgaT1WdMOgaJEmSpK2Nl9JJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6rwlgy5AkqT5sMcphw26BEnSAuaKkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6rwlgy5AkqT5cOqppw66BElSsxD/m+yKkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6jyDkSRJkqTOMxhJkiRJ6rxOBKMkOyc5P8ldSdYm+VKS9UlO7+uzSzu+Q5KrktyR5JYka5Ls3tfvkiTXzVPdX2w/h5Lc1rYPTXL5LJ7jz5J8vb3fl2zhXD9oP2e1RkmSJGmuLfpglCTAJcDqqtq1qpYBxwErgSOT7NG6ngm8varub/vHV9U+wMeBM9pcOwDLgO2T7DqHNS8BqKoXztU52nn2pPdZ7AUcDnwwyWPn8pySJEnSQrTogxFwGPDjqlo52lBV91TVWcBJwIokLwO2q6pV44xfDTy7bR8FXAZcSC9QTCjJs5Jcl2Rdkr/sW01JkjOS3NaOHdvaD01yTZJLga+0th9Mco792+rXTUm+2L+yNUWvAC6sqoeqagPwdWD/cc5zd5K/bvVen+TZrf2Z7fzrkvzlmGFPTPLpthK1MkkX/lmTJEnSVqoLv6zuBdw43oGq+gzwXXqrQm+eYPwRwLq2vRy4oL2WT3LeM4Ezq+p5wDf62o8C9gX2AV4MnJHkKe3YfsBbqupXJ5l71FeB36iqXwPeAfzVFMeNehpwX9/+N1rbeB5o7+Vs4P+0tjOBD7X2fx/Tf3/gRGBP4Fn03vejJHlDkpEkIxs3bpxm6ZIkSdLs6UIwepQkK9q9Qze0phXADVV1x5iuq5LcDBwInJxkZ2A34Nqq+hrwkyTP3cypDgAuatvn97UfBFxQVQ9X1beAq4Hnt2PXt5WbqdoeuKjdf/R+eiFwrlzQ9/OAtn1gX/u5Y/pfX1V3VdXDrc9BYyesqg9X1XBVDS9dunQuapYkSZKmpAvBaD29lRgAquoE4EXA6G/im9prrOOrat+qOrKq7gNeDewIbEhyNzDE5KtG0/XDafb/C+ALVfVceitb20xz/L8CT+/b/5XWNp6awvZE/TfXT5IkSRq4LgSjK4Ftkrypr23bGcyzHDi8qoaqaojeQxg2d5/RdcCr2nZ/v2uAY5M8NslS4GDg+hnUA70Vo9Eg8/oZjL8UOC7JLyZ5Jr0VsYlqObbv55fa9hp+9t6OH9N//3YP0mPamGtnUJ8kSZI0LxZ9MKqqAo4EDkmyIcn19O4pettU50gyBOxCL+yMzrsBeCDJCyYY9kfAW5PcSu/hDQ+09k8BtwK30Attf1JV35zOe+rz18B7ktwELJnu4KpaD3yS3sMePguc0C59I8lnkjy1r/uO7b28hd5DK2jbJyRZx8/fm3QDvfuRbgc20HvfkiRJ0oKUXm7QbEuyLfCjqqokxwHLq+oVg65rJtqlg8NV9Z25Osfw8HCNjIzM1fSSxKmnnjroEiRJzSD/m5xkbVUNj22f9iqDpmwZcHb7O0r3A/9jwPVIkiRJmoDBaAslOQU4ZkzzRVV1Gr1Hcs93PS8BTh/TvAtwz5i2DVX1yqnM2e6pkiRJkhYtg9EWagHotEHXMaqqrgCuGHQdkiRJ0tZk0T98QZIkSZImYzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HlLBl2AJEnz4dRTTx10CZKkBcwVI0mSJEmdZzCSJEmS1HkGI0mSJEmdZzCSJEmS1HkGI0mSJEmdl6oadA0SSTYC9wy6jkXgycB3Bl2EHuH3sbD4fSwsfh8Lh9/FwuL3Mfd2qaqlYxsNRtIikmSkqoYHXYd6/D4WFr+PhcXvY+Hwu1hY/D4Gx0vpJEmSJHWewUiSJElS5xmMpMXlw4MuQI/i97Gw+H0sLH4fC4ffxcLi9zEg3mMkSZIkqfNcMZIkSZLUeQYjaRFJ8sdJKsmT236SfCDJ15PcmmS/QdfYBUnOSPLV9pl/KskOfcf+rH0fdyR5ySDr7Iokh7fP++tJ/nTQ9XRNkqcn+UKSryRZn+Qtrf1JST6X5M72c8dB19oVSR6b5KYkl7f9Zyb5cvt35BNJHjfoGrsiyQ5JLm7/m3F7kgP8d2NwDEbSIpHk6cBvA/f2Nb8U2K293gB8aAClddHngOdW1d7A14A/A0iyJ3AcsBdwOPDBJI8dWJUd0D7fFfT+XdgTWN6+B82fn/7/7d1bjFXVHcfx709QjNpKxGsczXhBSdWixqLiJQ2aaJAwahWJRFF5KKkSJW1slVeNlyZaL2n64A0NSgQ1oPFSC40xbQEjMlzqQw2kEQNYq4ipsYbh14e9DMeJyGiHs2dm/z7Jyey99jr7/M9ZWXPO/6y11wF+aftHwFnAjaUNfgMssT0aWFL2oz1uBt5t2b8HuN/28cAnwIxaomqmB4BXbY8BxlK1S/pGTZIYRQwd9wO3Aq0XDnYBT7qyDBgp6YhaomsQ23+0vb3sLgM6ynYXMN/2f21vAN4DxtURY4OMA96zvd72l8B8qnaINrG9yfbKsv0Z1Qe/I6naYW6pNhe4tJ4Im0VSB3AJ8EjZFzABWFiqpC3aRNKBwPnAowC2v7S9lfSN2iQxihgCJHUBH9ju7nXoSOD9lv2NpSza5wbglbKd9mi/vOYDiKRO4DRgOXCY7U3l0GbgsJrCaprfUX2JtqPsjwK2tnyZkz7SPscA/wIeL1MbH5G0P+kbtRledwAR0TeS/gQc/g2H5gC3U02jizb5tvawvajUmUM1jWheO2OLGIgkHQA8B9xie1s1UFGxbUlZJncPkzQJ+ND225J+Wnc8wXDgdGCW7eWSHqDXtLn0jfZKYhQxSNi+8JvKJZ1C9a1Td/mg0QGslDQO+AA4qqV6RymL/9Ou2uMrkq4DJgEXeOfvIqQ92i+v+QAgaW+qpGie7edL8RZJR9jeVKb4flhfhI1xDjBZ0kRgX+CHVNe4jJQ0vIwapY+0z0Zgo+3lZX8hVWKUvlGTTKWLGORsr7F9qO1O251U/2hPt70ZWAxcW1anOwv4tGV4PvYQSRdTTVWZbPvzlkOLgamSRkg6hmpRjBV1xNggbwGjy6pb+1AtfrG45pgapVzD8ijwru37Wg4tBqaX7enAonbH1jS2b7PdUd4rpgJLbU8D/gxcUaqlLdqkvE+/L+nEUnQB8HfSN2qTEaOIoe1lYCLVRf6fA9fXG05jPAyMAF4vo3jLbM+0vU7Ss1RvfNuBG2331BjnkGd7u6SbgNeAYcBjttfVHFbTnANcA6yRtKqU3Q7cDTwraQbwT2BKTfEF/BqYL+kO4B3KYgDRFrOAeeWLm/VU79N7kb5RC+2c4REREREREdFMmUoXERERERGNl8QoIiIiIiIaL4lRREREREQ0XhKjiIiIiIhovCRGERERERHReEmMIiIiIiKi8ZIYRURE9CNJIyX9Yjd1OiVd3YdzdUpa23/RRUTEriQxioiI6F8jgW9NjIBOYLeJUbtJyg+/R0RjJTGKiIjoX3cDx0laJem35bZW0hpJV7XUOa/UmV1Ght6UtLLcxvflgSSdJGlFOc9qSaNL+bVlv1vSU6WsU9LSUr5E0tGl/AlJf5C0HLhX0nGSXpX0dolpTP+/RBERA49s1x1DRETEkCGpE3jJ9smSfgbMBC4GDgbeAs4ETgR+ZXtSuc9+wA7bX5Tk5hnbZ7SeaxeP9RCwzPY8SfsAw4BjgReA8bY/knSQ7Y8lvQgstD1X0g3AZNuXSnqixNZlu0fSEmCm7X9IOhO4y/aEPfBSRUQMKBkyj4iI2HPOpUpyeoAtkt4AfgJs61Vvb+BhSacCPcAJfTz/34A5kjqA50syMwFYYPsjANsfl7pnA5eX7aeAe1vOs6AkRQcA44EFkr46NqKPsUREDGpJjCIiIuo3G9gCjKWa5v5FX+5k++kyBe4S4GVJP/+ej/+f8ncvYKvtU7/neSIiBq1cYxQREdG/PgN+ULbfBK6SNEzSIcD5wIpedQAOBDbZ3gFcQzUlbrckHQust/0gsAj4MbAUuFLSqFLnoFL9r8DUsj2txPY1trcBGyRdWe4rSWP79KwjIga5JEYRERH9yPa/gb+UZbbPBlYD3VQJy622N5eynrI4wmzg98B0Sd3AGHaO4OzOFGCtpFXAycCTttcBdwJvlPPdV+rOAq6XtJoq+bp5F+ecBswo910HdH2Hpx8RMWhl8YWIiIiIiGi8jBhFRERERETjZfGFiIiIAU7SRcA9vYo32L6sjngiIoaiTKWLiIiIiIjGy1S6iIiIiIhovCRGERERERHReEmMIiIiIiKi8ZIYRURERERE4yUxioiIiIiIxvsfCX1QgpaE2GYAAAAASUVORK5CYII=\n"
          },
          "metadata": {
            "needs_background": "light"
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Tc7oifPnbssg"
      },
      "source": [
        "4. Finally, we can take a look at our best models using **py3Dmol**"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "LPpEhymrJ5gM",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 851
        },
        "outputId": "8d8347cd-80b3-4da0-99e0-43f721515fc8"
      },
      "source": [
        "#!pip install py3Dmol\n",
        "import py3Dmol\n",
        "view=py3Dmol.view()\n",
        "#Loading template\n",
        "view.addModel(open('5eh6.pdb', 'r').read(),'pdb')\n",
        "#Loading best DOPE score model\n",
        "view.addModel(open('GYPA_gorilla__0.pdb', 'r').read(),'pdb')\n",
        "view.zoomTo()\n",
        "view.setBackgroundColor('white')\n",
        "view.setStyle({'chain':'A'},{'cartoon': {'color':'purple'}})\n",
        "view.setStyle({'chain':' '},{'cartoon': {'color':'yellow'}})\n",
        "view.show()\n",
        "#The following is a small trick to be able to manipulate (rotate, zoom) your\n",
        "#representation in Google Colab and, once you are happy with the visualization,\n",
        "#press Enter to print out as an onscreen png image file that you can download\n",
        "input()\n",
        "view.png()"
      ],
      "execution_count": 26,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16642132334955251\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16642132334955251\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@latest/build/3Dmol-min.min.js');\n}\n\nvar viewer_16642132334955251 = null;\nvar warn = document.getElementById(\"3dmolwarning_16642132334955251\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16642132334955251 = $3Dmol.createViewer($(\"#3dmolviewer_16642132334955251\"),{backgroundColor:\"white\"});\nviewer_16642132334955251.zoomTo();\n\tviewer_16642132334955251.addModel(\"HEADER    MEMBRANE PROTEIN                        28-OCT-15   5EH6              \\nTITLE     CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE MONOMER IN       \\nTITLE    2 LIPIDIC CUBIC PHASE                                                  \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: GLYCOPHORIN-A;                                             \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 FRAGMENT: UNP RESIDEUS 89-117;                                       \\nCOMPND   5 SYNONYM: MN SIALOGLYCOPROTEIN,PAS-2,SIALOGLYCOPROTEIN ALPHA;         \\nCOMPND   6 ENGINEERED: YES;                                                     \\nCOMPND   7 MUTATION: YES                                                        \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606;                                                \\nSOURCE   5 GENE: GYPA, GPA;                                                     \\nSOURCE   6 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE   7 EXPRESSION_SYSTEM_TAXID: 562                                         \\nKEYWDS    RECEPTOR, LIPIDIC CUBIC PHASE, PEPTIDES, TRANSMEMBRANE, MEMBRANE      \\nKEYWDS   2 PROTEIN                                                              \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    M.J.CALL,M.E.CALL,R.TRENKER                                           \\nREVDAT   5   01-JAN-20 5EH6    1       REMARK                                   \\nREVDAT   4   17-JAN-18 5EH6    1       REMARK                                   \\nREVDAT   3   20-SEP-17 5EH6    1       REMARK                                   \\nREVDAT   2   06-JAN-16 5EH6    1       JRNL                                     \\nREVDAT   1   23-DEC-15 5EH6    0                                                \\nJRNL        AUTH   R.TRENKER,M.E.CALL,M.J.CALL                                  \\nJRNL        TITL   CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE DIMER   \\nJRNL        TITL 2 IN LIPIDIC CUBIC PHASE.                                      \\nJRNL        REF    J.AM.CHEM.SOC.                V. 137 15676 2015              \\nJRNL        REFN                   ESSN 1520-5126                               \\nJRNL        PMID   26642914                                                     \\nJRNL        DOI    10.1021/JACS.5B11354                                         \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.92 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX 1.9_1692                                      \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.92                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 26.60                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 1.340                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 97.5                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2139                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.225                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.224                           \\nREMARK   3   FREE R VALUE                     : 0.238                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 10.000                          \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 214                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 26.5982 -  2.4156    0.96      978   109  0.2219 0.2306        \\nREMARK   3     2  2.4156 -  1.9175    0.99      947   105  0.2312 0.2642        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.11                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.90                                          \\nREMARK   3   K_SOL              : NULL                                          \\nREMARK   3   B_SOL              : NULL                                          \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.150            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 29.650           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.007            199                                  \\nREMARK   3   ANGLE     :  0.890            271                                  \\nREMARK   3   CHIRALITY :  0.035             39                                  \\nREMARK   3   PLANARITY :  0.003             30                                  \\nREMARK   3   DIHEDRAL  :  8.879             65                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 5EH6 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 28-OCT-15.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000214088.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 13-AUG-15                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : 7.5                                \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : AUSTRALIAN SYNCHROTRON             \\nREMARK 200  BEAMLINE                       : MX2                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.9537                             \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315R                  \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : HKL-2000                           \\nREMARK 200  DATA SCALING SOFTWARE          : HKL-2000                           \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 2141                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.918                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 26.600                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 97.6                               \\nREMARK 200  DATA REDUNDANCY                : 9.600                              \\nREMARK 200  R MERGE                    (I) : 0.10850                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 9.9100                             \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 1.92                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 1.99                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : 93.6                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : 8.60                               \\nREMARK 200  R MERGE FOR SHELL          (I) : 0.94000                            \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 2.370                              \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \\nREMARK 200 SOFTWARE USED: PHENIX 1.9_1692                                       \\nREMARK 200 STARTING MODEL: 4WOL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: HEXAGONAL PLATES                                             \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 37.03                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.95                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: 0.1 M HEPES, 20% PEG8000, 10 MM TRIS,    \\nREMARK 280  40 MM NACL, PH 7.5, LIPIDIC CUBIC PHASE, TEMPERATURE 293K           \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: H 3 2                            \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z                                                \\nREMARK 290       3555   -X+Y,-X,Z                                               \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z                                               \\nREMARK 290       6555   -X,-X+Y,-Z                                              \\nREMARK 290       7555   X+2/3,Y+1/3,Z+1/3                                       \\nREMARK 290       8555   -Y+2/3,X-Y+1/3,Z+1/3                                    \\nREMARK 290       9555   -X+Y+2/3,-X+1/3,Z+1/3                                   \\nREMARK 290      10555   Y+2/3,X+1/3,-Z+1/3                                      \\nREMARK 290      11555   X-Y+2/3,-Y+1/3,-Z+1/3                                   \\nREMARK 290      12555   -X+2/3,-X+Y+1/3,-Z+1/3                                  \\nREMARK 290      13555   X+1/3,Y+2/3,Z+2/3                                       \\nREMARK 290      14555   -Y+1/3,X-Y+2/3,Z+2/3                                    \\nREMARK 290      15555   -X+Y+1/3,-X+2/3,Z+2/3                                   \\nREMARK 290      16555   Y+1/3,X+2/3,-Z+2/3                                      \\nREMARK 290      17555   X-Y+1/3,-Y+2/3,-Z+2/3                                   \\nREMARK 290      18555   -X+1/3,-X+Y+2/3,-Z+2/3                                  \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   7  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3   7  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   8 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   8  0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   8  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   9 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   9 -0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1  10 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  10  0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  11  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2  11  0.000000 -1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  12 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  12 -0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  14 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  14  0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  15 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  15 -0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  15  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  16 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  16  0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  17  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  17  0.000000 -1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  17  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  18 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  18 -0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  18  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     GLU A    70                                                      \\nREMARK 465     SCH A    99                                                      \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ARG A  96    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     ARG A  97    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     LEU A  98    CG   CD1  CD2                                       \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ARG A  97       31.10    -98.71                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 5EH4   RELATED DB: PDB                                   \\nREMARK 900 5EH4 IS THE SAME PEPTIDE CRYSTALLISED AS DIMER                       \\nDBREF  5EH6 A   70    98  UNP    P02724   GLPA_HUMAN      89    117             \\nSEQADV 5EH6 ILE A   81  UNP  P02724    MET   100 ENGINEERED MUTATION            \\nSEQRES   1 A   30  GLU PRO GLU ILE THR LEU ILE ILE PHE GLY VAL ILE ALA          \\nSEQRES   2 A   30  GLY VAL ILE GLY THR ILE LEU LEU ILE SER TYR GLY ILE          \\nSEQRES   3 A   30  ARG ARG LEU SCH                                              \\nHELIX    1 AA1 GLU A   72  ARG A   97  1                                  26    \\nCRYST1   31.312   31.312  136.287  90.00  90.00 120.00 H 3 2        18          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.031937  0.018439  0.000000        0.00000                         \\nSCALE2      0.000000  0.036877  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.007337        0.00000                         \\nATOM      1  N   PRO A  71      26.002   6.928 -18.824  1.00 72.28           N  \\nANISOU    1  N   PRO A  71    10939   9924   6602  -1650  -1647    429       N  \\nATOM      2  CA  PRO A  71      24.968   6.210 -18.070  1.00 75.23           C  \\nANISOU    2  CA  PRO A  71    10989  10418   7177  -1839  -1839    391       C  \\nATOM      3  C   PRO A  71      25.401   5.954 -16.631  1.00 59.78           C  \\nANISOU    3  C   PRO A  71     8903   8293   5518  -1825  -1641    348       C  \\nATOM      4  O   PRO A  71      24.631   6.137 -15.685  1.00 55.18           O  \\nANISOU    4  O   PRO A  71     7919   7869   5177  -1810  -1746    462       O  \\nATOM      5  CB  PRO A  71      24.820   4.899 -18.846  1.00 83.70           C  \\nANISOU    5  CB  PRO A  71    12311  11422   8068  -2143  -1887    151       C  \\nATOM      6  CG  PRO A  71      26.160   4.684 -19.474  1.00 82.77           C  \\nANISOU    6  CG  PRO A  71    12644  11040   7765  -2081  -1586    -13       C  \\nATOM      7  CD  PRO A  71      26.685   6.058 -19.798  1.00 79.89           C  \\nANISOU    7  CD  PRO A  71    12306  10735   7312  -1802  -1502    190       C  \\nATOM      8  N   GLU A  72      26.651   5.547 -16.470  1.00 43.89           N  \\nANISOU    8  N   GLU A  72     7210   5973   3492  -1803  -1334    180       N  \\nATOM      9  CA  GLU A  72      27.199   5.342 -15.151  1.00 41.57           C  \\nANISOU    9  CA  GLU A  72     6830   5492   3473  -1747  -1128    132       C  \\nATOM     10  C   GLU A  72      27.567   6.686 -14.541  1.00 28.78           C  \\nANISOU   10  C   GLU A  72     4985   3872   2077  -1389   -970    340       C  \\nATOM     11  O   GLU A  72      28.126   6.740 -13.453  1.00 27.05           O  \\nANISOU   11  O   GLU A  72     4644   3486   2148  -1252   -742    321       O  \\nATOM     12  CB  GLU A  72      28.412   4.415 -15.215  1.00 53.65           C  \\nANISOU   12  CB  GLU A  72     8727   6675   4983  -1767   -827   -124       C  \\nATOM     13  CG  GLU A  72      29.411   4.761 -16.308  1.00 55.89           C  \\nANISOU   13  CG  GLU A  72     9315   6881   5040  -1626   -621   -177       C  \\nATOM     14  CD  GLU A  72      30.587   3.800 -16.349  1.00 67.90           C  \\nANISOU   14  CD  GLU A  72    11070   8100   6628  -1563   -300   -410       C  \\nATOM     15  OE1 GLU A  72      31.018   3.327 -15.272  1.00 68.55           O  \\nANISOU   15  OE1 GLU A  72    11065   7996   6985  -1491   -160   -465       O  \\nATOM     16  OE2 GLU A  72      31.075   3.511 -17.462  1.00 79.31           O  \\nANISOU   16  OE2 GLU A  72    12751   9514   7868  -1553   -194   -519       O  \\nATOM     17  N   ILE A  73      27.238   7.766 -15.250  1.00 30.36           N  \\nANISOU   17  N   ILE A  73     5165   4244   2126  -1248  -1115    542       N  \\nATOM     18  CA  ILE A  73      27.532   9.124 -14.797  1.00 28.77           C  \\nANISOU   18  CA  ILE A  73     4824   3994   2114   -920   -993    745       C  \\nATOM     19  C   ILE A  73      26.893   9.424 -13.447  1.00 26.85           C  \\nANISOU   19  C   ILE A  73     4154   3812   2237   -780   -998    815       C  \\nATOM     20  O   ILE A  73      27.496  10.058 -12.581  1.00 24.23           O  \\nANISOU   20  O   ILE A  73     3756   3310   2141   -574   -773    839       O  \\nATOM     21  CB  ILE A  73      27.053  10.160 -15.815  1.00 34.40           C  \\nANISOU   21  CB  ILE A  73     5607   4872   2593   -799  -1219    987       C  \\nATOM     22  CG1 ILE A  73      27.291  11.568 -15.279  1.00 36.30           C  \\nANISOU   22  CG1 ILE A  73     5741   4990   3060   -462  -1102   1194       C  \\nATOM     23  CG2 ILE A  73      25.585   9.942 -16.166  1.00 46.47           C  \\nANISOU   23  CG2 ILE A  73     6867   6736   4054   -887  -1615   1059       C  \\nATOM     24  CD1 ILE A  73      28.740  11.858 -15.015  1.00 45.01           C  \\nANISOU   24  CD1 ILE A  73     7064   5789   4247   -413   -720   1120       C  \\nATOM     25  N   THR A  74      25.665   8.948 -13.292  1.00 40.83           N  \\nANISOU   25  N   THR A  74     5637   5847   4029   -921  -1252    836       N  \\nATOM     26  CA  THR A  74      24.932   9.005 -12.039  1.00 54.69           C  \\nANISOU   26  CA  THR A  74     6966   7727   6086   -849  -1236    876       C  \\nATOM     27  C   THR A  74      25.719   8.304 -10.935  1.00 33.06           C  \\nANISOU   27  C   THR A  74     4287   4741   3534   -920   -947    714       C  \\nATOM     28  O   THR A  74      25.960   8.870  -9.873  1.00 25.63           O  \\nANISOU   28  O   THR A  74     3197   3724   2819   -711   -764    744       O  \\nATOM     29  CB  THR A  74      23.528   8.346 -12.181  1.00 69.92           C  \\nANISOU   29  CB  THR A  74     8570  10023   7975  -1102  -1547    902       C  \\nATOM     30  OG1 THR A  74      23.670   6.968 -12.571  1.00 59.92           O  \\nANISOU   30  OG1 THR A  74     7546   8677   6544  -1516  -1596    711       O  \\nATOM     31  CG2 THR A  74      22.688   9.086 -13.223  1.00 60.41           C  \\nANISOU   31  CG2 THR A  74     7246   9113   6594   -991  -1891   1093       C  \\nATOM     32  N   LEU A  75      26.123   7.069 -11.215  1.00 30.54           N  \\nANISOU   32  N   LEU A  75     4220   4283   3099  -1205   -922    539       N  \\nATOM     33  CA  LEU A  75      26.853   6.255 -10.261  1.00 32.84           C  \\nANISOU   33  CA  LEU A  75     4607   4325   3546  -1271   -697    405       C  \\nATOM     34  C   LEU A  75      28.206   6.852  -9.911  1.00 25.79           C  \\nANISOU   34  C   LEU A  75     3865   3180   2754  -1010   -424    380       C  \\nATOM     35  O   LEU A  75      28.625   6.828  -8.754  1.00 24.52           O  \\nANISOU   35  O   LEU A  75     3612   2910   2796   -922   -264    366       O  \\nATOM     36  CB  LEU A  75      27.029   4.843 -10.809  1.00 28.01           C  \\nANISOU   36  CB  LEU A  75     4304   3560   2777  -1589   -744    220       C  \\nATOM     37  CG  LEU A  75      25.766   3.999 -10.722  1.00 27.87           C  \\nANISOU   37  CG  LEU A  75     4119   3735   2737  -1948   -981    222       C  \\nATOM     38  CD1 LEU A  75      26.030   2.653 -11.357  1.00 37.84           C  \\nANISOU   38  CD1 LEU A  75     5752   4769   3858  -2133   -958     39       C  \\nATOM     39  CD2 LEU A  75      25.374   3.858  -9.264  1.00 31.17           C  \\nANISOU   39  CD2 LEU A  75     4232   4196   3414  -1954   -873    303       C  \\nATOM     40  N   ILE A  76      28.895   7.392 -10.905  1.00 19.81           N  \\nANISOU   40  N   ILE A  76     3334   2354   1837   -914   -376    382       N  \\nATOM     41  CA  ILE A  76      30.199   7.987 -10.648  1.00 25.42           C  \\nANISOU   41  CA  ILE A  76     4150   2861   2646   -719   -117    364       C  \\nATOM     42  C   ILE A  76      30.078   9.202  -9.725  1.00 21.46           C  \\nANISOU   42  C   ILE A  76     3413   2382   2358   -495    -72    500       C  \\nATOM     43  O   ILE A  76      30.821   9.326  -8.751  1.00 19.33           O  \\nANISOU   43  O   ILE A  76     3097   1975   2274   -406     97    454       O  \\nATOM     44  CB  ILE A  76      30.897   8.377 -11.962  1.00 18.98           C  \\nANISOU   44  CB  ILE A  76     3616   2007   1589   -703    -49    365       C  \\nATOM     45  CG1 ILE A  76      31.346   7.115 -12.698  1.00 26.36           C  \\nANISOU   45  CG1 ILE A  76     4836   2851   2328   -878      5    155       C  \\nATOM     46  CG2 ILE A  76      32.095   9.264 -11.691  1.00 17.37           C  \\nANISOU   46  CG2 ILE A  76     3437   1654   1510   -533    201    401       C  \\nATOM     47  CD1 ILE A  76      31.830   7.370 -14.096  1.00 33.92           C  \\nANISOU   47  CD1 ILE A  76     6090   3835   2962   -903     69    138       C  \\nATOM     48  N   ILE A  77      29.127  10.081 -10.014  1.00 17.69           N  \\nANISOU   48  N   ILE A  77     2797   2074   1850   -390   -239    659       N  \\nATOM     49  CA  ILE A  77      28.965  11.285  -9.221  1.00 17.04           C  \\nANISOU   49  CA  ILE A  77     2546   1970   1957   -140   -190    764       C  \\nATOM     50  C   ILE A  77      28.530  10.901  -7.817  1.00 15.87           C  \\nANISOU   50  C   ILE A  77     2141   1884   2003   -135   -137    701       C  \\nATOM     51  O   ILE A  77      29.012  11.457  -6.840  1.00 14.34           O  \\nANISOU   51  O   ILE A  77     1914   1569   1964      3     13    673       O  \\nATOM     52  CB  ILE A  77      27.958  12.256  -9.867  1.00 32.82           C  \\nANISOU   52  CB  ILE A  77     4453   4127   3892     28   -399    957       C  \\nATOM     53  CG1 ILE A  77      28.527  12.779 -11.191  1.00 28.88           C  \\nANISOU   53  CG1 ILE A  77     4274   3532   3168     26   -424   1056       C  \\nATOM     54  CG2 ILE A  77      27.626  13.433  -8.929  1.00 28.78           C  \\nANISOU   54  CG2 ILE A  77     3765   3567   3602    331   -343   1032       C  \\nATOM     55  CD1 ILE A  77      27.508  13.469 -12.067  1.00 24.61           C  \\nANISOU   55  CD1 ILE A  77     3698   3167   2486    155   -703   1271       C  \\nATOM     56  N   PHE A  78      27.644   9.923  -7.722  1.00 20.44           N  \\nANISOU   56  N   PHE A  78     2565   2655   2547   -323   -258    678       N  \\nATOM     57  CA  PHE A  78      27.223   9.413  -6.424  1.00 29.06           C  \\nANISOU   57  CA  PHE A  78     3441   3824   3777   -381   -185    638       C  \\nATOM     58  C   PHE A  78      28.424   8.921  -5.639  1.00 18.03           C  \\nANISOU   58  C   PHE A  78     2225   2175   2450   -414      5    527       C  \\nATOM     59  O   PHE A  78      28.580   9.231  -4.459  1.00 15.70           O  \\nANISOU   59  O   PHE A  78     1836   1857   2272   -311    126    514       O  \\nATOM     60  CB  PHE A  78      26.204   8.271  -6.578  1.00 27.90           C  \\nANISOU   60  CB  PHE A  78     3148   3890   3561   -681   -340    636       C  \\nATOM     61  CG  PHE A  78      26.034   7.437  -5.332  1.00 22.44           C  \\nANISOU   61  CG  PHE A  78     2353   3209   2963   -840   -228    598       C  \\nATOM     62  CD1 PHE A  78      25.276   7.901  -4.270  1.00 18.61           C  \\nANISOU   62  CD1 PHE A  78     1545   2938   2588   -734   -147    659       C  \\nATOM     63  CD2 PHE A  78      26.641   6.192  -5.225  1.00 24.51           C  \\nANISOU   63  CD2 PHE A  78     2869   3255   3189  -1079   -191    508       C  \\nATOM     64  CE1 PHE A  78      25.123   7.138  -3.128  1.00 22.90           C  \\nANISOU   64  CE1 PHE A  78     2025   3507   3168   -907    -27    650       C  \\nATOM     65  CE2 PHE A  78      26.495   5.426  -4.086  1.00 28.95           C  \\nANISOU   65  CE2 PHE A  78     3396   3794   3809  -1191    -99    498       C  \\nATOM     66  CZ  PHE A  78      25.735   5.900  -3.036  1.00 29.26           C  \\nANISOU   66  CZ  PHE A  78     3134   4062   3920  -1117    -16    569       C  \\nATOM     67  N   GLY A  79      29.267   8.143  -6.306  1.00 16.77           N  \\nANISOU   67  N   GLY A  79     2326   1842   2204   -542     25    441       N  \\nATOM     68  CA  GLY A  79      30.397   7.527  -5.649  1.00 17.63           C  \\nANISOU   68  CA  GLY A  79     2578   1732   2390   -550    170    345       C  \\nATOM     69  C   GLY A  79      31.372   8.545  -5.111  1.00 10.12           C  \\nANISOU   69  C   GLY A  79     1596    716   1533   -325    273    327       C  \\nATOM     70  O   GLY A  79      31.913   8.383  -4.016  1.00 15.97           O  \\nANISOU   70  O   GLY A  79     2272   1484   2310   -256    275    257       O  \\nATOM     71  N   VAL A  80      31.591   9.601  -5.877  1.00 10.40           N  \\nANISOU   71  N   VAL A  80     1690    719   1543   -227    301    379       N  \\nATOM     72  CA  VAL A  80      32.536  10.628  -5.468  1.00 11.18           C  \\nANISOU   72  CA  VAL A  80     1745    819   1684    -83    338    312       C  \\nATOM     73  C   VAL A  80      32.013  11.384  -4.245  1.00 13.80           C  \\nANISOU   73  C   VAL A  80     1956   1172   2116     24    367    331       C  \\nATOM     74  O   VAL A  80      32.725  11.539  -3.271  1.00 10.65           O  \\nANISOU   74  O   VAL A  80     1511    805   1730     46    389    236       O  \\nATOM     75  CB  VAL A  80      32.830  11.609  -6.604  1.00 13.99           C  \\nANISOU   75  CB  VAL A  80     2206   1127   1982    -46    353    378       C  \\nATOM     76  CG1 VAL A  80      33.718  12.724  -6.111  1.00 17.30           C  \\nANISOU   76  CG1 VAL A  80     2580   1539   2454     21    389    320       C  \\nATOM     77  CG2 VAL A  80      33.500  10.886  -7.770  1.00 13.03           C  \\nANISOU   77  CG2 VAL A  80     2225    983   1741   -137    407    339       C  \\nATOM     78  N   ILE A  81      30.763  11.834  -4.294  1.00 13.20           N  \\nANISOU   78  N   ILE A  81     1795   1113   2106    113    358    485       N  \\nATOM     79  CA  ILE A  81      30.179  12.589  -3.190  1.00 11.57           C  \\nANISOU   79  CA  ILE A  81     1445    961   1989    277    400    478       C  \\nATOM     80  C   ILE A  81      30.108  11.708  -1.940  1.00 13.01           C  \\nANISOU   80  C   ILE A  81     1533   1227   2182    176    459    408       C  \\nATOM     81  O   ILE A  81      30.520  12.124  -0.855  1.00 16.50           O  \\nANISOU   81  O   ILE A  81     1995   1608   2667    245    548    337       O  \\nATOM     82  CB  ILE A  81      28.752  13.144  -3.549  1.00 17.96           C  \\nANISOU   82  CB  ILE A  81     2062   1970   2793    435    299    580       C  \\nATOM     83  CG1 ILE A  81      28.846  14.286  -4.572  1.00 19.73           C  \\nANISOU   83  CG1 ILE A  81     2429   2060   3007    602    234    685       C  \\nATOM     84  CG2 ILE A  81      28.016  13.617  -2.294  1.00 17.99           C  \\nANISOU   84  CG2 ILE A  81     1869   2090   2875    608    390    536       C  \\nATOM     85  CD1 ILE A  81      27.489  14.757  -5.143  1.00 20.93           C  \\nANISOU   85  CD1 ILE A  81     2389   2417   3146    790     73    823       C  \\nATOM     86  N   ALA A  82      29.615  10.479  -2.093  1.00 14.62           N  \\nANISOU   86  N   ALA A  82     1675   1555   2325    -16    399    431       N  \\nATOM     87  CA  ALA A  82      29.518   9.568  -0.954  1.00 15.60           C  \\nANISOU   87  CA  ALA A  82     1756   1734   2436   -146    451    410       C  \\nATOM     88  C   ALA A  82      30.904   9.253  -0.397  1.00 17.38           C  \\nANISOU   88  C   ALA A  82     2169   1796   2638   -135    450    304       C  \\nATOM     89  O   ALA A  82      31.093   9.224   0.822  1.00 14.20           O  \\nANISOU   89  O   ALA A  82     1762   1408   2225   -118    497    282       O  \\nATOM     90  CB  ALA A  82      28.785   8.280  -1.339  1.00 12.86           C  \\nANISOU   90  CB  ALA A  82     1365   1496   2025   -401    364    459       C  \\nATOM     91  N   GLY A  83      31.866   9.017  -1.287  1.00  9.24           N  \\nANISOU   91  N   GLY A  83     1253    707   1549   -128    365    233       N  \\nATOM     92  CA  GLY A  83      33.240   8.762  -0.875  1.00 13.08           C  \\nANISOU   92  CA  GLY A  83     1783   1188   1997    -85    327    150       C  \\nATOM     93  C   GLY A  83      33.832   9.924  -0.086  1.00 12.02           C  \\nANISOU   93  C   GLY A  83     1606   1064   1896     -5    381    116       C  \\nATOM     94  O   GLY A  83      34.436   9.734   0.966  1.00 14.41           O  \\nANISOU   94  O   GLY A  83     1913   1363   2200     -6    377     98       O  \\nATOM     95  N   VAL A  84      33.648  11.136  -0.584  1.00 11.13           N  \\nANISOU   95  N   VAL A  84     1484    936   1808     43    418    124       N  \\nATOM     96  CA  VAL A  84      34.146  12.312   0.104  1.00  9.86           C  \\nANISOU   96  CA  VAL A  84     1332    751   1663     74    438     92       C  \\nATOM     97  C   VAL A  84      33.478  12.454   1.478  1.00 23.28           C  \\nANISOU   97  C   VAL A  84     3029   2426   3392    111    507     85       C  \\nATOM     98  O   VAL A  84      34.162  12.648   2.494  1.00 13.72           O  \\nANISOU   98  O   VAL A  84     1857   1190   2165     98    492     31       O  \\nATOM     99  CB  VAL A  84      33.919  13.579  -0.731  1.00 11.25           C  \\nANISOU   99  CB  VAL A  84     1553    869   1854    123    442    115       C  \\nATOM    100  CG1 VAL A  84      34.115  14.841   0.110  1.00 10.57           C  \\nANISOU  100  CG1 VAL A  84     1521    705   1792    167    448     70       C  \\nATOM    101  CG2 VAL A  84      34.850  13.573  -1.937  1.00 17.63           C  \\nANISOU  101  CG2 VAL A  84     2388   1661   2650     80    458    124       C  \\nATOM    102  N   ILE A  85      32.154  12.329   1.520  1.00 14.94           N  \\nANISOU  102  N   ILE A  85     1923   1361   2394    179    603    145       N  \\nATOM    103  CA  ILE A  85      31.434  12.529   2.766  1.00  9.86           C  \\nANISOU  103  CA  ILE A  85     1243    764   1739    273    714    109       C  \\nATOM    104  C   ILE A  85      31.759  11.396   3.733  1.00 13.41           C  \\nANISOU  104  C   ILE A  85     1720   1264   2112    139    730    122       C  \\nATOM    105  O   ILE A  85      32.010  11.623   4.920  1.00 12.84           O  \\nANISOU  105  O   ILE A  85     1713   1218   1946    152    765     51       O  \\nATOM    106  CB  ILE A  85      29.908  12.608   2.539  1.00 11.50           C  \\nANISOU  106  CB  ILE A  85     1248   1175   1948    376    772    165       C  \\nATOM    107  CG1 ILE A  85      29.540  13.861   1.742  1.00 12.43           C  \\nANISOU  107  CG1 ILE A  85     1381   1240   2101    537    698    162       C  \\nATOM    108  CG2 ILE A  85      29.158  12.574   3.871  1.00 13.13           C  \\nANISOU  108  CG2 ILE A  85     1364   1588   2037    399    874    114       C  \\nATOM    109  CD1 ILE A  85      28.073  13.892   1.307  1.00 21.23           C  \\nANISOU  109  CD1 ILE A  85     2243   2580   3242    666    702    240       C  \\nATOM    110  N   GLY A  86      31.759  10.176   3.214  1.00  9.78           N  \\nANISOU  110  N   GLY A  86     1252    811   1654     -9    668    216       N  \\nATOM    111  CA  GLY A  86      32.067   9.016   4.018  1.00 10.45           C  \\nANISOU  111  CA  GLY A  86     1408    923   1640   -133    602    253       C  \\nATOM    112  C   GLY A  86      33.429   9.138   4.676  1.00 13.84           C  \\nANISOU  112  C   GLY A  86     1950   1238   2069    -97    542    201       C  \\nATOM    113  O   GLY A  86      33.573   8.900   5.878  1.00 11.38           O  \\nANISOU  113  O   GLY A  86     1720    951   1652   -142    558    232       O  \\nATOM    114  N   THR A  87      34.433   9.509   3.889  1.00 12.99           N  \\nANISOU  114  N   THR A  87     1814   1108   2012    -35    437    126       N  \\nATOM    115  CA  THR A  87      35.804   9.618   4.400  1.00 13.69           C  \\nANISOU  115  CA  THR A  87     1935   1150   2118    -27    347     91       C  \\nATOM    116  C   THR A  87      35.930  10.647   5.525  1.00 12.38           C  \\nANISOU  116  C   THR A  87     1850    902   1951    -16    409     19       C  \\nATOM    117  O   THR A  87      36.549  10.379   6.559  1.00 12.81           O  \\nANISOU  117  O   THR A  87     1982    937   1949    -48    338     17       O  \\nATOM    118  CB  THR A  87      36.779   9.989   3.278  1.00 17.27           C  \\nANISOU  118  CB  THR A  87     2321   1640   2602     -6    290     50       C  \\nATOM    119  OG1 THR A  87      36.788   8.937   2.313  1.00 18.68           O  \\nANISOU  119  OG1 THR A  87     2482   1837   2777     -8    289     77       O  \\nATOM    120  CG2 THR A  87      38.187  10.186   3.824  1.00 16.57           C  \\nANISOU  120  CG2 THR A  87     2227   1480   2587    -10    237     21       C  \\nATOM    121  N   ILE A  88      35.341  11.822   5.302  1.00 17.44           N  \\nANISOU  121  N   ILE A  88     2481   1553   2593     40    488    -55       N  \\nATOM    122  CA  ILE A  88      35.267  12.888   6.299  1.00 19.16           C  \\nANISOU  122  CA  ILE A  88     2814   1716   2751     84    545   -196       C  \\nATOM    123  C   ILE A  88      34.589  12.416   7.594  1.00 19.87           C  \\nANISOU  123  C   ILE A  88     2959   1969   2623     75    598   -200       C  \\nATOM    124  O   ILE A  88      35.102  12.650   8.689  1.00 15.46           O  \\nANISOU  124  O   ILE A  88     2516   1441   1917     35    536   -287       O  \\nATOM    125  CB  ILE A  88      34.504  14.104   5.735  1.00 13.93           C  \\nANISOU  125  CB  ILE A  88     2113   1077   2102    181    576   -223       C  \\nATOM    126  CG1 ILE A  88      35.327  14.779   4.646  1.00 13.96           C  \\nANISOU  126  CG1 ILE A  88     2082   1046   2176    134    455   -189       C  \\nATOM    127  CG2 ILE A  88      34.123  15.087   6.842  1.00 16.44           C  \\nANISOU  127  CG2 ILE A  88     2550   1378   2318    268    642   -381       C  \\nATOM    128  CD1 ILE A  88      34.637  15.963   4.066  1.00 18.02           C  \\nANISOU  128  CD1 ILE A  88     2625   1515   2706    226    466   -198       C  \\nATOM    129  N   LEU A  89      33.448  11.740   7.460  1.00 11.97           N  \\nANISOU  129  N   LEU A  89     1866   1110   1573     77    699    -94       N  \\nATOM    130  CA  LEU A  89      32.719  11.270   8.622  1.00 15.64           C  \\nANISOU  130  CA  LEU A  89     2357   1779   1807     30    786    -64       C  \\nATOM    131  C   LEU A  89      33.496  10.176   9.350  1.00 17.03           C  \\nANISOU  131  C   LEU A  89     2650   1960   1862   -109    644     50       C  \\nATOM    132  O   LEU A  89      33.421  10.062  10.574  1.00 22.83           O  \\nANISOU  132  O   LEU A  89     3503   2822   2348   -157    660     47       O  \\nATOM    133  CB  LEU A  89      31.337  10.757   8.223  1.00 15.16           C  \\nANISOU  133  CB  LEU A  89     2118   1893   1750      8    923     44       C  \\nATOM    134  CG  LEU A  89      30.305  11.772   7.741  1.00 18.76           C  \\nANISOU  134  CG  LEU A  89     2413   2428   2288    194   1065    -39       C  \\nATOM    135  CD1 LEU A  89      28.953  11.097   7.500  1.00 18.26           C  \\nANISOU  135  CD1 LEU A  89     2108   2619   2212    118   1127     82       C  \\nATOM    136  CD2 LEU A  89      30.145  12.864   8.755  1.00 17.08           C  \\nANISOU  136  CD2 LEU A  89     2287   2258   1946    347   1167   -223       C  \\nATOM    137  N   LEU A  90      34.249   9.376   8.604  1.00 17.01           N  \\nANISOU  137  N   LEU A  90     2634   1814   2016   -149    506    153       N  \\nATOM    138  CA  LEU A  90      35.074   8.327   9.215  1.00 21.28           C  \\nANISOU  138  CA  LEU A  90     3288   2315   2484   -214    341    279       C  \\nATOM    139  C   LEU A  90      36.266   8.919   9.981  1.00 19.34           C  \\nANISOU  139  C   LEU A  90     3105   2064   2180   -180    177    183       C  \\nATOM    140  O   LEU A  90      36.646   8.435  11.049  1.00 18.95           O  \\nANISOU  140  O   LEU A  90     3181   2085   1935   -222     50    260       O  \\nATOM    141  CB  LEU A  90      35.559   7.351   8.143  1.00 14.32           C  \\nANISOU  141  CB  LEU A  90     2375   1259   1808   -208    262    380       C  \\nATOM    142  CG  LEU A  90      35.982   5.938   8.518  1.00 26.43           C  \\nANISOU  142  CG  LEU A  90     4044   2695   3303   -244    131    560       C  \\nATOM    143  CD1 LEU A  90      34.949   5.281   9.413  1.00 31.06           C  \\nANISOU  143  CD1 LEU A  90     4753   3391   3656   -397    202    703       C  \\nATOM    144  CD2 LEU A  90      36.156   5.148   7.244  1.00 21.46           C  \\nANISOU  144  CD2 LEU A  90     3291   1963   2900   -193    106    550       C  \\nATOM    145  N   ILE A  91      36.861   9.967   9.434  1.00 16.80           N  \\nANISOU  145  N   ILE A  91     2705   1662   2015   -132    162     29       N  \\nATOM    146  CA  ILE A  91      37.923  10.669  10.141  1.00 16.69           C  \\nANISOU  146  CA  ILE A  91     2731   1659   1952   -161      0    -90       C  \\nATOM    147  C   ILE A  91      37.387  11.282  11.438  1.00 21.02           C  \\nANISOU  147  C   ILE A  91     3458   2337   2192   -197     42   -207       C  \\nATOM    148  O   ILE A  91      37.971  11.111  12.510  1.00 23.14           O  \\nANISOU  148  O   ILE A  91     3838   2702   2251   -260   -131   -205       O  \\nATOM    149  CB  ILE A  91      38.548  11.761   9.261  1.00 16.48           C  \\nANISOU  149  CB  ILE A  91     2608   1501   2152   -163     12   -230       C  \\nATOM    150  CG1 ILE A  91      39.209  11.120   8.036  1.00 14.90           C  \\nANISOU  150  CG1 ILE A  91     2238   1215   2208   -130     -8   -127       C  \\nATOM    151  CG2 ILE A  91      39.553  12.591  10.062  1.00 20.71           C  \\nANISOU  151  CG2 ILE A  91     3190   2054   2625   -264   -162   -378       C  \\nATOM    152  CD1 ILE A  91      39.872  12.107   7.107  1.00 15.47           C  \\nANISOU  152  CD1 ILE A  91     2214   1180   2484   -172     32   -222       C  \\nATOM    153  N   SER A  92      36.265  11.984  11.320  1.00 21.14           N  \\nANISOU  153  N   SER A  92     3497   2366   2168   -137    273   -310       N  \\nATOM    154  CA  SER A  92      35.596  12.619  12.448  1.00 19.47           C  \\nANISOU  154  CA  SER A  92     3452   2280   1664   -123    395   -459       C  \\nATOM    155  C   SER A  92      35.332  11.605  13.560  1.00 21.28           C  \\nANISOU  155  C   SER A  92     3792   2717   1575   -208    375   -314       C  \\nATOM    156  O   SER A  92      35.657  11.825  14.733  1.00 23.88           O  \\nANISOU  156  O   SER A  92     4318   3152   1603   -269    293   -398       O  \\nATOM    157  CB  SER A  92      34.292  13.264  11.965  1.00 29.49           C  \\nANISOU  157  CB  SER A  92     4650   3558   2997     23    673   -538       C  \\nATOM    158  OG  SER A  92      33.609  13.945  12.998  1.00 39.92           O  \\nANISOU  158  OG  SER A  92     6117   5000   4052     92    844   -718       O  \\nATOM    159  N   TYR A  93      34.755  10.473  13.174  1.00 26.54           N  \\nANISOU  159  N   TYR A  93     4366   3429   2290   -241    439    -86       N  \\nATOM    160  CA  TYR A  93      34.474   9.408  14.121  1.00 27.68           C  \\nANISOU  160  CA  TYR A  93     4642   3725   2149   -357    429    110       C  \\nATOM    161  C   TYR A  93      35.794   8.928  14.720  1.00 27.37           C  \\nANISOU  161  C   TYR A  93     4738   3639   2021   -399    102    200       C  \\nATOM    162  O   TYR A  93      35.933   8.818  15.933  1.00 28.88           O  \\nANISOU  162  O   TYR A  93     5135   3977   1861   -470     24    228       O  \\nATOM    163  CB  TYR A  93      33.712   8.267  13.436  1.00 21.26           C  \\nANISOU  163  CB  TYR A  93     3721   2895   1460   -429    526    339       C  \\nATOM    164  CG  TYR A  93      33.489   7.046  14.302  1.00 33.29           C  \\nANISOU  164  CG  TYR A  93     5422   4503   2724   -592    501    595       C  \\nATOM    165  CD1 TYR A  93      32.502   7.034  15.280  1.00 28.66           C  \\nANISOU  165  CD1 TYR A  93     4837   4158   1893   -651    694    604       C  \\nATOM    166  CD2 TYR A  93      34.254   5.906  14.132  1.00 23.51           C  \\nANISOU  166  CD2 TYR A  93     4283   3078   1573   -625    269    812       C  \\nATOM    167  CE1 TYR A  93      32.295   5.937  16.063  1.00 29.00           C  \\nANISOU  167  CE1 TYR A  93     4966   4240   1811   -750    652    817       C  \\nATOM    168  CE2 TYR A  93      34.049   4.806  14.910  1.00 26.12           C  \\nANISOU  168  CE2 TYR A  93     4692   3432   1802   -692    224    999       C  \\nATOM    169  CZ  TYR A  93      33.070   4.827  15.874  1.00 32.94           C  \\nANISOU  169  CZ  TYR A  93     5569   4524   2424   -777    417   1012       C  \\nATOM    170  OH  TYR A  93      32.866   3.724  16.653  1.00 37.68           O  \\nANISOU  170  OH  TYR A  93     6289   5122   2905   -863    388   1217       O  \\nATOM    171  N   GLY A  94      36.775   8.688  13.859  1.00 27.56           N  \\nANISOU  171  N   GLY A  94     4631   3486   2355   -341    -90    240       N  \\nATOM    172  CA  GLY A  94      38.087   8.260  14.302  1.00 26.35           C  \\nANISOU  172  CA  GLY A  94     4513   3313   2186   -331   -418    327       C  \\nATOM    173  C   GLY A  94      38.758   9.167  15.325  1.00 30.85           C  \\nANISOU  173  C   GLY A  94     5189   4010   2522   -387   -592    152       C  \\nATOM    174  O   GLY A  94      39.384   8.678  16.259  1.00 34.40           O  \\nANISOU  174  O   GLY A  94     5764   4561   2745   -420   -848    270       O  \\nATOM    175  N   ILE A  95      38.646  10.484  15.172  1.00 30.72           N  \\nANISOU  175  N   ILE A  95     5156   3973   2545   -403   -483   -126       N  \\nATOM    176  CA  ILE A  95      39.352  11.367  16.102  1.00 36.00           C  \\nANISOU  176  CA  ILE A  95     5959   4723   2995   -499   -676   -329       C  \\nATOM    177  C   ILE A  95      38.577  11.567  17.405  1.00 41.88           C  \\nANISOU  177  C   ILE A  95     7009   5651   3252   -556   -564   -411       C  \\nATOM    178  O   ILE A  95      39.175  11.835  18.445  1.00 44.79           O  \\nANISOU  178  O   ILE A  95     7564   6144   3311   -660   -790   -498       O  \\nATOM    179  CB  ILE A  95      39.666  12.730  15.469  1.00 31.59           C  \\nANISOU  179  CB  ILE A  95     5328   4008   2665   -527   -628   -604       C  \\nATOM    180  CG1 ILE A  95      38.392  13.480  15.107  1.00 44.59           C  \\nANISOU  180  CG1 ILE A  95     7048   5569   4326   -434   -264   -754       C  \\nATOM    181  CG2 ILE A  95      40.500  12.536  14.236  1.00 28.34           C  \\nANISOU  181  CG2 ILE A  95     4621   3465   2683   -501   -716   -515       C  \\nATOM    182  CD1 ILE A  95      38.625  14.614  14.129  1.00 56.29           C  \\nANISOU  182  CD1 ILE A  95     8453   6814   6122   -422   -200   -928       C  \\nATOM    183  N   ARG A  96      37.255  11.416  17.350  1.00 44.07           N  \\nANISOU  183  N   ARG A  96     7323   5977   3443   -500   -216   -383       N  \\nATOM    184  CA  ARG A  96      36.445  11.402  18.559  1.00 48.82           C  \\nANISOU  184  CA  ARG A  96     8152   6800   3599   -539    -44   -413       C  \\nATOM    185  C   ARG A  96      36.929  10.279  19.462  1.00 76.51           C  \\nANISOU  185  C   ARG A  96    11748  10437   6886   -616   -281   -135       C  \\nATOM    186  O   ARG A  96      37.215  10.485  20.641  1.00 72.86           O  \\nANISOU  186  O   ARG A  96    11427  10122   6136   -653   -391   -197       O  \\nATOM    187  CB  ARG A  96      34.965  11.213  18.224  1.00 37.76           C  \\nANISOU  187  CB  ARG A  96     6623   5470   2255   -460    359   -360       C  \\nATOM    188  N   ARG A  97      37.055   9.096  18.870  1.00109.45           N  \\nANISOU  188  N   ARG A  97    15842  14523  11220   -617   -370    171       N  \\nATOM    189  CA  ARG A  97      37.355   7.874  19.604  1.00126.28           C  \\nANISOU  189  CA  ARG A  97    18046  16706  13230   -638   -546    477       C  \\nATOM    190  C   ARG A  97      38.833   7.466  19.569  1.00135.45           C  \\nANISOU  190  C   ARG A  97    19165  17788  14510   -587   -990    594       C  \\nATOM    191  O   ARG A  97      39.151   6.276  19.611  1.00149.94           O  \\nANISOU  191  O   ARG A  97    20993  19546  16431   -527  -1126    878       O  \\nATOM    192  CB  ARG A  97      36.497   6.726  19.058  1.00126.38           C  \\nANISOU  192  CB  ARG A  97    17986  16628  13406   -649   -349    735       C  \\nATOM    193  N   LEU A  98      39.734   8.442  19.490  1.00 91.41           N  \\nANISOU  193  N   LEU A  98    13524  12225   8981   -599  -1204    365       N  \\nATOM    194  CA  LEU A  98      41.164   8.149  19.561  1.00 61.19           C  \\nANISOU  194  CA  LEU A  98     9556   8401   5292   -545  -1627    456       C  \\nATOM    195  C   LEU A  98      41.950   9.379  19.988  1.00 66.32           C  \\nANISOU  195  C   LEU A  98    10163   9158   5878   -647  -1813    155       C  \\nATOM    196  O   LEU A  98      43.006   9.262  20.609  1.00 71.98           O  \\nANISOU  196  O   LEU A  98    10796  10000   6555   -646  -2141    198       O  \\nATOM    197  CB  LEU A  98      41.686   7.629  18.218  1.00 51.35           C  \\nANISOU  197  CB  LEU A  98     8064   6955   4491   -419  -1724    582       C  \\nTER     198      LEU A  98                                                      \\nMASTER      286    0    0    1    0    0    0    6  197    1    0    3          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16642132334955251.addModel(\"HEADER                                            26-SEP-22   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 26-SEP-22                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2022.34+release.9ec33c9fe00                                 \\nATOM      1  N   ASP A   1      -4.602  -2.188 -13.100  1.00  0.00           N  \\nATOM      2  CA  ASP A   1      -4.515  -1.079 -14.042  1.00  0.00           C  \\nATOM      3  C   ASP A   1      -3.242  -1.155 -14.870  1.00  0.00           C  \\nATOM      4  O   ASP A   1      -2.211  -1.628 -14.392  1.00  0.00           O  \\nATOM      5  CB  ASP A   1      -4.568   0.258 -13.303  1.00  0.00           C  \\nATOM      6  CG  ASP A   1      -5.925   0.531 -12.670  1.00  0.00           C  \\nATOM      7  OD1 ASP A   1      -6.872  -0.126 -13.033  1.00  0.00           O  \\nATOM      8  OD2 ASP A   1      -6.002   1.394 -11.829  1.00  0.00           O  \\nATOM      9 1H   ASP A   1      -5.456  -2.111 -12.566  1.00  0.00           H  \\nATOM     10 2H   ASP A   1      -4.600  -3.062 -13.606  1.00  0.00           H  \\nATOM     11 3H   ASP A   1      -3.810  -2.161 -12.473  1.00  0.00           H  \\nATOM     12  HA  ASP A   1      -5.366  -1.136 -14.722  1.00  0.00           H  \\nATOM     13 1HB  ASP A   1      -3.808   0.272 -12.521  1.00  0.00           H  \\nATOM     14 2HB  ASP A   1      -4.338   1.068 -13.998  1.00  0.00           H  \\nATOM     15  N   ASN A   2      -3.320  -0.690 -16.118  1.00  0.00           N  \\nATOM     16  CA  ASN A   2      -2.167  -0.705 -17.011  1.00  0.00           C  \\nATOM     17  C   ASN A   2      -1.332   0.561 -16.873  1.00  0.00           C  \\nATOM     18  O   ASN A   2      -0.326   0.721 -17.564  1.00  0.00           O  \\nATOM     19  CB  ASN A   2      -2.613  -0.893 -18.447  1.00  0.00           C  \\nATOM     20  CG  ASN A   2      -3.139  -2.274 -18.712  1.00  0.00           C  \\nATOM     21  OD1 ASN A   2      -2.781  -3.232 -18.016  1.00  0.00           O  \\nATOM     22  ND2 ASN A   2      -3.982  -2.397 -19.704  1.00  0.00           N  \\nATOM     23  H   ASN A   2      -4.194  -0.310 -16.453  1.00  0.00           H  \\nATOM     24  HA  ASN A   2      -1.524  -1.542 -16.733  1.00  0.00           H  \\nATOM     25 1HB  ASN A   2      -3.392  -0.168 -18.684  1.00  0.00           H  \\nATOM     26 2HB  ASN A   2      -1.774  -0.702 -19.116  1.00  0.00           H  \\nATOM     27 1HD2 ASN A   2      -4.366  -3.294 -19.927  1.00  0.00           H  \\nATOM     28 2HD2 ASN A   2      -4.244  -1.595 -20.240  1.00  0.00           H  \\nATOM     29  N   GLY A   3      -1.756   1.457 -15.990  1.00  0.00           N  \\nATOM     30  CA  GLY A   3      -1.045   2.708 -15.790  1.00  0.00           C  \\nATOM     31  C   GLY A   3       0.205   2.438 -15.001  1.00  0.00           C  \\nATOM     32  O   GLY A   3       0.356   1.369 -14.408  1.00  0.00           O  \\nATOM     33  H   GLY A   3      -2.585   1.271 -15.443  1.00  0.00           H  \\nATOM     34 1HA  GLY A   3      -0.806   3.149 -16.758  1.00  0.00           H  \\nATOM     35 2HA  GLY A   3      -1.690   3.413 -15.267  1.00  0.00           H  \\nATOM     36  N   GLU A   4       1.119   3.393 -14.974  1.00  0.00           N  \\nATOM     37  CA  GLU A   4       2.325   3.194 -14.211  1.00  0.00           C  \\nATOM     38  C   GLU A   4       1.981   3.052 -12.749  1.00  0.00           C  \\nATOM     39  O   GLU A   4       1.208   3.836 -12.197  1.00  0.00           O  \\nATOM     40  CB  GLU A   4       3.295   4.356 -14.418  1.00  0.00           C  \\nATOM     41  CG  GLU A   4       4.631   4.186 -13.727  1.00  0.00           C  \\nATOM     42  CD  GLU A   4       5.573   5.325 -13.989  1.00  0.00           C  \\nATOM     43  OE1 GLU A   4       5.888   5.561 -15.131  1.00  0.00           O  \\nATOM     44  OE2 GLU A   4       5.978   5.961 -13.046  1.00  0.00           O  \\nATOM     45  H   GLU A   4       0.967   4.256 -15.475  1.00  0.00           H  \\nATOM     46  HA  GLU A   4       2.806   2.278 -14.557  1.00  0.00           H  \\nATOM     47 1HB  GLU A   4       3.483   4.488 -15.483  1.00  0.00           H  \\nATOM     48 2HB  GLU A   4       2.843   5.277 -14.049  1.00  0.00           H  \\nATOM     49 1HG  GLU A   4       4.465   4.105 -12.653  1.00  0.00           H  \\nATOM     50 2HG  GLU A   4       5.087   3.258 -14.067  1.00  0.00           H  \\nATOM     51  N   TRP A   5       2.548   2.050 -12.107  1.00  0.00           N  \\nATOM     52  CA  TRP A   5       2.299   1.806 -10.705  1.00  0.00           C  \\nATOM     53  C   TRP A   5       2.998   2.843  -9.869  1.00  0.00           C  \\nATOM     54  O   TRP A   5       4.179   3.018 -10.072  1.00  0.00           O  \\nATOM     55  CB  TRP A   5       2.772   0.420 -10.304  1.00  0.00           C  \\nATOM     56  CG  TRP A   5       1.944  -0.683 -10.879  1.00  0.00           C  \\nATOM     57  CD1 TRP A   5       2.252  -1.457 -11.957  1.00  0.00           C  \\nATOM     58  CD2 TRP A   5       0.658  -1.145 -10.407  1.00  0.00           C  \\nATOM     59  NE1 TRP A   5       1.249  -2.366 -12.189  1.00  0.00           N  \\nATOM     60  CE2 TRP A   5       0.265  -2.191 -11.249  1.00  0.00           C  \\nATOM     61  CE3 TRP A   5      -0.182  -0.764  -9.353  1.00  0.00           C  \\nATOM     62  CZ2 TRP A   5      -0.934  -2.864 -11.075  1.00  0.00           C  \\nATOM     63  CZ3 TRP A   5      -1.385  -1.440  -9.178  1.00  0.00           C  \\nATOM     64  CH2 TRP A   5      -1.749  -2.464 -10.017  1.00  0.00           C  \\nATOM     65  H   TRP A   5       3.170   1.438 -12.615  1.00  0.00           H  \\nATOM     66  HA  TRP A   5       1.226   1.870 -10.528  1.00  0.00           H  \\nATOM     67 1HB  TRP A   5       3.804   0.280 -10.630  1.00  0.00           H  \\nATOM     68 2HB  TRP A   5       2.757   0.331  -9.217  1.00  0.00           H  \\nATOM     69  HD1 TRP A   5       3.161  -1.368 -12.548  1.00  0.00           H  \\nATOM     70  HE1 TRP A   5       1.236  -3.051 -12.930  1.00  0.00           H  \\nATOM     71  HE3 TRP A   5       0.104   0.047  -8.682  1.00  0.00           H  \\nATOM     72  HZ2 TRP A   5      -1.241  -3.679 -11.731  1.00  0.00           H  \\nATOM     73  HZ3 TRP A   5      -2.033  -1.137  -8.355  1.00  0.00           H  \\nATOM     74  HH2 TRP A   5      -2.699  -2.973  -9.851  1.00  0.00           H  \\nATOM     75  N   ILE A   6       2.238   3.478  -8.999  1.00  0.00           N  \\nATOM     76  CA  ILE A   6       2.640   4.549  -8.107  1.00  0.00           C  \\nATOM     77  C   ILE A   6       3.836   4.210  -7.212  1.00  0.00           C  \\nATOM     78  O   ILE A   6       4.527   5.107  -6.730  1.00  0.00           O  \\nATOM     79  CB  ILE A   6       1.441   4.956  -7.226  1.00  0.00           C  \\nATOM     80  CG1 ILE A   6       1.716   6.292  -6.535  1.00  0.00           C  \\nATOM     81  CG2 ILE A   6       1.145   3.884  -6.209  1.00  0.00           C  \\nATOM     82  CD1 ILE A   6       1.834   7.459  -7.488  1.00  0.00           C  \\nATOM     83  H   ILE A   6       1.269   3.196  -8.952  1.00  0.00           H  \\nATOM     84  HA  ILE A   6       2.940   5.406  -8.708  1.00  0.00           H  \\nATOM     85  HB  ILE A   6       0.562   5.101  -7.853  1.00  0.00           H  \\nATOM     86 1HG1 ILE A   6       0.915   6.508  -5.829  1.00  0.00           H  \\nATOM     87 2HG1 ILE A   6       2.643   6.223  -5.965  1.00  0.00           H  \\nATOM     88 1HG2 ILE A   6       0.298   4.187  -5.596  1.00  0.00           H  \\nATOM     89 2HG2 ILE A   6       0.907   2.952  -6.721  1.00  0.00           H  \\nATOM     90 3HG2 ILE A   6       2.004   3.737  -5.583  1.00  0.00           H  \\nATOM     91 1HD1 ILE A   6       2.030   8.371  -6.923  1.00  0.00           H  \\nATOM     92 2HD1 ILE A   6       2.655   7.280  -8.183  1.00  0.00           H  \\nATOM     93 3HD1 ILE A   6       0.904   7.570  -8.044  1.00  0.00           H  \\nATOM     94  N   GLN A   7       4.054   2.906  -6.984  1.00  0.00           N  \\nATOM     95  CA  GLN A   7       5.150   2.442  -6.139  1.00  0.00           C  \\nATOM     96  C   GLN A   7       6.488   2.387  -6.866  1.00  0.00           C  \\nATOM     97  O   GLN A   7       7.514   2.100  -6.250  1.00  0.00           O  \\nATOM     98  CB  GLN A   7       4.820   1.056  -5.571  1.00  0.00           C  \\nATOM     99  CG  GLN A   7       3.596   1.028  -4.672  1.00  0.00           C  \\nATOM    100  CD  GLN A   7       3.762   1.902  -3.437  1.00  0.00           C  \\nATOM    101  OE1 GLN A   7       4.716   1.743  -2.671  1.00  0.00           O  \\nATOM    102  NE2 GLN A   7       2.835   2.826  -3.240  1.00  0.00           N  \\nATOM    103  H   GLN A   7       3.444   2.225  -7.413  1.00  0.00           H  \\nATOM    104  HA  GLN A   7       5.272   3.143  -5.314  1.00  0.00           H  \\nATOM    105 1HB  GLN A   7       4.651   0.358  -6.391  1.00  0.00           H  \\nATOM    106 2HB  GLN A   7       5.668   0.687  -4.996  1.00  0.00           H  \\nATOM    107 1HG  GLN A   7       2.737   1.391  -5.234  1.00  0.00           H  \\nATOM    108 2HG  GLN A   7       3.422   0.004  -4.345  1.00  0.00           H  \\nATOM    109 1HE2 GLN A   7       2.891   3.431  -2.445  1.00  0.00           H  \\nATOM    110 2HE2 GLN A   7       2.080   2.921  -3.885  1.00  0.00           H  \\nATOM    111  N   LEU A   8       6.505   2.698  -8.161  1.00  0.00           N  \\nATOM    112  CA  LEU A   8       7.728   2.607  -8.948  1.00  0.00           C  \\nATOM    113  C   LEU A   8       8.852   3.492  -8.445  1.00  0.00           C  \\nATOM    114  O   LEU A   8       8.647   4.626  -8.011  1.00  0.00           O  \\nATOM    115  CB  LEU A   8       7.438   2.975 -10.419  1.00  0.00           C  \\nATOM    116  CG  LEU A   8       8.613   2.778 -11.403  1.00  0.00           C  \\nATOM    117  CD1 LEU A   8       8.913   1.295 -11.546  1.00  0.00           C  \\nATOM    118  CD2 LEU A   8       8.262   3.392 -12.736  1.00  0.00           C  \\nATOM    119  H   LEU A   8       5.678   2.901  -8.701  1.00  0.00           H  \\nATOM    120  HA  LEU A   8       8.092   1.581  -8.904  1.00  0.00           H  \\nATOM    121 1HB  LEU A   8       6.607   2.368 -10.771  1.00  0.00           H  \\nATOM    122 2HB  LEU A   8       7.140   4.023 -10.463  1.00  0.00           H  \\nATOM    123  HG  LEU A   8       9.500   3.259 -11.007  1.00  0.00           H  \\nATOM    124 1HD1 LEU A   8       9.742   1.156 -12.240  1.00  0.00           H  \\nATOM    125 2HD1 LEU A   8       9.182   0.882 -10.574  1.00  0.00           H  \\nATOM    126 3HD1 LEU A   8       8.031   0.781 -11.928  1.00  0.00           H  \\nATOM    127 1HD2 LEU A   8       9.091   3.254 -13.430  1.00  0.00           H  \\nATOM    128 2HD2 LEU A   8       7.372   2.909 -13.134  1.00  0.00           H  \\nATOM    129 3HD2 LEU A   8       8.070   4.454 -12.607  1.00  0.00           H  \\nATOM    130  N   VAL A   9      10.051   2.940  -8.522  1.00  0.00           N  \\nATOM    131  CA  VAL A   9      11.295   3.575  -8.136  1.00  0.00           C  \\nATOM    132  C   VAL A   9      12.214   3.703  -9.371  1.00  0.00           C  \\nATOM    133  O   VAL A   9      12.277   2.798 -10.200  1.00  0.00           O  \\nATOM    134  CB  VAL A   9      11.989   2.757  -7.038  1.00  0.00           C  \\nATOM    135  CG1 VAL A   9      11.134   2.720  -5.781  1.00  0.00           C  \\nATOM    136  CG2 VAL A   9      12.271   1.352  -7.540  1.00  0.00           C  \\nATOM    137  H   VAL A   9      10.097   2.003  -8.894  1.00  0.00           H  \\nATOM    138  HA  VAL A   9      11.075   4.539  -7.750  1.00  0.00           H  \\nATOM    139  HB  VAL A   9      12.927   3.245  -6.774  1.00  0.00           H  \\nATOM    140 1HG1 VAL A   9      11.641   2.137  -5.013  1.00  0.00           H  \\nATOM    141 2HG1 VAL A   9      10.975   3.736  -5.419  1.00  0.00           H  \\nATOM    142 3HG1 VAL A   9      10.173   2.261  -6.008  1.00  0.00           H  \\nATOM    143 1HG2 VAL A   9      12.764   0.778  -6.756  1.00  0.00           H  \\nATOM    144 2HG2 VAL A   9      11.332   0.866  -7.810  1.00  0.00           H  \\nATOM    145 3HG2 VAL A   9      12.919   1.401  -8.416  1.00  0.00           H  \\nATOM    146  N   PRO A  10      12.935   4.852  -9.475  1.00  0.00           N  \\nATOM    147  CA  PRO A  10      13.832   5.075 -10.583  1.00  0.00           C  \\nATOM    148  C   PRO A  10      14.952   4.062 -10.745  1.00  0.00           C  \\nATOM    149  O   PRO A  10      15.525   3.584  -9.766  1.00  0.00           O  \\nATOM    150  CB  PRO A  10      14.416   6.481 -10.255  1.00  0.00           C  \\nATOM    151  CG  PRO A  10      13.358   7.129  -9.381  1.00  0.00           C  \\nATOM    152  CD  PRO A  10      12.815   6.002  -8.555  1.00  0.00           C  \\nATOM    153  HA  PRO A  10      13.254   5.088 -11.519  1.00  0.00           H  \\nATOM    154 1HB  PRO A  10      15.385   6.375  -9.745  1.00  0.00           H  \\nATOM    155 2HB  PRO A  10      14.601   7.038 -11.185  1.00  0.00           H  \\nATOM    156 1HG  PRO A  10      13.806   7.925  -8.769  1.00  0.00           H  \\nATOM    157 2HG  PRO A  10      12.588   7.604 -10.006  1.00  0.00           H  \\nATOM    158 1HD  PRO A  10      13.434   5.872  -7.664  1.00  0.00           H  \\nATOM    159 2HD  PRO A  10      11.792   6.216  -8.281  1.00  0.00           H  \\nATOM    160  N   ARG A  11      15.234   3.732 -11.998  1.00  0.00           N  \\nATOM    161  CA  ARG A  11      16.291   2.803 -12.362  1.00  0.00           C  \\nATOM    162  C   ARG A  11      17.546   3.576 -12.706  1.00  0.00           C  \\nATOM    163  O   ARG A  11      17.469   4.540 -13.467  1.00  0.00           O  \\nATOM    164  CB  ARG A  11      15.880   1.942 -13.544  1.00  0.00           C  \\nATOM    165  CG  ARG A  11      16.909   0.904 -13.968  1.00  0.00           C  \\nATOM    166  CD  ARG A  11      16.407   0.054 -15.077  1.00  0.00           C  \\nATOM    167  NE  ARG A  11      17.392  -0.931 -15.495  1.00  0.00           N  \\nATOM    168  CZ  ARG A  11      17.227  -1.791 -16.519  1.00  0.00           C  \\nATOM    169  NH1 ARG A  11      16.114  -1.774 -17.219  1.00  0.00           N  \\nATOM    170  NH2 ARG A  11      18.185  -2.650 -16.821  1.00  0.00           N  \\nATOM    171  H   ARG A  11      14.709   4.175 -12.738  1.00  0.00           H  \\nATOM    172  HA  ARG A  11      16.484   2.148 -11.512  1.00  0.00           H  \\nATOM    173 1HB  ARG A  11      14.958   1.414 -13.307  1.00  0.00           H  \\nATOM    174 2HB  ARG A  11      15.681   2.578 -14.406  1.00  0.00           H  \\nATOM    175 1HG  ARG A  11      17.816   1.407 -14.304  1.00  0.00           H  \\nATOM    176 2HG  ARG A  11      17.144   0.258 -13.121  1.00  0.00           H  \\nATOM    177 1HD  ARG A  11      15.511  -0.475 -14.753  1.00  0.00           H  \\nATOM    178 2HD  ARG A  11      16.167   0.681 -15.935  1.00  0.00           H  \\nATOM    179  HE  ARG A  11      18.261  -0.973 -14.980  1.00  0.00           H  \\nATOM    180 1HH1 ARG A  11      15.382  -1.117 -16.989  1.00  0.00           H  \\nATOM    181 2HH1 ARG A  11      15.991  -2.419 -17.987  1.00  0.00           H  \\nATOM    182 1HH2 ARG A  11      19.041  -2.663 -16.283  1.00  0.00           H  \\nATOM    183 2HH2 ARG A  11      18.062  -3.294 -17.588  1.00  0.00           H  \\nATOM    184  N   PHE A  12      18.687   3.167 -12.177  1.00  0.00           N  \\nATOM    185  CA  PHE A  12      19.919   3.866 -12.484  1.00  0.00           C  \\nATOM    186  C   PHE A  12      20.456   3.618 -13.878  1.00  0.00           C  \\nATOM    187  O   PHE A  12      20.555   2.484 -14.345  1.00  0.00           O  \\nATOM    188  CB  PHE A  12      20.987   3.481 -11.476  1.00  0.00           C  \\nATOM    189  CG  PHE A  12      20.754   4.037 -10.111  1.00  0.00           C  \\nATOM    190  CD1 PHE A  12      20.104   3.289  -9.146  1.00  0.00           C  \\nATOM    191  CD2 PHE A  12      21.186   5.313  -9.790  1.00  0.00           C  \\nATOM    192  CE1 PHE A  12      19.888   3.801  -7.885  1.00  0.00           C  \\nATOM    193  CE2 PHE A  12      20.972   5.828  -8.527  1.00  0.00           C  \\nATOM    194  CZ  PHE A  12      20.321   5.070  -7.573  1.00  0.00           C  \\nATOM    195  H   PHE A  12      18.709   2.367 -11.562  1.00  0.00           H  \\nATOM    196  HA  PHE A  12      19.735   4.939 -12.415  1.00  0.00           H  \\nATOM    197 1HB  PHE A  12      21.038   2.397 -11.397  1.00  0.00           H  \\nATOM    198 2HB  PHE A  12      21.960   3.831 -11.824  1.00  0.00           H  \\nATOM    199  HD1 PHE A  12      19.762   2.283  -9.393  1.00  0.00           H  \\nATOM    200  HD2 PHE A  12      21.698   5.911 -10.542  1.00  0.00           H  \\nATOM    201  HE1 PHE A  12      19.374   3.200  -7.135  1.00  0.00           H  \\nATOM    202  HE2 PHE A  12      21.315   6.834  -8.281  1.00  0.00           H  \\nATOM    203  HZ  PHE A  12      20.150   5.476  -6.578  1.00  0.00           H  \\nATOM    204  N   SER A  13      20.817   4.698 -14.543  1.00  0.00           N  \\nATOM    205  CA  SER A  13      21.367   4.693 -15.884  1.00  0.00           C  \\nATOM    206  C   SER A  13      22.868   4.413 -15.941  1.00  0.00           C  \\nATOM    207  O   SER A  13      23.583   4.575 -14.970  1.00  0.00           O  \\nATOM    208  CB  SER A  13      21.083   6.018 -16.540  1.00  0.00           C  \\nATOM    209  OG  SER A  13      21.788   7.053 -15.907  1.00  0.00           O  \\nATOM    210  H   SER A  13      20.687   5.588 -14.084  1.00  0.00           H  \\nATOM    211  HA  SER A  13      20.882   3.899 -16.454  1.00  0.00           H  \\nATOM    212 1HB  SER A  13      21.364   5.972 -17.582  1.00  0.00           H  \\nATOM    213 2HB  SER A  13      20.015   6.224 -16.498  1.00  0.00           H  \\nATOM    214  HG  SER A  13      22.718   6.830 -15.990  1.00  0.00           H  \\nATOM    215  N   GLY A  14      23.335   3.965 -17.112  1.00  0.00           N  \\nATOM    216  CA  GLY A  14      24.788   3.724 -17.375  1.00  0.00           C  \\nATOM    217  C   GLY A  14      25.084   4.078 -18.873  1.00  0.00           C  \\nATOM    218  O   GLY A  14      25.328   3.170 -19.669  1.00  0.00           O  \\nATOM    219  H   GLY A  14      22.680   3.801 -17.864  1.00  0.00           H  \\nATOM    220 1HA  GLY A  14      25.251   4.199 -16.854  1.00  0.00           H  \\nATOM    221 2HA  GLY A  14      25.029   2.685 -17.163  1.00  0.00           H  \\nATOM    222  N   PRO A  15      25.117   5.598 -19.338  1.00  0.00           N  \\nATOM    223  CA  PRO A  15      24.947   7.066 -18.483  1.00  0.00           C  \\nATOM    224  C   PRO A  15      25.143   6.909 -16.902  1.00  0.00           C  \\nATOM    225  O   PRO A  15      24.527   7.638 -16.119  1.00  0.00           O  \\nATOM    226  CB  PRO A  15      23.773   7.402 -18.763  1.00  0.00           C  \\nATOM    227  CG  PRO A  15      23.436   6.541 -20.460  1.00  0.00           C  \\nATOM    228  CD  PRO A  15      24.172   5.207 -20.359  1.00  0.00           C  \\nATOM    229  HA  PRO A  15      25.674   7.796 -18.869  1.00  0.00           H  \\nATOM    230 1HB  PRO A  15      23.114   7.082 -18.000  1.00  0.00           H  \\nATOM    231 2HB  PRO A  15      23.698   8.496 -18.815  1.00  0.00           H  \\nATOM    232 1HG  PRO A  15      22.354   6.417 -20.615  1.00  0.00           H  \\nATOM    233 2HG  PRO A  15      23.805   7.168 -21.283  1.00  0.00           H  \\nATOM    234 1HD  PRO A  15      23.518   4.463 -19.884  1.00  0.00           H  \\nATOM    235 2HD  PRO A  15      24.470   4.876 -21.365  1.00  0.00           H  \\nATOM    236  N   GLU A  16      26.048   6.046 -16.622  1.00  0.00           N  \\nATOM    237  CA  GLU A  16      26.464   5.587 -15.268  1.00  0.00           C  \\nATOM    238  C   GLU A  16      26.881   6.657 -14.294  1.00  0.00           C  \\nATOM    239  O   GLU A  16      26.860   6.314 -13.101  1.00  0.00           O  \\nATOM    240  CB  GLU A  16      27.634   4.587 -15.395  1.00  0.00           C  \\nATOM    241  CG  GLU A  16      28.914   5.183 -15.992  1.00  0.00           C  \\nATOM    242  CD  GLU A  16      28.924   5.160 -17.493  1.00  0.00           C  \\nATOM    243  OE1 GLU A  16      27.878   5.038 -18.071  1.00  0.00           O  \\nATOM    244  OE2 GLU A  16      29.986   5.267 -18.062  1.00  0.00           O  \\nATOM    245  H   GLU A  16      26.509   5.585 -17.393  1.00  0.00           H  \\nATOM    246  HA  GLU A  16      25.642   5.108 -14.809  1.00  0.00           H  \\nATOM    247 1HB  GLU A  16      27.878   4.187 -14.411  1.00  0.00           H  \\nATOM    248 2HB  GLU A  16      27.331   3.752 -16.021  1.00  0.00           H  \\nATOM    249 1HG  GLU A  16      29.014   6.201 -15.659  1.00  0.00           H  \\nATOM    250 2HG  GLU A  16      29.770   4.623 -15.620  1.00  0.00           H  \\nATOM    251  N   ILE A  17      27.274   7.824 -14.812  1.00  0.00           N  \\nATOM    252  CA  ILE A  17      27.723   8.940 -13.994  1.00  0.00           C  \\nATOM    253  C   ILE A  17      26.648   9.245 -13.034  1.00  0.00           C  \\nATOM    254  O   ILE A  17      26.803   9.730 -11.936  1.00  0.00           O  \\nATOM    255  CB  ILE A  17      28.050  10.186 -14.837  1.00  0.00           C  \\nATOM    256  CG1 ILE A  17      26.797  10.684 -15.564  1.00  0.00           C  \\nATOM    257  CG2 ILE A  17      29.156   9.881 -15.828  1.00  0.00           C  \\nATOM    258  CD1 ILE A  17      26.972  12.024 -16.233  1.00  0.00           C  \\nATOM    259  H   ILE A  17      27.230   7.949 -15.813  1.00  0.00           H  \\nATOM    260  HA  ILE A  17      28.539   8.671 -13.523  1.00  0.00           H  \\nATOM    261  HB  ILE A  17      28.376  10.994 -14.181  1.00  0.00           H  \\nATOM    262 1HG1 ILE A  17      26.504   9.958 -16.324  1.00  0.00           H  \\nATOM    263 2HG1 ILE A  17      25.977  10.763 -14.858  1.00  0.00           H  \\nATOM    264 1HG2 ILE A  17      29.374  10.772 -16.415  1.00  0.00           H  \\nATOM    265 2HG2 ILE A  17      30.043   9.576 -15.294  1.00  0.00           H  \\nATOM    266 3HG2 ILE A  17      28.837   9.077 -16.493  1.00  0.00           H  \\nATOM    267 1HD1 ILE A  17      26.043  12.309 -16.725  1.00  0.00           H  \\nATOM    268 2HD1 ILE A  17      27.232  12.774 -15.484  1.00  0.00           H  \\nATOM    269 3HD1 ILE A  17      27.769  11.959 -16.972  1.00  0.00           H  \\nATOM    270  N   THR A  18      25.354   8.751 -13.325  1.00  0.00           N  \\nATOM    271  CA  THR A  18      24.461   9.203 -12.165  1.00  0.00           C  \\nATOM    272  C   THR A  18      25.250   8.927 -10.746  1.00  0.00           C  \\nATOM    273  O   THR A  18      25.248   9.805  -9.871  1.00  0.00           O  \\nATOM    274  CB  THR A  18      23.106   8.471 -12.189  1.00  0.00           C  \\nATOM    275  OG1 THR A  18      22.431   8.749 -13.424  1.00  0.00           O  \\nATOM    276  CG2 THR A  18      22.235   8.924 -11.028  1.00  0.00           C  \\nATOM    277  H   THR A  18      25.051   8.294 -14.180  1.00  0.00           H  \\nATOM    278  HA  THR A  18      24.276  10.273 -12.267  1.00  0.00           H  \\nATOM    279  HB  THR A  18      23.271   7.396 -12.115  1.00  0.00           H  \\nATOM    280  HG1 THR A  18      22.971   8.445 -14.159  1.00  0.00           H  \\nATOM    281 1HG2 THR A  18      21.282   8.396 -11.060  1.00  0.00           H  \\nATOM    282 2HG2 THR A  18      22.741   8.703 -10.086  1.00  0.00           H  \\nATOM    283 3HG2 THR A  18      22.058   9.995 -11.102  1.00  0.00           H  \\nATOM    284  N   LEU A  19      25.933   7.677 -10.605  1.00  0.00           N  \\nATOM    285  CA  LEU A  19      26.643   7.179  -9.375  1.00  0.00           C  \\nATOM    286  C   LEU A  19      28.153   7.560  -9.538  1.00  0.00           C  \\nATOM    287  O   LEU A  19      28.938   7.845  -8.646  1.00  0.00           O  \\nATOM    288  CB  LEU A  19      26.479   5.665  -9.208  1.00  0.00           C  \\nATOM    289  CG  LEU A  19      25.034   5.166  -9.032  1.00  0.00           C  \\nATOM    290  CD1 LEU A  19      25.027   3.648  -8.962  1.00  0.00           C  \\nATOM    291  CD2 LEU A  19      24.442   5.769  -7.782  1.00  0.00           C  \\nATOM    292  H   LEU A  19      26.126   7.166 -11.457  1.00  0.00           H  \\nATOM    293  HA  LEU A  19      26.235   7.621  -8.570  1.00  0.00           H  \\nATOM    294 1HB  LEU A  19      26.895   5.171 -10.085  1.00  0.00           H  \\nATOM    295 2HB  LEU A  19      27.048   5.347  -8.334  1.00  0.00           H  \\nATOM    296  HG  LEU A  19      24.438   5.463  -9.897  1.00  0.00           H  \\nATOM    297 1HD1 LEU A  19      24.002   3.294  -8.838  1.00  0.00           H  \\nATOM    298 2HD1 LEU A  19      25.441   3.239  -9.884  1.00  0.00           H  \\nATOM    299 3HD1 LEU A  19      25.629   3.320  -8.116  1.00  0.00           H  \\nATOM    300 1HD2 LEU A  19      23.428   5.420  -7.659  1.00  0.00           H  \\nATOM    301 2HD2 LEU A  19      25.034   5.472  -6.919  1.00  0.00           H  \\nATOM    302 3HD2 LEU A  19      24.443   6.849  -7.866  1.00  0.00           H  \\nATOM    303  N   ILE A  20      28.562   7.739 -10.816  1.00  0.00           N  \\nATOM    304  CA  ILE A  20      30.145   8.089 -10.812  1.00  0.00           C  \\nATOM    305  C   ILE A  20      30.200   9.244  -9.685  1.00  0.00           C  \\nATOM    306  O   ILE A  20      31.000   9.116  -8.747  1.00  0.00           O  \\nATOM    307  CB  ILE A  20      30.674   8.583 -12.142  1.00  0.00           C  \\nATOM    308  CG1 ILE A  20      30.570   7.406 -13.270  1.00  0.00           C  \\nATOM    309  CG2 ILE A  20      32.105   9.059 -11.999  1.00  0.00           C  \\nATOM    310  CD1 ILE A  20      31.395   6.189 -12.955  1.00  0.00           C  \\nATOM    311  H   ILE A  20      27.978   7.383 -11.570  1.00  0.00           H  \\nATOM    312  HA  ILE A  20      30.697   7.185 -10.561  1.00  0.00           H  \\nATOM    313  HB  ILE A  20      30.069   9.398 -12.488  1.00  0.00           H  \\nATOM    314 1HG1 ILE A  20      29.580   7.110 -13.371  1.00  0.00           H  \\nATOM    315 2HG1 ILE A  20      30.892   7.794 -14.228  1.00  0.00           H  \\nATOM    316 1HG2 ILE A  20      32.471   9.410 -12.963  1.00  0.00           H  \\nATOM    317 2HG2 ILE A  20      32.147   9.874 -11.277  1.00  0.00           H  \\nATOM    318 3HG2 ILE A  20      32.730   8.236 -11.653  1.00  0.00           H  \\nATOM    319 1HD1 ILE A  20      31.275   5.452 -13.749  1.00  0.00           H  \\nATOM    320 2HD1 ILE A  20      32.445   6.471 -12.879  1.00  0.00           H  \\nATOM    321 3HD1 ILE A  20      31.065   5.760 -12.010  1.00  0.00           H  \\nATOM    322  N   ILE A  21      29.276  10.202  -9.860  1.00  0.00           N  \\nATOM    323  CA  ILE A  21      29.108  11.403  -8.993  1.00  0.00           C  \\nATOM    324  C   ILE A  21      28.644  11.049  -7.530  1.00  0.00           C  \\nATOM    325  O   ILE A  21      29.429  11.488  -6.683  1.00  0.00           O  \\nATOM    326  CB  ILE A  21      28.101  12.370  -9.622  1.00  0.00           C  \\nATOM    327  CG1 ILE A  21      28.660  12.954 -10.919  1.00  0.00           C  \\nATOM    328  CG2 ILE A  21      27.747  13.482  -8.642  1.00  0.00           C  \\nATOM    329  CD1 ILE A  21      27.637  13.694 -11.746  1.00  0.00           C  \\nATOM    330  H   ILE A  21      28.602  10.084 -10.617  1.00  0.00           H  \\nATOM    331  HA  ILE A  21      30.071  11.906  -8.908  1.00  0.00           H  \\nATOM    332  HB  ILE A  21      27.192  11.829  -9.886  1.00  0.00           H  \\nATOM    333 1HG1 ILE A  21      29.473  13.640 -10.687  1.00  0.00           H  \\nATOM    334 2HG1 ILE A  21      29.076  12.150 -11.531  1.00  0.00           H  \\nATOM    335 1HG2 ILE A  21      27.031  14.160  -9.105  1.00  0.00           H  \\nATOM    336 2HG2 ILE A  21      27.308  13.049  -7.744  1.00  0.00           H  \\nATOM    337 3HG2 ILE A  21      28.649  14.033  -8.376  1.00  0.00           H  \\nATOM    338 1HD1 ILE A  21      28.109  14.079 -12.651  1.00  0.00           H  \\nATOM    339 2HD1 ILE A  21      26.827  13.013 -12.020  1.00  0.00           H  \\nATOM    340 3HD1 ILE A  21      27.233  14.524 -11.168  1.00  0.00           H  \\nATOM    341  N   PHE A  22      27.615  10.220  -7.406  1.00  0.00           N  \\nATOM    342  CA  PHE A  22      26.986   9.763  -6.128  1.00  0.00           C  \\nATOM    343  C   PHE A  22      28.021   9.383  -5.132  1.00  0.00           C  \\nATOM    344  O   PHE A  22      27.910   9.912  -4.026  1.00  0.00           O  \\nATOM    345  CB  PHE A  22      26.095   8.606  -6.340  1.00  0.00           C  \\nATOM    346  CG  PHE A  22      25.484   8.076  -5.079  1.00  0.00           C  \\nATOM    347  CD1 PHE A  22      24.401   8.712  -4.497  1.00  0.00           C  \\nATOM    348  CD2 PHE A  22      25.994   6.936  -4.473  1.00  0.00           C  \\nATOM    349  CE1 PHE A  22      23.837   8.224  -3.335  1.00  0.00           C  \\nATOM    350  CE2 PHE A  22      25.431   6.445  -3.312  1.00  0.00           C  \\nATOM    351  CZ  PHE A  22      24.351   7.090  -2.742  1.00  0.00           C  \\nATOM    352  H   PHE A  22      27.026  10.016  -8.215  1.00  0.00           H  \\nATOM    353  HA  PHE A  22      26.393  10.584  -5.721  1.00  0.00           H  \\nATOM    354 1HB  PHE A  22      25.289   8.887  -7.016  1.00  0.00           H  \\nATOM    355 2HB  PHE A  22      26.633   7.828  -6.796  1.00  0.00           H  \\nATOM    356  HD1 PHE A  22      23.994   9.607  -4.967  1.00  0.00           H  \\nATOM    357  HD2 PHE A  22      26.847   6.427  -4.923  1.00  0.00           H  \\nATOM    358  HE1 PHE A  22      22.985   8.735  -2.888  1.00  0.00           H  \\nATOM    359  HE2 PHE A  22      25.838   5.549  -2.844  1.00  0.00           H  \\nATOM    360  HZ  PHE A  22      23.907   6.705  -1.825  1.00  0.00           H  \\nATOM    361  N   GLY A  23      28.933   8.495  -5.519  1.00  0.00           N  \\nATOM    362  CA  GLY A  23      29.956   8.035  -4.633  1.00  0.00           C  \\nATOM    363  C   GLY A  23      31.022   9.105  -4.534  1.00  0.00           C  \\nATOM    364  O   GLY A  23      31.605   9.453  -3.522  1.00  0.00           O  \\nATOM    365  H   GLY A  23      28.914   8.218  -6.499  1.00  0.00           H  \\nATOM    366 1HA  GLY A  23      29.529   7.825  -3.670  1.00  0.00           H  \\nATOM    367 2HA  GLY A  23      30.372   7.102  -5.010  1.00  0.00           H  \\nATOM    368  N   VAL A  24      31.285   9.935  -5.690  1.00  0.00           N  \\nATOM    369  CA  VAL A  24      32.454  10.861  -5.240  1.00  0.00           C  \\nATOM    370  C   VAL A  24      31.943  11.513  -3.897  1.00  0.00           C  \\nATOM    371  O   VAL A  24      32.733  11.511  -2.944  1.00  0.00           O  \\nATOM    372  CB  VAL A  24      32.778  11.946  -6.283  1.00  0.00           C  \\nATOM    373  CG1 VAL A  24      33.766  12.952  -5.715  1.00  0.00           C  \\nATOM    374  CG2 VAL A  24      33.327  11.305  -7.538  1.00  0.00           C  \\nATOM    375  H   VAL A  24      31.038   9.815  -6.669  1.00  0.00           H  \\nATOM    376  HA  VAL A  24      33.345  10.250  -5.097  1.00  0.00           H  \\nATOM    377  HB  VAL A  24      31.865  12.492  -6.523  1.00  0.00           H  \\nATOM    378 1HG1 VAL A  24      33.984  13.712  -6.466  1.00  0.00           H  \\nATOM    379 2HG1 VAL A  24      33.334  13.428  -4.832  1.00  0.00           H  \\nATOM    380 3HG1 VAL A  24      34.687  12.441  -5.440  1.00  0.00           H  \\nATOM    381 1HG2 VAL A  24      33.553  12.078  -8.272  1.00  0.00           H  \\nATOM    382 2HG2 VAL A  24      34.236  10.755  -7.298  1.00  0.00           H  \\nATOM    383 3HG2 VAL A  24      32.598  10.630  -7.944  1.00  0.00           H  \\nATOM    384  N   MET A  25      30.633  11.848  -3.876  1.00  0.00           N  \\nATOM    385  CA  MET A  25      30.011  12.496  -2.745  1.00  0.00           C  \\nATOM    386  C   MET A  25      30.008  11.685  -1.478  1.00  0.00           C  \\nATOM    387  O   MET A  25      30.476  12.288  -0.510  1.00  0.00           O  \\nATOM    388  CB  MET A  25      28.583  12.874  -3.107  1.00  0.00           C  \\nATOM    389  CG  MET A  25      28.469  13.980  -4.135  1.00  0.00           C  \\nATOM    390  SD  MET A  25      26.772  14.273  -4.653  1.00  0.00           S  \\nATOM    391  CE  MET A  25      26.084  14.994  -3.166  1.00  0.00           C  \\nATOM    392  H   MET A  25      30.136  11.730  -4.753  1.00  0.00           H  \\nATOM    393  HA  MET A  25      30.569  13.395  -2.513  1.00  0.00           H  \\nATOM    394 1HB  MET A  25      28.084  12.035  -3.483  1.00  0.00           H  \\nATOM    395 2HB  MET A  25      28.053  13.195  -2.211  1.00  0.00           H  \\nATOM    396 1HG  MET A  25      28.867  14.904  -3.719  1.00  0.00           H  \\nATOM    397 2HG  MET A  25      29.058  13.720  -5.016  1.00  0.00           H  \\nATOM    398 1HE  MET A  25      25.032  15.232  -3.329  1.00  0.00           H  \\nATOM    399 2HE  MET A  25      26.173  14.284  -2.342  1.00  0.00           H  \\nATOM    400 3HE  MET A  25      26.628  15.906  -2.918  1.00  0.00           H  \\nATOM    401  N   ALA A  26      29.665  10.401  -1.622  1.00  0.00           N  \\nATOM    402  CA  ALA A  26      29.527   9.465  -0.530  1.00  0.00           C  \\nATOM    403  C   ALA A  26      30.771   9.489   0.305  1.00  0.00           C  \\nATOM    404  O   ALA A  26      30.561   9.560   1.512  1.00  0.00           O  \\nATOM    405  CB  ALA A  26      29.262   8.066  -1.031  1.00  0.00           C  \\nATOM    406  H   ALA A  26      29.238  10.171  -2.512  1.00  0.00           H  \\nATOM    407  HA  ALA A  26      28.679   9.778   0.080  1.00  0.00           H  \\nATOM    408 1HB  ALA A  26      29.196   7.383  -0.185  1.00  0.00           H  \\nATOM    409 2HB  ALA A  26      28.321   8.049  -1.585  1.00  0.00           H  \\nATOM    410 3HB  ALA A  26      30.066   7.757  -1.680  1.00  0.00           H  \\nATOM    411  N   GLY A  27      31.933   9.458  -0.357  1.00  0.00           N  \\nATOM    412  CA  GLY A  27      33.186   9.474   0.360  1.00  0.00           C  \\nATOM    413  C   GLY A  27      33.441  10.888   0.846  1.00  0.00           C  \\nATOM    414  O   GLY A  27      33.907  11.207   1.928  1.00  0.00           O  \\nATOM    415  H   GLY A  27      31.872   9.428  -1.371  1.00  0.00           H  \\nATOM    416 1HA  GLY A  27      33.137   8.773   1.193  1.00  0.00           H  \\nATOM    417 2HA  GLY A  27      33.985   9.134  -0.298  1.00  0.00           H  \\nATOM    418  N   ILE A  28      32.913  11.962   0.057  1.00  0.00           N  \\nATOM    419  CA  ILE A  28      33.348  13.293   0.702  1.00  0.00           C  \\nATOM    420  C   ILE A  28      32.927  13.124   2.180  1.00  0.00           C  \\nATOM    421  O   ILE A  28      33.780  13.381   3.041  1.00  0.00           O  \\nATOM    422  CB  ILE A  28      32.678  14.529   0.073  1.00  0.00           C  \\nATOM    423  CG1 ILE A  28      33.187  14.742  -1.356  1.00  0.00           C  \\nATOM    424  CG2 ILE A  28      32.934  15.763   0.922  1.00  0.00           C  \\nATOM    425  CD1 ILE A  28      32.383  15.752  -2.147  1.00  0.00           C  \\nATOM    426  H   ILE A  28      32.636  11.874  -0.914  1.00  0.00           H  \\nATOM    427  HA  ILE A  28      34.424  13.403   0.576  1.00  0.00           H  \\nATOM    428  HB  ILE A  28      31.608  14.365   0.003  1.00  0.00           H  \\nATOM    429 1HG1 ILE A  28      34.222  15.078  -1.327  1.00  0.00           H  \\nATOM    430 2HG1 ILE A  28      33.167  13.797  -1.894  1.00  0.00           H  \\nATOM    431 1HG2 ILE A  28      32.455  16.626   0.463  1.00  0.00           H  \\nATOM    432 2HG2 ILE A  28      32.525  15.609   1.921  1.00  0.00           H  \\nATOM    433 3HG2 ILE A  28      34.008  15.939   0.993  1.00  0.00           H  \\nATOM    434 1HD1 ILE A  28      32.804  15.848  -3.148  1.00  0.00           H  \\nATOM    435 2HD1 ILE A  28      31.352  15.418  -2.219  1.00  0.00           H  \\nATOM    436 3HD1 ILE A  28      32.418  16.717  -1.646  1.00  0.00           H  \\nATOM    437  N   ILE A  29      31.704  12.568   2.346  1.00  0.00           N  \\nATOM    438  CA  ILE A  29      31.101  12.369   3.638  1.00  0.00           C  \\nATOM    439  C   ILE A  29      31.788  11.395   4.533  1.00  0.00           C  \\nATOM    440  O   ILE A  29      32.107  11.870   5.631  1.00  0.00           O  \\nATOM    441  CB  ILE A  29      29.644  11.914   3.466  1.00  0.00           C  \\nATOM    442  CG1 ILE A  29      28.807  13.031   2.857  1.00  0.00           C  \\nATOM    443  CG2 ILE A  29      29.061  11.476   4.806  1.00  0.00           C  \\nATOM    444  CD1 ILE A  29      27.441  12.586   2.395  1.00  0.00           C  \\nATOM    445  H   ILE A  29      31.208  12.400   1.476  1.00  0.00           H  \\nATOM    446  HA  ILE A  29      31.117  13.316   4.176  1.00  0.00           H  \\nATOM    447  HB  ILE A  29      29.605  11.104   2.795  1.00  0.00           H  \\nATOM    448 1HG1 ILE A  29      28.677  13.828   3.588  1.00  0.00           H  \\nATOM    449 2HG1 ILE A  29      29.335  13.456   2.000  1.00  0.00           H  \\nATOM    450 1HG2 ILE A  29      28.029  11.157   4.666  1.00  0.00           H  \\nATOM    451 2HG2 ILE A  29      29.645  10.648   5.203  1.00  0.00           H  \\nATOM    452 3HG2 ILE A  29      29.091  12.310   5.506  1.00  0.00           H  \\nATOM    453 1HD1 ILE A  29      26.905  13.436   1.973  1.00  0.00           H  \\nATOM    454 2HD1 ILE A  29      27.548  11.810   1.634  1.00  0.00           H  \\nATOM    455 3HD1 ILE A  29      26.882  12.190   3.241  1.00  0.00           H  \\nATOM    456  N   GLY A  30      32.071  10.217   3.944  1.00  0.00           N  \\nATOM    457  CA  GLY A  30      32.679   9.060   4.534  1.00  0.00           C  \\nATOM    458  C   GLY A  30      33.871   9.429   5.316  1.00  0.00           C  \\nATOM    459  O   GLY A  30      33.880   8.997   6.470  1.00  0.00           O  \\nATOM    460  H   GLY A  30      31.672  10.057   3.030  1.00  0.00           H  \\nATOM    461 1HA  GLY A  30      31.957   8.558   5.178  1.00  0.00           H  \\nATOM    462 2HA  GLY A  30      32.955   8.356   3.751  1.00  0.00           H  \\nATOM    463  N   THR A  31      34.760  10.176   4.680  1.00  0.00           N  \\nATOM    464  CA  THR A  31      35.958  10.590   5.314  1.00  0.00           C  \\nATOM    465  C   THR A  31      35.672  11.741   6.234  1.00  0.00           C  \\nATOM    466  O   THR A  31      36.110  11.851   7.355  1.00  0.00           O  \\nATOM    467  CB  THR A  31      37.021  10.984   4.287  1.00  0.00           C  \\nATOM    468  OG1 THR A  31      36.525  12.053   3.466  1.00  0.00           O  \\nATOM    469  CG2 THR A  31      37.376   9.799   3.408  1.00  0.00           C  \\nATOM    470  H   THR A  31      34.526  10.476   3.741  1.00  0.00           H  \\nATOM    471  HA  THR A  31      36.333   9.778   5.889  1.00  0.00           H  \\nATOM    472  HB  THR A  31      37.918  11.327   4.805  1.00  0.00           H  \\nATOM    473  HG1 THR A  31      35.731  11.763   3.009  1.00  0.00           H  \\nATOM    474 1HG2 THR A  31      38.134  10.097   2.685  1.00  0.00           H  \\nATOM    475 2HG2 THR A  31      37.764   8.989   4.028  1.00  0.00           H  \\nATOM    476 3HG2 THR A  31      36.485   9.457   2.881  1.00  0.00           H  \\nATOM    477  N   ILE A  32      34.652  12.729   5.885  1.00  0.00           N  \\nATOM    478  CA  ILE A  32      34.768  13.698   7.109  1.00  0.00           C  \\nATOM    479  C   ILE A  32      34.581  12.822   8.425  1.00  0.00           C  \\nATOM    480  O   ILE A  32      35.359  13.136   9.335  1.00  0.00           O  \\nATOM    481  CB  ILE A  32      33.723  14.817   7.063  1.00  0.00           C  \\nATOM    482  CG1 ILE A  32      34.016  15.773   5.906  1.00  0.00           C  \\nATOM    483  CG2 ILE A  32      33.692  15.570   8.385  1.00  0.00           C  \\nATOM    484  CD1 ILE A  32      32.895  16.742   5.617  1.00  0.00           C  \\nATOM    485  H   ILE A  32      34.195  12.902   4.997  1.00  0.00           H  \\nATOM    486  HA  ILE A  32      35.755  14.161   7.091  1.00  0.00           H  \\nATOM    487  HB  ILE A  32      32.738  14.388   6.876  1.00  0.00           H  \\nATOM    488 1HG1 ILE A  32      34.915  16.347   6.127  1.00  0.00           H  \\nATOM    489 2HG1 ILE A  32      34.211  15.198   5.000  1.00  0.00           H  \\nATOM    490 1HG2 ILE A  32      32.945  16.361   8.335  1.00  0.00           H  \\nATOM    491 2HG2 ILE A  32      33.437  14.881   9.190  1.00  0.00           H  \\nATOM    492 3HG2 ILE A  32      34.671  16.008   8.578  1.00  0.00           H  \\nATOM    493 1HD1 ILE A  32      33.176  17.386   4.783  1.00  0.00           H  \\nATOM    494 2HD1 ILE A  32      31.991  16.186   5.357  1.00  0.00           H  \\nATOM    495 3HD1 ILE A  32      32.705  17.352   6.498  1.00  0.00           H  \\nATOM    496  N   LEU A  33      33.704  11.793   8.348  1.00  0.00           N  \\nATOM    497  CA  LEU A  33      33.412  10.943   9.482  1.00  0.00           C  \\nATOM    498  C   LEU A  33      34.590  10.167   9.993  1.00  0.00           C  \\nATOM    499  O   LEU A  33      34.849  10.359  11.187  1.00  0.00           O  \\nATOM    500  CB  LEU A  33      32.300   9.962   9.110  1.00  0.00           C  \\nATOM    501  CG  LEU A  33      30.915  10.570   8.929  1.00  0.00           C  \\nATOM    502  CD1 LEU A  33      29.972   9.524   8.360  1.00  0.00           C  \\nATOM    503  CD2 LEU A  33      30.412  11.092  10.264  1.00  0.00           C  \\nATOM    504  H   LEU A  33      33.173  11.691   7.488  1.00  0.00           H  \\nATOM    505  HA  LEU A  33      33.071  11.571  10.305  1.00  0.00           H  \\nATOM    506 1HB  LEU A  33      32.559   9.487   8.212  1.00  0.00           H  \\nATOM    507 2HB  LEU A  33      32.229   9.205   9.891  1.00  0.00           H  \\nATOM    508  HG  LEU A  33      30.970  11.394   8.214  1.00  0.00           H  \\nATOM    509 1HD1 LEU A  33      28.981   9.959   8.230  1.00  0.00           H  \\nATOM    510 2HD1 LEU A  33      30.347   9.181   7.394  1.00  0.00           H  \\nATOM    511 3HD1 LEU A  33      29.910   8.680   9.046  1.00  0.00           H  \\nATOM    512 1HD2 LEU A  33      29.421  11.528  10.134  1.00  0.00           H  \\nATOM    513 2HD2 LEU A  33      30.356  10.270  10.978  1.00  0.00           H  \\nATOM    514 3HD2 LEU A  33      31.097  11.853  10.639  1.00  0.00           H  \\nATOM    515  N   LEU A  34      35.276   9.507   9.049  1.00  0.00           N  \\nATOM    516  CA  LEU A  34      36.393   8.625   9.274  1.00  0.00           C  \\nATOM    517  C   LEU A  34      37.406   9.280  10.109  1.00  0.00           C  \\nATOM    518  O   LEU A  34      37.832   8.572  11.023  1.00  0.00           O  \\nATOM    519  CB  LEU A  34      37.019   8.204   7.960  1.00  0.00           C  \\nATOM    520  CG  LEU A  34      38.222   7.281   8.069  1.00  0.00           C  \\nATOM    521  CD1 LEU A  34      37.800   5.971   8.717  1.00  0.00           C  \\nATOM    522  CD2 LEU A  34      38.805   7.043   6.686  1.00  0.00           C  \\nATOM    523  H   LEU A  34      34.869   9.469   8.123  1.00  0.00           H  \\nATOM    524  HA  LEU A  34      36.032   7.733   9.784  1.00  0.00           H  \\nATOM    525 1HB  LEU A  34      36.263   7.694   7.364  1.00  0.00           H  \\nATOM    526 2HB  LEU A  34      37.327   9.076   7.435  1.00  0.00           H  \\nATOM    527  HG  LEU A  34      38.978   7.741   8.708  1.00  0.00           H  \\nATOM    528 1HD1 LEU A  34      38.662   5.310   8.795  1.00  0.00           H  \\nATOM    529 2HD1 LEU A  34      37.403   6.169   9.714  1.00  0.00           H  \\nATOM    530 3HD1 LEU A  34      37.032   5.495   8.108  1.00  0.00           H  \\nATOM    531 1HD2 LEU A  34      39.668   6.381   6.764  1.00  0.00           H  \\nATOM    532 2HD2 LEU A  34      38.051   6.582   6.047  1.00  0.00           H  \\nATOM    533 3HD2 LEU A  34      39.115   7.995   6.253  1.00  0.00           H  \\nATOM    534  N   ILE A  35      37.723  10.522   9.754  1.00  0.00           N  \\nATOM    535  CA  ILE A  35      38.687  11.299  10.446  1.00  0.00           C  \\nATOM    536  C   ILE A  35      38.161  11.799  11.749  1.00  0.00           C  \\nATOM    537  O   ILE A  35      38.815  11.718  12.798  1.00  0.00           O  \\nATOM    538  CB  ILE A  35      39.143  12.490   9.584  1.00  0.00           C  \\nATOM    539  CG1 ILE A  35      39.897  12.000   8.358  1.00  0.00           C  \\nATOM    540  CG2 ILE A  35      39.971  13.397  10.366  1.00  0.00           C  \\nATOM    541  CD1 ILE A  35      40.158  13.075   7.334  1.00  0.00           C  \\nATOM    542  H   ILE A  35      37.223  10.925   8.966  1.00  0.00           H  \\nATOM    543  HA  ILE A  35      39.546  10.676  10.653  1.00  0.00           H  \\nATOM    544  HB  ILE A  35      38.268  13.033   9.221  1.00  0.00           H  \\nATOM    545 1HG1 ILE A  35      40.855  11.579   8.664  1.00  0.00           H  \\nATOM    546 2HG1 ILE A  35      39.328  11.200   7.875  1.00  0.00           H  \\nATOM    547 1HG2 ILE A  35      40.284  14.232   9.742  1.00  0.00           H  \\nATOM    548 2HG2 ILE A  35      39.399  13.772  11.213  1.00  0.00           H  \\nATOM    549 3HG2 ILE A  35      40.791  12.901  10.705  1.00  0.00           H  \\nATOM    550 1HD1 ILE A  35      40.699  12.648   6.489  1.00  0.00           H  \\nATOM    551 2HD1 ILE A  35      39.208  13.486   6.987  1.00  0.00           H  \\nATOM    552 3HD1 ILE A  35      40.755  13.867   7.783  1.00  0.00           H  \\nATOM    553  N   SER A  36      36.854  12.215  11.818  1.00  0.00           N  \\nATOM    554  CA  SER A  36      36.544  12.688  13.192  1.00  0.00           C  \\nATOM    555  C   SER A  36      36.946  11.487  14.124  1.00  0.00           C  \\nATOM    556  O   SER A  36      37.469  11.702  15.211  1.00  0.00           O  \\nATOM    557  CB  SER A  36      35.083  13.048  13.351  1.00  0.00           C  \\nATOM    558  OG  SER A  36      34.267  11.917  13.195  1.00  0.00           O  \\nATOM    559  H   SER A  36      36.255  12.377  11.017  1.00  0.00           H  \\nATOM    560  HA  SER A  36      37.135  13.584  13.391  1.00  0.00           H  \\nATOM    561 1HB  SER A  36      34.921  13.483  14.337  1.00  0.00           H  \\nATOM    562 2HB  SER A  36      34.811  13.801  12.612  1.00  0.00           H  \\nATOM    563  HG  SER A  36      34.438  11.588  12.309  1.00  0.00           H  \\nATOM    564  N   TYR A  37      36.783  10.237  13.630  1.00  0.00           N  \\nATOM    565  CA  TYR A  37      37.141   9.135  14.519  1.00  0.00           C  \\nATOM    566  C   TYR A  37      38.647   8.860  14.655  1.00  0.00           C  \\nATOM    567  O   TYR A  37      39.163   8.803  15.771  1.00  0.00           O  \\nATOM    568  CB  TYR A  37      36.426   7.877  14.046  1.00  0.00           C  \\nATOM    569  CG  TYR A  37      34.947   7.904  14.266  1.00  0.00           C  \\nATOM    570  CD1 TYR A  37      34.080   7.840  13.179  1.00  0.00           C  \\nATOM    571  CD2 TYR A  37      34.440   7.992  15.551  1.00  0.00           C  \\nATOM    572  CE1 TYR A  37      32.717   7.864  13.382  1.00  0.00           C  \\nATOM    573  CE2 TYR A  37      33.074   8.017  15.754  1.00  0.00           C  \\nATOM    574  CZ  TYR A  37      32.215   7.953  14.677  1.00  0.00           C  \\nATOM    575  OH  TYR A  37      30.855   7.978  14.879  1.00  0.00           O  \\nATOM    576  H   TYR A  37      36.262  10.132  12.765  1.00  0.00           H  \\nATOM    577  HA  TYR A  37      36.811   9.383  15.528  1.00  0.00           H  \\nATOM    578 1HB  TYR A  37      36.608   7.734  12.987  1.00  0.00           H  \\nATOM    579 2HB  TYR A  37      36.832   7.011  14.567  1.00  0.00           H  \\nATOM    580  HD1 TYR A  37      34.480   7.770  12.166  1.00  0.00           H  \\nATOM    581  HD2 TYR A  37      35.118   8.043  16.403  1.00  0.00           H  \\nATOM    582  HE1 TYR A  37      32.039   7.814  12.531  1.00  0.00           H  \\nATOM    583  HE2 TYR A  37      32.675   8.087  16.766  1.00  0.00           H  \\nATOM    584  HH  TYR A  37      30.405   7.926  14.032  1.00  0.00           H  \\nATOM    585  N   SER A  38      39.359   8.728  13.531  1.00  0.00           N  \\nATOM    586  CA  SER A  38      40.787   8.377  13.536  1.00  0.00           C  \\nATOM    587  C   SER A  38      41.771   9.456  13.982  1.00  0.00           C  \\nATOM    588  O   SER A  38      42.844   9.146  14.500  1.00  0.00           O  \\nATOM    589  CB  SER A  38      41.182   7.928  12.142  1.00  0.00           C  \\nATOM    590  OG  SER A  38      41.089   8.978  11.236  1.00  0.00           O  \\nATOM    591  H   SER A  38      38.876   8.771  12.644  1.00  0.00           H  \\nATOM    592  HA  SER A  38      40.940   7.557  14.239  1.00  0.00           H  \\nATOM    593 1HB  SER A  38      42.203   7.548  12.157  1.00  0.00           H  \\nATOM    594 2HB  SER A  38      40.534   7.112  11.825  1.00  0.00           H  \\nATOM    595  HG  SER A  38      41.688   9.654  11.552  1.00  0.00           H  \\nATOM    596  N   ILE A  39      41.427  10.705  13.759  1.00  0.00           N  \\nATOM    597  CA  ILE A  39      42.288  11.828  14.113  1.00  0.00           C  \\nATOM    598  C   ILE A  39      41.676  12.496  15.415  1.00  0.00           C  \\nATOM    599  O   ILE A  39      42.402  12.822  16.351  1.00  0.00           O  \\nATOM    600  CB  ILE A  39      42.383  12.815  13.010  1.00  0.00           C  \\nATOM    601  CG1 ILE A  39      42.927  12.143  11.734  1.00  0.00           C  \\nATOM    602  CG2 ILE A  39      43.120  13.790  13.349  1.00  0.00           C  \\nATOM    603  CD1 ILE A  39      44.291  11.526  11.904  1.00  0.00           C  \\nATOM    604  H   ILE A  39      40.534  10.953  13.354  1.00  0.00           H  \\nATOM    605  HA  ILE A  39      43.288  11.447  14.311  1.00  0.00           H  \\nATOM    606  HB  ILE A  39      41.399  13.185  12.772  1.00  0.00           H  \\nATOM    607 1HG1 ILE A  39      42.239  11.366  11.413  1.00  0.00           H  \\nATOM    608 2HG1 ILE A  39      42.984  12.879  10.932  1.00  0.00           H  \\nATOM    609 1HG2 ILE A  39      43.180  14.503  12.529  1.00  0.00           H  \\nATOM    610 2HG2 ILE A  39      42.737  14.231  14.140  1.00  0.00           H  \\nATOM    611 3HG2 ILE A  39      44.118  13.426  13.586  1.00  0.00           H  \\nATOM    612 1HD1 ILE A  39      44.605  11.073  10.964  1.00  0.00           H  \\nATOM    613 2HD1 ILE A  39      45.005  12.297  12.193  1.00  0.00           H  \\nATOM    614 3HD1 ILE A  39      44.249  10.761  12.677  1.00  0.00           H  \\nATOM    615  N   ARG A  40      40.349  12.674  15.415  1.00  0.00           N  \\nATOM    616  CA  ARG A  40      39.662  13.379  16.601  1.00  0.00           C  \\nATOM    617  C   ARG A  40      40.303  14.871  16.734  1.00  0.00           C  \\nATOM    618  O   ARG A  40      40.316  15.469  17.810  1.00  0.00           O  \\nATOM    619  CB  ARG A  40      39.855  12.591  17.885  1.00  0.00           C  \\nATOM    620  CG  ARG A  40      39.313  11.199  17.854  1.00  0.00           C  \\nATOM    621  CD  ARG A  40      39.463  10.539  19.132  1.00  0.00           C  \\nATOM    622  NE  ARG A  40      38.944   9.186  19.106  1.00  0.00           N  \\nATOM    623  CZ  ARG A  40      39.024   8.316  20.131  1.00  0.00           C  \\nATOM    624  NH1 ARG A  40      39.604   8.676  21.255  1.00  0.00           N  \\nATOM    625  NH2 ARG A  40      38.518   7.101  20.008  1.00  0.00           N  \\nATOM    626  H   ARG A  40      39.791  12.378  14.617  1.00  0.00           H  \\nATOM    627  HA  ARG A  40      38.596  13.436  16.398  1.00  0.00           H  \\nATOM    628 1HB  ARG A  40      40.906  12.527  18.113  1.00  0.00           H  \\nATOM    629 2HB  ARG A  40      39.373  13.114  18.709  1.00  0.00           H  \\nATOM    630 1HG  ARG A  40      38.254  11.227  17.606  1.00  0.00           H  \\nATOM    631 2HG  ARG A  40      39.842  10.622  17.110  1.00  0.00           H  \\nATOM    632 1HD  ARG A  40      40.519  10.493  19.396  1.00  0.00           H  \\nATOM    633 2HD  ARG A  40      38.923  11.096  19.896  1.00  0.00           H  \\nATOM    634  HE  ARG A  40      38.485   8.869  18.248  1.00  0.00           H  \\nATOM    635 1HH1 ARG A  40      39.991   9.604  21.350  1.00  0.00           H  \\nATOM    636 2HH1 ARG A  40      39.664   8.024  22.024  1.00  0.00           H  \\nATOM    637 1HH2 ARG A  40      38.072   6.825  19.144  1.00  0.00           H  \\nATOM    638 2HH2 ARG A  40      38.578   6.449  20.776  1.00  0.00           H  \\nATOM    639  N   ARG A  41      40.806  15.378  15.617  1.00  0.00           N  \\nATOM    640  CA  ARG A  41      41.422  16.780  15.567  1.00  0.00           C  \\nATOM    641  C   ARG A  41      40.560  17.741  14.776  1.00  0.00           C  \\nATOM    642  O   ARG A  41      40.936  18.900  14.601  1.00  0.00           O  \\nATOM    643  CB  ARG A  41      42.802  16.767  14.939  1.00  0.00           C  \\nATOM    644  CG  ARG A  41      43.896  16.020  15.771  1.00  0.00           C  \\nATOM    645  CD  ARG A  41      45.223  16.055  15.092  1.00  0.00           C  \\nATOM    646  NE  ARG A  41      46.230  15.314  15.833  1.00  0.00           N  \\nATOM    647  CZ  ARG A  41      46.977  15.826  16.830  1.00  0.00           C  \\nATOM    648  NH1 ARG A  41      46.818  17.080  17.193  1.00  0.00           N  \\nATOM    649  NH2 ARG A  41      47.869  15.069  17.444  1.00  0.00           N  \\nATOM    650  H   ARG A  41      40.775  14.827  14.771  1.00  0.00           H  \\nATOM    651  HA  ARG A  41      41.504  17.158  16.586  1.00  0.00           H  \\nATOM    652 1HB  ARG A  41      42.753  16.310  13.996  1.00  0.00           H  \\nATOM    653 2HB  ARG A  41      43.143  17.791  14.790  1.00  0.00           H  \\nATOM    654 1HG  ARG A  41      43.998  16.494  16.746  1.00  0.00           H  \\nATOM    655 2HG  ARG A  41      43.611  14.998  15.899  1.00  0.00           H  \\nATOM    656 1HD  ARG A  41      45.136  15.616  14.106  1.00  0.00           H  \\nATOM    657 2HD  ARG A  41      45.558  17.087  15.001  1.00  0.00           H  \\nATOM    658  HE  ARG A  41      46.381  14.346  15.582  1.00  0.00           H  \\nATOM    659 1HH1 ARG A  41      46.136  17.659  16.724  1.00  0.00           H  \\nATOM    660 2HH1 ARG A  41      47.378  17.464  17.941  1.00  0.00           H  \\nATOM    661 1HH2 ARG A  41      47.991  14.105  17.164  1.00  0.00           H  \\nATOM    662 2HH2 ARG A  41      48.429  15.452  18.191  1.00  0.00           H  \\nATOM    663  N   LEU A  42      39.417  17.272  14.274  1.00  0.00           N  \\nATOM    664  CA  LEU A  42      38.557  18.133  13.483  1.00  0.00           C  \\nATOM    665  C   LEU A  42      38.023  19.268  14.348  1.00  0.00           C  \\nATOM    666  O   LEU A  42      38.031  20.430  13.940  1.00  0.00           O  \\nATOM    667  OXT LEU A  42      37.577  19.030  15.470  1.00  0.00           O  \\nATOM    668  CB  LEU A  42      37.387  17.331  12.888  1.00  0.00           C  \\nATOM    669  CG  LEU A  42      36.453  18.110  11.955  1.00  0.00           C  \\nATOM    670  CD1 LEU A  42      37.215  18.521  10.702  1.00  0.00           C  \\nATOM    671  CD2 LEU A  42      35.247  17.247  11.604  1.00  0.00           C  \\nATOM    672  H   LEU A  42      39.150  16.314  14.450  1.00  0.00           H  \\nATOM    673  HA  LEU A  42      39.143  18.549  12.665  1.00  0.00           H  \\nATOM    674 1HB  LEU A  42      37.792  16.491  12.324  1.00  0.00           H  \\nATOM    675 2HB  LEU A  42      36.784  16.936  13.706  1.00  0.00           H  \\nATOM    676  HG  LEU A  42      36.116  19.019  12.453  1.00  0.00           H  \\nATOM    677 1HD1 LEU A  42      36.551  19.076  10.038  1.00  0.00           H  \\nATOM    678 2HD1 LEU A  42      38.059  19.153  10.979  1.00  0.00           H  \\nATOM    679 3HD1 LEU A  42      37.579  17.631  10.189  1.00  0.00           H  \\nATOM    680 1HD2 LEU A  42      34.583  17.802  10.941  1.00  0.00           H  \\nATOM    681 2HD2 LEU A  42      35.583  16.337  11.104  1.00  0.00           H  \\nATOM    682 3HD2 LEU A  42      34.712  16.983  12.516  1.00  0.00           H  \\nTER                                                                             \\n##Begin comments##\\nMembraneInfo1 MembraneCenter   29.203   9.309  -0.621\\nMembraneInfo2 MembraneNormal   -0.144  -0.187  -0.972\\nMembraneInfo3 ATOM   9999  X   MEM A 999      29.203   9.309  -0.621\\nMembraneInfo4 ATOM   9999  Y   MEM A 999      27.038   6.511 -15.198\\nMembraneInfo5 ATOM   9999  Z   MEM A 999      31.367  12.107  13.956\\n##End comments##\\n\\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\nlabel fa_atr fa_rep fa_intra_rep fa_mbenv fa_mbsolv fa_pair hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_ss_dst dslf_cs_ang dslf_ss_dih dslf_ca_dih atom_pair_constraint rama omega fa_dun p_aa_pp ref cart_bonded Menv_smooth total\\nweights 0.8 0.44 0.004 0.3 0.35 0.49 1.17 1.17 2.34 2.2 0.5 2 5 5 0.5 0.2 0.5 0.56 0.32 1 0.5 0.5 NA\\npose -127.659 63.7097 0.50038 10.2388 18.328 -0.13863 -47.036 -1.34777 -0.03314 0 0 0 0 0 2.47073 -1.03172 -2.11779 28.7125 -0.30158 -4.81 138.177 -5.94256 71.7193\\nASP:NtermProteinFull_1 -1.15455 0.08465 0.00321 0.55185 0.44176 -0.0142 0 0 0 0 0 0 0 0 0 0 -0.00617 1.54618 0 -0.67 0.00389 -0.45202 0.33459\\nASN_2 -0.64035 0.03388 0.00548 0.02975 0.25058 -0.0142 0 0 0 0 0 0 0 0 0 -0.30587 -0.07928 1.21874 -0.13027 -0.89 0.0116 -0.21203 -0.72198\\nGLY_3 -1.13406 0.12242 0.00016 0.05548 0.36595 0 0 0 0 0 0 0 0 0 0 -0.10538 -0.07426 0 0.0621 -0.17 0.09235 1.61365 0.82842\\nGLU_4 -1.6319 0.11335 0.00508 0.67481 0.33759 0 0 0 0 0 0 0 0 0 0 -0.17897 -0.06788 2.77392 -0.0055 -0.81 0.13739 -0.72342 0.62448\\nTRP_5 -2.58249 0.21529 0.0152 0.16449 0.53907 0 0 0 0 0 0 0 0 0 0 -0.12856 -0.03498 0.99079 -0.03482 0.91 0.4328 -0.9775 -0.49072\\nILE_6 -1.78063 0.28172 0.02929 -0.30771 0.18285 0 0 0 0 0 0 0 0 0 0 -0.20554 -0.21937 0.53795 0.21042 0.24 0.53448 -0.50327 -0.99979\\nGLN_7 -1.96752 0.39655 0.00753 2.06962 0.19059 0 0 0 0 0 0 0 0 0 0 -0.23206 -0.0443 1.68367 -0.09515 -0.97 0.38702 0.99688 2.42282\\nLEU_8 -2.75615 0.32012 0.0129 0.28982 0.36901 0 0 0 0 0 0 0 0 0 0 -0.19939 -0.06516 0.88129 0.00309 -0.1 0.48024 -0.16339 -0.92763\\nVAL_9 -1.92101 0.78407 0.01696 -0.11286 0.2328 0 0 0 0 0 0 0 0 0 0 -0.13086 -0.07293 0.82923 -0.34749 0.29 1.19054 -0.26398 0.49446\\nPRO_10 -1.10234 0.6322 0.00202 -0.32041 0.1815 0 0 0 0 0 0 0 0 0 0 -0.27296 -0.08718 0.10343 -0.4791 0.02 1.22466 -0.0973 -0.1955\\nARG_11 -1.10794 0.03584 0.00964 0.54743 0.18665 0 0 0 0 0 0 0 0 0 0 -0.05783 -0.0505 1.2432 0.01543 -0.98 0.35995 -1.14663 -0.94476\\nPHE_12 -2.18987 0.20115 0.0178 -0.37804 0.27932 0 0 0 0 0 0 0 0 0 0 -0.18386 -0.04178 1.66143 0.00514 0.63 0.22744 -0.41925 -0.19051\\nSER_13 -2.4887 0.80308 0.00158 0.62409 0.51546 -0.03609 0 0 0 0 0 0 0 0 0 -0.13531 -0.00037 0.19953 -0.12331 -0.37 0.26844 0.25191 -0.4897\\nGLY_14 -1.67733 4.88315 9e-05 0.08478 0.39637 0 0 0 0 0 0 0 0 0 0 0.82272 0.04364 0 0.47152 -0.17 14.0452 0.58809 19.4882\\nPRO_15 -3.01763 0.58655 0.006 0.11389 0.63769 0 0 0 0 0 0 0 0 0 0 1.95468 0.06131 0.53727 0.43792 0.02 23.4103 -0.20164 24.5464\\nGLU_16 -4.36227 6.24293 0.01262 0.68059 0.89476 0.00995 0 0 0 0 0 0 0 0 0 -0.28707 -0.1861 2.47476 -0.12801 -0.81 10.2588 0.54547 15.3464\\nILE_17 -4.47266 4.75379 0.02202 0.0756 0.5423 0 0 0 0 0 0 0 0 0 0 -0.1735 -0.22526 0.62556 0.22754 0.24 3.58364 0.57309 5.77213\\nTHR_18 -3.17305 3.08505 0.00513 1.07485 0.47226 -0.04604 0 0 0 0 0 0 0 0 0 0.06405 -0.03168 0.0264 0.06053 -0.27 5.37933 0.36232 7.00915\\nLEU_19 -4.83694 4.422 0.01291 -0.24419 0.51935 0 0 0 0 0 0 0 0 0 0.24289 -0.06443 -0.02405 0.04059 -0.13159 -0.1 4.98873 -0.78806 4.03721\\nILE_20 -4.13459 4.03395 0.01786 -0.1535 0.41052 0 0 0 0 0 0 0 0 0 0 0.02347 0.00382 1.02846 -0.00895 0.24 5.63213 -0.79518 6.29799\\nILE_21 -4.34977 0.62603 0.02346 -0.23124 0.42552 0 0 0 0 0 0 0 0 0 0 -0.07757 -0.07399 0.06307 -0.2544 0.24 5.00212 -0.81269 0.58054\\nPHE_22 -4.39049 1.91278 0.02094 0.00979 0.42552 0 0 0 0 0 0 0 0 0 0.27307 0.04519 -0.03751 1.13427 -0.24897 0.63 2.62601 -0.90695 1.49364\\nGLY_23 -3.04152 3.29021 4e-05 0.90419 0.34196 0 0 0 0 0 0 0 0 0 0.24048 -0.01709 0.00055 0 0.4167 -0.17 4.62387 1.1475 7.73689\\nVAL_24 -3.31063 2.59465 0.01053 -0.00297 0.36377 0 0 0 0 0 0 0 0 0 0 -0.18388 -0.05161 0.15606 0.04412 0.29 5.24612 -0.33828 4.81786\\nMET_25 -4.7822 1.36472 0.00767 0.2046 0.49298 0 0 0 0 0 0 0 0 0 0 -0.1542 -0.01493 0.89039 0.04074 -0.34 3.05985 -0.0681 0.70152\\nALA_26 -3.30185 0.67738 0.00144 0.4956 0.32973 0 0 0 0 0 0 0 0 0 0.24449 -0.15427 -0.01285 0 -0.04765 0.16 2.34482 0.50989 1.24672\\nGLY_27 -2.96175 2.47786 4e-05 0.9042 0.33342 0 0 0 0 0 0 0 0 0 0.24289 -0.06813 -0.0384 0 0.38995 -0.17 3.73141 1.03797 5.87945\\nILE_28 -3.91087 2.19185 0.01635 -0.24031 0.42714 0 0 0 0 0 0 0 0 0 0 -0.09814 -0.00368 0.23171 -0.04783 0.24 4.43374 -0.79518 2.44477\\nILE_29 -4.37171 1.02319 0.02872 -0.24051 0.44112 0 0 0 0 0 0 0 0 0 0 -0.06906 -0.05551 0.06361 -0.25256 0.24 3.11151 -0.79716 -0.87837\\nGLY_30 -3.06246 0.32941 0.00038 0.9037 0.3044 0 0 0 0 0 0 0 0 0 0.5075 0.24462 -0.15782 0 0.11416 -0.17 2.48002 1.07453 2.56844\\nTHR_31 -4.16594 3.42261 0.00493 1.39175 0.46635 0 0 0 0 0 0 0 0 0 0.24048 -0.07025 -0.02983 0.40577 0.03239 -0.27 5.11119 0.57813 7.11758\\nILE_32 -3.50607 2.51612 0.01959 -0.25356 0.38854 0 0 0 0 0 0 0 0 0 0 -0.20739 -0.09679 0.44635 0.05272 0.24 6.2189 -0.79518 5.02324\\nLEU_33 -4.51414 0.85857 0.01496 -0.28639 0.55971 0 0 0 0 0 0 0 0 0 0 -0.11286 0.01975 0.62037 -0.10187 -0.1 3.16194 -0.82389 -0.70385\\nLEU_34 -4.1324 0.53621 0.017 -0.3061 0.48257 0 0 0 0 0 0 0 0 0 0.24449 -0.11408 -0.01398 0.24701 -0.08028 -0.1 2.16795 -0.82365 -1.87526\\nILE_35 -5.30782 2.22559 0.02886 -0.39747 0.78403 0 0 0 0 0 0 0 0 0 0 -0.12446 -0.05246 0.26351 -0.15088 0.24 2.47571 -0.77614 -0.79155\\nSER_36 -3.54997 0.915 0.001 0.8579 0.70093 -0.01903 0 0 0 0 0 0 0 0 0 -0.20402 -0.10651 0.30188 0.07206 -0.37 2.46677 0.19332 1.25932\\nTYR_37 -3.71988 0.90866 0.01852 0.35565 0.72938 0 0 0 -0.01657 0 0 0 0 0 0 0.08649 -0.02239 2.05525 -0.04267 0.51 1.25175 -0.04825 2.06594\\nSER_38 -2.64361 0.18717 0.00171 0.94348 0.47974 -0.01903 0 0 0 0 0 0 0 0 0.23444 -0.25719 -0.03895 0.23214 0.07354 -0.37 0.62038 0.25136 -0.3048\\nILE_39 -4.47886 1.97098 0.04616 -0.32023 0.61575 0 0 0 0 0 0 0 0 0 0 0.14516 -0.02747 0.50887 -0.2545 0.24 1.60265 0.57309 0.62162\\nARG_40 -2.82774 0.47416 0.00973 0.13724 0.63456 0 0 0 -0.01657 0 0 0 0 0 0 0.20834 -0.04812 1.00752 0.00135 -0.98 2.17129 -1.19737 -0.4256\\nARG_41 -2.05892 1.14598 0.01195 0.15282 0.40455 0 0 0 0 0 0 0 0 0 0 -0.05225 -0.05284 1.39552 -0.06721 -0.98 1.20877 -1.15715 -0.04878\\nLEU:CtermProteinFull_42 -1.13846 0.02882 0.01292 -0.26365 0.25 0 0 0 0 0 0 0 0 0 0 0 0 0.28683 0 -0.1 0.41176 -0.15607 -0.66785\\n#END_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\n\\n\",\"pdb\");\n\tviewer_16642132334955251.zoomTo();\n\tviewer_16642132334955251.setBackgroundColor(\"white\");\n\tviewer_16642132334955251.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"purple\"}});\n\tviewer_16642132334955251.setStyle({\"chain\": \" \"},{\"cartoon\": {\"color\": \"yellow\"}});\nviewer_16642132334955251.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16642132334955251\"  style=\"position: relative; width: 640px; height: 480px\">\n",
              "        <p id=\"3dmolwarning_16642132334955251\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@latest/build/3Dmol-min.min.js');\n",
              "}\n",
              "\n",
              "var viewer_16642132334955251 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16642132334955251\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16642132334955251 = $3Dmol.createViewer($(\"#3dmolviewer_16642132334955251\"),{backgroundColor:\"white\"});\n",
              "viewer_16642132334955251.zoomTo();\n",
              "\tviewer_16642132334955251.addModel(\"HEADER    MEMBRANE PROTEIN                        28-OCT-15   5EH6              \\nTITLE     CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE MONOMER IN       \\nTITLE    2 LIPIDIC CUBIC PHASE                                                  \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: GLYCOPHORIN-A;                                             \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 FRAGMENT: UNP RESIDEUS 89-117;                                       \\nCOMPND   5 SYNONYM: MN SIALOGLYCOPROTEIN,PAS-2,SIALOGLYCOPROTEIN ALPHA;         \\nCOMPND   6 ENGINEERED: YES;                                                     \\nCOMPND   7 MUTATION: YES                                                        \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606;                                                \\nSOURCE   5 GENE: GYPA, GPA;                                                     \\nSOURCE   6 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE   7 EXPRESSION_SYSTEM_TAXID: 562                                         \\nKEYWDS    RECEPTOR, LIPIDIC CUBIC PHASE, PEPTIDES, TRANSMEMBRANE, MEMBRANE      \\nKEYWDS   2 PROTEIN                                                              \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    M.J.CALL,M.E.CALL,R.TRENKER                                           \\nREVDAT   5   01-JAN-20 5EH6    1       REMARK                                   \\nREVDAT   4   17-JAN-18 5EH6    1       REMARK                                   \\nREVDAT   3   20-SEP-17 5EH6    1       REMARK                                   \\nREVDAT   2   06-JAN-16 5EH6    1       JRNL                                     \\nREVDAT   1   23-DEC-15 5EH6    0                                                \\nJRNL        AUTH   R.TRENKER,M.E.CALL,M.J.CALL                                  \\nJRNL        TITL   CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE DIMER   \\nJRNL        TITL 2 IN LIPIDIC CUBIC PHASE.                                      \\nJRNL        REF    J.AM.CHEM.SOC.                V. 137 15676 2015              \\nJRNL        REFN                   ESSN 1520-5126                               \\nJRNL        PMID   26642914                                                     \\nJRNL        DOI    10.1021/JACS.5B11354                                         \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.92 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX 1.9_1692                                      \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.92                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 26.60                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 1.340                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 97.5                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2139                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.225                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.224                           \\nREMARK   3   FREE R VALUE                     : 0.238                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 10.000                          \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 214                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 26.5982 -  2.4156    0.96      978   109  0.2219 0.2306        \\nREMARK   3     2  2.4156 -  1.9175    0.99      947   105  0.2312 0.2642        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.11                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.90                                          \\nREMARK   3   K_SOL              : NULL                                          \\nREMARK   3   B_SOL              : NULL                                          \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.150            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 29.650           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.007            199                                  \\nREMARK   3   ANGLE     :  0.890            271                                  \\nREMARK   3   CHIRALITY :  0.035             39                                  \\nREMARK   3   PLANARITY :  0.003             30                                  \\nREMARK   3   DIHEDRAL  :  8.879             65                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 5EH6 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 28-OCT-15.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000214088.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 13-AUG-15                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : 7.5                                \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : AUSTRALIAN SYNCHROTRON             \\nREMARK 200  BEAMLINE                       : MX2                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.9537                             \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315R                  \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : HKL-2000                           \\nREMARK 200  DATA SCALING SOFTWARE          : HKL-2000                           \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 2141                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.918                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 26.600                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 97.6                               \\nREMARK 200  DATA REDUNDANCY                : 9.600                              \\nREMARK 200  R MERGE                    (I) : 0.10850                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 9.9100                             \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 1.92                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 1.99                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : 93.6                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : 8.60                               \\nREMARK 200  R MERGE FOR SHELL          (I) : 0.94000                            \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 2.370                              \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \\nREMARK 200 SOFTWARE USED: PHENIX 1.9_1692                                       \\nREMARK 200 STARTING MODEL: 4WOL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: HEXAGONAL PLATES                                             \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 37.03                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.95                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: 0.1 M HEPES, 20% PEG8000, 10 MM TRIS,    \\nREMARK 280  40 MM NACL, PH 7.5, LIPIDIC CUBIC PHASE, TEMPERATURE 293K           \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: H 3 2                            \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z                                                \\nREMARK 290       3555   -X+Y,-X,Z                                               \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z                                               \\nREMARK 290       6555   -X,-X+Y,-Z                                              \\nREMARK 290       7555   X+2/3,Y+1/3,Z+1/3                                       \\nREMARK 290       8555   -Y+2/3,X-Y+1/3,Z+1/3                                    \\nREMARK 290       9555   -X+Y+2/3,-X+1/3,Z+1/3                                   \\nREMARK 290      10555   Y+2/3,X+1/3,-Z+1/3                                      \\nREMARK 290      11555   X-Y+2/3,-Y+1/3,-Z+1/3                                   \\nREMARK 290      12555   -X+2/3,-X+Y+1/3,-Z+1/3                                  \\nREMARK 290      13555   X+1/3,Y+2/3,Z+2/3                                       \\nREMARK 290      14555   -Y+1/3,X-Y+2/3,Z+2/3                                    \\nREMARK 290      15555   -X+Y+1/3,-X+2/3,Z+2/3                                   \\nREMARK 290      16555   Y+1/3,X+2/3,-Z+2/3                                      \\nREMARK 290      17555   X-Y+1/3,-Y+2/3,-Z+2/3                                   \\nREMARK 290      18555   -X+1/3,-X+Y+2/3,-Z+2/3                                  \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   7  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3   7  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   8 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   8  0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   8  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   9 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   9 -0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1  10 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  10  0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  11  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2  11  0.000000 -1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  12 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  12 -0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  14 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  14  0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  15 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  15 -0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  15  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  16 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  16  0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  17  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  17  0.000000 -1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  17  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  18 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  18 -0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  18  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     GLU A    70                                                      \\nREMARK 465     SCH A    99                                                      \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ARG A  96    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     ARG A  97    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     LEU A  98    CG   CD1  CD2                                       \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ARG A  97       31.10    -98.71                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 5EH4   RELATED DB: PDB                                   \\nREMARK 900 5EH4 IS THE SAME PEPTIDE CRYSTALLISED AS DIMER                       \\nDBREF  5EH6 A   70    98  UNP    P02724   GLPA_HUMAN      89    117             \\nSEQADV 5EH6 ILE A   81  UNP  P02724    MET   100 ENGINEERED MUTATION            \\nSEQRES   1 A   30  GLU PRO GLU ILE THR LEU ILE ILE PHE GLY VAL ILE ALA          \\nSEQRES   2 A   30  GLY VAL ILE GLY THR ILE LEU LEU ILE SER TYR GLY ILE          \\nSEQRES   3 A   30  ARG ARG LEU SCH                                              \\nHELIX    1 AA1 GLU A   72  ARG A   97  1                                  26    \\nCRYST1   31.312   31.312  136.287  90.00  90.00 120.00 H 3 2        18          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.031937  0.018439  0.000000        0.00000                         \\nSCALE2      0.000000  0.036877  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.007337        0.00000                         \\nATOM      1  N   PRO A  71      26.002   6.928 -18.824  1.00 72.28           N  \\nANISOU    1  N   PRO A  71    10939   9924   6602  -1650  -1647    429       N  \\nATOM      2  CA  PRO A  71      24.968   6.210 -18.070  1.00 75.23           C  \\nANISOU    2  CA  PRO A  71    10989  10418   7177  -1839  -1839    391       C  \\nATOM      3  C   PRO A  71      25.401   5.954 -16.631  1.00 59.78           C  \\nANISOU    3  C   PRO A  71     8903   8293   5518  -1825  -1641    348       C  \\nATOM      4  O   PRO A  71      24.631   6.137 -15.685  1.00 55.18           O  \\nANISOU    4  O   PRO A  71     7919   7869   5177  -1810  -1746    462       O  \\nATOM      5  CB  PRO A  71      24.820   4.899 -18.846  1.00 83.70           C  \\nANISOU    5  CB  PRO A  71    12311  11422   8068  -2143  -1887    151       C  \\nATOM      6  CG  PRO A  71      26.160   4.684 -19.474  1.00 82.77           C  \\nANISOU    6  CG  PRO A  71    12644  11040   7765  -2081  -1586    -13       C  \\nATOM      7  CD  PRO A  71      26.685   6.058 -19.798  1.00 79.89           C  \\nANISOU    7  CD  PRO A  71    12306  10735   7312  -1802  -1502    190       C  \\nATOM      8  N   GLU A  72      26.651   5.547 -16.470  1.00 43.89           N  \\nANISOU    8  N   GLU A  72     7210   5973   3492  -1803  -1334    180       N  \\nATOM      9  CA  GLU A  72      27.199   5.342 -15.151  1.00 41.57           C  \\nANISOU    9  CA  GLU A  72     6830   5492   3473  -1747  -1128    132       C  \\nATOM     10  C   GLU A  72      27.567   6.686 -14.541  1.00 28.78           C  \\nANISOU   10  C   GLU A  72     4985   3872   2077  -1389   -970    340       C  \\nATOM     11  O   GLU A  72      28.126   6.740 -13.453  1.00 27.05           O  \\nANISOU   11  O   GLU A  72     4644   3486   2148  -1252   -742    321       O  \\nATOM     12  CB  GLU A  72      28.412   4.415 -15.215  1.00 53.65           C  \\nANISOU   12  CB  GLU A  72     8727   6675   4983  -1767   -827   -124       C  \\nATOM     13  CG  GLU A  72      29.411   4.761 -16.308  1.00 55.89           C  \\nANISOU   13  CG  GLU A  72     9315   6881   5040  -1626   -621   -177       C  \\nATOM     14  CD  GLU A  72      30.587   3.800 -16.349  1.00 67.90           C  \\nANISOU   14  CD  GLU A  72    11070   8100   6628  -1563   -300   -410       C  \\nATOM     15  OE1 GLU A  72      31.018   3.327 -15.272  1.00 68.55           O  \\nANISOU   15  OE1 GLU A  72    11065   7996   6985  -1491   -160   -465       O  \\nATOM     16  OE2 GLU A  72      31.075   3.511 -17.462  1.00 79.31           O  \\nANISOU   16  OE2 GLU A  72    12751   9514   7868  -1553   -194   -519       O  \\nATOM     17  N   ILE A  73      27.238   7.766 -15.250  1.00 30.36           N  \\nANISOU   17  N   ILE A  73     5165   4244   2126  -1248  -1115    542       N  \\nATOM     18  CA  ILE A  73      27.532   9.124 -14.797  1.00 28.77           C  \\nANISOU   18  CA  ILE A  73     4824   3994   2114   -920   -993    745       C  \\nATOM     19  C   ILE A  73      26.893   9.424 -13.447  1.00 26.85           C  \\nANISOU   19  C   ILE A  73     4154   3812   2237   -780   -998    815       C  \\nATOM     20  O   ILE A  73      27.496  10.058 -12.581  1.00 24.23           O  \\nANISOU   20  O   ILE A  73     3756   3310   2141   -574   -773    839       O  \\nATOM     21  CB  ILE A  73      27.053  10.160 -15.815  1.00 34.40           C  \\nANISOU   21  CB  ILE A  73     5607   4872   2593   -799  -1219    987       C  \\nATOM     22  CG1 ILE A  73      27.291  11.568 -15.279  1.00 36.30           C  \\nANISOU   22  CG1 ILE A  73     5741   4990   3060   -462  -1102   1194       C  \\nATOM     23  CG2 ILE A  73      25.585   9.942 -16.166  1.00 46.47           C  \\nANISOU   23  CG2 ILE A  73     6867   6736   4054   -887  -1615   1059       C  \\nATOM     24  CD1 ILE A  73      28.740  11.858 -15.015  1.00 45.01           C  \\nANISOU   24  CD1 ILE A  73     7064   5789   4247   -413   -720   1120       C  \\nATOM     25  N   THR A  74      25.665   8.948 -13.292  1.00 40.83           N  \\nANISOU   25  N   THR A  74     5637   5847   4029   -921  -1252    836       N  \\nATOM     26  CA  THR A  74      24.932   9.005 -12.039  1.00 54.69           C  \\nANISOU   26  CA  THR A  74     6966   7727   6086   -849  -1236    876       C  \\nATOM     27  C   THR A  74      25.719   8.304 -10.935  1.00 33.06           C  \\nANISOU   27  C   THR A  74     4287   4741   3534   -920   -947    714       C  \\nATOM     28  O   THR A  74      25.960   8.870  -9.873  1.00 25.63           O  \\nANISOU   28  O   THR A  74     3197   3724   2819   -711   -764    744       O  \\nATOM     29  CB  THR A  74      23.528   8.346 -12.181  1.00 69.92           C  \\nANISOU   29  CB  THR A  74     8570  10023   7975  -1102  -1547    902       C  \\nATOM     30  OG1 THR A  74      23.670   6.968 -12.571  1.00 59.92           O  \\nANISOU   30  OG1 THR A  74     7546   8677   6544  -1516  -1596    711       O  \\nATOM     31  CG2 THR A  74      22.688   9.086 -13.223  1.00 60.41           C  \\nANISOU   31  CG2 THR A  74     7246   9113   6594   -991  -1891   1093       C  \\nATOM     32  N   LEU A  75      26.123   7.069 -11.215  1.00 30.54           N  \\nANISOU   32  N   LEU A  75     4220   4283   3099  -1205   -922    539       N  \\nATOM     33  CA  LEU A  75      26.853   6.255 -10.261  1.00 32.84           C  \\nANISOU   33  CA  LEU A  75     4607   4325   3546  -1271   -697    405       C  \\nATOM     34  C   LEU A  75      28.206   6.852  -9.911  1.00 25.79           C  \\nANISOU   34  C   LEU A  75     3865   3180   2754  -1010   -424    380       C  \\nATOM     35  O   LEU A  75      28.625   6.828  -8.754  1.00 24.52           O  \\nANISOU   35  O   LEU A  75     3612   2910   2796   -922   -264    366       O  \\nATOM     36  CB  LEU A  75      27.029   4.843 -10.809  1.00 28.01           C  \\nANISOU   36  CB  LEU A  75     4304   3560   2777  -1589   -744    220       C  \\nATOM     37  CG  LEU A  75      25.766   3.999 -10.722  1.00 27.87           C  \\nANISOU   37  CG  LEU A  75     4119   3735   2737  -1948   -981    222       C  \\nATOM     38  CD1 LEU A  75      26.030   2.653 -11.357  1.00 37.84           C  \\nANISOU   38  CD1 LEU A  75     5752   4769   3858  -2133   -958     39       C  \\nATOM     39  CD2 LEU A  75      25.374   3.858  -9.264  1.00 31.17           C  \\nANISOU   39  CD2 LEU A  75     4232   4196   3414  -1954   -873    303       C  \\nATOM     40  N   ILE A  76      28.895   7.392 -10.905  1.00 19.81           N  \\nANISOU   40  N   ILE A  76     3334   2354   1837   -914   -376    382       N  \\nATOM     41  CA  ILE A  76      30.199   7.987 -10.648  1.00 25.42           C  \\nANISOU   41  CA  ILE A  76     4150   2861   2646   -719   -117    364       C  \\nATOM     42  C   ILE A  76      30.078   9.202  -9.725  1.00 21.46           C  \\nANISOU   42  C   ILE A  76     3413   2382   2358   -495    -72    500       C  \\nATOM     43  O   ILE A  76      30.821   9.326  -8.751  1.00 19.33           O  \\nANISOU   43  O   ILE A  76     3097   1975   2274   -406     97    454       O  \\nATOM     44  CB  ILE A  76      30.897   8.377 -11.962  1.00 18.98           C  \\nANISOU   44  CB  ILE A  76     3616   2007   1589   -703    -49    365       C  \\nATOM     45  CG1 ILE A  76      31.346   7.115 -12.698  1.00 26.36           C  \\nANISOU   45  CG1 ILE A  76     4836   2851   2328   -878      5    155       C  \\nATOM     46  CG2 ILE A  76      32.095   9.264 -11.691  1.00 17.37           C  \\nANISOU   46  CG2 ILE A  76     3437   1654   1510   -533    201    401       C  \\nATOM     47  CD1 ILE A  76      31.830   7.370 -14.096  1.00 33.92           C  \\nANISOU   47  CD1 ILE A  76     6090   3835   2962   -903     69    138       C  \\nATOM     48  N   ILE A  77      29.127  10.081 -10.014  1.00 17.69           N  \\nANISOU   48  N   ILE A  77     2797   2074   1850   -390   -239    659       N  \\nATOM     49  CA  ILE A  77      28.965  11.285  -9.221  1.00 17.04           C  \\nANISOU   49  CA  ILE A  77     2546   1970   1957   -140   -190    764       C  \\nATOM     50  C   ILE A  77      28.530  10.901  -7.817  1.00 15.87           C  \\nANISOU   50  C   ILE A  77     2141   1884   2003   -135   -137    701       C  \\nATOM     51  O   ILE A  77      29.012  11.457  -6.840  1.00 14.34           O  \\nANISOU   51  O   ILE A  77     1914   1569   1964      3     13    673       O  \\nATOM     52  CB  ILE A  77      27.958  12.256  -9.867  1.00 32.82           C  \\nANISOU   52  CB  ILE A  77     4453   4127   3892     28   -399    957       C  \\nATOM     53  CG1 ILE A  77      28.527  12.779 -11.191  1.00 28.88           C  \\nANISOU   53  CG1 ILE A  77     4274   3532   3168     26   -424   1056       C  \\nATOM     54  CG2 ILE A  77      27.626  13.433  -8.929  1.00 28.78           C  \\nANISOU   54  CG2 ILE A  77     3765   3567   3602    331   -343   1032       C  \\nATOM     55  CD1 ILE A  77      27.508  13.469 -12.067  1.00 24.61           C  \\nANISOU   55  CD1 ILE A  77     3698   3167   2486    155   -703   1271       C  \\nATOM     56  N   PHE A  78      27.644   9.923  -7.722  1.00 20.44           N  \\nANISOU   56  N   PHE A  78     2565   2655   2547   -323   -258    678       N  \\nATOM     57  CA  PHE A  78      27.223   9.413  -6.424  1.00 29.06           C  \\nANISOU   57  CA  PHE A  78     3441   3824   3777   -381   -185    638       C  \\nATOM     58  C   PHE A  78      28.424   8.921  -5.639  1.00 18.03           C  \\nANISOU   58  C   PHE A  78     2225   2175   2450   -414      5    527       C  \\nATOM     59  O   PHE A  78      28.580   9.231  -4.459  1.00 15.70           O  \\nANISOU   59  O   PHE A  78     1836   1857   2272   -311    126    514       O  \\nATOM     60  CB  PHE A  78      26.204   8.271  -6.578  1.00 27.90           C  \\nANISOU   60  CB  PHE A  78     3148   3890   3561   -681   -340    636       C  \\nATOM     61  CG  PHE A  78      26.034   7.437  -5.332  1.00 22.44           C  \\nANISOU   61  CG  PHE A  78     2353   3209   2963   -840   -228    598       C  \\nATOM     62  CD1 PHE A  78      25.276   7.901  -4.270  1.00 18.61           C  \\nANISOU   62  CD1 PHE A  78     1545   2938   2588   -734   -147    659       C  \\nATOM     63  CD2 PHE A  78      26.641   6.192  -5.225  1.00 24.51           C  \\nANISOU   63  CD2 PHE A  78     2869   3255   3189  -1079   -191    508       C  \\nATOM     64  CE1 PHE A  78      25.123   7.138  -3.128  1.00 22.90           C  \\nANISOU   64  CE1 PHE A  78     2025   3507   3168   -907    -27    650       C  \\nATOM     65  CE2 PHE A  78      26.495   5.426  -4.086  1.00 28.95           C  \\nANISOU   65  CE2 PHE A  78     3396   3794   3809  -1191    -99    498       C  \\nATOM     66  CZ  PHE A  78      25.735   5.900  -3.036  1.00 29.26           C  \\nANISOU   66  CZ  PHE A  78     3134   4062   3920  -1117    -16    569       C  \\nATOM     67  N   GLY A  79      29.267   8.143  -6.306  1.00 16.77           N  \\nANISOU   67  N   GLY A  79     2326   1842   2204   -542     25    441       N  \\nATOM     68  CA  GLY A  79      30.397   7.527  -5.649  1.00 17.63           C  \\nANISOU   68  CA  GLY A  79     2578   1732   2390   -550    170    345       C  \\nATOM     69  C   GLY A  79      31.372   8.545  -5.111  1.00 10.12           C  \\nANISOU   69  C   GLY A  79     1596    716   1533   -325    273    327       C  \\nATOM     70  O   GLY A  79      31.913   8.383  -4.016  1.00 15.97           O  \\nANISOU   70  O   GLY A  79     2272   1484   2310   -256    275    257       O  \\nATOM     71  N   VAL A  80      31.591   9.601  -5.877  1.00 10.40           N  \\nANISOU   71  N   VAL A  80     1690    719   1543   -227    301    379       N  \\nATOM     72  CA  VAL A  80      32.536  10.628  -5.468  1.00 11.18           C  \\nANISOU   72  CA  VAL A  80     1745    819   1684    -83    338    312       C  \\nATOM     73  C   VAL A  80      32.013  11.384  -4.245  1.00 13.80           C  \\nANISOU   73  C   VAL A  80     1956   1172   2116     24    367    331       C  \\nATOM     74  O   VAL A  80      32.725  11.539  -3.271  1.00 10.65           O  \\nANISOU   74  O   VAL A  80     1511    805   1730     46    389    236       O  \\nATOM     75  CB  VAL A  80      32.830  11.609  -6.604  1.00 13.99           C  \\nANISOU   75  CB  VAL A  80     2206   1127   1982    -46    353    378       C  \\nATOM     76  CG1 VAL A  80      33.718  12.724  -6.111  1.00 17.30           C  \\nANISOU   76  CG1 VAL A  80     2580   1539   2454     21    389    320       C  \\nATOM     77  CG2 VAL A  80      33.500  10.886  -7.770  1.00 13.03           C  \\nANISOU   77  CG2 VAL A  80     2225    983   1741   -137    407    339       C  \\nATOM     78  N   ILE A  81      30.763  11.834  -4.294  1.00 13.20           N  \\nANISOU   78  N   ILE A  81     1795   1113   2106    113    358    485       N  \\nATOM     79  CA  ILE A  81      30.179  12.589  -3.190  1.00 11.57           C  \\nANISOU   79  CA  ILE A  81     1445    961   1989    277    400    478       C  \\nATOM     80  C   ILE A  81      30.108  11.708  -1.940  1.00 13.01           C  \\nANISOU   80  C   ILE A  81     1533   1227   2182    176    459    408       C  \\nATOM     81  O   ILE A  81      30.520  12.124  -0.855  1.00 16.50           O  \\nANISOU   81  O   ILE A  81     1995   1608   2667    245    548    337       O  \\nATOM     82  CB  ILE A  81      28.752  13.144  -3.549  1.00 17.96           C  \\nANISOU   82  CB  ILE A  81     2062   1970   2793    435    299    580       C  \\nATOM     83  CG1 ILE A  81      28.846  14.286  -4.572  1.00 19.73           C  \\nANISOU   83  CG1 ILE A  81     2429   2060   3007    602    234    685       C  \\nATOM     84  CG2 ILE A  81      28.016  13.617  -2.294  1.00 17.99           C  \\nANISOU   84  CG2 ILE A  81     1869   2090   2875    608    390    536       C  \\nATOM     85  CD1 ILE A  81      27.489  14.757  -5.143  1.00 20.93           C  \\nANISOU   85  CD1 ILE A  81     2389   2417   3146    790     73    823       C  \\nATOM     86  N   ALA A  82      29.615  10.479  -2.093  1.00 14.62           N  \\nANISOU   86  N   ALA A  82     1675   1555   2325    -16    399    431       N  \\nATOM     87  CA  ALA A  82      29.518   9.568  -0.954  1.00 15.60           C  \\nANISOU   87  CA  ALA A  82     1756   1734   2436   -146    451    410       C  \\nATOM     88  C   ALA A  82      30.904   9.253  -0.397  1.00 17.38           C  \\nANISOU   88  C   ALA A  82     2169   1796   2638   -135    450    304       C  \\nATOM     89  O   ALA A  82      31.093   9.224   0.822  1.00 14.20           O  \\nANISOU   89  O   ALA A  82     1762   1408   2225   -118    497    282       O  \\nATOM     90  CB  ALA A  82      28.785   8.280  -1.339  1.00 12.86           C  \\nANISOU   90  CB  ALA A  82     1365   1496   2025   -401    364    459       C  \\nATOM     91  N   GLY A  83      31.866   9.017  -1.287  1.00  9.24           N  \\nANISOU   91  N   GLY A  83     1253    707   1549   -128    365    233       N  \\nATOM     92  CA  GLY A  83      33.240   8.762  -0.875  1.00 13.08           C  \\nANISOU   92  CA  GLY A  83     1783   1188   1997    -85    327    150       C  \\nATOM     93  C   GLY A  83      33.832   9.924  -0.086  1.00 12.02           C  \\nANISOU   93  C   GLY A  83     1606   1064   1896     -5    381    116       C  \\nATOM     94  O   GLY A  83      34.436   9.734   0.966  1.00 14.41           O  \\nANISOU   94  O   GLY A  83     1913   1363   2200     -6    377     98       O  \\nATOM     95  N   VAL A  84      33.648  11.136  -0.584  1.00 11.13           N  \\nANISOU   95  N   VAL A  84     1484    936   1808     43    418    124       N  \\nATOM     96  CA  VAL A  84      34.146  12.312   0.104  1.00  9.86           C  \\nANISOU   96  CA  VAL A  84     1332    751   1663     74    438     92       C  \\nATOM     97  C   VAL A  84      33.478  12.454   1.478  1.00 23.28           C  \\nANISOU   97  C   VAL A  84     3029   2426   3392    111    507     85       C  \\nATOM     98  O   VAL A  84      34.162  12.648   2.494  1.00 13.72           O  \\nANISOU   98  O   VAL A  84     1857   1190   2165     98    492     31       O  \\nATOM     99  CB  VAL A  84      33.919  13.579  -0.731  1.00 11.25           C  \\nANISOU   99  CB  VAL A  84     1553    869   1854    123    442    115       C  \\nATOM    100  CG1 VAL A  84      34.115  14.841   0.110  1.00 10.57           C  \\nANISOU  100  CG1 VAL A  84     1521    705   1792    167    448     70       C  \\nATOM    101  CG2 VAL A  84      34.850  13.573  -1.937  1.00 17.63           C  \\nANISOU  101  CG2 VAL A  84     2388   1661   2650     80    458    124       C  \\nATOM    102  N   ILE A  85      32.154  12.329   1.520  1.00 14.94           N  \\nANISOU  102  N   ILE A  85     1923   1361   2394    179    603    145       N  \\nATOM    103  CA  ILE A  85      31.434  12.529   2.766  1.00  9.86           C  \\nANISOU  103  CA  ILE A  85     1243    764   1739    273    714    109       C  \\nATOM    104  C   ILE A  85      31.759  11.396   3.733  1.00 13.41           C  \\nANISOU  104  C   ILE A  85     1720   1264   2112    139    730    122       C  \\nATOM    105  O   ILE A  85      32.010  11.623   4.920  1.00 12.84           O  \\nANISOU  105  O   ILE A  85     1713   1218   1946    152    765     51       O  \\nATOM    106  CB  ILE A  85      29.908  12.608   2.539  1.00 11.50           C  \\nANISOU  106  CB  ILE A  85     1248   1175   1948    376    772    165       C  \\nATOM    107  CG1 ILE A  85      29.540  13.861   1.742  1.00 12.43           C  \\nANISOU  107  CG1 ILE A  85     1381   1240   2101    537    698    162       C  \\nATOM    108  CG2 ILE A  85      29.158  12.574   3.871  1.00 13.13           C  \\nANISOU  108  CG2 ILE A  85     1364   1588   2037    399    874    114       C  \\nATOM    109  CD1 ILE A  85      28.073  13.892   1.307  1.00 21.23           C  \\nANISOU  109  CD1 ILE A  85     2243   2580   3242    666    702    240       C  \\nATOM    110  N   GLY A  86      31.759  10.176   3.214  1.00  9.78           N  \\nANISOU  110  N   GLY A  86     1252    811   1654     -9    668    216       N  \\nATOM    111  CA  GLY A  86      32.067   9.016   4.018  1.00 10.45           C  \\nANISOU  111  CA  GLY A  86     1408    923   1640   -133    602    253       C  \\nATOM    112  C   GLY A  86      33.429   9.138   4.676  1.00 13.84           C  \\nANISOU  112  C   GLY A  86     1950   1238   2069    -97    542    201       C  \\nATOM    113  O   GLY A  86      33.573   8.900   5.878  1.00 11.38           O  \\nANISOU  113  O   GLY A  86     1720    951   1652   -142    558    232       O  \\nATOM    114  N   THR A  87      34.433   9.509   3.889  1.00 12.99           N  \\nANISOU  114  N   THR A  87     1814   1108   2012    -35    437    126       N  \\nATOM    115  CA  THR A  87      35.804   9.618   4.400  1.00 13.69           C  \\nANISOU  115  CA  THR A  87     1935   1150   2118    -27    347     91       C  \\nATOM    116  C   THR A  87      35.930  10.647   5.525  1.00 12.38           C  \\nANISOU  116  C   THR A  87     1850    902   1951    -16    409     19       C  \\nATOM    117  O   THR A  87      36.549  10.379   6.559  1.00 12.81           O  \\nANISOU  117  O   THR A  87     1982    937   1949    -48    338     17       O  \\nATOM    118  CB  THR A  87      36.779   9.989   3.278  1.00 17.27           C  \\nANISOU  118  CB  THR A  87     2321   1640   2602     -6    290     50       C  \\nATOM    119  OG1 THR A  87      36.788   8.937   2.313  1.00 18.68           O  \\nANISOU  119  OG1 THR A  87     2482   1837   2777     -8    289     77       O  \\nATOM    120  CG2 THR A  87      38.187  10.186   3.824  1.00 16.57           C  \\nANISOU  120  CG2 THR A  87     2227   1480   2587    -10    237     21       C  \\nATOM    121  N   ILE A  88      35.341  11.822   5.302  1.00 17.44           N  \\nANISOU  121  N   ILE A  88     2481   1553   2593     40    488    -55       N  \\nATOM    122  CA  ILE A  88      35.267  12.888   6.299  1.00 19.16           C  \\nANISOU  122  CA  ILE A  88     2814   1716   2751     84    545   -196       C  \\nATOM    123  C   ILE A  88      34.589  12.416   7.594  1.00 19.87           C  \\nANISOU  123  C   ILE A  88     2959   1969   2623     75    598   -200       C  \\nATOM    124  O   ILE A  88      35.102  12.650   8.689  1.00 15.46           O  \\nANISOU  124  O   ILE A  88     2516   1441   1917     35    536   -287       O  \\nATOM    125  CB  ILE A  88      34.504  14.104   5.735  1.00 13.93           C  \\nANISOU  125  CB  ILE A  88     2113   1077   2102    181    576   -223       C  \\nATOM    126  CG1 ILE A  88      35.327  14.779   4.646  1.00 13.96           C  \\nANISOU  126  CG1 ILE A  88     2082   1046   2176    134    455   -189       C  \\nATOM    127  CG2 ILE A  88      34.123  15.087   6.842  1.00 16.44           C  \\nANISOU  127  CG2 ILE A  88     2550   1378   2318    268    642   -381       C  \\nATOM    128  CD1 ILE A  88      34.637  15.963   4.066  1.00 18.02           C  \\nANISOU  128  CD1 ILE A  88     2625   1515   2706    226    466   -198       C  \\nATOM    129  N   LEU A  89      33.448  11.740   7.460  1.00 11.97           N  \\nANISOU  129  N   LEU A  89     1866   1110   1573     77    699    -94       N  \\nATOM    130  CA  LEU A  89      32.719  11.270   8.622  1.00 15.64           C  \\nANISOU  130  CA  LEU A  89     2357   1779   1807     30    786    -64       C  \\nATOM    131  C   LEU A  89      33.496  10.176   9.350  1.00 17.03           C  \\nANISOU  131  C   LEU A  89     2650   1960   1862   -109    644     50       C  \\nATOM    132  O   LEU A  89      33.421  10.062  10.574  1.00 22.83           O  \\nANISOU  132  O   LEU A  89     3503   2822   2348   -157    660     47       O  \\nATOM    133  CB  LEU A  89      31.337  10.757   8.223  1.00 15.16           C  \\nANISOU  133  CB  LEU A  89     2118   1893   1750      8    923     44       C  \\nATOM    134  CG  LEU A  89      30.305  11.772   7.741  1.00 18.76           C  \\nANISOU  134  CG  LEU A  89     2413   2428   2288    194   1065    -39       C  \\nATOM    135  CD1 LEU A  89      28.953  11.097   7.500  1.00 18.26           C  \\nANISOU  135  CD1 LEU A  89     2108   2619   2212    118   1127     82       C  \\nATOM    136  CD2 LEU A  89      30.145  12.864   8.755  1.00 17.08           C  \\nANISOU  136  CD2 LEU A  89     2287   2258   1946    347   1167   -223       C  \\nATOM    137  N   LEU A  90      34.249   9.376   8.604  1.00 17.01           N  \\nANISOU  137  N   LEU A  90     2634   1814   2016   -149    506    153       N  \\nATOM    138  CA  LEU A  90      35.074   8.327   9.215  1.00 21.28           C  \\nANISOU  138  CA  LEU A  90     3288   2315   2484   -214    341    279       C  \\nATOM    139  C   LEU A  90      36.266   8.919   9.981  1.00 19.34           C  \\nANISOU  139  C   LEU A  90     3105   2064   2180   -180    177    183       C  \\nATOM    140  O   LEU A  90      36.646   8.435  11.049  1.00 18.95           O  \\nANISOU  140  O   LEU A  90     3181   2085   1935   -222     50    260       O  \\nATOM    141  CB  LEU A  90      35.559   7.351   8.143  1.00 14.32           C  \\nANISOU  141  CB  LEU A  90     2375   1259   1808   -208    262    380       C  \\nATOM    142  CG  LEU A  90      35.982   5.938   8.518  1.00 26.43           C  \\nANISOU  142  CG  LEU A  90     4044   2695   3303   -244    131    560       C  \\nATOM    143  CD1 LEU A  90      34.949   5.281   9.413  1.00 31.06           C  \\nANISOU  143  CD1 LEU A  90     4753   3391   3656   -397    202    703       C  \\nATOM    144  CD2 LEU A  90      36.156   5.148   7.244  1.00 21.46           C  \\nANISOU  144  CD2 LEU A  90     3291   1963   2900   -193    106    550       C  \\nATOM    145  N   ILE A  91      36.861   9.967   9.434  1.00 16.80           N  \\nANISOU  145  N   ILE A  91     2705   1662   2015   -132    162     29       N  \\nATOM    146  CA  ILE A  91      37.923  10.669  10.141  1.00 16.69           C  \\nANISOU  146  CA  ILE A  91     2731   1659   1952   -161      0    -90       C  \\nATOM    147  C   ILE A  91      37.387  11.282  11.438  1.00 21.02           C  \\nANISOU  147  C   ILE A  91     3458   2337   2192   -197     42   -207       C  \\nATOM    148  O   ILE A  91      37.971  11.111  12.510  1.00 23.14           O  \\nANISOU  148  O   ILE A  91     3838   2702   2251   -260   -131   -205       O  \\nATOM    149  CB  ILE A  91      38.548  11.761   9.261  1.00 16.48           C  \\nANISOU  149  CB  ILE A  91     2608   1501   2152   -163     12   -230       C  \\nATOM    150  CG1 ILE A  91      39.209  11.120   8.036  1.00 14.90           C  \\nANISOU  150  CG1 ILE A  91     2238   1215   2208   -130     -8   -127       C  \\nATOM    151  CG2 ILE A  91      39.553  12.591  10.062  1.00 20.71           C  \\nANISOU  151  CG2 ILE A  91     3190   2054   2625   -264   -162   -378       C  \\nATOM    152  CD1 ILE A  91      39.872  12.107   7.107  1.00 15.47           C  \\nANISOU  152  CD1 ILE A  91     2214   1180   2484   -172     32   -222       C  \\nATOM    153  N   SER A  92      36.265  11.984  11.320  1.00 21.14           N  \\nANISOU  153  N   SER A  92     3497   2366   2168   -137    273   -310       N  \\nATOM    154  CA  SER A  92      35.596  12.619  12.448  1.00 19.47           C  \\nANISOU  154  CA  SER A  92     3452   2280   1664   -123    395   -459       C  \\nATOM    155  C   SER A  92      35.332  11.605  13.560  1.00 21.28           C  \\nANISOU  155  C   SER A  92     3792   2717   1575   -208    375   -314       C  \\nATOM    156  O   SER A  92      35.657  11.825  14.733  1.00 23.88           O  \\nANISOU  156  O   SER A  92     4318   3152   1603   -269    293   -398       O  \\nATOM    157  CB  SER A  92      34.292  13.264  11.965  1.00 29.49           C  \\nANISOU  157  CB  SER A  92     4650   3558   2997     23    673   -538       C  \\nATOM    158  OG  SER A  92      33.609  13.945  12.998  1.00 39.92           O  \\nANISOU  158  OG  SER A  92     6117   5000   4052     92    844   -718       O  \\nATOM    159  N   TYR A  93      34.755  10.473  13.174  1.00 26.54           N  \\nANISOU  159  N   TYR A  93     4366   3429   2290   -241    439    -86       N  \\nATOM    160  CA  TYR A  93      34.474   9.408  14.121  1.00 27.68           C  \\nANISOU  160  CA  TYR A  93     4642   3725   2149   -357    429    110       C  \\nATOM    161  C   TYR A  93      35.794   8.928  14.720  1.00 27.37           C  \\nANISOU  161  C   TYR A  93     4738   3639   2021   -399    102    200       C  \\nATOM    162  O   TYR A  93      35.933   8.818  15.933  1.00 28.88           O  \\nANISOU  162  O   TYR A  93     5135   3977   1861   -470     24    228       O  \\nATOM    163  CB  TYR A  93      33.712   8.267  13.436  1.00 21.26           C  \\nANISOU  163  CB  TYR A  93     3721   2895   1460   -429    526    339       C  \\nATOM    164  CG  TYR A  93      33.489   7.046  14.302  1.00 33.29           C  \\nANISOU  164  CG  TYR A  93     5422   4503   2724   -592    501    595       C  \\nATOM    165  CD1 TYR A  93      32.502   7.034  15.280  1.00 28.66           C  \\nANISOU  165  CD1 TYR A  93     4837   4158   1893   -651    694    604       C  \\nATOM    166  CD2 TYR A  93      34.254   5.906  14.132  1.00 23.51           C  \\nANISOU  166  CD2 TYR A  93     4283   3078   1573   -625    269    812       C  \\nATOM    167  CE1 TYR A  93      32.295   5.937  16.063  1.00 29.00           C  \\nANISOU  167  CE1 TYR A  93     4966   4240   1811   -750    652    817       C  \\nATOM    168  CE2 TYR A  93      34.049   4.806  14.910  1.00 26.12           C  \\nANISOU  168  CE2 TYR A  93     4692   3432   1802   -692    224    999       C  \\nATOM    169  CZ  TYR A  93      33.070   4.827  15.874  1.00 32.94           C  \\nANISOU  169  CZ  TYR A  93     5569   4524   2424   -777    417   1012       C  \\nATOM    170  OH  TYR A  93      32.866   3.724  16.653  1.00 37.68           O  \\nANISOU  170  OH  TYR A  93     6289   5122   2905   -863    388   1217       O  \\nATOM    171  N   GLY A  94      36.775   8.688  13.859  1.00 27.56           N  \\nANISOU  171  N   GLY A  94     4631   3486   2355   -341    -90    240       N  \\nATOM    172  CA  GLY A  94      38.087   8.260  14.302  1.00 26.35           C  \\nANISOU  172  CA  GLY A  94     4513   3313   2186   -331   -418    327       C  \\nATOM    173  C   GLY A  94      38.758   9.167  15.325  1.00 30.85           C  \\nANISOU  173  C   GLY A  94     5189   4010   2522   -387   -592    152       C  \\nATOM    174  O   GLY A  94      39.384   8.678  16.259  1.00 34.40           O  \\nANISOU  174  O   GLY A  94     5764   4561   2745   -420   -848    270       O  \\nATOM    175  N   ILE A  95      38.646  10.484  15.172  1.00 30.72           N  \\nANISOU  175  N   ILE A  95     5156   3973   2545   -403   -483   -126       N  \\nATOM    176  CA  ILE A  95      39.352  11.367  16.102  1.00 36.00           C  \\nANISOU  176  CA  ILE A  95     5959   4723   2995   -499   -676   -329       C  \\nATOM    177  C   ILE A  95      38.577  11.567  17.405  1.00 41.88           C  \\nANISOU  177  C   ILE A  95     7009   5651   3252   -556   -564   -411       C  \\nATOM    178  O   ILE A  95      39.175  11.835  18.445  1.00 44.79           O  \\nANISOU  178  O   ILE A  95     7564   6144   3311   -660   -790   -498       O  \\nATOM    179  CB  ILE A  95      39.666  12.730  15.469  1.00 31.59           C  \\nANISOU  179  CB  ILE A  95     5328   4008   2665   -527   -628   -604       C  \\nATOM    180  CG1 ILE A  95      38.392  13.480  15.107  1.00 44.59           C  \\nANISOU  180  CG1 ILE A  95     7048   5569   4326   -434   -264   -754       C  \\nATOM    181  CG2 ILE A  95      40.500  12.536  14.236  1.00 28.34           C  \\nANISOU  181  CG2 ILE A  95     4621   3465   2683   -501   -716   -515       C  \\nATOM    182  CD1 ILE A  95      38.625  14.614  14.129  1.00 56.29           C  \\nANISOU  182  CD1 ILE A  95     8453   6814   6122   -422   -200   -928       C  \\nATOM    183  N   ARG A  96      37.255  11.416  17.350  1.00 44.07           N  \\nANISOU  183  N   ARG A  96     7323   5977   3443   -500   -216   -383       N  \\nATOM    184  CA  ARG A  96      36.445  11.402  18.559  1.00 48.82           C  \\nANISOU  184  CA  ARG A  96     8152   6800   3599   -539    -44   -413       C  \\nATOM    185  C   ARG A  96      36.929  10.279  19.462  1.00 76.51           C  \\nANISOU  185  C   ARG A  96    11748  10437   6886   -616   -281   -135       C  \\nATOM    186  O   ARG A  96      37.215  10.485  20.641  1.00 72.86           O  \\nANISOU  186  O   ARG A  96    11427  10122   6136   -653   -391   -197       O  \\nATOM    187  CB  ARG A  96      34.965  11.213  18.224  1.00 37.76           C  \\nANISOU  187  CB  ARG A  96     6623   5470   2255   -460    359   -360       C  \\nATOM    188  N   ARG A  97      37.055   9.096  18.870  1.00109.45           N  \\nANISOU  188  N   ARG A  97    15842  14523  11220   -617   -370    171       N  \\nATOM    189  CA  ARG A  97      37.355   7.874  19.604  1.00126.28           C  \\nANISOU  189  CA  ARG A  97    18046  16706  13230   -638   -546    477       C  \\nATOM    190  C   ARG A  97      38.833   7.466  19.569  1.00135.45           C  \\nANISOU  190  C   ARG A  97    19165  17788  14510   -587   -990    594       C  \\nATOM    191  O   ARG A  97      39.151   6.276  19.611  1.00149.94           O  \\nANISOU  191  O   ARG A  97    20993  19546  16431   -527  -1126    878       O  \\nATOM    192  CB  ARG A  97      36.497   6.726  19.058  1.00126.38           C  \\nANISOU  192  CB  ARG A  97    17986  16628  13406   -649   -349    735       C  \\nATOM    193  N   LEU A  98      39.734   8.442  19.490  1.00 91.41           N  \\nANISOU  193  N   LEU A  98    13524  12225   8981   -599  -1204    365       N  \\nATOM    194  CA  LEU A  98      41.164   8.149  19.561  1.00 61.19           C  \\nANISOU  194  CA  LEU A  98     9556   8401   5292   -545  -1627    456       C  \\nATOM    195  C   LEU A  98      41.950   9.379  19.988  1.00 66.32           C  \\nANISOU  195  C   LEU A  98    10163   9158   5878   -647  -1813    155       C  \\nATOM    196  O   LEU A  98      43.006   9.262  20.609  1.00 71.98           O  \\nANISOU  196  O   LEU A  98    10796  10000   6555   -646  -2141    198       O  \\nATOM    197  CB  LEU A  98      41.686   7.629  18.218  1.00 51.35           C  \\nANISOU  197  CB  LEU A  98     8064   6955   4491   -419  -1724    582       C  \\nTER     198      LEU A  98                                                      \\nMASTER      286    0    0    1    0    0    0    6  197    1    0    3          \\nEND                                                                             \\n\",\"pdb\");\n",
              "\tviewer_16642132334955251.addModel(\"HEADER                                            26-SEP-22   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 26-SEP-22                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2022.34+release.9ec33c9fe00                                 \\nATOM      1  N   ASP A   1      -4.602  -2.188 -13.100  1.00  0.00           N  \\nATOM      2  CA  ASP A   1      -4.515  -1.079 -14.042  1.00  0.00           C  \\nATOM      3  C   ASP A   1      -3.242  -1.155 -14.870  1.00  0.00           C  \\nATOM      4  O   ASP A   1      -2.211  -1.628 -14.392  1.00  0.00           O  \\nATOM      5  CB  ASP A   1      -4.568   0.258 -13.303  1.00  0.00           C  \\nATOM      6  CG  ASP A   1      -5.925   0.531 -12.670  1.00  0.00           C  \\nATOM      7  OD1 ASP A   1      -6.872  -0.126 -13.033  1.00  0.00           O  \\nATOM      8  OD2 ASP A   1      -6.002   1.394 -11.829  1.00  0.00           O  \\nATOM      9 1H   ASP A   1      -5.456  -2.111 -12.566  1.00  0.00           H  \\nATOM     10 2H   ASP A   1      -4.600  -3.062 -13.606  1.00  0.00           H  \\nATOM     11 3H   ASP A   1      -3.810  -2.161 -12.473  1.00  0.00           H  \\nATOM     12  HA  ASP A   1      -5.366  -1.136 -14.722  1.00  0.00           H  \\nATOM     13 1HB  ASP A   1      -3.808   0.272 -12.521  1.00  0.00           H  \\nATOM     14 2HB  ASP A   1      -4.338   1.068 -13.998  1.00  0.00           H  \\nATOM     15  N   ASN A   2      -3.320  -0.690 -16.118  1.00  0.00           N  \\nATOM     16  CA  ASN A   2      -2.167  -0.705 -17.011  1.00  0.00           C  \\nATOM     17  C   ASN A   2      -1.332   0.561 -16.873  1.00  0.00           C  \\nATOM     18  O   ASN A   2      -0.326   0.721 -17.564  1.00  0.00           O  \\nATOM     19  CB  ASN A   2      -2.613  -0.893 -18.447  1.00  0.00           C  \\nATOM     20  CG  ASN A   2      -3.139  -2.274 -18.712  1.00  0.00           C  \\nATOM     21  OD1 ASN A   2      -2.781  -3.232 -18.016  1.00  0.00           O  \\nATOM     22  ND2 ASN A   2      -3.982  -2.397 -19.704  1.00  0.00           N  \\nATOM     23  H   ASN A   2      -4.194  -0.310 -16.453  1.00  0.00           H  \\nATOM     24  HA  ASN A   2      -1.524  -1.542 -16.733  1.00  0.00           H  \\nATOM     25 1HB  ASN A   2      -3.392  -0.168 -18.684  1.00  0.00           H  \\nATOM     26 2HB  ASN A   2      -1.774  -0.702 -19.116  1.00  0.00           H  \\nATOM     27 1HD2 ASN A   2      -4.366  -3.294 -19.927  1.00  0.00           H  \\nATOM     28 2HD2 ASN A   2      -4.244  -1.595 -20.240  1.00  0.00           H  \\nATOM     29  N   GLY A   3      -1.756   1.457 -15.990  1.00  0.00           N  \\nATOM     30  CA  GLY A   3      -1.045   2.708 -15.790  1.00  0.00           C  \\nATOM     31  C   GLY A   3       0.205   2.438 -15.001  1.00  0.00           C  \\nATOM     32  O   GLY A   3       0.356   1.369 -14.408  1.00  0.00           O  \\nATOM     33  H   GLY A   3      -2.585   1.271 -15.443  1.00  0.00           H  \\nATOM     34 1HA  GLY A   3      -0.806   3.149 -16.758  1.00  0.00           H  \\nATOM     35 2HA  GLY A   3      -1.690   3.413 -15.267  1.00  0.00           H  \\nATOM     36  N   GLU A   4       1.119   3.393 -14.974  1.00  0.00           N  \\nATOM     37  CA  GLU A   4       2.325   3.194 -14.211  1.00  0.00           C  \\nATOM     38  C   GLU A   4       1.981   3.052 -12.749  1.00  0.00           C  \\nATOM     39  O   GLU A   4       1.208   3.836 -12.197  1.00  0.00           O  \\nATOM     40  CB  GLU A   4       3.295   4.356 -14.418  1.00  0.00           C  \\nATOM     41  CG  GLU A   4       4.631   4.186 -13.727  1.00  0.00           C  \\nATOM     42  CD  GLU A   4       5.573   5.325 -13.989  1.00  0.00           C  \\nATOM     43  OE1 GLU A   4       5.888   5.561 -15.131  1.00  0.00           O  \\nATOM     44  OE2 GLU A   4       5.978   5.961 -13.046  1.00  0.00           O  \\nATOM     45  H   GLU A   4       0.967   4.256 -15.475  1.00  0.00           H  \\nATOM     46  HA  GLU A   4       2.806   2.278 -14.557  1.00  0.00           H  \\nATOM     47 1HB  GLU A   4       3.483   4.488 -15.483  1.00  0.00           H  \\nATOM     48 2HB  GLU A   4       2.843   5.277 -14.049  1.00  0.00           H  \\nATOM     49 1HG  GLU A   4       4.465   4.105 -12.653  1.00  0.00           H  \\nATOM     50 2HG  GLU A   4       5.087   3.258 -14.067  1.00  0.00           H  \\nATOM     51  N   TRP A   5       2.548   2.050 -12.107  1.00  0.00           N  \\nATOM     52  CA  TRP A   5       2.299   1.806 -10.705  1.00  0.00           C  \\nATOM     53  C   TRP A   5       2.998   2.843  -9.869  1.00  0.00           C  \\nATOM     54  O   TRP A   5       4.179   3.018 -10.072  1.00  0.00           O  \\nATOM     55  CB  TRP A   5       2.772   0.420 -10.304  1.00  0.00           C  \\nATOM     56  CG  TRP A   5       1.944  -0.683 -10.879  1.00  0.00           C  \\nATOM     57  CD1 TRP A   5       2.252  -1.457 -11.957  1.00  0.00           C  \\nATOM     58  CD2 TRP A   5       0.658  -1.145 -10.407  1.00  0.00           C  \\nATOM     59  NE1 TRP A   5       1.249  -2.366 -12.189  1.00  0.00           N  \\nATOM     60  CE2 TRP A   5       0.265  -2.191 -11.249  1.00  0.00           C  \\nATOM     61  CE3 TRP A   5      -0.182  -0.764  -9.353  1.00  0.00           C  \\nATOM     62  CZ2 TRP A   5      -0.934  -2.864 -11.075  1.00  0.00           C  \\nATOM     63  CZ3 TRP A   5      -1.385  -1.440  -9.178  1.00  0.00           C  \\nATOM     64  CH2 TRP A   5      -1.749  -2.464 -10.017  1.00  0.00           C  \\nATOM     65  H   TRP A   5       3.170   1.438 -12.615  1.00  0.00           H  \\nATOM     66  HA  TRP A   5       1.226   1.870 -10.528  1.00  0.00           H  \\nATOM     67 1HB  TRP A   5       3.804   0.280 -10.630  1.00  0.00           H  \\nATOM     68 2HB  TRP A   5       2.757   0.331  -9.217  1.00  0.00           H  \\nATOM     69  HD1 TRP A   5       3.161  -1.368 -12.548  1.00  0.00           H  \\nATOM     70  HE1 TRP A   5       1.236  -3.051 -12.930  1.00  0.00           H  \\nATOM     71  HE3 TRP A   5       0.104   0.047  -8.682  1.00  0.00           H  \\nATOM     72  HZ2 TRP A   5      -1.241  -3.679 -11.731  1.00  0.00           H  \\nATOM     73  HZ3 TRP A   5      -2.033  -1.137  -8.355  1.00  0.00           H  \\nATOM     74  HH2 TRP A   5      -2.699  -2.973  -9.851  1.00  0.00           H  \\nATOM     75  N   ILE A   6       2.238   3.478  -8.999  1.00  0.00           N  \\nATOM     76  CA  ILE A   6       2.640   4.549  -8.107  1.00  0.00           C  \\nATOM     77  C   ILE A   6       3.836   4.210  -7.212  1.00  0.00           C  \\nATOM     78  O   ILE A   6       4.527   5.107  -6.730  1.00  0.00           O  \\nATOM     79  CB  ILE A   6       1.441   4.956  -7.226  1.00  0.00           C  \\nATOM     80  CG1 ILE A   6       1.716   6.292  -6.535  1.00  0.00           C  \\nATOM     81  CG2 ILE A   6       1.145   3.884  -6.209  1.00  0.00           C  \\nATOM     82  CD1 ILE A   6       1.834   7.459  -7.488  1.00  0.00           C  \\nATOM     83  H   ILE A   6       1.269   3.196  -8.952  1.00  0.00           H  \\nATOM     84  HA  ILE A   6       2.940   5.406  -8.708  1.00  0.00           H  \\nATOM     85  HB  ILE A   6       0.562   5.101  -7.853  1.00  0.00           H  \\nATOM     86 1HG1 ILE A   6       0.915   6.508  -5.829  1.00  0.00           H  \\nATOM     87 2HG1 ILE A   6       2.643   6.223  -5.965  1.00  0.00           H  \\nATOM     88 1HG2 ILE A   6       0.298   4.187  -5.596  1.00  0.00           H  \\nATOM     89 2HG2 ILE A   6       0.907   2.952  -6.721  1.00  0.00           H  \\nATOM     90 3HG2 ILE A   6       2.004   3.737  -5.583  1.00  0.00           H  \\nATOM     91 1HD1 ILE A   6       2.030   8.371  -6.923  1.00  0.00           H  \\nATOM     92 2HD1 ILE A   6       2.655   7.280  -8.183  1.00  0.00           H  \\nATOM     93 3HD1 ILE A   6       0.904   7.570  -8.044  1.00  0.00           H  \\nATOM     94  N   GLN A   7       4.054   2.906  -6.984  1.00  0.00           N  \\nATOM     95  CA  GLN A   7       5.150   2.442  -6.139  1.00  0.00           C  \\nATOM     96  C   GLN A   7       6.488   2.387  -6.866  1.00  0.00           C  \\nATOM     97  O   GLN A   7       7.514   2.100  -6.250  1.00  0.00           O  \\nATOM     98  CB  GLN A   7       4.820   1.056  -5.571  1.00  0.00           C  \\nATOM     99  CG  GLN A   7       3.596   1.028  -4.672  1.00  0.00           C  \\nATOM    100  CD  GLN A   7       3.762   1.902  -3.437  1.00  0.00           C  \\nATOM    101  OE1 GLN A   7       4.716   1.743  -2.671  1.00  0.00           O  \\nATOM    102  NE2 GLN A   7       2.835   2.826  -3.240  1.00  0.00           N  \\nATOM    103  H   GLN A   7       3.444   2.225  -7.413  1.00  0.00           H  \\nATOM    104  HA  GLN A   7       5.272   3.143  -5.314  1.00  0.00           H  \\nATOM    105 1HB  GLN A   7       4.651   0.358  -6.391  1.00  0.00           H  \\nATOM    106 2HB  GLN A   7       5.668   0.687  -4.996  1.00  0.00           H  \\nATOM    107 1HG  GLN A   7       2.737   1.391  -5.234  1.00  0.00           H  \\nATOM    108 2HG  GLN A   7       3.422   0.004  -4.345  1.00  0.00           H  \\nATOM    109 1HE2 GLN A   7       2.891   3.431  -2.445  1.00  0.00           H  \\nATOM    110 2HE2 GLN A   7       2.080   2.921  -3.885  1.00  0.00           H  \\nATOM    111  N   LEU A   8       6.505   2.698  -8.161  1.00  0.00           N  \\nATOM    112  CA  LEU A   8       7.728   2.607  -8.948  1.00  0.00           C  \\nATOM    113  C   LEU A   8       8.852   3.492  -8.445  1.00  0.00           C  \\nATOM    114  O   LEU A   8       8.647   4.626  -8.011  1.00  0.00           O  \\nATOM    115  CB  LEU A   8       7.438   2.975 -10.419  1.00  0.00           C  \\nATOM    116  CG  LEU A   8       8.613   2.778 -11.403  1.00  0.00           C  \\nATOM    117  CD1 LEU A   8       8.913   1.295 -11.546  1.00  0.00           C  \\nATOM    118  CD2 LEU A   8       8.262   3.392 -12.736  1.00  0.00           C  \\nATOM    119  H   LEU A   8       5.678   2.901  -8.701  1.00  0.00           H  \\nATOM    120  HA  LEU A   8       8.092   1.581  -8.904  1.00  0.00           H  \\nATOM    121 1HB  LEU A   8       6.607   2.368 -10.771  1.00  0.00           H  \\nATOM    122 2HB  LEU A   8       7.140   4.023 -10.463  1.00  0.00           H  \\nATOM    123  HG  LEU A   8       9.500   3.259 -11.007  1.00  0.00           H  \\nATOM    124 1HD1 LEU A   8       9.742   1.156 -12.240  1.00  0.00           H  \\nATOM    125 2HD1 LEU A   8       9.182   0.882 -10.574  1.00  0.00           H  \\nATOM    126 3HD1 LEU A   8       8.031   0.781 -11.928  1.00  0.00           H  \\nATOM    127 1HD2 LEU A   8       9.091   3.254 -13.430  1.00  0.00           H  \\nATOM    128 2HD2 LEU A   8       7.372   2.909 -13.134  1.00  0.00           H  \\nATOM    129 3HD2 LEU A   8       8.070   4.454 -12.607  1.00  0.00           H  \\nATOM    130  N   VAL A   9      10.051   2.940  -8.522  1.00  0.00           N  \\nATOM    131  CA  VAL A   9      11.295   3.575  -8.136  1.00  0.00           C  \\nATOM    132  C   VAL A   9      12.214   3.703  -9.371  1.00  0.00           C  \\nATOM    133  O   VAL A   9      12.277   2.798 -10.200  1.00  0.00           O  \\nATOM    134  CB  VAL A   9      11.989   2.757  -7.038  1.00  0.00           C  \\nATOM    135  CG1 VAL A   9      11.134   2.720  -5.781  1.00  0.00           C  \\nATOM    136  CG2 VAL A   9      12.271   1.352  -7.540  1.00  0.00           C  \\nATOM    137  H   VAL A   9      10.097   2.003  -8.894  1.00  0.00           H  \\nATOM    138  HA  VAL A   9      11.075   4.539  -7.750  1.00  0.00           H  \\nATOM    139  HB  VAL A   9      12.927   3.245  -6.774  1.00  0.00           H  \\nATOM    140 1HG1 VAL A   9      11.641   2.137  -5.013  1.00  0.00           H  \\nATOM    141 2HG1 VAL A   9      10.975   3.736  -5.419  1.00  0.00           H  \\nATOM    142 3HG1 VAL A   9      10.173   2.261  -6.008  1.00  0.00           H  \\nATOM    143 1HG2 VAL A   9      12.764   0.778  -6.756  1.00  0.00           H  \\nATOM    144 2HG2 VAL A   9      11.332   0.866  -7.810  1.00  0.00           H  \\nATOM    145 3HG2 VAL A   9      12.919   1.401  -8.416  1.00  0.00           H  \\nATOM    146  N   PRO A  10      12.935   4.852  -9.475  1.00  0.00           N  \\nATOM    147  CA  PRO A  10      13.832   5.075 -10.583  1.00  0.00           C  \\nATOM    148  C   PRO A  10      14.952   4.062 -10.745  1.00  0.00           C  \\nATOM    149  O   PRO A  10      15.525   3.584  -9.766  1.00  0.00           O  \\nATOM    150  CB  PRO A  10      14.416   6.481 -10.255  1.00  0.00           C  \\nATOM    151  CG  PRO A  10      13.358   7.129  -9.381  1.00  0.00           C  \\nATOM    152  CD  PRO A  10      12.815   6.002  -8.555  1.00  0.00           C  \\nATOM    153  HA  PRO A  10      13.254   5.088 -11.519  1.00  0.00           H  \\nATOM    154 1HB  PRO A  10      15.385   6.375  -9.745  1.00  0.00           H  \\nATOM    155 2HB  PRO A  10      14.601   7.038 -11.185  1.00  0.00           H  \\nATOM    156 1HG  PRO A  10      13.806   7.925  -8.769  1.00  0.00           H  \\nATOM    157 2HG  PRO A  10      12.588   7.604 -10.006  1.00  0.00           H  \\nATOM    158 1HD  PRO A  10      13.434   5.872  -7.664  1.00  0.00           H  \\nATOM    159 2HD  PRO A  10      11.792   6.216  -8.281  1.00  0.00           H  \\nATOM    160  N   ARG A  11      15.234   3.732 -11.998  1.00  0.00           N  \\nATOM    161  CA  ARG A  11      16.291   2.803 -12.362  1.00  0.00           C  \\nATOM    162  C   ARG A  11      17.546   3.576 -12.706  1.00  0.00           C  \\nATOM    163  O   ARG A  11      17.469   4.540 -13.467  1.00  0.00           O  \\nATOM    164  CB  ARG A  11      15.880   1.942 -13.544  1.00  0.00           C  \\nATOM    165  CG  ARG A  11      16.909   0.904 -13.968  1.00  0.00           C  \\nATOM    166  CD  ARG A  11      16.407   0.054 -15.077  1.00  0.00           C  \\nATOM    167  NE  ARG A  11      17.392  -0.931 -15.495  1.00  0.00           N  \\nATOM    168  CZ  ARG A  11      17.227  -1.791 -16.519  1.00  0.00           C  \\nATOM    169  NH1 ARG A  11      16.114  -1.774 -17.219  1.00  0.00           N  \\nATOM    170  NH2 ARG A  11      18.185  -2.650 -16.821  1.00  0.00           N  \\nATOM    171  H   ARG A  11      14.709   4.175 -12.738  1.00  0.00           H  \\nATOM    172  HA  ARG A  11      16.484   2.148 -11.512  1.00  0.00           H  \\nATOM    173 1HB  ARG A  11      14.958   1.414 -13.307  1.00  0.00           H  \\nATOM    174 2HB  ARG A  11      15.681   2.578 -14.406  1.00  0.00           H  \\nATOM    175 1HG  ARG A  11      17.816   1.407 -14.304  1.00  0.00           H  \\nATOM    176 2HG  ARG A  11      17.144   0.258 -13.121  1.00  0.00           H  \\nATOM    177 1HD  ARG A  11      15.511  -0.475 -14.753  1.00  0.00           H  \\nATOM    178 2HD  ARG A  11      16.167   0.681 -15.935  1.00  0.00           H  \\nATOM    179  HE  ARG A  11      18.261  -0.973 -14.980  1.00  0.00           H  \\nATOM    180 1HH1 ARG A  11      15.382  -1.117 -16.989  1.00  0.00           H  \\nATOM    181 2HH1 ARG A  11      15.991  -2.419 -17.987  1.00  0.00           H  \\nATOM    182 1HH2 ARG A  11      19.041  -2.663 -16.283  1.00  0.00           H  \\nATOM    183 2HH2 ARG A  11      18.062  -3.294 -17.588  1.00  0.00           H  \\nATOM    184  N   PHE A  12      18.687   3.167 -12.177  1.00  0.00           N  \\nATOM    185  CA  PHE A  12      19.919   3.866 -12.484  1.00  0.00           C  \\nATOM    186  C   PHE A  12      20.456   3.618 -13.878  1.00  0.00           C  \\nATOM    187  O   PHE A  12      20.555   2.484 -14.345  1.00  0.00           O  \\nATOM    188  CB  PHE A  12      20.987   3.481 -11.476  1.00  0.00           C  \\nATOM    189  CG  PHE A  12      20.754   4.037 -10.111  1.00  0.00           C  \\nATOM    190  CD1 PHE A  12      20.104   3.289  -9.146  1.00  0.00           C  \\nATOM    191  CD2 PHE A  12      21.186   5.313  -9.790  1.00  0.00           C  \\nATOM    192  CE1 PHE A  12      19.888   3.801  -7.885  1.00  0.00           C  \\nATOM    193  CE2 PHE A  12      20.972   5.828  -8.527  1.00  0.00           C  \\nATOM    194  CZ  PHE A  12      20.321   5.070  -7.573  1.00  0.00           C  \\nATOM    195  H   PHE A  12      18.709   2.367 -11.562  1.00  0.00           H  \\nATOM    196  HA  PHE A  12      19.735   4.939 -12.415  1.00  0.00           H  \\nATOM    197 1HB  PHE A  12      21.038   2.397 -11.397  1.00  0.00           H  \\nATOM    198 2HB  PHE A  12      21.960   3.831 -11.824  1.00  0.00           H  \\nATOM    199  HD1 PHE A  12      19.762   2.283  -9.393  1.00  0.00           H  \\nATOM    200  HD2 PHE A  12      21.698   5.911 -10.542  1.00  0.00           H  \\nATOM    201  HE1 PHE A  12      19.374   3.200  -7.135  1.00  0.00           H  \\nATOM    202  HE2 PHE A  12      21.315   6.834  -8.281  1.00  0.00           H  \\nATOM    203  HZ  PHE A  12      20.150   5.476  -6.578  1.00  0.00           H  \\nATOM    204  N   SER A  13      20.817   4.698 -14.543  1.00  0.00           N  \\nATOM    205  CA  SER A  13      21.367   4.693 -15.884  1.00  0.00           C  \\nATOM    206  C   SER A  13      22.868   4.413 -15.941  1.00  0.00           C  \\nATOM    207  O   SER A  13      23.583   4.575 -14.970  1.00  0.00           O  \\nATOM    208  CB  SER A  13      21.083   6.018 -16.540  1.00  0.00           C  \\nATOM    209  OG  SER A  13      21.788   7.053 -15.907  1.00  0.00           O  \\nATOM    210  H   SER A  13      20.687   5.588 -14.084  1.00  0.00           H  \\nATOM    211  HA  SER A  13      20.882   3.899 -16.454  1.00  0.00           H  \\nATOM    212 1HB  SER A  13      21.364   5.972 -17.582  1.00  0.00           H  \\nATOM    213 2HB  SER A  13      20.015   6.224 -16.498  1.00  0.00           H  \\nATOM    214  HG  SER A  13      22.718   6.830 -15.990  1.00  0.00           H  \\nATOM    215  N   GLY A  14      23.335   3.965 -17.112  1.00  0.00           N  \\nATOM    216  CA  GLY A  14      24.788   3.724 -17.375  1.00  0.00           C  \\nATOM    217  C   GLY A  14      25.084   4.078 -18.873  1.00  0.00           C  \\nATOM    218  O   GLY A  14      25.328   3.170 -19.669  1.00  0.00           O  \\nATOM    219  H   GLY A  14      22.680   3.801 -17.864  1.00  0.00           H  \\nATOM    220 1HA  GLY A  14      25.251   4.199 -16.854  1.00  0.00           H  \\nATOM    221 2HA  GLY A  14      25.029   2.685 -17.163  1.00  0.00           H  \\nATOM    222  N   PRO A  15      25.117   5.598 -19.338  1.00  0.00           N  \\nATOM    223  CA  PRO A  15      24.947   7.066 -18.483  1.00  0.00           C  \\nATOM    224  C   PRO A  15      25.143   6.909 -16.902  1.00  0.00           C  \\nATOM    225  O   PRO A  15      24.527   7.638 -16.119  1.00  0.00           O  \\nATOM    226  CB  PRO A  15      23.773   7.402 -18.763  1.00  0.00           C  \\nATOM    227  CG  PRO A  15      23.436   6.541 -20.460  1.00  0.00           C  \\nATOM    228  CD  PRO A  15      24.172   5.207 -20.359  1.00  0.00           C  \\nATOM    229  HA  PRO A  15      25.674   7.796 -18.869  1.00  0.00           H  \\nATOM    230 1HB  PRO A  15      23.114   7.082 -18.000  1.00  0.00           H  \\nATOM    231 2HB  PRO A  15      23.698   8.496 -18.815  1.00  0.00           H  \\nATOM    232 1HG  PRO A  15      22.354   6.417 -20.615  1.00  0.00           H  \\nATOM    233 2HG  PRO A  15      23.805   7.168 -21.283  1.00  0.00           H  \\nATOM    234 1HD  PRO A  15      23.518   4.463 -19.884  1.00  0.00           H  \\nATOM    235 2HD  PRO A  15      24.470   4.876 -21.365  1.00  0.00           H  \\nATOM    236  N   GLU A  16      26.048   6.046 -16.622  1.00  0.00           N  \\nATOM    237  CA  GLU A  16      26.464   5.587 -15.268  1.00  0.00           C  \\nATOM    238  C   GLU A  16      26.881   6.657 -14.294  1.00  0.00           C  \\nATOM    239  O   GLU A  16      26.860   6.314 -13.101  1.00  0.00           O  \\nATOM    240  CB  GLU A  16      27.634   4.587 -15.395  1.00  0.00           C  \\nATOM    241  CG  GLU A  16      28.914   5.183 -15.992  1.00  0.00           C  \\nATOM    242  CD  GLU A  16      28.924   5.160 -17.493  1.00  0.00           C  \\nATOM    243  OE1 GLU A  16      27.878   5.038 -18.071  1.00  0.00           O  \\nATOM    244  OE2 GLU A  16      29.986   5.267 -18.062  1.00  0.00           O  \\nATOM    245  H   GLU A  16      26.509   5.585 -17.393  1.00  0.00           H  \\nATOM    246  HA  GLU A  16      25.642   5.108 -14.809  1.00  0.00           H  \\nATOM    247 1HB  GLU A  16      27.878   4.187 -14.411  1.00  0.00           H  \\nATOM    248 2HB  GLU A  16      27.331   3.752 -16.021  1.00  0.00           H  \\nATOM    249 1HG  GLU A  16      29.014   6.201 -15.659  1.00  0.00           H  \\nATOM    250 2HG  GLU A  16      29.770   4.623 -15.620  1.00  0.00           H  \\nATOM    251  N   ILE A  17      27.274   7.824 -14.812  1.00  0.00           N  \\nATOM    252  CA  ILE A  17      27.723   8.940 -13.994  1.00  0.00           C  \\nATOM    253  C   ILE A  17      26.648   9.245 -13.034  1.00  0.00           C  \\nATOM    254  O   ILE A  17      26.803   9.730 -11.936  1.00  0.00           O  \\nATOM    255  CB  ILE A  17      28.050  10.186 -14.837  1.00  0.00           C  \\nATOM    256  CG1 ILE A  17      26.797  10.684 -15.564  1.00  0.00           C  \\nATOM    257  CG2 ILE A  17      29.156   9.881 -15.828  1.00  0.00           C  \\nATOM    258  CD1 ILE A  17      26.972  12.024 -16.233  1.00  0.00           C  \\nATOM    259  H   ILE A  17      27.230   7.949 -15.813  1.00  0.00           H  \\nATOM    260  HA  ILE A  17      28.539   8.671 -13.523  1.00  0.00           H  \\nATOM    261  HB  ILE A  17      28.376  10.994 -14.181  1.00  0.00           H  \\nATOM    262 1HG1 ILE A  17      26.504   9.958 -16.324  1.00  0.00           H  \\nATOM    263 2HG1 ILE A  17      25.977  10.763 -14.858  1.00  0.00           H  \\nATOM    264 1HG2 ILE A  17      29.374  10.772 -16.415  1.00  0.00           H  \\nATOM    265 2HG2 ILE A  17      30.043   9.576 -15.294  1.00  0.00           H  \\nATOM    266 3HG2 ILE A  17      28.837   9.077 -16.493  1.00  0.00           H  \\nATOM    267 1HD1 ILE A  17      26.043  12.309 -16.725  1.00  0.00           H  \\nATOM    268 2HD1 ILE A  17      27.232  12.774 -15.484  1.00  0.00           H  \\nATOM    269 3HD1 ILE A  17      27.769  11.959 -16.972  1.00  0.00           H  \\nATOM    270  N   THR A  18      25.354   8.751 -13.325  1.00  0.00           N  \\nATOM    271  CA  THR A  18      24.461   9.203 -12.165  1.00  0.00           C  \\nATOM    272  C   THR A  18      25.250   8.927 -10.746  1.00  0.00           C  \\nATOM    273  O   THR A  18      25.248   9.805  -9.871  1.00  0.00           O  \\nATOM    274  CB  THR A  18      23.106   8.471 -12.189  1.00  0.00           C  \\nATOM    275  OG1 THR A  18      22.431   8.749 -13.424  1.00  0.00           O  \\nATOM    276  CG2 THR A  18      22.235   8.924 -11.028  1.00  0.00           C  \\nATOM    277  H   THR A  18      25.051   8.294 -14.180  1.00  0.00           H  \\nATOM    278  HA  THR A  18      24.276  10.273 -12.267  1.00  0.00           H  \\nATOM    279  HB  THR A  18      23.271   7.396 -12.115  1.00  0.00           H  \\nATOM    280  HG1 THR A  18      22.971   8.445 -14.159  1.00  0.00           H  \\nATOM    281 1HG2 THR A  18      21.282   8.396 -11.060  1.00  0.00           H  \\nATOM    282 2HG2 THR A  18      22.741   8.703 -10.086  1.00  0.00           H  \\nATOM    283 3HG2 THR A  18      22.058   9.995 -11.102  1.00  0.00           H  \\nATOM    284  N   LEU A  19      25.933   7.677 -10.605  1.00  0.00           N  \\nATOM    285  CA  LEU A  19      26.643   7.179  -9.375  1.00  0.00           C  \\nATOM    286  C   LEU A  19      28.153   7.560  -9.538  1.00  0.00           C  \\nATOM    287  O   LEU A  19      28.938   7.845  -8.646  1.00  0.00           O  \\nATOM    288  CB  LEU A  19      26.479   5.665  -9.208  1.00  0.00           C  \\nATOM    289  CG  LEU A  19      25.034   5.166  -9.032  1.00  0.00           C  \\nATOM    290  CD1 LEU A  19      25.027   3.648  -8.962  1.00  0.00           C  \\nATOM    291  CD2 LEU A  19      24.442   5.769  -7.782  1.00  0.00           C  \\nATOM    292  H   LEU A  19      26.126   7.166 -11.457  1.00  0.00           H  \\nATOM    293  HA  LEU A  19      26.235   7.621  -8.570  1.00  0.00           H  \\nATOM    294 1HB  LEU A  19      26.895   5.171 -10.085  1.00  0.00           H  \\nATOM    295 2HB  LEU A  19      27.048   5.347  -8.334  1.00  0.00           H  \\nATOM    296  HG  LEU A  19      24.438   5.463  -9.897  1.00  0.00           H  \\nATOM    297 1HD1 LEU A  19      24.002   3.294  -8.838  1.00  0.00           H  \\nATOM    298 2HD1 LEU A  19      25.441   3.239  -9.884  1.00  0.00           H  \\nATOM    299 3HD1 LEU A  19      25.629   3.320  -8.116  1.00  0.00           H  \\nATOM    300 1HD2 LEU A  19      23.428   5.420  -7.659  1.00  0.00           H  \\nATOM    301 2HD2 LEU A  19      25.034   5.472  -6.919  1.00  0.00           H  \\nATOM    302 3HD2 LEU A  19      24.443   6.849  -7.866  1.00  0.00           H  \\nATOM    303  N   ILE A  20      28.562   7.739 -10.816  1.00  0.00           N  \\nATOM    304  CA  ILE A  20      30.145   8.089 -10.812  1.00  0.00           C  \\nATOM    305  C   ILE A  20      30.200   9.244  -9.685  1.00  0.00           C  \\nATOM    306  O   ILE A  20      31.000   9.116  -8.747  1.00  0.00           O  \\nATOM    307  CB  ILE A  20      30.674   8.583 -12.142  1.00  0.00           C  \\nATOM    308  CG1 ILE A  20      30.570   7.406 -13.270  1.00  0.00           C  \\nATOM    309  CG2 ILE A  20      32.105   9.059 -11.999  1.00  0.00           C  \\nATOM    310  CD1 ILE A  20      31.395   6.189 -12.955  1.00  0.00           C  \\nATOM    311  H   ILE A  20      27.978   7.383 -11.570  1.00  0.00           H  \\nATOM    312  HA  ILE A  20      30.697   7.185 -10.561  1.00  0.00           H  \\nATOM    313  HB  ILE A  20      30.069   9.398 -12.488  1.00  0.00           H  \\nATOM    314 1HG1 ILE A  20      29.580   7.110 -13.371  1.00  0.00           H  \\nATOM    315 2HG1 ILE A  20      30.892   7.794 -14.228  1.00  0.00           H  \\nATOM    316 1HG2 ILE A  20      32.471   9.410 -12.963  1.00  0.00           H  \\nATOM    317 2HG2 ILE A  20      32.147   9.874 -11.277  1.00  0.00           H  \\nATOM    318 3HG2 ILE A  20      32.730   8.236 -11.653  1.00  0.00           H  \\nATOM    319 1HD1 ILE A  20      31.275   5.452 -13.749  1.00  0.00           H  \\nATOM    320 2HD1 ILE A  20      32.445   6.471 -12.879  1.00  0.00           H  \\nATOM    321 3HD1 ILE A  20      31.065   5.760 -12.010  1.00  0.00           H  \\nATOM    322  N   ILE A  21      29.276  10.202  -9.860  1.00  0.00           N  \\nATOM    323  CA  ILE A  21      29.108  11.403  -8.993  1.00  0.00           C  \\nATOM    324  C   ILE A  21      28.644  11.049  -7.530  1.00  0.00           C  \\nATOM    325  O   ILE A  21      29.429  11.488  -6.683  1.00  0.00           O  \\nATOM    326  CB  ILE A  21      28.101  12.370  -9.622  1.00  0.00           C  \\nATOM    327  CG1 ILE A  21      28.660  12.954 -10.919  1.00  0.00           C  \\nATOM    328  CG2 ILE A  21      27.747  13.482  -8.642  1.00  0.00           C  \\nATOM    329  CD1 ILE A  21      27.637  13.694 -11.746  1.00  0.00           C  \\nATOM    330  H   ILE A  21      28.602  10.084 -10.617  1.00  0.00           H  \\nATOM    331  HA  ILE A  21      30.071  11.906  -8.908  1.00  0.00           H  \\nATOM    332  HB  ILE A  21      27.192  11.829  -9.886  1.00  0.00           H  \\nATOM    333 1HG1 ILE A  21      29.473  13.640 -10.687  1.00  0.00           H  \\nATOM    334 2HG1 ILE A  21      29.076  12.150 -11.531  1.00  0.00           H  \\nATOM    335 1HG2 ILE A  21      27.031  14.160  -9.105  1.00  0.00           H  \\nATOM    336 2HG2 ILE A  21      27.308  13.049  -7.744  1.00  0.00           H  \\nATOM    337 3HG2 ILE A  21      28.649  14.033  -8.376  1.00  0.00           H  \\nATOM    338 1HD1 ILE A  21      28.109  14.079 -12.651  1.00  0.00           H  \\nATOM    339 2HD1 ILE A  21      26.827  13.013 -12.020  1.00  0.00           H  \\nATOM    340 3HD1 ILE A  21      27.233  14.524 -11.168  1.00  0.00           H  \\nATOM    341  N   PHE A  22      27.615  10.220  -7.406  1.00  0.00           N  \\nATOM    342  CA  PHE A  22      26.986   9.763  -6.128  1.00  0.00           C  \\nATOM    343  C   PHE A  22      28.021   9.383  -5.132  1.00  0.00           C  \\nATOM    344  O   PHE A  22      27.910   9.912  -4.026  1.00  0.00           O  \\nATOM    345  CB  PHE A  22      26.095   8.606  -6.340  1.00  0.00           C  \\nATOM    346  CG  PHE A  22      25.484   8.076  -5.079  1.00  0.00           C  \\nATOM    347  CD1 PHE A  22      24.401   8.712  -4.497  1.00  0.00           C  \\nATOM    348  CD2 PHE A  22      25.994   6.936  -4.473  1.00  0.00           C  \\nATOM    349  CE1 PHE A  22      23.837   8.224  -3.335  1.00  0.00           C  \\nATOM    350  CE2 PHE A  22      25.431   6.445  -3.312  1.00  0.00           C  \\nATOM    351  CZ  PHE A  22      24.351   7.090  -2.742  1.00  0.00           C  \\nATOM    352  H   PHE A  22      27.026  10.016  -8.215  1.00  0.00           H  \\nATOM    353  HA  PHE A  22      26.393  10.584  -5.721  1.00  0.00           H  \\nATOM    354 1HB  PHE A  22      25.289   8.887  -7.016  1.00  0.00           H  \\nATOM    355 2HB  PHE A  22      26.633   7.828  -6.796  1.00  0.00           H  \\nATOM    356  HD1 PHE A  22      23.994   9.607  -4.967  1.00  0.00           H  \\nATOM    357  HD2 PHE A  22      26.847   6.427  -4.923  1.00  0.00           H  \\nATOM    358  HE1 PHE A  22      22.985   8.735  -2.888  1.00  0.00           H  \\nATOM    359  HE2 PHE A  22      25.838   5.549  -2.844  1.00  0.00           H  \\nATOM    360  HZ  PHE A  22      23.907   6.705  -1.825  1.00  0.00           H  \\nATOM    361  N   GLY A  23      28.933   8.495  -5.519  1.00  0.00           N  \\nATOM    362  CA  GLY A  23      29.956   8.035  -4.633  1.00  0.00           C  \\nATOM    363  C   GLY A  23      31.022   9.105  -4.534  1.00  0.00           C  \\nATOM    364  O   GLY A  23      31.605   9.453  -3.522  1.00  0.00           O  \\nATOM    365  H   GLY A  23      28.914   8.218  -6.499  1.00  0.00           H  \\nATOM    366 1HA  GLY A  23      29.529   7.825  -3.670  1.00  0.00           H  \\nATOM    367 2HA  GLY A  23      30.372   7.102  -5.010  1.00  0.00           H  \\nATOM    368  N   VAL A  24      31.285   9.935  -5.690  1.00  0.00           N  \\nATOM    369  CA  VAL A  24      32.454  10.861  -5.240  1.00  0.00           C  \\nATOM    370  C   VAL A  24      31.943  11.513  -3.897  1.00  0.00           C  \\nATOM    371  O   VAL A  24      32.733  11.511  -2.944  1.00  0.00           O  \\nATOM    372  CB  VAL A  24      32.778  11.946  -6.283  1.00  0.00           C  \\nATOM    373  CG1 VAL A  24      33.766  12.952  -5.715  1.00  0.00           C  \\nATOM    374  CG2 VAL A  24      33.327  11.305  -7.538  1.00  0.00           C  \\nATOM    375  H   VAL A  24      31.038   9.815  -6.669  1.00  0.00           H  \\nATOM    376  HA  VAL A  24      33.345  10.250  -5.097  1.00  0.00           H  \\nATOM    377  HB  VAL A  24      31.865  12.492  -6.523  1.00  0.00           H  \\nATOM    378 1HG1 VAL A  24      33.984  13.712  -6.466  1.00  0.00           H  \\nATOM    379 2HG1 VAL A  24      33.334  13.428  -4.832  1.00  0.00           H  \\nATOM    380 3HG1 VAL A  24      34.687  12.441  -5.440  1.00  0.00           H  \\nATOM    381 1HG2 VAL A  24      33.553  12.078  -8.272  1.00  0.00           H  \\nATOM    382 2HG2 VAL A  24      34.236  10.755  -7.298  1.00  0.00           H  \\nATOM    383 3HG2 VAL A  24      32.598  10.630  -7.944  1.00  0.00           H  \\nATOM    384  N   MET A  25      30.633  11.848  -3.876  1.00  0.00           N  \\nATOM    385  CA  MET A  25      30.011  12.496  -2.745  1.00  0.00           C  \\nATOM    386  C   MET A  25      30.008  11.685  -1.478  1.00  0.00           C  \\nATOM    387  O   MET A  25      30.476  12.288  -0.510  1.00  0.00           O  \\nATOM    388  CB  MET A  25      28.583  12.874  -3.107  1.00  0.00           C  \\nATOM    389  CG  MET A  25      28.469  13.980  -4.135  1.00  0.00           C  \\nATOM    390  SD  MET A  25      26.772  14.273  -4.653  1.00  0.00           S  \\nATOM    391  CE  MET A  25      26.084  14.994  -3.166  1.00  0.00           C  \\nATOM    392  H   MET A  25      30.136  11.730  -4.753  1.00  0.00           H  \\nATOM    393  HA  MET A  25      30.569  13.395  -2.513  1.00  0.00           H  \\nATOM    394 1HB  MET A  25      28.084  12.035  -3.483  1.00  0.00           H  \\nATOM    395 2HB  MET A  25      28.053  13.195  -2.211  1.00  0.00           H  \\nATOM    396 1HG  MET A  25      28.867  14.904  -3.719  1.00  0.00           H  \\nATOM    397 2HG  MET A  25      29.058  13.720  -5.016  1.00  0.00           H  \\nATOM    398 1HE  MET A  25      25.032  15.232  -3.329  1.00  0.00           H  \\nATOM    399 2HE  MET A  25      26.173  14.284  -2.342  1.00  0.00           H  \\nATOM    400 3HE  MET A  25      26.628  15.906  -2.918  1.00  0.00           H  \\nATOM    401  N   ALA A  26      29.665  10.401  -1.622  1.00  0.00           N  \\nATOM    402  CA  ALA A  26      29.527   9.465  -0.530  1.00  0.00           C  \\nATOM    403  C   ALA A  26      30.771   9.489   0.305  1.00  0.00           C  \\nATOM    404  O   ALA A  26      30.561   9.560   1.512  1.00  0.00           O  \\nATOM    405  CB  ALA A  26      29.262   8.066  -1.031  1.00  0.00           C  \\nATOM    406  H   ALA A  26      29.238  10.171  -2.512  1.00  0.00           H  \\nATOM    407  HA  ALA A  26      28.679   9.778   0.080  1.00  0.00           H  \\nATOM    408 1HB  ALA A  26      29.196   7.383  -0.185  1.00  0.00           H  \\nATOM    409 2HB  ALA A  26      28.321   8.049  -1.585  1.00  0.00           H  \\nATOM    410 3HB  ALA A  26      30.066   7.757  -1.680  1.00  0.00           H  \\nATOM    411  N   GLY A  27      31.933   9.458  -0.357  1.00  0.00           N  \\nATOM    412  CA  GLY A  27      33.186   9.474   0.360  1.00  0.00           C  \\nATOM    413  C   GLY A  27      33.441  10.888   0.846  1.00  0.00           C  \\nATOM    414  O   GLY A  27      33.907  11.207   1.928  1.00  0.00           O  \\nATOM    415  H   GLY A  27      31.872   9.428  -1.371  1.00  0.00           H  \\nATOM    416 1HA  GLY A  27      33.137   8.773   1.193  1.00  0.00           H  \\nATOM    417 2HA  GLY A  27      33.985   9.134  -0.298  1.00  0.00           H  \\nATOM    418  N   ILE A  28      32.913  11.962   0.057  1.00  0.00           N  \\nATOM    419  CA  ILE A  28      33.348  13.293   0.702  1.00  0.00           C  \\nATOM    420  C   ILE A  28      32.927  13.124   2.180  1.00  0.00           C  \\nATOM    421  O   ILE A  28      33.780  13.381   3.041  1.00  0.00           O  \\nATOM    422  CB  ILE A  28      32.678  14.529   0.073  1.00  0.00           C  \\nATOM    423  CG1 ILE A  28      33.187  14.742  -1.356  1.00  0.00           C  \\nATOM    424  CG2 ILE A  28      32.934  15.763   0.922  1.00  0.00           C  \\nATOM    425  CD1 ILE A  28      32.383  15.752  -2.147  1.00  0.00           C  \\nATOM    426  H   ILE A  28      32.636  11.874  -0.914  1.00  0.00           H  \\nATOM    427  HA  ILE A  28      34.424  13.403   0.576  1.00  0.00           H  \\nATOM    428  HB  ILE A  28      31.608  14.365   0.003  1.00  0.00           H  \\nATOM    429 1HG1 ILE A  28      34.222  15.078  -1.327  1.00  0.00           H  \\nATOM    430 2HG1 ILE A  28      33.167  13.797  -1.894  1.00  0.00           H  \\nATOM    431 1HG2 ILE A  28      32.455  16.626   0.463  1.00  0.00           H  \\nATOM    432 2HG2 ILE A  28      32.525  15.609   1.921  1.00  0.00           H  \\nATOM    433 3HG2 ILE A  28      34.008  15.939   0.993  1.00  0.00           H  \\nATOM    434 1HD1 ILE A  28      32.804  15.848  -3.148  1.00  0.00           H  \\nATOM    435 2HD1 ILE A  28      31.352  15.418  -2.219  1.00  0.00           H  \\nATOM    436 3HD1 ILE A  28      32.418  16.717  -1.646  1.00  0.00           H  \\nATOM    437  N   ILE A  29      31.704  12.568   2.346  1.00  0.00           N  \\nATOM    438  CA  ILE A  29      31.101  12.369   3.638  1.00  0.00           C  \\nATOM    439  C   ILE A  29      31.788  11.395   4.533  1.00  0.00           C  \\nATOM    440  O   ILE A  29      32.107  11.870   5.631  1.00  0.00           O  \\nATOM    441  CB  ILE A  29      29.644  11.914   3.466  1.00  0.00           C  \\nATOM    442  CG1 ILE A  29      28.807  13.031   2.857  1.00  0.00           C  \\nATOM    443  CG2 ILE A  29      29.061  11.476   4.806  1.00  0.00           C  \\nATOM    444  CD1 ILE A  29      27.441  12.586   2.395  1.00  0.00           C  \\nATOM    445  H   ILE A  29      31.208  12.400   1.476  1.00  0.00           H  \\nATOM    446  HA  ILE A  29      31.117  13.316   4.176  1.00  0.00           H  \\nATOM    447  HB  ILE A  29      29.605  11.104   2.795  1.00  0.00           H  \\nATOM    448 1HG1 ILE A  29      28.677  13.828   3.588  1.00  0.00           H  \\nATOM    449 2HG1 ILE A  29      29.335  13.456   2.000  1.00  0.00           H  \\nATOM    450 1HG2 ILE A  29      28.029  11.157   4.666  1.00  0.00           H  \\nATOM    451 2HG2 ILE A  29      29.645  10.648   5.203  1.00  0.00           H  \\nATOM    452 3HG2 ILE A  29      29.091  12.310   5.506  1.00  0.00           H  \\nATOM    453 1HD1 ILE A  29      26.905  13.436   1.973  1.00  0.00           H  \\nATOM    454 2HD1 ILE A  29      27.548  11.810   1.634  1.00  0.00           H  \\nATOM    455 3HD1 ILE A  29      26.882  12.190   3.241  1.00  0.00           H  \\nATOM    456  N   GLY A  30      32.071  10.217   3.944  1.00  0.00           N  \\nATOM    457  CA  GLY A  30      32.679   9.060   4.534  1.00  0.00           C  \\nATOM    458  C   GLY A  30      33.871   9.429   5.316  1.00  0.00           C  \\nATOM    459  O   GLY A  30      33.880   8.997   6.470  1.00  0.00           O  \\nATOM    460  H   GLY A  30      31.672  10.057   3.030  1.00  0.00           H  \\nATOM    461 1HA  GLY A  30      31.957   8.558   5.178  1.00  0.00           H  \\nATOM    462 2HA  GLY A  30      32.955   8.356   3.751  1.00  0.00           H  \\nATOM    463  N   THR A  31      34.760  10.176   4.680  1.00  0.00           N  \\nATOM    464  CA  THR A  31      35.958  10.590   5.314  1.00  0.00           C  \\nATOM    465  C   THR A  31      35.672  11.741   6.234  1.00  0.00           C  \\nATOM    466  O   THR A  31      36.110  11.851   7.355  1.00  0.00           O  \\nATOM    467  CB  THR A  31      37.021  10.984   4.287  1.00  0.00           C  \\nATOM    468  OG1 THR A  31      36.525  12.053   3.466  1.00  0.00           O  \\nATOM    469  CG2 THR A  31      37.376   9.799   3.408  1.00  0.00           C  \\nATOM    470  H   THR A  31      34.526  10.476   3.741  1.00  0.00           H  \\nATOM    471  HA  THR A  31      36.333   9.778   5.889  1.00  0.00           H  \\nATOM    472  HB  THR A  31      37.918  11.327   4.805  1.00  0.00           H  \\nATOM    473  HG1 THR A  31      35.731  11.763   3.009  1.00  0.00           H  \\nATOM    474 1HG2 THR A  31      38.134  10.097   2.685  1.00  0.00           H  \\nATOM    475 2HG2 THR A  31      37.764   8.989   4.028  1.00  0.00           H  \\nATOM    476 3HG2 THR A  31      36.485   9.457   2.881  1.00  0.00           H  \\nATOM    477  N   ILE A  32      34.652  12.729   5.885  1.00  0.00           N  \\nATOM    478  CA  ILE A  32      34.768  13.698   7.109  1.00  0.00           C  \\nATOM    479  C   ILE A  32      34.581  12.822   8.425  1.00  0.00           C  \\nATOM    480  O   ILE A  32      35.359  13.136   9.335  1.00  0.00           O  \\nATOM    481  CB  ILE A  32      33.723  14.817   7.063  1.00  0.00           C  \\nATOM    482  CG1 ILE A  32      34.016  15.773   5.906  1.00  0.00           C  \\nATOM    483  CG2 ILE A  32      33.692  15.570   8.385  1.00  0.00           C  \\nATOM    484  CD1 ILE A  32      32.895  16.742   5.617  1.00  0.00           C  \\nATOM    485  H   ILE A  32      34.195  12.902   4.997  1.00  0.00           H  \\nATOM    486  HA  ILE A  32      35.755  14.161   7.091  1.00  0.00           H  \\nATOM    487  HB  ILE A  32      32.738  14.388   6.876  1.00  0.00           H  \\nATOM    488 1HG1 ILE A  32      34.915  16.347   6.127  1.00  0.00           H  \\nATOM    489 2HG1 ILE A  32      34.211  15.198   5.000  1.00  0.00           H  \\nATOM    490 1HG2 ILE A  32      32.945  16.361   8.335  1.00  0.00           H  \\nATOM    491 2HG2 ILE A  32      33.437  14.881   9.190  1.00  0.00           H  \\nATOM    492 3HG2 ILE A  32      34.671  16.008   8.578  1.00  0.00           H  \\nATOM    493 1HD1 ILE A  32      33.176  17.386   4.783  1.00  0.00           H  \\nATOM    494 2HD1 ILE A  32      31.991  16.186   5.357  1.00  0.00           H  \\nATOM    495 3HD1 ILE A  32      32.705  17.352   6.498  1.00  0.00           H  \\nATOM    496  N   LEU A  33      33.704  11.793   8.348  1.00  0.00           N  \\nATOM    497  CA  LEU A  33      33.412  10.943   9.482  1.00  0.00           C  \\nATOM    498  C   LEU A  33      34.590  10.167   9.993  1.00  0.00           C  \\nATOM    499  O   LEU A  33      34.849  10.359  11.187  1.00  0.00           O  \\nATOM    500  CB  LEU A  33      32.300   9.962   9.110  1.00  0.00           C  \\nATOM    501  CG  LEU A  33      30.915  10.570   8.929  1.00  0.00           C  \\nATOM    502  CD1 LEU A  33      29.972   9.524   8.360  1.00  0.00           C  \\nATOM    503  CD2 LEU A  33      30.412  11.092  10.264  1.00  0.00           C  \\nATOM    504  H   LEU A  33      33.173  11.691   7.488  1.00  0.00           H  \\nATOM    505  HA  LEU A  33      33.071  11.571  10.305  1.00  0.00           H  \\nATOM    506 1HB  LEU A  33      32.559   9.487   8.212  1.00  0.00           H  \\nATOM    507 2HB  LEU A  33      32.229   9.205   9.891  1.00  0.00           H  \\nATOM    508  HG  LEU A  33      30.970  11.394   8.214  1.00  0.00           H  \\nATOM    509 1HD1 LEU A  33      28.981   9.959   8.230  1.00  0.00           H  \\nATOM    510 2HD1 LEU A  33      30.347   9.181   7.394  1.00  0.00           H  \\nATOM    511 3HD1 LEU A  33      29.910   8.680   9.046  1.00  0.00           H  \\nATOM    512 1HD2 LEU A  33      29.421  11.528  10.134  1.00  0.00           H  \\nATOM    513 2HD2 LEU A  33      30.356  10.270  10.978  1.00  0.00           H  \\nATOM    514 3HD2 LEU A  33      31.097  11.853  10.639  1.00  0.00           H  \\nATOM    515  N   LEU A  34      35.276   9.507   9.049  1.00  0.00           N  \\nATOM    516  CA  LEU A  34      36.393   8.625   9.274  1.00  0.00           C  \\nATOM    517  C   LEU A  34      37.406   9.280  10.109  1.00  0.00           C  \\nATOM    518  O   LEU A  34      37.832   8.572  11.023  1.00  0.00           O  \\nATOM    519  CB  LEU A  34      37.019   8.204   7.960  1.00  0.00           C  \\nATOM    520  CG  LEU A  34      38.222   7.281   8.069  1.00  0.00           C  \\nATOM    521  CD1 LEU A  34      37.800   5.971   8.717  1.00  0.00           C  \\nATOM    522  CD2 LEU A  34      38.805   7.043   6.686  1.00  0.00           C  \\nATOM    523  H   LEU A  34      34.869   9.469   8.123  1.00  0.00           H  \\nATOM    524  HA  LEU A  34      36.032   7.733   9.784  1.00  0.00           H  \\nATOM    525 1HB  LEU A  34      36.263   7.694   7.364  1.00  0.00           H  \\nATOM    526 2HB  LEU A  34      37.327   9.076   7.435  1.00  0.00           H  \\nATOM    527  HG  LEU A  34      38.978   7.741   8.708  1.00  0.00           H  \\nATOM    528 1HD1 LEU A  34      38.662   5.310   8.795  1.00  0.00           H  \\nATOM    529 2HD1 LEU A  34      37.403   6.169   9.714  1.00  0.00           H  \\nATOM    530 3HD1 LEU A  34      37.032   5.495   8.108  1.00  0.00           H  \\nATOM    531 1HD2 LEU A  34      39.668   6.381   6.764  1.00  0.00           H  \\nATOM    532 2HD2 LEU A  34      38.051   6.582   6.047  1.00  0.00           H  \\nATOM    533 3HD2 LEU A  34      39.115   7.995   6.253  1.00  0.00           H  \\nATOM    534  N   ILE A  35      37.723  10.522   9.754  1.00  0.00           N  \\nATOM    535  CA  ILE A  35      38.687  11.299  10.446  1.00  0.00           C  \\nATOM    536  C   ILE A  35      38.161  11.799  11.749  1.00  0.00           C  \\nATOM    537  O   ILE A  35      38.815  11.718  12.798  1.00  0.00           O  \\nATOM    538  CB  ILE A  35      39.143  12.490   9.584  1.00  0.00           C  \\nATOM    539  CG1 ILE A  35      39.897  12.000   8.358  1.00  0.00           C  \\nATOM    540  CG2 ILE A  35      39.971  13.397  10.366  1.00  0.00           C  \\nATOM    541  CD1 ILE A  35      40.158  13.075   7.334  1.00  0.00           C  \\nATOM    542  H   ILE A  35      37.223  10.925   8.966  1.00  0.00           H  \\nATOM    543  HA  ILE A  35      39.546  10.676  10.653  1.00  0.00           H  \\nATOM    544  HB  ILE A  35      38.268  13.033   9.221  1.00  0.00           H  \\nATOM    545 1HG1 ILE A  35      40.855  11.579   8.664  1.00  0.00           H  \\nATOM    546 2HG1 ILE A  35      39.328  11.200   7.875  1.00  0.00           H  \\nATOM    547 1HG2 ILE A  35      40.284  14.232   9.742  1.00  0.00           H  \\nATOM    548 2HG2 ILE A  35      39.399  13.772  11.213  1.00  0.00           H  \\nATOM    549 3HG2 ILE A  35      40.791  12.901  10.705  1.00  0.00           H  \\nATOM    550 1HD1 ILE A  35      40.699  12.648   6.489  1.00  0.00           H  \\nATOM    551 2HD1 ILE A  35      39.208  13.486   6.987  1.00  0.00           H  \\nATOM    552 3HD1 ILE A  35      40.755  13.867   7.783  1.00  0.00           H  \\nATOM    553  N   SER A  36      36.854  12.215  11.818  1.00  0.00           N  \\nATOM    554  CA  SER A  36      36.544  12.688  13.192  1.00  0.00           C  \\nATOM    555  C   SER A  36      36.946  11.487  14.124  1.00  0.00           C  \\nATOM    556  O   SER A  36      37.469  11.702  15.211  1.00  0.00           O  \\nATOM    557  CB  SER A  36      35.083  13.048  13.351  1.00  0.00           C  \\nATOM    558  OG  SER A  36      34.267  11.917  13.195  1.00  0.00           O  \\nATOM    559  H   SER A  36      36.255  12.377  11.017  1.00  0.00           H  \\nATOM    560  HA  SER A  36      37.135  13.584  13.391  1.00  0.00           H  \\nATOM    561 1HB  SER A  36      34.921  13.483  14.337  1.00  0.00           H  \\nATOM    562 2HB  SER A  36      34.811  13.801  12.612  1.00  0.00           H  \\nATOM    563  HG  SER A  36      34.438  11.588  12.309  1.00  0.00           H  \\nATOM    564  N   TYR A  37      36.783  10.237  13.630  1.00  0.00           N  \\nATOM    565  CA  TYR A  37      37.141   9.135  14.519  1.00  0.00           C  \\nATOM    566  C   TYR A  37      38.647   8.860  14.655  1.00  0.00           C  \\nATOM    567  O   TYR A  37      39.163   8.803  15.771  1.00  0.00           O  \\nATOM    568  CB  TYR A  37      36.426   7.877  14.046  1.00  0.00           C  \\nATOM    569  CG  TYR A  37      34.947   7.904  14.266  1.00  0.00           C  \\nATOM    570  CD1 TYR A  37      34.080   7.840  13.179  1.00  0.00           C  \\nATOM    571  CD2 TYR A  37      34.440   7.992  15.551  1.00  0.00           C  \\nATOM    572  CE1 TYR A  37      32.717   7.864  13.382  1.00  0.00           C  \\nATOM    573  CE2 TYR A  37      33.074   8.017  15.754  1.00  0.00           C  \\nATOM    574  CZ  TYR A  37      32.215   7.953  14.677  1.00  0.00           C  \\nATOM    575  OH  TYR A  37      30.855   7.978  14.879  1.00  0.00           O  \\nATOM    576  H   TYR A  37      36.262  10.132  12.765  1.00  0.00           H  \\nATOM    577  HA  TYR A  37      36.811   9.383  15.528  1.00  0.00           H  \\nATOM    578 1HB  TYR A  37      36.608   7.734  12.987  1.00  0.00           H  \\nATOM    579 2HB  TYR A  37      36.832   7.011  14.567  1.00  0.00           H  \\nATOM    580  HD1 TYR A  37      34.480   7.770  12.166  1.00  0.00           H  \\nATOM    581  HD2 TYR A  37      35.118   8.043  16.403  1.00  0.00           H  \\nATOM    582  HE1 TYR A  37      32.039   7.814  12.531  1.00  0.00           H  \\nATOM    583  HE2 TYR A  37      32.675   8.087  16.766  1.00  0.00           H  \\nATOM    584  HH  TYR A  37      30.405   7.926  14.032  1.00  0.00           H  \\nATOM    585  N   SER A  38      39.359   8.728  13.531  1.00  0.00           N  \\nATOM    586  CA  SER A  38      40.787   8.377  13.536  1.00  0.00           C  \\nATOM    587  C   SER A  38      41.771   9.456  13.982  1.00  0.00           C  \\nATOM    588  O   SER A  38      42.844   9.146  14.500  1.00  0.00           O  \\nATOM    589  CB  SER A  38      41.182   7.928  12.142  1.00  0.00           C  \\nATOM    590  OG  SER A  38      41.089   8.978  11.236  1.00  0.00           O  \\nATOM    591  H   SER A  38      38.876   8.771  12.644  1.00  0.00           H  \\nATOM    592  HA  SER A  38      40.940   7.557  14.239  1.00  0.00           H  \\nATOM    593 1HB  SER A  38      42.203   7.548  12.157  1.00  0.00           H  \\nATOM    594 2HB  SER A  38      40.534   7.112  11.825  1.00  0.00           H  \\nATOM    595  HG  SER A  38      41.688   9.654  11.552  1.00  0.00           H  \\nATOM    596  N   ILE A  39      41.427  10.705  13.759  1.00  0.00           N  \\nATOM    597  CA  ILE A  39      42.288  11.828  14.113  1.00  0.00           C  \\nATOM    598  C   ILE A  39      41.676  12.496  15.415  1.00  0.00           C  \\nATOM    599  O   ILE A  39      42.402  12.822  16.351  1.00  0.00           O  \\nATOM    600  CB  ILE A  39      42.383  12.815  13.010  1.00  0.00           C  \\nATOM    601  CG1 ILE A  39      42.927  12.143  11.734  1.00  0.00           C  \\nATOM    602  CG2 ILE A  39      43.120  13.790  13.349  1.00  0.00           C  \\nATOM    603  CD1 ILE A  39      44.291  11.526  11.904  1.00  0.00           C  \\nATOM    604  H   ILE A  39      40.534  10.953  13.354  1.00  0.00           H  \\nATOM    605  HA  ILE A  39      43.288  11.447  14.311  1.00  0.00           H  \\nATOM    606  HB  ILE A  39      41.399  13.185  12.772  1.00  0.00           H  \\nATOM    607 1HG1 ILE A  39      42.239  11.366  11.413  1.00  0.00           H  \\nATOM    608 2HG1 ILE A  39      42.984  12.879  10.932  1.00  0.00           H  \\nATOM    609 1HG2 ILE A  39      43.180  14.503  12.529  1.00  0.00           H  \\nATOM    610 2HG2 ILE A  39      42.737  14.231  14.140  1.00  0.00           H  \\nATOM    611 3HG2 ILE A  39      44.118  13.426  13.586  1.00  0.00           H  \\nATOM    612 1HD1 ILE A  39      44.605  11.073  10.964  1.00  0.00           H  \\nATOM    613 2HD1 ILE A  39      45.005  12.297  12.193  1.00  0.00           H  \\nATOM    614 3HD1 ILE A  39      44.249  10.761  12.677  1.00  0.00           H  \\nATOM    615  N   ARG A  40      40.349  12.674  15.415  1.00  0.00           N  \\nATOM    616  CA  ARG A  40      39.662  13.379  16.601  1.00  0.00           C  \\nATOM    617  C   ARG A  40      40.303  14.871  16.734  1.00  0.00           C  \\nATOM    618  O   ARG A  40      40.316  15.469  17.810  1.00  0.00           O  \\nATOM    619  CB  ARG A  40      39.855  12.591  17.885  1.00  0.00           C  \\nATOM    620  CG  ARG A  40      39.313  11.199  17.854  1.00  0.00           C  \\nATOM    621  CD  ARG A  40      39.463  10.539  19.132  1.00  0.00           C  \\nATOM    622  NE  ARG A  40      38.944   9.186  19.106  1.00  0.00           N  \\nATOM    623  CZ  ARG A  40      39.024   8.316  20.131  1.00  0.00           C  \\nATOM    624  NH1 ARG A  40      39.604   8.676  21.255  1.00  0.00           N  \\nATOM    625  NH2 ARG A  40      38.518   7.101  20.008  1.00  0.00           N  \\nATOM    626  H   ARG A  40      39.791  12.378  14.617  1.00  0.00           H  \\nATOM    627  HA  ARG A  40      38.596  13.436  16.398  1.00  0.00           H  \\nATOM    628 1HB  ARG A  40      40.906  12.527  18.113  1.00  0.00           H  \\nATOM    629 2HB  ARG A  40      39.373  13.114  18.709  1.00  0.00           H  \\nATOM    630 1HG  ARG A  40      38.254  11.227  17.606  1.00  0.00           H  \\nATOM    631 2HG  ARG A  40      39.842  10.622  17.110  1.00  0.00           H  \\nATOM    632 1HD  ARG A  40      40.519  10.493  19.396  1.00  0.00           H  \\nATOM    633 2HD  ARG A  40      38.923  11.096  19.896  1.00  0.00           H  \\nATOM    634  HE  ARG A  40      38.485   8.869  18.248  1.00  0.00           H  \\nATOM    635 1HH1 ARG A  40      39.991   9.604  21.350  1.00  0.00           H  \\nATOM    636 2HH1 ARG A  40      39.664   8.024  22.024  1.00  0.00           H  \\nATOM    637 1HH2 ARG A  40      38.072   6.825  19.144  1.00  0.00           H  \\nATOM    638 2HH2 ARG A  40      38.578   6.449  20.776  1.00  0.00           H  \\nATOM    639  N   ARG A  41      40.806  15.378  15.617  1.00  0.00           N  \\nATOM    640  CA  ARG A  41      41.422  16.780  15.567  1.00  0.00           C  \\nATOM    641  C   ARG A  41      40.560  17.741  14.776  1.00  0.00           C  \\nATOM    642  O   ARG A  41      40.936  18.900  14.601  1.00  0.00           O  \\nATOM    643  CB  ARG A  41      42.802  16.767  14.939  1.00  0.00           C  \\nATOM    644  CG  ARG A  41      43.896  16.020  15.771  1.00  0.00           C  \\nATOM    645  CD  ARG A  41      45.223  16.055  15.092  1.00  0.00           C  \\nATOM    646  NE  ARG A  41      46.230  15.314  15.833  1.00  0.00           N  \\nATOM    647  CZ  ARG A  41      46.977  15.826  16.830  1.00  0.00           C  \\nATOM    648  NH1 ARG A  41      46.818  17.080  17.193  1.00  0.00           N  \\nATOM    649  NH2 ARG A  41      47.869  15.069  17.444  1.00  0.00           N  \\nATOM    650  H   ARG A  41      40.775  14.827  14.771  1.00  0.00           H  \\nATOM    651  HA  ARG A  41      41.504  17.158  16.586  1.00  0.00           H  \\nATOM    652 1HB  ARG A  41      42.753  16.310  13.996  1.00  0.00           H  \\nATOM    653 2HB  ARG A  41      43.143  17.791  14.790  1.00  0.00           H  \\nATOM    654 1HG  ARG A  41      43.998  16.494  16.746  1.00  0.00           H  \\nATOM    655 2HG  ARG A  41      43.611  14.998  15.899  1.00  0.00           H  \\nATOM    656 1HD  ARG A  41      45.136  15.616  14.106  1.00  0.00           H  \\nATOM    657 2HD  ARG A  41      45.558  17.087  15.001  1.00  0.00           H  \\nATOM    658  HE  ARG A  41      46.381  14.346  15.582  1.00  0.00           H  \\nATOM    659 1HH1 ARG A  41      46.136  17.659  16.724  1.00  0.00           H  \\nATOM    660 2HH1 ARG A  41      47.378  17.464  17.941  1.00  0.00           H  \\nATOM    661 1HH2 ARG A  41      47.991  14.105  17.164  1.00  0.00           H  \\nATOM    662 2HH2 ARG A  41      48.429  15.452  18.191  1.00  0.00           H  \\nATOM    663  N   LEU A  42      39.417  17.272  14.274  1.00  0.00           N  \\nATOM    664  CA  LEU A  42      38.557  18.133  13.483  1.00  0.00           C  \\nATOM    665  C   LEU A  42      38.023  19.268  14.348  1.00  0.00           C  \\nATOM    666  O   LEU A  42      38.031  20.430  13.940  1.00  0.00           O  \\nATOM    667  OXT LEU A  42      37.577  19.030  15.470  1.00  0.00           O  \\nATOM    668  CB  LEU A  42      37.387  17.331  12.888  1.00  0.00           C  \\nATOM    669  CG  LEU A  42      36.453  18.110  11.955  1.00  0.00           C  \\nATOM    670  CD1 LEU A  42      37.215  18.521  10.702  1.00  0.00           C  \\nATOM    671  CD2 LEU A  42      35.247  17.247  11.604  1.00  0.00           C  \\nATOM    672  H   LEU A  42      39.150  16.314  14.450  1.00  0.00           H  \\nATOM    673  HA  LEU A  42      39.143  18.549  12.665  1.00  0.00           H  \\nATOM    674 1HB  LEU A  42      37.792  16.491  12.324  1.00  0.00           H  \\nATOM    675 2HB  LEU A  42      36.784  16.936  13.706  1.00  0.00           H  \\nATOM    676  HG  LEU A  42      36.116  19.019  12.453  1.00  0.00           H  \\nATOM    677 1HD1 LEU A  42      36.551  19.076  10.038  1.00  0.00           H  \\nATOM    678 2HD1 LEU A  42      38.059  19.153  10.979  1.00  0.00           H  \\nATOM    679 3HD1 LEU A  42      37.579  17.631  10.189  1.00  0.00           H  \\nATOM    680 1HD2 LEU A  42      34.583  17.802  10.941  1.00  0.00           H  \\nATOM    681 2HD2 LEU A  42      35.583  16.337  11.104  1.00  0.00           H  \\nATOM    682 3HD2 LEU A  42      34.712  16.983  12.516  1.00  0.00           H  \\nTER                                                                             \\n##Begin comments##\\nMembraneInfo1 MembraneCenter   29.203   9.309  -0.621\\nMembraneInfo2 MembraneNormal   -0.144  -0.187  -0.972\\nMembraneInfo3 ATOM   9999  X   MEM A 999      29.203   9.309  -0.621\\nMembraneInfo4 ATOM   9999  Y   MEM A 999      27.038   6.511 -15.198\\nMembraneInfo5 ATOM   9999  Z   MEM A 999      31.367  12.107  13.956\\n##End comments##\\n\\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\nlabel fa_atr fa_rep fa_intra_rep fa_mbenv fa_mbsolv fa_pair hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_ss_dst dslf_cs_ang dslf_ss_dih dslf_ca_dih atom_pair_constraint rama omega fa_dun p_aa_pp ref cart_bonded Menv_smooth total\\nweights 0.8 0.44 0.004 0.3 0.35 0.49 1.17 1.17 2.34 2.2 0.5 2 5 5 0.5 0.2 0.5 0.56 0.32 1 0.5 0.5 NA\\npose -127.659 63.7097 0.50038 10.2388 18.328 -0.13863 -47.036 -1.34777 -0.03314 0 0 0 0 0 2.47073 -1.03172 -2.11779 28.7125 -0.30158 -4.81 138.177 -5.94256 71.7193\\nASP:NtermProteinFull_1 -1.15455 0.08465 0.00321 0.55185 0.44176 -0.0142 0 0 0 0 0 0 0 0 0 0 -0.00617 1.54618 0 -0.67 0.00389 -0.45202 0.33459\\nASN_2 -0.64035 0.03388 0.00548 0.02975 0.25058 -0.0142 0 0 0 0 0 0 0 0 0 -0.30587 -0.07928 1.21874 -0.13027 -0.89 0.0116 -0.21203 -0.72198\\nGLY_3 -1.13406 0.12242 0.00016 0.05548 0.36595 0 0 0 0 0 0 0 0 0 0 -0.10538 -0.07426 0 0.0621 -0.17 0.09235 1.61365 0.82842\\nGLU_4 -1.6319 0.11335 0.00508 0.67481 0.33759 0 0 0 0 0 0 0 0 0 0 -0.17897 -0.06788 2.77392 -0.0055 -0.81 0.13739 -0.72342 0.62448\\nTRP_5 -2.58249 0.21529 0.0152 0.16449 0.53907 0 0 0 0 0 0 0 0 0 0 -0.12856 -0.03498 0.99079 -0.03482 0.91 0.4328 -0.9775 -0.49072\\nILE_6 -1.78063 0.28172 0.02929 -0.30771 0.18285 0 0 0 0 0 0 0 0 0 0 -0.20554 -0.21937 0.53795 0.21042 0.24 0.53448 -0.50327 -0.99979\\nGLN_7 -1.96752 0.39655 0.00753 2.06962 0.19059 0 0 0 0 0 0 0 0 0 0 -0.23206 -0.0443 1.68367 -0.09515 -0.97 0.38702 0.99688 2.42282\\nLEU_8 -2.75615 0.32012 0.0129 0.28982 0.36901 0 0 0 0 0 0 0 0 0 0 -0.19939 -0.06516 0.88129 0.00309 -0.1 0.48024 -0.16339 -0.92763\\nVAL_9 -1.92101 0.78407 0.01696 -0.11286 0.2328 0 0 0 0 0 0 0 0 0 0 -0.13086 -0.07293 0.82923 -0.34749 0.29 1.19054 -0.26398 0.49446\\nPRO_10 -1.10234 0.6322 0.00202 -0.32041 0.1815 0 0 0 0 0 0 0 0 0 0 -0.27296 -0.08718 0.10343 -0.4791 0.02 1.22466 -0.0973 -0.1955\\nARG_11 -1.10794 0.03584 0.00964 0.54743 0.18665 0 0 0 0 0 0 0 0 0 0 -0.05783 -0.0505 1.2432 0.01543 -0.98 0.35995 -1.14663 -0.94476\\nPHE_12 -2.18987 0.20115 0.0178 -0.37804 0.27932 0 0 0 0 0 0 0 0 0 0 -0.18386 -0.04178 1.66143 0.00514 0.63 0.22744 -0.41925 -0.19051\\nSER_13 -2.4887 0.80308 0.00158 0.62409 0.51546 -0.03609 0 0 0 0 0 0 0 0 0 -0.13531 -0.00037 0.19953 -0.12331 -0.37 0.26844 0.25191 -0.4897\\nGLY_14 -1.67733 4.88315 9e-05 0.08478 0.39637 0 0 0 0 0 0 0 0 0 0 0.82272 0.04364 0 0.47152 -0.17 14.0452 0.58809 19.4882\\nPRO_15 -3.01763 0.58655 0.006 0.11389 0.63769 0 0 0 0 0 0 0 0 0 0 1.95468 0.06131 0.53727 0.43792 0.02 23.4103 -0.20164 24.5464\\nGLU_16 -4.36227 6.24293 0.01262 0.68059 0.89476 0.00995 0 0 0 0 0 0 0 0 0 -0.28707 -0.1861 2.47476 -0.12801 -0.81 10.2588 0.54547 15.3464\\nILE_17 -4.47266 4.75379 0.02202 0.0756 0.5423 0 0 0 0 0 0 0 0 0 0 -0.1735 -0.22526 0.62556 0.22754 0.24 3.58364 0.57309 5.77213\\nTHR_18 -3.17305 3.08505 0.00513 1.07485 0.47226 -0.04604 0 0 0 0 0 0 0 0 0 0.06405 -0.03168 0.0264 0.06053 -0.27 5.37933 0.36232 7.00915\\nLEU_19 -4.83694 4.422 0.01291 -0.24419 0.51935 0 0 0 0 0 0 0 0 0 0.24289 -0.06443 -0.02405 0.04059 -0.13159 -0.1 4.98873 -0.78806 4.03721\\nILE_20 -4.13459 4.03395 0.01786 -0.1535 0.41052 0 0 0 0 0 0 0 0 0 0 0.02347 0.00382 1.02846 -0.00895 0.24 5.63213 -0.79518 6.29799\\nILE_21 -4.34977 0.62603 0.02346 -0.23124 0.42552 0 0 0 0 0 0 0 0 0 0 -0.07757 -0.07399 0.06307 -0.2544 0.24 5.00212 -0.81269 0.58054\\nPHE_22 -4.39049 1.91278 0.02094 0.00979 0.42552 0 0 0 0 0 0 0 0 0 0.27307 0.04519 -0.03751 1.13427 -0.24897 0.63 2.62601 -0.90695 1.49364\\nGLY_23 -3.04152 3.29021 4e-05 0.90419 0.34196 0 0 0 0 0 0 0 0 0 0.24048 -0.01709 0.00055 0 0.4167 -0.17 4.62387 1.1475 7.73689\\nVAL_24 -3.31063 2.59465 0.01053 -0.00297 0.36377 0 0 0 0 0 0 0 0 0 0 -0.18388 -0.05161 0.15606 0.04412 0.29 5.24612 -0.33828 4.81786\\nMET_25 -4.7822 1.36472 0.00767 0.2046 0.49298 0 0 0 0 0 0 0 0 0 0 -0.1542 -0.01493 0.89039 0.04074 -0.34 3.05985 -0.0681 0.70152\\nALA_26 -3.30185 0.67738 0.00144 0.4956 0.32973 0 0 0 0 0 0 0 0 0 0.24449 -0.15427 -0.01285 0 -0.04765 0.16 2.34482 0.50989 1.24672\\nGLY_27 -2.96175 2.47786 4e-05 0.9042 0.33342 0 0 0 0 0 0 0 0 0 0.24289 -0.06813 -0.0384 0 0.38995 -0.17 3.73141 1.03797 5.87945\\nILE_28 -3.91087 2.19185 0.01635 -0.24031 0.42714 0 0 0 0 0 0 0 0 0 0 -0.09814 -0.00368 0.23171 -0.04783 0.24 4.43374 -0.79518 2.44477\\nILE_29 -4.37171 1.02319 0.02872 -0.24051 0.44112 0 0 0 0 0 0 0 0 0 0 -0.06906 -0.05551 0.06361 -0.25256 0.24 3.11151 -0.79716 -0.87837\\nGLY_30 -3.06246 0.32941 0.00038 0.9037 0.3044 0 0 0 0 0 0 0 0 0 0.5075 0.24462 -0.15782 0 0.11416 -0.17 2.48002 1.07453 2.56844\\nTHR_31 -4.16594 3.42261 0.00493 1.39175 0.46635 0 0 0 0 0 0 0 0 0 0.24048 -0.07025 -0.02983 0.40577 0.03239 -0.27 5.11119 0.57813 7.11758\\nILE_32 -3.50607 2.51612 0.01959 -0.25356 0.38854 0 0 0 0 0 0 0 0 0 0 -0.20739 -0.09679 0.44635 0.05272 0.24 6.2189 -0.79518 5.02324\\nLEU_33 -4.51414 0.85857 0.01496 -0.28639 0.55971 0 0 0 0 0 0 0 0 0 0 -0.11286 0.01975 0.62037 -0.10187 -0.1 3.16194 -0.82389 -0.70385\\nLEU_34 -4.1324 0.53621 0.017 -0.3061 0.48257 0 0 0 0 0 0 0 0 0 0.24449 -0.11408 -0.01398 0.24701 -0.08028 -0.1 2.16795 -0.82365 -1.87526\\nILE_35 -5.30782 2.22559 0.02886 -0.39747 0.78403 0 0 0 0 0 0 0 0 0 0 -0.12446 -0.05246 0.26351 -0.15088 0.24 2.47571 -0.77614 -0.79155\\nSER_36 -3.54997 0.915 0.001 0.8579 0.70093 -0.01903 0 0 0 0 0 0 0 0 0 -0.20402 -0.10651 0.30188 0.07206 -0.37 2.46677 0.19332 1.25932\\nTYR_37 -3.71988 0.90866 0.01852 0.35565 0.72938 0 0 0 -0.01657 0 0 0 0 0 0 0.08649 -0.02239 2.05525 -0.04267 0.51 1.25175 -0.04825 2.06594\\nSER_38 -2.64361 0.18717 0.00171 0.94348 0.47974 -0.01903 0 0 0 0 0 0 0 0 0.23444 -0.25719 -0.03895 0.23214 0.07354 -0.37 0.62038 0.25136 -0.3048\\nILE_39 -4.47886 1.97098 0.04616 -0.32023 0.61575 0 0 0 0 0 0 0 0 0 0 0.14516 -0.02747 0.50887 -0.2545 0.24 1.60265 0.57309 0.62162\\nARG_40 -2.82774 0.47416 0.00973 0.13724 0.63456 0 0 0 -0.01657 0 0 0 0 0 0 0.20834 -0.04812 1.00752 0.00135 -0.98 2.17129 -1.19737 -0.4256\\nARG_41 -2.05892 1.14598 0.01195 0.15282 0.40455 0 0 0 0 0 0 0 0 0 0 -0.05225 -0.05284 1.39552 -0.06721 -0.98 1.20877 -1.15715 -0.04878\\nLEU:CtermProteinFull_42 -1.13846 0.02882 0.01292 -0.26365 0.25 0 0 0 0 0 0 0 0 0 0 0 0 0.28683 0 -0.1 0.41176 -0.15607 -0.66785\\n#END_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\n\\n\",\"pdb\");\n",
              "\tviewer_16642132334955251.zoomTo();\n",
              "\tviewer_16642132334955251.setBackgroundColor(\"white\");\n",
              "\tviewer_16642132334955251.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"purple\"}});\n",
              "\tviewer_16642132334955251.setStyle({\"chain\": \" \"},{\"cartoon\": {\"color\": \"yellow\"}});\n",
              "viewer_16642132334955251.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "error",
          "ename": "KeyboardInterrupt",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-26-270e697b7c16>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     14\u001b[0m \u001b[0;31m#representation in Google Colab and, once you are happy with the visualization,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m \u001b[0;31m#press Enter to print out as an onscreen png image file that you can download\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 16\u001b[0;31m \u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     17\u001b[0m \u001b[0mview\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpng\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36mraw_input\u001b[0;34m(self, prompt)\u001b[0m\n\u001b[1;32m    861\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_ident\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    862\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_header\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 863\u001b[0;31m             \u001b[0mpassword\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    864\u001b[0m         )\n\u001b[1;32m    865\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    902\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    903\u001b[0m                 \u001b[0;31m# re-raise KeyboardInterrupt, to truncate traceback\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 904\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Interrupted by user\"\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    905\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    906\u001b[0m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mlog\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwarning\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Invalid Message:\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mexc_info\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mKeyboardInterrupt\u001b[0m: Interrupted by user"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JFpJRYPwJ8w6"
      },
      "source": [
        "**üìöHOMEWORK:** As a challenge, using what you have learned in the previous tutorials to superimpose structures and determine their similarity using RMSD, take your best structures and compare them to answer the following question: **Is the TM region different between orthologs?**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7CV7AsCQIFrB"
      },
      "source": [
        "5. To finalize, you can download your output files as indicated in the code cell below"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "oy4qyfAGMDFp"
      },
      "source": [
        "#download your output\n",
        "%%bash\n",
        "zip -r example.zip ./outs\n",
        "mkdir outs\n",
        "cp *.pdb ./outs/"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "lJpeTpYPIKfB"
      },
      "source": [
        "**This is the end of the fifth tutorial. Good Science!**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "s9giUnnEAast"
      },
      "source": [
        "#Appendix A"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "n1NoW-mIX3Fp"
      },
      "source": [
        "## Generating the input files using the standalone command-line version of Rosetta"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "FF5pgAKmh14Y"
      },
      "source": [
        "First, you need to get a Rosetta license [HERE](https://els2.comotion.uw.edu/product/rosetta), as you did for PyRosetta"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "pZ8OMgudYJi9"
      },
      "source": [
        "### Download and extract Rosetta in Google Colab (ETA ~15min)"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9eLMeKl_vyuf"
      },
      "source": [
        "!wget --user=Academic_User --password=Xry3x4 https://www.rosettacommons.org/downloads/academic/3.12/rosetta_bin_linux_3.12_bundle.tgz\n",
        "!tar xvf /content/rosetta_bin_linux_3.12_bundle.tgz"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "5XUiyaTK6nxy"
      },
      "source": [
        "!unzip input_files.zip"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "oovXJfZUl5g7"
      },
      "source": [
        "### Grishin File Generation"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ZaQNZszVl8bf"
      },
      "source": [
        "The Grishin alignment file format is documented [here](https://www.rosettacommons.org/docs/latest/rosetta_basics/file_types/Grishan-format-alignment). The following is an example of its structure\n",
        "```\n",
        "## target_name template_name.pdb\n",
        "#\n",
        "scores from program: 0\n",
        "0 HERE_GOES_YOUR_TARGET_SEQUENCE_ALIGNMENT_IN_ONE_LINE \n",
        "0 HERE_GOES_YOUR_TEMPLATE_SEQUENCE_ALIGNMENT_IN_ONE_LINE\n",
        "```\n",
        "This is one of the real Grishin files used in this tutorial\n",
        "```\n",
        "## gorillaGYPA 5eh6_A.pdb\n",
        "#\n",
        "scores from program: 0\n",
        "0 DNGEWIQLVPRFSGPEITLIIFGVMAGIIGTILLISYSIRRL \n",
        "0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n",
        "```\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "RmZi9ejLmvYF"
      },
      "source": [
        "#Modify and run this cell to create grishin from clustal alignment\n",
        "%%bash\n",
        "echo \\\n",
        "\"\"\"\\\n",
        "## gorillaGYPA 5eh6_A.pdb\n",
        "#\n",
        "scores from program: 0\n",
        "0 DNGEWIQLVPRFSGPEITLIIFGVMAGIIGTILLISYSIRRL \n",
        "0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n",
        "\"\"\" > /content/input_files/grishin/5eh6_on_gorillaGYPA.grishin\n",
        "\n",
        "echo \\\n",
        "\"\"\"\\\n",
        "## GYPA 5eh6_A.pdb\n",
        "#\n",
        "scores from program: 0\n",
        "0 ETGERVQLAHHFSEPEITLIIFGVMAGVIGTILLISYGIRRL \n",
        "0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n",
        "\"\"\" > /content/input_files/grishin/5eh6_on_humanGYPA.grishin\n",
        "\n",
        "echo \\\n",
        "\"\"\"\\\n",
        "## marmotaGYPA 5eh6_A.pdb\n",
        "#\n",
        "scores from program: 0\n",
        "0 -KQGRNQIIHPFSEPVIILIIFAVMFGIIGTILLISFCVRRL\n",
        "0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n",
        "\"\"\" > /content/input_files/grishin/5eh6_on_marmotaGYPA.grishin\n",
        "\n",
        "\n",
        "echo \\\n",
        "\"\"\"\\\n",
        "## OrnithorhynchusGYPA 5eh6_A.pdb\n",
        "#\n",
        "scores from program: 0\n",
        "0 ---IDPEIEHIFTGPVAAVIIYAVVCGVIGTILFIALVIKVV\n",
        "0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n",
        "\"\"\" > /content/input_files/grishin/5eh6_on_ornithorhynchusGYPA.grishin"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "iAfdLyUYYV40"
      },
      "source": [
        "### Partial Threading"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "COXQteMfi0Ku",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a7d4eb10-b528-4677-c0b4-43c5c2183ad9"
      },
      "source": [
        "import urllib.request \n",
        "urllib.request.urlretrieve('http://files.rcsb.org/download/5eh6.pdb', '/content/input_files/pdbs/5eh6.pdb')"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "('/content/input_files/pdbs/5eh6.pdb',\n",
              " <http.client.HTTPMessage at 0x7ff0199e8390>)"
            ]
          },
          "metadata": {},
          "execution_count": 26
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "z6Z-k7TDigtJ",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 481
        },
        "outputId": "6d1f6b45-7936-412a-80b2-70ef47f41ec5"
      },
      "source": [
        "%%bash\n",
        "cd /content/input_files/pdbs\n",
        "/content/rosetta_bin_linux_2020.08.61146_bundle/tools/protein_tools/scripts/clean_pdb.py 5eh6.pdb A\n",
        "mv 5eh6_A.* ../cleaned"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "bash: line 2: /content/rosetta_bin_linux_2020.08.61146_bundle/tools/protein_tools/scripts/clean_pdb.py: No such file or directory\n",
            "mv: cannot stat '5eh6_A.*': No such file or directory\n"
          ]
        },
        {
          "output_type": "error",
          "ename": "CalledProcessError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mCalledProcessError\u001b[0m                        Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-27-84facaa76a9b>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrun_cell_magic\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'bash'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'cd /content/input_files/pdbs\\n/content/rosetta_bin_linux_2020.08.61146_bundle/tools/protein_tools/scripts/clean_pdb.py 5eh6.pdb A\\nmv 5eh6_A.* ../cleaned\\n'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/IPython/core/interactiveshell.py\u001b[0m in \u001b[0;36mrun_cell_magic\u001b[0;34m(self, magic_name, line, cell)\u001b[0m\n\u001b[1;32m   2357\u001b[0m             \u001b[0;32mwith\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbuiltin_trap\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2358\u001b[0m                 \u001b[0margs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mmagic_arg_s\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 2359\u001b[0;31m                 \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mfn\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   2360\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2361\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/IPython/core/magics/script.py\u001b[0m in \u001b[0;36mnamed_script_magic\u001b[0;34m(line, cell)\u001b[0m\n\u001b[1;32m    140\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    141\u001b[0m                 \u001b[0mline\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mscript\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 142\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mshebang\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    143\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    144\u001b[0m         \u001b[0;31m# write a basic docstring:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m<decorator-gen-103>\u001b[0m in \u001b[0;36mshebang\u001b[0;34m(self, line, cell)\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/IPython/core/magic.py\u001b[0m in \u001b[0;36m<lambda>\u001b[0;34m(f, *a, **k)\u001b[0m\n\u001b[1;32m    185\u001b[0m     \u001b[0;31m# but it's overkill for just that one bit of state.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    186\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mmagic_deco\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 187\u001b[0;31m         \u001b[0mcall\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mlambda\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m*\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mk\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mk\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    188\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    189\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mcallable\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/IPython/core/magics/script.py\u001b[0m in \u001b[0;36mshebang\u001b[0;34m(self, line, cell)\u001b[0m\n\u001b[1;32m    243\u001b[0m             \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstderr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflush\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    244\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0margs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mraise_error\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreturncode\u001b[0m\u001b[0;34m!=\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 245\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mCalledProcessError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreturncode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0moutput\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0merr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    246\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    247\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_run_script\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mp\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcell\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mto_close\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mCalledProcessError\u001b[0m: Command 'b'cd /content/input_files/pdbs\\n/content/rosetta_bin_linux_2020.08.61146_bundle/tools/protein_tools/scripts/clean_pdb.py 5eh6.pdb A\\nmv 5eh6_A.* ../cleaned\\n'' returned non-zero exit status 1."
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Yw46R_v1560X"
      },
      "source": [
        "!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/GYPA.fasta -in:file:alignment ./grishin/5eh6.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n",
        "!mv 5eh6_A.pdb.pdb 5eh6_A.human.pdb\n",
        "!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/ornithorhynchusGYPA.fasta -in:file:alignment ./grishin/5eh6_on_ornithorhynchusGYPA.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n",
        "!mv 5eh6_A.pdb.pdb 5eh6_A.ornithorhynchus.pdb\n",
        "!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/marmotaGYPA.fasta -in:file:alignment ./grishin/5eh6_on_marmotaGYPA.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n",
        "!mv 5eh6_A.pdb.pdb 5eh6_A.gorilla.pdb\n",
        "!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/gorillaGYPA.fasta -in:file:alignment ./grishin/5eh6_on_gorillaGYPA.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n",
        "!mv 5eh6_A.pdb.pdb 5eh6_A.marmota.pdb\n",
        "!mv 5eh6_A.*pdb ./input_files/threaded_pdbs/"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "2Wn-5LuU44Wm"
      },
      "source": [
        "### Getting the fragments from oldRobetta server\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "LmPktANT5uEe"
      },
      "source": [
        "As stated in the Rosetta documentation fragments are:\n",
        "\n",
        "\n",
        "> (...) used in the assembly of proteins whether for structure prediction or design, to cut down on the size of the protein-folding search space. They are a core part of the Rosetta design.\n",
        "\n",
        "For the purpose of this tutorial, we skipped this part. However, in the following figure you can see a screenshot of the [Robetta-server](http://old.robetta.org/fragmentsubmit.jsp) where you can obtain the fragments. For this, you have to register and submit a FASTA sequence of your target proteins. Do keep in mind that the length of your target will impact in the time needed for the fragment picking process."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Neo5VCZVJoBl"
      },
      "source": [
        "<figure>\n",
        "<center>\n",
        "<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_04.png'/>\n",
        "</figcaption></center>\n",
        "</figure>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xvXqKZZdZNjJ"
      },
      "source": [
        "When your job is finished you will get an email with the corresponding fragments. You have to download them and place them inside the frags folder and make sure that their filename matches the one in XML RosettaScripts cell"
      ]
    }
  ]
}