{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[]},"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"}},"cells":[{"cell_type":"markdown","metadata":{"id":"gDvK9xHtcjTr"},"source":["#Lab.05 / IBM3202 ‚Äì¬†Membrane Protein Modelling using PyRosetta"]},{"cell_type":"markdown","metadata":{"id":"CTFZC7Mafuzu"},"source":["##Theoretical Aspects"]},{"cell_type":"markdown","metadata":{"id":"VWTkCQifjWXY"},"source":["As stated by Alford et al. 2020 [Biophys J 2020 118(8), 2042-2055], membrane proteins partner with the surrounding lipid environment to perform essential life processes. They constitute 30% of all proteins and are targets for over 60% of pharmaceuticals. However, experimental difficulties have limited our insights into their molecular mechanisms of function."]},{"cell_type":"markdown","metadata":{"id":"0nPD5ByvctQy"},"source":["<figure>\n","<center>\n","<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_01.jpg'/>\n","<figcaption>FIGURE 1. The challenge of solving the structure of membrane proteins. As shown in this graph, the number of membrane proteins solved before the extensive use of cryoEM was (and still is) significantly inferior to the number of soluble proteins.\n","<br>Bijelic A & Rompel A (2015) <i>Coord Chem Rev 299, 22-38.\n","</i></figcaption></center>\n","</figure>"]},{"cell_type":"markdown","metadata":{"id":"iHpJP9wCdgqp"},"source":["Therefore, membrane protein modelling is ideal for determining the structure of these proteins and for developing new therapeutics. In this tutorial we will learn to model membrane proteins in PyRosetta and provide examples of the types of problems one can solve with this framework."]},{"cell_type":"markdown","metadata":{"id":"wr-vcvaVTKTf"},"source":["##Overview"]},{"cell_type":"markdown","metadata":{"id":"62vLxb1sSVVf"},"source":["In this tutorial, we will model different homologs with varying sequence identities of the membrane protein Glycophorin A, also known as GYPA. This is a pore-forming protein that, in the case of the human ortholog, has been broadly studied using different structural biology methods such as NMR and X-Ray crystallography.\n","\n","This overview and tutorial include information from this [tutorial](http://www.meilerlab.org/jobs/downloadfile/name/May2018_rosetta_cm_presentation.pdf) by Nica Marlow and Georg Kuenze"]},{"cell_type":"markdown","metadata":{"id":"jmsmt7SGbTmQ"},"source":["**The Algorithm**\n","\n","At a high-level, the algorithm consists of a long Monte Carlo trajectory starting from a randomly-chosen template. The MC trajectory employs the following moves:\n","\n","* Fragment insertion in unaligned regions\n","\n","* Replacement of a randomly-chosen segment with that from a different template structure\n","\n","* Cartesian-space minimization using a smooth (differentiable) version of the Rosetta centroid energy function. Finally, this is followed by all-atom optimization.\n","\n","See more at the [official docs](https://www.rosettacommons.org/docs/latest/application_documentation/structure_prediction/RosettaCM)"]},{"cell_type":"markdown","metadata":{"id":"_rm1_EjsTMWF"},"source":["**Differences between Single and Multiple Template modeling**\n","\n","Single Template Modeling:\n","- Single template as input\n","- Uses sequence and template derived\n","fragments\n","- Used when available templates have very\n","high identity (>60%)\n","\n","Multiple Template Modeling:\n","- Multiple templates as input\n","- Combine sections of multiple threaded\n","models and sequence derived fragments\n","- Used when available templates have low\n","identity (30-50%)"]},{"cell_type":"markdown","metadata":{"id":"eiINEFYXfajF"},"source":["**The protocol**\n","\n","Rosetta comparative modelling (**RosettaCM**) performs a complex series of steps that are illustrated in the scheme below, but that can be summarize in three main steps:\n","\n","1. Align target sequence to template sequences\n","\n","2. Perform a partial-thread of the target sequence onto template structures\n","\n","3. Combine pieces from different templates using RosettaCM hybridize"]},{"cell_type":"markdown","metadata":{"id":"NK5aEeL_e2w_"},"source":["<figure>\n","<center>\n","<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_02.jpg'/>\n","<figcaption>FIGURE 2. A simple scheme of the RosettaCM protocol\n","<br>Song Y et al (2013) <i>Structure 21(10), 1735-1742</i></figcaption></center>\n","</figure>"]},{"cell_type":"markdown","metadata":{"id":"eTZzTm6SUOIH"},"source":["**Input Files for RosettaCM**\n","\n","Bare minimum:\n","\n","* Partial-threaded structures (‚úÖ)\n","\n","* Mover definition and options (‚úÖ)\n","\n","Specific to membrane proteins (not needed if modeling\n","soluble proteins):\n","\n","* Membrane spanning regions (span file) (üôã)\n","\n","* Membrane weight patches  (‚úÖ)\n","\n","Optional files based on available information:\n","\n","* Constraint information (eg. atom pair connectivity)\n","\n","* Disulfide Connectivity"]},{"cell_type":"markdown","metadata":{"id":"SKd7eVaiRunj"},"source":["**For a deep dive into membrane protein modelling using Rosetta, we suggest to check the following articles:**\n","1. Alford RF, Leman JK et al., ‚ÄúAn Integrated Framework Advancing Membrane Protein Modeling and Design,‚Äù PLOS Computational Biology. 2015;11(9).\n","2. Alford RF, Fleming PJ, Fleming KG, Gray JJ (2020) ‚ÄúProtein structure prediction and design in a biologically-realistic implicit membrane‚Äù BioRxiv\n","3. Koehler Leman J, Muller BK, Gray JJ (2017) ‚ÄúExpanding the toolkit for membrane protein modeling in Rosetta‚Äù Bioinformatics 33(5):754-756.\n","4. Koehler Leman J & Gray JJ (2015) ‚ÄúComputational modeling of membrane proteins‚Äù Proteins Structure, Function, and Bioinformatics 83(1):1-24."]},{"cell_type":"markdown","metadata":{"id":"U2vWpWg8klhM"},"source":["#Part I ‚Äì Exploring the sequences of GYPA orthologs"]},{"cell_type":"markdown","metadata":{"id":"D-u6sgce8Zxm"},"source":["1. The sequences that we are going to be working with today  correspond to three ortologs of the Glycophorin A membrane protein. In our particular case we are going to model the following ortologs using PyRosetta:\n","\n","  *   [Marmota flaviventris](https://www.ncbi.nlm.nih.gov/protein/XP_027782564.1?report=fasta&log$=seqview)\n","\n","  *   [Gorilla gorilla](https://www.ncbi.nlm.nih.gov/protein/XP_007666620.2?report=fasta&log$=seqview)\n","  \n","  *   [Ornithorhynchus anatinus](https://www.uniprot.org/uniprot/O18798)"]},{"cell_type":"markdown","metadata":{"id":"xHLARFm0-8ae"},"source":["<figure>\n","<center>\n","<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_03.png'/>\n","</figcaption></center>\n","</figure>"]},{"cell_type":"markdown","metadata":{"id":"ESDFFpOd-pYU"},"source":["2. For the sake of time we are going to provide the already aligned and trimmed sequences in a Grishin format file (yet another alignment format), the partial-threaded structures required to run the modelling protocol and the fragment files:\n","\n","  You will find these files inside its corresponding folder inside input_files. If you want to pursue the generation of these input files on your own, check **Appendix A**.\n","\n"]},{"cell_type":"code","metadata":{"id":"ImD_xOcegVuQ","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446322539,"user_tz":-120,"elapsed":3089,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"5c0f3030-8661-4887-93fe-e3279f4ebc9a"},"source":["#Here we download the input files from Github\n","!wget https://raw.githubusercontent.com/pb3lab/ibm3202/master/input_files/input_files_mp.zip\n","#This remove command is just in case you had some previous input files\n","!rm -rf input_files\n","#Here, we unzip the downloaded files\n","!unzip input_files_mp.zip"],"execution_count":1,"outputs":[{"output_type":"stream","name":"stdout","text":["--2024-10-20 17:45:19--  https://raw.githubusercontent.com/pb3lab/ibm3202/master/input_files/input_files_mp.zip\n","Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n","Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 6407895 (6.1M) [application/zip]\n","Saving to: ‚Äòinput_files_mp.zip‚Äô\n","\n","input_files_mp.zip  100%[===================>]   6.11M  37.0MB/s    in 0.2s    \n","\n","2024-10-20 17:45:20 (37.0 MB/s) - ‚Äòinput_files_mp.zip‚Äô saved [6407895/6407895]\n","\n","Archive:  input_files_mp.zip\n","   creating: input_files/\n","   creating: input_files/grishin/\n","  inflating: __MACOSX/input_files/._grishin  \n","  inflating: input_files/.DS_Store   \n","  inflating: __MACOSX/input_files/._.DS_Store  \n","   creating: input_files/cleaned/\n","  inflating: __MACOSX/input_files/._cleaned  \n","   creating: input_files/fasta/\n","  inflating: __MACOSX/input_files/._fasta  \n","  inflating: input_files/gcm.xml     \n","   creating: input_files/threaded_pdbs/\n","  inflating: __MACOSX/input_files/._threaded_pdbs  \n","  inflating: input_files/gcm.options  \n","   creating: input_files/pdbs/\n","  inflating: __MACOSX/input_files/._pdbs  \n","   creating: input_files/octopus/\n","  inflating: __MACOSX/input_files/._octopus  \n","   creating: input_files/wts/\n","  inflating: __MACOSX/input_files/._wts  \n","   creating: input_files/frags/\n","  inflating: __MACOSX/input_files/._frags  \n","   creating: input_files/clustal/\n","  inflating: __MACOSX/input_files/._clustal  \n","  inflating: input_files/grishin/.DS_Store  \n","  inflating: input_files/grishin/Icon  \n","  inflating: __MACOSX/input_files/grishin/._Icon  \n","  inflating: input_files/grishin/5eh6_on_humanGYPA.grishin  \n","  inflating: input_files/grishin/5eh6_on_ornithorhynchusGYPA.grishin  \n","  inflating: input_files/grishin/5eh6_on_gorillaGYPA.grishin  \n","  inflating: input_files/grishin/5eh6_on_marmotaGYPA.grishin  \n","  inflating: input_files/cleaned/.DS_Store  \n","  inflating: input_files/cleaned/Icon  \n","  inflating: __MACOSX/input_files/cleaned/._Icon  \n","  inflating: input_files/cleaned/5eh6_A.pdb  \n","  inflating: input_files/cleaned/5eh6_A.fasta  \n","  inflating: __MACOSX/input_files/cleaned/._5eh6_A.fasta  \n","  inflating: input_files/fasta/.DS_Store  \n","  inflating: __MACOSX/input_files/fasta/._.DS_Store  \n","  inflating: input_files/fasta/Icon  \n","  inflating: __MACOSX/input_files/fasta/._Icon  \n","  inflating: input_files/fasta/gorillaGYPA.fasta  \n","  inflating: __MACOSX/input_files/fasta/._gorillaGYPA.fasta  \n","  inflating: input_files/fasta/ornithorhynchusGYPA.fasta  \n","  inflating: __MACOSX/input_files/fasta/._ornithorhynchusGYPA.fasta  \n","  inflating: input_files/fasta/marmotaGYPA.fasta  \n","  inflating: __MACOSX/input_files/fasta/._marmotaGYPA.fasta  \n","  inflating: input_files/fasta/GYPA.fasta  \n","  inflating: __MACOSX/input_files/fasta/._GYPA.fasta  \n","  inflating: input_files/threaded_pdbs/.DS_Store  \n","  inflating: input_files/threaded_pdbs/5eh6_A.human.pdb  \n","  inflating: input_files/threaded_pdbs/Icon  \n","  inflating: __MACOSX/input_files/threaded_pdbs/._Icon  \n","  inflating: input_files/threaded_pdbs/5eh6_A.marmota.pdb  \n","  inflating: input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb  \n","  inflating: input_files/threaded_pdbs/5eh6_A.gorilla.pdb  \n","  inflating: input_files/pdbs/5eh6.pdb  \n","  inflating: __MACOSX/input_files/pdbs/._5eh6.pdb  \n","  inflating: input_files/pdbs/.DS_Store  \n","  inflating: input_files/pdbs/Icon   \n","  inflating: __MACOSX/input_files/pdbs/._Icon  \n","  inflating: input_files/octopus/.DS_Store  \n","  inflating: __MACOSX/input_files/octopus/._.DS_Store  \n","  inflating: input_files/octopus/Icon  \n","  inflating: __MACOSX/input_files/octopus/._Icon  \n","  inflating: input_files/octopus/ornithorhynchus.topo.txt  \n","  inflating: __MACOSX/input_files/octopus/._ornithorhynchus.topo.txt  \n","  inflating: input_files/octopus/marmota.span  \n","  inflating: input_files/octopus/octopus2span.pl  \n","  inflating: input_files/octopus/gorilla.topo.txt  \n","  inflating: __MACOSX/input_files/octopus/._gorilla.topo.txt  \n","  inflating: input_files/octopus/gorilla.span  \n","  inflating: input_files/octopus/marmota.topo.txt  \n","  inflating: __MACOSX/input_files/octopus/._marmota.topo.txt  \n","  inflating: input_files/octopus/GYPA.topo.txt  \n","  inflating: __MACOSX/input_files/octopus/._GYPA.topo.txt  \n","  inflating: input_files/octopus/ornithorhynchus.span  \n","  inflating: input_files/octopus/GYPA.span  \n","  inflating: input_files/wts/stage1_membrane.wts  \n","  inflating: input_files/wts/rosetta_cm_3.8.xml  \n","  inflating: input_files/wts/.DS_Store  \n","  inflating: input_files/wts/Icon    \n","  inflating: __MACOSX/input_files/wts/._Icon  \n","  inflating: input_files/wts/ref2015.wts  \n","  inflating: input_files/wts/stage3_rlx_membrane.wts  \n","  inflating: input_files/wts/stage2_membrane.wts  \n","  inflating: input_files/frags/GYPA.9mers  \n","  inflating: __MACOSX/input_files/frags/._GYPA.9mers  \n","  inflating: input_files/frags/.DS_Store  \n","  inflating: input_files/frags/marmotaGYPA.9mers  \n","  inflating: __MACOSX/input_files/frags/._marmotaGYPA.9mers  \n","  inflating: input_files/frags/Icon  \n","  inflating: __MACOSX/input_files/frags/._Icon  \n","  inflating: input_files/frags/ornithorhynchusGYPA.3mers  \n","  inflating: __MACOSX/input_files/frags/._ornithorhynchusGYPA.3mers  \n","  inflating: input_files/frags/gorillaGYPA.3mers  \n","  inflating: __MACOSX/input_files/frags/._gorillaGYPA.3mers  \n","  inflating: input_files/frags/gorillaGYPA.9mers  \n","  inflating: __MACOSX/input_files/frags/._gorillaGYPA.9mers  \n","  inflating: input_files/frags/ornithorhynchusGYPA.9mers  \n","  inflating: __MACOSX/input_files/frags/._ornithorhynchusGYPA.9mers  \n","  inflating: input_files/frags/GYPA.3mers  \n","  inflating: __MACOSX/input_files/frags/._GYPA.3mers  \n","  inflating: input_files/frags/marmotaGYPA.3mers  \n","  inflating: __MACOSX/input_files/frags/._marmotaGYPA.3mers  \n","  inflating: input_files/clustal/clustalo-I20201006-164940-0747-5352144-p1m.trimmed.aln  \n","  inflating: __MACOSX/input_files/clustal/._clustalo-I20201006-164940-0747-5352144-p1m.trimmed.aln  \n","  inflating: input_files/clustal/.DS_Store  \n","  inflating: input_files/clustal/.crtz_promals3d_full_edit_alt.fasta.swp  \n","  inflating: input_files/clustal/Icon  \n","  inflating: __MACOSX/input_files/clustal/._Icon  \n","  inflating: input_files/clustal/clustalo-I20201006-164940-0747-5352144-p1m.full.aln  \n","  inflating: __MACOSX/input_files/clustal/._clustalo-I20201006-164940-0747-5352144-p1m.full.aln  \n"]}]},{"cell_type":"markdown","metadata":{"id":"agqXm_0dmf8k"},"source":["#Part II ‚Äì Determining the membrane spanning regions (span file) using OCTOPUS server"]},{"cell_type":"markdown","metadata":{"id":"a0Qhw-MSmtjk"},"source":["As we briefly noted at the begining of this tutorial (Overview), we need to give Rosetta a way of knowing to which residues a specific and implicit \"membrane\" potential needs to be applied. These instructions are generated based on the transmembrane (TM) prediction server [**OCTOPUS**](https://octopus.cbr.su.se)"]},{"cell_type":"markdown","metadata":{"id":"ZIycpMeaCwI4"},"source":["1. In [**OCTOPUS**](https://octopus.cbr.su.se) you need to input the target protein sequence for the TM prediction. You can find these protein sequences inside the FASTA folder within the downloaded input files."]},{"cell_type":"markdown","metadata":{"id":"jNyDLK3CCShq"},"source":["2. The, the topology files must be processed to convert them into span files as shown in the code cell below:"]},{"cell_type":"code","metadata":{"id":"u_pxQELHbPX7","executionInfo":{"status":"ok","timestamp":1729446328256,"user_tz":-120,"elapsed":293,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}}},"source":["%%bash\n","cd /content/input_files/octopus\n","perl octopus2span.pl GYPA.topo.txt > GYPA.span\n","perl octopus2span.pl marmota.topo.txt > marmota.span\n","perl octopus2span.pl ornithorhynchus.topo.txt > ornithorhynchus.span\n","perl octopus2span.pl gorilla.topo.txt > gorilla.span"],"execution_count":2,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"zjUSFLdJCY7f"},"source":["3. Once you obtained all the span files from **OCTOPUS**, remember to place them inside the following folder\n","\n","```\n","/content/input_files/octopus #Sweet trick! You can open the folder clicking this path\n","```\n","\n"]},{"cell_type":"markdown","metadata":{"id":"o3hcWpFQXoUx"},"source":["# Part III ‚Äì Setting Up PyRosetta and Executing the Membrane Protein Modelling protocols\n"]},{"cell_type":"markdown","metadata":{"id":"i4mcFtHR6u_-"},"source":["1. We start by setting up PyRosetta in Google Colab, which will be installed in your Google Drive, for which you must provide access."]},{"cell_type":"code","metadata":{"id":"SePtDf1JST8a","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446416325,"user_tz":-120,"elapsed":81748,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"a2acb3d4-b2e1-44b9-e5ef-8d88a2b94c39"},"source":["!pip install pyrosettacolabsetup\n","import pyrosettacolabsetup; pyrosettacolabsetup.install_pyrosetta()\n","import pyrosetta; pyrosetta.init()"],"execution_count":3,"outputs":[{"output_type":"stream","name":"stdout","text":["Collecting pyrosettacolabsetup\n","  Downloading pyrosettacolabsetup-1.0.9-py3-none-any.whl.metadata (294 bytes)\n","Downloading pyrosettacolabsetup-1.0.9-py3-none-any.whl (4.9 kB)\n","Installing collected packages: pyrosettacolabsetup\n","Successfully installed pyrosettacolabsetup-1.0.9\n","Mounted at /content/google_drive\n","\n","Note that USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE.\n","See https://github.com/RosettaCommons/rosetta/blob/main/LICENSE.md or email license@uw.edu for details.\n","\n","Looking for compatible PyRosetta wheel file at google-drive/PyRosetta/colab.bin//wheels...\n","Found compatible wheel: /content/google_drive/MyDrive/PyRosetta/colab.bin/wheels//content/google_drive/MyDrive/PyRosetta/colab.bin/wheels/pyrosetta-2024.42+release.3366cf78a3-cp310-cp310-linux_x86_64.whl\n","\n","\n","‚îå‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îê\n","‚îÇ                                 PyRosetta-4                                  ‚îÇ\n","‚îÇ              Created in JHU by Sergey Lyskov and PyRosetta Team              ‚îÇ\n","‚îÇ              (C) Copyright Rosetta Commons Member Institutions               ‚îÇ\n","‚îÇ                                                                              ‚îÇ\n","‚îÇ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE ‚îÇ\n","‚îÇ         See LICENSE.PyRosetta.md or email license@uw.edu for details         ‚îÇ\n","‚îî‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îò\n","PyRosetta-4 2024 [Rosetta PyRosetta4.MinSizeRel.python310.ubuntu 2024.42+release.3366cf78a3df04339d1982e94531b77b098ddb99 2024-10-11T08:24:04] retrieved from: http://www.pyrosetta.org\n","core.init: Checking for fconfig files in pwd and ./rosetta/flags\n","core.init: Rosetta version: PyRosetta4.MinSizeRel.python310.ubuntu r388 2024.42+release.3366cf78a3 3366cf78a3df04339d1982e94531b77b098ddb99 http://www.pyrosetta.org 2024-10-11T08:24:04\n","core.init: Rosetta extras: []\n","core.init: command: PyRosetta -ex1 -ex2aro -database /usr/local/lib/python3.10/dist-packages/pyrosetta/database\n","basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=1180306035 seed_offset=0 real_seed=1180306035\n","basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=1180306035 RG_type=mt19937\n"]}]},{"cell_type":"markdown","metadata":{"id":"MglIIrmAhWZ0"},"source":["2. In the following cell code, we import the PyRosetta modules that we are going to use"]},{"cell_type":"code","metadata":{"id":"e02ruAYR076e","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446416768,"user_tz":-120,"elapsed":446,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"7b39a66c-d304-40d6-c929-a92dd250a5f4"},"source":["#Here we import the PyRosetta modules that we are going to be using\n","%%time\n","import pandas as pd\n","from pyrosetta import *\n","from pyrosetta.rosetta.protocols.rosetta_scripts import *\n","from pyrosetta import (\n","    init, pose_from_sequence, pose_from_file, Pose, MoveMap, create_score_function, get_fa_scorefxn,\n","    MonteCarlo, TrialMover, SwitchResidueTypeSetMover, PyJobDistributor,\n",")\n"],"execution_count":4,"outputs":[{"output_type":"stream","name":"stdout","text":["CPU times: user 420 ms, sys: 50.2 ms, total: 470 ms\n","Wall time: 514 ms\n"]}]},{"cell_type":"markdown","metadata":{"id":"4vFqicSeD-NQ"},"source":["3. Next, we performing the Membrane Protein Modelling protocol. This is briefly described for the GYPA from _Gorilla_, whereas the modelling of GYPA for the other orthologs is just a modification of this protocol."]},{"cell_type":"markdown","metadata":{"id":"wlmDdcxmXZ9T"},"source":["#### Gorilla ü¶ç"]},{"cell_type":"markdown","metadata":{"id":"cQc4_jK8Oug2"},"source":["1. In the following cell code we give PyRosetta all the needed **Flags**. These flags are equivalent to command-line instructions in order to set the path for files and set the value of different (and numerous) options. We pass them into PyRosetta, thus initializing it\n","\n","\n","```\n","pyrosetta.init(flags)\n","```\n","\n"]},{"cell_type":"code","metadata":{"id":"vG9dJLT4bZkk","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446425051,"user_tz":-120,"elapsed":388,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"d2878738-3235-40ec-e1b0-ea3aee5c68e0"},"source":["#Flags are equivalent to command-line instructions in order to set the path for files and set the value of different options\n","flags = f\"\"\"\n","-no_fconfig\n","-in:file:fasta ./input_files/fasta/gorillaGYPA.fasta\n","-out:prefix GYPA_gorilla_\n","-out:path:all ./output_files\n","-in:file:spanfile ./input_files/octopus/gorilla.span\n","-membrane:no_interpolate_Mpair\n","-membrane:Menv_penalties\n","-rg_reweight .1\n","-relax:fast\n","-relax:minimize_bond_angles\n","-relax:minimize_bond_lengths\n","-relax:jump_move true\n","-default_max_cycles 2\n","-relax:min_type lbfgs_armijo_nonmonotone\n","-score:weights input_files/wts/stage3_rlx_membrane.wts\n","-use_bicubic_interpolation\n","-hybridize:stage1_probability 1.0\n","-sog_upper_bound 15\n","-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n","-analytic_etable_evaluation true\n","-linmem_ig 10\n","-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n","\"\"\"\n","pyrosetta.init(flags)"],"execution_count":5,"outputs":[{"output_type":"stream","name":"stdout","text":["‚îå‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îê\n","‚îÇ                                 PyRosetta-4                                  ‚îÇ\n","‚îÇ              Created in JHU by Sergey Lyskov and PyRosetta Team              ‚îÇ\n","‚îÇ              (C) Copyright Rosetta Commons Member Institutions               ‚îÇ\n","‚îÇ                                                                              ‚îÇ\n","‚îÇ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE ‚îÇ\n","‚îÇ         See LICENSE.PyRosetta.md or email license@uw.edu for details         ‚îÇ\n","‚îî‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îò\n","PyRosetta-4 2024 [Rosetta PyRosetta4.MinSizeRel.python310.ubuntu 2024.42+release.3366cf78a3df04339d1982e94531b77b098ddb99 2024-10-11T08:24:04] retrieved from: http://www.pyrosetta.org\n","core.init: Rosetta version: PyRosetta4.MinSizeRel.python310.ubuntu r388 2024.42+release.3366cf78a3 3366cf78a3df04339d1982e94531b77b098ddb99 http://www.pyrosetta.org 2024-10-11T08:24:04\n","core.init: Rosetta extras: []\n","core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/gorillaGYPA.fasta -out:prefix GYPA_gorilla_ -out:path:all ./output_files -in:file:spanfile ./input_files/octopus/gorilla.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /usr/local/lib/python3.10/dist-packages/pyrosetta/database\n","basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=1709332272 seed_offset=0 real_seed=1709332272\n","basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=1709332272 RG_type=mt19937\n"]}]},{"cell_type":"markdown","metadata":{"id":"3EwFD1YdO-xV"},"source":["2. One of the crucial concepts in PyRosetta are **poses**. They contain the information, sequence or structure, to which the protocol will be applied.\n","\n","  With this in mind, here we create a **pose object** from a sequence, which in this case is the sequence of the GYPA from **_Gorilla_**. Also, we specify a **res_type** which corresponds to the **fullatom_standard**."]},{"cell_type":"code","metadata":{"id":"MQRxSQh4bZku","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446428980,"user_tz":-120,"elapsed":1337,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"8b9bb3aa-f0fb-4bca-ad0a-4b2333f469e2"},"source":["#Here we create a pose object from a sequence\n","#Also, we specify a res_type which corresponds to the fullatom_standard\n","%%time\n","pose = pose_from_sequence('DNGEWIQLVPRFSGPEITLIIFGVMAGIIGTILLISYSIRRL', 'fa_standard')"],"execution_count":6,"outputs":[{"output_type":"stream","name":"stdout","text":["CPU times: user 953 ms, sys: 73.8 ms, total: 1.03 s\n","Wall time: 1.05 s\n"]}]},{"cell_type":"markdown","metadata":{"id":"40X9c4WjPXeq"},"source":["3. While there are multiple ways of creating a protocol in Rosetta, in our case we are going to create a protocol from a **XML object**.\n","\n","**NOTE:** In the **MOVERS** tag we specify the **fragment files** and the **threaded_pdb**!!"]},{"cell_type":"code","metadata":{"id":"pLfWzrs0bZky","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446445544,"user_tz":-120,"elapsed":14473,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"eee440f7-d698-4a00-86de-cd2dbcd40470"},"source":["#Creating a Rosetta protocol from a XML object\n","#Note that in the <MOVERS> we specify the fragment files and the threaded_pdb!!\n","\n","%%time\n","xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n","<ROSETTASCRIPTS>\n","    <TASKOPERATIONS>\n","    </TASKOPERATIONS>\n","    <RESIDUE_SELECTORS>\n","    </RESIDUE_SELECTORS>\n","    <SCOREFXNS>\n","      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n","        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t</ScoreFunction>\n","    </SCOREFXNS>\n","    <FILTERS>\n","    </FILTERS>\n","    <MOVERS>\n","        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n","       \t\t\t<Fragments three_mers=\"/content/input_files/frags/gorillaGYPA.9mers\" nine_mers=\"/content/input_files/frags/gorillaGYPA.9mers\" />\n","\t          \t<Template pdb=\"/content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n","\t       </Hybridize>\n","    </MOVERS>\n","    <APPLY_TO_POSE>\n","    </APPLY_TO_POSE>\n","    <PROTOCOLS>\n","      Add mover=\"add_memb\"\n","\t    <Add mover=\"hybridize\"/>\n","    </PROTOCOLS>\n","<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","    \"\"\").get_mover(\"ParsedProtocol\")"],"execution_count":7,"outputs":[{"output_type":"stream","name":"stdout","text":["protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n","protocols.rosetta_scripts.RosettaScriptsSchemaValidator: Generating XML Schema for rosetta_scripts...\n","protocols.rosetta_scripts.RosettaScriptsSchemaValidator: ...done\n","protocols.rosetta_scripts.RosettaScriptsSchemaValidator: Initializing schema validator...\n","protocols.rosetta_scripts.RosettaScriptsSchemaValidator: ...done\n","protocols.rosetta_scripts.RosettaScriptsParser: ...done\n","protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n","<ROSETTASCRIPTS>\n","\t<TASKOPERATIONS/>\n","\t<RESIDUE_SELECTORS/>\n","\t<SCOREFXNS>\n","\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n","\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t</SCOREFXNS>\n","\t<FILTERS/>\n","\t<MOVERS>\n","\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n","\t\t\t<Fragments nine_mers=\"/content/input_files/frags/gorillaGYPA.9mers\" three_mers=\"/content/input_files/frags/gorillaGYPA.9mers\"/>\n","\t\t\t<Template cst_file=\"AUTO\" pdb=\"/content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\" weight=\"1.000\"/>\n","\t\t</Hybridize>\n","\t</MOVERS>\n","\t<APPLY_TO_POSE/>\n","\t<PROTOCOLS>\n","\t\t<Add mover=\"hybridize\"/>\n","\t</PROTOCOLS>\n","\t<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.scoring.etable: Starting energy table calculation\n","core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n","core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n","core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n","core.scoring.etable: Finished calculating energy tables.\n","core.scoring.P_AA: shapovalov_lib::shap_p_aa_pp_smooth_level of 1( aka low_smooth ) got activated.\n","core.scoring.ramachandran: shapovalov_lib::shap_rama_smooth_level of 4( aka highest_smooth ) got activated.\n","core.energy_methods.CartesianBondedEnergy: Initializing IdealParametersDatabase with default Ks=300 , 80 , 80 , 10 , 80\n","core.energy_methods.CartesianBondedEnergy: Read 759 bb-independent lengths.\n","core.energy_methods.CartesianBondedEnergy: Read 1434 bb-independent angles.\n","core.energy_methods.CartesianBondedEnergy: Read 1 bb-independent torsions.\n","core.energy_methods.CartesianBondedEnergy: Read 529 bb-independent improper tors.\n","core.scoring.etable: Starting energy table calculation\n","core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n","core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n","core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n","core.scoring.etable: Finished calculating energy tables.\n","core.scoring.etable: Using alternate parameters: LK_DGFREE in MembEtable construction.\n","core.scoring.etable: Using alternate parameters: MEMB_LK_DGFREE in MembEtable construction.\n","core.scoring.etable: Using alternate parameters: LK_DGREFCE in MembEtable construction.\n","core.scoring.etable: Using alternate parameters: MEMB_LK_DGREFCE in MembEtable construction.\n","core.scoring.etable: Starting membrane specific energy table calculation\n","core.scoring.etable: Starting energy table calculation\n","core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n","core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n","core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n","core.scoring.etable: Finished calculating energy tables.\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.io.fragments: reading fragments from file: /content/input_files/frags/gorillaGYPA.9mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file /content/input_files/frags/gorillaGYPA.9mers\n","core.import_pose.import_pose: File '/content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb' automatically determined to be of type PDB\n","protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n","protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n","protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n","CPU times: user 13.8 s, sys: 198 ms, total: 14 s\n","Wall time: 14.2 s\n"]}]},{"cell_type":"markdown","metadata":{"id":"Dn9Gu8TkjmVU"},"source":["4. Once our protocol is set up, in the following code cells:\n","  * We create a membrane score function\n","  * Then, we call the PyJobDistrubutor and pass three arguments\n","    * A name prefix to output our models as PDB files\n","    * The number of models to generate\n","    * The score function to employ\n","  * We pass our pose to the job distributor that we just created\n","  * Then, we create a Pandas dataframe to store the score results\n","  * Finally, we loop while the jobs are not completed"]},{"cell_type":"code","metadata":{"id":"flBA8JKLbZk6","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446841555,"user_tz":-120,"elapsed":128094,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"328d0d2d-a25c-4fce-82ad-26a26a7c1f79"},"source":["#Here we create a membrane score function\n","membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n","#Here we create a job distributor object and specify the prefix, number of output models or poses and the scorefunction\n","jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_gorilla_\",nstruct=2,scorefxn=membrane)\n","#Here we pass our previously created pose to the Job Distributor\n","jd.native_pose = pose\n","#Here we create a panda dataframe in order to store our score results as we are going to see later\n","gdf = pd.DataFrame()\n","#Here we loop until all n jobs are completed\n","while not jd.job_complete:\n","    test_pose = pose.clone()\n","    xml.apply(test_pose)\n","    test_gdf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n","    gdf = pd.concat([gdf, test_gdf])\n","    jd.output_decoy(test_pose)"],"execution_count":11,"outputs":[{"output_type":"stream","name":"stdout","text":["Working on decoy: GYPA_gorilla__0.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 9\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n","core.fragment: compute strand/loop fractions for 42 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLEEEEELLLLLLLEEEEEHHHHHHHHHHHHHHHHHHHHL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 30 1  EDGE 30 1 -1  EDGE 30 42 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 7 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.71484 0\n","protocols.moves.TrialCounter:      RandomMover trials=      7;  accepts= 1.0000;  energy_drop/trial=  -0.46829\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.76685 0.460805\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4264;  energy_drop/trial=  -0.00543\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 11.1634 0.658309\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   1.98716\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4850;  energy_drop/trial=   0.00628\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.93511 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4679;  energy_drop/trial=   0.00831\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.12829 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      7;  accepts= 0.2857;  energy_drop/trial=   0.00550\n","protocols.moves.TrialCounter: WeightedFragment trials=    993;  accepts= 0.4300;  energy_drop/trial=   0.00169\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.27019 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.4438;  energy_drop/trial=   0.00689\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.17212 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.5000;  energy_drop/trial=  -0.86270\n","protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.4598;  energy_drop/trial=   0.00698\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.39685 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4453;  energy_drop/trial=   0.00199\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.36715 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   0.64641\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4319;  energy_drop/trial=   0.00067\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.66346 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4885;  energy_drop/trial=   0.00928\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.84156 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4334;  energy_drop/trial=   0.00944\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.6864 -0.590209\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4835;  energy_drop/trial=   0.00929\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -8.03154 -8.03154\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.0505;  energy_drop/trial=  -0.04043\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -3.09008 -8.90098\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.1313;  energy_drop/trial=   0.01780\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -5.70575 -5.70575\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.1256;  energy_drop/trial=  -0.00233\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -35.6902 -36.3705\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     65;  accepts= 0.0769;  energy_drop/trial=  -0.01958\n","protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.0667;  energy_drop/trial=  -0.06573\n","protocols.moves.TrialCounter:      picker_rand trials=     20;  accepts= 0.1500;  energy_drop/trial=  -0.29622\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -34.4664 -41.9672\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    137;  accepts= 0.0365;  energy_drop/trial=  -0.00929\n","protocols.moves.TrialCounter:           picker trials=     26;  accepts= 0.1923;  energy_drop/trial=   0.15913\n","protocols.moves.TrialCounter:      picker_rand trials=     37;  accepts= 0.1892;  energy_drop/trial=  -0.26552\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -40.2409 -40.2409\n","protocols.moves.TrialCounter:         frag_15+ trials=     81 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     13;  accepts= 0.0769;  energy_drop/trial=  -0.23654\n","protocols.moves.TrialCounter:      picker_rand trials=      6;  accepts= 0.5000;  energy_drop/trial=  -0.16016\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -46.4511 -46.4511\n","protocols.moves.TrialCounter:         frag_15+ trials=    154 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     27;  accepts= 0.0741;  energy_drop/trial=  -0.25370\n","protocols.moves.TrialCounter:      picker_rand trials=     19;  accepts= 0.2105;  energy_drop/trial=  -0.17875\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -46.2425 -48.2827\n","protocols.moves.TrialCounter:         frag_15+ trials=     64 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     16;  accepts= 0.0625;  energy_drop/trial=   0.12751\n","protocols.moves.TrialCounter:      picker_rand trials=     20;  accepts= 0.1000;  energy_drop/trial=  -0.23234\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -40.9021 -48.2827\n","protocols.moves.TrialCounter:         frag_15+ trials=    135 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     29;  accepts= 0.2069;  energy_drop/trial=   0.35234\n","protocols.moves.TrialCounter:      picker_rand trials=     36;  accepts= 0.1389;  energy_drop/trial=  -0.20789\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -38.1512 -39.7262\n","protocols.moves.TrialCounter:           picker trials=     49;  accepts= 0.2041;  energy_drop/trial=  -0.19510\n","protocols.moves.TrialCounter:      picker_rand trials=     51;  accepts= 0.2745;  energy_drop/trial=   0.04567\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -35.1882 Finite Diff= 7.30928e+08\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.relax.FastRelax: CMD: repeat  13165.1  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -59.0328  0.270182  0.270182  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -48.0859  0.174908  0.174908  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -42.7272  0.222856  0.222856  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -32.8556  0.383469  0.383469  0.44\n","protocols.relax.FastRelax: MRP: 0  -32.8556  -32.8556  0.383469  0.383469\n","protocols.relax.FastRelax: CMD: accept_to_best  -32.8556  0.383469  0.383469  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -32.8556  0.383469  0.383469  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -56.8523  0.108577  0.108577  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -44.2662  0.338943  0.338943  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -39.4718  0.230561  0.230561  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -31.2982  0.346961  0.346961  0.44\n","protocols.relax.FastRelax: MRP: 1  -31.2982  -32.8556  0.383469  0.383469\n","protocols.relax.FastRelax: CMD: accept_to_best  -31.2982  0.346961  0.346961  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -31.2982  0.346961  0.346961  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -59.1072  0.163209  0.163209  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -49.0683  0.276118  0.276118  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -44.0696  0.218758  0.218758  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -33.3614  0.381631  0.381631  0.44\n","protocols.relax.FastRelax: MRP: 2  -33.3614  -33.3614  0.381631  0.381631\n","protocols.relax.FastRelax: CMD: accept_to_best  -33.3614  0.381631  0.381631  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -33.3614  0.381631  0.381631  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -62.2853  0.238672  0.238672  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -50.1085  0.186247  0.186247  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -41.962  0.250965  0.250965  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.349  0.342066  0.342066  0.44\n","protocols.relax.FastRelax: MRP: 3  -34.349  -34.349  0.342066  0.342066\n","protocols.relax.FastRelax: CMD: accept_to_best  -34.349  0.342066  0.342066  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -34.349  0.342066  0.342066  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -59.7085  0.144945  0.144945  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -48.465  0.27613  0.27613  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -42.6913  0.233424  0.233424  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.7615  0.340362  0.340362  0.44\n","protocols.relax.FastRelax: MRP: 4  -34.7615  -34.7615  0.340362  0.340362\n","protocols.relax.FastRelax: CMD: accept_to_best  -34.7615  0.340362  0.340362  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -34.7615  0.340362  0.340362  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n","Working on decoy: GYPA_gorilla__1.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 9\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.gorilla.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGIIGTILLISYSIRRL\n","core.fragment: compute strand/loop fractions for 42 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLEEEEELLLLLLLEEEEEHHHHHHHHHHHHHHHHHHHHL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 28 1  EDGE 28 1 -1  EDGE 28 42 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 5 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n","protocols.moves.TrialCounter:      RandomMover trials=      5;  accepts= 0.8000;  energy_drop/trial=  -0.79857\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.46008 0.459663\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4795;  energy_drop/trial=  -0.00167\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.90391 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.2809 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.1667;  energy_drop/trial=   0.06536\n","protocols.moves.TrialCounter: WeightedFragment trials=    943;  accepts= 0.4231;  energy_drop/trial=   0.00967\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.93398 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.099 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    509;  accepts= 0.3929;  energy_drop/trial=   0.02013\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.60783 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4685;  energy_drop/trial=   0.00276\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.21794 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4259;  energy_drop/trial=   0.00137\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.17495 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4419;  energy_drop/trial=   0.00533\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.5348 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.09637\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.3992;  energy_drop/trial=   0.01100\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 3.91458 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4299;  energy_drop/trial=   0.00407\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.3137 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.03895\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4111;  energy_drop/trial=   0.01233\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.6753 -0.145181\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.00816\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4484;  energy_drop/trial=   0.00882\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.46086 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.4361 -0.145181\n","protocols.moves.TrialCounter: WeightedFragment trials=    393;  accepts= 0.5089;  energy_drop/trial=   0.03201\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.365612 -5.92911\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.0400;  energy_drop/trial=  -1.02357\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -4.08744 -6.67183\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.0500;  energy_drop/trial=   0.00920\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -0.762648 -7.58044\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.1061;  energy_drop/trial=   0.02470\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -31.3773 -42.0915\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     69;  accepts= 0.4493;  energy_drop/trial=  -0.00892\n","protocols.moves.TrialCounter:           picker trials=     16;  accepts= 0.1250;  energy_drop/trial=  -0.16608\n","protocols.moves.TrialCounter:      picker_rand trials=     15;  accepts= 0.5333;  energy_drop/trial=  -0.06472\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -39.0468 -42.0915\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    139;  accepts= 0.4388;  energy_drop/trial=  -0.04800\n","protocols.moves.TrialCounter:           picker trials=     30;  accepts= 0.1000;  energy_drop/trial=  -0.08205\n","protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.4839;  energy_drop/trial=  -0.08965\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -41.8523 -43.3873\n","protocols.moves.TrialCounter:         frag_15+ trials=     72;  accepts= 0.3194;  energy_drop/trial=   0.06739\n","protocols.moves.TrialCounter:           picker trials=     16;  accepts= 0.3125;  energy_drop/trial=  -0.67419\n","protocols.moves.TrialCounter:      picker_rand trials=     12;  accepts= 0.0833;  energy_drop/trial=   0.03881\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -44.1211 -47.4712\n","protocols.moves.TrialCounter:         frag_15+ trials=    149;  accepts= 0.3624;  energy_drop/trial=   0.04143\n","protocols.moves.TrialCounter:           picker trials=     24;  accepts= 0.2500;  energy_drop/trial=  -0.45229\n","protocols.moves.TrialCounter:      picker_rand trials=     27;  accepts= 0.1111;  energy_drop/trial=  -0.11320\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -50.5978 -54.1068\n","protocols.moves.TrialCounter:         frag_15+ trials=     76;  accepts= 0.2763;  energy_drop/trial=  -0.15374\n","protocols.moves.TrialCounter:           picker trials=     12 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     12;  accepts= 0.1667;  energy_drop/trial=   0.27589\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -53.7967 -54.1068\n","protocols.moves.TrialCounter:         frag_15+ trials=    152;  accepts= 0.2500;  energy_drop/trial=  -0.08628\n","protocols.moves.TrialCounter:           picker trials=     24;  accepts= 0.0417;  energy_drop/trial=   0.02743\n","protocols.moves.TrialCounter:      picker_rand trials=     24;  accepts= 0.1667;  energy_drop/trial=   0.03682\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -47.3427 -49.6691\n","protocols.moves.TrialCounter:           picker trials=     50;  accepts= 0.0400;  energy_drop/trial=   0.13133\n","protocols.moves.TrialCounter:      picker_rand trials=     50;  accepts= 0.1000;  energy_drop/trial=  -0.43732\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 9.53674e-09 Deriv= -112.053 Finite Diff= 2.88984e+07\n","core.optimization.LineMinimizer: Inaccurate G in step= 9.56908e-09 Deriv= -29.2972 Finite Diff= 3.63591e+08\n","core.optimization.LineMinimizer: Inaccurate G in step= 5.7871e-07 Deriv= -173.804 Finite Diff= 2.68169e+06\n","core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#17\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.relax.FastRelax: CMD: repeat  14515.5  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -53.5193  0.333289  0.333289  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -35.9834  0.261165  0.261165  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -30.3567  0.314058  0.314058  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -25.4215  0.338078  0.338078  0.44\n","protocols.relax.FastRelax: MRP: 0  -25.4215  -25.4215  0.338078  0.338078\n","protocols.relax.FastRelax: CMD: accept_to_best  -25.4215  0.338078  0.338078  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -25.4215  0.338078  0.338078  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -54.0517  0.209175  0.209175  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -36.6871  0.30638  0.30638  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -32.8824  0.265298  0.265298  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -26.0597  0.318358  0.318358  0.44\n","protocols.relax.FastRelax: MRP: 1  -26.0597  -26.0597  0.318358  0.318358\n","protocols.relax.FastRelax: CMD: accept_to_best  -26.0597  0.318358  0.318358  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -26.0597  0.318358  0.318358  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -55.3539  0.213446  0.213446  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -39.2062  0.273444  0.273444  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.1834  0.27654  0.27654  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -26.2748  0.309422  0.309422  0.44\n","protocols.relax.FastRelax: MRP: 2  -26.2748  -26.2748  0.309422  0.309422\n","protocols.relax.FastRelax: CMD: accept_to_best  -26.2748  0.309422  0.309422  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -26.2748  0.309422  0.309422  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -53.9553  0.231935  0.231935  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.5929  0.358561  0.358561  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -35.8524  0.238912  0.238912  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -25.9875  0.368852  0.368852  0.44\n","protocols.relax.FastRelax: MRP: 3  -25.9875  -26.2748  0.309422  0.309422\n","protocols.relax.FastRelax: CMD: accept_to_best  -25.9875  0.368852  0.368852  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -25.9875  0.368852  0.368852  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -55.9732  0.237099  0.237099  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -41.6224  0.225542  0.225542  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -36.688  0.279611  0.279611  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -26.8952  0.299636  0.299636  0.44\n","protocols.relax.FastRelax: MRP: 4  -26.8952  -26.8952  0.299636  0.299636\n","protocols.relax.FastRelax: CMD: accept_to_best  -26.8952  0.299636  0.299636  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -26.8952  0.299636  0.299636  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n"]}]},{"cell_type":"markdown","metadata":{"id":"Mh_oZyC4XgY6"},"source":["#### Ornitorrinchus ü•ö"]},{"cell_type":"code","metadata":{"id":"PmDCmCogbbja","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446856336,"user_tz":-120,"elapsed":281,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"f358723f-3732-43b9-b652-5ff7acd4a5ab"},"source":["flags = f\"\"\"\n","-no_fconfig\n","-in:file:fasta ./input_files/fasta/ornithorhynchusGYPA.fasta\n","-out:prefix GYPA_ornithorhynchus_\n","-in:file:spanfile ./input_files/octopus/ornithorhynchus.span\n","-membrane:no_interpolate_Mpair\n","-membrane:Menv_penalties\n","-rg_reweight .1\n","-relax:fast\n","-relax:minimize_bond_angles\n","-relax:minimize_bond_lengths\n","-relax:jump_move true\n","-default_max_cycles 2\n","-relax:min_type lbfgs_armijo_nonmonotone\n","-score:weights input_files/wts/stage3_rlx_membrane.wts\n","-use_bicubic_interpolation\n","-hybridize:stage1_probability 1.0\n","-sog_upper_bound 15\n","-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n","-analytic_etable_evaluation true\n","-linmem_ig 10\n","-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n","\"\"\"\n","pyrosetta.init(flags)"],"execution_count":12,"outputs":[{"output_type":"stream","name":"stdout","text":["‚îå‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îê\n","‚îÇ                                 PyRosetta-4                                  ‚îÇ\n","‚îÇ              Created in JHU by Sergey Lyskov and PyRosetta Team              ‚îÇ\n","‚îÇ              (C) Copyright Rosetta Commons Member Institutions               ‚îÇ\n","‚îÇ                                                                              ‚îÇ\n","‚îÇ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE ‚îÇ\n","‚îÇ         See LICENSE.PyRosetta.md or email license@uw.edu for details         ‚îÇ\n","‚îî‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îò\n","PyRosetta-4 2024 [Rosetta PyRosetta4.MinSizeRel.python310.ubuntu 2024.42+release.3366cf78a3df04339d1982e94531b77b098ddb99 2024-10-11T08:24:04] retrieved from: http://www.pyrosetta.org\n","core.init: Rosetta version: PyRosetta4.MinSizeRel.python310.ubuntu r388 2024.42+release.3366cf78a3 3366cf78a3df04339d1982e94531b77b098ddb99 http://www.pyrosetta.org 2024-10-11T08:24:04\n","core.init: Rosetta extras: []\n","core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/ornithorhynchusGYPA.fasta -out:prefix GYPA_ornithorhynchus_ -in:file:spanfile ./input_files/octopus/ornithorhynchus.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /usr/local/lib/python3.10/dist-packages/pyrosetta/database\n","basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=1082027819 seed_offset=0 real_seed=1082027819\n","basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=1082027819 RG_type=mt19937\n"]}]},{"cell_type":"code","metadata":{"id":"NxxxpSsKbbjd","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446858647,"user_tz":-120,"elapsed":261,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"16dfd02c-f0c3-4867-9171-2852e02c7533"},"source":["%%time\n","pose = pose_from_sequence('IDPEIEHIFTGPVAAVIIYAVVCGVIGTILFIALVIKVV', 'fa_standard')"],"execution_count":13,"outputs":[{"output_type":"stream","name":"stdout","text":["CPU times: user 12.2 ms, sys: 0 ns, total: 12.2 ms\n","Wall time: 14.4 ms\n"]}]},{"cell_type":"code","metadata":{"id":"XKkrl3q8bbjf","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446860380,"user_tz":-120,"elapsed":552,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"b34f41bb-eec8-4ef8-c53b-9b4aeed4f36c"},"source":["%%time\n","xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n","<ROSETTASCRIPTS>\n","    <TASKOPERATIONS>\n","    </TASKOPERATIONS>\n","    <RESIDUE_SELECTORS>\n","    </RESIDUE_SELECTORS>\n","    <SCOREFXNS>\n","      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n","        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t</ScoreFunction>\n","    </SCOREFXNS>\n","    <FILTERS>\n","    </FILTERS>\n","    <MOVERS>\n","          AddMembraneMover name=\"add_memb\"\n","        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n","       \t\t\t<Fragments three_mers=\"/content/input_files/frags/ornithorhynchusGYPA.3mers\" nine_mers=\"/content/input_files/frags/ornithorhynchusGYPA.9mers\" />\n","\t          \t<Template pdb=\"/content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n","\t       </Hybridize>\n","    </MOVERS>\n","    <APPLY_TO_POSE>\n","    </APPLY_TO_POSE>\n","    <PROTOCOLS>\n","      Add mover=\"add_memb\"\n","\t    <Add mover=\"hybridize\"/>\n","    </PROTOCOLS>\n","<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","    \"\"\").get_mover(\"ParsedProtocol\")"],"execution_count":14,"outputs":[{"output_type":"stream","name":"stdout","text":["protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n","protocols.rosetta_scripts.RosettaScriptsParser: ...done\n","protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n","<ROSETTASCRIPTS>\n","\t<TASKOPERATIONS/>\n","\t<RESIDUE_SELECTORS/>\n","\t<SCOREFXNS>\n","\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n","\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t</SCOREFXNS>\n","\t<FILTERS/>\n","\t<MOVERS>\n","\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n","\t\t\t<Fragments nine_mers=\"/content/input_files/frags/ornithorhynchusGYPA.9mers\" three_mers=\"/content/input_files/frags/ornithorhynchusGYPA.3mers\"/>\n","\t\t\t<Template cst_file=\"AUTO\" pdb=\"/content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\" weight=\"1.000\"/>\n","\t\t</Hybridize>\n","\t</MOVERS>\n","\t<APPLY_TO_POSE/>\n","\t<PROTOCOLS>\n","\t\t<Add mover=\"hybridize\"/>\n","\t</PROTOCOLS>\n","\t<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.io.fragments: reading fragments from file: /content/input_files/frags/ornithorhynchusGYPA.3mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 3mer fragments from file /content/input_files/frags/ornithorhynchusGYPA.3mers\n","core.io.fragments: reading fragments from file: /content/input_files/frags/ornithorhynchusGYPA.9mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file /content/input_files/frags/ornithorhynchusGYPA.9mers\n","core.import_pose.import_pose: File '/content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb' automatically determined to be of type PDB\n","protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n","protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n","protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n","CPU times: user 384 ms, sys: 14 ms, total: 398 ms\n","Wall time: 401 ms\n"]}]},{"cell_type":"code","metadata":{"id":"_a7SPqrMbbji","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446962129,"user_tz":-120,"elapsed":85266,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"7012355b-2f69-4966-b96a-03463a65fb56"},"source":["%%time\n","membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n","jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_ornitorrinchus\",nstruct=2,scorefxn=membrane)\n","jd.native_pose = pose\n","odf = pd.DataFrame()\n","while not jd.job_complete:\n","    test_pose = pose.clone()\n","    xml.apply(test_pose)\n","    test_odf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n","    odf = pd.concat([odf, test_odf])\n","    jd.output_decoy(test_pose)"],"execution_count":15,"outputs":[{"output_type":"stream","name":"stdout","text":["Working on decoy: GYPA_ornitorrinchus_1.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: Found 1 domains using template /content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\n","protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 39  cut: 0  size: 39  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeProtocol:\n","protocols.hybridization.HybridizeProtocol:\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n","core.fragment: compute strand/loop fractions for 39 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLHEELLLHHHHHHHHHHHHHHHHHHHHHHEEEEL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 39 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 40 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 40 32 1  EDGE 32 1 -1  EDGE 32 39 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99638 275\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00362 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 6 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.87046 0.227374\n","protocols.moves.TrialCounter:      RandomMover trials=      6;  accepts= 0.8333;  energy_drop/trial=  -4.66522\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.7458 6.74537\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      8;  accepts= 0.2500;  energy_drop/trial=   0.55094\n","protocols.moves.TrialCounter: WeightedFragment trials=    992;  accepts= 0.4627;  energy_drop/trial=  -0.00673\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.1493 6.68979\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   0.28154\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4569;  energy_drop/trial=   0.00418\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.02957 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.10312 6.4406\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    424;  accepts= 0.4717;  energy_drop/trial=   0.00569\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.77828 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.47635 6.4406\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    419;  accepts= 0.4773;  energy_drop/trial=   0.00247\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.79547 6.35979\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4605;  energy_drop/trial=   0.00136\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.46349 6.22489\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.6667;  energy_drop/trial=  -0.30156\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4895;  energy_drop/trial=   0.00201\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.00364 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.62778 5.03938\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.00622\n","protocols.moves.TrialCounter: WeightedFragment trials=    430;  accepts= 0.4628;  energy_drop/trial=   0.00329\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.36028 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.47311 5.03938\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    387;  accepts= 0.5168;  energy_drop/trial=   0.00629\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.02738 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.39139 5.03938\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.06672\n","protocols.moves.TrialCounter: WeightedFragment trials=    407;  accepts= 0.4889;  energy_drop/trial=   0.00512\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.07653 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.76541 5.03938\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    411;  accepts= 0.4866;  energy_drop/trial=   0.00663\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.02164 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.73363 5.03938\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    454;  accepts= 0.4405;  energy_drop/trial=   0.00593\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 14.625 11.9388\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.3000;  energy_drop/trial=  -1.00875\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.2635 9.20037\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.1457;  energy_drop/trial=  -0.02082\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 13.182 9.36312\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.2915;  energy_drop/trial=  -0.03906\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=39   prot_res=39\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -25.8857 -26.1428\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     67;  accepts= 0.0746;  energy_drop/trial=   0.08338\n","protocols.moves.TrialCounter:           picker trials=     20;  accepts= 0.5500;  energy_drop/trial=  -1.01671\n","protocols.moves.TrialCounter:      picker_rand trials=     13;  accepts= 0.3846;  energy_drop/trial=  -0.11139\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -28.119 -29.2999\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    132;  accepts= 0.0379;  energy_drop/trial=   0.04232\n","protocols.moves.TrialCounter:           picker trials=     35;  accepts= 0.3714;  energy_drop/trial=  -0.49142\n","protocols.moves.TrialCounter:      picker_rand trials=     33;  accepts= 0.3939;  energy_drop/trial=  -0.20654\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -30.3209 -30.48\n","protocols.moves.TrialCounter:         frag_15+ trials=     71 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     17;  accepts= 0.0588;  energy_drop/trial=  -0.30307\n","protocols.moves.TrialCounter:      picker_rand trials=     12;  accepts= 0.0833;  energy_drop/trial=   0.01326\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -34.1876 -34.1876\n","protocols.moves.TrialCounter:         frag_15+ trials=    137 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     32;  accepts= 0.1250;  energy_drop/trial=   0.05440\n","protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.2258;  energy_drop/trial=  -0.34195\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -39.3822 -39.3822\n","protocols.moves.TrialCounter:         frag_15+ trials=     74 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=      9 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     17;  accepts= 0.3529;  energy_drop/trial=  -0.62257\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -39.1001 -39.3822\n","protocols.moves.TrialCounter:         frag_15+ trials=    149 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     22;  accepts= 0.2273;  energy_drop/trial=   0.27114\n","protocols.moves.TrialCounter:      picker_rand trials=     29;  accepts= 0.2759;  energy_drop/trial=  -0.56092\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -27.192 -30.4287\n","protocols.moves.TrialCounter:           picker trials=     46;  accepts= 0.0870;  energy_drop/trial=  -0.05087\n","protocols.moves.TrialCounter:      picker_rand trials=     54;  accepts= 0.3333;  energy_drop/trial=  -0.13253\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 6.10352e-07 Deriv= -45.0226 Finite Diff= 5.11269e+06\n","core.optimization.LineMinimizer: Inaccurate G in step= 9.86744e-09 Deriv= -116.189 Finite Diff= 9.63877e+07\n","core.optimization.LineMinimizer: Inaccurate G in step= 2.53589e-09 Deriv= -113.63 Finite Diff= 1.73885e+09\n","core.optimization.LineMinimizer: Inaccurate G in step= 2.55365e-09 Deriv= -60.1557 Finite Diff= 2.94472e+09\n","core.optimization.LineMinimizer: Inaccurate G in step= 9.3678e-09 Deriv= -40.6947 Finite Diff= 4.12072e+07\n","core.optimization.LineMinimizer: Inaccurate G in step= 5.65832e-07 Deriv= -181.805 Finite Diff= 1.09127e+06\n","core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#24\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n","protocols.relax.FastRelax: CMD: repeat  12365.5  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -69.2062  0.136555  0.136555  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -55.2301  0.390886  0.390886  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -46.1748  0.391982  0.391982  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -33.0462  0.560875  0.560875  0.44\n","protocols.relax.FastRelax: MRP: 0  -33.0462  -33.0462  0.560875  0.560875\n","protocols.relax.FastRelax: CMD: accept_to_best  -33.0462  0.560875  0.560875  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -33.0462  0.560875  0.560875  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -68.9012  0.301628  0.301628  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -57.8096  0.344615  0.344615  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -48.5111  0.42119  0.42119  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.928  0.548477  0.548477  0.44\n","protocols.relax.FastRelax: MRP: 1  -34.928  -34.928  0.548477  0.548477\n","protocols.relax.FastRelax: CMD: accept_to_best  -34.928  0.548477  0.548477  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -34.928  0.548477  0.548477  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -71.0108  0.33093  0.33093  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -58.8336  0.309458  0.309458  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -48.3928  0.481077  0.481077  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -36.074  0.487369  0.487369  0.44\n","protocols.relax.FastRelax: MRP: 2  -36.074  -36.074  0.487369  0.487369\n","protocols.relax.FastRelax: CMD: accept_to_best  -36.074  0.487369  0.487369  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -36.074  0.487369  0.487369  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -68.9621  0.2079  0.2079  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -56.157  0.426079  0.426079  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -48.5146  0.381462  0.381462  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -33.5125  0.59117  0.59117  0.44\n","protocols.relax.FastRelax: MRP: 3  -33.5125  -36.074  0.487369  0.487369\n","protocols.relax.FastRelax: CMD: accept_to_best  -33.5125  0.59117  0.59117  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -33.5125  0.59117  0.59117  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -67.9666  0.271476  0.271476  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -58.0174  0.36935  0.36935  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -49.8091  0.409624  0.409624  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -36.4778  0.540873  0.540873  0.44\n","protocols.relax.FastRelax: MRP: 4  -36.4778  -36.4778  0.540873  0.540873\n","protocols.relax.FastRelax: CMD: accept_to_best  -36.4778  0.540873  0.540873  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -36.4778  0.540873  0.540873  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n","Working on decoy: GYPA_ornitorrinchus_0.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.ornithorhynchus.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PVAAVIIYAVVCGVIGTILFIALVIKVV\n","core.fragment: compute strand/loop fractions for 39 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLHEELLLHHHHHHHHHHHHHHHHHHHHHHEEEEL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 14  stop: 37  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 39 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 40 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 40 24 1  EDGE 24 1 -1  EDGE 24 39 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99638 275\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00362 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 5 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n","protocols.moves.TrialCounter:      RandomMover trials=      5;  accepts= 0.8000;  energy_drop/trial=  -5.97236\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.74538 6.74537\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      7;  accepts= 0.1429;  energy_drop/trial=   0.20392\n","protocols.moves.TrialCounter: WeightedFragment trials=    993;  accepts= 0.4753;  energy_drop/trial=  -0.00144\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.3162 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4689;  energy_drop/trial=   0.00003\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.39682 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.01270\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4654;  energy_drop/trial=   0.00141\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.45259 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.6667;  energy_drop/trial=   0.42140\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4754;  energy_drop/trial=   0.00116\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.08708 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4635;  energy_drop/trial=   0.00206\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 9.00618 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.4815;  energy_drop/trial=   0.00298\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.23289 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.43245 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    750;  accepts= 0.5333;  energy_drop/trial=   0.00187\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.96713 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -1.27042\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.4875;  energy_drop/trial=   0.00677\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.30172 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=  -0.00078\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4633;  energy_drop/trial=   0.00128\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 1.21474 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.5766 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    390;  accepts= 0.5128;  energy_drop/trial=   0.00653\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.88401 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.51307 6.03165\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    410;  accepts= 0.4878;  energy_drop/trial=   0.00361\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 14.1083 8.43666\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.1508;  energy_drop/trial=  -0.03283\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 10.2178 7.66966\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1450;  energy_drop/trial=  -0.02769\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 15.1919 7.33895\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.2261;  energy_drop/trial=   0.02805\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=39   prot_res=39\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -27.0022 -27.0022\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     66;  accepts= 0.3182;  energy_drop/trial=  -0.14000\n","protocols.moves.TrialCounter:           picker trials=     16 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.1667;  energy_drop/trial=  -0.04244\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -22.8966 -27.5083\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    136;  accepts= 0.3015;  energy_drop/trial=  -0.06082\n","protocols.moves.TrialCounter:           picker trials=     35 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     29;  accepts= 0.1724;  energy_drop/trial=   0.08183\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -20.3908 -26.7602\n","protocols.moves.TrialCounter:         frag_15+ trials=     77;  accepts= 0.1299;  energy_drop/trial=  -0.01245\n","protocols.moves.TrialCounter:           picker trials=     13;  accepts= 0.1538;  energy_drop/trial=  -0.11765\n","protocols.moves.TrialCounter:      picker_rand trials=     10;  accepts= 0.2000;  energy_drop/trial=   0.21344\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -16.2026 -26.7602\n","protocols.moves.TrialCounter:         frag_15+ trials=    155;  accepts= 0.2774;  energy_drop/trial=   0.08829\n","protocols.moves.TrialCounter:           picker trials=     23;  accepts= 0.1739;  energy_drop/trial=  -0.21754\n","protocols.moves.TrialCounter:      picker_rand trials=     22;  accepts= 0.4091;  energy_drop/trial=  -0.22031\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -18.984 -27.5779\n","protocols.moves.TrialCounter:         frag_15+ trials=     76;  accepts= 0.3026;  energy_drop/trial=   0.01675\n","protocols.moves.TrialCounter:           picker trials=     14;  accepts= 0.0714;  energy_drop/trial=   0.36012\n","protocols.moves.TrialCounter:      picker_rand trials=     10;  accepts= 0.1000;  energy_drop/trial=   0.22790\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -20.5518 -27.5779\n","protocols.moves.TrialCounter:         frag_15+ trials=    157;  accepts= 0.1465;  energy_drop/trial=   0.00811\n","protocols.moves.TrialCounter:           picker trials=     19;  accepts= 0.0526;  energy_drop/trial=   0.26535\n","protocols.moves.TrialCounter:      picker_rand trials=     24;  accepts= 0.1667;  energy_drop/trial=   0.02963\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -23.1348 -28.6403\n","protocols.moves.TrialCounter:           picker trials=     55;  accepts= 0.1273;  energy_drop/trial=   0.13435\n","protocols.moves.TrialCounter:      picker_rand trials=     45;  accepts= 0.2000;  energy_drop/trial=  -0.48050\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 9.53674e-09 Deriv= -45.3176 Finite Diff= 1.07532e+08\n","core.optimization.LineMinimizer: Inaccurate G in step= 2.92106e-09 Deriv= -13.867 Finite Diff= 1.35478e+09\n","core.optimization.LineMinimizer: Inaccurate G in step= 2.72097e-09 Deriv= -8.30305 Finite Diff= 1.60926e+09\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (39)\n","protocols.relax.FastRelax: CMD: repeat  9817.06  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -60.2128  0.206215  0.206215  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -49.1752  0.310401  0.310401  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -41.6393  0.328943  0.328943  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -29.9525  0.375841  0.375841  0.44\n","protocols.relax.FastRelax: MRP: 0  -29.9525  -29.9525  0.375841  0.375841\n","protocols.relax.FastRelax: CMD: accept_to_best  -29.9525  0.375841  0.375841  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -29.9525  0.375841  0.375841  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -63.7162  0.183127  0.183127  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -51.6778  0.292899  0.292899  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -42.9438  0.326095  0.326095  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -30.2625  0.437308  0.437308  0.44\n","protocols.relax.FastRelax: MRP: 1  -30.2625  -30.2625  0.437308  0.437308\n","protocols.relax.FastRelax: CMD: accept_to_best  -30.2625  0.437308  0.437308  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -30.2625  0.437308  0.437308  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -63.1111  0.220293  0.220293  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -52.6669  0.247259  0.247259  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -42.9747  0.352039  0.352039  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -31.8202  0.381807  0.381807  0.44\n","protocols.relax.FastRelax: MRP: 2  -31.8202  -31.8202  0.381807  0.381807\n","protocols.relax.FastRelax: CMD: accept_to_best  -31.8202  0.381807  0.381807  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -31.8202  0.381807  0.381807  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -62.7972  0.193832  0.193832  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -52.2644  0.276  0.276  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -43.569  0.317305  0.317305  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -31.5763  0.41481  0.41481  0.44\n","protocols.relax.FastRelax: MRP: 3  -31.5763  -31.8202  0.381807  0.381807\n","protocols.relax.FastRelax: CMD: accept_to_best  -31.5763  0.41481  0.41481  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -31.5763  0.41481  0.41481  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -64.1293  0.210306  0.210306  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -53.4489  0.239053  0.239053  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -43.9491  0.329201  0.329201  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -32.4721  0.352466  0.352466  0.44\n","protocols.relax.FastRelax: MRP: 4  -32.4721  -32.4721  0.352466  0.352466\n","protocols.relax.FastRelax: CMD: accept_to_best  -32.4721  0.352466  0.352466  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -32.4721  0.352466  0.352466  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n","CPU times: user 1min 19s, sys: 628 ms, total: 1min 20s\n","Wall time: 1min 25s\n"]}]},{"cell_type":"markdown","metadata":{"id":"feibleVGXjZX"},"source":["####Marmota ü¶°"]},{"cell_type":"code","metadata":{"id":"sh2p_Qq0bee4","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446962129,"user_tz":-120,"elapsed":15,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"203d682a-0ce9-465a-e827-1e9b6755ff8c"},"source":["%%time\n","flags = f\"\"\"\n","-no_fconfig\n","-in:file:fasta ./input_files/fasta/marmotaGYPA.fasta\n","-out:prefix GYPA_marmota_\n","-out:path:all ./output_files\n","-in:file:spanfile ./input_files/octopus/marmota.span\n","-membrane:no_interpolate_Mpair\n","-membrane:Menv_penalties\n","-rg_reweight .1\n","-relax:fast\n","-relax:minimize_bond_angles\n","-relax:minimize_bond_lengths\n","-relax:jump_move true\n","-default_max_cycles 2\n","-relax:min_type lbfgs_armijo_nonmonotone\n","-score:weights input_files/wts/stage3_rlx_membrane.wts\n","-use_bicubic_interpolation\n","-hybridize:stage1_probability 1.0\n","-sog_upper_bound 15\n","-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n","-analytic_etable_evaluation true\n","-linmem_ig 10\n","-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n","\"\"\"\n","pyrosetta.init(flags)"],"execution_count":16,"outputs":[{"output_type":"stream","name":"stdout","text":["‚îå‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îê\n","‚îÇ                                 PyRosetta-4                                  ‚îÇ\n","‚îÇ              Created in JHU by Sergey Lyskov and PyRosetta Team              ‚îÇ\n","‚îÇ              (C) Copyright Rosetta Commons Member Institutions               ‚îÇ\n","‚îÇ                                                                              ‚îÇ\n","‚îÇ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE ‚îÇ\n","‚îÇ         See LICENSE.PyRosetta.md or email license@uw.edu for details         ‚îÇ\n","‚îî‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îò\n","PyRosetta-4 2024 [Rosetta PyRosetta4.MinSizeRel.python310.ubuntu 2024.42+release.3366cf78a3df04339d1982e94531b77b098ddb99 2024-10-11T08:24:04] retrieved from: http://www.pyrosetta.org\n","core.init: Rosetta version: PyRosetta4.MinSizeRel.python310.ubuntu r388 2024.42+release.3366cf78a3 3366cf78a3df04339d1982e94531b77b098ddb99 http://www.pyrosetta.org 2024-10-11T08:24:04\n","core.init: Rosetta extras: []\n","core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/marmotaGYPA.fasta -out:prefix GYPA_marmota_ -out:path:all ./output_files -in:file:spanfile ./input_files/octopus/marmota.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /usr/local/lib/python3.10/dist-packages/pyrosetta/database\n","basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=1124140540 seed_offset=0 real_seed=1124140540\n","basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=1124140540 RG_type=mt19937\n","CPU times: user 39.9 ms, sys: 0 ns, total: 39.9 ms\n","Wall time: 41.4 ms\n"]}]},{"cell_type":"code","metadata":{"id":"GahGjthKbee7","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446962129,"user_tz":-120,"elapsed":5,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"8f9d912b-6fbe-44ca-8969-e2b7212a0d6e"},"source":["%%time\n","pose = pose_from_sequence('KQGRNQIIHPFSEPVIILIIFAVMFGIIGTILLISFCVRRL', 'fa_standard')"],"execution_count":17,"outputs":[{"output_type":"stream","name":"stdout","text":["CPU times: user 9.1 ms, sys: 0 ns, total: 9.1 ms\n","Wall time: 9.41 ms\n"]}]},{"cell_type":"code","metadata":{"id":"BhLwUqtBbee9","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729446962515,"user_tz":-120,"elapsed":389,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"d8118e59-199b-46dc-d5d6-4c1b039fdb37"},"source":["%%time\n","xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n","<ROSETTASCRIPTS>\n","    <TASKOPERATIONS>\n","    </TASKOPERATIONS>\n","    <RESIDUE_SELECTORS>\n","    </RESIDUE_SELECTORS>\n","    <SCOREFXNS>\n","      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n","        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t</ScoreFunction>\n","    </SCOREFXNS>\n","    <FILTERS>\n","    </FILTERS>\n","    <MOVERS>\n","          AddMembraneMover name=\"add_memb\"\n","        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n","       \t\t\t<Fragments three_mers=\"./input_files/frags/marmotaGYPA.3mers\" nine_mers=\"./input_files/frags/marmotaGYPA.9mers\" />\n","\t          \t<Template pdb=\"./input_files/threaded_pdbs/5eh6_A.marmota.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n","\t       </Hybridize>\n","    </MOVERS>\n","    <APPLY_TO_POSE>\n","    </APPLY_TO_POSE>\n","    <PROTOCOLS>\n","      Add mover=\"add_memb\"\n","\t    <Add mover=\"hybridize\"/>\n","    </PROTOCOLS>\n","<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","    \"\"\").get_mover(\"ParsedProtocol\")"],"execution_count":18,"outputs":[{"output_type":"stream","name":"stdout","text":["protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n","protocols.rosetta_scripts.RosettaScriptsParser: ...done\n","protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n","<ROSETTASCRIPTS>\n","\t<TASKOPERATIONS/>\n","\t<RESIDUE_SELECTORS/>\n","\t<SCOREFXNS>\n","\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n","\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t</SCOREFXNS>\n","\t<FILTERS/>\n","\t<MOVERS>\n","\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n","\t\t\t<Fragments nine_mers=\"./input_files/frags/marmotaGYPA.9mers\" three_mers=\"./input_files/frags/marmotaGYPA.3mers\"/>\n","\t\t\t<Template cst_file=\"AUTO\" pdb=\"./input_files/threaded_pdbs/5eh6_A.marmota.pdb\" weight=\"1.000\"/>\n","\t\t</Hybridize>\n","\t</MOVERS>\n","\t<APPLY_TO_POSE/>\n","\t<PROTOCOLS>\n","\t\t<Add mover=\"hybridize\"/>\n","\t</PROTOCOLS>\n","\t<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.io.fragments: reading fragments from file: ./input_files/frags/marmotaGYPA.3mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 3mer fragments from file ./input_files/frags/marmotaGYPA.3mers\n","core.io.fragments: reading fragments from file: ./input_files/frags/marmotaGYPA.9mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file ./input_files/frags/marmotaGYPA.9mers\n","core.import_pose.import_pose: File './input_files/threaded_pdbs/5eh6_A.marmota.pdb' automatically determined to be of type PDB\n","protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n","protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n","protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n","CPU times: user 421 ms, sys: 24 ms, total: 445 ms\n","Wall time: 453 ms\n"]}]},{"cell_type":"code","metadata":{"id":"l2l4g1onbefA","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729447090582,"user_tz":-120,"elapsed":127545,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"71d29ff3-22c6-436c-83b4-109cbc82041a"},"source":["membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n","jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_marmota_\",nstruct=2,scorefxn=membrane)\n","jd.native_pose = pose\n","mdf = pd.DataFrame()\n","while not jd.job_complete:\n","    test_pose = pose.clone()\n","    xml.apply(test_pose)\n","    test_mdf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n","    mdf = pd.concat([mdf, test_mdf])\n","    jd.output_decoy(test_pose)"],"execution_count":19,"outputs":[{"output_type":"stream","name":"stdout","text":["Working on decoy: GYPA_marmota__0.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: Found 1 domains using template ./input_files/threaded_pdbs/5eh6_A.marmota.pdb\n","protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 41  cut: 0  size: 41  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeProtocol:\n","protocols.hybridization.HybridizeProtocol:\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 ./input_files/threaded_pdbs/5eh6_A.marmota.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n","core.fragment: compute strand/loop fractions for 41 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLLLLLLLLLEEHHHHHHHHHHHHHHHHHHHHHHHHL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 41 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 42 32 1  EDGE 32 1 -1  EDGE 32 41 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99693 325\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00307 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 10 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n","protocols.moves.TrialCounter:      RandomMover trials=     10;  accepts= 0.9000;  energy_drop/trial=  -3.24724\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.514753 0.459412\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5090;  energy_drop/trial=  -0.00467\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.989289 0.039558\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5110;  energy_drop/trial=  -0.00004\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.20428 0.039558\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   0.01322\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5220;  energy_drop/trial=   0.00114\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.24099 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.4000;  energy_drop/trial=   0.80116\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5156;  energy_drop/trial=  -0.00282\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.34322 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.44748\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4824;  energy_drop/trial=   0.00050\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.397829 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=  -0.22510\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5251;  energy_drop/trial=   0.00169\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.19712 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4965;  energy_drop/trial=   0.00159\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.18258 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 4.88492 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    395;  accepts= 0.5063;  energy_drop/trial=   0.01335\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.29277 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5206;  energy_drop/trial=   0.00670\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.75832 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 3.89024 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.45752\n","protocols.moves.TrialCounter: WeightedFragment trials=    377;  accepts= 0.5279;  energy_drop/trial=   0.01014\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.07681 -0.388556\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5190;  energy_drop/trial=   0.00147\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -10.5583 -10.5583\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    198;  accepts= 0.1869;  energy_drop/trial=  -1.10236\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -3.77409 -11.2827\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.1457;  energy_drop/trial=   0.01332\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -8.44428 -11.7217\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1200;  energy_drop/trial=  -0.13024\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=41   prot_res=41\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -41.0682 -43.3071\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     78;  accepts= 0.0641;  energy_drop/trial=   0.02499\n","protocols.moves.TrialCounter:           picker trials=     11;  accepts= 0.4545;  energy_drop/trial=   0.04286\n","protocols.moves.TrialCounter:      picker_rand trials=     11;  accepts= 0.3636;  energy_drop/trial=  -0.82914\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -44.1233 -45.227\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    150;  accepts= 0.1800;  energy_drop/trial=   0.02449\n","protocols.moves.TrialCounter:           picker trials=     23;  accepts= 0.3043;  energy_drop/trial=   0.02165\n","protocols.moves.TrialCounter:      picker_rand trials=     27;  accepts= 0.2963;  energy_drop/trial=  -0.51576\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -41.8862 -41.8862\n","protocols.moves.TrialCounter:         frag_15+ trials=     71;  accepts= 0.0423;  energy_drop/trial=  -0.29060\n","protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.4000;  energy_drop/trial=  -0.00134\n","protocols.moves.TrialCounter:      picker_rand trials=     14;  accepts= 0.1429;  energy_drop/trial=  -0.04160\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -40.2136 -47.58\n","protocols.moves.TrialCounter:         frag_15+ trials=    146;  accepts= 0.1370;  energy_drop/trial=  -0.04582\n","protocols.moves.TrialCounter:           picker trials=     28;  accepts= 0.2500;  energy_drop/trial=  -0.12696\n","protocols.moves.TrialCounter:      picker_rand trials=     26;  accepts= 0.3077;  energy_drop/trial=  -0.35837\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -44.414 -48.6799\n","protocols.moves.TrialCounter:         frag_15+ trials=     74 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     13;  accepts= 0.2308;  energy_drop/trial=  -0.12922\n","protocols.moves.TrialCounter:      picker_rand trials=     13;  accepts= 0.4615;  energy_drop/trial=   0.31145\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -48.2452 -48.6799\n","protocols.moves.TrialCounter:         frag_15+ trials=    143;  accepts= 0.1259;  energy_drop/trial=  -0.01144\n","protocols.moves.TrialCounter:           picker trials=     29;  accepts= 0.1724;  energy_drop/trial=  -0.04913\n","protocols.moves.TrialCounter:      picker_rand trials=     28;  accepts= 0.3571;  energy_drop/trial=   0.05710\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -49.4206 -49.4206\n","protocols.moves.TrialCounter:           picker trials=     47;  accepts= 0.1702;  energy_drop/trial=   0.07342\n","protocols.moves.TrialCounter:      picker_rand trials=     53;  accepts= 0.2830;  energy_drop/trial=  -0.22442\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 9.53674e-09 Deriv= -103.366 Finite Diff= 3.02068e+08\n","core.optimization.LineMinimizer: Inaccurate G in step= 8.41017e-09 Deriv= -200.689 Finite Diff= 3.46104e+08\n","core.optimization.LineMinimizer: Inaccurate G in step= 5.38552e-07 Deriv= -200.689 Finite Diff= 7.50761e+06\n","core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#7\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n","protocols.relax.FastRelax: CMD: repeat  16428.8  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -99.6531  0.193014  0.193014  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -83.1473  0.235292  0.235292  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -76.9643  0.191996  0.191996  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -65.0507  0.321134  0.321134  0.44\n","protocols.relax.FastRelax: MRP: 0  -65.0507  -65.0507  0.321134  0.321134\n","protocols.relax.FastRelax: CMD: accept_to_best  -65.0507  0.321134  0.321134  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -65.0507  0.321134  0.321134  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -102.778  0.116073  0.116073  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -85.1074  0.205388  0.205388  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -78.7813  0.201367  0.201367  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -66.3119  0.290525  0.290525  0.44\n","protocols.relax.FastRelax: MRP: 1  -66.3119  -66.3119  0.290525  0.290525\n","protocols.relax.FastRelax: CMD: accept_to_best  -66.3119  0.290525  0.290525  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -66.3119  0.290525  0.290525  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -103.208  0.138175  0.138175  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -87.2592  0.132585  0.132585  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -76.1903  0.275941  0.275941  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -67.2376  0.220617  0.220617  0.44\n","protocols.relax.FastRelax: MRP: 2  -67.2376  -67.2376  0.220617  0.220617\n","protocols.relax.FastRelax: CMD: accept_to_best  -67.2376  0.220617  0.220617  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -67.2376  0.220617  0.220617  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -106.662  0.217038  0.217038  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -88.0345  0.162865  0.162865  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -80.1198  0.179675  0.179675  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -67.9477  0.298782  0.298782  0.44\n","protocols.relax.FastRelax: MRP: 3  -67.9477  -67.9477  0.298782  0.298782\n","protocols.relax.FastRelax: CMD: accept_to_best  -67.9477  0.298782  0.298782  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -67.9477  0.298782  0.298782  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -105.394  0.141362  0.141362  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -86.2399  0.192438  0.192438  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -79.767  0.201238  0.201238  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -67.6727  0.274023  0.274023  0.44\n","protocols.relax.FastRelax: MRP: 4  -67.6727  -67.9477  0.298782  0.298782\n","protocols.relax.FastRelax: CMD: accept_to_best  -67.6727  0.274023  0.274023  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  -67.6727  0.274023  0.274023  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n","Working on decoy: GYPA_marmota__1.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 ./input_files/threaded_pdbs/5eh6_A.marmota.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PVIILIIFAVMFGIIGTILLISFCVRRL\n","core.fragment: compute strand/loop fractions for 41 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLLLLLLLLLLEEHHHHHHHHHHHHHHHHHHHHHHHHL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 16  stop: 39  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 41 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 42 23 1  EDGE 23 1 -1  EDGE 23 41 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99693 325\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00307 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 3 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n","protocols.moves.TrialCounter:      RandomMover trials=      3;  accepts= 0.6667;  energy_drop/trial= -10.82414\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.03965 0.459412\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=  -0.06790\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5476;  energy_drop/trial=   0.00781\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.10319 0.156006\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.5447;  energy_drop/trial=   0.00001\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.27187 0.156006\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      6;  accepts= 0.3333;  energy_drop/trial=   0.16961\n","protocols.moves.TrialCounter: WeightedFragment trials=    994;  accepts= 0.5060;  energy_drop/trial=   0.00010\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.08088 -0.172074\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=   0.53991\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5135;  energy_drop/trial=  -0.00070\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.50017 -0.172074\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4975;  energy_drop/trial=   0.00872\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 4.27726 -0.172074\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5517;  energy_drop/trial=   0.00446\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.44743 -0.241451\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.05385\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5281;  energy_drop/trial=   0.00073\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.14966 -0.241451\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.55778\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.5345;  energy_drop/trial=   0.00083\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.12595 -0.648432\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      5;  accepts= 0.2000;  energy_drop/trial=   0.20487\n","protocols.moves.TrialCounter: WeightedFragment trials=    995;  accepts= 0.4894;  energy_drop/trial=   0.00034\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.10058 -0.648432\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5240;  energy_drop/trial=   0.00175\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.946414 -0.648432\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   2.55126\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.4870;  energy_drop/trial=  -0.00351\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -2.75184 -2.75184\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.0653;  energy_drop/trial=  -1.12372\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -5.43353 -8.28619\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.2513;  energy_drop/trial=  -0.01549\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -7.90196 -7.90196\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.1960;  energy_drop/trial=  -0.08157\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=41   prot_res=41\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -36.5616 -36.5616\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     68;  accepts= 0.0294;  energy_drop/trial=  -0.02000\n","protocols.moves.TrialCounter:           picker trials=     17 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     15;  accepts= 0.0667;  energy_drop/trial=  -0.03004\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -36.5616 -36.5616\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    141;  accepts= 0.0142;  energy_drop/trial=  -0.00965\n","protocols.moves.TrialCounter:           picker trials=     28 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     31;  accepts= 0.0323;  energy_drop/trial=  -0.01453\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -30.8241 -30.8241\n","protocols.moves.TrialCounter:         frag_15+ trials=     72 NO ACCEPTS.\n","protocols.moves.TrialCounter:           picker trials=     15 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     13 NO ACCEPTS.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -42.1349 -42.3177\n","protocols.moves.TrialCounter:         frag_15+ trials=    136;  accepts= 0.1691;  energy_drop/trial=  -0.07222\n","protocols.moves.TrialCounter:           picker trials=     32;  accepts= 0.0312;  energy_drop/trial=  -0.11763\n","protocols.moves.TrialCounter:      picker_rand trials=     32;  accepts= 0.0938;  energy_drop/trial=   0.07110\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -40.2897 -40.9781\n","protocols.moves.TrialCounter:         frag_15+ trials=     70;  accepts= 0.2857;  energy_drop/trial=  -0.51213\n","protocols.moves.TrialCounter:           picker trials=     17 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     13;  accepts= 0.1538;  energy_drop/trial=   0.56589\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -38.2139 -43.139\n","protocols.moves.TrialCounter:         frag_15+ trials=    142;  accepts= 0.2183;  energy_drop/trial=  -0.22733\n","protocols.moves.TrialCounter:           picker trials=     32 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     26;  accepts= 0.1538;  energy_drop/trial=   0.22555\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -23.3507 -23.9666\n","protocols.moves.TrialCounter:           picker trials=     50 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     50;  accepts= 0.2400;  energy_drop/trial=  -0.39231\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -204.944 Finite Diff= 4.09264e+09\n","core.optimization.LineMinimizer: Inaccurate G in step= 3.38224e-07 Deriv= -508.83 Finite Diff= 7.46591e+07\n","core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#17\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (41)\n","protocols.relax.FastRelax: CMD: repeat  19033.9  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  274.653  0.558143  0.558143  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  353.532  0.741664  0.741664  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  385.552  0.967348  0.967348  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  379.057  1.05767  1.05767  0.44\n","protocols.relax.FastRelax: MRP: 0  379.057  379.057  1.05767  1.05767\n","protocols.relax.FastRelax: CMD: accept_to_best  379.057  1.05767  1.05767  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  379.057  1.05767  1.05767  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  221.387  0.496782  0.496782  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  291.727  0.600173  0.600173  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  317.456  0.838503  0.838503  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  338.178  0.922163  0.922163  0.44\n","protocols.relax.FastRelax: MRP: 1  338.178  338.178  0.922163  0.922163\n","protocols.relax.FastRelax: CMD: accept_to_best  338.178  0.922163  0.922163  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  338.178  0.922163  0.922163  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  209.552  0.427968  0.427968  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  278.391  0.680931  0.680931  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  308.2  0.876828  0.876828  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  323.409  0.996711  0.996711  0.44\n","protocols.relax.FastRelax: MRP: 2  323.409  323.409  0.996711  0.996711\n","protocols.relax.FastRelax: CMD: accept_to_best  323.409  0.996711  0.996711  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  323.409  0.996711  0.996711  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  206.327  0.3801  0.3801  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  268.017  0.709045  0.709045  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  291.186  0.836984  0.836984  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  309.665  0.9771  0.9771  0.44\n","protocols.relax.FastRelax: MRP: 3  309.665  309.665  0.9771  0.9771\n","protocols.relax.FastRelax: CMD: accept_to_best  309.665  0.9771  0.9771  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  309.665  0.9771  0.9771  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  201.561  0.644652  0.644652  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  260.649  0.706766  0.706766  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  288.164  0.833394  0.833394  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  311.056  0.934555  0.934555  0.44\n","protocols.relax.FastRelax: MRP: 4  311.056  309.665  0.9771  0.9771\n","protocols.relax.FastRelax: CMD: accept_to_best  311.056  0.934555  0.934555  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  311.056  0.934555  0.934555  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n"]}]},{"cell_type":"markdown","metadata":{"id":"0ehjClu0YhXk"},"source":["####Human üßë"]},{"cell_type":"code","metadata":{"id":"slUPRFWuoOQR","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729447090583,"user_tz":-120,"elapsed":14,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"7c753f7a-e0d9-4326-dd5a-cf99d91527c3"},"source":["%%time\n","flags = f\"\"\"\n","-no_fconfig\n","-in:file:fasta ./input_files/fasta/GYPA.fasta\n","-parser:protocol ./input_files/cm.xml\n","-out:prefix GYPA_\n","-out:path:all ./output_files\n","-in:file:spanfile ./input_files/octopus/GYPA.span\n","-membrane:no_interpolate_Mpair\n","-membrane:Menv_penalties\n","-rg_reweight .1\n","-relax:fast\n","-relax:minimize_bond_angles\n","-relax:minimize_bond_lengths\n","-relax:jump_move true\n","-default_max_cycles 2\n","-relax:min_type lbfgs_armijo_nonmonotone\n","-score:weights input_files/wts/stage3_rlx_membrane.wts\n","-use_bicubic_interpolation\n","-hybridize:stage1_probability 1.0\n","-sog_upper_bound 15\n","-chemical:exclude_patches LowerDNA  UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated  tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated  lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm\n","-analytic_etable_evaluation true\n","-linmem_ig 10\n","-out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack\n","\"\"\"\n","pyrosetta.init(flags)"],"execution_count":20,"outputs":[{"output_type":"stream","name":"stdout","text":["‚îå‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îê\n","‚îÇ                                 PyRosetta-4                                  ‚îÇ\n","‚îÇ              Created in JHU by Sergey Lyskov and PyRosetta Team              ‚îÇ\n","‚îÇ              (C) Copyright Rosetta Commons Member Institutions               ‚îÇ\n","‚îÇ                                                                              ‚îÇ\n","‚îÇ NOTE: USE OF PyRosetta FOR COMMERCIAL PURPOSES REQUIRE PURCHASE OF A LICENSE ‚îÇ\n","‚îÇ         See LICENSE.PyRosetta.md or email license@uw.edu for details         ‚îÇ\n","‚îî‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îò\n","PyRosetta-4 2024 [Rosetta PyRosetta4.MinSizeRel.python310.ubuntu 2024.42+release.3366cf78a3df04339d1982e94531b77b098ddb99 2024-10-11T08:24:04] retrieved from: http://www.pyrosetta.org\n","core.init: Rosetta version: PyRosetta4.MinSizeRel.python310.ubuntu r388 2024.42+release.3366cf78a3 3366cf78a3df04339d1982e94531b77b098ddb99 http://www.pyrosetta.org 2024-10-11T08:24:04\n","core.init: Rosetta extras: []\n","core.init: command: PyRosetta -no_fconfig -in:file:fasta ./input_files/fasta/GYPA.fasta -parser:protocol ./input_files/cm.xml -out:prefix GYPA_ -out:path:all ./output_files -in:file:spanfile ./input_files/octopus/GYPA.span -membrane:no_interpolate_Mpair -membrane:Menv_penalties -rg_reweight .1 -relax:fast -relax:minimize_bond_angles -relax:minimize_bond_lengths -relax:jump_move true -default_max_cycles 2 -relax:min_type lbfgs_armijo_nonmonotone -score:weights input_files/wts/stage3_rlx_membrane.wts -use_bicubic_interpolation -hybridize:stage1_probability 1.0 -sog_upper_bound 15 -chemical:exclude_patches LowerDNA UpperDNA Cterm_amidation SpecialRotamer VirtualBB ShoveBB VirtualDNAPhosphate VirtualNTerm CTermConnect sc_orbitals pro_hydroxylated_case1 pro_hydroxylated_case2 ser_phosphorylated thr_phosphorylated tyr_phosphorylated tyr_sulfated lys_dimethylated lys_monomethylated lys_trimethylated lys_acetylated glu_carboxylated cys_acetylated tyr_diiodinated N_acetylated C_methylamidated MethylatedProteinCterm -analytic_etable_evaluation true -linmem_ig 10 -out:mute core.io.pose_from_sfr.PoseFromSFRBuilder core.scoring.MembranePotential core.chemical.GlobalResidueTypeSet basic.io.database core.scoring.MembraneTopology core.scoring.CartesianBondedEnergy core.conformation.Conformation protocols.jd2.parser.ScoreFunctionLoader core.optimization.AtomTreeMinimizer core.pack -database /usr/local/lib/python3.10/dist-packages/pyrosetta/database\n","basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=1533108389 seed_offset=0 real_seed=1533108389\n","basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=1533108389 RG_type=mt19937\n","CPU times: user 51.1 ms, sys: 10 ¬µs, total: 51.1 ms\n","Wall time: 52.8 ms\n"]}]},{"cell_type":"code","metadata":{"id":"dYwsIljin-vA","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729447090583,"user_tz":-120,"elapsed":5,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"a90ae941-c5ff-49ec-ef2c-7f5edd4a81d5"},"source":["%%time\n","pose = pose_from_sequence('ETGERVQLAHHFSEPEITLIIFGVMAGVIGTILLISYGIRRL', 'fa_standard')"],"execution_count":21,"outputs":[{"output_type":"stream","name":"stdout","text":["CPU times: user 9.16 ms, sys: 989 ¬µs, total: 10.1 ms\n","Wall time: 10 ms\n"]}]},{"cell_type":"code","metadata":{"id":"oYRxZf4M8_Un","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729447090971,"user_tz":-120,"elapsed":391,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"d0053ccb-ffa6-4d54-a7cf-b6484d94053d"},"source":["%%time\n","xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n","<ROSETTASCRIPTS>\n","    <TASKOPERATIONS>\n","    </TASKOPERATIONS>\n","    <RESIDUE_SELECTORS>\n","    </RESIDUE_SELECTORS>\n","    <SCOREFXNS>\n","      <ScoreFunction name=\"stage1\" weights=\"input_files/wts/stage1_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"stage2\" weights=\"input_files/wts/stage2_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"fullatom\" weights=\"input_files/wts/stage3_rlx_membrane.wts\" symmetric=\"0\">\n","        <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","      </ScoreFunction>\n","      <ScoreFunction name=\"membrane\" weights=\"membrane_highres_Menv_smooth\" symmetric=\"0\">\n","        <Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","        <Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t <Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t</ScoreFunction>\n","    </SCOREFXNS>\n","    <FILTERS>\n","    </FILTERS>\n","    <MOVERS>\n","          AddMembraneMover name=\"add_memb\"\n","        \t <Hybridize name=\"hybridize\" stage1_scorefxn=\"stage1\" stage2_scorefxn=\"stage2\" fa_scorefxn=\"fullatom\" batch=\"1\" stage1_increase_cycles=\"0.5\" stage2_increase_cycles=\"0.5\" linmin_only=\"1\" add_hetatm=\"0\" hetatm_to_protein_cst_weight=\"0\"  >\n","       \t\t\t<Fragments three_mers=\"./input_files/frags/GYPA.3mers\" nine_mers=\"./input_files/frags/GYPA.9mers\" />\n","\t          \t<Template pdb=\"/content/input_files/threaded_pdbs/5eh6_A.human.pdb\" cst_file=\"AUTO\" weight=\"1.000\" />\n","\t       </Hybridize>\n","    </MOVERS>\n","    <APPLY_TO_POSE>\n","    </APPLY_TO_POSE>\n","    <PROTOCOLS>\n","      Add mover=\"add_memb\"\n","\t    <Add mover=\"hybridize\"/>\n","    </PROTOCOLS>\n","<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","    \"\"\").get_mover(\"ParsedProtocol\")"],"execution_count":22,"outputs":[{"output_type":"stream","name":"stdout","text":["protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n","protocols.rosetta_scripts.RosettaScriptsParser: ...done\n","protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n","<ROSETTASCRIPTS>\n","\t<TASKOPERATIONS/>\n","\t<RESIDUE_SELECTORS/>\n","\t<SCOREFXNS>\n","\t\t<ScoreFunction name=\"stage1\" symmetric=\"0\" weights=\"input_files/wts/stage1_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"1\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"stage2\" symmetric=\"0\" weights=\"input_files/wts/stage2_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"fullatom\" symmetric=\"0\" weights=\"input_files/wts/stage3_rlx_membrane.wts\">\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t\t<ScoreFunction name=\"membrane\" symmetric=\"0\" weights=\"membrane_highres_Menv_smooth\">\n","\t\t\t<Reweight scoretype=\"cart_bonded\" weight=\"0.5\"/>\n","\t\t\t<Reweight scoretype=\"pro_close\" weight=\"0\"/>\n","\t\t\t<Reweight scoretype=\"atom_pair_constraint\" weight=\"0.5\"/>\n","\t\t</ScoreFunction>\n","\t</SCOREFXNS>\n","\t<FILTERS/>\n","\t<MOVERS>\n","\t\t<Hybridize add_hetatm=\"0\" batch=\"1\" fa_scorefxn=\"fullatom\" hetatm_to_protein_cst_weight=\"0\" linmin_only=\"1\" name=\"hybridize\" stage1_increase_cycles=\"0.5\" stage1_scorefxn=\"stage1\" stage2_increase_cycles=\"0.5\" stage2_scorefxn=\"stage2\">\n","\t\t\t<Fragments nine_mers=\"./input_files/frags/GYPA.9mers\" three_mers=\"./input_files/frags/GYPA.3mers\"/>\n","\t\t\t<Template cst_file=\"AUTO\" pdb=\"/content/input_files/threaded_pdbs/5eh6_A.human.pdb\" weight=\"1.000\"/>\n","\t\t</Hybridize>\n","\t</MOVERS>\n","\t<APPLY_TO_POSE/>\n","\t<PROTOCOLS>\n","\t\t<Add mover=\"hybridize\"/>\n","\t</PROTOCOLS>\n","\t<OUTPUT scorefxn=\"membrane\"/>\n","</ROSETTASCRIPTS>\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.scoring.ScoreFunctionFactory: SCOREFUNCTION: input_files/wts/stage3_rlx_membrane.wts\n","core.io.fragments: reading fragments from file: ./input_files/frags/GYPA.3mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 3mer fragments from file ./input_files/frags/GYPA.3mers\n","core.io.fragments: reading fragments from file: ./input_files/frags/GYPA.9mers ...\n","core.io.fragments: rosetta++ fileformat detected! Calling legacy reader...\n","core.fragments.ConstantLengthFragSet: finished reading top 200 9mer fragments from file ./input_files/frags/GYPA.9mers\n","core.import_pose.import_pose: File '/content/input_files/threaded_pdbs/5eh6_A.human.pdb' automatically determined to be of type PDB\n","protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hybridize\" of type Hybridize\n","protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n","protocols.rosetta_scripts.ParsedProtocol: Added mover \"hybridize\"\n","CPU times: user 424 ms, sys: 23 ms, total: 447 ms\n","Wall time: 457 ms\n"]}]},{"cell_type":"code","metadata":{"id":"gvyV2upvoAW1","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1729447195408,"user_tz":-120,"elapsed":104439,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"75aaa350-14ce-4aa1-f5fd-ca22fb851400"},"source":["%%time\n","membrane = pyrosetta.create_score_function(\"membrane_highres_Menv_smooth\")\n","jd = pyrosetta.toolbox.py_jobdistributor.PyJobDistributor(pdb_name=\"GYPA_human_\",nstruct=2,scorefxn=membrane)\n","jd.native_pose = pose\n","#Name of the score dataframe\n","hdf = pd.DataFrame()\n","while not jd.job_complete:\n","    test_pose = pose.clone()\n","    xml.apply(test_pose)\n","    test_hdf = pd.DataFrame.from_records(dict(test_pose.scores), index=[jd.current_name])\n","    hdf = pd.concat([hdf, test_hdf])\n","    jd.output_decoy(test_pose)"],"execution_count":23,"outputs":[{"output_type":"stream","name":"stdout","text":["Working on decoy: GYPA_human__1.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: Found 1 domains using template /content/input_files/threaded_pdbs/5eh6_A.human.pdb\n","protocols.hybridization.HybridizeProtocol: domain 1: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.HybridizeProtocol: LOOP start: 1  stop: 42  cut: 0  size: 42  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeProtocol:\n","protocols.hybridization.HybridizeProtocol:\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.human.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n","core.fragment: compute strand/loop fractions for 42 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLHHHHLLLLLLLEEEEHHHHHHHHHHHHHHHHHHHHLL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 24 1  EDGE 24 1 -1  EDGE 24 42 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 4 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0 0\n","protocols.moves.TrialCounter:      RandomMover trials=      4;  accepts= 1.0000;  energy_drop/trial=  -2.46217\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.07811 0.459687\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.6152;  energy_drop/trial=  -0.00582\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.767934 0.417341\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 1.0000;  energy_drop/trial=   0.24268\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5130;  energy_drop/trial=  -0.00083\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.42089 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.6032;  energy_drop/trial=   0.00602\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.5324 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5521;  energy_drop/trial=   0.00873\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.04324 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.6667;  energy_drop/trial=  -0.07851\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5336;  energy_drop/trial=   0.00146\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.41435 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.19432\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5472;  energy_drop/trial=   0.00584\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.83408 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.5085;  energy_drop/trial=   0.00201\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.71567 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      7 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    993;  accepts= 0.5408;  energy_drop/trial=   0.00594\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.3718 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5552;  energy_drop/trial=   0.00858\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 12.6382 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5607;  energy_drop/trial=   0.01285\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.04298 -0.17797\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=   1.79267\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5962;  energy_drop/trial=  -0.00237\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -8.98314 -8.98314\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.1650;  energy_drop/trial=  -1.13696\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.63823 -4.91777\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.3450;  energy_drop/trial=  -1.04101\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -0.346739 -2.2552\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.2900;  energy_drop/trial=  -0.01391\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.5641 -42.3147\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     75;  accepts= 0.3467;  energy_drop/trial=  -0.03735\n","protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.1333;  energy_drop/trial=  -0.53329\n","protocols.moves.TrialCounter:      picker_rand trials=     10;  accepts= 0.1000;  energy_drop/trial=  -0.47238\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -33.4653 -42.3147\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    156;  accepts= 0.2308;  energy_drop/trial=   0.01210\n","protocols.moves.TrialCounter:           picker trials=     24;  accepts= 0.1667;  energy_drop/trial=  -0.40841\n","protocols.moves.TrialCounter:      picker_rand trials=     20;  accepts= 0.1500;  energy_drop/trial=  -0.17557\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -33.3686 -41.4715\n","protocols.moves.TrialCounter:         frag_15+ trials=     74;  accepts= 0.5000;  energy_drop/trial=   0.05892\n","protocols.moves.TrialCounter:           picker trials=     15;  accepts= 0.0667;  energy_drop/trial=   0.05787\n","protocols.moves.TrialCounter:      picker_rand trials=     11;  accepts= 0.1818;  energy_drop/trial=   0.01156\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -35.2958 -41.4715\n","protocols.moves.TrialCounter:         frag_15+ trials=    148;  accepts= 0.3851;  energy_drop/trial=   0.02434\n","protocols.moves.TrialCounter:           picker trials=     27;  accepts= 0.0370;  energy_drop/trial=   0.03215\n","protocols.moves.TrialCounter:      picker_rand trials=     25;  accepts= 0.1600;  energy_drop/trial=  -0.04170\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -43.7007 -43.7007\n","protocols.moves.TrialCounter:         frag_15+ trials=     71;  accepts= 0.2394;  energy_drop/trial=  -0.13108\n","protocols.moves.TrialCounter:           picker trials=     12 NO ACCEPTS.\n","protocols.moves.TrialCounter:      picker_rand trials=     17;  accepts= 0.1176;  energy_drop/trial=  -0.16461\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -37.5633 -43.7007\n","protocols.moves.TrialCounter:         frag_15+ trials=    144;  accepts= 0.2431;  energy_drop/trial=  -0.07619\n","protocols.moves.TrialCounter:           picker trials=     31;  accepts= 0.0323;  energy_drop/trial=   0.25165\n","protocols.moves.TrialCounter:      picker_rand trials=     25;  accepts= 0.0800;  energy_drop/trial=  -0.11193\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -36.4388 -36.4388\n","protocols.moves.TrialCounter:           picker trials=     52;  accepts= 0.0192;  energy_drop/trial=  -0.01457\n","protocols.moves.TrialCounter:      picker_rand trials=     48;  accepts= 0.0625;  energy_drop/trial=  -0.06021\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","core.optimization.LineMinimizer: Inaccurate G in step= 4.76837e-09 Deriv= -150.665 Finite Diff= 4.49534e+10\n","core.optimization.LineMinimizer: Inaccurate G in step= 6.21561e-07 Deriv= -150.665 Finite Diff= 3.4149e+08\n","core.optimization.Minimizer: Line search failed even after resetting Hessian; aborting at iter#2\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.relax.FastRelax: CMD: repeat  18163.5  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -58.3082  0.13802  0.13802  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -30.6664  0.170791  0.170791  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -12.0545  0.137108  0.137108  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  4.16602  0.115811  0.115811  0.44\n","protocols.relax.FastRelax: MRP: 0  4.16602  4.16602  0.115811  0.115811\n","protocols.relax.FastRelax: CMD: accept_to_best  4.16602  0.115811  0.115811  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  4.16602  0.115811  0.115811  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -47.6052  0.224679  0.224679  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -30.8609  0.155924  0.155924  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -15.6296  0.119034  0.119034  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  7.9784  0.220285  0.220285  0.44\n","protocols.relax.FastRelax: MRP: 1  7.9784  4.16602  0.115811  0.115811\n","protocols.relax.FastRelax: CMD: accept_to_best  7.9784  0.220285  0.220285  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  7.9784  0.220285  0.220285  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -55.2205  0.134334  0.134334  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.0988  0.136108  0.136108  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -15.3063  0.133965  0.133965  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  3.21221  0.126461  0.126461  0.44\n","protocols.relax.FastRelax: MRP: 2  3.21221  3.21221  0.126461  0.126461\n","protocols.relax.FastRelax: CMD: accept_to_best  3.21221  0.126461  0.126461  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  3.21221  0.126461  0.126461  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -56.9014  0.118714  0.118714  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -30.9984  0.169286  0.169286  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -16.019  0.0959475  0.0959475  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  3.42747  0.116778  0.116778  0.44\n","protocols.relax.FastRelax: MRP: 3  3.42747  3.21221  0.126461  0.126461\n","protocols.relax.FastRelax: CMD: accept_to_best  3.42747  0.116778  0.116778  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  3.42747  0.116778  0.116778  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -59.0047  0.17062  0.17062  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -34.4107  0.140061  0.140061  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -18.0215  0.123678  0.123678  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  3.45247  0.17821  0.17821  0.44\n","protocols.relax.FastRelax: MRP: 4  3.45247  3.21221  0.126461  0.126461\n","protocols.relax.FastRelax: CMD: accept_to_best  3.45247  0.17821  0.17821  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  3.45247  0.17821  0.17821  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n","Working on decoy: GYPA_human__0.pdb\n","protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Hybridize - hybridize=======================\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS small max length: 3\n","protocols.hybridization.HybridizeProtocol: FRAGMENTS big max length: 9\n","protocols.hybridization.HybridizeProtocol: Using initial template:    1 /content/input_files/threaded_pdbs/5eh6_A.human.pdb\n","protocols.hybridization.HybridizeProtocol: Align domain with TMscore of    1.000\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n","protocols.hybridization.HybridizeProtocol: ::::::::::::::::::::::::::::\n","protocols.hybridization.HybridizeProtocol: PEITLIIFGVMAGVIGTILLISYGIRRL\n","core.fragment: compute strand/loop fractions for 42 residues...\n","protocols.hybridization.FoldTreeHybridize: Secondary structure from fragments: LLLLLHHHHLLLLLLLEEEEHHHHHHHHHHHHHHHHHHHHLL\n","protocols.hybridization.FoldTreeHybridize: Steal chunks with rate = 0\n","protocols.hybridization.FoldTreeHybridize: Chunks used for foldtree setup:\n","protocols.hybridization.FoldTreeHybridize: LOOP  begin  end  cut  skip_rate  extended\n","protocols.hybridization.FoldTreeHybridize: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.FoldTreeHybridize:\n","protocols.hybridization.HybridizeFoldtreeDynamic: Identified template core chunk with index: 1\n","protocols.hybridization.HybridizeFoldtreeDynamic: LOOP start: 17  stop: 40  cut: 0  size: 24  skip rate: 0  extended?: False\n","protocols.hybridization.HybridizeFoldtreeDynamic:\n","protocols.loops.loops_main: [ WARNING ] Residue 42 is not compatible with cutpoints; variant type not changed.\n","protocols.loops.loops_main: [ WARNING ] Residue 43 is not compatible with cutpoints; variant type not changed.\n","protocols.hybridization.FoldTreeHybridize: FOLD_TREE  EDGE 43 23 1  EDGE 23 1 -1  EDGE 23 42 -1\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for 1mer fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for small fragments: 0.00000 0\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for big fragments: 0.99715 350\n","protocols.hybridization.FoldTreeHybridize: auto_frag_insertion_weight for template chunks: 0.00285 1\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 1                                                         \n","   Folding with score0 for max of 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 1: Replaced extended chains after 31 cycles.\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.172318 0\n","protocols.moves.TrialCounter:      RandomMover trials=     31;  accepts= 0.9355;  energy_drop/trial=  -0.31214\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 2                                                         \n","   Folding with score1 for 1000\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.46018 0.460046\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5752;  energy_drop/trial=  -0.00644\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 3                                                         \n","   Folding with score2 and score5 for 1000\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 1\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.42225 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5421;  energy_drop/trial=   0.00111\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 2\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.19621 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5701;  energy_drop/trial=   0.00762\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.42612 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      2;  accepts= 0.5000;  energy_drop/trial=  -0.32749\n","protocols.moves.TrialCounter: WeightedFragment trials=    998;  accepts= 0.5772;  energy_drop/trial=   0.00751\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 4\n","protocols.hybridization.FoldTreeHybridize: stop cycles due to convergence, converge_rms_big: 2.43404 converge_rms_small: 0\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.2355 -0.411641\n","protocols.moves.TrialCounter: WeightedFragment trials=    716;  accepts= 0.5587;  energy_drop/trial=   0.00230\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 5\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 5.53301 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5517;  energy_drop/trial=   0.00596\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score5...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 6\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 8.06203 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=   0.70970\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.6046;  energy_drop/trial=   0.00777\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 7\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 2.54042 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      3;  accepts= 0.3333;  energy_drop/trial=   0.18979\n","protocols.moves.TrialCounter: WeightedFragment trials=    997;  accepts= 0.5888;  energy_drop/trial=   0.00239\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 8\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 7.10595 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.00343\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.5924;  energy_drop/trial=   0.00753\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 9\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 0.985241 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1;  accepts= 1.0000;  energy_drop/trial=  -0.50863\n","protocols.moves.TrialCounter: WeightedFragment trials=    999;  accepts= 0.5786;  energy_drop/trial=   0.00191\n","protocols.hybridization.FoldTreeHybridize: Stage 3 select score2...\n","protocols.hybridization.FoldTreeHybridize: Stage 3 loop iteration 10\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 1.24339 -0.411641\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      4;  accepts= 0.2500;  energy_drop/trial=   0.03561\n","protocols.moves.TrialCounter: WeightedFragment trials=    996;  accepts= 0.6124;  energy_drop/trial=   0.00152\n","protocols.hybridization.FoldTreeHybridize: \n","===================================================================\n","   Stage 4                                                         \n","   Folding with score3 for 200\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 1: small fragments trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -0.00299701 -1.24008\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.3518;  energy_drop/trial=  -0.06863\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 2: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: 6.57896 -2.62408\n","protocols.moves.TrialCounter: WeightedFragment trials=    200;  accepts= 0.4650;  energy_drop/trial=   0.03687\n","protocols.hybridization.FoldTreeHybridize: Stage 4 loop iteration 3: small fragments smooth trials\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -2.71729 -4.20972\n","protocols.moves.TrialCounter:  ChunkTrialMover trials=      1 NO ACCEPTS.\n","protocols.moves.TrialCounter: WeightedFragment trials=    199;  accepts= 0.3015;  energy_drop/trial=  -0.02390\n","protocols.hybridization.HybridizeProtocol: Realigning template domains to stage1 pose.\n","core.util.switchresiduetypeset: [ WARNING ] When switching to a centroid ResidueTypeSet:  Pose already contains centroid ResidueTypes.\n","protocols.hybridization.CartesianHybridize: RUNNING FOR 4 MACROCYCLES\n","protocols.hybridization.CartesianHybridize:  total_res=42   prot_res=42\n","protocols.hybridization.CartesianHybridize: CYCLE 1\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -29.1406 -30.1328\n","protocols.moves.TrialCounter:       fragNS_15+ trials=     65;  accepts= 0.1077;  energy_drop/trial=   0.13388\n","protocols.moves.TrialCounter:           picker trials=     17;  accepts= 0.4118;  energy_drop/trial=  -0.34877\n","protocols.moves.TrialCounter:      picker_rand trials=     18;  accepts= 0.2778;  energy_drop/trial=  -0.34357\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -31.6306 -31.6306\n","protocols.moves.TrialCounter:       fragNS_15+ trials=    138;  accepts= 0.2609;  energy_drop/trial=   0.07619\n","protocols.moves.TrialCounter:           picker trials=     30;  accepts= 0.3000;  energy_drop/trial=  -0.31836\n","protocols.moves.TrialCounter:      picker_rand trials=     32;  accepts= 0.2188;  energy_drop/trial=  -0.21452\n","protocols.hybridization.CartesianHybridize: CYCLE 2\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.0005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -20.6334 -30.6339\n","protocols.moves.TrialCounter:         frag_15+ trials=     67;  accepts= 0.4030;  energy_drop/trial=  -0.09092\n","protocols.moves.TrialCounter:           picker trials=     12;  accepts= 0.3333;  energy_drop/trial=   0.57820\n","protocols.moves.TrialCounter:      picker_rand trials=     21;  accepts= 0.4286;  energy_drop/trial=  -0.00391\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -29.8433 -30.9692\n","protocols.moves.TrialCounter:         frag_15+ trials=    140;  accepts= 0.4571;  energy_drop/trial=  -0.08521\n","protocols.moves.TrialCounter:           picker trials=     27;  accepts= 0.2222;  energy_drop/trial=   0.23764\n","protocols.moves.TrialCounter:      picker_rand trials=     33;  accepts= 0.3939;  energy_drop/trial=  -0.08884\n","protocols.hybridization.CartesianHybridize: CYCLE 3\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.005\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 0.3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -13.821 -30.6134\n","protocols.moves.TrialCounter:         frag_15+ trials=     75;  accepts= 0.2000;  energy_drop/trial=   0.12096\n","protocols.moves.TrialCounter:           picker trials=     14;  accepts= 0.2143;  energy_drop/trial=  -0.23593\n","protocols.moves.TrialCounter:      picker_rand trials=     11;  accepts= 0.2727;  energy_drop/trial=   0.59661\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -23.6534 -30.6134\n","protocols.moves.TrialCounter:         frag_15+ trials=    144;  accepts= 0.2083;  energy_drop/trial=   0.09720\n","protocols.moves.TrialCounter:           picker trials=     30;  accepts= 0.3333;  energy_drop/trial=  -0.37977\n","protocols.moves.TrialCounter:      picker_rand trials=     26;  accepts= 0.3462;  energy_drop/trial=  -0.00403\n","protocols.hybridization.CartesianHybridize: CYCLE 4\n","protocols.hybridization.CartesianHybridize:   setting bonded weight = 0.05\n","protocols.hybridization.CartesianHybridize:   setting bonded angle weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded length weight = 0\n","protocols.hybridization.CartesianHybridize:   setting bonded torsion weight = 0\n","protocols.hybridization.CartesianHybridize:   setting cst    weight = 0.5\n","protocols.hybridization.CartesianHybridize:   setting vdw    weight = 3\n","protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -17.9804 -26.9636\n","protocols.moves.TrialCounter:           picker trials=     46;  accepts= 0.1957;  energy_drop/trial=  -0.15103\n","protocols.moves.TrialCounter:      picker_rand trials=     54;  accepts= 0.2593;  energy_drop/trial=   0.14807\n","protocols.hybridization.HybridizeProtocol: History :   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42\n","protocols.hybridization.HybridizeProtocol: History :   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1\n","protocols.hybridization.HybridizeProtocol:  batch_relax 1 :\n","core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (42)\n","protocols.relax.FastRelax: CMD: repeat  13839.4  0  0  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -24.8811  0.234902  0.234902  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -8.90957  0.296697  0.296697  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  4.48673  0.348856  0.348856  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  21.6411  0.481375  0.481375  0.44\n","protocols.relax.FastRelax: MRP: 0  21.6411  21.6411  0.481375  0.481375\n","protocols.relax.FastRelax: CMD: accept_to_best  21.6411  0.481375  0.481375  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  21.6411  0.481375  0.481375  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -23.0997  0.148237  0.148237  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -3.75627  0.430413  0.430413  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  4.04361  0.363193  0.363193  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  20.2659  0.425794  0.425794  0.44\n","protocols.relax.FastRelax: MRP: 1  20.2659  20.2659  0.425794  0.425794\n","protocols.relax.FastRelax: CMD: accept_to_best  20.2659  0.425794  0.425794  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  20.2659  0.425794  0.425794  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -27.6733  0.148993  0.148993  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -5.90004  0.31988  0.31988  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  4.3001  0.331134  0.331134  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  19.5546  0.394087  0.394087  0.44\n","protocols.relax.FastRelax: MRP: 2  19.5546  19.5546  0.394087  0.394087\n","protocols.relax.FastRelax: CMD: accept_to_best  19.5546  0.394087  0.394087  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  19.5546  0.394087  0.394087  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -27.0853  0.2567  0.2567  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -11.059  0.297227  0.297227  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  2.17411  0.380414  0.380414  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  18.2127  0.481749  0.481749  0.44\n","protocols.relax.FastRelax: MRP: 3  18.2127  18.2127  0.481749  0.481749\n","protocols.relax.FastRelax: CMD: accept_to_best  18.2127  0.481749  0.481749  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  18.2127  0.481749  0.481749  0.44\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -25.6257  0.171133  0.171133  0.0088\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  -6.97472  0.43603  0.43603  0.11\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  1.10708  0.384623  0.384623  0.242\n","core.optimization.Minimizer: [ WARNING ] LBFGS MAX CYCLES 2 EXCEEDED, BUT FUNC NOT CONVERGED!\n","protocols.relax.FastRelax: CMD: ramp_repack_min  17.1892  0.456945  0.456945  0.44\n","protocols.relax.FastRelax: MRP: 4  17.1892  17.1892  0.456945  0.456945\n","protocols.relax.FastRelax: CMD: accept_to_best  17.1892  0.456945  0.456945  0.44\n","protocols.relax.FastRelax: CMD: endrepeat  17.1892  0.456945  0.456945  0.44\n","protocols::checkpoint: Deleting checkpoints of FastRelax\n","protocols.rosetta_scripts.ParsedProtocol: setting status to success\n","CPU times: user 1min 43s, sys: 712 ms, total: 1min 43s\n","Wall time: 1min 44s\n"]}]},{"cell_type":"markdown","metadata":{"id":"1GSWo_C0qE3M"},"source":["#Part IV ‚Äì Analyze and visualize the modelled structures"]},{"cell_type":"markdown","metadata":{"id":"eVDe0mz3QRQi"},"source":["1. Now, we need to determine our best models by analyzing their score dataframes. For this, we are going to obtain the data frame for each of the models that we generated with PyRosetta.\n","\n","  We can visually inspect these data frames just writing their name in a code chunk."]},{"cell_type":"code","metadata":{"id":"sD8nrCKprPpr","colab":{"base_uri":"https://localhost:8080/","height":161},"executionInfo":{"status":"ok","timestamp":1729447195408,"user_tz":-120,"elapsed":7,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"e355e0b0-32c2-45b8-c628-4fa7e4a74fed"},"source":["#Call here the name of the different dataframes that were created\n","hdf\n"],"execution_count":24,"outputs":[{"output_type":"execute_result","data":{"text/plain":["                   Menv_smooth  atom_pair_constraint  cart_bonded  \\\n","GYPA_human__1.pdb    -2.602461              4.677751   167.787798   \n","GYPA_human__0.pdb    -5.413962              5.382300   168.358186   \n","\n","                   dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n","GYPA_human__1.pdb          0.0          0.0          0.0          0.0   \n","GYPA_human__0.pdb          0.0          0.0          0.0          0.0   \n","\n","                       fa_atr     fa_dun  fa_intra_rep  ...     fa_rep  \\\n","GYPA_human__1.pdb -185.079815  48.942319    117.095685  ...  90.489134   \n","GYPA_human__0.pdb -151.427025  45.066502    120.515500  ...  60.122577   \n","\n","                   hbond_bb_sc  hbond_lr_bb  hbond_sc  hbond_sr_bb     omega  \\\n","GYPA_human__1.pdb    -0.442582          0.0  0.000000   -34.828282  0.895635   \n","GYPA_human__0.pdb    -0.341836          0.0 -0.261213   -39.221145 -3.122443   \n","\n","                    p_aa_pp       rama   ref  total_score  \n","GYPA_human__1.pdb  1.832539 -18.474291 -4.61    -3.209094  \n","GYPA_human__0.pdb -7.371308 -20.630833 -4.61   -15.326076  \n","\n","[2 rows x 23 columns]"],"text/html":["\n","  <div id=\"df-8fbb3b76-59b9-464b-a9a5-c2d6c1816337\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Menv_smooth</th>\n","      <th>atom_pair_constraint</th>\n","      <th>cart_bonded</th>\n","      <th>dslf_ca_dih</th>\n","      <th>dslf_cs_ang</th>\n","      <th>dslf_ss_dih</th>\n","      <th>dslf_ss_dst</th>\n","      <th>fa_atr</th>\n","      <th>fa_dun</th>\n","      <th>fa_intra_rep</th>\n","      <th>...</th>\n","      <th>fa_rep</th>\n","      <th>hbond_bb_sc</th>\n","      <th>hbond_lr_bb</th>\n","      <th>hbond_sc</th>\n","      <th>hbond_sr_bb</th>\n","      <th>omega</th>\n","      <th>p_aa_pp</th>\n","      <th>rama</th>\n","      <th>ref</th>\n","      <th>total_score</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>GYPA_human__1.pdb</th>\n","      <td>-2.602461</td>\n","      <td>4.677751</td>\n","      <td>167.787798</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-185.079815</td>\n","      <td>48.942319</td>\n","      <td>117.095685</td>\n","      <td>...</td>\n","      <td>90.489134</td>\n","      <td>-0.442582</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>-34.828282</td>\n","      <td>0.895635</td>\n","      <td>1.832539</td>\n","      <td>-18.474291</td>\n","      <td>-4.61</td>\n","      <td>-3.209094</td>\n","    </tr>\n","    <tr>\n","      <th>GYPA_human__0.pdb</th>\n","      <td>-5.413962</td>\n","      <td>5.382300</td>\n","      <td>168.358186</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-151.427025</td>\n","      <td>45.066502</td>\n","      <td>120.515500</td>\n","      <td>...</td>\n","      <td>60.122577</td>\n","      <td>-0.341836</td>\n","      <td>0.0</td>\n","      <td>-0.261213</td>\n","      <td>-39.221145</td>\n","      <td>-3.122443</td>\n","      <td>-7.371308</td>\n","      <td>-20.630833</td>\n","      <td>-4.61</td>\n","      <td>-15.326076</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>2 rows √ó 23 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-8fbb3b76-59b9-464b-a9a5-c2d6c1816337')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-8fbb3b76-59b9-464b-a9a5-c2d6c1816337 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-8fbb3b76-59b9-464b-a9a5-c2d6c1816337');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-51d148b9-02e3-490f-99b6-7ca4dd9748dc\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-51d148b9-02e3-490f-99b6-7ca4dd9748dc')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-51d148b9-02e3-490f-99b6-7ca4dd9748dc button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_f14d91bc-01cd-4a1a-997e-ba8b29536533\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('hdf')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_f14d91bc-01cd-4a1a-997e-ba8b29536533 button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('hdf');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"hdf"}},"metadata":{},"execution_count":24}]},{"cell_type":"code","metadata":{"id":"GZ1NnFdOlIT0","colab":{"base_uri":"https://localhost:8080/","height":161},"executionInfo":{"status":"ok","timestamp":1729447195408,"user_tz":-120,"elapsed":6,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"36841634-d551-4b79-baca-c66273849fb3"},"source":["odf\n"],"execution_count":25,"outputs":[{"output_type":"execute_result","data":{"text/plain":["                           Menv_smooth  atom_pair_constraint  cart_bonded  \\\n","GYPA_ornitorrinchus_1.pdb     6.801851              5.455776   172.821656   \n","GYPA_ornitorrinchus_0.pdb    11.359693              5.327946   154.337543   \n","\n","                           dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n","GYPA_ornitorrinchus_1.pdb          0.0          0.0          0.0          0.0   \n","GYPA_ornitorrinchus_0.pdb          0.0          0.0          0.0          0.0   \n","\n","                               fa_atr     fa_dun  fa_intra_rep  ...  \\\n","GYPA_ornitorrinchus_1.pdb -179.652744  29.388224    128.965199  ...   \n","GYPA_ornitorrinchus_0.pdb -166.299082  27.638869    123.135727  ...   \n","\n","                              fa_rep  hbond_bb_sc  hbond_lr_bb  hbond_sc  \\\n","GYPA_ornitorrinchus_1.pdb  52.207441     0.000000          0.0       0.0   \n","GYPA_ornitorrinchus_0.pdb  55.907397    -0.069027          0.0       0.0   \n","\n","                           hbond_sr_bb     omega   p_aa_pp       rama  ref  \\\n","GYPA_ornitorrinchus_1.pdb   -44.985920 -1.803436 -8.130390 -17.206839  5.0   \n","GYPA_ornitorrinchus_0.pdb   -43.846114 -3.701601 -8.635703 -22.432960  5.0   \n","\n","                           total_score  \n","GYPA_ornitorrinchus_1.pdb   -37.664419  \n","GYPA_ornitorrinchus_0.pdb   -34.373575  \n","\n","[2 rows x 23 columns]"],"text/html":["\n","  <div id=\"df-d1b82a8f-c313-4d9d-b499-d0bc9f4690f3\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Menv_smooth</th>\n","      <th>atom_pair_constraint</th>\n","      <th>cart_bonded</th>\n","      <th>dslf_ca_dih</th>\n","      <th>dslf_cs_ang</th>\n","      <th>dslf_ss_dih</th>\n","      <th>dslf_ss_dst</th>\n","      <th>fa_atr</th>\n","      <th>fa_dun</th>\n","      <th>fa_intra_rep</th>\n","      <th>...</th>\n","      <th>fa_rep</th>\n","      <th>hbond_bb_sc</th>\n","      <th>hbond_lr_bb</th>\n","      <th>hbond_sc</th>\n","      <th>hbond_sr_bb</th>\n","      <th>omega</th>\n","      <th>p_aa_pp</th>\n","      <th>rama</th>\n","      <th>ref</th>\n","      <th>total_score</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>GYPA_ornitorrinchus_1.pdb</th>\n","      <td>6.801851</td>\n","      <td>5.455776</td>\n","      <td>172.821656</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-179.652744</td>\n","      <td>29.388224</td>\n","      <td>128.965199</td>\n","      <td>...</td>\n","      <td>52.207441</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-44.985920</td>\n","      <td>-1.803436</td>\n","      <td>-8.130390</td>\n","      <td>-17.206839</td>\n","      <td>5.0</td>\n","      <td>-37.664419</td>\n","    </tr>\n","    <tr>\n","      <th>GYPA_ornitorrinchus_0.pdb</th>\n","      <td>11.359693</td>\n","      <td>5.327946</td>\n","      <td>154.337543</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-166.299082</td>\n","      <td>27.638869</td>\n","      <td>123.135727</td>\n","      <td>...</td>\n","      <td>55.907397</td>\n","      <td>-0.069027</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-43.846114</td>\n","      <td>-3.701601</td>\n","      <td>-8.635703</td>\n","      <td>-22.432960</td>\n","      <td>5.0</td>\n","      <td>-34.373575</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>2 rows √ó 23 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d1b82a8f-c313-4d9d-b499-d0bc9f4690f3')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-d1b82a8f-c313-4d9d-b499-d0bc9f4690f3 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-d1b82a8f-c313-4d9d-b499-d0bc9f4690f3');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-7b7d2b96-c1b8-46cb-86ed-111d180e76f9\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-7b7d2b96-c1b8-46cb-86ed-111d180e76f9')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-7b7d2b96-c1b8-46cb-86ed-111d180e76f9 button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_df266a92-66db-4f62-bd6d-78411289fa27\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('odf')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_df266a92-66db-4f62-bd6d-78411289fa27 button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('odf');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"odf"}},"metadata":{},"execution_count":25}]},{"cell_type":"code","metadata":{"id":"AixtSW1JlJXz","colab":{"base_uri":"https://localhost:8080/","height":161},"executionInfo":{"status":"ok","timestamp":1729447195408,"user_tz":-120,"elapsed":5,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"fef00d3b-34c0-4369-822a-a8096967045d"},"source":["mdf\n"],"execution_count":26,"outputs":[{"output_type":"execute_result","data":{"text/plain":["                     Menv_smooth  atom_pair_constraint  cart_bonded  \\\n","GYPA_marmota__0.pdb   -18.686129              4.961442   110.212905   \n","GYPA_marmota__1.pdb   -14.384180              5.214092   712.509674   \n","\n","                     dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n","GYPA_marmota__0.pdb          0.0          0.0          0.0          0.0   \n","GYPA_marmota__1.pdb          0.0          0.0          0.0          0.0   \n","\n","                        fa_atr     fa_dun  fa_intra_rep  ...      fa_rep  \\\n","GYPA_marmota__0.pdb -183.55478  57.813045    150.691747  ...   46.933810   \n","GYPA_marmota__1.pdb -195.38371  60.815309    245.529592  ...  124.967307   \n","\n","                     hbond_bb_sc  hbond_lr_bb  hbond_sc  hbond_sr_bb  \\\n","GYPA_marmota__0.pdb    -0.528759          0.0  0.000000   -35.741563   \n","GYPA_marmota__1.pdb     0.000000          0.0 -0.593745   -37.025284   \n","\n","                        omega   p_aa_pp       rama   ref  total_score  \n","GYPA_marmota__0.pdb -3.535251 -3.941944 -18.473909 -1.24   -68.240693  \n","GYPA_marmota__1.pdb  1.406749 -2.927329 -12.514954 -1.24   266.560859  \n","\n","[2 rows x 23 columns]"],"text/html":["\n","  <div id=\"df-18cb9fa4-d769-4d9a-a167-e8bdc6376536\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Menv_smooth</th>\n","      <th>atom_pair_constraint</th>\n","      <th>cart_bonded</th>\n","      <th>dslf_ca_dih</th>\n","      <th>dslf_cs_ang</th>\n","      <th>dslf_ss_dih</th>\n","      <th>dslf_ss_dst</th>\n","      <th>fa_atr</th>\n","      <th>fa_dun</th>\n","      <th>fa_intra_rep</th>\n","      <th>...</th>\n","      <th>fa_rep</th>\n","      <th>hbond_bb_sc</th>\n","      <th>hbond_lr_bb</th>\n","      <th>hbond_sc</th>\n","      <th>hbond_sr_bb</th>\n","      <th>omega</th>\n","      <th>p_aa_pp</th>\n","      <th>rama</th>\n","      <th>ref</th>\n","      <th>total_score</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>GYPA_marmota__0.pdb</th>\n","      <td>-18.686129</td>\n","      <td>4.961442</td>\n","      <td>110.212905</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-183.55478</td>\n","      <td>57.813045</td>\n","      <td>150.691747</td>\n","      <td>...</td>\n","      <td>46.933810</td>\n","      <td>-0.528759</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>-35.741563</td>\n","      <td>-3.535251</td>\n","      <td>-3.941944</td>\n","      <td>-18.473909</td>\n","      <td>-1.24</td>\n","      <td>-68.240693</td>\n","    </tr>\n","    <tr>\n","      <th>GYPA_marmota__1.pdb</th>\n","      <td>-14.384180</td>\n","      <td>5.214092</td>\n","      <td>712.509674</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-195.38371</td>\n","      <td>60.815309</td>\n","      <td>245.529592</td>\n","      <td>...</td>\n","      <td>124.967307</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>-0.593745</td>\n","      <td>-37.025284</td>\n","      <td>1.406749</td>\n","      <td>-2.927329</td>\n","      <td>-12.514954</td>\n","      <td>-1.24</td>\n","      <td>266.560859</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>2 rows √ó 23 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-18cb9fa4-d769-4d9a-a167-e8bdc6376536')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-18cb9fa4-d769-4d9a-a167-e8bdc6376536 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-18cb9fa4-d769-4d9a-a167-e8bdc6376536');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-733ef9ea-66ee-49ee-be25-02497e691f9f\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-733ef9ea-66ee-49ee-be25-02497e691f9f')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-733ef9ea-66ee-49ee-be25-02497e691f9f button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_d1857dc0-9b08-4b99-8494-44b691fce767\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('mdf')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_d1857dc0-9b08-4b99-8494-44b691fce767 button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('mdf');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"mdf"}},"metadata":{},"execution_count":26}]},{"cell_type":"code","metadata":{"id":"yvfyIrqClKJL","colab":{"base_uri":"https://localhost:8080/","height":161},"executionInfo":{"status":"ok","timestamp":1729447195408,"user_tz":-120,"elapsed":5,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"735dd875-c9d9-4891-f8ce-d1ad6080f306"},"source":["gdf"],"execution_count":27,"outputs":[{"output_type":"execute_result","data":{"text/plain":["                     Menv_smooth  atom_pair_constraint  cart_bonded  \\\n","GYPA_gorilla__0.pdb    -9.721510              6.292492   167.746912   \n","GYPA_gorilla__1.pdb    -3.875817              4.847885   165.079096   \n","\n","                     dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n","GYPA_gorilla__0.pdb          0.0          0.0          0.0          0.0   \n","GYPA_gorilla__1.pdb          0.0          0.0          0.0          0.0   \n","\n","                         fa_atr     fa_dun  fa_intra_rep  ...     fa_rep  \\\n","GYPA_gorilla__0.pdb -166.504488  45.505765    128.476668  ...  42.112057   \n","GYPA_gorilla__1.pdb -174.246776  47.752706    133.345832  ...  48.862424   \n","\n","                     hbond_bb_sc  hbond_lr_bb  hbond_sc  hbond_sr_bb  \\\n","GYPA_gorilla__0.pdb     -0.56426          0.0       0.0   -39.371535   \n","GYPA_gorilla__1.pdb      0.00000          0.0       0.0   -39.315911   \n","\n","                        omega   p_aa_pp       rama   ref  total_score  \n","GYPA_gorilla__0.pdb -3.707682 -8.545411 -22.047972 -4.81   -35.436458  \n","GYPA_gorilla__1.pdb -3.443383 -1.472890 -25.490143 -4.81   -27.300132  \n","\n","[2 rows x 23 columns]"],"text/html":["\n","  <div id=\"df-e3bdc8b4-c808-4270-b489-908d04fe1039\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Menv_smooth</th>\n","      <th>atom_pair_constraint</th>\n","      <th>cart_bonded</th>\n","      <th>dslf_ca_dih</th>\n","      <th>dslf_cs_ang</th>\n","      <th>dslf_ss_dih</th>\n","      <th>dslf_ss_dst</th>\n","      <th>fa_atr</th>\n","      <th>fa_dun</th>\n","      <th>fa_intra_rep</th>\n","      <th>...</th>\n","      <th>fa_rep</th>\n","      <th>hbond_bb_sc</th>\n","      <th>hbond_lr_bb</th>\n","      <th>hbond_sc</th>\n","      <th>hbond_sr_bb</th>\n","      <th>omega</th>\n","      <th>p_aa_pp</th>\n","      <th>rama</th>\n","      <th>ref</th>\n","      <th>total_score</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>GYPA_gorilla__0.pdb</th>\n","      <td>-9.721510</td>\n","      <td>6.292492</td>\n","      <td>167.746912</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-166.504488</td>\n","      <td>45.505765</td>\n","      <td>128.476668</td>\n","      <td>...</td>\n","      <td>42.112057</td>\n","      <td>-0.56426</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-39.371535</td>\n","      <td>-3.707682</td>\n","      <td>-8.545411</td>\n","      <td>-22.047972</td>\n","      <td>-4.81</td>\n","      <td>-35.436458</td>\n","    </tr>\n","    <tr>\n","      <th>GYPA_gorilla__1.pdb</th>\n","      <td>-3.875817</td>\n","      <td>4.847885</td>\n","      <td>165.079096</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-174.246776</td>\n","      <td>47.752706</td>\n","      <td>133.345832</td>\n","      <td>...</td>\n","      <td>48.862424</td>\n","      <td>0.00000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-39.315911</td>\n","      <td>-3.443383</td>\n","      <td>-1.472890</td>\n","      <td>-25.490143</td>\n","      <td>-4.81</td>\n","      <td>-27.300132</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>2 rows √ó 23 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-e3bdc8b4-c808-4270-b489-908d04fe1039')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-e3bdc8b4-c808-4270-b489-908d04fe1039 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-e3bdc8b4-c808-4270-b489-908d04fe1039');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-362d1d60-bb64-4a05-a53d-feb6ea585b35\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-362d1d60-bb64-4a05-a53d-feb6ea585b35')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-362d1d60-bb64-4a05-a53d-feb6ea585b35 button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","  <div id=\"id_5a2ddb3f-85be-4d6a-b245-2da5ed54dca6\">\n","    <style>\n","      .colab-df-generate {\n","        background-color: #E8F0FE;\n","        border: none;\n","        border-radius: 50%;\n","        cursor: pointer;\n","        display: none;\n","        fill: #1967D2;\n","        height: 32px;\n","        padding: 0 0 0 0;\n","        width: 32px;\n","      }\n","\n","      .colab-df-generate:hover {\n","        background-color: #E2EBFA;\n","        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","        fill: #174EA6;\n","      }\n","\n","      [theme=dark] .colab-df-generate {\n","        background-color: #3B4455;\n","        fill: #D2E3FC;\n","      }\n","\n","      [theme=dark] .colab-df-generate:hover {\n","        background-color: #434B5C;\n","        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","        fill: #FFFFFF;\n","      }\n","    </style>\n","    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('gdf')\"\n","            title=\"Generate code using this dataframe.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","       width=\"24px\">\n","    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n","  </svg>\n","    </button>\n","    <script>\n","      (() => {\n","      const buttonEl =\n","        document.querySelector('#id_5a2ddb3f-85be-4d6a-b245-2da5ed54dca6 button.colab-df-generate');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      buttonEl.onclick = () => {\n","        google.colab.notebook.generateWithVariable('gdf');\n","      }\n","      })();\n","    </script>\n","  </div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"gdf"}},"metadata":{},"execution_count":27}]},{"cell_type":"markdown","metadata":{"id":"gKSGvDM8Okly"},"source":["2. As Rosetta uses an stochastic algorithm for comparative modelling, we are only interested in the low energy structures. Thus, we can sort our models by the column total_score as in the following cell code:"]},{"cell_type":"code","metadata":{"id":"0XFEzgf4Li4f","colab":{"base_uri":"https://localhost:8080/","height":161},"executionInfo":{"status":"ok","timestamp":1729447195717,"user_tz":-120,"elapsed":313,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"7f6ac29a-8f36-4135-e07c-f5cb70263990"},"source":["#hdf.sort_values('total_score')\n","gdf.sort_values('total_score')"],"execution_count":28,"outputs":[{"output_type":"execute_result","data":{"text/plain":["                     Menv_smooth  atom_pair_constraint  cart_bonded  \\\n","GYPA_gorilla__0.pdb    -9.721510              6.292492   167.746912   \n","GYPA_gorilla__1.pdb    -3.875817              4.847885   165.079096   \n","\n","                     dslf_ca_dih  dslf_cs_ang  dslf_ss_dih  dslf_ss_dst  \\\n","GYPA_gorilla__0.pdb          0.0          0.0          0.0          0.0   \n","GYPA_gorilla__1.pdb          0.0          0.0          0.0          0.0   \n","\n","                         fa_atr     fa_dun  fa_intra_rep  ...     fa_rep  \\\n","GYPA_gorilla__0.pdb -166.504488  45.505765    128.476668  ...  42.112057   \n","GYPA_gorilla__1.pdb -174.246776  47.752706    133.345832  ...  48.862424   \n","\n","                     hbond_bb_sc  hbond_lr_bb  hbond_sc  hbond_sr_bb  \\\n","GYPA_gorilla__0.pdb     -0.56426          0.0       0.0   -39.371535   \n","GYPA_gorilla__1.pdb      0.00000          0.0       0.0   -39.315911   \n","\n","                        omega   p_aa_pp       rama   ref  total_score  \n","GYPA_gorilla__0.pdb -3.707682 -8.545411 -22.047972 -4.81   -35.436458  \n","GYPA_gorilla__1.pdb -3.443383 -1.472890 -25.490143 -4.81   -27.300132  \n","\n","[2 rows x 23 columns]"],"text/html":["\n","  <div id=\"df-79aecfbf-370c-4274-a4c4-3cfeaae5fd28\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Menv_smooth</th>\n","      <th>atom_pair_constraint</th>\n","      <th>cart_bonded</th>\n","      <th>dslf_ca_dih</th>\n","      <th>dslf_cs_ang</th>\n","      <th>dslf_ss_dih</th>\n","      <th>dslf_ss_dst</th>\n","      <th>fa_atr</th>\n","      <th>fa_dun</th>\n","      <th>fa_intra_rep</th>\n","      <th>...</th>\n","      <th>fa_rep</th>\n","      <th>hbond_bb_sc</th>\n","      <th>hbond_lr_bb</th>\n","      <th>hbond_sc</th>\n","      <th>hbond_sr_bb</th>\n","      <th>omega</th>\n","      <th>p_aa_pp</th>\n","      <th>rama</th>\n","      <th>ref</th>\n","      <th>total_score</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>GYPA_gorilla__0.pdb</th>\n","      <td>-9.721510</td>\n","      <td>6.292492</td>\n","      <td>167.746912</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-166.504488</td>\n","      <td>45.505765</td>\n","      <td>128.476668</td>\n","      <td>...</td>\n","      <td>42.112057</td>\n","      <td>-0.56426</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-39.371535</td>\n","      <td>-3.707682</td>\n","      <td>-8.545411</td>\n","      <td>-22.047972</td>\n","      <td>-4.81</td>\n","      <td>-35.436458</td>\n","    </tr>\n","    <tr>\n","      <th>GYPA_gorilla__1.pdb</th>\n","      <td>-3.875817</td>\n","      <td>4.847885</td>\n","      <td>165.079096</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-174.246776</td>\n","      <td>47.752706</td>\n","      <td>133.345832</td>\n","      <td>...</td>\n","      <td>48.862424</td>\n","      <td>0.00000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>-39.315911</td>\n","      <td>-3.443383</td>\n","      <td>-1.472890</td>\n","      <td>-25.490143</td>\n","      <td>-4.81</td>\n","      <td>-27.300132</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>2 rows √ó 23 columns</p>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-79aecfbf-370c-4274-a4c4-3cfeaae5fd28')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-79aecfbf-370c-4274-a4c4-3cfeaae5fd28 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-79aecfbf-370c-4274-a4c4-3cfeaae5fd28');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-0350e53b-00e8-40fc-ba05-f7d39dd2e72a\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-0350e53b-00e8-40fc-ba05-f7d39dd2e72a')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-0350e53b-00e8-40fc-ba05-f7d39dd2e72a button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe"}},"metadata":{},"execution_count":28}]},{"cell_type":"markdown","metadata":{"id":"jE8R-lrqRdXe"},"source":["3. Having too many tables or tables with too much information would be challenging to analyze our results. For simplicity, we can make a plot with the **total_score** of each PDB for each species using **matplotlib**"]},{"cell_type":"code","metadata":{"id":"WU7m7LhCq8DG","colab":{"base_uri":"https://localhost:8080/","height":555},"executionInfo":{"status":"ok","timestamp":1729447198759,"user_tz":-120,"elapsed":3045,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"4b8d2f83-5678-4bb6-802a-0168e27bd6df"},"source":["import matplotlib\n","import seaborn\n","frames = [hdf,odf,mdf,gdf]\n","results = pd.concat(frames)\n","matplotlib.rcParams['figure.figsize'] = [12.0, 8.0]\n","seaborn.barplot(x=\"total_score\", y=\"index\", data=results.sort_values('total_score').reset_index())"],"execution_count":29,"outputs":[{"output_type":"execute_result","data":{"text/plain":["<Axes: xlabel='total_score', ylabel='index'>"]},"metadata":{},"execution_count":29},{"output_type":"display_data","data":{"text/plain":["<Figure size 1200x800 with 1 Axes>"],"image/png":"iVBORw0KGgoAAAANSUhEUgAABI0AAAKnCAYAAAALL71eAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAB+f0lEQVR4nOzde1RVdf7/8dcRBOXqiIqKlAKKSEomWsRvzFEJxbFUyqxRMMks0XAUUZu8ZZmTqBGEl8Lr9MU0zSy74PRNM1RIAs0EzMQvmmgjKhTehd8fLs902pCI6PHyfKy112p/9v58Pu99ztQsXuvz2cdUUVFRIQAAAAAAAOA36li7AAAAAAAAANx8CI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAICBrbULAO5k5eXlOnz4sJydnWUymaxdDgAAAADgNldRUaFffvlFzZs3V506f7yWiNAIsKLDhw/L09PT2mUAAAAAAO4wBw8eVIsWLf7wHkIjwIqcnZ0lXfqX1cXFxcrVAAAAAABud6WlpfL09DT/PfpHCI0AK7q8Jc3FxYXQCAAAAABww1TnFSm8CBsAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADHgRNoBq6zR+ubVLAKota3aEtUsAAAAAbmmsNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoRFQTUOHDlW/fv3M5926ddOYMWOsVg8AAAAAANfTHRMaHTlyRDExMfLx8VG9evXk7u6u4OBgzZ8/X927d1doaKihT3Jysho0aKBDhw5p06ZNMplM5sPd3V3h4eHav3+/RZ/U1FTZ2NgoOjr6Rj3aTcUaQcrq1avVtm1b1atXT+3bt9cnn3xyQ+cHAAAAAOB2dEeERvv371fHjh2VlpammTNnKjs7W9u2bVNcXJw+/vhjRUZGKiMjQwsXLjT3KSgoUFxcnBITE9WiRQtze35+vg4fPqzVq1fr+++/V9++fXXx4kXz9ZSUFMXFxSk1NVVnzpy5oc9ZHRUVFbpw4YK1y6g1W7du1ZNPPqmoqChlZ2erX79+6tevn3bv3m3t0gAAAAAAuKXdEaHRyJEjZWtrqx07dmjgwIHy8/OTl5eXHn30UW3YsEERERFKSEhQbGysCgoKVFFRoaioKD388MMaMmSIxVhNmjRRs2bN1LVrV02ZMkV79uzRvn37JF0KmrZu3aqJEyeqTZs2Wrt2bbVrXLp0qRo0aKCPP/5Yvr6+cnBw0GOPPaZTp05p2bJlatmypf70pz/phRdesAipVqxYocDAQDk7O6tp06Z66qmn9PPPP5uvX14h9emnn6pTp06yt7fX119/rW7dumn06NEaM2aM/vSnP8nd3V1vv/22ysrK9PTTT8vZ2Vk+Pj769NNPLercvHmzunTpInt7ezVr1kwTJ040h1BDhw7V5s2blZCQYF6RdeDAAV28eFFRUVFq1aqV6tevL19fXyUkJFz191iZhIQE9erVS+PHj5efn59mzJih++67T0lJSVX2mTZtmu69914tXLhQnp6ecnBw0MCBA1VSUmK+5+LFixo7dqwaNGggNzc3xcXFqaKiwjDWhQsXNGrUKLm6uqpRo0aaPHlypfcBAAAAAHCrue1Do+LiYqWlpSk6OlqOjo6V3mMymRQZGakePXpo2LBhSkpK0u7duy1WHlWmfv36kqRz585JkpYsWaI+ffrI1dVVgwcPVkpKylXVeurUKb355ptauXKlPvvsM23atEn9+/fXJ598ok8++UQrVqzQwoUL9f7775v7nD9/XjNmzNDOnTu1bt06HThwQEOHDjWMPXHiRM2aNUu5ubnq0KGDJGnZsmVq1KiRMjMzNXr0aD3//PN6/PHH9eCDD+rbb781h2anTp2SJP30008KCwtT586dtXPnTs2fP18pKSl65ZVXJF0KcIKCgjR8+HAVFRWpqKhInp6eKi8vV4sWLbR69Wrt2bNHU6ZM0YsvvqhVq1Zd1edTmW3btqlnz54WbaGhodq2bdsf9tu3b59WrVqljz76SJ999pmys7M1cuRI8/U5c+Zo6dKlWrx4sb7++msdP35cH3zwgWGcZcuWydbWVpmZmUpISNDcuXP1zjvvVDnv2bNnVVpaanEAAAAAAHAzsrV2Adfbvn37VFFRIV9fX4v2Ro0ambePRUdH65///KcWLVokf39/ffXVV1qzZo0aN25c5bhFRUWKj4+Xh4eHfH19VV5erqVLlyoxMVGSNGjQII0bN04FBQVq1apVtWo9f/685s+fL29vb0nSY489phUrVujo0aNycnJSu3bt9Je//EVffvmlnnjiCUnSsGHDzP29vLz05ptvqnPnzvr111/l5ORkvvbyyy8rJCTEYr6AgAC99NJLkqRJkyZp1qxZatSokYYPHy5JmjJliubPn69du3bpgQceUHJysjw9PZWUlCSTyaS2bdvq8OHDmjBhgqZMmSJXV1fZ2dnJwcFBTZs2Nc9jY2Oj6dOnm89btWqlbdu2adWqVRo4cGC1PpuqHDlyRO7u7hZt7u7uOnLkyB/2O3PmjJYvXy4PDw9JUmJiovr06aM5c+aoadOmeuONNzRp0iQNGDBAkrRgwQJ9/vnnhnE8PT01b948mUwm+fr66rvvvtO8efPMn+HvvfbaaxafBQAAAAAAN6vbfqVRVTIzM5WTkyN/f3+dPXtW0qWtZyNGjJCfn5/Fr2T9VosWLeTo6KjmzZurrKxMa9askZ2dnTZu3KiysjKFhYVJuhRKhYSEaPHixdWuycHBwRwYSZfCj5YtW1qEP+7u7hbbz7KystS3b1/dddddcnZ21kMPPSRJKiwstBg7MDDQMN/lFUfSpWDHzc1N7du3t5hLknm+3NxcBQUFyWQyme8JDg7Wr7/+qkOHDv3hs7311lvq1KmTGjduLCcnJy1atMhQ44101113mQMjSQoKClJ5ebny8/NVUlKioqIi3X///ebrtra2lX6GDzzwgMXnERQUpB9++MFiC+FvTZo0SSUlJebj4MGDtfhUAAAAAADUntt+pZGPj49MJpPy8/Mt2r28vCT9d4vZZba2trK1rfpj2bJli1xcXNSkSRM5Ozub21NSUnT8+HGL8crLy7Vr1y5Nnz5ddepcOZ+rW7euxbnJZKq0rby8XJJUVlam0NBQhYaG6t1331Xjxo1VWFio0NBQ85a5yyrbmnel+S6HIZfnq6mVK1cqNjZWc+bMUVBQkJydnTV79mxlZGRc07iS1LRpUx09etSi7ejRoxYrnW4m9vb2sre3t3YZAAAAAABc0W2/0sjNzU0hISFKSkpSWVnZNY/XqlUreXt7WwRGxcXF+vDDD7Vy5Url5OSYj+zsbJ04cUJpaWnXPG9l8vLyVFxcrFmzZunPf/6z2rZta7EKqbb5+flp27ZtFi96Tk9Pl7Ozs/kX5uzs7AyrbNLT0/Xggw9q5MiR6tixo3x8fPTjjz/WSk1BQUH64osvLNo2btyooKCgP+xXWFiow4cPm8+3b9+uOnXqyNfXV66urmrWrJlFqHXhwgVlZWUZxvl98LV9+3a1bt1aNjY2NXkcAAAAAABuGrd9aCRJycnJunDhggIDA/Xee+8pNzdX+fn5+te//qW8vLxr/gN/xYoVcnNz08CBA3XPPfeYj4CAAIWFhV31C7Gr66677pKdnZ0SExO1f/9+rV+/XjNmzLguc0mXfoXu4MGDGj16tPLy8vThhx9q6tSpGjt2rHklVcuWLZWRkaEDBw7o2LFjKi8vV+vWrbVjxw59/vnn2rt3ryZPnqxvvvmmVmqKiYnRZ599pjlz5igvL0/Tpk3Tjh07NGrUKPM9kyZNUkREhEW/evXqKTIyUjt37tSWLVv0wgsvaODAgeYVSjExMZo1a5bWrVunvLw8jRw5UidPnjTMX1hYqLFjxyo/P1+pqalKTExUTExMrTwbAAAAAADWdEeERt7e3srOzlbPnj01adIkBQQEKDAwUImJiYqNjb3moGXx4sXq37+/xbttLgsPD9f69et17Nixa5qjMo0bN9bSpUu1evVqtWvXTrNmzVJ8fHytz3OZh4eHPvnkE2VmZiogIEDPPfecoqKizC/TlqTY2FjZ2NioXbt25u1yI0aM0IABA/TEE0/o/vvvV3FxscUvlV2LBx98UP/zP/+jRYsWKSAgQO+//77WrVune+65x3xPUVGR4f1JPj4+GjBggMLCwvTwww+rQ4cOSk5ONl8fN26chgwZosjISPOWuv79+xvmj4iI0OnTp9WlSxdFR0crJiZGzz77bK08GwAAAAAA1mSq+O1eI+AOMG3aNK1bt045OTnWLkWlpaVydXVVSUmJXFxcrF3OFXUav9zaJQDVljU74so3AQAAAHeYq/k79I5YaQQAAAAAAICrQ2h0g/Tu3VtOTk6VHjNnzrR2eVZV1efi5OQkGxubKq9t2bLF2qUDAAAAAHDbYnvaDfLTTz/p9OnTlV5r2LChGjZseIMrunns27evymunT59W/fr1K73m4eFR5bVbBdvTgOuH7WkAAACA0dX8HWp7g2q643l4eFi7hJuWj4+PtUsAAAAAAAC/w/Y0AAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADCwtXYBAG4dWbMjrF0CAAAAAOAGYaURAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGttYuAABuZ53GL7d2CXesrNkR1i4BAAAAuKWx0ggAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEZWNHToUPXr1++Gzjlt2jTde++9tTbepk2bZDKZdPLkyVob82b1+8/OGt8fAAAAAAA3itVDoyNHjigmJkY+Pj6qV6+e3N3dFRwcrPnz56t79+4KDQ019ElOTlaDBg106NAhc2hx+XB3d1d4eLj2799v0Sc1NVU2NjaKjo6+UY92RQkJCVq6dKn5vFu3bhozZsx1nTM2NlZffPHFdZ3jRvr+++8VHh6uli1bymQy6Y033rB2SQAAAAAA3BasGhrt379fHTt2VFpammbOnKns7Gxt27ZNcXFx+vjjjxUZGamMjAwtXLjQ3KegoEBxcXFKTExUixYtzO35+fk6fPiwVq9ere+//159+/bVxYsXzddTUlIUFxen1NRUnTlz5ro+17lz56p1n6urqxo0aHBD5q+oqNCFCxfk5OQkNze3Wp/TWk6dOiUvLy/NmjVLTZs2tXY5AAAAAADcNqwaGo0cOVK2trbasWOHBg4cKD8/P3l5eenRRx/Vhg0bFBERoYSEBMXGxqqgoEAVFRWKiorSww8/rCFDhliM1aRJEzVr1kxdu3bVlClTtGfPHu3bt0/SpaBp69atmjhxotq0aaO1a9deVZ1r1qyRv7+/7O3t1bJlS82ZM8fiesuWLTVjxgxFRETIxcVFzz77rJYuXaoGDRro888/l5+fn5ycnNSrVy8VFRWZ+/12e9PQoUO1efNmJSQkmFdNHThwQJK0efNmdenSRfb29mrWrJkmTpyoCxcumMfp1q2bRo0apTFjxqhRo0YKDQ01r8D69NNP1alTJ9nb2+vrr7+ucotVfHy8mjVrJjc3N0VHR+v8+fPme86ePasJEybI09NT9vb28vHxUUpKisVnkJWVpcDAQDk4OOjBBx9Ufn5+pc952ZgxY9StWzfz+fvvv6/27durfv36cnNzU8+ePVVWVnbF76Zz586aPXu2Bg0aJHt7+yveL8n83axbt06tW7dWvXr1FBoaqoMHD1rcN2vWLLm7u8vZ2VlRUVFVho3Tp09X48aN5eLioueee67aoSEAAAAAADczq4VGxcXFSktLU3R0tBwdHSu9x2QyKTIyUj169NCwYcOUlJSk3bt3W6w8qkz9+vUl/XfFzZIlS9SnTx+5urpq8ODBhsDjj2RlZWngwIEaNGiQvvvuO02bNk2TJ0+22FYmSfHx8QoICFB2drYmT54s6dIqmPj4eK1YsUJfffWVCgsLFRsbW+k8CQkJCgoK0vDhw1VUVKSioiJ5enrqp59+UlhYmDp37qydO3dq/vz5SklJ0SuvvGLRf9myZbKzs1N6eroWLFhgbp84caJmzZql3NxcdejQodK5v/zyS/3444/68ssvtWzZMi1dutTi+SIiIpSamqo333xTubm5WrhwoZycnCzG+Mc//qE5c+Zox44dsrW11bBhw6r7EauoqEhPPvmkhg0bptzcXG3atEkDBgxQRUVFtce4WqdOndKrr76q5cuXKz09XSdPntSgQYPM11etWqVp06Zp5syZ2rFjh5o1a6bk5GTDOF988YW55tTUVK1du1bTp0+vct6zZ8+qtLTU4gAAAAAA4GZka62J9+3bp4qKCvn6+lq0N2rUyLyiIzo6Wv/85z+1aNEi+fv766uvvtKaNWvUuHHjKsctKipSfHy8PDw85Ovrq/Lyci1dulSJiYmSpEGDBmncuHEqKChQq1atrljn3Llz1aNHD3MQ1KZNG+3Zs0ezZ8/W0KFDzfd1795d48aNM59v2bJF58+f14IFC+Tt7S1JGjVqlF5++eVK53F1dZWdnZ0cHBwstlklJyfL09NTSUlJMplMatu2rQ4fPqwJEyZoypQpqlPnUu7XunVrvf766xafgyS9/PLLCgkJ+cNn/NOf/qSkpCTZ2Niobdu26tOnj7744gsNHz5ce/fu1apVq7Rx40b17NlTkuTl5WUY49VXX9VDDz0k6VJQ1adPH505c0b16tX7w7kv13rhwgUNGDBAd999tySpffv2V+x3Lc6fP6+kpCTdf//9ki6Fbn5+fsrMzFSXLl30xhtvKCoqSlFRUZKkV155Rf/+978Nq43s7Oy0ePFiOTg4yN/fXy+//LLGjx+vGTNmmL+b33rttdf+MFQCAAAAAOBmYfUXYf9eZmamcnJy5O/vr7Nnz0q6tPVsxIgR8vPzq/LXqlq0aCFHR0c1b95cZWVlWrNmjezs7LRx40aVlZUpLCxM0qVQKiQkRIsXL65WPbm5uQoODrZoCw4O1g8//GDxzqTAwEBDXwcHB3NgJEnNmjXTzz//XK15fzt/UFCQTCaTxfy//vqrDh06ZG7r1KlTpf0rq+v3/P39ZWNjU2mdOTk5srGxMQdCVfntKqZmzZpJUrWfNSAgQD169FD79u31+OOP6+2339aJEyeq1bembG1t1blzZ/N527Zt1aBBA+Xm5kq69LlfDpQuCwoKqrR2BwcHi3t+/fVXw1a3yyZNmqSSkhLzUdV9AAAAAABYm9VWGvn4+MhkMlm8+0b67yqWy1vMLrO1tZWtbdXlbtmyRS4uLmrSpImcnZ3N7SkpKTp+/LjFeOXl5dq1a5emT59e6WqQmqhsi13dunUtzk0m03XbclXVFr+q2n+rsjrLy8slGb+H6oxxOeC6PEadOnUMz/3bdybZ2Nho48aN2rp1q9LS0pSYmKh//OMfysjIqNZqsFuJvb19td+9BAAAAACANVltpZGbm5tCQkKUlJRUrRceX0mrVq3k7e1tERgVFxfrww8/1MqVK5WTk2M+srOzdeLECaWlpV1xXD8/P6Wnp1u0paenq02bNharc2qDnZ2dxeqly/Nv27bNInRJT0+Xs7Ozxa/HXS/t27dXeXm5Nm/eXOMxGjdubPECcOnSCqbfMplMCg4O1vTp05WdnS07Ozt98MEHNZ7zSi5cuKAdO3aYz/Pz83Xy5En5+flJuvS5Z2RkWPTZvn27YZydO3fq9OnTFvc4OTnJ09PzOlUOAAAAAMCNYdXtacnJybpw4YICAwP13nvvKTc3V/n5+frXv/6lvLy8aw5lVqxYITc3Nw0cOFD33HOP+QgICFBYWFi1Xog9btw4ffHFF5oxY4b27t2rZcuWKSkpqcoXWl+Lli1bKiMjQwcOHNCxY8dUXl6ukSNH6uDBgxo9erTy8vL04YcfaurUqRo7dmytrZK6Uk2RkZEaNmyY1q1bp4KCAm3atEmrVq2q9hjdu3fXjh07tHz5cv3www+aOnWqdu/ebb6ekZFhfuF0YWGh1q5dq//85z/mAOePnDt3zhwGnjt3Tj/99JNycnLMv5wnSUlJSerRo4dFv7p162r06NHKyMhQVlaWhg4dqgceeEBdunSRJMXExGjx4sVasmSJ9u7dq6lTp+r777+vdP6oqCjt2bNHn3zyiaZOnapRo0bdkO8GAAAAAIDryap/2Xp7eys7O1s9e/bUpEmTFBAQoMDAQCUmJio2NlYzZsy4pvEXL16s/v37W7wP6LLw8HCtX79ex44d+8Mx7rvvPq1atUorV67UPffcoylTpujll1+2eAl2bYmNjZWNjY3atWunxo0bq7CwUB4eHvrkk0+UmZmpgIAAPffcc4qKitJLL71U6/NXZf78+Xrsscc0cuRItW3bVsOHD7+q1WGhoaGaPHmy4uLi1LlzZ/3yyy+KiIgwX3dxcdFXX32lsLAwtWnTRi+99JLmzJmj3r17X3Hsw4cPq2PHjurYsaP5JegdO3bUM888Y77n2LFj+vHHHy36OTg4aMKECXrqqacUHBwsJycnvffee+brTzzxhLnmTp066f/+7//0/PPPG+bv0aOHWrdura5du+qJJ57QI488omnTplX7swEAAAAA4GZlqriev2sO3ISWLl2qMWPG6OTJk9YuRaWlpXJ1dVVJSYlcXFysXQ6ug07jl1u7hDtW1uyIK98EAAAA3GGu5u9Q9tAAAAAAAADA4I4PjXr37i0nJ6dKj5kzZ1q7vDteVd+Nk5OTtmzZYu3yAAAAAAC4bVX9G/Z3iHfeecfi169+q2HDhje4Gvze739l7bc8PDxqNObQoUOvyzupAAAAAAC4ndzxoVFNgwfcGD4+PtYuAQAAAACAO9Idvz0NAAAAAAAARoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAxsrV0AANzOsmZHWLsEAAAAAKgRVhoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBga+0CAOBO02n8cmuXcEfImh1h7RIAAACAWxorjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaIQa2bRpk0wmk06ePClJWrp0qRo0aGC+Pm3aNN17771Wqe16MplMWrdunSTpwIEDMplMysnJsWpNAAAAAABcD4RG1+DIkSOKiYmRj4+P6tWrJ3d3dwUHB2v+/Pnq3r27QkNDDX2Sk5PVoEEDHTp0yBy8XD7c3d0VHh6u/fv3W/RJTU2VjY2NoqOjb9SjXdGDDz6ooqIiubq6WrsUnTlzRtHR0XJzc5OTk5PCw8N19OhRa5cFAAAAAMAtjdCohvbv36+OHTsqLS1NM2fOVHZ2trZt26a4uDh9/PHHioyMVEZGhhYuXGjuU1BQoLi4OCUmJqpFixbm9vz8fB0+fFirV6/W999/r759++rixYvm6ykpKYqLi1NqaqrOnDlzQ5+zMufPn5ednZ2aNm0qk8lk7XL097//XR999JFWr16tzZs36/DhwxowYIC1ywIAAAAA4JZGaFRDI0eOlK2trXbs2KGBAwfKz89PXl5eevTRR7VhwwZFREQoISFBsbGxKigoUEVFhaKiovTwww9ryJAhFmM1adJEzZo1U9euXTVlyhTt2bNH+/btk3QpaNq6dasmTpyoNm3aaO3atVdV59tvvy1PT085ODiof//+mjt3rsU2MkmaP3++vL29ZWdnJ19fX61YscLiuslk0vz58/XII4/I0dFRr776qmF72pV88803CgkJUaNGjeTq6qqHHnpI33777VU9S2VKSkqUkpKiuXPnqnv37urUqZOWLFmirVu3avv27VX2a9mypWbMmKEnn3xSjo6O8vDw0FtvvWVxzw8//KCuXbuqXr16ateunTZu3FjpWHl5eXrwwQdVr1493XPPPdq8efM1PxcAAAAAANZGaFQDxcXFSktLU3R0tBwdHSu9x2QyKTIyUj169NCwYcOUlJSk3bt3W6w8qkz9+vUlSefOnZMkLVmyRH369JGrq6sGDx6slJSUateZnp6u5557TjExMcrJyVFISIheffVVi3s++OADxcTEaNy4cdq9e7dGjBihp59+Wl9++aXFfdOmTVP//v313XffadiwYdWu4bJffvlFkZGR+vrrr7V9+3a1bt1aYWFh+uWXX656rN/KysrS+fPn1bNnT3Nb27Ztddddd2nbtm1/2Hf27NkKCAhQdna2Jk6cqJiYGHMwVF5ergEDBsjOzk4ZGRlasGCBJkyYUOk448eP17hx45Sdna2goCD17dtXxcXFld579uxZlZaWWhwAAAAAANyMCI1qYN++faqoqJCvr69Fe6NGjeTk5CQnJydzwLBo0SLt3r1bY8aM0aJFi9S4ceMqxy0qKlJ8fLw8PDzk6+ur8vJyLV26VIMHD5YkDRo0SF9//bUKCgqqVWdiYqJ69+6t2NhYtWnTRiNHjlTv3r0t7omPj9fQoUM1cuRItWnTRmPHjtWAAQMUHx9vcd9TTz2lp59+Wl5eXrrrrruqNf9vde/eXYMHD1bbtm3l5+enRYsW6dSpU9e8KufIkSOys7MzrJ5yd3fXkSNH/rBvcHCweQXX6NGj9dhjj2nevHmSpH//+9/Ky8vT8uXLFRAQoK5du2rmzJmVjjNq1CiFh4fLz89P8+fPl6ura5Xh3muvvSZXV1fz4enpefUPDQAAAADADUBoVIsyMzOVk5Mjf39/nT17VtKlrWcjRoyQn5+f+vXrV2m/Fi1ayNHRUc2bN1dZWZnWrFkjOzs7bdy4UWVlZQoLC5N0KZQKCQnR4sWLq1VPfn6+unTpYtH2+/Pc3FwFBwdbtAUHBys3N9eiLTAwsFpzVuXo0aMaPny4WrduLVdXV7m4uOjXX39VYWHhNY17LYKCggznl587NzdXnp6eat68eZX3V9Zua2urwMBAw+d32aRJk1RSUmI+Dh48eK2PAQAAAADAdWFr7QJuRT4+PjKZTMrPz7do9/LykvTfLWaX2drayta26o96y5YtcnFxUZMmTeTs7GxuT0lJ0fHjxy3GKy8v165duzR9+nTVqXPjMr+qtuFVV2RkpIqLi5WQkKC7775b9vb2CgoKMm/Dq6mmTZvq3LlzOnnypMVqo6NHj6pp06bXNPb1YG9vL3t7e2uXAQAAAADAFbHSqAbc3NwUEhKipKQklZWVXfN4rVq1kre3t0VgVFxcrA8//FArV65UTk6O+cjOztaJEyeUlpZ2xXF9fX31zTffWLT9/tzPz0/p6ekWbenp6WrXrt01PJFRenq6XnjhBYWFhcnf31/29vY6duzYNY/bqVMn1a1bV1988YW5LT8/X4WFhVWuDLrs9y/K3r59u/z8/CRd+lwOHjyooqKiKu+vrP3ChQvKysoyjwMAAAAAwK2KlUY1lJycrODgYAUGBmratGnq0KGD6tSpo2+++UZ5eXnq1KnTNY2/YsUKubm5aeDAgYaftQ8LC1NKSop69er1h2OMHj1aXbt21dy5c9W3b1/97//+rz799FOL8caPH6+BAweqY8eO6tmzpz766COtXbtW//73v6+p/t9r3bq1VqxYocDAQJWWlmr8+PGGFVk14erqqqioKI0dO1YNGzaUi4uLRo8eraCgID3wwAPm+9q2bavXXntN/fv3N7elp6fr9ddfV79+/bRx40atXr1aGzZskCT17NlTbdq0UWRkpGbPnq3S0lL94x//qLSGt956S61bt5afn5/mzZunEydO1Ohl4QAAAAAA3ExYaVRD3t7eys7OVs+ePTVp0iQFBAQoMDBQiYmJio2N1YwZM65p/MWLF6t///6GwEiSwsPDtX79+iuu1AkODtaCBQs0d+5cBQQE6LPPPtPf//531atXz3xPv379lJCQoPj4ePn7+2vhwoVasmSJunXrdk31/15KSopOnDih++67T0OGDNELL7ygJk2a1MrY8+bN01//+leFh4era9euatq0qdauXWtxT35+vkpKSizaxo0bpx07dqhjx4565ZVXNHfuXIWGhkqS6tSpow8++ECnT59Wly5d9Mwzzxh+ee6yWbNmadasWQoICNDXX3+t9evXq1GjRrXybAAAAAAAWIupoqKiwtpF4MYZPny48vLytGXLFmuXYlUtW7bUmDFjNGbMGKvWUVpaKldXV5WUlMjFxcWqteDG6TR+ubVLuCNkzY6wdgkAAADATedq/g5le9ptLj4+XiEhIXJ0dNSnn36qZcuWKTk52dplAQAAAACAmxzb025hvXv3lpOTU6XHzJkzJUmZmZkKCQlR+/bttWDBAr355pt65plnrFy5pXfffbfK52jVqlWV1/z9/a1dOgAAAAAAty1WGt3C3nnnHZ0+fbrSaw0bNpQkrVq16kaWVCOPPPKI7r///kqv1a1bV+fPn6/yWk0dOHCgxn0BAAAAALgTEBrdwjw8PKxdQq1wdnaWs7OztcsAAAAAAAC/wfY0AAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADCwtXYBAHCnyZodYe0SAAAAAOCKWGkEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAICBrbULAIA7Wafxy61dwm0ra3aEtUsAAAAAbmmsNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoZEVDR06VP369buhc06bNk333ntvrY23adMmmUwmnTx5stbGvFn9/vvq1q2bxowZY7V6AAAAAAC4nqweGh05ckQxMTHy8fFRvXr15O7uruDgYM2fP1/du3dXaGiooU9ycrIaNGigQ4cOmUOLy4e7u7vCw8O1f/9+iz6pqamysbFRdHT0jXq0K0pISNDSpUvN5zcihIiNjdUXX3xxXee40VavXq22bduqXr16at++vT755BNrlwQAAAAAwC3PqqHR/v371bFjR6WlpWnmzJnKzs7Wtm3bFBcXp48//liRkZHKyMjQwoULzX0KCgoUFxenxMREtWjRwtyen5+vw4cPa/Xq1fr+++/Vt29fXbx40Xw9JSVFcXFxSk1N1ZkzZ67rc507d65a97m6uqpBgwY3ZP6KigpduHBBTk5OcnNzq/U5rWXr1q168sknFRUVpezsbPXr10/9+vXT7t27rV0aAAAAAAC3NKuGRiNHjpStra127NihgQMHys/PT15eXnr00Ue1YcMGRUREKCEhQbGxsSooKFBFRYWioqL08MMPa8iQIRZjNWnSRM2aNVPXrl01ZcoU7dmzR/v27ZN0KWjaunWrJk6cqDZt2mjt2rVXVeeaNWvk7+8ve3t7tWzZUnPmzLG43rJlS82YMUMRERFycXHRs88+q6VLl6pBgwb6/PPP5efnJycnJ/Xq1UtFRUXmfr/d7jR06FBt3rxZCQkJ5lVTBw4ckCRt3rxZXbp0kb29vZo1a6aJEyfqwoUL5nG6deumUaNGacyYMWrUqJFCQ0PNK7A+/fRTderUSfb29vr6668N29Mu1xAfH69mzZrJzc1N0dHROn/+vPmes2fPasKECfL09JS9vb18fHyUkpJi8RlkZWUpMDBQDg4OevDBB5Wfn1/pc142ZswYdevWzXz+/vvvq3379qpfv77c3NzUs2dPlZWVXfG7SUhIUK9evTR+/Hj5+flpxowZuu+++5SUlFRln8ufwcKFC+Xp6SkHBwcNHDhQJSUl5nsuXryosWPHqkGDBnJzc1NcXJwqKioMY124cEGjRo2Sq6urGjVqpMmTJ1d6HwAAAAAAtxqrhUbFxcVKS0tTdHS0HB0dK73HZDIpMjJSPXr00LBhw5SUlKTdu3dbrDyqTP369SX9d8XNkiVL1KdPH7m6umrw4MGGwOOPZGVlaeDAgRo0aJC+++47TZs2TZMnT7bYViZJ8fHxCggIUHZ2tiZPnixJOnXqlOLj47VixQp99dVXKiwsVGxsbKXzJCQkKCgoSMOHD1dRUZGKiork6empn376SWFhYercubN27typ+fPnKyUlRa+88opF/2XLlsnOzk7p6elasGCBuX3ixImaNWuWcnNz1aFDh0rn/vLLL/Xjjz/qyy+/1LJly7R06VKL54uIiFBqaqrefPNN5ebmauHChXJycrIY4x//+IfmzJmjHTt2yNbWVsOGDavuR6yioiI9+eSTGjZsmHJzc7Vp0yYNGDCgWuHLtm3b1LNnT4u20NBQbdu27Q/77du3T6tWrdJHH32kzz77TNnZ2Ro5cqT5+pw5c7R06VItXrxYX3/9tY4fP64PPvjAMM6yZctka2urzMxMJSQkaO7cuXrnnXeqnPfs2bMqLS21OAAAAAAAuBnZWmviffv2qaKiQr6+vhbtjRo1Mm8fi46O1j//+U8tWrRI/v7++uqrr7RmzRo1bty4ynGLiooUHx8vDw8P+fr6qry8XEuXLlViYqIkadCgQRo3bpwKCgrUqlWrK9Y5d+5c9ejRwxwEtWnTRnv27NHs2bM1dOhQ833du3fXuHHjzOdbtmzR+fPntWDBAnl7e0uSRo0apZdffrnSeVxdXWVnZycHBwc1bdrU3J6cnCxPT08lJSXJZDKpbdu2Onz4sCZMmKApU6aoTp1LuV/r1q31+uuvW3wOkvTyyy8rJCTkD5/xT3/6k5KSkmRjY6O2bduqT58++uKLLzR8+HDt3btXq1at0saNG83hjJeXl2GMV199VQ899JCkS0FVnz59dObMGdWrV+8P575c64ULFzRgwADdfffdkqT27dtfsZ906Z1Y7u7uFm3u7u46cuTIH/Y7c+aMli9fLg8PD0lSYmKi+vTpozlz5qhp06Z64403NGnSJA0YMECStGDBAn3++eeGcTw9PTVv3jyZTCb5+vrqu+++07x58zR8+PBK533ttdc0ffr0aj0bAAAAAADWZPUXYf9eZmamcnJy5O/vr7Nnz0q6tPVsxIgR8vPzq/LXxlq0aCFHR0c1b95cZWVlWrNmjezs7LRx40aVlZUpLCxM0qVQKiQkRIsXL65WPbm5uQoODrZoCw4O1g8//GDxzqTAwEBDXwcHB3NgJEnNmjXTzz//XK15fzt/UFCQTCaTxfy//vqrDh06ZG7r1KlTpf0rq+v3/P39ZWNjU2mdOTk5srGxMQdCVfntKqZmzZpJUrWfNSAgQD169FD79u31+OOP6+2339aJEyeq1bem7rrrLnNgJElBQUEqLy9Xfn6+SkpKVFRUpPvvv9983dbWttLP8oEHHrD4boKCggz/2/itSZMmqaSkxHwcPHiwFp8KAAAAAIDaY7WVRj4+PjKZTBbvvpH+u4rl8hazy2xtbWVrW3W5W7ZskYuLi5o0aSJnZ2dze0pKio4fP24xXnl5uXbt2qXp06ebV+pcq8q22NWtW9fi3GQyXbf33VS1xa+q9t+qrM7y8nJJxu+hOmNcDlEuj1GnTh3Dc//2nUk2NjbauHGjtm7dqrS0NCUmJuof//iHMjIyrrgarGnTpjp69KhF29GjRy1Wa91M7O3tZW9vb+0yAAAAAAC4IqutNHJzc1NISIiSkpKq9cLjK2nVqpW8vb0tAqPi4mJ9+OGHWrlypXJycsxHdna2Tpw4obS0tCuO6+fnp/T0dIu29PR0tWnTxmJ1Tm2ws7MzrFDx8/PTtm3bLEKX9PR0OTs7W/x63PXSvn17lZeXa/PmzTUeo3HjxhYvAJcurWD6LZPJpODgYE2fPl3Z2dmys7Or9B1CvxcUFKQvvvjCom3jxo0KCgr6w36FhYU6fPiw+Xz79u2qU6eOfH195erqqmbNmikjI8N8/cKFC8rKyjKM89t7Lo/TunXrWv/fBgAAAAAAN5pVt6clJyfrwoULCgwM1Hvvvafc3Fzl5+frX//6l/Ly8q75D+8VK1bIzc1NAwcO1D333GM+AgICFBYWVq0XYo8bN05ffPGFZsyYob1792rZsmVKSkqq8oXW16Jly5bKyMjQgQMHdOzYMZWXl2vkyJE6ePCgRo8erby8PH344YeaOnWqxo4dW2urpK5UU2RkpIYNG6Z169apoKBAmzZt0qpVq6o9Rvfu3bVjxw4tX75cP/zwg6ZOnardu3ebr2dkZGjmzJnasWOHCgsLtXbtWv3nP/+Rn5/fFceOiYnRZ599pjlz5igvL0/Tpk3Tjh07NGrUKPM9kyZNUkREhEW/evXqKTIyUjt37tSWLVv0wgsvaODAgeYVSjExMZo1a5bWrVunvLw8jRw5UidPnjTMX1hYqLFjxyo/P1+pqalKTExUTExMtT8bAAAAAABuVlbbniZJ3t7eys7O1syZMzVp0iQdOnRI9vb2ateunWJjYy1+zaomFi9erP79+1u8c+ay8PBwDRkyRMeOHVOjRo2qHOO+++7TqlWrNGXKFM2YMUPNmjXTyy+/bPES7NoSGxuryMhItWvXTqdPn1ZBQYFatmypTz75ROPHj1dAQIAaNmyoqKgovfTSS7U+f1Xmz5+vF198USNHjlRxcbHuuusuvfjii9XuHxoaqsmTJysuLk5nzpzRsGHDFBERoe+++06S5OLioq+++kpvvPGGSktLdffdd2vOnDnq3bv3Fcd+8MEH9T//8z966aWX9OKLL6p169Zat26d7rnnHvM9RUVFKiwstOjn4+OjAQMGKCwsTMePH9df//pXJScnm6+PGzdORUVFioyMVJ06dTRs2DD1799fJSUlFuNERETo9OnT6tKli2xsbBQTE6Nnn3222p8NAAAAAAA3K1PF9XrJDnCTmjZtmtatW2fYImcNpaWlcnV1VUlJiVxcXKxdDqyg0/jl1i7htpU1O+LKNwEAAAB3mKv5O/Sm+/U0AAAAAAAAWN8dHxr17t1bTk5OlR4zZ860dnl3vKq+GycnJ23ZssXa5QEAAAAAcNu647en/fTTTzp9+nSl1xo2bKiGDRve4IrwW/v27avymoeHh+rXr38Dq6l9bE8D29OuH7anAQAAAEZX83eoVV+EfTPw8PCwdgn4Az4+PtYuAQAAAACAO9Idvz0NAAAAAAAARoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAxqFBr95z//qfLad999V+NiAAAAAAAAcHOoUWjUvn17bdiwwdAeHx+vLl26XHNRAAAAAAAAsK4ahUZjx45VeHi4nn/+eZ0+fVo//fSTevTooddff13/8z//U9s1AgAAAAAA4AarUWgUFxenbdu2acuWLerQoYM6dOgge3t77dq1S/3796/tGgEAAAAAAHCD1fhF2D4+Prrnnnt04MABlZaW6oknnlDTpk1rszYAAAAAAABYSY1Co/T0dHXo0EE//PCDdu3apfnz52v06NF64okndOLEidquEQAAAAAAADdYjUKj7t2764knntD27dvl5+enZ555RtnZ2SosLFT79u1ru0YAAAAAAADcYLY16ZSWlqaHHnrIos3b21vp6el69dVXa6UwAAAAAAAAWI+poqKioqad9+3bpx9//FFdu3ZV/fr1VVFRIZPJVJv1Abe10tJSubq6qqSkRC4uLtYuBwAAAABwm7uav0NrtD2tuLhYPXr0UJs2bRQWFqaioiJJUlRUlGJjY2syJAAAAAAAAG4iNQqN/v73v6tu3boqLCyUg4ODuf2JJ57Qp59+WmvFAQAAAAAAwDpq/E6jzz//XC1atLBob926tf7v//6vVgoDAAAAAACA9dRopVFZWZnFCqPLjh8/Lnt7+2suCgAAAAAAANZVo9Doz3/+s5YvX24+N5lMKi8v1+uvv66//OUvtVYcAAAAAAAArKNG29Nef/119ejRQzt27NC5c+cUFxen77//XsePH1d6enpt1wgAAAAAAIAbrEYrje655x7t3btX/+///T89+uijKisr04ABA5SdnS1vb+/arhEAAAAAAAA3mKmioqLC2kUAd6rS0lK5urqqpKRELi4u1i4HAAAAAHCbu5q/Q6u9PW3Xrl3VLqBDhw7VvhcAAAAAAAA3n2qHRvfee69MJpMqKipkMpnM7ZcXKv227eLFi7VYIgAAAAAAAG60ar/TqKCgQPv371dBQYHWrFmjVq1aKTk5WTk5OcrJyVFycrK8vb21Zs2a61kvAAAAAAAAboBqrzS6++67zf/8+OOP680331RYWJi5rUOHDvL09NTkyZPVr1+/Wi0SAAAAAAAAN1aNfj3tu+++U6tWrQztrVq10p49e665KAAAAAAAAFhXjUIjPz8/vfbaazp37py57dy5c3rttdfk5+dXa8UBAAAAAADAOqq9Pe23FixYoL59+6pFixbmX0rbtWuXTCaTPvroo1otEABwZZ3GL7d2CTedrNkR1i4BAAAAuKXVKDTq0qWL9u/fr3fffVd5eXmSpCeeeEJPPfWUHB0da7VAAAAAAAAA3Hg1Co0kydHRUc8++2xt1gIAAAAAAICbRI1Dox9++EFffvmlfv75Z5WXl1tcmzJlyjUXBgAAAAAAAOupUWj09ttv6/nnn1ejRo3UtGlTmUwm8zWTyURoBAAAAAAAcIurUWj0yiuv6NVXX9WECRNqux4AAAAAAADcBOrUpNOJEyf0+OOP13YtAAAAAAAAuEnUKDR6/PHHlZaWVtu1AAAAAAAA4CZRo+1pPj4+mjx5srZv36727durbt26FtdfeOGFWikOAAAAAAAA1lGj0GjRokVycnLS5s2btXnzZotrJpOJ0AgAAAAAAOAWV6PQqKCgoLbrAAAAAAAAwE2k2qHR2LFjNWPGDDk6Omrs2LFV3mcymTRnzpxaKQ4AAAAAAADWUe3QKDs7W+fPnzf/c1VMJtO1VwUAAAAAAACrqnZo9OWXX1b6zwAAAAAAALj91LF2AQAAAAAAALj5EBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0Qo1s2rRJJpNJJ0+elCQtXbpUDRo0MF+fNm2a7r33XqvUdj21bNlSb7zxhvncZDJp3bp1VqsHAAAAAIDrhdDoGhw5ckQxMTHy8fFRvXr15O7uruDgYM2fP1/du3dXaGiooU9ycrIaNGigQ4cOmYOXy4e7u7vCw8O1f/9+iz6pqamysbFRdHT0jXq0K3rwwQdVVFQkV1dXa5eiRYsWqVu3bnJxcbEIsgAAAAAAQM0RGtXQ/v371bFjR6WlpWnmzJnKzs7Wtm3bFBcXp48//liRkZHKyMjQwoULzX0KCgoUFxenxMREtWjRwtyen5+vw4cPa/Xq1fr+++/Vt29fXbx40Xw9JSVFcXFxSk1N1ZkzZ27oc1bm/PnzsrOzU9OmTWUymaxdjk6dOqVevXrpxRdftHYpAAAAAADcNgiNamjkyJGytbXVjh07NHDgQPn5+cnLy0uPPvqoNmzYoIiICCUkJCg2NlYFBQWqqKhQVFSUHn74YQ0ZMsRirCZNmqhZs2bq2rWrpkyZoj179mjfvn2SLgVNW7du1cSJE9WmTRutXbv2qup8++235enpKQcHB/Xv319z58612EYmSfPnz5e3t7fs7Ozk6+urFStWWFw3mUyaP3++HnnkETk6OurVV181bE+7km+++UYhISFq1KiRXF1d9dBDD+nbb7+9qmepypgxYzRx4kQ98MAD1e7TrVs3jRo1SqNGjZKrq6saNWqkyZMnq6KiwnzPzz//rL59+6p+/fpq1aqV3n333UrHKioqUu/evVW/fn15eXnp/fffv+ZnAgAAAADA2giNaqC4uFhpaWmKjo6Wo6NjpfeYTCZFRkaqR48eGjZsmJKSkrR7926LlUeVqV+/viTp3LlzkqQlS5aoT58+cnV11eDBg5WSklLtOtPT0/Xcc88pJiZGOTk5CgkJ0auvvmpxzwcffKCYmBiNGzdOu3fv1ogRI/T000/ryy+/tLhv2rRp6t+/v7777jsNGzas2jVc9ssvvygyMlJff/21tm/frtatWyssLEy//PLLVY9VW5YtWyZbW1tlZmYqISFBc+fO1TvvvGO+PnToUB08eFBffvml3n//fSUnJ+vnn382jDN58mSFh4dr586d+tvf/qZBgwYpNze30jnPnj2r0tJSiwMAAAAAgJuRrbULuBXt27dPFRUV8vX1tWhv1KiReftYdHS0/vnPf2rRokXy9/fXV199pTVr1qhx48ZVjltUVKT4+Hh5eHjI19dX5eXlWrp0qRITEyVJgwYN0rhx41RQUKBWrVpdsc7ExET17t1bsbGxkqQ2bdpo69at+vjjj833xMfHa+jQoRo5cqQkaezYsdq+fbvi4+P1l7/8xXzfU089paefftp8/vv3Ll1J9+7dLc4XLVqkBg0aaPPmzfrrX/96VWPVFk9PT82bN08mk0m+vr767rvvNG/ePA0fPlx79+7Vp59+qszMTHXu3FnSpW2Cfn5+hnEef/xxPfPMM5KkGTNmaOPGjUpMTFRycrLh3tdee03Tp0+/vg8GAAAAAEAtYKVRLcrMzFROTo78/f119uxZSZe2no0YMUJ+fn7q169fpf1atGghR0dHNW/eXGVlZVqzZo3s7Oy0ceNGlZWVKSwsTNKlUCokJESLFy+uVj35+fnq0qWLRdvvz3NzcxUcHGzRFhwcbFgpExgYWK05q3L06FENHz5crVu3lqurq1xcXPTrr7+qsLDwmsa9Fg888IDFO5mCgoL0ww8/6OLFi8rNzZWtra06depkvt62bVvD1r7L/X5/XtVKo0mTJqmkpMR8HDx4sHYeBgAAAACAWsZKoxrw8fGRyWRSfn6+RbuXl5ek/24xu8zW1la2tlV/1Fu2bJGLi4uaNGkiZ2dnc3tKSoqOHz9uMV55ebl27dql6dOnq06dG5f5VbUNr7oiIyNVXFyshIQE3X333bK3t1dQUJB5G96dwt7eXvb29tYuAwAAAACAK2KlUQ24ubkpJCRESUlJKisru+bxWrVqJW9vb4vAqLi4WB9++KFWrlypnJwc85Gdna0TJ04oLS3tiuP6+vrqm2++sWj7/bmfn5/S09Mt2tLT09WuXbtreCKj9PR0vfDCCwoLC5O/v7/s7e117NixWp3jamVkZFicX37Xko2Njdq2basLFy4oKyvLfD0/P7/SF39v377dcF7ZNjYAAAAAAG4lrDSqoeTkZAUHByswMFDTpk1Thw4dVKdOHX3zzTfKy8uz2NZUEytWrJCbm5sGDhxo+Fn7sLAwpaSkqFevXn84xujRo9W1a1fNnTtXffv21f/+7//q008/tRhv/PjxGjhwoDp27KiePXvqo48+0tq1a/Xvf//7mur/vdatW2vFihUKDAxUaWmpxo8fb1iRVVNHjhzRkSNHzL84991338nZ2Vl33XWXGjZsKEnq0aOH+vfvr1GjRpn7FRYWauzYsRoxYoS+/fZbJSYmas6cOZIuBW69evXSiBEjNH/+fNna2mrMmDGV1rx69WoFBgbq//2//6d3331XmZmZV/XCcgAAAAAAbkasNKohb29vZWdnq2fPnpo0aZICAgIUGBioxMRExcbGasaMGdc0/uLFi9W/f39DYCRJ4eHhWr9+/RVX6gQHB2vBggWaO3euAgIC9Nlnn+nvf/+76tWrZ76nX79+SkhIUHx8vPz9/bVw4UItWbJE3bp1u6b6fy8lJUUnTpzQfffdpyFDhuiFF15QkyZNamXsBQsWqGPHjho+fLgkqWvXrurYsaPWr19vvufHH380fF4RERE6ffq0unTpoujoaMXExOjZZ581X1+yZImaN2+uhx56SAMGDNCzzz5bac3Tp0/XypUr1aFDBy1fvlypqam1vlILAAAAAIAbzVRRUVFh7SJw4wwfPlx5eXnasmWLtUuxqm7duunee+/VG2+8YdU6SktL5erqqpKSErm4uFi1FtzaOo1fbu0SbjpZsyOsXQIAAABw07mav0PZnnabi4+PV0hIiBwdHfXpp59q2bJllf4UPAAAAAAAwG+xPe0W1rt3bzk5OVV6zJw5U5KUmZmpkJAQtW/fXgsWLNCbb76pZ555xsqVW3r33XerfI5WrVpVec3f39/apQMAAAAAcNtipdEt7J133tHp06crvXb5BdCrVq26kSXVyCOPPKL777+/0mt169bV+fPnq7xWU5s2bapxXwAAAAAA7gSERrcwDw8Pa5dQK5ydneXs7GztMgAAAAAAwG+wPQ0AAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADGytXQAA4NplzY6wdgkAAAAAbjOsNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwMDW2gUAAG5dncYvt3YJVcqaHWHtEgAAAIBbGiuNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBodJtYunSpGjRoYO0ybnsmk0nr1q2TJB04cEAmk0k5OTlWrQkAAAAAgOuB0Kiajhw5opiYGPn4+KhevXpyd3dXcHCw5s+fr+7duys0NNTQJzk5WQ0aNNChQ4e0adMmmUwm8+Hu7q7w8HDt37/fok9qaqpsbGwUHR19ox7tlnfmzBlFR0fLzc1NTk5OCg8P19GjR61dFgAAAAAAtzRCo2rYv3+/OnbsqLS0NM2cOVPZ2dnatm2b4uLi9PHHHysyMlIZGRlauHChuU9BQYHi4uKUmJioFi1amNvz8/N1+PBhrV69Wt9//7369u2rixcvmq+npKQoLi5OqampOnPmzA19zlvV3//+d3300UdavXq1Nm/erMOHD2vAgAHWLgsAAAAAgFsaoVE1jBw5Ura2ttqxY4cGDhwoPz8/eXl56dFHH9WGDRsUERGhhIQExcbGqqCgQBUVFYqKitLDDz+sIUOGWIzVpEkTNWvWTF27dtWUKVO0Z88e7du3T9KloGnr1q2aOHGi2rRpo7Vr1151rZ9//rn8/Pzk5OSkXr16qaioyHytW7duGjNmjMX9/fr109ChQ83nLVu21CuvvKKIiAg5OTnp7rvv1vr16/Wf//xHjz76qJycnNShQwft2LHD3Ke4uFhPPvmkPDw85ODgoPbt2ys1NdVinm7duumFF15QXFycGjZsqKZNm2ratGlX/Xy/V1JSopSUFM2dO1fdu3dXp06dtGTJEm3dulXbt2+vsl/Lli01Y8YMPfnkk3J0dJSHh4feeusti3t++OEHde3aVfXq1VO7du20cePGSsfKy8vTgw8+qHr16umee+7R5s2br/m5AAAAAACwNkKjKyguLlZaWpqio6Pl6OhY6T0mk0mRkZHq0aOHhg0bpqSkJO3evdti5VFl6tevL0k6d+6cJGnJkiXq06ePXF1dNXjwYKWkpFxVradOnVJ8fLxWrFihr776SoWFhYqNjb2qMSRp3rx5Cg4OVnZ2tvr06aMhQ4YoIiJCgwcP1rfffitvb29FRESooqJC0qXtYZ06ddKGDRu0e/duPfvssxoyZIgyMzMtxl22bJkcHR2VkZGh119/XS+//HKVQUx1ZWVl6fz58+rZs6e5rW3btrrrrru0bdu2P+w7e/ZsBQQEKDs7WxMnTlRMTIy5nvLycg0YMEB2dnbKyMjQggULNGHChErHGT9+vMaNG6fs7GwFBQWpb9++Ki4urvTes2fPqrS01OIAAAAAAOBmRGh0Bfv27VNFRYV8fX0t2hs1aiQnJyc5OTmZw4RFixZp9+7dGjNmjBYtWqTGjRtXOW5RUZHi4+Pl4eEhX19flZeXa+nSpRo8eLAkadCgQfr6669VUFBQ7VrPnz+vBQsWKDAwUPfdd59GjRqlL7744qqfOSwsTCNGjFDr1q01ZcoUlZaWqnPnznr88cfVpk0bTZgwQbm5ueb3Bnl4eCg2Nlb33nuvvLy8NHr0aPXq1UurVq2yGLdDhw6aOnWqWrdurYiICAUGBtaovt86cuSI7OzsDC8Bd3d315EjR/6wb3BwsHlV1+jRo/XYY49p3rx5kqR///vfysvL0/LlyxUQEKCuXbtq5syZlY4zatQohYeHy8/PT/Pnz5erq2uVgd9rr70mV1dX8+Hp6Xn1Dw0AAAAAwA1AaFRDmZmZysnJkb+/v86ePSvp0tazESNGyM/PT/369au0X4sWLeTo6KjmzZurrKxMa9askZ2dnTZu3KiysjKFhYVJuhRKhYSEaPHixdWuycHBQd7e3ubzZs2a6eeff77qZ+vQoYP5n93d3SVJ7du3N7RdHvvixYuaMWOG2rdvr4YNG8rJyUmff/65CgsLqxz3WuqrLUFBQYbz3NxcSVJubq48PT3VvHnzKu+vrN3W1laBgYHmcX5v0qRJKikpMR8HDx681scAAAAAAOC6sLV2ATc7Hx8fmUwm5efnW7R7eXlJ+u8Ws8tsbW1la1v1x7plyxa5uLioSZMmcnZ2NrenpKTo+PHjFuOVl5dr165dmj59uurUuXK+V7duXYtzk8lk3kImSXXq1LE4ly6tTvqjcUwmU5Vt5eXlki5t80pISNAbb7yh9u3by9HRUWPGjDFvu/uj+i6PUVNNmzbVuXPndPLkSYvVRkePHlXTpk2vaezrwd7eXvb29tYuAwAAAACAK2Kl0RW4ubkpJCRESUlJKisru+bxWrVqJW9vb4vAqLi4WB9++KFWrlypnJwc85Gdna0TJ04oLS3tmueVpMaNG1u8GPvixYvavXv3NY+bnp6uRx99VIMHD1ZAQIC8vLy0d+/eax63Ojp16qS6detabHPLz89XYWFhlSuDLvv9i7K3b98uPz8/SZKfn58OHjxo8XlV9WLt37ZfuHBBWVlZ5nEAAAAAALhVERpVQ3Jysi5cuKDAwEC99957ys3NVX5+vv71r38pLy9PNjY21zT+ihUr5ObmpoEDB+qee+4xHwEBAQoLC7vqF2JXpXv37tqwYYM2bNigvLw8Pf/88zp58uQ1j9u6dWtt3LhRW7duVW5urkaMGGF+39H15urqqqioKI0dO1ZffvmlsrKy9PTTTysoKEgPPPCA+b62bdvqgw8+sOibnp6u119/XXv37tVbb72l1atXKyYmRpLUs2dPtWnTRpGRkdq5c6e2bNmif/zjH5XW8NZbb+mDDz5QXl6eoqOjdeLECQ0bNuz6PTQAAAAAADcA29OqwdvbW9nZ2Zo5c6YmTZqkQ4cOyd7eXu3atVNsbKxGjhx5TeMvXrxY/fv3N2/7+q3w8HANGTJEx44dU6NGja5pnmHDhmnnzp2KiIiQra2t/v73v+svf/nLNY0pSS+99JL279+v0NBQOTg46Nlnn1W/fv1UUlJyzWNXx7x581SnTh2Fh4fr7NmzCg0NVXJyssU9+fn5hnrGjRunHTt2aPr06XJxcdHcuXMVGhoq6dJWvg8++EBRUVHq0qWLWrZsqTfffFO9evUyzD9r1izNmjVLOTk58vHx0fr166/5uwIAAAAAwNpMFb9/yQ1wB2jZsqXGjBmjMWPGWLWO0tJSubq6qqSkRC4uLlatBaiJTuOXW7uEKmXNjrB2CQAAAMBN52r+DmV7GgAAAAAAAAwIjW4RvXv3lpOTU6XHzJkzrV3eNXn33XerfLZWrVpVec3f39/apQMAAAAAcNvinUa3iHfeeUenT5+u9FrDhg1vcDW165FHHtH9999f6bW6devq/PnzVV6rqQMHDtS4LwAAAAAAdwJCo1uEh4eHtUu4bpydneXs7GztMgAAAAAAwG+wPQ0AAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCAAAAAACAAaERAAAAAAAADGytXQAA4NaVNTvC2iUAAAAAuE5YaQQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgIGttQsAAKAmOo1f/ofXs2ZH3KBKAAAAgNsTK40AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGh0m1i6dKkaNGhg7TJuey1bttQbb7xhPjeZTFq3bp3V6gEAAAAA4HohNKqmI0eOKCYmRj4+PqpXr57c3d0VHBys+fPnq3v37goNDTX0SU5OVoMGDXTo0CFt2rRJJpPJfLi7uys8PFz79++36JOamiobGxtFR0ffqEe75S1atEjdunWTi4uLTCaTTp48ae2SAAAAAAC45REaVcP+/fvVsWNHpaWlaebMmcrOzta2bdsUFxenjz/+WJGRkcrIyNDChQvNfQoKChQXF6fExES1aNHC3J6fn6/Dhw9r9erV+v7779W3b19dvHjRfD0lJUVxcXFKTU3VmTNnbuhz3qpOnTqlXr166cUXX7R2KQAAAAAA3DYIjaph5MiRsrW11Y4dOzRw4ED5+fnJy8tLjz76qDZs2KCIiAglJCQoNjZWBQUFqqioUFRUlB5++GENGTLEYqwmTZqoWbNm6tq1q6ZMmaI9e/Zo3759ki4FTVu3btXEiRPVpk0brV279qpr/fzzz+Xn5ycnJyf16tVLRUVF5mvdunXTmDFjLO7v16+fhg4daj5v2bKlXnnlFUVERMjJyUl333231q9fr//85z969NFH5eTkpA4dOmjHjh3mPsXFxXryySfl4eEhBwcHtW/fXqmpqRbzdOvWTS+88ILi4uLUsGFDNW3aVNOmTbvq56vMmDFjNHHiRD3wwAPV7tOtWzeNGjVKo0aNkqurqxo1aqTJkyeroqLCfM/PP/+svn37qn79+mrVqpXefffdSscqKipS7969Vb9+fXl5een999+/5mcCAAAAAMDaCI2uoLi4WGlpaYqOjpajo2Ol95hMJkVGRqpHjx4aNmyYkpKStHv3bouVR5WpX7++JOncuXOSpCVLlqhPnz5ydXXV4MGDlZKSclW1njp1SvHx8VqxYoW++uorFRYWKjY29qrGkKR58+YpODhY2dnZ6tOnj4YMGaKIiAgNHjxY3377rby9vRUREWEOWM6cOaNOnTppw4YN2r17t5599lkNGTJEmZmZFuMuW7ZMjo6OysjI0Ouvv66XX35ZGzduvOr6asuyZctka2urzMxMJSQkaO7cuXrnnXfM14cOHaqDBw/qyy+/1Pvvv6/k5GT9/PPPhnEmT56s8PBw7dy5U3/72980aNAg5ebmVjrn2bNnVVpaanEAAAAAAHAzIjS6gn379qmiokK+vr4W7Y0aNZKTk5OcnJw0YcIESZferbN7926NGTNGixYtUuPGjasct6ioSPHx8fLw8JCvr6/Ky8u1dOlSDR48WJI0aNAgff311yooKKh2refPn9eCBQsUGBio++67T6NGjdIXX3xx1c8cFhamESNGqHXr1poyZYpKS0vVuXNnPf7442rTpo0mTJig3NxcHT16VJLk4eGh2NhY3XvvvfLy8tLo0aPVq1cvrVq1ymLcDh06aOrUqWrdurUiIiIUGBhYo/pqi6enp+bNmydfX1/97W9/0+jRozVv3jxJ0t69e/Xpp5/q7bff1gMPPKBOnTopJSVFp0+fNozz+OOP65lnnlGbNm00Y8YMBQYGKjExsdI5X3vtNbm6upoPT0/P6/qMAAAAAADUFKFRDWVmZionJ0f+/v46e/aspEtbz0aMGCE/Pz/169ev0n4tWrSQo6OjmjdvrrKyMq1Zs0Z2dnbauHGjysrKFBYWJulSKBUSEqLFixdXuyYHBwd5e3ubz5s1a1bpypgr6dChg/mf3d3dJUnt27c3tF0e++LFi5oxY4bat2+vhg0bysnJSZ9//rkKCwurHPda6qstDzzwgEwmk/k8KChIP/zwgy5evKjc3FzZ2tqqU6dO5utt27at9BfqgoKCDOdVrTSaNGmSSkpKzMfBgwdr52EAAAAAAKhlttYu4Gbn4+Mjk8mk/Px8i3YvLy9J/91idpmtra1sbav+WLds2SIXFxc1adJEzs7O5vaUlBQdP37cYrzy8nLt2rVL06dPV506V8736tata3FuMpks3tFTp04di3Pp0uqkPxrncqhSWVt5ebkkafbs2UpISNAbb7yh9u3by9HRUWPGjDFvu/uj+i6Pcaewt7eXvb29tcsAAAAAAOCKWGl0BW5ubgoJCVFSUpLKysquebxWrVrJ29vbIjAqLi7Whx9+qJUrVyonJ8d8ZGdn68SJE0pLS7vmeSWpcePGFi/Gvnjxonbv3n3N46anp+vRRx/V4MGDFRAQIC8vL+3du/eax73eMjIyLM63b9+u1q1by8bGRm3bttWFCxeUlZVlvp6fn6+TJ08axtm+fbvh3M/P77rUDAAAAADAjUJoVA3Jycm6cOGCAgMD9d577yk3N1f5+fn617/+pby8PNnY2FzT+CtWrJCbm5sGDhyoe+65x3wEBAQoLCzsql+IXZXu3btrw4YN2rBhg/Ly8vT8889XGoJcrdatW2vjxo3aunWrcnNzNWLECPP7jm6EI0eOKCcnx/wrdN99951ycnJ0/Phx8z09evRQUlKSRb/CwkKNHTtW+fn5Sk1NVWJiomJiYiRJvr6+6tWrl0aMGKGMjAxlZWXpmWeeMawsk6TVq1dr8eLF2rt3r6ZOnarMzEyNGjXqOj4xAAAAAADXH6FRNXh7eys7O1s9e/bUpEmTFBAQYH7ZcWxsrGbMmHFN4y9evFj9+/e3eL/OZeHh4Vq/fr2OHTt2TXNI0rBhwxQZGamIiAg99NBD8vLy0l/+8pdrHvell17Sfffdp9DQUHXr1k1Nmzat8p1O18OCBQvUsWNHDR8+XJLUtWtXdezYUevXrzff8+OPPxo+w4iICJ0+fVpdunRRdHS0YmJi9Oyzz5qvL1myRM2bN9dDDz2kAQMG6Nlnn1WTJk0M80+fPl0rV65Uhw4dtHz5cqWmpqpdu3bX6WkBAAAAALgxTBW/f8kNcAfo1q2b7r33Xr3xxhtWraO0tFSurq4qKSmRi4uLVWsBbjWdxi//w+tZsyNuUCUAAADAreNq/g5lpREAAAAAAAAMCI1uEb1795aTk1Olx8yZM61d3jV59913q3y2Vq1aVXnN39/f2qUDAAAAAHDbqvq34XFTeeedd3T69OlKrzVs2PAGV1O7HnnkEd1///2VXqtbt67Onz9f5bWa2rRpU437AgAAAABwJyA0ukV4eHhYu4TrxtnZWc7OztYuAwAAAAAA/Abb0wAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAwNbaBQAAUBNZsyOsXQIAAABwW2OlEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABrbWLgAAgOuh0/jl1i4BAAAAd5is2RHWLqFWsdIIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGAAAAAAAAMCA0AgAAAAAAgAGhEQAAAAAAAAwIjQAAAAAAAGBAaAQAAAAAAAADQiMAAAAAAAAYEBoBAAAAAADAgNAIAAAAAAAABoRGQDVNmzZN9957r/l86NCh6tevn9XqAQAAAADgerpjQqMjR44oJiZGPj4+qlevntzd3RUcHKz58+ere/fuCg0NNfRJTk5WgwYNdOjQIW3atEkmk8l8uLu7Kzw8XPv377fok5qaKhsbG0VHR9+oR7updOvWTWPGjLlh833//fcKDw9Xy5YtZTKZ9MYbb9ywuQEAAAAAuJ3dEaHR/v371bFjR6WlpWnmzJnKzs7Wtm3bFBcXp48//liRkZHKyMjQwoULzX0KCgoUFxenxMREtWjRwtyen5+vw4cPa/Xq1fr+++/Vt29fXbx40Xw9JSVFcXFxSk1N1ZkzZ27oc1ZHRUWFLly4YO0yas2pU6fk5eWlWbNmqWnTptYuBwAAAACA28YdERqNHDlStra22rFjhwYOHCg/Pz95eXnp0Ucf1YYNGxQREaGEhATFxsaqoKBAFRUVioqK0sMPP6whQ4ZYjNWkSRM1a9ZMXbt21ZQpU7Rnzx7t27dP0qWgaevWrZo4caLatGmjtWvXVrvGpUuXqkGDBvr444/l6+srBwcHPfbYYzp16pSWLVumli1b6k9/+pNeeOEFi5BqxYoVCgwMlLOzs5o2baqnnnpKP//8s/n65RVSn376qTp16iR7e3t9/fXX6tatm0aPHq0xY8boT3/6k9zd3fX222+rrKxMTz/9tJydneXj46NPP/3Uos7NmzerS5cusre3V7NmzTRx4kRzCDV06FBt3rxZCQkJ5hVZBw4c0MWLFxUVFaVWrVqpfv368vX1VUJCwlV/j5Xp3LmzZs+erUGDBsne3r5afS5/1uvWrVPr1q1Vr149hYaG6uDBgxb3zZo1S+7u7nJ2dlZUVFSVIeD06dPVuHFjubi46LnnntO5c+eu+bkAAAAAALC22z40Ki4uVlpamqKjo+Xo6FjpPSaTSZGRkerRo4eGDRumpKQk7d6922LlUWXq168vSeaQYMmSJerTp49cXV01ePBgpaSkXFWtp06d0ptvvqmVK1fqs88+06ZNm9S/f3998skn+uSTT7RixQotXLhQ77//vrnP+fPnNWPGDO3cuVPr1q3TgQMHNHToUMPYEydO1KxZs5Sbm6sOHTpIkpYtW6ZGjRopMzNTo0eP1vPPP6/HH39cDz74oL799ltzaHbq1ClJ0k8//aSwsDB17txZO3fu1Pz585WSkqJXXnlFkpSQkKCgoCANHz5cRUVFKioqkqenp8rLy9WiRQutXr1ae/bs0ZQpU/Tiiy9q1apVV/X51KZTp07p1Vdf1fLly5Wenq6TJ09q0KBB5uurVq3StGnTNHPmTO3YsUPNmjVTcnKyYZwvvvhCubm52rRpk1JTU7V27VpNnz69ynnPnj2r0tJSiwMAAAAAgJuRrbULuN727duniooK+fr6WrQ3atTIvHIkOjpa//znP7Vo0SL5+/vrq6++0po1a9S4ceMqxy0qKlJ8fLw8PDzk6+ur8vJyLV26VImJiZKkQYMGady4cSooKFCrVq2qVev58+c1f/58eXt7S5Iee+wxrVixQkePHpWTk5PatWunv/zlL/ryyy/1xBNPSJKGDRtm7u/l5aU333xTnTt31q+//ionJyfztZdfflkhISEW8wUEBOill16SJE2aNEmzZs1So0aNNHz4cEnSlClTNH/+fO3atUsPPPCAkpOT5enpqaSkJJlMJrVt21aHDx/WhAkTNGXKFLm6usrOzk4ODg4WW8VsbGwsgpRWrVpp27ZtWrVqlQYOHFitz6a2nT9/XklJSbr//vslXQrQ/Pz8lJmZqS5duuiNN95QVFSUoqKiJEmvvPKK/v3vfxtWG9nZ2Wnx4sVycHCQv7+/Xn75ZY0fP14zZsxQnTrGTPa11177w1AJAAAAAICbxW2/0qgqmZmZysnJkb+/v86ePSvp0tazESNGyM/Pr8pfxWrRooUcHR3VvHlzlZWVac2aNbKzs9PGjRtVVlamsLAwSZdCqZCQEC1evLjaNTk4OJgDI0lyd3dXy5YtLcIfd3d3i+1nWVlZ6tu3r+666y45OzvroYcekiQVFhZajB0YGGiY7/KKI+lSsOPm5qb27dtbzCXJPF9ubq6CgoJkMpnM9wQHB+vXX3/VoUOH/vDZ3nrrLXXq1EmNGzeWk5OTFi1aZKjxRrK1tVXnzp3N523btlWDBg2Um5sr6dKzXg6ULgsKCjKMExAQIAcHB4t7fv31V8NWt8smTZqkkpIS81HVfQAAAAAAWNttv9LIx8dHJpNJ+fn5Fu1eXl6S/rvF7DJbW1vZ2lb9sWzZskUuLi5q0qSJnJ2dze0pKSk6fvy4xXjl5eXatWuXpk+fXumqk9+rW7euxbnJZKq0rby8XJJUVlam0NBQhYaG6t1331Xjxo1VWFio0NBQw3t1Ktuad6X5LodDl+erqZUrVyo2NlZz5sxRUFCQnJ2dNXv2bGVkZFzTuLcie3v7ar97CQAAAAAAa7rtVxq5ubkpJCRESUlJKisru+bxWrVqJW9vb4vAqLi4WB9++KFWrlypnJwc85Gdna0TJ04oLS3tmuetTF5enoqLizVr1iz9+c9/Vtu2bS1WIdU2Pz8/bdu2TRUVFea29PR0OTs7m39hzs7OzuJF3ZfvefDBBzVy5Eh17NhRPj4++vHHH69bndVx4cIF7dixw3yen5+vkydPys/PT9KlZ/19qLV9+3bDODt37tTp06ct7nFycpKnp+d1qhwAAAAAgBvjtg+NJCk5OVkXLlxQYGCg3nvvPeXm5io/P1//+te/lJeXJxsbm2saf8WKFXJzc9PAgQN1zz33mI+AgACFhYVd9Quxq+uuu+6SnZ2dEhMTtX//fq1fv14zZsy4LnNJl36F7uDBgxo9erTy8vL04YcfaurUqRo7dqx5JVXLli2VkZGhAwcO6NixYyovL1fr1q21Y8cOff7559q7d68mT56sb775plZqOnfunDmkO3funH766Sfl5OSYf9FOkpKSktSjRw+LfnXr1tXo0aOVkZGhrKwsDR06VA888IC6dOkiSYqJidHixYu1ZMkS7d27V1OnTtX3339f6fxRUVHas2ePPvnkE02dOlWjRo2q1soyAAAAAABuZnfEX7be3t7Kzs5Wz549NWnSJAUEBCgwMFCJiYmKjY295qBl8eLF6t+/v8W7fi4LDw/X+vXrdezYsWuaozKNGzfW0qVLtXr1arVr106zZs1SfHx8rc9zmYeHhz755BNlZmYqICBAzz33nKKioswv05ak2NhY2djYqF27dubtciNGjNCAAQP0xBNP6P7771dxcbFGjhxZKzUdPnxYHTt2VMeOHc0vJ+/YsaOeeeYZ8z3Hjh0zrGxycHDQhAkT9NRTTyk4OFhOTk567733zNefeOIJTZ48WXFxcerUqZP+7//+T88//7xh/h49eqh169bq2rWrnnjiCT3yyCOaNm1arTwbAAAAAADWZKr47V4j4A6wdOlSjRkzRidPnrR2KSotLZWrq6tKSkrk4uJi7XKA20qn8cutXQIAAADuMFmzI6xdwhVdzd+hd8RKIwAAAAAAAFwdQqMbpHfv3nJycqr0mDlzprXLs6qqPhcnJyfZ2NhUeW3Lli3WLh0AAAAAgNtW1b8tj1r1zjvvWPzK1m81bNjwBldzc8nJyany2unTp1W/fv1Kr3l4eNRovqFDh2ro0KE16gsAAAAAwJ2C0OgGqWnAcSfw8fGxdgkAAAAAAOB32J4GAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAaERgAAAAAAADAgNAIAAAAAAIABoREAAAAAAAAMCI0AAAAAAABgQGgEAAAAAAAAA0IjAAAAAAAAGBAaAQAAAAAAwIDQCAAAAAAAAAa21i4AAIDrIWt2hLVLAAAAAG5prDQCAAAAAACAAaERAAAAAAAADAiNAAAAAAAAYEBoBAAAAAAAAANCIwAAAAAAABgQGgEAAAAAAMDA1toFAHeyiooKSVJpaamVKwEAAAAA3Aku//15+e/RP0JoBFjRL7/8Ikny9PS0ciUAAAAAgDvJL7/8IldX1z+8x1RRnWgJwHVRXl6uw4cPy9nZWSaTydrl3JZKS0vl6empgwcPysXFxdrl4Abhe78z8b3fmfje71x893cmvvc7E9977aqoqNAvv/yi5s2bq06dP35rESuNACuqU6eOWrRoYe0y7gguLi78H8wdiO/9zsT3fmfie79z8d3fmfje70x877XnSiuMLuNF2AAAAAAAADAgNAIAAAAAAIABoRGA25q9vb2mTp0qe3t7a5eCG4jv/c7E935n4nu/c/Hd35n43u9MfO/Ww4uwAQAAAAAAYMBKIwAAAAAAABgQGgEAAAAAAMCA0AgAAAAAAAAGhEYAAAAAAAAwIDQCcNtq2bKlTCaTxTFr1iyLe3bt2qU///nPqlevnjw9PfX6669bqVrUlrfeekstW7ZUvXr1dP/99yszM9PaJaEWTZs2zfDvddu2bc3Xz5w5o+joaLm5ucnJyUnh4eE6evSoFStGTX311Vfq27evmjdvLpPJpHXr1llcr6io0JQpU9SsWTPVr19fPXv21A8//GBxz/Hjx/W3v/1NLi4uatCggaKiovTrr7/ewKfA1brS9z506FDDfwN69eplcQ/f+63ntddeU+fOneXs7KwmTZqoX79+ys/Pt7inOv99LywsVJ8+feTg4KAmTZpo/PjxunDhwo18FFyF6nzv3bp1M/w7/9xzz1ncw/d+fREaAbitvfzyyyoqKjIfo0ePNl8rLS3Vww8/rLvvvltZWVmaPXu2pk2bpkWLFlmxYlyL9957T2PHjtXUqVP17bffKiAgQKGh/7+9ew+KqnzjAP4FZFcUuckKqIggyk0k8cJsFjbBqFgzGDOK5uQlExUsTWy8azaO+svU1NTumVOjqWmUoyWh4CjrKgh4AUkIXE02EkXFS1z2+f3ReKYNVCRggb6fmTPDOe/7nn1en3MO68PZs8NRWlpq6dCoEQUFBZmd18eOHVPa3nzzTfzwww/YvXs30tLScPXqVcTExFgwWmqoO3fuICQkBJs3b66z/d1338XGjRvx4YcfQq/Xo2PHjhg+fDju37+v9Bk/fjzOnz+P5ORk7N+/H0ePHkVcXFxzTYEa4HF5B4ARI0aYXQN27Nhh1s68tz5paWlISEjAiRMnkJycjKqqKgwbNgx37txR+jzu+l5TU4MXXngBlZWVSE9Px5dffolt27Zh6dKllpgS1UN98g4AU6dONTvn//5HXua9GQgRURvl5eUl69evf2j7li1bxNnZWf78809l27x588TPz68ZoqOmMHjwYElISFDWa2pqpGvXrrJq1SoLRkWNadmyZRISElJnW3l5udja2sru3buVbXl5eQJAdDpdM0VITQGA7Nu3T1k3mUzi7u4ua9asUbaVl5eLWq2WHTt2iIhIbm6uAJBTp04pfQ4ePChWVlby22+/NVvs1HD/zLuIyMSJEyU6OvqhY5j3tqG0tFQASFpamojU7/p+4MABsba2FqPRqPTZunWrODg4mL3Xo5brn3kXERk6dKjMmjXroWOY96bHO42IqE1bvXo1OnfujP79+2PNmjVmt6rqdDqEh4dDpVIp24YPH478/HzcuHHDEuHSv1BZWYnMzExERkYq26ytrREZGQmdTmfByKixXbx4EV27doWPjw/Gjx8Pg8EAAMjMzERVVZXZMeDv748ePXrwGGhjioqKYDQazXLt6OiIsLAwJdc6nQ5OTk4YOHCg0icyMhLW1tbQ6/XNHjM1ntTUVHTp0gV+fn6YMWMGysrKlDbmvW24efMmAMDFxQVA/a7vOp0OwcHBcHNzU/oMHz4ct27dwvnz55sxemqof+b9ga+//hqurq7o27cvFixYgLt37yptzHvTa2fpAIiImsobb7yB0NBQuLi4ID09HQsWLEBJSQnWrVsHADAajfD29jYb8+AXjtFohLOzc7PHTA137do11NTUmL1pAP7K6YULFywUFTW2sLAwbNu2DX5+figpKcHy5cvx7LPP4ty5czAajVCpVHBycjIb4+bmBqPRaJmAqUk8yGdd5/uDNqPRiC5dupi1t2vXDi4uLjweWrERI0YgJiYG3t7eKCwsxMKFCxEVFQWdTgcbGxvmvQ0wmUyYPXs2hgwZgr59+wJAva7vRqOxzmvCgzZq2erKOwC8/PLL8PLyQteuXXHmzBnMmzcP+fn52Lt3LwDmvTmwaERErcr8+fPxv//975F98vLy4O/vjzlz5ijb+vXrB5VKhWnTpmHVqlVQq9VNHSoRNYGoqCjl5379+iEsLAxeXl7YtWsX7OzsLBgZETWHsWPHKj8HBwejX79+6NWrF1JTUxEREWHByKixJCQk4Ny5c2bPq6O272F5//vzyIKDg+Hh4YGIiAgUFhaiV69ezR3mfxI/nkZErUpiYiLy8vIeufj4+NQ5NiwsDNXV1SguLgYAuLu71/rWjQfr7u7uTToPanyurq6wsbGpM6fMZ9vl5OSEPn36oKCgAO7u7qisrER5eblZHx4Dbc+DfD7qfHd3d6/1EPzq6mpcv36dx0Mb4uPjA1dXVxQUFABg3lu7mTNnYv/+/Thy5Ai6d++ubK/P9Z3v61qvh+W9LmFhYQBgds4z702LRSMialU0Gg38/f0fufz9GUV/l52dDWtra+W2da1Wi6NHj6Kqqkrpk5ycDD8/P340rRVSqVQYMGAAUlJSlG0mkwkpKSnQarUWjIyaUkVFBQoLC+Hh4YEBAwbA1tbW7BjIz8+HwWDgMdDGeHt7w93d3SzXt27dgl6vV3Kt1WpRXl6OzMxMpc/hw4dhMpmU/3RQ63flyhWUlZXBw8MDAPPeWokIZs6ciX379uHw4cO1Hh9Qn+u7VqvF2bNnzYqGycnJcHBwQGBgYPNMhJ7I4/Jel+zsbAAwO+eZ9yZm6SdxExE1hfT0dFm/fr1kZ2dLYWGhfPXVV6LRaGTChAlKn/LycnFzc5NXXnlFzp07Jzt37pQOHTrIRx99ZMHI6d/YuXOnqNVq2bZtm+Tm5kpcXJw4OTmZfaMGtW6JiYmSmpoqRUVFcvz4cYmMjBRXV1cpLS0VEZHp06dLjx495PDhw5KRkSFarVa0Wq2Fo6aGuH37tmRlZUlWVpYAkHXr1klWVpZcunRJRERWr14tTk5OkpSUJGfOnJHo6Gjx9vaWe/fuKfsYMWKE9O/fX/R6vRw7dkx69+4t48aNs9SUqB4elffbt2/L3LlzRafTSVFRkfz8888SGhoqvXv3lvv37yv7YN5bnxkzZoijo6OkpqZKSUmJsty9e1fp87jre3V1tfTt21eGDRsm2dnZ8uOPP4pGo5EFCxZYYkpUD4/Le0FBgbzzzjuSkZEhRUVFkpSUJD4+PhIeHq7sg3lveiwaEVGblJmZKWFhYeLo6Cjt27eXgIAAWblypdmbShGRnJwceeaZZ0StVku3bt1k9erVFoqYGsumTZukR48eolKpZPDgwXLixAlLh0SNKDY2Vjw8PESlUkm3bt0kNjZWCgoKlPZ79+5JfHy8ODs7S4cOHeSll16SkpISC0ZMDXXkyBEBUGuZOHGiiIiYTCZZsmSJuLm5iVqtloiICMnPzzfbR1lZmYwbN07s7e3FwcFBJk+eLLdv37bAbKi+HpX3u3fvyrBhw0Sj0Yitra14eXnJ1KlTa/1hgHlvferKOQD54osvlD71ub4XFxdLVFSU2NnZiaurqyQmJkpVVVUzz4bq63F5NxgMEh4eLi4uLqJWq8XX11feeustuXnzptl+mPemZSUi0nz3NRERERERERERUWvAZxoREREREREREVEtLBoREREREREREVEtLBoREREREREREVEtLBoREREREREREVEtLBoREREREREREVEtLBoREREREREREVEtLBoREREREREREVEtLBoRERERUYszadIkjBo1ytJhEBER/aexaERERERE9fLcc89h9uzZTT6GiIiIWgYWjYiIiIiI6qmystLSIRARETUbFo2IiIiI6LEmTZqEtLQ0bNiwAVZWVrCyskJxcTHS0tIwePBgqNVqeHh4YP78+aiurn7kmJqaGkyZMgXe3t6ws7ODn58fNmzY0ODY9uzZg+DgYNjZ2aFz586IjIzEnTt3lPbPP/8cQUFBSowzZ85U2gwGA6Kjo2Fvbw8HBweMGTMGv//+u9L+9ttv46mnnsKnn34Kb29vtG/fHgBQXl6O1157DRqNBg4ODnj++eeRk5PT4DkQERG1RCwaEREREdFjbdiwAVqtFlOnTkVJSQlKSkpga2uLkSNHYtCgQcjJycHWrVvx2WefYcWKFQ8d4+npCZPJhO7du2P37t3Izc3F0qVLsXDhQuzateuJ4yopKcG4cePw6quvIi8vD6mpqYiJiYGIAAC2bt2KhIQExMXF4ezZs/j+++/h6+sLADCZTIiOjsb169eRlpaG5ORk/Prrr4iNjTV7jYKCAnz77bfYu3cvsrOzAQCjR49GaWkpDh48iMzMTISGhiIiIgLXr1//F//KRERELUs7SwdARERERC2fo6MjVCoVOnToAHd3dwDAokWL4OnpiQ8++ABWVlbw9/fH1atXMW/ePCxdurTOMQBgY2OD5cuXK+ve3t7Q6XTYtWsXxowZ80RxlZSUoLq6GjExMfDy8gIABAcHK+0rVqxAYmIiZs2apWwbNGgQACAlJQVnz55FUVERPD09AQDbt29HUFAQTp06pfSrrKzE9u3bodFoAADHjh3DyZMnUVpaCrVaDQB477338N1332HPnj2Ii4t7ojkQERG1VLzTiIiIiIgaJC8vD1qtFlZWVsq2IUOGoKKiAleuXHnk2M2bN2PAgAHQaDSwt7fHxx9/DIPB8MQxhISEICIiAsHBwRg9ejQ++eQT3LhxAwBQWlqKq1evIiIi4qHxe3p6KgUjAAgMDISTkxPy8vKUbV5eXkrBCABycnJQUVGBzp07w97eXlmKiopQWFj4xHMgIiJqqXinERERERE1q507d2Lu3LlYu3YttFotOnXqhDVr1kCv1z/xvmxsbJCcnIz09HQcOnQImzZtwqJFi6DX6+Hq6too8Xbs2NFsvaKiAh4eHkhNTa3V18nJqVFek4iIqCXgnUZEREREVC8qlQo1NTXKekBAAHQ6nfL8IAA4fvw4OnXqhO7du9c55kGfp59+GvHx8ejfvz98fX3/1R06VlZWGDJkCJYvX46srCyoVCrs27cPnTp1Qs+ePZGSklLnuICAAFy+fBmXL19WtuXm5qK8vByBgYEPfb3Q0FAYjUa0a9cOvr6+ZktjFaqIiIhaAhaNiIiIiKheevbsCb1ej+LiYly7dg3x8fG4fPkyXn/9dVy4cAFJSUlYtmwZ5syZA2tr6zrHmEwm9O7dGxkZGfjpp5/wyy+/YMmSJTh16lSDYtLr9Vi5ciUyMjJgMBiwd+9e/PHHHwgICADw17efrV27Fhs3bsTFixdx+vRpbNq0CQAQGRmJ4OBgjB8/HqdPn8bJkycxYcIEDB06FAMHDnzoa0ZGRkKr1WLUqFE4dOgQiouLkZ6ejkWLFiEjI6NB8yAiImqJWDQiIiIionqZO3cubGxsEBgYCI1Gg6qqKhw4cAAnT55ESEgIpk+fjilTpmDx4sUPHWMwGDBt2jTExMQgNjYWYWFhKCsrQ3x8fINicnBwwNGjRzFy5Ej06dMHixcvxtq1axEVFQUAmDhxIt5//31s2bIFQUFBePHFF3Hx4kUAf92hlJSUBGdnZ4SHhyMyMhI+Pj745ptvHvmaVlZWOHDgAMLDwzF58mT06dMHY8eOxaVLl+Dm5tageRAREbVEVvL3+4mJiIiIiIiIiIjAO42IiIiIiIiIiKgOLBoRERERUYtlMBjMvtb+n4vBYLB0iERERG0WP55GRERERC1WdXU1iouLH9res2dPtGvXrvkCIiIi+g9h0YiIiIiIiIiIiGrhx9OIiIiIiIiIiKgWFo2IiIiIiIiIiKgWFo2IiIiIiIiIiKgWFo2IiIiIiIiIiKgWFo2IiIiIiIiIiKgWFo2IiIiIiIiIiKgWFo2IiIiIiIiIiKgWFo2IiIiIiIiIiKiW/wN3LQRDeDUCnwAAAABJRU5ErkJggg==\n"},"metadata":{}}]},{"cell_type":"markdown","metadata":{"id":"Tc7oifPnbssg"},"source":["4. Finally, we can take a look at our best models using **py3Dmol**"]},{"cell_type":"code","metadata":{"id":"LPpEhymrJ5gM","colab":{"base_uri":"https://localhost:8080/","height":497},"executionInfo":{"status":"ok","timestamp":1729448043466,"user_tz":-120,"elapsed":485,"user":{"displayName":"C√©sar Ram√≠rez-Sarmiento","userId":"07473609926097268448"}},"outputId":"5a9e7b15-b913-41fd-e1cc-b5939496e455"},"source":["#!pip install py3Dmol\n","import py3Dmol\n","view=py3Dmol.view()\n","view.setBackgroundColor('white')\n","#Loading template\n","view.addModel(open('input_files/pdbs/5eh6.pdb', 'r').read(),'pdb')\n","view.addModel(open('GYPA_gorilla__0.pdb', 'r').read(),'pdb')\n","view.setStyle({'chain':'A'},{'cartoon': {'color':'spectrum'}})\n","view.zoomTo()\n","view.show()\n","#The following is a small trick to be able to manipulate (rotate, zoom) your\n","#representation in Google Colab and, once you are happy with the visualization,\n","#press Enter to print out as an onscreen png image file that you can download\n","#input()\n","#view.png()"],"execution_count":49,"outputs":[{"output_type":"display_data","data":{"application/3dmoljs_load.v0":"<div id=\"3dmolviewer_17294480429883013\"  style=\"position: relative; width: 640px; height: 480px;\">\n        <p id=\"3dmolwarning_17294480429883013\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n}\n\nvar viewer_17294480429883013 = null;\nvar warn = document.getElementById(\"3dmolwarning_17294480429883013\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17294480429883013 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17294480429883013\"),{backgroundColor:\"white\"});\nviewer_17294480429883013.zoomTo();\n\tviewer_17294480429883013.setBackgroundColor(\"white\");\n\tviewer_17294480429883013.addModel(\"HEADER    MEMBRANE PROTEIN                        28-OCT-15   5EH6              \\nTITLE     CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE MONOMER IN       \\nTITLE    2 LIPIDIC CUBIC PHASE                                                  \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: GLYCOPHORIN-A;                                             \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 FRAGMENT: UNP RESIDEUS 89-117;                                       \\nCOMPND   5 SYNONYM: MN SIALOGLYCOPROTEIN,PAS-2,SIALOGLYCOPROTEIN ALPHA;         \\nCOMPND   6 ENGINEERED: YES;                                                     \\nCOMPND   7 MUTATION: YES                                                        \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606;                                                \\nSOURCE   5 GENE: GYPA, GPA;                                                     \\nSOURCE   6 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE   7 EXPRESSION_SYSTEM_TAXID: 562                                         \\nKEYWDS    RECEPTOR, LIPIDIC CUBIC PHASE, PEPTIDES, TRANSMEMBRANE, MEMBRANE      \\nKEYWDS   2 PROTEIN                                                              \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    M.J.CALL,M.E.CALL,R.TRENKER                                           \\nREVDAT   5   01-JAN-20 5EH6    1       REMARK                                   \\nREVDAT   4   17-JAN-18 5EH6    1       REMARK                                   \\nREVDAT   3   20-SEP-17 5EH6    1       REMARK                                   \\nREVDAT   2   06-JAN-16 5EH6    1       JRNL                                     \\nREVDAT   1   23-DEC-15 5EH6    0                                                \\nJRNL        AUTH   R.TRENKER,M.E.CALL,M.J.CALL                                  \\nJRNL        TITL   CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE DIMER   \\nJRNL        TITL 2 IN LIPIDIC CUBIC PHASE.                                      \\nJRNL        REF    J.AM.CHEM.SOC.                V. 137 15676 2015              \\nJRNL        REFN                   ESSN 1520-5126                               \\nJRNL        PMID   26642914                                                     \\nJRNL        DOI    10.1021/JACS.5B11354                                         \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.92 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX 1.9_1692                                      \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.92                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 26.60                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 1.340                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 97.5                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2139                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.225                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.224                           \\nREMARK   3   FREE R VALUE                     : 0.238                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 10.000                          \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 214                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 26.5982 -  2.4156    0.96      978   109  0.2219 0.2306        \\nREMARK   3     2  2.4156 -  1.9175    0.99      947   105  0.2312 0.2642        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.11                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.90                                          \\nREMARK   3   K_SOL              : NULL                                          \\nREMARK   3   B_SOL              : NULL                                          \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.150            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 29.650           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.007            199                                  \\nREMARK   3   ANGLE     :  0.890            271                                  \\nREMARK   3   CHIRALITY :  0.035             39                                  \\nREMARK   3   PLANARITY :  0.003             30                                  \\nREMARK   3   DIHEDRAL  :  8.879             65                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 5EH6 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 28-OCT-15.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000214088.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 13-AUG-15                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : 7.5                                \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : AUSTRALIAN SYNCHROTRON             \\nREMARK 200  BEAMLINE                       : MX2                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.9537                             \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315R                  \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : HKL-2000                           \\nREMARK 200  DATA SCALING SOFTWARE          : HKL-2000                           \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 2141                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.918                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 26.600                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 97.6                               \\nREMARK 200  DATA REDUNDANCY                : 9.600                              \\nREMARK 200  R MERGE                    (I) : 0.10850                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 9.9100                             \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 1.92                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 1.99                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : 93.6                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : 8.60                               \\nREMARK 200  R MERGE FOR SHELL          (I) : 0.94000                            \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 2.370                              \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \\nREMARK 200 SOFTWARE USED: PHENIX 1.9_1692                                       \\nREMARK 200 STARTING MODEL: 4WOL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: HEXAGONAL PLATES                                             \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 37.03                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.95                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: 0.1 M HEPES, 20% PEG8000, 10 MM TRIS,    \\nREMARK 280  40 MM NACL, PH 7.5, LIPIDIC CUBIC PHASE, TEMPERATURE 293K           \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: H 3 2                            \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z                                                \\nREMARK 290       3555   -X+Y,-X,Z                                               \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z                                               \\nREMARK 290       6555   -X,-X+Y,-Z                                              \\nREMARK 290       7555   X+2/3,Y+1/3,Z+1/3                                       \\nREMARK 290       8555   -Y+2/3,X-Y+1/3,Z+1/3                                    \\nREMARK 290       9555   -X+Y+2/3,-X+1/3,Z+1/3                                   \\nREMARK 290      10555   Y+2/3,X+1/3,-Z+1/3                                      \\nREMARK 290      11555   X-Y+2/3,-Y+1/3,-Z+1/3                                   \\nREMARK 290      12555   -X+2/3,-X+Y+1/3,-Z+1/3                                  \\nREMARK 290      13555   X+1/3,Y+2/3,Z+2/3                                       \\nREMARK 290      14555   -Y+1/3,X-Y+2/3,Z+2/3                                    \\nREMARK 290      15555   -X+Y+1/3,-X+2/3,Z+2/3                                   \\nREMARK 290      16555   Y+1/3,X+2/3,-Z+2/3                                      \\nREMARK 290      17555   X-Y+1/3,-Y+2/3,-Z+2/3                                   \\nREMARK 290      18555   -X+1/3,-X+Y+2/3,-Z+2/3                                  \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   7  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3   7  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   8 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   8  0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   8  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   9 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   9 -0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1  10 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  10  0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  11  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2  11  0.000000 -1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  12 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  12 -0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  14 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  14  0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  15 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  15 -0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  15  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  16 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  16  0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  17  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  17  0.000000 -1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  17  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  18 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  18 -0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  18  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     GLU A    70                                                      \\nREMARK 465     SCH A    99                                                      \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ARG A  96    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     ARG A  97    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     LEU A  98    CG   CD1  CD2                                       \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ARG A  97       31.10    -98.71                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 5EH4   RELATED DB: PDB                                   \\nREMARK 900 5EH4 IS THE SAME PEPTIDE CRYSTALLISED AS DIMER                       \\nDBREF  5EH6 A   70    98  UNP    P02724   GLPA_HUMAN      89    117             \\nSEQADV 5EH6 ILE A   81  UNP  P02724    MET   100 ENGINEERED MUTATION            \\nSEQRES   1 A   30  GLU PRO GLU ILE THR LEU ILE ILE PHE GLY VAL ILE ALA          \\nSEQRES   2 A   30  GLY VAL ILE GLY THR ILE LEU LEU ILE SER TYR GLY ILE          \\nSEQRES   3 A   30  ARG ARG LEU SCH                                              \\nHELIX    1 AA1 GLU A   72  ARG A   97  1                                  26    \\nCRYST1   31.312   31.312  136.287  90.00  90.00 120.00 H 3 2        18          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.031937  0.018439  0.000000        0.00000                         \\nSCALE2      0.000000  0.036877  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.007337        0.00000                         \\nATOM      1  N   PRO A  71      26.002   6.928 -18.824  1.00 72.28           N  \\nANISOU    1  N   PRO A  71    10939   9924   6602  -1650  -1647    429       N  \\nATOM      2  CA  PRO A  71      24.968   6.210 -18.070  1.00 75.23           C  \\nANISOU    2  CA  PRO A  71    10989  10418   7177  -1839  -1839    391       C  \\nATOM      3  C   PRO A  71      25.401   5.954 -16.631  1.00 59.78           C  \\nANISOU    3  C   PRO A  71     8903   8293   5518  -1825  -1641    348       C  \\nATOM      4  O   PRO A  71      24.631   6.137 -15.685  1.00 55.18           O  \\nANISOU    4  O   PRO A  71     7919   7869   5177  -1810  -1746    462       O  \\nATOM      5  CB  PRO A  71      24.820   4.899 -18.846  1.00 83.70           C  \\nANISOU    5  CB  PRO A  71    12311  11422   8068  -2143  -1887    151       C  \\nATOM      6  CG  PRO A  71      26.160   4.684 -19.474  1.00 82.77           C  \\nANISOU    6  CG  PRO A  71    12644  11040   7765  -2081  -1586    -13       C  \\nATOM      7  CD  PRO A  71      26.685   6.058 -19.798  1.00 79.89           C  \\nANISOU    7  CD  PRO A  71    12306  10735   7312  -1802  -1502    190       C  \\nATOM      8  N   GLU A  72      26.651   5.547 -16.470  1.00 43.89           N  \\nANISOU    8  N   GLU A  72     7210   5973   3492  -1803  -1334    180       N  \\nATOM      9  CA  GLU A  72      27.199   5.342 -15.151  1.00 41.57           C  \\nANISOU    9  CA  GLU A  72     6830   5492   3473  -1747  -1128    132       C  \\nATOM     10  C   GLU A  72      27.567   6.686 -14.541  1.00 28.78           C  \\nANISOU   10  C   GLU A  72     4985   3872   2077  -1389   -970    340       C  \\nATOM     11  O   GLU A  72      28.126   6.740 -13.453  1.00 27.05           O  \\nANISOU   11  O   GLU A  72     4644   3486   2148  -1252   -742    321       O  \\nATOM     12  CB  GLU A  72      28.412   4.415 -15.215  1.00 53.65           C  \\nANISOU   12  CB  GLU A  72     8727   6675   4983  -1767   -827   -124       C  \\nATOM     13  CG  GLU A  72      29.411   4.761 -16.308  1.00 55.89           C  \\nANISOU   13  CG  GLU A  72     9315   6881   5040  -1626   -621   -177       C  \\nATOM     14  CD  GLU A  72      30.587   3.800 -16.349  1.00 67.90           C  \\nANISOU   14  CD  GLU A  72    11070   8100   6628  -1563   -300   -410       C  \\nATOM     15  OE1 GLU A  72      31.018   3.327 -15.272  1.00 68.55           O  \\nANISOU   15  OE1 GLU A  72    11065   7996   6985  -1491   -160   -465       O  \\nATOM     16  OE2 GLU A  72      31.075   3.511 -17.462  1.00 79.31           O  \\nANISOU   16  OE2 GLU A  72    12751   9514   7868  -1553   -194   -519       O  \\nATOM     17  N   ILE A  73      27.238   7.766 -15.250  1.00 30.36           N  \\nANISOU   17  N   ILE A  73     5165   4244   2126  -1248  -1115    542       N  \\nATOM     18  CA  ILE A  73      27.532   9.124 -14.797  1.00 28.77           C  \\nANISOU   18  CA  ILE A  73     4824   3994   2114   -920   -993    745       C  \\nATOM     19  C   ILE A  73      26.893   9.424 -13.447  1.00 26.85           C  \\nANISOU   19  C   ILE A  73     4154   3812   2237   -780   -998    815       C  \\nATOM     20  O   ILE A  73      27.496  10.058 -12.581  1.00 24.23           O  \\nANISOU   20  O   ILE A  73     3756   3310   2141   -574   -773    839       O  \\nATOM     21  CB  ILE A  73      27.053  10.160 -15.815  1.00 34.40           C  \\nANISOU   21  CB  ILE A  73     5607   4872   2593   -799  -1219    987       C  \\nATOM     22  CG1 ILE A  73      27.291  11.568 -15.279  1.00 36.30           C  \\nANISOU   22  CG1 ILE A  73     5741   4990   3060   -462  -1102   1194       C  \\nATOM     23  CG2 ILE A  73      25.585   9.942 -16.166  1.00 46.47           C  \\nANISOU   23  CG2 ILE A  73     6867   6736   4054   -887  -1615   1059       C  \\nATOM     24  CD1 ILE A  73      28.740  11.858 -15.015  1.00 45.01           C  \\nANISOU   24  CD1 ILE A  73     7064   5789   4247   -413   -720   1120       C  \\nATOM     25  N   THR A  74      25.665   8.948 -13.292  1.00 40.83           N  \\nANISOU   25  N   THR A  74     5637   5847   4029   -921  -1252    836       N  \\nATOM     26  CA  THR A  74      24.932   9.005 -12.039  1.00 54.69           C  \\nANISOU   26  CA  THR A  74     6966   7727   6086   -849  -1236    876       C  \\nATOM     27  C   THR A  74      25.719   8.304 -10.935  1.00 33.06           C  \\nANISOU   27  C   THR A  74     4287   4741   3534   -920   -947    714       C  \\nATOM     28  O   THR A  74      25.960   8.870  -9.873  1.00 25.63           O  \\nANISOU   28  O   THR A  74     3197   3724   2819   -711   -764    744       O  \\nATOM     29  CB  THR A  74      23.528   8.346 -12.181  1.00 69.92           C  \\nANISOU   29  CB  THR A  74     8570  10023   7975  -1102  -1547    902       C  \\nATOM     30  OG1 THR A  74      23.670   6.968 -12.571  1.00 59.92           O  \\nANISOU   30  OG1 THR A  74     7546   8677   6544  -1516  -1596    711       O  \\nATOM     31  CG2 THR A  74      22.688   9.086 -13.223  1.00 60.41           C  \\nANISOU   31  CG2 THR A  74     7246   9113   6594   -991  -1891   1093       C  \\nATOM     32  N   LEU A  75      26.123   7.069 -11.215  1.00 30.54           N  \\nANISOU   32  N   LEU A  75     4220   4283   3099  -1205   -922    539       N  \\nATOM     33  CA  LEU A  75      26.853   6.255 -10.261  1.00 32.84           C  \\nANISOU   33  CA  LEU A  75     4607   4325   3546  -1271   -697    405       C  \\nATOM     34  C   LEU A  75      28.206   6.852  -9.911  1.00 25.79           C  \\nANISOU   34  C   LEU A  75     3865   3180   2754  -1010   -424    380       C  \\nATOM     35  O   LEU A  75      28.625   6.828  -8.754  1.00 24.52           O  \\nANISOU   35  O   LEU A  75     3612   2910   2796   -922   -264    366       O  \\nATOM     36  CB  LEU A  75      27.029   4.843 -10.809  1.00 28.01           C  \\nANISOU   36  CB  LEU A  75     4304   3560   2777  -1589   -744    220       C  \\nATOM     37  CG  LEU A  75      25.766   3.999 -10.722  1.00 27.87           C  \\nANISOU   37  CG  LEU A  75     4119   3735   2737  -1948   -981    222       C  \\nATOM     38  CD1 LEU A  75      26.030   2.653 -11.357  1.00 37.84           C  \\nANISOU   38  CD1 LEU A  75     5752   4769   3858  -2133   -958     39       C  \\nATOM     39  CD2 LEU A  75      25.374   3.858  -9.264  1.00 31.17           C  \\nANISOU   39  CD2 LEU A  75     4232   4196   3414  -1954   -873    303       C  \\nATOM     40  N   ILE A  76      28.895   7.392 -10.905  1.00 19.81           N  \\nANISOU   40  N   ILE A  76     3334   2354   1837   -914   -376    382       N  \\nATOM     41  CA  ILE A  76      30.199   7.987 -10.648  1.00 25.42           C  \\nANISOU   41  CA  ILE A  76     4150   2861   2646   -719   -117    364       C  \\nATOM     42  C   ILE A  76      30.078   9.202  -9.725  1.00 21.46           C  \\nANISOU   42  C   ILE A  76     3413   2382   2358   -495    -72    500       C  \\nATOM     43  O   ILE A  76      30.821   9.326  -8.751  1.00 19.33           O  \\nANISOU   43  O   ILE A  76     3097   1975   2274   -406     97    454       O  \\nATOM     44  CB  ILE A  76      30.897   8.377 -11.962  1.00 18.98           C  \\nANISOU   44  CB  ILE A  76     3616   2007   1589   -703    -49    365       C  \\nATOM     45  CG1 ILE A  76      31.346   7.115 -12.698  1.00 26.36           C  \\nANISOU   45  CG1 ILE A  76     4836   2851   2328   -878      5    155       C  \\nATOM     46  CG2 ILE A  76      32.095   9.264 -11.691  1.00 17.37           C  \\nANISOU   46  CG2 ILE A  76     3437   1654   1510   -533    201    401       C  \\nATOM     47  CD1 ILE A  76      31.830   7.370 -14.096  1.00 33.92           C  \\nANISOU   47  CD1 ILE A  76     6090   3835   2962   -903     69    138       C  \\nATOM     48  N   ILE A  77      29.127  10.081 -10.014  1.00 17.69           N  \\nANISOU   48  N   ILE A  77     2797   2074   1850   -390   -239    659       N  \\nATOM     49  CA  ILE A  77      28.965  11.285  -9.221  1.00 17.04           C  \\nANISOU   49  CA  ILE A  77     2546   1970   1957   -140   -190    764       C  \\nATOM     50  C   ILE A  77      28.530  10.901  -7.817  1.00 15.87           C  \\nANISOU   50  C   ILE A  77     2141   1884   2003   -135   -137    701       C  \\nATOM     51  O   ILE A  77      29.012  11.457  -6.840  1.00 14.34           O  \\nANISOU   51  O   ILE A  77     1914   1569   1964      3     13    673       O  \\nATOM     52  CB  ILE A  77      27.958  12.256  -9.867  1.00 32.82           C  \\nANISOU   52  CB  ILE A  77     4453   4127   3892     28   -399    957       C  \\nATOM     53  CG1 ILE A  77      28.527  12.779 -11.191  1.00 28.88           C  \\nANISOU   53  CG1 ILE A  77     4274   3532   3168     26   -424   1056       C  \\nATOM     54  CG2 ILE A  77      27.626  13.433  -8.929  1.00 28.78           C  \\nANISOU   54  CG2 ILE A  77     3765   3567   3602    331   -343   1032       C  \\nATOM     55  CD1 ILE A  77      27.508  13.469 -12.067  1.00 24.61           C  \\nANISOU   55  CD1 ILE A  77     3698   3167   2486    155   -703   1271       C  \\nATOM     56  N   PHE A  78      27.644   9.923  -7.722  1.00 20.44           N  \\nANISOU   56  N   PHE A  78     2565   2655   2547   -323   -258    678       N  \\nATOM     57  CA  PHE A  78      27.223   9.413  -6.424  1.00 29.06           C  \\nANISOU   57  CA  PHE A  78     3441   3824   3777   -381   -185    638       C  \\nATOM     58  C   PHE A  78      28.424   8.921  -5.639  1.00 18.03           C  \\nANISOU   58  C   PHE A  78     2225   2175   2450   -414      5    527       C  \\nATOM     59  O   PHE A  78      28.580   9.231  -4.459  1.00 15.70           O  \\nANISOU   59  O   PHE A  78     1836   1857   2272   -311    126    514       O  \\nATOM     60  CB  PHE A  78      26.204   8.271  -6.578  1.00 27.90           C  \\nANISOU   60  CB  PHE A  78     3148   3890   3561   -681   -340    636       C  \\nATOM     61  CG  PHE A  78      26.034   7.437  -5.332  1.00 22.44           C  \\nANISOU   61  CG  PHE A  78     2353   3209   2963   -840   -228    598       C  \\nATOM     62  CD1 PHE A  78      25.276   7.901  -4.270  1.00 18.61           C  \\nANISOU   62  CD1 PHE A  78     1545   2938   2588   -734   -147    659       C  \\nATOM     63  CD2 PHE A  78      26.641   6.192  -5.225  1.00 24.51           C  \\nANISOU   63  CD2 PHE A  78     2869   3255   3189  -1079   -191    508       C  \\nATOM     64  CE1 PHE A  78      25.123   7.138  -3.128  1.00 22.90           C  \\nANISOU   64  CE1 PHE A  78     2025   3507   3168   -907    -27    650       C  \\nATOM     65  CE2 PHE A  78      26.495   5.426  -4.086  1.00 28.95           C  \\nANISOU   65  CE2 PHE A  78     3396   3794   3809  -1191    -99    498       C  \\nATOM     66  CZ  PHE A  78      25.735   5.900  -3.036  1.00 29.26           C  \\nANISOU   66  CZ  PHE A  78     3134   4062   3920  -1117    -16    569       C  \\nATOM     67  N   GLY A  79      29.267   8.143  -6.306  1.00 16.77           N  \\nANISOU   67  N   GLY A  79     2326   1842   2204   -542     25    441       N  \\nATOM     68  CA  GLY A  79      30.397   7.527  -5.649  1.00 17.63           C  \\nANISOU   68  CA  GLY A  79     2578   1732   2390   -550    170    345       C  \\nATOM     69  C   GLY A  79      31.372   8.545  -5.111  1.00 10.12           C  \\nANISOU   69  C   GLY A  79     1596    716   1533   -325    273    327       C  \\nATOM     70  O   GLY A  79      31.913   8.383  -4.016  1.00 15.97           O  \\nANISOU   70  O   GLY A  79     2272   1484   2310   -256    275    257       O  \\nATOM     71  N   VAL A  80      31.591   9.601  -5.877  1.00 10.40           N  \\nANISOU   71  N   VAL A  80     1690    719   1543   -227    301    379       N  \\nATOM     72  CA  VAL A  80      32.536  10.628  -5.468  1.00 11.18           C  \\nANISOU   72  CA  VAL A  80     1745    819   1684    -83    338    312       C  \\nATOM     73  C   VAL A  80      32.013  11.384  -4.245  1.00 13.80           C  \\nANISOU   73  C   VAL A  80     1956   1172   2116     24    367    331       C  \\nATOM     74  O   VAL A  80      32.725  11.539  -3.271  1.00 10.65           O  \\nANISOU   74  O   VAL A  80     1511    805   1730     46    389    236       O  \\nATOM     75  CB  VAL A  80      32.830  11.609  -6.604  1.00 13.99           C  \\nANISOU   75  CB  VAL A  80     2206   1127   1982    -46    353    378       C  \\nATOM     76  CG1 VAL A  80      33.718  12.724  -6.111  1.00 17.30           C  \\nANISOU   76  CG1 VAL A  80     2580   1539   2454     21    389    320       C  \\nATOM     77  CG2 VAL A  80      33.500  10.886  -7.770  1.00 13.03           C  \\nANISOU   77  CG2 VAL A  80     2225    983   1741   -137    407    339       C  \\nATOM     78  N   ILE A  81      30.763  11.834  -4.294  1.00 13.20           N  \\nANISOU   78  N   ILE A  81     1795   1113   2106    113    358    485       N  \\nATOM     79  CA  ILE A  81      30.179  12.589  -3.190  1.00 11.57           C  \\nANISOU   79  CA  ILE A  81     1445    961   1989    277    400    478       C  \\nATOM     80  C   ILE A  81      30.108  11.708  -1.940  1.00 13.01           C  \\nANISOU   80  C   ILE A  81     1533   1227   2182    176    459    408       C  \\nATOM     81  O   ILE A  81      30.520  12.124  -0.855  1.00 16.50           O  \\nANISOU   81  O   ILE A  81     1995   1608   2667    245    548    337       O  \\nATOM     82  CB  ILE A  81      28.752  13.144  -3.549  1.00 17.96           C  \\nANISOU   82  CB  ILE A  81     2062   1970   2793    435    299    580       C  \\nATOM     83  CG1 ILE A  81      28.846  14.286  -4.572  1.00 19.73           C  \\nANISOU   83  CG1 ILE A  81     2429   2060   3007    602    234    685       C  \\nATOM     84  CG2 ILE A  81      28.016  13.617  -2.294  1.00 17.99           C  \\nANISOU   84  CG2 ILE A  81     1869   2090   2875    608    390    536       C  \\nATOM     85  CD1 ILE A  81      27.489  14.757  -5.143  1.00 20.93           C  \\nANISOU   85  CD1 ILE A  81     2389   2417   3146    790     73    823       C  \\nATOM     86  N   ALA A  82      29.615  10.479  -2.093  1.00 14.62           N  \\nANISOU   86  N   ALA A  82     1675   1555   2325    -16    399    431       N  \\nATOM     87  CA  ALA A  82      29.518   9.568  -0.954  1.00 15.60           C  \\nANISOU   87  CA  ALA A  82     1756   1734   2436   -146    451    410       C  \\nATOM     88  C   ALA A  82      30.904   9.253  -0.397  1.00 17.38           C  \\nANISOU   88  C   ALA A  82     2169   1796   2638   -135    450    304       C  \\nATOM     89  O   ALA A  82      31.093   9.224   0.822  1.00 14.20           O  \\nANISOU   89  O   ALA A  82     1762   1408   2225   -118    497    282       O  \\nATOM     90  CB  ALA A  82      28.785   8.280  -1.339  1.00 12.86           C  \\nANISOU   90  CB  ALA A  82     1365   1496   2025   -401    364    459       C  \\nATOM     91  N   GLY A  83      31.866   9.017  -1.287  1.00  9.24           N  \\nANISOU   91  N   GLY A  83     1253    707   1549   -128    365    233       N  \\nATOM     92  CA  GLY A  83      33.240   8.762  -0.875  1.00 13.08           C  \\nANISOU   92  CA  GLY A  83     1783   1188   1997    -85    327    150       C  \\nATOM     93  C   GLY A  83      33.832   9.924  -0.086  1.00 12.02           C  \\nANISOU   93  C   GLY A  83     1606   1064   1896     -5    381    116       C  \\nATOM     94  O   GLY A  83      34.436   9.734   0.966  1.00 14.41           O  \\nANISOU   94  O   GLY A  83     1913   1363   2200     -6    377     98       O  \\nATOM     95  N   VAL A  84      33.648  11.136  -0.584  1.00 11.13           N  \\nANISOU   95  N   VAL A  84     1484    936   1808     43    418    124       N  \\nATOM     96  CA  VAL A  84      34.146  12.312   0.104  1.00  9.86           C  \\nANISOU   96  CA  VAL A  84     1332    751   1663     74    438     92       C  \\nATOM     97  C   VAL A  84      33.478  12.454   1.478  1.00 23.28           C  \\nANISOU   97  C   VAL A  84     3029   2426   3392    111    507     85       C  \\nATOM     98  O   VAL A  84      34.162  12.648   2.494  1.00 13.72           O  \\nANISOU   98  O   VAL A  84     1857   1190   2165     98    492     31       O  \\nATOM     99  CB  VAL A  84      33.919  13.579  -0.731  1.00 11.25           C  \\nANISOU   99  CB  VAL A  84     1553    869   1854    123    442    115       C  \\nATOM    100  CG1 VAL A  84      34.115  14.841   0.110  1.00 10.57           C  \\nANISOU  100  CG1 VAL A  84     1521    705   1792    167    448     70       C  \\nATOM    101  CG2 VAL A  84      34.850  13.573  -1.937  1.00 17.63           C  \\nANISOU  101  CG2 VAL A  84     2388   1661   2650     80    458    124       C  \\nATOM    102  N   ILE A  85      32.154  12.329   1.520  1.00 14.94           N  \\nANISOU  102  N   ILE A  85     1923   1361   2394    179    603    145       N  \\nATOM    103  CA  ILE A  85      31.434  12.529   2.766  1.00  9.86           C  \\nANISOU  103  CA  ILE A  85     1243    764   1739    273    714    109       C  \\nATOM    104  C   ILE A  85      31.759  11.396   3.733  1.00 13.41           C  \\nANISOU  104  C   ILE A  85     1720   1264   2112    139    730    122       C  \\nATOM    105  O   ILE A  85      32.010  11.623   4.920  1.00 12.84           O  \\nANISOU  105  O   ILE A  85     1713   1218   1946    152    765     51       O  \\nATOM    106  CB  ILE A  85      29.908  12.608   2.539  1.00 11.50           C  \\nANISOU  106  CB  ILE A  85     1248   1175   1948    376    772    165       C  \\nATOM    107  CG1 ILE A  85      29.540  13.861   1.742  1.00 12.43           C  \\nANISOU  107  CG1 ILE A  85     1381   1240   2101    537    698    162       C  \\nATOM    108  CG2 ILE A  85      29.158  12.574   3.871  1.00 13.13           C  \\nANISOU  108  CG2 ILE A  85     1364   1588   2037    399    874    114       C  \\nATOM    109  CD1 ILE A  85      28.073  13.892   1.307  1.00 21.23           C  \\nANISOU  109  CD1 ILE A  85     2243   2580   3242    666    702    240       C  \\nATOM    110  N   GLY A  86      31.759  10.176   3.214  1.00  9.78           N  \\nANISOU  110  N   GLY A  86     1252    811   1654     -9    668    216       N  \\nATOM    111  CA  GLY A  86      32.067   9.016   4.018  1.00 10.45           C  \\nANISOU  111  CA  GLY A  86     1408    923   1640   -133    602    253       C  \\nATOM    112  C   GLY A  86      33.429   9.138   4.676  1.00 13.84           C  \\nANISOU  112  C   GLY A  86     1950   1238   2069    -97    542    201       C  \\nATOM    113  O   GLY A  86      33.573   8.900   5.878  1.00 11.38           O  \\nANISOU  113  O   GLY A  86     1720    951   1652   -142    558    232       O  \\nATOM    114  N   THR A  87      34.433   9.509   3.889  1.00 12.99           N  \\nANISOU  114  N   THR A  87     1814   1108   2012    -35    437    126       N  \\nATOM    115  CA  THR A  87      35.804   9.618   4.400  1.00 13.69           C  \\nANISOU  115  CA  THR A  87     1935   1150   2118    -27    347     91       C  \\nATOM    116  C   THR A  87      35.930  10.647   5.525  1.00 12.38           C  \\nANISOU  116  C   THR A  87     1850    902   1951    -16    409     19       C  \\nATOM    117  O   THR A  87      36.549  10.379   6.559  1.00 12.81           O  \\nANISOU  117  O   THR A  87     1982    937   1949    -48    338     17       O  \\nATOM    118  CB  THR A  87      36.779   9.989   3.278  1.00 17.27           C  \\nANISOU  118  CB  THR A  87     2321   1640   2602     -6    290     50       C  \\nATOM    119  OG1 THR A  87      36.788   8.937   2.313  1.00 18.68           O  \\nANISOU  119  OG1 THR A  87     2482   1837   2777     -8    289     77       O  \\nATOM    120  CG2 THR A  87      38.187  10.186   3.824  1.00 16.57           C  \\nANISOU  120  CG2 THR A  87     2227   1480   2587    -10    237     21       C  \\nATOM    121  N   ILE A  88      35.341  11.822   5.302  1.00 17.44           N  \\nANISOU  121  N   ILE A  88     2481   1553   2593     40    488    -55       N  \\nATOM    122  CA  ILE A  88      35.267  12.888   6.299  1.00 19.16           C  \\nANISOU  122  CA  ILE A  88     2814   1716   2751     84    545   -196       C  \\nATOM    123  C   ILE A  88      34.589  12.416   7.594  1.00 19.87           C  \\nANISOU  123  C   ILE A  88     2959   1969   2623     75    598   -200       C  \\nATOM    124  O   ILE A  88      35.102  12.650   8.689  1.00 15.46           O  \\nANISOU  124  O   ILE A  88     2516   1441   1917     35    536   -287       O  \\nATOM    125  CB  ILE A  88      34.504  14.104   5.735  1.00 13.93           C  \\nANISOU  125  CB  ILE A  88     2113   1077   2102    181    576   -223       C  \\nATOM    126  CG1 ILE A  88      35.327  14.779   4.646  1.00 13.96           C  \\nANISOU  126  CG1 ILE A  88     2082   1046   2176    134    455   -189       C  \\nATOM    127  CG2 ILE A  88      34.123  15.087   6.842  1.00 16.44           C  \\nANISOU  127  CG2 ILE A  88     2550   1378   2318    268    642   -381       C  \\nATOM    128  CD1 ILE A  88      34.637  15.963   4.066  1.00 18.02           C  \\nANISOU  128  CD1 ILE A  88     2625   1515   2706    226    466   -198       C  \\nATOM    129  N   LEU A  89      33.448  11.740   7.460  1.00 11.97           N  \\nANISOU  129  N   LEU A  89     1866   1110   1573     77    699    -94       N  \\nATOM    130  CA  LEU A  89      32.719  11.270   8.622  1.00 15.64           C  \\nANISOU  130  CA  LEU A  89     2357   1779   1807     30    786    -64       C  \\nATOM    131  C   LEU A  89      33.496  10.176   9.350  1.00 17.03           C  \\nANISOU  131  C   LEU A  89     2650   1960   1862   -109    644     50       C  \\nATOM    132  O   LEU A  89      33.421  10.062  10.574  1.00 22.83           O  \\nANISOU  132  O   LEU A  89     3503   2822   2348   -157    660     47       O  \\nATOM    133  CB  LEU A  89      31.337  10.757   8.223  1.00 15.16           C  \\nANISOU  133  CB  LEU A  89     2118   1893   1750      8    923     44       C  \\nATOM    134  CG  LEU A  89      30.305  11.772   7.741  1.00 18.76           C  \\nANISOU  134  CG  LEU A  89     2413   2428   2288    194   1065    -39       C  \\nATOM    135  CD1 LEU A  89      28.953  11.097   7.500  1.00 18.26           C  \\nANISOU  135  CD1 LEU A  89     2108   2619   2212    118   1127     82       C  \\nATOM    136  CD2 LEU A  89      30.145  12.864   8.755  1.00 17.08           C  \\nANISOU  136  CD2 LEU A  89     2287   2258   1946    347   1167   -223       C  \\nATOM    137  N   LEU A  90      34.249   9.376   8.604  1.00 17.01           N  \\nANISOU  137  N   LEU A  90     2634   1814   2016   -149    506    153       N  \\nATOM    138  CA  LEU A  90      35.074   8.327   9.215  1.00 21.28           C  \\nANISOU  138  CA  LEU A  90     3288   2315   2484   -214    341    279       C  \\nATOM    139  C   LEU A  90      36.266   8.919   9.981  1.00 19.34           C  \\nANISOU  139  C   LEU A  90     3105   2064   2180   -180    177    183       C  \\nATOM    140  O   LEU A  90      36.646   8.435  11.049  1.00 18.95           O  \\nANISOU  140  O   LEU A  90     3181   2085   1935   -222     50    260       O  \\nATOM    141  CB  LEU A  90      35.559   7.351   8.143  1.00 14.32           C  \\nANISOU  141  CB  LEU A  90     2375   1259   1808   -208    262    380       C  \\nATOM    142  CG  LEU A  90      35.982   5.938   8.518  1.00 26.43           C  \\nANISOU  142  CG  LEU A  90     4044   2695   3303   -244    131    560       C  \\nATOM    143  CD1 LEU A  90      34.949   5.281   9.413  1.00 31.06           C  \\nANISOU  143  CD1 LEU A  90     4753   3391   3656   -397    202    703       C  \\nATOM    144  CD2 LEU A  90      36.156   5.148   7.244  1.00 21.46           C  \\nANISOU  144  CD2 LEU A  90     3291   1963   2900   -193    106    550       C  \\nATOM    145  N   ILE A  91      36.861   9.967   9.434  1.00 16.80           N  \\nANISOU  145  N   ILE A  91     2705   1662   2015   -132    162     29       N  \\nATOM    146  CA  ILE A  91      37.923  10.669  10.141  1.00 16.69           C  \\nANISOU  146  CA  ILE A  91     2731   1659   1952   -161      0    -90       C  \\nATOM    147  C   ILE A  91      37.387  11.282  11.438  1.00 21.02           C  \\nANISOU  147  C   ILE A  91     3458   2337   2192   -197     42   -207       C  \\nATOM    148  O   ILE A  91      37.971  11.111  12.510  1.00 23.14           O  \\nANISOU  148  O   ILE A  91     3838   2702   2251   -260   -131   -205       O  \\nATOM    149  CB  ILE A  91      38.548  11.761   9.261  1.00 16.48           C  \\nANISOU  149  CB  ILE A  91     2608   1501   2152   -163     12   -230       C  \\nATOM    150  CG1 ILE A  91      39.209  11.120   8.036  1.00 14.90           C  \\nANISOU  150  CG1 ILE A  91     2238   1215   2208   -130     -8   -127       C  \\nATOM    151  CG2 ILE A  91      39.553  12.591  10.062  1.00 20.71           C  \\nANISOU  151  CG2 ILE A  91     3190   2054   2625   -264   -162   -378       C  \\nATOM    152  CD1 ILE A  91      39.872  12.107   7.107  1.00 15.47           C  \\nANISOU  152  CD1 ILE A  91     2214   1180   2484   -172     32   -222       C  \\nATOM    153  N   SER A  92      36.265  11.984  11.320  1.00 21.14           N  \\nANISOU  153  N   SER A  92     3497   2366   2168   -137    273   -310       N  \\nATOM    154  CA  SER A  92      35.596  12.619  12.448  1.00 19.47           C  \\nANISOU  154  CA  SER A  92     3452   2280   1664   -123    395   -459       C  \\nATOM    155  C   SER A  92      35.332  11.605  13.560  1.00 21.28           C  \\nANISOU  155  C   SER A  92     3792   2717   1575   -208    375   -314       C  \\nATOM    156  O   SER A  92      35.657  11.825  14.733  1.00 23.88           O  \\nANISOU  156  O   SER A  92     4318   3152   1603   -269    293   -398       O  \\nATOM    157  CB  SER A  92      34.292  13.264  11.965  1.00 29.49           C  \\nANISOU  157  CB  SER A  92     4650   3558   2997     23    673   -538       C  \\nATOM    158  OG  SER A  92      33.609  13.945  12.998  1.00 39.92           O  \\nANISOU  158  OG  SER A  92     6117   5000   4052     92    844   -718       O  \\nATOM    159  N   TYR A  93      34.755  10.473  13.174  1.00 26.54           N  \\nANISOU  159  N   TYR A  93     4366   3429   2290   -241    439    -86       N  \\nATOM    160  CA  TYR A  93      34.474   9.408  14.121  1.00 27.68           C  \\nANISOU  160  CA  TYR A  93     4642   3725   2149   -357    429    110       C  \\nATOM    161  C   TYR A  93      35.794   8.928  14.720  1.00 27.37           C  \\nANISOU  161  C   TYR A  93     4738   3639   2021   -399    102    200       C  \\nATOM    162  O   TYR A  93      35.933   8.818  15.933  1.00 28.88           O  \\nANISOU  162  O   TYR A  93     5135   3977   1861   -470     24    228       O  \\nATOM    163  CB  TYR A  93      33.712   8.267  13.436  1.00 21.26           C  \\nANISOU  163  CB  TYR A  93     3721   2895   1460   -429    526    339       C  \\nATOM    164  CG  TYR A  93      33.489   7.046  14.302  1.00 33.29           C  \\nANISOU  164  CG  TYR A  93     5422   4503   2724   -592    501    595       C  \\nATOM    165  CD1 TYR A  93      32.502   7.034  15.280  1.00 28.66           C  \\nANISOU  165  CD1 TYR A  93     4837   4158   1893   -651    694    604       C  \\nATOM    166  CD2 TYR A  93      34.254   5.906  14.132  1.00 23.51           C  \\nANISOU  166  CD2 TYR A  93     4283   3078   1573   -625    269    812       C  \\nATOM    167  CE1 TYR A  93      32.295   5.937  16.063  1.00 29.00           C  \\nANISOU  167  CE1 TYR A  93     4966   4240   1811   -750    652    817       C  \\nATOM    168  CE2 TYR A  93      34.049   4.806  14.910  1.00 26.12           C  \\nANISOU  168  CE2 TYR A  93     4692   3432   1802   -692    224    999       C  \\nATOM    169  CZ  TYR A  93      33.070   4.827  15.874  1.00 32.94           C  \\nANISOU  169  CZ  TYR A  93     5569   4524   2424   -777    417   1012       C  \\nATOM    170  OH  TYR A  93      32.866   3.724  16.653  1.00 37.68           O  \\nANISOU  170  OH  TYR A  93     6289   5122   2905   -863    388   1217       O  \\nATOM    171  N   GLY A  94      36.775   8.688  13.859  1.00 27.56           N  \\nANISOU  171  N   GLY A  94     4631   3486   2355   -341    -90    240       N  \\nATOM    172  CA  GLY A  94      38.087   8.260  14.302  1.00 26.35           C  \\nANISOU  172  CA  GLY A  94     4513   3313   2186   -331   -418    327       C  \\nATOM    173  C   GLY A  94      38.758   9.167  15.325  1.00 30.85           C  \\nANISOU  173  C   GLY A  94     5189   4010   2522   -387   -592    152       C  \\nATOM    174  O   GLY A  94      39.384   8.678  16.259  1.00 34.40           O  \\nANISOU  174  O   GLY A  94     5764   4561   2745   -420   -848    270       O  \\nATOM    175  N   ILE A  95      38.646  10.484  15.172  1.00 30.72           N  \\nANISOU  175  N   ILE A  95     5156   3973   2545   -403   -483   -126       N  \\nATOM    176  CA  ILE A  95      39.352  11.367  16.102  1.00 36.00           C  \\nANISOU  176  CA  ILE A  95     5959   4723   2995   -499   -676   -329       C  \\nATOM    177  C   ILE A  95      38.577  11.567  17.405  1.00 41.88           C  \\nANISOU  177  C   ILE A  95     7009   5651   3252   -556   -564   -411       C  \\nATOM    178  O   ILE A  95      39.175  11.835  18.445  1.00 44.79           O  \\nANISOU  178  O   ILE A  95     7564   6144   3311   -660   -790   -498       O  \\nATOM    179  CB  ILE A  95      39.666  12.730  15.469  1.00 31.59           C  \\nANISOU  179  CB  ILE A  95     5328   4008   2665   -527   -628   -604       C  \\nATOM    180  CG1 ILE A  95      38.392  13.480  15.107  1.00 44.59           C  \\nANISOU  180  CG1 ILE A  95     7048   5569   4326   -434   -264   -754       C  \\nATOM    181  CG2 ILE A  95      40.500  12.536  14.236  1.00 28.34           C  \\nANISOU  181  CG2 ILE A  95     4621   3465   2683   -501   -716   -515       C  \\nATOM    182  CD1 ILE A  95      38.625  14.614  14.129  1.00 56.29           C  \\nANISOU  182  CD1 ILE A  95     8453   6814   6122   -422   -200   -928       C  \\nATOM    183  N   ARG A  96      37.255  11.416  17.350  1.00 44.07           N  \\nANISOU  183  N   ARG A  96     7323   5977   3443   -500   -216   -383       N  \\nATOM    184  CA  ARG A  96      36.445  11.402  18.559  1.00 48.82           C  \\nANISOU  184  CA  ARG A  96     8152   6800   3599   -539    -44   -413       C  \\nATOM    185  C   ARG A  96      36.929  10.279  19.462  1.00 76.51           C  \\nANISOU  185  C   ARG A  96    11748  10437   6886   -616   -281   -135       C  \\nATOM    186  O   ARG A  96      37.215  10.485  20.641  1.00 72.86           O  \\nANISOU  186  O   ARG A  96    11427  10122   6136   -653   -391   -197       O  \\nATOM    187  CB  ARG A  96      34.965  11.213  18.224  1.00 37.76           C  \\nANISOU  187  CB  ARG A  96     6623   5470   2255   -460    359   -360       C  \\nATOM    188  N   ARG A  97      37.055   9.096  18.870  1.00109.45           N  \\nANISOU  188  N   ARG A  97    15842  14523  11220   -617   -370    171       N  \\nATOM    189  CA  ARG A  97      37.355   7.874  19.604  1.00126.28           C  \\nANISOU  189  CA  ARG A  97    18046  16706  13230   -638   -546    477       C  \\nATOM    190  C   ARG A  97      38.833   7.466  19.569  1.00135.45           C  \\nANISOU  190  C   ARG A  97    19165  17788  14510   -587   -990    594       C  \\nATOM    191  O   ARG A  97      39.151   6.276  19.611  1.00149.94           O  \\nANISOU  191  O   ARG A  97    20993  19546  16431   -527  -1126    878       O  \\nATOM    192  CB  ARG A  97      36.497   6.726  19.058  1.00126.38           C  \\nANISOU  192  CB  ARG A  97    17986  16628  13406   -649   -349    735       C  \\nATOM    193  N   LEU A  98      39.734   8.442  19.490  1.00 91.41           N  \\nANISOU  193  N   LEU A  98    13524  12225   8981   -599  -1204    365       N  \\nATOM    194  CA  LEU A  98      41.164   8.149  19.561  1.00 61.19           C  \\nANISOU  194  CA  LEU A  98     9556   8401   5292   -545  -1627    456       C  \\nATOM    195  C   LEU A  98      41.950   9.379  19.988  1.00 66.32           C  \\nANISOU  195  C   LEU A  98    10163   9158   5878   -647  -1813    155       C  \\nATOM    196  O   LEU A  98      43.006   9.262  20.609  1.00 71.98           O  \\nANISOU  196  O   LEU A  98    10796  10000   6555   -646  -2141    198       O  \\nATOM    197  CB  LEU A  98      41.686   7.629  18.218  1.00 51.35           C  \\nANISOU  197  CB  LEU A  98     8064   6955   4491   -419  -1724    582       C  \\nTER     198      LEU A  98                                                      \\nMASTER      286    0    0    1    0    0    0    6  197    1    0    3          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_17294480429883013.addModel(\"HEADER                                            20-OCT-24   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 20-OCT-24                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2024.42+release.3366cf78a3                                  \\nATOM      1  N   ASP A   1       7.436  21.211 -12.237  1.00  0.00           N  \\nATOM      2  CA  ASP A   1       6.048  21.655 -12.263  1.00  0.00           C  \\nATOM      3  C   ASP A   1       5.092  20.480 -12.423  1.00  0.00           C  \\nATOM      4  O   ASP A   1       4.104  20.565 -13.153  1.00  0.00           O  \\nATOM      5  CB  ASP A   1       5.827  22.657 -13.399  1.00  0.00           C  \\nATOM      6  CG  ASP A   1       6.087  22.058 -14.775  1.00  0.00           C  \\nATOM      7  OD1 ASP A   1       6.466  20.913 -14.842  1.00  0.00           O  \\nATOM      8  OD2 ASP A   1       5.905  22.752 -15.747  1.00  0.00           O  \\nATOM      9 1H   ASP A   1       8.044  22.011 -12.130  1.00  0.00           H  \\nATOM     10 2H   ASP A   1       7.573  20.579 -11.461  1.00  0.00           H  \\nATOM     11 3H   ASP A   1       7.655  20.736 -13.101  1.00  0.00           H  \\nATOM     12  HA  ASP A   1       5.823  22.147 -11.316  1.00  0.00           H  \\nATOM     13 1HB  ASP A   1       4.802  23.024 -13.366  1.00  0.00           H  \\nATOM     14 2HB  ASP A   1       6.487  23.514 -13.261  1.00  0.00           H  \\nATOM     15  N   ASN A   2       5.419  19.372 -11.775  1.00  0.00           N  \\nATOM     16  CA  ASN A   2       4.593  18.172 -11.839  1.00  0.00           C  \\nATOM     17  C   ASN A   2       3.919  17.836 -10.525  1.00  0.00           C  \\nATOM     18  O   ASN A   2       3.470  16.709 -10.316  1.00  0.00           O  \\nATOM     19  CB  ASN A   2       5.422  16.995 -12.311  1.00  0.00           C  \\nATOM     20  CG  ASN A   2       5.858  17.133 -13.743  1.00  0.00           C  \\nATOM     21  OD1 ASN A   2       5.072  16.901 -14.669  1.00  0.00           O  \\nATOM     22  ND2 ASN A   2       7.096  17.506 -13.943  1.00  0.00           N  \\nATOM     23  H   ASN A   2       6.247  19.366 -11.196  1.00  0.00           H  \\nATOM     24  HA  ASN A   2       3.786  18.345 -12.552  1.00  0.00           H  \\nATOM     25 1HB  ASN A   2       6.308  16.898 -11.681  1.00  0.00           H  \\nATOM     26 2HB  ASN A   2       4.844  16.077 -12.207  1.00  0.00           H  \\nATOM     27 1HD2 ASN A   2       7.441  17.614 -14.876  1.00  0.00           H  \\nATOM     28 2HD2 ASN A   2       7.697  17.682 -13.164  1.00  0.00           H  \\nATOM     29  N   GLY A   3       3.876  18.809  -9.629  1.00  0.00           N  \\nATOM     30  CA  GLY A   3       3.270  18.656  -8.321  1.00  0.00           C  \\nATOM     31  C   GLY A   3       4.017  19.439  -7.269  1.00  0.00           C  \\nATOM     32  O   GLY A   3       5.118  19.934  -7.513  1.00  0.00           O  \\nATOM     33  H   GLY A   3       4.266  19.708  -9.875  1.00  0.00           H  \\nATOM     34 1HA  GLY A   3       2.234  18.992  -8.359  1.00  0.00           H  \\nATOM     35 2HA  GLY A   3       3.256  17.601  -8.049  1.00  0.00           H  \\nATOM     36  N   GLU A   4       3.418  19.550  -6.100  1.00  0.00           N  \\nATOM     37  CA  GLU A   4       3.951  20.271  -4.966  1.00  0.00           C  \\nATOM     38  C   GLU A   4       5.064  19.514  -4.308  1.00  0.00           C  \\nATOM     39  O   GLU A   4       5.212  18.318  -4.533  1.00  0.00           O  \\nATOM     40  CB  GLU A   4       2.855  20.552  -3.947  1.00  0.00           C  \\nATOM     41  CG  GLU A   4       1.754  21.468  -4.446  1.00  0.00           C  \\nATOM     42  CD  GLU A   4       2.231  22.869  -4.704  1.00  0.00           C  \\nATOM     43  OE1 GLU A   4       2.854  23.433  -3.837  1.00  0.00           O  \\nATOM     44  OE2 GLU A   4       1.973  23.378  -5.771  1.00  0.00           O  \\nATOM     45  H   GLU A   4       2.521  19.098  -5.988  1.00  0.00           H  \\nATOM     46  HA  GLU A   4       4.351  21.222  -5.318  1.00  0.00           H  \\nATOM     47 1HB  GLU A   4       2.398  19.619  -3.641  1.00  0.00           H  \\nATOM     48 2HB  GLU A   4       3.293  21.009  -3.058  1.00  0.00           H  \\nATOM     49 1HG  GLU A   4       1.348  21.060  -5.371  1.00  0.00           H  \\nATOM     50 2HG  GLU A   4       0.953  21.492  -3.709  1.00  0.00           H  \\nATOM     51  N   TRP A   5       5.858  20.154  -3.467  1.00  0.00           N  \\nATOM     52  CA  TRP A   5       6.909  19.411  -2.794  1.00  0.00           C  \\nATOM     53  C   TRP A   5       6.366  18.204  -2.044  1.00  0.00           C  \\nATOM     54  O   TRP A   5       7.076  17.215  -1.861  1.00  0.00           O  \\nATOM     55  CB  TRP A   5       7.655  20.320  -1.821  1.00  0.00           C  \\nATOM     56  CG  TRP A   5       6.816  20.776  -0.665  1.00  0.00           C  \\nATOM     57  CD1 TRP A   5       6.077  21.916  -0.595  1.00  0.00           C  \\nATOM     58  CD2 TRP A   5       6.625  20.095   0.604  1.00  0.00           C  \\nATOM     59  NE1 TRP A   5       5.443  21.996   0.621  1.00  0.00           N  \\nATOM     60  CE2 TRP A   5       5.767  20.888   1.365  1.00  0.00           C  \\nATOM     61  CE3 TRP A   5       7.107  18.895   1.140  1.00  0.00           C  \\nATOM     62  CZ2 TRP A   5       5.376  20.525   2.645  1.00  0.00           C  \\nATOM     63  CZ3 TRP A   5       6.715  18.530   2.421  1.00  0.00           C  \\nATOM     64  CH2 TRP A   5       5.871  19.324   3.153  1.00  0.00           C  \\nATOM     65  H   TRP A   5       5.738  21.140  -3.288  1.00  0.00           H  \\nATOM     66  HA  TRP A   5       7.610  19.047  -3.545  1.00  0.00           H  \\nATOM     67 1HB  TRP A   5       8.525  19.795  -1.428  1.00  0.00           H  \\nATOM     68 2HB  TRP A   5       8.016  21.201  -2.351  1.00  0.00           H  \\nATOM     69  HD1 TRP A   5       5.999  22.658  -1.387  1.00  0.00           H  \\nATOM     70  HE1 TRP A   5       4.837  22.746   0.921  1.00  0.00           H  \\nATOM     71  HE3 TRP A   5       7.776  18.259   0.561  1.00  0.00           H  \\nATOM     72  HZ2 TRP A   5       4.706  21.143   3.243  1.00  0.00           H  \\nATOM     73  HZ3 TRP A   5       7.096  17.595   2.831  1.00  0.00           H  \\nATOM     74  HH2 TRP A   5       5.583  19.008   4.156  1.00  0.00           H  \\nATOM     75  N   ILE A   6       5.119  18.281  -1.571  1.00  0.00           N  \\nATOM     76  CA  ILE A   6       4.508  17.222  -0.803  1.00  0.00           C  \\nATOM     77  C   ILE A   6       3.888  16.122  -1.645  1.00  0.00           C  \\nATOM     78  O   ILE A   6       3.406  15.131  -1.097  1.00  0.00           O  \\nATOM     79  CB  ILE A   6       3.429  17.801   0.126  1.00  0.00           C  \\nATOM     80  CG1 ILE A   6       3.132  16.832   1.265  1.00  0.00           C  \\nATOM     81  CG2 ILE A   6       2.168  18.110  -0.654  1.00  0.00           C  \\nATOM     82  CD1 ILE A   6       2.310  17.434   2.380  1.00  0.00           C  \\nATOM     83  H   ILE A   6       4.585  19.121  -1.744  1.00  0.00           H  \\nATOM     84  HA  ILE A   6       5.277  16.746  -0.198  1.00  0.00           H  \\nATOM     85  HB  ILE A   6       3.796  18.718   0.580  1.00  0.00           H  \\nATOM     86 1HG1 ILE A   6       2.598  15.967   0.876  1.00  0.00           H  \\nATOM     87 2HG1 ILE A   6       4.070  16.472   1.690  1.00  0.00           H  \\nATOM     88 1HG2 ILE A   6       1.416  18.518   0.018  1.00  0.00           H  \\nATOM     89 2HG2 ILE A   6       2.391  18.835  -1.431  1.00  0.00           H  \\nATOM     90 3HG2 ILE A   6       1.790  17.196  -1.110  1.00  0.00           H  \\nATOM     91 1HD1 ILE A   6       2.141  16.685   3.153  1.00  0.00           H  \\nATOM     92 2HD1 ILE A   6       2.845  18.284   2.808  1.00  0.00           H  \\nATOM     93 3HD1 ILE A   6       1.353  17.770   1.985  1.00  0.00           H  \\nATOM     94  N   GLN A   7       3.785  16.339  -2.958  1.00  0.00           N  \\nATOM     95  CA  GLN A   7       3.173  15.423  -3.904  1.00  0.00           C  \\nATOM     96  C   GLN A   7       4.251  14.682  -4.647  1.00  0.00           C  \\nATOM     97  O   GLN A   7       3.970  13.839  -5.500  1.00  0.00           O  \\nATOM     98  CB  GLN A   7       2.269  16.166  -4.885  1.00  0.00           C  \\nATOM     99  CG  GLN A   7       1.077  16.826  -4.246  1.00  0.00           C  \\nATOM    100  CD  GLN A   7       0.226  17.581  -5.250  1.00  0.00           C  \\nATOM    101  OE1 GLN A   7       0.711  18.477  -5.945  1.00  0.00           O  \\nATOM    102  NE2 GLN A   7      -1.049  17.222  -5.331  1.00  0.00           N  \\nATOM    103  H   GLN A   7       4.240  17.108  -3.418  1.00  0.00           H  \\nATOM    104  HA  GLN A   7       2.567  14.708  -3.350  1.00  0.00           H  \\nATOM    105 1HB  GLN A   7       2.842  16.933  -5.398  1.00  0.00           H  \\nATOM    106 2HB  GLN A   7       1.903  15.471  -5.640  1.00  0.00           H  \\nATOM    107 1HG  GLN A   7       0.457  16.062  -3.780  1.00  0.00           H  \\nATOM    108 2HG  GLN A   7       1.424  17.529  -3.498  1.00  0.00           H  \\nATOM    109 1HE2 GLN A   7      -1.659  17.685  -5.975  1.00  0.00           H  \\nATOM    110 2HE2 GLN A   7      -1.401  16.490  -4.748  1.00  0.00           H  \\nATOM    111  N   LEU A   8       5.489  15.050  -4.366  1.00  0.00           N  \\nATOM    112  CA  LEU A   8       6.617  14.441  -5.027  1.00  0.00           C  \\nATOM    113  C   LEU A   8       7.456  13.679  -4.026  1.00  0.00           C  \\nATOM    114  O   LEU A   8       7.820  14.204  -2.973  1.00  0.00           O  \\nATOM    115  CB  LEU A   8       7.463  15.509  -5.723  1.00  0.00           C  \\nATOM    116  CG  LEU A   8       6.733  16.340  -6.822  1.00  0.00           C  \\nATOM    117  CD1 LEU A   8       7.660  17.419  -7.339  1.00  0.00           C  \\nATOM    118  CD2 LEU A   8       6.289  15.419  -7.943  1.00  0.00           C  \\nATOM    119  H   LEU A   8       5.656  15.746  -3.653  1.00  0.00           H  \\nATOM    120  HA  LEU A   8       6.244  13.748  -5.779  1.00  0.00           H  \\nATOM    121 1HB  LEU A   8       7.832  16.206  -4.970  1.00  0.00           H  \\nATOM    122 2HB  LEU A   8       8.320  15.024  -6.190  1.00  0.00           H  \\nATOM    123  HG  LEU A   8       5.864  16.829  -6.389  1.00  0.00           H  \\nATOM    124 1HD1 LEU A   8       7.149  18.002  -8.108  1.00  0.00           H  \\nATOM    125 2HD1 LEU A   8       7.948  18.077  -6.517  1.00  0.00           H  \\nATOM    126 3HD1 LEU A   8       8.551  16.960  -7.765  1.00  0.00           H  \\nATOM    127 1HD2 LEU A   8       5.777  16.001  -8.711  1.00  0.00           H  \\nATOM    128 2HD2 LEU A   8       7.160  14.930  -8.378  1.00  0.00           H  \\nATOM    129 3HD2 LEU A   8       5.611  14.665  -7.547  1.00  0.00           H  \\nATOM    130  N   VAL A   9       7.808  12.453  -4.383  1.00  0.00           N  \\nATOM    131  CA  VAL A   9       8.595  11.589  -3.523  1.00  0.00           C  \\nATOM    132  C   VAL A   9       9.915  11.283  -4.227  1.00  0.00           C  \\nATOM    133  O   VAL A   9       9.960  11.156  -5.453  1.00  0.00           O  \\nATOM    134  CB  VAL A   9       7.847  10.285  -3.216  1.00  0.00           C  \\nATOM    135  CG1 VAL A   9       6.562  10.575  -2.457  1.00  0.00           C  \\nATOM    136  CG2 VAL A   9       7.553   9.540  -4.510  1.00  0.00           C  \\nATOM    137  H   VAL A   9       7.482  12.086  -5.266  1.00  0.00           H  \\nATOM    138  HA  VAL A   9       8.774  12.104  -2.593  1.00  0.00           H  \\nATOM    139  HB  VAL A   9       8.468   9.664  -2.572  1.00  0.00           H  \\nATOM    140 1HG1 VAL A   9       6.045   9.639  -2.248  1.00  0.00           H  \\nATOM    141 2HG1 VAL A   9       6.799  11.075  -1.519  1.00  0.00           H  \\nATOM    142 3HG1 VAL A   9       5.920  11.217  -3.061  1.00  0.00           H  \\nATOM    143 1HG2 VAL A   9       7.024   8.616  -4.285  1.00  0.00           H  \\nATOM    144 2HG2 VAL A   9       6.936  10.165  -5.158  1.00  0.00           H  \\nATOM    145 3HG2 VAL A   9       8.493   9.306  -5.018  1.00  0.00           H  \\nATOM    146  N   PRO A  10      10.993  11.132  -3.438  1.00  0.00           N  \\nATOM    147  CA  PRO A  10      12.249  10.849  -4.075  1.00  0.00           C  \\nATOM    148  C   PRO A  10      12.238   9.632  -4.980  1.00  0.00           C  \\nATOM    149  O   PRO A  10      11.698   8.572  -4.665  1.00  0.00           O  \\nATOM    150  CB  PRO A  10      13.166  10.636  -2.869  1.00  0.00           C  \\nATOM    151  CG  PRO A  10      12.527  11.465  -1.762  1.00  0.00           C  \\nATOM    152  CD  PRO A  10      11.049  11.294  -1.973  1.00  0.00           C  \\nATOM    153  HA  PRO A  10      12.560  11.722  -4.667  1.00  0.00           H  \\nATOM    154 1HB  PRO A  10      13.222   9.566  -2.623  1.00  0.00           H  \\nATOM    155 2HB  PRO A  10      14.188  10.963  -3.112  1.00  0.00           H  \\nATOM    156 1HG  PRO A  10      12.859  11.105  -0.778  1.00  0.00           H  \\nATOM    157 2HG  PRO A  10      12.847  12.515  -1.841  1.00  0.00           H  \\nATOM    158 1HD  PRO A  10      10.707  10.387  -1.455  1.00  0.00           H  \\nATOM    159 2HD  PRO A  10      10.522  12.175  -1.598  1.00  0.00           H  \\nATOM    160  N   ARG A  11      12.886   9.835  -6.115  1.00  0.00           N  \\nATOM    161  CA  ARG A  11      13.026   8.873  -7.192  1.00  0.00           C  \\nATOM    162  C   ARG A  11      14.441   8.327  -7.158  1.00  0.00           C  \\nATOM    163  O   ARG A  11      15.347   8.950  -6.606  1.00  0.00           O  \\nATOM    164  CB  ARG A  11      12.743   9.507  -8.539  1.00  0.00           C  \\nATOM    165  CG  ARG A  11      11.357  10.116  -8.682  1.00  0.00           C  \\nATOM    166  CD  ARG A  11      10.303   9.074  -8.726  1.00  0.00           C  \\nATOM    167  NE  ARG A  11       8.984   9.646  -8.959  1.00  0.00           N  \\nATOM    168  CZ  ARG A  11       7.844   8.932  -9.038  1.00  0.00           C  \\nATOM    169  NH1 ARG A  11       7.876   7.625  -8.901  1.00  0.00           N  \\nATOM    170  NH2 ARG A  11       6.693   9.546  -9.252  1.00  0.00           N  \\nATOM    171  H   ARG A  11      13.279  10.753  -6.262  1.00  0.00           H  \\nATOM    172  HA  ARG A  11      12.305   8.071  -7.036  1.00  0.00           H  \\nATOM    173 1HB  ARG A  11      13.470  10.295  -8.730  1.00  0.00           H  \\nATOM    174 2HB  ARG A  11      12.858   8.759  -9.324  1.00  0.00           H  \\nATOM    175 1HG  ARG A  11      11.158  10.769  -7.832  1.00  0.00           H  \\nATOM    176 2HG  ARG A  11      11.307  10.694  -9.604  1.00  0.00           H  \\nATOM    177 1HD  ARG A  11      10.518   8.374  -9.532  1.00  0.00           H  \\nATOM    178 2HD  ARG A  11      10.280   8.539  -7.776  1.00  0.00           H  \\nATOM    179  HE  ARG A  11       8.919  10.649  -9.069  1.00  0.00           H  \\nATOM    180 1HH1 ARG A  11       8.755   7.156  -8.737  1.00  0.00           H  \\nATOM    181 2HH1 ARG A  11       7.021   7.091  -8.960  1.00  0.00           H  \\nATOM    182 1HH2 ARG A  11       6.668  10.552  -9.357  1.00  0.00           H  \\nATOM    183 2HH2 ARG A  11       5.839   9.012  -9.311  1.00  0.00           H  \\nATOM    184  N   PHE A  12      14.617   7.150  -7.742  1.00  0.00           N  \\nATOM    185  CA  PHE A  12      15.949   6.579  -7.838  1.00  0.00           C  \\nATOM    186  C   PHE A  12      16.653   7.364  -8.919  1.00  0.00           C  \\nATOM    187  O   PHE A  12      16.189   7.397 -10.059  1.00  0.00           O  \\nATOM    188  CB  PHE A  12      15.917   5.099  -8.180  1.00  0.00           C  \\nATOM    189  CG  PHE A  12      17.272   4.464  -8.241  1.00  0.00           C  \\nATOM    190  CD1 PHE A  12      17.956   4.149  -7.081  1.00  0.00           C  \\nATOM    191  CD2 PHE A  12      17.866   4.180  -9.464  1.00  0.00           C  \\nATOM    192  CE1 PHE A  12      19.206   3.563  -7.136  1.00  0.00           C  \\nATOM    193  CE2 PHE A  12      19.114   3.594  -9.522  1.00  0.00           C  \\nATOM    194  CZ  PHE A  12      19.785   3.285  -8.357  1.00  0.00           C  \\nATOM    195  H   PHE A  12      13.837   6.673  -8.171  1.00  0.00           H  \\nATOM    196  HA  PHE A  12      16.444   6.691  -6.872  1.00  0.00           H  \\nATOM    197 1HB  PHE A  12      15.325   4.567  -7.437  1.00  0.00           H  \\nATOM    198 2HB  PHE A  12      15.433   4.959  -9.145  1.00  0.00           H  \\nATOM    199  HD1 PHE A  12      17.498   4.367  -6.117  1.00  0.00           H  \\nATOM    200  HD2 PHE A  12      17.336   4.425 -10.385  1.00  0.00           H  \\nATOM    201  HE1 PHE A  12      19.734   3.320  -6.215  1.00  0.00           H  \\nATOM    202  HE2 PHE A  12      19.570   3.375 -10.487  1.00  0.00           H  \\nATOM    203  HZ  PHE A  12      20.771   2.824  -8.401  1.00  0.00           H  \\nATOM    204  N   SER A  13      17.780   7.979  -8.597  1.00  0.00           N  \\nATOM    205  CA  SER A  13      18.446   8.760  -9.621  1.00  0.00           C  \\nATOM    206  C   SER A  13      19.538   7.917 -10.248  1.00  0.00           C  \\nATOM    207  O   SER A  13      20.607   7.744  -9.664  1.00  0.00           O  \\nATOM    208  CB  SER A  13      19.029  10.029  -9.037  1.00  0.00           C  \\nATOM    209  OG  SER A  13      19.680  10.785 -10.025  1.00  0.00           O  \\nATOM    210  H   SER A  13      18.153   7.914  -7.661  1.00  0.00           H  \\nATOM    211  HA  SER A  13      17.711   9.035 -10.379  1.00  0.00           H  \\nATOM    212 1HB  SER A  13      18.233  10.622  -8.587  1.00  0.00           H  \\nATOM    213 2HB  SER A  13      19.734   9.774  -8.247  1.00  0.00           H  \\nATOM    214  HG  SER A  13      20.378  10.223 -10.372  1.00  0.00           H  \\nATOM    215  N   GLY A  14      19.243   7.333 -11.402  1.00  0.00           N  \\nATOM    216  CA  GLY A  14      20.182   6.436 -12.025  1.00  0.00           C  \\nATOM    217  C   GLY A  14      20.902   6.893 -13.341  1.00  0.00           C  \\nATOM    218  O   GLY A  14      21.690   6.124 -13.896  1.00  0.00           O  \\nATOM    219  H   GLY A  14      18.355   7.515 -11.846  1.00  0.00           H  \\nATOM    220 1HA  GLY A  14      20.979   6.198 -11.321  1.00  0.00           H  \\nATOM    221 2HA  GLY A  14      19.682   5.501 -12.272  1.00  0.00           H  \\nATOM    222  N   PRO A  15      20.592   8.133 -13.888  1.00  0.00           N  \\nATOM    223  CA  PRO A  15      21.215   8.505 -15.121  1.00  0.00           C  \\nATOM    224  C   PRO A  15      22.676   8.545 -14.705  1.00  0.00           C  \\nATOM    225  O   PRO A  15      22.814   9.134 -13.639  1.00  0.00           O  \\nATOM    226  CB  PRO A  15      20.639   9.866 -15.467  1.00  0.00           C  \\nATOM    227  CG  PRO A  15      19.312   9.919 -14.702  1.00  0.00           C  \\nATOM    228  CD  PRO A  15      19.588   9.160 -13.412  1.00  0.00           C  \\nATOM    229  HA  PRO A  15      20.937   7.781 -15.900  1.00  0.00           H  \\nATOM    230 1HB  PRO A  15      21.340  10.661 -15.162  1.00  0.00           H  \\nATOM    231 2HB  PRO A  15      20.508   9.954 -16.555  1.00  0.00           H  \\nATOM    232 1HG  PRO A  15      19.017  10.963 -14.525  1.00  0.00           H  \\nATOM    233 2HG  PRO A  15      18.511   9.460 -15.300  1.00  0.00           H  \\nATOM    234 1HD  PRO A  15      20.010   9.832 -12.684  1.00  0.00           H  \\nATOM    235 2HD  PRO A  15      18.684   8.734 -13.051  1.00  0.00           H  \\nATOM    236  N   GLU A  16      23.636   8.080 -15.492  1.00  0.00           N  \\nATOM    237  CA  GLU A  16      25.064   8.066 -15.192  1.00  0.00           C  \\nATOM    238  C   GLU A  16      25.671   9.436 -14.897  1.00  0.00           C  \\nATOM    239  O   GLU A  16      26.438   9.738 -14.008  1.00  0.00           O  \\nATOM    240  CB  GLU A  16      25.825   7.437 -16.351  1.00  0.00           C  \\nATOM    241  CG  GLU A  16      27.310   7.248 -16.098  1.00  0.00           C  \\nATOM    242  CD  GLU A  16      28.009   6.535 -17.220  1.00  0.00           C  \\nATOM    243  OE1 GLU A  16      27.355   6.180 -18.173  1.00  0.00           O  \\nATOM    244  OE2 GLU A  16      29.198   6.343 -17.127  1.00  0.00           O  \\nATOM    245  H   GLU A  16      23.353   7.671 -16.371  1.00  0.00           H  \\nATOM    246  HA  GLU A  16      25.218   7.487 -14.322  1.00  0.00           H  \\nATOM    247 1HB  GLU A  16      25.398   6.460 -16.580  1.00  0.00           H  \\nATOM    248 2HB  GLU A  16      25.714   8.059 -17.239  1.00  0.00           H  \\nATOM    249 1HG  GLU A  16      27.772   8.225 -15.960  1.00  0.00           H  \\nATOM    250 2HG  GLU A  16      27.442   6.681 -15.175  1.00  0.00           H  \\nATOM    251  N   ILE A  17      25.122  10.530 -15.525  1.00  0.00           N  \\nATOM    252  CA  ILE A  17      25.956  11.732 -15.075  1.00  0.00           C  \\nATOM    253  C   ILE A  17      25.809  11.855 -13.502  1.00  0.00           C  \\nATOM    254  O   ILE A  17      26.844  11.929 -12.830  1.00  0.00           O  \\nATOM    255  CB  ILE A  17      25.506  13.031 -15.753  1.00  0.00           C  \\nATOM    256  CG1 ILE A  17      25.813  12.986 -17.248  1.00  0.00           C  \\nATOM    257  CG2 ILE A  17      26.177  14.225 -15.107  1.00  0.00           C  \\nATOM    258  CD1 ILE A  17      25.150  14.088 -18.041  1.00  0.00           C  \\nATOM    259  H   ILE A  17      24.375  10.536 -16.204  1.00  0.00           H  \\nATOM    260  HA  ILE A  17      26.997  11.552 -15.346  1.00  0.00           H  \\nATOM    261  HB  ILE A  17      24.425  13.137 -15.656  1.00  0.00           H  \\nATOM    262 1HG1 ILE A  17      26.889  13.055 -17.401  1.00  0.00           H  \\nATOM    263 2HG1 ILE A  17      25.486  12.029 -17.658  1.00  0.00           H  \\nATOM    264 1HG2 ILE A  17      25.847  15.138 -15.600  1.00  0.00           H  \\nATOM    265 2HG2 ILE A  17      25.911  14.266 -14.058  1.00  0.00           H  \\nATOM    266 3HG2 ILE A  17      27.258  14.131 -15.204  1.00  0.00           H  \\nATOM    267 1HD1 ILE A  17      25.415  13.989 -19.094  1.00  0.00           H  \\nATOM    268 2HD1 ILE A  17      24.067  14.016 -17.930  1.00  0.00           H  \\nATOM    269 3HD1 ILE A  17      25.489  15.055 -17.672  1.00  0.00           H  \\nATOM    270  N   THR A  18      24.557  11.813 -13.058  1.00  0.00           N  \\nATOM    271  CA  THR A  18      24.114  11.918 -11.659  1.00  0.00           C  \\nATOM    272  C   THR A  18      24.577  10.716 -10.863  1.00  0.00           C  \\nATOM    273  O   THR A  18      24.989  10.733  -9.704  1.00  0.00           O  \\nATOM    274  CB  THR A  18      22.604  12.040 -11.558  1.00  0.00           C  \\nATOM    275  OG1 THR A  18      22.000  10.913 -12.145  1.00  0.00           O  \\nATOM    276  CG2 THR A  18      22.124  13.288 -12.263  1.00  0.00           C  \\nATOM    277  H   THR A  18      23.830  11.750 -13.756  1.00  0.00           H  \\nATOM    278  HA  THR A  18      24.552  12.797 -11.229  1.00  0.00           H  \\nATOM    279  HB  THR A  18      22.312  12.087 -10.510  1.00  0.00           H  \\nATOM    280  HG1 THR A  18      22.255  10.858 -13.070  1.00  0.00           H  \\nATOM    281 1HG2 THR A  18      21.040  13.359 -12.180  1.00  0.00           H  \\nATOM    282 2HG2 THR A  18      22.582  14.164 -11.803  1.00  0.00           H  \\nATOM    283 3HG2 THR A  18      22.406  13.242 -13.316  1.00  0.00           H  \\nATOM    284  N   LEU A  19      24.686   9.580 -11.529  1.00  0.00           N  \\nATOM    285  CA  LEU A  19      25.117   8.419 -10.754  1.00  0.00           C  \\nATOM    286  C   LEU A  19      26.450   8.758 -10.136  1.00  0.00           C  \\nATOM    287  O   LEU A  19      26.491   8.670  -8.905  1.00  0.00           O  \\nATOM    288  CB  LEU A  19      25.236   7.190 -11.617  1.00  0.00           C  \\nATOM    289  CG  LEU A  19      25.741   5.945 -10.920  1.00  0.00           C  \\nATOM    290  CD1 LEU A  19      24.764   5.547  -9.821  1.00  0.00           C  \\nATOM    291  CD2 LEU A  19      25.905   4.828 -11.935  1.00  0.00           C  \\nATOM    292  H   LEU A  19      24.255   9.469 -12.430  1.00  0.00           H  \\nATOM    293  HA  LEU A  19      24.372   8.222  -9.983  1.00  0.00           H  \\nATOM    294 1HB  LEU A  19      24.255   6.961 -12.035  1.00  0.00           H  \\nATOM    295 2HB  LEU A  19      25.898   7.402 -12.417  1.00  0.00           H  \\nATOM    296  HG  LEU A  19      26.703   6.155 -10.451  1.00  0.00           H  \\nATOM    297 1HD1 LEU A  19      25.128   4.651  -9.319  1.00  0.00           H  \\nATOM    298 2HD1 LEU A  19      24.680   6.359  -9.097  1.00  0.00           H  \\nATOM    299 3HD1 LEU A  19      23.785   5.347 -10.258  1.00  0.00           H  \\nATOM    300 1HD2 LEU A  19      26.270   3.931 -11.433  1.00  0.00           H  \\nATOM    301 2HD2 LEU A  19      24.943   4.616 -12.403  1.00  0.00           H  \\nATOM    302 3HD2 LEU A  19      26.622   5.132 -12.700  1.00  0.00           H  \\nATOM    303  N   ILE A  20      27.352   9.217 -11.012  1.00  0.00           N  \\nATOM    304  CA  ILE A  20      28.706   9.560 -10.665  1.00  0.00           C  \\nATOM    305  C   ILE A  20      28.760  10.681  -9.651  1.00  0.00           C  \\nATOM    306  O   ILE A  20      29.514  10.412  -8.725  1.00  0.00           O  \\nATOM    307  CB  ILE A  20      29.498   9.962 -11.912  1.00  0.00           C  \\nATOM    308  CG1 ILE A  20      29.704   8.755 -12.823  1.00  0.00           C  \\nATOM    309  CG2 ILE A  20      30.835  10.572 -11.519  1.00  0.00           C  \\nATOM    310  CD1 ILE A  20      30.228   9.109 -14.191  1.00  0.00           C  \\nATOM    311  H   ILE A  20      27.078   9.279 -11.986  1.00  0.00           H  \\nATOM    312  HA  ILE A  20      29.182   8.685 -10.224  1.00  0.00           H  \\nATOM    313  HB  ILE A  20      28.932  10.690 -12.479  1.00  0.00           H  \\nATOM    314 1HG1 ILE A  20      30.404   8.064 -12.358  1.00  0.00           H  \\nATOM    315 2HG1 ILE A  20      28.755   8.226 -12.948  1.00  0.00           H  \\nATOM    316 1HG2 ILE A  20      31.385  10.851 -12.416  1.00  0.00           H  \\nATOM    317 2HG2 ILE A  20      30.666  11.458 -10.906  1.00  0.00           H  \\nATOM    318 3HG2 ILE A  20      31.415   9.844 -10.951  1.00  0.00           H  \\nATOM    319 1HD1 ILE A  20      30.349   8.199 -14.781  1.00  0.00           H  \\nATOM    320 2HD1 ILE A  20      29.522   9.776 -14.691  1.00  0.00           H  \\nATOM    321 3HD1 ILE A  20      31.191   9.606 -14.093  1.00  0.00           H  \\nATOM    322  N   ILE A  21      27.917  11.710  -9.831  1.00  0.00           N  \\nATOM    323  CA  ILE A  21      27.917  12.857  -8.929  1.00  0.00           C  \\nATOM    324  C   ILE A  21      27.664  12.462  -7.518  1.00  0.00           C  \\nATOM    325  O   ILE A  21      28.520  12.873  -6.735  1.00  0.00           O  \\nATOM    326  CB  ILE A  21      26.858  13.895  -9.348  1.00  0.00           C  \\nATOM    327  CG1 ILE A  21      27.247  14.551 -10.687  1.00  0.00           C  \\nATOM    328  CG2 ILE A  21      26.689  14.951  -8.263  1.00  0.00           C  \\nATOM    329  CD1 ILE A  21      26.132  15.364 -11.320  1.00  0.00           C  \\nATOM    330  H   ILE A  21      27.437  11.785 -10.722  1.00  0.00           H  \\nATOM    331  HA  ILE A  21      28.897  13.331  -8.973  1.00  0.00           H  \\nATOM    332  HB  ILE A  21      25.908  13.399  -9.506  1.00  0.00           H  \\nATOM    333 1HG1 ILE A  21      28.103  15.207 -10.535  1.00  0.00           H  \\nATOM    334 2HG1 ILE A  21      27.547  13.785 -11.391  1.00  0.00           H  \\nATOM    335 1HG2 ILE A  21      25.938  15.675  -8.575  1.00  0.00           H  \\nATOM    336 2HG2 ILE A  21      26.370  14.474  -7.339  1.00  0.00           H  \\nATOM    337 3HG2 ILE A  21      27.638  15.460  -8.100  1.00  0.00           H  \\nATOM    338 1HD1 ILE A  21      26.481  15.792 -12.257  1.00  0.00           H  \\nATOM    339 2HD1 ILE A  21      25.292  14.729 -11.510  1.00  0.00           H  \\nATOM    340 3HD1 ILE A  21      25.837  16.163 -10.644  1.00  0.00           H  \\nATOM    341  N   PHE A  22      26.628  11.680  -7.320  1.00  0.00           N  \\nATOM    342  CA  PHE A  22      26.147  11.174  -6.080  1.00  0.00           C  \\nATOM    343  C   PHE A  22      27.240  10.378  -5.431  1.00  0.00           C  \\nATOM    344  O   PHE A  22      27.427  10.762  -4.274  1.00  0.00           O  \\nATOM    345  CB  PHE A  22      24.911  10.307  -6.283  1.00  0.00           C  \\nATOM    346  CG  PHE A  22      24.377   9.711  -5.017  1.00  0.00           C  \\nATOM    347  CD1 PHE A  22      23.622  10.473  -4.140  1.00  0.00           C  \\nATOM    348  CD2 PHE A  22      24.630   8.384  -4.699  1.00  0.00           C  \\nATOM    349  CE1 PHE A  22      23.129   9.923  -2.971  1.00  0.00           C  \\nATOM    350  CE2 PHE A  22      24.138   7.832  -3.532  1.00  0.00           C  \\nATOM    351  CZ  PHE A  22      23.386   8.603  -2.667  1.00  0.00           C  \\nATOM    352  H   PHE A  22      26.082  11.447  -8.137  1.00  0.00           H  \\nATOM    353  HA  PHE A  22      25.864  12.017  -5.448  1.00  0.00           H  \\nATOM    354 1HB  PHE A  22      24.120  10.902  -6.738  1.00  0.00           H  \\nATOM    355 2HB  PHE A  22      25.144   9.497  -6.967  1.00  0.00           H  \\nATOM    356  HD1 PHE A  22      23.418  11.516  -4.381  1.00  0.00           H  \\nATOM    357  HD2 PHE A  22      25.224   7.775  -5.382  1.00  0.00           H  \\nATOM    358  HE1 PHE A  22      22.537  10.534  -2.290  1.00  0.00           H  \\nATOM    359  HE2 PHE A  22      24.342   6.788  -3.293  1.00  0.00           H  \\nATOM    360  HZ  PHE A  22      22.999   8.170  -1.746  1.00  0.00           H  \\nATOM    361  N   GLY A  23      27.886   9.490  -6.217  1.00  0.00           N  \\nATOM    362  CA  GLY A  23      28.910   8.610  -5.709  1.00  0.00           C  \\nATOM    363  C   GLY A  23      30.005   9.394  -5.050  1.00  0.00           C  \\nATOM    364  O   GLY A  23      30.211   9.001  -3.905  1.00  0.00           O  \\nATOM    365  H   GLY A  23      27.570   9.283  -7.158  1.00  0.00           H  \\nATOM    366 1HA  GLY A  23      28.470   7.914  -4.996  1.00  0.00           H  \\nATOM    367 2HA  GLY A  23      29.317   8.017  -6.527  1.00  0.00           H  \\nATOM    368  N   VAL A  24      30.441  10.466  -5.722  1.00  0.00           N  \\nATOM    369  CA  VAL A  24      31.529  11.313  -5.306  1.00  0.00           C  \\nATOM    370  C   VAL A  24      31.255  12.009  -4.026  1.00  0.00           C  \\nATOM    371  O   VAL A  24      32.165  11.881  -3.200  1.00  0.00           O  \\nATOM    372  CB  VAL A  24      31.825  12.370  -6.389  1.00  0.00           C  \\nATOM    373  CG1 VAL A  24      32.838  13.386  -5.879  1.00  0.00           C  \\nATOM    374  CG2 VAL A  24      32.331  11.692  -7.648  1.00  0.00           C  \\nATOM    375  H   VAL A  24      30.100  10.590  -6.669  1.00  0.00           H  \\nATOM    376  HA  VAL A  24      32.416  10.694  -5.168  1.00  0.00           H  \\nATOM    377  HB  VAL A  24      30.917  12.909  -6.610  1.00  0.00           H  \\nATOM    378 1HG1 VAL A  24      33.035  14.125  -6.656  1.00  0.00           H  \\nATOM    379 2HG1 VAL A  24      32.439  13.886  -4.996  1.00  0.00           H  \\nATOM    380 3HG1 VAL A  24      33.766  12.877  -5.621  1.00  0.00           H  \\nATOM    381 1HG2 VAL A  24      32.537  12.444  -8.408  1.00  0.00           H  \\nATOM    382 2HG2 VAL A  24      33.244  11.143  -7.423  1.00  0.00           H  \\nATOM    383 3HG2 VAL A  24      31.594  11.020  -8.008  1.00  0.00           H  \\nATOM    384  N   MET A  25      30.038  12.572  -3.929  1.00  0.00           N  \\nATOM    385  CA  MET A  25      29.677  13.309  -2.760  1.00  0.00           C  \\nATOM    386  C   MET A  25      29.721  12.432  -1.553  1.00  0.00           C  \\nATOM    387  O   MET A  25      30.374  12.882  -0.610  1.00  0.00           O  \\nATOM    388  CB  MET A  25      28.296  13.923  -2.928  1.00  0.00           C  \\nATOM    389  CG  MET A  25      28.226  15.041  -3.951  1.00  0.00           C  \\nATOM    390  SD  MET A  25      26.547  15.619  -4.234  1.00  0.00           S  \\nATOM    391  CE  MET A  25      26.202  16.433  -2.677  1.00  0.00           C  \\nATOM    392  H   MET A  25      29.443  12.590  -4.753  1.00  0.00           H  \\nATOM    393  HA  MET A  25      30.400  14.113  -2.622  1.00  0.00           H  \\nATOM    394 1HB  MET A  25      27.589  13.152  -3.228  1.00  0.00           H  \\nATOM    395 2HB  MET A  25      27.957  14.323  -1.972  1.00  0.00           H  \\nATOM    396 1HG  MET A  25      28.827  15.883  -3.610  1.00  0.00           H  \\nATOM    397 2HG  MET A  25      28.634  14.693  -4.898  1.00  0.00           H  \\nATOM    398 1HE  MET A  25      25.193  16.845  -2.697  1.00  0.00           H  \\nATOM    399 2HE  MET A  25      26.285  15.712  -1.862  1.00  0.00           H  \\nATOM    400 3HE  MET A  25      26.920  17.240  -2.522  1.00  0.00           H  \\nATOM    401  N   ALA A  26      29.170  11.222  -1.765  1.00  0.00           N  \\nATOM    402  CA  ALA A  26      29.021  10.244  -0.728  1.00  0.00           C  \\nATOM    403  C   ALA A  26      30.351   9.850  -0.208  1.00  0.00           C  \\nATOM    404  O   ALA A  26      30.373   9.930   1.010  1.00  0.00           O  \\nATOM    405  CB  ALA A  26      28.269   9.019  -1.231  1.00  0.00           C  \\nATOM    406  H   ALA A  26      28.716  11.000  -2.642  1.00  0.00           H  \\nATOM    407  HA  ALA A  26      28.447  10.694   0.081  1.00  0.00           H  \\nATOM    408 1HB  ALA A  26      28.192   8.285  -0.430  1.00  0.00           H  \\nATOM    409 2HB  ALA A  26      27.268   9.312  -1.553  1.00  0.00           H  \\nATOM    410 3HB  ALA A  26      28.799   8.589  -2.061  1.00  0.00           H  \\nATOM    411  N   GLY A  27      31.293   9.645  -1.121  1.00  0.00           N  \\nATOM    412  CA  GLY A  27      32.620   9.207  -0.861  1.00  0.00           C  \\nATOM    413  C   GLY A  27      33.309  10.147   0.077  1.00  0.00           C  \\nATOM    414  O   GLY A  27      33.800   9.561   1.047  1.00  0.00           O  \\nATOM    415  H   GLY A  27      31.019   9.588  -2.094  1.00  0.00           H  \\nATOM    416 1HA  GLY A  27      32.596   8.205  -0.434  1.00  0.00           H  \\nATOM    417 2HA  GLY A  27      33.172   9.144  -1.798  1.00  0.00           H  \\nATOM    418  N   ILE A  28      33.176  11.459  -0.212  1.00  0.00           N  \\nATOM    419  CA  ILE A  28      33.843  12.464   0.572  1.00  0.00           C  \\nATOM    420  C   ILE A  28      33.336  12.426   1.980  1.00  0.00           C  \\nATOM    421  O   ILE A  28      34.206  12.378   2.847  1.00  0.00           O  \\nATOM    422  CB  ILE A  28      33.633  13.863  -0.021  1.00  0.00           C  \\nATOM    423  CG1 ILE A  28      34.351  13.986  -1.363  1.00  0.00           C  \\nATOM    424  CG2 ILE A  28      34.121  14.930   0.948  1.00  0.00           C  \\nATOM    425  CD1 ILE A  28      33.976  15.220  -2.143  1.00  0.00           C  \\nATOM    426  H   ILE A  28      32.733  11.705  -1.093  1.00  0.00           H  \\nATOM    427  HA  ILE A  28      34.912  12.253   0.568  1.00  0.00           H  \\nATOM    428  HB  ILE A  28      32.576  14.019  -0.215  1.00  0.00           H  \\nATOM    429 1HG1 ILE A  28      35.427  13.999  -1.199  1.00  0.00           H  \\nATOM    430 2HG1 ILE A  28      34.126  13.114  -1.977  1.00  0.00           H  \\nATOM    431 1HG2 ILE A  28      33.965  15.915   0.512  1.00  0.00           H  \\nATOM    432 2HG2 ILE A  28      33.565  14.854   1.882  1.00  0.00           H  \\nATOM    433 3HG2 ILE A  28      35.183  14.784   1.145  1.00  0.00           H  \\nATOM    434 1HD1 ILE A  28      34.527  15.237  -3.084  1.00  0.00           H  \\nATOM    435 2HD1 ILE A  28      32.903  15.209  -2.350  1.00  0.00           H  \\nATOM    436 3HD1 ILE A  28      34.225  16.107  -1.563  1.00  0.00           H  \\nATOM    437  N   ILE A  29      32.005  12.308   2.083  1.00  0.00           N  \\nATOM    438  CA  ILE A  29      31.361  12.299   3.364  1.00  0.00           C  \\nATOM    439  C   ILE A  29      31.839  11.173   4.210  1.00  0.00           C  \\nATOM    440  O   ILE A  29      32.136  11.521   5.343  1.00  0.00           O  \\nATOM    441  CB  ILE A  29      29.834  12.206   3.212  1.00  0.00           C  \\nATOM    442  CG1 ILE A  29      29.281  13.489   2.600  1.00  0.00           C  \\nATOM    443  CG2 ILE A  29      29.182  11.930   4.556  1.00  0.00           C  \\nATOM    444  CD1 ILE A  29      27.842  13.382   2.157  1.00  0.00           C  \\nATOM    445  H   ILE A  29      31.446  12.391   1.238  1.00  0.00           H  \\nATOM    446  HA  ILE A  29      31.597  13.230   3.875  1.00  0.00           H  \\nATOM    447  HB  ILE A  29      29.593  11.419   2.544  1.00  0.00           H  \\nATOM    448 1HG1 ILE A  29      29.356  14.299   3.324  1.00  0.00           H  \\nATOM    449 2HG1 ILE A  29      29.882  13.768   1.736  1.00  0.00           H  \\nATOM    450 1HG2 ILE A  29      28.102  11.867   4.430  1.00  0.00           H  \\nATOM    451 2HG2 ILE A  29      29.555  10.989   4.954  1.00  0.00           H  \\nATOM    452 3HG2 ILE A  29      29.420  12.737   5.248  1.00  0.00           H  \\nATOM    453 1HD1 ILE A  29      27.521  14.333   1.732  1.00  0.00           H  \\nATOM    454 2HD1 ILE A  29      27.751  12.599   1.404  1.00  0.00           H  \\nATOM    455 3HD1 ILE A  29      27.216  13.138   3.012  1.00  0.00           H  \\nATOM    456  N   GLY A  30      31.948   9.995   3.597  1.00  0.00           N  \\nATOM    457  CA  GLY A  30      32.340   8.755   4.186  1.00  0.00           C  \\nATOM    458  C   GLY A  30      33.686   8.897   4.817  1.00  0.00           C  \\nATOM    459  O   GLY A  30      33.691   8.501   5.973  1.00  0.00           O  \\nATOM    460  H   GLY A  30      31.571   9.953   2.660  1.00  0.00           H  \\nATOM    461 1HA  GLY A  30      31.605   8.457   4.926  1.00  0.00           H  \\nATOM    462 2HA  GLY A  30      32.359   7.979   3.422  1.00  0.00           H  \\nATOM    463  N   THR A  31      34.588   9.538   4.072  1.00  0.00           N  \\nATOM    464  CA  THR A  31      35.941   9.714   4.510  1.00  0.00           C  \\nATOM    465  C   THR A  31      35.955  10.493   5.800  1.00  0.00           C  \\nATOM    466  O   THR A  31      36.556   9.928   6.708  1.00  0.00           O  \\nATOM    467  CB  THR A  31      36.786  10.431   3.447  1.00  0.00           C  \\nATOM    468  OG1 THR A  31      36.832   9.637   2.254  1.00  0.00           O  \\nATOM    469  CG2 THR A  31      38.197  10.658   3.954  1.00  0.00           C  \\nATOM    470  H   THR A  31      34.354   9.734   3.106  1.00  0.00           H  \\nATOM    471  HA  THR A  31      36.380   8.732   4.680  1.00  0.00           H  \\nATOM    472  HB  THR A  31      36.337  11.384   3.212  1.00  0.00           H  \\nATOM    473  HG1 THR A  31      35.940   9.509   1.921  1.00  0.00           H  \\nATOM    474 1HG2 THR A  31      38.781  11.167   3.188  1.00  0.00           H  \\nATOM    475 2HG2 THR A  31      38.166  11.270   4.854  1.00  0.00           H  \\nATOM    476 3HG2 THR A  31      38.659   9.699   4.183  1.00  0.00           H  \\nATOM    477  N   ILE A  32      35.139  11.563   5.810  1.00  0.00           N  \\nATOM    478  CA  ILE A  32      35.107  12.441   6.960  1.00  0.00           C  \\nATOM    479  C   ILE A  32      34.666  11.706   8.197  1.00  0.00           C  \\nATOM    480  O   ILE A  32      35.385  11.892   9.171  1.00  0.00           O  \\nATOM    481  CB  ILE A  32      34.170  13.634   6.714  1.00  0.00           C  \\nATOM    482  CG1 ILE A  32      34.747  14.551   5.638  1.00  0.00           C  \\nATOM    483  CG2 ILE A  32      33.938  14.402   8.003  1.00  0.00           C  \\nATOM    484  CD1 ILE A  32      33.774  15.589   5.136  1.00  0.00           C  \\nATOM    485  H   ILE A  32      34.710  11.835   4.931  1.00  0.00           H  \\nATOM    486  HA  ILE A  32      36.107  12.823   7.129  1.00  0.00           H  \\nATOM    487  HB  ILE A  32      33.233  13.281   6.348  1.00  0.00           H  \\nATOM    488 1HG1 ILE A  32      35.622  15.066   6.031  1.00  0.00           H  \\nATOM    489 2HG1 ILE A  32      35.075  13.953   4.787  1.00  0.00           H  \\nATOM    490 1HG2 ILE A  32      33.274  15.243   7.811  1.00  0.00           H  \\nATOM    491 2HG2 ILE A  32      33.484  13.742   8.743  1.00  0.00           H  \\nATOM    492 3HG2 ILE A  32      34.891  14.771   8.383  1.00  0.00           H  \\nATOM    493 1HD1 ILE A  32      34.256  16.202   4.374  1.00  0.00           H  \\nATOM    494 2HD1 ILE A  32      32.903  15.093   4.705  1.00  0.00           H  \\nATOM    495 3HD1 ILE A  32      33.459  16.222   5.963  1.00  0.00           H  \\nATOM    496  N   LEU A  33      33.626  10.891   8.025  1.00  0.00           N  \\nATOM    497  CA  LEU A  33      33.014  10.127   9.080  1.00  0.00           C  \\nATOM    498  C   LEU A  33      34.017   9.207   9.723  1.00  0.00           C  \\nATOM    499  O   LEU A  33      34.043   9.318  10.945  1.00  0.00           O  \\nATOM    500  CB  LEU A  33      31.836   9.312   8.533  1.00  0.00           C  \\nATOM    501  CG  LEU A  33      30.606  10.120   8.121  1.00  0.00           C  \\nATOM    502  CD1 LEU A  33      29.613   9.209   7.422  1.00  0.00           C  \\nATOM    503  CD2 LEU A  33      29.987  10.762   9.348  1.00  0.00           C  \\nATOM    504  H   LEU A  33      33.158  10.960   7.128  1.00  0.00           H  \\nATOM    505  HA  LEU A  33      32.638  10.818   9.834  1.00  0.00           H  \\nATOM    506 1HB  LEU A  33      32.166   8.769   7.683  1.00  0.00           H  \\nATOM    507 2HB  LEU A  33      31.523   8.599   9.294  1.00  0.00           H  \\nATOM    508  HG  LEU A  33      30.900  10.892   7.418  1.00  0.00           H  \\nATOM    509 1HD1 LEU A  33      28.736   9.785   7.127  1.00  0.00           H  \\nATOM    510 2HD1 LEU A  33      30.074   8.780   6.541  1.00  0.00           H  \\nATOM    511 3HD1 LEU A  33      29.312   8.411   8.099  1.00  0.00           H  \\nATOM    512 1HD2 LEU A  33      29.110  11.339   9.053  1.00  0.00           H  \\nATOM    513 2HD2 LEU A  33      29.691   9.986  10.054  1.00  0.00           H  \\nATOM    514 3HD2 LEU A  33      30.716  11.423   9.819  1.00  0.00           H  \\nATOM    515  N   LEU A  34      34.803   8.508   8.885  1.00  0.00           N  \\nATOM    516  CA  LEU A  34      35.787   7.543   9.321  1.00  0.00           C  \\nATOM    517  C   LEU A  34      36.803   8.218  10.213  1.00  0.00           C  \\nATOM    518  O   LEU A  34      37.027   7.622  11.265  1.00  0.00           O  \\nATOM    519  CB  LEU A  34      36.484   6.902   8.116  1.00  0.00           C  \\nATOM    520  CG  LEU A  34      35.623   5.970   7.271  1.00  0.00           C  \\nATOM    521  CD1 LEU A  34      36.388   5.565   6.023  1.00  0.00           C  \\nATOM    522  CD2 LEU A  34      35.236   4.754   8.094  1.00  0.00           C  \\nATOM    523  H   LEU A  34      34.589   8.560   7.896  1.00  0.00           H  \\nATOM    524  HA  LEU A  34      35.276   6.757   9.880  1.00  0.00           H  \\nATOM    525 1HB  LEU A  34      36.846   7.686   7.473  1.00  0.00           H  \\nATOM    526 2HB  LEU A  34      37.339   6.330   8.473  1.00  0.00           H  \\nATOM    527  HG  LEU A  34      34.728   6.492   6.956  1.00  0.00           H  \\nATOM    528 1HD1 LEU A  34      35.772   4.899   5.420  1.00  0.00           H  \\nATOM    529 2HD1 LEU A  34      36.636   6.453   5.444  1.00  0.00           H  \\nATOM    530 3HD1 LEU A  34      37.304   5.051   6.310  1.00  0.00           H  \\nATOM    531 1HD2 LEU A  34      34.619   4.087   7.490  1.00  0.00           H  \\nATOM    532 2HD2 LEU A  34      36.136   4.227   8.411  1.00  0.00           H  \\nATOM    533 3HD2 LEU A  34      34.673   5.072   8.973  1.00  0.00           H  \\nATOM    534  N   ILE A  35      37.195   9.432   9.779  1.00  0.00           N  \\nATOM    535  CA  ILE A  35      38.213  10.180  10.482  1.00  0.00           C  \\nATOM    536  C   ILE A  35      37.736  10.460  11.868  1.00  0.00           C  \\nATOM    537  O   ILE A  35      38.540  10.095  12.727  1.00  0.00           O  \\nATOM    538  CB  ILE A  35      38.546  11.501   9.765  1.00  0.00           C  \\nATOM    539  CG1 ILE A  35      39.239  11.224   8.430  1.00  0.00           C  \\nATOM    540  CG2 ILE A  35      39.416  12.380  10.648  1.00  0.00           C  \\nATOM    541  CD1 ILE A  35      39.349  12.435   7.538  1.00  0.00           C  \\nATOM    542  H   ILE A  35      36.932   9.723   8.843  1.00  0.00           H  \\nATOM    543  HA  ILE A  35      39.121   9.580  10.517  1.00  0.00           H  \\nATOM    544  HB  ILE A  35      37.647  12.019   9.542  1.00  0.00           H  \\nATOM    545 1HG1 ILE A  35      40.242  10.843   8.614  1.00  0.00           H  \\nATOM    546 2HG1 ILE A  35      38.694  10.455   7.892  1.00  0.00           H  \\nATOM    547 1HG2 ILE A  35      39.642  13.309  10.126  1.00  0.00           H  \\nATOM    548 2HG2 ILE A  35      38.886  12.604  11.573  1.00  0.00           H  \\nATOM    549 3HG2 ILE A  35      40.344  11.858  10.878  1.00  0.00           H  \\nATOM    550 1HD1 ILE A  35      39.851  12.160   6.611  1.00  0.00           H  \\nATOM    551 2HD1 ILE A  35      38.357  12.811   7.312  1.00  0.00           H  \\nATOM    552 3HD1 ILE A  35      39.923  13.209   8.044  1.00  0.00           H  \\nATOM    553  N   SER A  36      36.462  10.885  11.962  1.00  0.00           N  \\nATOM    554  CA  SER A  36      35.904  11.232  13.238  1.00  0.00           C  \\nATOM    555  C   SER A  36      35.873  10.058  14.160  1.00  0.00           C  \\nATOM    556  O   SER A  36      36.269  10.363  15.284  1.00  0.00           O  \\nATOM    557  CB  SER A  36      34.502  11.780  13.063  1.00  0.00           C  \\nATOM    558  OG  SER A  36      34.522  12.998  12.371  1.00  0.00           O  \\nATOM    559  H   SER A  36      35.991  11.122  11.096  1.00  0.00           H  \\nATOM    560  HA  SER A  36      36.528  12.003  13.690  1.00  0.00           H  \\nATOM    561 1HB  SER A  36      33.900  11.065  12.522  1.00  0.00           H  \\nATOM    562 2HB  SER A  36      34.042  11.922  14.039  1.00  0.00           H  \\nATOM    563  HG  SER A  36      34.919  12.810  11.516  1.00  0.00           H  \\nATOM    564  N   TYR A  37      35.561   8.876  13.600  1.00  0.00           N  \\nATOM    565  CA  TYR A  37      35.453   7.701  14.418  1.00  0.00           C  \\nATOM    566  C   TYR A  37      36.709   7.421  15.154  1.00  0.00           C  \\nATOM    567  O   TYR A  37      36.483   7.279  16.353  1.00  0.00           O  \\nATOM    568  CB  TYR A  37      35.071   6.494  13.571  1.00  0.00           C  \\nATOM    569  CG  TYR A  37      34.960   5.209  14.357  1.00  0.00           C  \\nATOM    570  CD1 TYR A  37      33.825   4.954  15.110  1.00  0.00           C  \\nATOM    571  CD2 TYR A  37      35.994   4.284  14.324  1.00  0.00           C  \\nATOM    572  CE1 TYR A  37      33.723   3.778  15.829  1.00  0.00           C  \\nATOM    573  CE2 TYR A  37      35.892   3.110  15.042  1.00  0.00           C  \\nATOM    574  CZ  TYR A  37      34.763   2.855  15.792  1.00  0.00           C  \\nATOM    575  OH  TYR A  37      34.662   1.684  16.508  1.00  0.00           O  \\nATOM    576  H   TYR A  37      35.199   8.914  12.656  1.00  0.00           H  \\nATOM    577  HA  TYR A  37      34.667   7.863  15.155  1.00  0.00           H  \\nATOM    578 1HB  TYR A  37      34.113   6.679  13.084  1.00  0.00           H  \\nATOM    579 2HB  TYR A  37      35.808   6.351  12.794  1.00  0.00           H  \\nATOM    580  HD1 TYR A  37      33.013   5.680  15.135  1.00  0.00           H  \\nATOM    581  HD2 TYR A  37      36.888   4.486  13.732  1.00  0.00           H  \\nATOM    582  HE1 TYR A  37      32.831   3.577  16.421  1.00  0.00           H  \\nATOM    583  HE2 TYR A  37      36.705   2.383  15.018  1.00  0.00           H  \\nATOM    584  HH  TYR A  37      35.456   1.161  16.377  1.00  0.00           H  \\nATOM    585  N   SER A  38      37.834   7.499  14.439  1.00  0.00           N  \\nATOM    586  CA  SER A  38      39.096   7.223  15.069  1.00  0.00           C  \\nATOM    587  C   SER A  38      39.422   8.338  16.019  1.00  0.00           C  \\nATOM    588  O   SER A  38      39.796   8.248  17.182  1.00  0.00           O  \\nATOM    589  CB  SER A  38      40.183   7.071  14.032  1.00  0.00           C  \\nATOM    590  OG  SER A  38      40.401   8.280  13.352  1.00  0.00           O  \\nATOM    591  H   SER A  38      37.722   7.670  13.443  1.00  0.00           H  \\nATOM    592  HA  SER A  38      39.009   6.286  15.622  1.00  0.00           H  \\nATOM    593 1HB  SER A  38      41.104   6.752  14.514  1.00  0.00           H  \\nATOM    594 2HB  SER A  38      39.901   6.297  13.320  1.00  0.00           H  \\nATOM    595  HG  SER A  38      39.566   8.503  12.934  1.00  0.00           H  \\nATOM    596  N   ILE A  39      38.994   9.590  15.672  1.00  0.00           N  \\nATOM    597  CA  ILE A  39      39.422  10.550  16.696  1.00  0.00           C  \\nATOM    598  C   ILE A  39      38.775  10.186  18.030  1.00  0.00           C  \\nATOM    599  O   ILE A  39      39.406  10.245  19.086  1.00  0.00           O  \\nATOM    600  CB  ILE A  39      39.052  11.983  16.296  1.00  0.00           C  \\nATOM    601  CG1 ILE A  39      39.878  12.429  15.093  1.00  0.00           C  \\nATOM    602  CG2 ILE A  39      39.255  12.929  17.463  1.00  0.00           C  \\nATOM    603  CD1 ILE A  39      39.399  13.710  14.469  1.00  0.00           C  \\nATOM    604  H   ILE A  39      38.657   9.815  14.746  1.00  0.00           H  \\nATOM    605  HA  ILE A  39      40.506  10.490  16.796  1.00  0.00           H  \\nATOM    606  HB  ILE A  39      38.011  12.017  15.992  1.00  0.00           H  \\nATOM    607 1HG1 ILE A  39      40.915  12.563  15.394  1.00  0.00           H  \\nATOM    608 2HG1 ILE A  39      39.858  11.651  14.330  1.00  0.00           H  \\nATOM    609 1HG2 ILE A  39      38.989  13.941  17.162  1.00  0.00           H  \\nATOM    610 2HG2 ILE A  39      38.623  12.620  18.296  1.00  0.00           H  \\nATOM    611 3HG2 ILE A  39      40.300  12.906  17.773  1.00  0.00           H  \\nATOM    612 1HD1 ILE A  39      40.035  13.962  13.621  1.00  0.00           H  \\nATOM    613 2HD1 ILE A  39      38.370  13.586  14.128  1.00  0.00           H  \\nATOM    614 3HD1 ILE A  39      39.443  14.511  15.205  1.00  0.00           H  \\nATOM    615  N   ARG A  40      37.510   9.765  17.984  1.00  0.00           N  \\nATOM    616  CA  ARG A  40      36.805   9.426  19.195  1.00  0.00           C  \\nATOM    617  C   ARG A  40      37.229   8.135  19.858  1.00  0.00           C  \\nATOM    618  O   ARG A  40      37.352   8.064  21.081  1.00  0.00           O  \\nATOM    619  CB  ARG A  40      35.318   9.349  18.903  1.00  0.00           C  \\nATOM    620  CG  ARG A  40      34.451   9.011  20.098  1.00  0.00           C  \\nATOM    621  CD  ARG A  40      34.548  10.046  21.156  1.00  0.00           C  \\nATOM    622  NE  ARG A  40      33.683   9.749  22.287  1.00  0.00           N  \\nATOM    623  CZ  ARG A  40      33.775  10.336  23.496  1.00  0.00           C  \\nATOM    624  NH1 ARG A  40      34.696  11.248  23.715  1.00  0.00           N  \\nATOM    625  NH2 ARG A  40      32.939   9.995  24.462  1.00  0.00           N  \\nATOM    626  H   ARG A  40      37.036   9.805  17.091  1.00  0.00           H  \\nATOM    627  HA  ARG A  40      36.988  10.208  19.932  1.00  0.00           H  \\nATOM    628 1HB  ARG A  40      34.975  10.303  18.507  1.00  0.00           H  \\nATOM    629 2HB  ARG A  40      35.136   8.593  18.140  1.00  0.00           H  \\nATOM    630 1HG  ARG A  40      33.410   8.939  19.783  1.00  0.00           H  \\nATOM    631 2HG  ARG A  40      34.769   8.057  20.521  1.00  0.00           H  \\nATOM    632 1HD  ARG A  40      35.574  10.102  21.517  1.00  0.00           H  \\nATOM    633 2HD  ARG A  40      34.254  11.012  20.747  1.00  0.00           H  \\nATOM    634  HE  ARG A  40      32.961   9.052  22.156  1.00  0.00           H  \\nATOM    635 1HH1 ARG A  40      35.335  11.508  22.977  1.00  0.00           H  \\nATOM    636 2HH1 ARG A  40      34.766  11.688  24.622  1.00  0.00           H  \\nATOM    637 1HH2 ARG A  40      32.231   9.294  24.293  1.00  0.00           H  \\nATOM    638 2HH2 ARG A  40      33.008  10.435  25.367  1.00  0.00           H  \\nATOM    639  N   ARG A  41      37.469   7.104  19.052  1.00  0.00           N  \\nATOM    640  CA  ARG A  41      37.814   5.797  19.566  1.00  0.00           C  \\nATOM    641  C   ARG A  41      39.212   5.753  20.138  1.00  0.00           C  \\nATOM    642  O   ARG A  41      39.448   5.114  21.163  1.00  0.00           O  \\nATOM    643  CB  ARG A  41      37.692   4.751  18.468  1.00  0.00           C  \\nATOM    644  CG  ARG A  41      38.019   3.332  18.899  1.00  0.00           C  \\nATOM    645  CD  ARG A  41      37.086   2.848  19.949  1.00  0.00           C  \\nATOM    646  NE  ARG A  41      37.369   1.472  20.333  1.00  0.00           N  \\nATOM    647  CZ  ARG A  41      36.891   0.873  21.445  1.00  0.00           C  \\nATOM    648  NH1 ARG A  41      36.113   1.540  22.268  1.00  0.00           N  \\nATOM    649  NH2 ARG A  41      37.207  -0.384  21.706  1.00  0.00           N  \\nATOM    650  H   ARG A  41      37.298   7.189  18.064  1.00  0.00           H  \\nATOM    651  HA  ARG A  41      37.117   5.545  20.366  1.00  0.00           H  \\nATOM    652 1HB  ARG A  41      36.674   4.750  18.078  1.00  0.00           H  \\nATOM    653 2HB  ARG A  41      38.358   5.008  17.645  1.00  0.00           H  \\nATOM    654 1HG  ARG A  41      37.945   2.665  18.040  1.00  0.00           H  \\nATOM    655 2HG  ARG A  41      39.034   3.298  19.299  1.00  0.00           H  \\nATOM    656 1HD  ARG A  41      37.179   3.475  20.835  1.00  0.00           H  \\nATOM    657 2HD  ARG A  41      36.063   2.894  19.577  1.00  0.00           H  \\nATOM    658  HE  ARG A  41      37.965   0.926  19.725  1.00  0.00           H  \\nATOM    659 1HH1 ARG A  41      35.872   2.501  22.068  1.00  0.00           H  \\nATOM    660 2HH1 ARG A  41      35.755   1.092  23.099  1.00  0.00           H  \\nATOM    661 1HH2 ARG A  41      37.805  -0.897  21.073  1.00  0.00           H  \\nATOM    662 2HH2 ARG A  41      36.849  -0.831  22.537  1.00  0.00           H  \\nATOM    663  N   LEU A  42      40.126   6.446  19.478  1.00  0.00           N  \\nATOM    664  CA  LEU A  42      41.504   6.475  19.919  1.00  0.00           C  \\nATOM    665  C   LEU A  42      41.711   7.665  20.846  1.00  0.00           C  \\nATOM    666  O   LEU A  42      42.176   7.518  21.976  1.00  0.00           O  \\nATOM    667  OXT LEU A  42      41.413   8.796  20.469  1.00  0.00           O  \\nATOM    668  CB  LEU A  42      42.444   6.559  18.718  1.00  0.00           C  \\nATOM    669  CG  LEU A  42      42.380   5.384  17.744  1.00  0.00           C  \\nATOM    670  CD1 LEU A  42      43.298   5.654  16.563  1.00  0.00           C  \\nATOM    671  CD2 LEU A  42      42.778   4.108  18.463  1.00  0.00           C  \\nATOM    672  H   LEU A  42      39.948   6.959  18.629  1.00  0.00           H  \\nATOM    673  HA  LEU A  42      41.713   5.554  20.462  1.00  0.00           H  \\nATOM    674 1HB  LEU A  42      42.213   7.466  18.159  1.00  0.00           H  \\nATOM    675 2HB  LEU A  42      43.468   6.633  19.082  1.00  0.00           H  \\nATOM    676  HG  LEU A  42      41.361   5.283  17.359  1.00  0.00           H  \\nATOM    677 1HD1 LEU A  42      43.253   4.816  15.868  1.00  0.00           H  \\nATOM    678 2HD1 LEU A  42      42.979   6.564  16.055  1.00  0.00           H  \\nATOM    679 3HD1 LEU A  42      44.320   5.776  16.918  1.00  0.00           H  \\nATOM    680 1HD2 LEU A  42      42.732   3.269  17.768  1.00  0.00           H  \\nATOM    681 2HD2 LEU A  42      43.793   4.207  18.846  1.00  0.00           H  \\nATOM    682 3HD2 LEU A  42      42.092   3.929  19.294  1.00  0.00           H  \\nTER                                                                             \\n##Begin comments##\\nMembraneInfo1 MembraneCenter   31.246   8.804  -0.232\\nMembraneInfo2 MembraneNormal   -0.323   0.059  -0.944\\nMembraneInfo3 ATOM   9999  X   MEM A 999      31.246   8.804  -0.232\\nMembraneInfo4 ATOM   9999  Y   MEM A 999      26.396   9.689 -14.398\\nMembraneInfo5 ATOM   9999  Z   MEM A 999      36.097   7.919  13.935\\n##End comments##\\n\\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\nlabel fa_atr fa_rep fa_intra_rep fa_mbenv fa_mbsolv fa_pair hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_ss_dst dslf_cs_ang dslf_ss_dih dslf_ca_dih atom_pair_constraint rama omega fa_dun p_aa_pp ref cart_bonded Menv_smooth total\\nweights 0.8 0.44 0.004 0.3 0.35 0.49 1.17 1.17 2.34 2.2 0.5 2 5 5 0.5 0.2 0.5 0.56 0.32 1 0.5 0.5 NA\\npose -133.204 18.5293 0.51391 13.033 19.6014 -0.35959 -46.0647 0 -1.32037 0 0 0 0 0 3.14625 -4.40959 -1.85384 25.4832 -2.73453 -4.81 83.8735 -4.86075 -35.4365\\nASP:NtermProteinFull_1 -0.8812 0.03743 0.00881 0.20092 0.60858 -0.06886 0 0 0 0 0 0 0 0 0 0 -0.00444 1.68137 0 -0.67 0.01721 -0.47447 0.45534\\nASN_2 -1.19997 0.04721 0.00545 0.09386 0.61421 -0.06886 0 0 0 0 0 0 0 0 0 -0.20887 -0.06538 1.27439 -0.29817 -0.89 0.06512 -0.24228 -0.8733\\nGLY_3 -1.81568 0.03233 0.0001 0.21393 0.32191 0 0 0 0 0 0 0 0 0 0 0.20176 -0.13284 0 -0.80782 -0.17 0.10881 1.13347 -0.91403\\nGLU_4 -3.67394 0.45303 0.0061 1.66872 0.61829 -0.04331 0 0 -0.66018 0 0 0 0 0 0 -0.01485 -0.02709 2.10747 0.09166 -0.81 0.21631 0.5479 0.48009\\nTRP_5 -3.0895 0.25889 0.01904 0.71348 0.33786 0 0 0 0 0 0 0 0 0 0 -0.34807 -0.13453 0.72036 0.03411 0.91 0.19796 -1.07633 -1.45672\\nILE_6 -3.41931 0.51227 0.02918 -0.24913 0.34045 0 0 0 0 0 0 0 0 0 0 -0.02471 -0.0413 1.27233 0.29943 0.24 0.45945 -0.85696 -1.4383\\nGLN_7 -3.69205 0.22846 0.00567 1.01614 0.54269 -0.04331 0 0 -0.66018 0 0 0 0 0 0 -0.16153 -0.04536 2.03188 -0.03889 -0.97 0.64349 -0.13084 -1.27385\\nLEU_8 -3.54818 0.17229 0.01229 0.22718 0.36021 0 0 0 0 0 0 0 0 0 0 -0.01281 -0.03945 0.36171 -0.09974 -0.1 0.2947 -0.23339 -2.60518\\nVAL_9 -1.84956 0.39364 0.01636 -0.04219 0.3013 0 0 0 0 0 0 0 0 0 0 -0.10662 -0.04099 0.30634 -0.25392 0.29 1.32864 -0.26398 0.07902\\nPRO_10 -1.26129 0.36307 0.0021 -0.34493 0.18842 0 0 0 0 0 0 0 0 0 0 -0.22533 -0.10212 0.04599 -0.47478 0.02 1.72771 1.68253 1.62137\\nARG_11 -1.63291 0.03172 0.01243 1.44491 0.25504 0 0 0 0 0 0 0 0 0 0 -0.02125 0.00078 1.58813 -0.02273 -0.98 0.58324 -1.12468 0.13469\\nPHE_12 -1.53114 0.05915 0.01765 -0.13696 0.18359 0 0 0 0 0 0 0 0 0 0 -0.10723 -0.07209 1.45649 0.02143 0.63 0.40197 -0.89708 0.02581\\nSER_13 -1.91931 0.12437 0.00223 1.38431 0.26082 -0.04604 0 0 0 0 0 0 0 0 0 0.35871 -0.05476 0.3125 0.26376 -0.37 0.37033 0.22997 0.91688\\nGLY_14 -1.56416 1.19084 7e-05 0.43443 0.31006 0 0 0 0 0 0 0 0 0 0 0.08099 -0.08511 0 0.35659 -0.17 2.26647 0.84949 3.66968\\nPRO_15 -3.01901 1.42109 0.00625 0.03984 0.58601 0 0 0 0 0 0 0 0 0 0 -0.26335 -0.06729 0.24461 -0.38492 0.02 4.07814 -0.0863 2.57508\\nGLU_16 -3.16976 1.44294 0.00699 0.75059 0.53041 0 0 0 0 0 0 0 0 0 0 -0.3423 -0.10944 2.49511 -0.17141 -0.81 4.6598 0.54816 5.83109\\nILE_17 -2.96934 1.31148 0.01914 0.24509 0.4145 0 0 0 0 0 0 0 0 0 0 -0.07495 -0.0514 0.05137 -0.22534 0.24 3.78576 0.76207 3.50838\\nTHR_18 -4.67974 0.8269 0.00505 1.00136 0.73201 -0.04604 0 0 0 0 0 0 0 0 0 -0.19807 -0.04565 0.23534 0.0787 -0.27 2.23611 0.42631 0.30227\\nLEU_19 -4.18292 0.76821 0.01588 -0.20728 0.52403 0 0 0 0 0 0 0 0 0 0.3127 -0.0776 -0.00204 0.22853 -0.09066 -0.1 2.19306 -0.75613 -1.37422\\nILE_20 -4.2199 0.40024 0.02514 -0.11028 0.47671 0 0 0 0 0 0 0 0 0 0 -0.17508 -0.05449 0.08958 -0.17222 0.24 2.50203 -0.81275 -1.81101\\nILE_21 -4.75324 0.53115 0.02405 -0.17224 0.45335 0 0 0 0 0 0 0 0 0 0 -0.0583 -0.01291 0.06376 -0.18209 0.24 2.57585 -0.7762 -2.06683\\nPHE_22 -3.56332 0.20102 0.01847 0.00897 0.35512 0 0 0 0 0 0 0 0 0 0.29334 -0.15042 -0.00532 1.11516 -0.18203 0.63 2.78826 -0.85394 0.6553\\nGLY_23 -2.64764 0.19947 0.00048 0.90403 0.27185 0 0 0 0 0 0 0 0 0 0.32058 0.07334 -0.0175 0 0.28323 -0.17 3.35163 1.1475 3.71697\\nVAL_24 -3.71703 0.45312 0.01828 -0.00285 0.36828 0 0 0 0 0 0 0 0 0 0 -0.14387 -0.05187 -0.01962 -0.16555 0.29 2.84638 -0.3403 -0.46505\\nMET_25 -4.11805 0.2132 0.00705 0.20459 0.40197 0 0 0 0 0 0 0 0 0 0 -0.17829 0.00093 0.8954 0.02295 -0.34 2.41801 -0.08931 -0.56153\\nALA_26 -3.33029 0.47514 0.00181 0.4956 0.34027 0 0 0 0 0 0 0 0 0 0.30151 -0.22741 -0.03388 0 -0.07522 0.16 3.28653 0.61198 2.00602\\nGLY_27 -2.50118 0.13833 0.00043 0.9042 0.2563 0 0 0 0 0 0 0 0 0 0.3127 0.08575 -0.02999 0 0.27193 -0.17 3.54377 1.10311 3.91535\\nILE_28 -4.02139 0.29903 0.02254 -0.2403 0.39894 0 0 0 0 0 0 0 0 0 0 -0.15584 -0.0529 0.06409 -0.20478 0.24 2.39861 -0.81275 -2.06475\\nILE_29 -4.72961 0.63369 0.02528 -0.24032 0.45649 0 0 0 0 0 0 0 0 0 0 -0.15346 -0.03193 0.07165 -0.18618 0.24 2.29648 -0.7762 -2.39411\\nGLY_30 -3.27799 0.41572 0.00045 0.90405 0.33037 0 0 0 0 0 0 0 0 0 0.63833 0.05919 -0.04993 0 0.28854 -0.17 2.95836 1.09951 3.19661\\nTHR_31 -3.91161 0.39671 0.00811 1.39446 0.40126 0 0 0 0 0 0 0 0 0 0.32058 -0.06251 -0.01783 0.00657 0.00825 -0.27 2.98322 0.65874 1.91596\\nILE_32 -4.48193 0.55526 0.02429 -0.24789 0.44959 0 0 0 0 0 0 0 0 0 0 -0.15662 -0.04132 0.06643 -0.19496 0.24 2.47212 -0.79691 -2.11194\\nLEU_33 -4.21751 0.52847 0.01478 -0.2695 0.4453 0 0 0 0 0 0 0 0 0 0 -0.17762 -0.02661 0.4769 -0.11139 -0.1 2.26238 -0.7971 -1.9719\\nLEU_34 -3.70315 0.42801 0.01498 -0.32112 0.45417 0 0 0 0 0 0 0 0 0 0.30151 -0.22678 -0.03241 0.29478 -0.12759 -0.1 2.3894 -0.82365 -1.45186\\nILE_35 -4.50459 0.46524 0.02165 -0.38204 0.63195 0 0 0 0 0 0 0 0 0 0 -0.14298 -0.03537 0.05611 -0.20746 0.24 2.6883 -0.79369 -1.96287\\nSER_36 -3.60748 0.32969 0.00177 1.1487 0.64266 -0.02158 0 0 0 0 0 0 0 0 0 -0.17502 -0.05642 0.38896 0.14356 -0.37 3.15475 0.21548 1.79507\\nTYR_37 -3.89391 0.32948 0.01903 0.16036 0.72315 0 0 0 0 0 0 0 0 0 0 -0.1376 -0.02297 1.15229 -0.13909 0.51 3.36784 -0.05565 2.01294\\nSER_38 -4.04219 0.80239 0.00129 0.48447 0.99577 -0.02158 0 0 0 0 0 0 0 0 0.34499 -0.22894 -0.03858 0.35898 0.12092 -0.37 3.92083 0.27404 2.60239\\nILE_39 -3.43427 0.749 0.02201 -0.1292 0.81194 0 0 0 0 0 0 0 0 0 0 -0.21125 -0.07176 0.19036 -0.10577 0.24 2.6186 0.52484 1.20451\\nARG_40 -2.38993 0.20285 0.01922 0.04752 0.63795 0 0 0 0 0 0 0 0 0 0 -0.13801 -0.04016 1.63828 -0.03883 -0.98 0.65317 -2.50356 -2.89148\\nARG_41 -2.73166 0.04311 0.01737 0.04145 0.54951 0 0 0 0 0 0 0 0 0 0 -0.18183 -0.01013 1.71744 -0.05806 -0.98 0.41158 -2.14879 -3.33\\nLEU:CtermProteinFull_42 -2.30674 0.06368 0.01464 -0.00396 0.71812 0 0 0 0 0 0 0 0 0 0 0 0 0.44217 0 -0.1 0.35107 1.84739 1.02637\\n#END_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\n\\n\",\"pdb\");\n\tviewer_17294480429883013.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_17294480429883013.zoomTo();\nviewer_17294480429883013.render();\n});\n</script>","text/html":["<div id=\"3dmolviewer_17294480429883013\"  style=\"position: relative; width: 640px; height: 480px;\">\n","        <p id=\"3dmolwarning_17294480429883013\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n","        </div>\n","<script>\n","\n","var loadScriptAsync = function(uri){\n","  return new Promise((resolve, reject) => {\n","    //this is to ignore the existence of requirejs amd\n","    var savedexports, savedmodule;\n","    if (typeof exports !== 'undefined') savedexports = exports;\n","    else exports = {}\n","    if (typeof module !== 'undefined') savedmodule = module;\n","    else module = {}\n","\n","    var tag = document.createElement('script');\n","    tag.src = uri;\n","    tag.async = true;\n","    tag.onload = () => {\n","        exports = savedexports;\n","        module = savedmodule;\n","        resolve();\n","    };\n","  var firstScriptTag = document.getElementsByTagName('script')[0];\n","  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n","});\n","};\n","\n","if(typeof $3Dmolpromise === 'undefined') {\n","$3Dmolpromise = null;\n","  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.0/3Dmol-min.js');\n","}\n","\n","var viewer_17294480429883013 = null;\n","var warn = document.getElementById(\"3dmolwarning_17294480429883013\");\n","if(warn) {\n","    warn.parentNode.removeChild(warn);\n","}\n","$3Dmolpromise.then(function() {\n","viewer_17294480429883013 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17294480429883013\"),{backgroundColor:\"white\"});\n","viewer_17294480429883013.zoomTo();\n","\tviewer_17294480429883013.setBackgroundColor(\"white\");\n","\tviewer_17294480429883013.addModel(\"HEADER    MEMBRANE PROTEIN                        28-OCT-15   5EH6              \\nTITLE     CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE MONOMER IN       \\nTITLE    2 LIPIDIC CUBIC PHASE                                                  \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: GLYCOPHORIN-A;                                             \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 FRAGMENT: UNP RESIDEUS 89-117;                                       \\nCOMPND   5 SYNONYM: MN SIALOGLYCOPROTEIN,PAS-2,SIALOGLYCOPROTEIN ALPHA;         \\nCOMPND   6 ENGINEERED: YES;                                                     \\nCOMPND   7 MUTATION: YES                                                        \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606;                                                \\nSOURCE   5 GENE: GYPA, GPA;                                                     \\nSOURCE   6 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \\nSOURCE   7 EXPRESSION_SYSTEM_TAXID: 562                                         \\nKEYWDS    RECEPTOR, LIPIDIC CUBIC PHASE, PEPTIDES, TRANSMEMBRANE, MEMBRANE      \\nKEYWDS   2 PROTEIN                                                              \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    M.J.CALL,M.E.CALL,R.TRENKER                                           \\nREVDAT   5   01-JAN-20 5EH6    1       REMARK                                   \\nREVDAT   4   17-JAN-18 5EH6    1       REMARK                                   \\nREVDAT   3   20-SEP-17 5EH6    1       REMARK                                   \\nREVDAT   2   06-JAN-16 5EH6    1       JRNL                                     \\nREVDAT   1   23-DEC-15 5EH6    0                                                \\nJRNL        AUTH   R.TRENKER,M.E.CALL,M.J.CALL                                  \\nJRNL        TITL   CRYSTAL STRUCTURE OF THE GLYCOPHORIN A TRANSMEMBRANE DIMER   \\nJRNL        TITL 2 IN LIPIDIC CUBIC PHASE.                                      \\nJRNL        REF    J.AM.CHEM.SOC.                V. 137 15676 2015              \\nJRNL        REFN                   ESSN 1520-5126                               \\nJRNL        PMID   26642914                                                     \\nJRNL        DOI    10.1021/JACS.5B11354                                         \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.92 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX 1.9_1692                                      \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.92                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 26.60                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 1.340                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 97.5                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2139                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.225                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.224                           \\nREMARK   3   FREE R VALUE                     : 0.238                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 10.000                          \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 214                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 26.5982 -  2.4156    0.96      978   109  0.2219 0.2306        \\nREMARK   3     2  2.4156 -  1.9175    0.99      947   105  0.2312 0.2642        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.11                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.90                                          \\nREMARK   3   K_SOL              : NULL                                          \\nREMARK   3   B_SOL              : NULL                                          \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.150            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 29.650           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.007            199                                  \\nREMARK   3   ANGLE     :  0.890            271                                  \\nREMARK   3   CHIRALITY :  0.035             39                                  \\nREMARK   3   PLANARITY :  0.003             30                                  \\nREMARK   3   DIHEDRAL  :  8.879             65                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 5EH6 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 28-OCT-15.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000214088.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 13-AUG-15                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : 7.5                                \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : AUSTRALIAN SYNCHROTRON             \\nREMARK 200  BEAMLINE                       : MX2                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.9537                             \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315R                  \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : HKL-2000                           \\nREMARK 200  DATA SCALING SOFTWARE          : HKL-2000                           \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 2141                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 1.918                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 26.600                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 97.6                               \\nREMARK 200  DATA REDUNDANCY                : 9.600                              \\nREMARK 200  R MERGE                    (I) : 0.10850                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 9.9100                             \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 1.92                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 1.99                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : 93.6                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : 8.60                               \\nREMARK 200  R MERGE FOR SHELL          (I) : 0.94000                            \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 2.370                              \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \\nREMARK 200 SOFTWARE USED: PHENIX 1.9_1692                                       \\nREMARK 200 STARTING MODEL: 4WOL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: HEXAGONAL PLATES                                             \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 37.03                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.95                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: 0.1 M HEPES, 20% PEG8000, 10 MM TRIS,    \\nREMARK 280  40 MM NACL, PH 7.5, LIPIDIC CUBIC PHASE, TEMPERATURE 293K           \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: H 3 2                            \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z                                                \\nREMARK 290       3555   -X+Y,-X,Z                                               \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z                                               \\nREMARK 290       6555   -X,-X+Y,-Z                                              \\nREMARK 290       7555   X+2/3,Y+1/3,Z+1/3                                       \\nREMARK 290       8555   -Y+2/3,X-Y+1/3,Z+1/3                                    \\nREMARK 290       9555   -X+Y+2/3,-X+1/3,Z+1/3                                   \\nREMARK 290      10555   Y+2/3,X+1/3,-Z+1/3                                      \\nREMARK 290      11555   X-Y+2/3,-Y+1/3,-Z+1/3                                   \\nREMARK 290      12555   -X+2/3,-X+Y+1/3,-Z+1/3                                  \\nREMARK 290      13555   X+1/3,Y+2/3,Z+2/3                                       \\nREMARK 290      14555   -Y+1/3,X-Y+2/3,Z+2/3                                    \\nREMARK 290      15555   -X+Y+1/3,-X+2/3,Z+2/3                                   \\nREMARK 290      16555   Y+1/3,X+2/3,-Z+2/3                                      \\nREMARK 290      17555   X-Y+1/3,-Y+2/3,-Z+2/3                                   \\nREMARK 290      18555   -X+1/3,-X+Y+2/3,-Z+2/3                                  \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   7  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3   7  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   8 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   8  0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   8  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1   9 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2   9 -0.866025 -0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       45.42900            \\nREMARK 290   SMTRY1  10 -0.500000  0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  10  0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  11  1.000000  0.000000  0.000000       15.65600            \\nREMARK 290   SMTRY2  11  0.000000 -1.000000  0.000000        9.03900            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  12 -0.500000 -0.866025  0.000000       15.65600            \\nREMARK 290   SMTRY2  12 -0.866025  0.500000  0.000000        9.03900            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       45.42900            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  14 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  14  0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  15 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  15 -0.866025 -0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  15  0.000000  0.000000  1.000000       90.85800            \\nREMARK 290   SMTRY1  16 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  16  0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  17  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  17  0.000000 -1.000000  0.000000       18.07799            \\nREMARK 290   SMTRY3  17  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290   SMTRY1  18 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  18 -0.866025  0.500000  0.000000       18.07799            \\nREMARK 290   SMTRY3  18  0.000000  0.000000 -1.000000       90.85800            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     GLU A    70                                                      \\nREMARK 465     SCH A    99                                                      \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ARG A  96    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     ARG A  97    CG   CD   NE   CZ   NH1  NH2                        \\nREMARK 470     LEU A  98    CG   CD1  CD2                                       \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ARG A  97       31.10    -98.71                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 5EH4   RELATED DB: PDB                                   \\nREMARK 900 5EH4 IS THE SAME PEPTIDE CRYSTALLISED AS DIMER                       \\nDBREF  5EH6 A   70    98  UNP    P02724   GLPA_HUMAN      89    117             \\nSEQADV 5EH6 ILE A   81  UNP  P02724    MET   100 ENGINEERED MUTATION            \\nSEQRES   1 A   30  GLU PRO GLU ILE THR LEU ILE ILE PHE GLY VAL ILE ALA          \\nSEQRES   2 A   30  GLY VAL ILE GLY THR ILE LEU LEU ILE SER TYR GLY ILE          \\nSEQRES   3 A   30  ARG ARG LEU SCH                                              \\nHELIX    1 AA1 GLU A   72  ARG A   97  1                                  26    \\nCRYST1   31.312   31.312  136.287  90.00  90.00 120.00 H 3 2        18          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.031937  0.018439  0.000000        0.00000                         \\nSCALE2      0.000000  0.036877  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.007337        0.00000                         \\nATOM      1  N   PRO A  71      26.002   6.928 -18.824  1.00 72.28           N  \\nANISOU    1  N   PRO A  71    10939   9924   6602  -1650  -1647    429       N  \\nATOM      2  CA  PRO A  71      24.968   6.210 -18.070  1.00 75.23           C  \\nANISOU    2  CA  PRO A  71    10989  10418   7177  -1839  -1839    391       C  \\nATOM      3  C   PRO A  71      25.401   5.954 -16.631  1.00 59.78           C  \\nANISOU    3  C   PRO A  71     8903   8293   5518  -1825  -1641    348       C  \\nATOM      4  O   PRO A  71      24.631   6.137 -15.685  1.00 55.18           O  \\nANISOU    4  O   PRO A  71     7919   7869   5177  -1810  -1746    462       O  \\nATOM      5  CB  PRO A  71      24.820   4.899 -18.846  1.00 83.70           C  \\nANISOU    5  CB  PRO A  71    12311  11422   8068  -2143  -1887    151       C  \\nATOM      6  CG  PRO A  71      26.160   4.684 -19.474  1.00 82.77           C  \\nANISOU    6  CG  PRO A  71    12644  11040   7765  -2081  -1586    -13       C  \\nATOM      7  CD  PRO A  71      26.685   6.058 -19.798  1.00 79.89           C  \\nANISOU    7  CD  PRO A  71    12306  10735   7312  -1802  -1502    190       C  \\nATOM      8  N   GLU A  72      26.651   5.547 -16.470  1.00 43.89           N  \\nANISOU    8  N   GLU A  72     7210   5973   3492  -1803  -1334    180       N  \\nATOM      9  CA  GLU A  72      27.199   5.342 -15.151  1.00 41.57           C  \\nANISOU    9  CA  GLU A  72     6830   5492   3473  -1747  -1128    132       C  \\nATOM     10  C   GLU A  72      27.567   6.686 -14.541  1.00 28.78           C  \\nANISOU   10  C   GLU A  72     4985   3872   2077  -1389   -970    340       C  \\nATOM     11  O   GLU A  72      28.126   6.740 -13.453  1.00 27.05           O  \\nANISOU   11  O   GLU A  72     4644   3486   2148  -1252   -742    321       O  \\nATOM     12  CB  GLU A  72      28.412   4.415 -15.215  1.00 53.65           C  \\nANISOU   12  CB  GLU A  72     8727   6675   4983  -1767   -827   -124       C  \\nATOM     13  CG  GLU A  72      29.411   4.761 -16.308  1.00 55.89           C  \\nANISOU   13  CG  GLU A  72     9315   6881   5040  -1626   -621   -177       C  \\nATOM     14  CD  GLU A  72      30.587   3.800 -16.349  1.00 67.90           C  \\nANISOU   14  CD  GLU A  72    11070   8100   6628  -1563   -300   -410       C  \\nATOM     15  OE1 GLU A  72      31.018   3.327 -15.272  1.00 68.55           O  \\nANISOU   15  OE1 GLU A  72    11065   7996   6985  -1491   -160   -465       O  \\nATOM     16  OE2 GLU A  72      31.075   3.511 -17.462  1.00 79.31           O  \\nANISOU   16  OE2 GLU A  72    12751   9514   7868  -1553   -194   -519       O  \\nATOM     17  N   ILE A  73      27.238   7.766 -15.250  1.00 30.36           N  \\nANISOU   17  N   ILE A  73     5165   4244   2126  -1248  -1115    542       N  \\nATOM     18  CA  ILE A  73      27.532   9.124 -14.797  1.00 28.77           C  \\nANISOU   18  CA  ILE A  73     4824   3994   2114   -920   -993    745       C  \\nATOM     19  C   ILE A  73      26.893   9.424 -13.447  1.00 26.85           C  \\nANISOU   19  C   ILE A  73     4154   3812   2237   -780   -998    815       C  \\nATOM     20  O   ILE A  73      27.496  10.058 -12.581  1.00 24.23           O  \\nANISOU   20  O   ILE A  73     3756   3310   2141   -574   -773    839       O  \\nATOM     21  CB  ILE A  73      27.053  10.160 -15.815  1.00 34.40           C  \\nANISOU   21  CB  ILE A  73     5607   4872   2593   -799  -1219    987       C  \\nATOM     22  CG1 ILE A  73      27.291  11.568 -15.279  1.00 36.30           C  \\nANISOU   22  CG1 ILE A  73     5741   4990   3060   -462  -1102   1194       C  \\nATOM     23  CG2 ILE A  73      25.585   9.942 -16.166  1.00 46.47           C  \\nANISOU   23  CG2 ILE A  73     6867   6736   4054   -887  -1615   1059       C  \\nATOM     24  CD1 ILE A  73      28.740  11.858 -15.015  1.00 45.01           C  \\nANISOU   24  CD1 ILE A  73     7064   5789   4247   -413   -720   1120       C  \\nATOM     25  N   THR A  74      25.665   8.948 -13.292  1.00 40.83           N  \\nANISOU   25  N   THR A  74     5637   5847   4029   -921  -1252    836       N  \\nATOM     26  CA  THR A  74      24.932   9.005 -12.039  1.00 54.69           C  \\nANISOU   26  CA  THR A  74     6966   7727   6086   -849  -1236    876       C  \\nATOM     27  C   THR A  74      25.719   8.304 -10.935  1.00 33.06           C  \\nANISOU   27  C   THR A  74     4287   4741   3534   -920   -947    714       C  \\nATOM     28  O   THR A  74      25.960   8.870  -9.873  1.00 25.63           O  \\nANISOU   28  O   THR A  74     3197   3724   2819   -711   -764    744       O  \\nATOM     29  CB  THR A  74      23.528   8.346 -12.181  1.00 69.92           C  \\nANISOU   29  CB  THR A  74     8570  10023   7975  -1102  -1547    902       C  \\nATOM     30  OG1 THR A  74      23.670   6.968 -12.571  1.00 59.92           O  \\nANISOU   30  OG1 THR A  74     7546   8677   6544  -1516  -1596    711       O  \\nATOM     31  CG2 THR A  74      22.688   9.086 -13.223  1.00 60.41           C  \\nANISOU   31  CG2 THR A  74     7246   9113   6594   -991  -1891   1093       C  \\nATOM     32  N   LEU A  75      26.123   7.069 -11.215  1.00 30.54           N  \\nANISOU   32  N   LEU A  75     4220   4283   3099  -1205   -922    539       N  \\nATOM     33  CA  LEU A  75      26.853   6.255 -10.261  1.00 32.84           C  \\nANISOU   33  CA  LEU A  75     4607   4325   3546  -1271   -697    405       C  \\nATOM     34  C   LEU A  75      28.206   6.852  -9.911  1.00 25.79           C  \\nANISOU   34  C   LEU A  75     3865   3180   2754  -1010   -424    380       C  \\nATOM     35  O   LEU A  75      28.625   6.828  -8.754  1.00 24.52           O  \\nANISOU   35  O   LEU A  75     3612   2910   2796   -922   -264    366       O  \\nATOM     36  CB  LEU A  75      27.029   4.843 -10.809  1.00 28.01           C  \\nANISOU   36  CB  LEU A  75     4304   3560   2777  -1589   -744    220       C  \\nATOM     37  CG  LEU A  75      25.766   3.999 -10.722  1.00 27.87           C  \\nANISOU   37  CG  LEU A  75     4119   3735   2737  -1948   -981    222       C  \\nATOM     38  CD1 LEU A  75      26.030   2.653 -11.357  1.00 37.84           C  \\nANISOU   38  CD1 LEU A  75     5752   4769   3858  -2133   -958     39       C  \\nATOM     39  CD2 LEU A  75      25.374   3.858  -9.264  1.00 31.17           C  \\nANISOU   39  CD2 LEU A  75     4232   4196   3414  -1954   -873    303       C  \\nATOM     40  N   ILE A  76      28.895   7.392 -10.905  1.00 19.81           N  \\nANISOU   40  N   ILE A  76     3334   2354   1837   -914   -376    382       N  \\nATOM     41  CA  ILE A  76      30.199   7.987 -10.648  1.00 25.42           C  \\nANISOU   41  CA  ILE A  76     4150   2861   2646   -719   -117    364       C  \\nATOM     42  C   ILE A  76      30.078   9.202  -9.725  1.00 21.46           C  \\nANISOU   42  C   ILE A  76     3413   2382   2358   -495    -72    500       C  \\nATOM     43  O   ILE A  76      30.821   9.326  -8.751  1.00 19.33           O  \\nANISOU   43  O   ILE A  76     3097   1975   2274   -406     97    454       O  \\nATOM     44  CB  ILE A  76      30.897   8.377 -11.962  1.00 18.98           C  \\nANISOU   44  CB  ILE A  76     3616   2007   1589   -703    -49    365       C  \\nATOM     45  CG1 ILE A  76      31.346   7.115 -12.698  1.00 26.36           C  \\nANISOU   45  CG1 ILE A  76     4836   2851   2328   -878      5    155       C  \\nATOM     46  CG2 ILE A  76      32.095   9.264 -11.691  1.00 17.37           C  \\nANISOU   46  CG2 ILE A  76     3437   1654   1510   -533    201    401       C  \\nATOM     47  CD1 ILE A  76      31.830   7.370 -14.096  1.00 33.92           C  \\nANISOU   47  CD1 ILE A  76     6090   3835   2962   -903     69    138       C  \\nATOM     48  N   ILE A  77      29.127  10.081 -10.014  1.00 17.69           N  \\nANISOU   48  N   ILE A  77     2797   2074   1850   -390   -239    659       N  \\nATOM     49  CA  ILE A  77      28.965  11.285  -9.221  1.00 17.04           C  \\nANISOU   49  CA  ILE A  77     2546   1970   1957   -140   -190    764       C  \\nATOM     50  C   ILE A  77      28.530  10.901  -7.817  1.00 15.87           C  \\nANISOU   50  C   ILE A  77     2141   1884   2003   -135   -137    701       C  \\nATOM     51  O   ILE A  77      29.012  11.457  -6.840  1.00 14.34           O  \\nANISOU   51  O   ILE A  77     1914   1569   1964      3     13    673       O  \\nATOM     52  CB  ILE A  77      27.958  12.256  -9.867  1.00 32.82           C  \\nANISOU   52  CB  ILE A  77     4453   4127   3892     28   -399    957       C  \\nATOM     53  CG1 ILE A  77      28.527  12.779 -11.191  1.00 28.88           C  \\nANISOU   53  CG1 ILE A  77     4274   3532   3168     26   -424   1056       C  \\nATOM     54  CG2 ILE A  77      27.626  13.433  -8.929  1.00 28.78           C  \\nANISOU   54  CG2 ILE A  77     3765   3567   3602    331   -343   1032       C  \\nATOM     55  CD1 ILE A  77      27.508  13.469 -12.067  1.00 24.61           C  \\nANISOU   55  CD1 ILE A  77     3698   3167   2486    155   -703   1271       C  \\nATOM     56  N   PHE A  78      27.644   9.923  -7.722  1.00 20.44           N  \\nANISOU   56  N   PHE A  78     2565   2655   2547   -323   -258    678       N  \\nATOM     57  CA  PHE A  78      27.223   9.413  -6.424  1.00 29.06           C  \\nANISOU   57  CA  PHE A  78     3441   3824   3777   -381   -185    638       C  \\nATOM     58  C   PHE A  78      28.424   8.921  -5.639  1.00 18.03           C  \\nANISOU   58  C   PHE A  78     2225   2175   2450   -414      5    527       C  \\nATOM     59  O   PHE A  78      28.580   9.231  -4.459  1.00 15.70           O  \\nANISOU   59  O   PHE A  78     1836   1857   2272   -311    126    514       O  \\nATOM     60  CB  PHE A  78      26.204   8.271  -6.578  1.00 27.90           C  \\nANISOU   60  CB  PHE A  78     3148   3890   3561   -681   -340    636       C  \\nATOM     61  CG  PHE A  78      26.034   7.437  -5.332  1.00 22.44           C  \\nANISOU   61  CG  PHE A  78     2353   3209   2963   -840   -228    598       C  \\nATOM     62  CD1 PHE A  78      25.276   7.901  -4.270  1.00 18.61           C  \\nANISOU   62  CD1 PHE A  78     1545   2938   2588   -734   -147    659       C  \\nATOM     63  CD2 PHE A  78      26.641   6.192  -5.225  1.00 24.51           C  \\nANISOU   63  CD2 PHE A  78     2869   3255   3189  -1079   -191    508       C  \\nATOM     64  CE1 PHE A  78      25.123   7.138  -3.128  1.00 22.90           C  \\nANISOU   64  CE1 PHE A  78     2025   3507   3168   -907    -27    650       C  \\nATOM     65  CE2 PHE A  78      26.495   5.426  -4.086  1.00 28.95           C  \\nANISOU   65  CE2 PHE A  78     3396   3794   3809  -1191    -99    498       C  \\nATOM     66  CZ  PHE A  78      25.735   5.900  -3.036  1.00 29.26           C  \\nANISOU   66  CZ  PHE A  78     3134   4062   3920  -1117    -16    569       C  \\nATOM     67  N   GLY A  79      29.267   8.143  -6.306  1.00 16.77           N  \\nANISOU   67  N   GLY A  79     2326   1842   2204   -542     25    441       N  \\nATOM     68  CA  GLY A  79      30.397   7.527  -5.649  1.00 17.63           C  \\nANISOU   68  CA  GLY A  79     2578   1732   2390   -550    170    345       C  \\nATOM     69  C   GLY A  79      31.372   8.545  -5.111  1.00 10.12           C  \\nANISOU   69  C   GLY A  79     1596    716   1533   -325    273    327       C  \\nATOM     70  O   GLY A  79      31.913   8.383  -4.016  1.00 15.97           O  \\nANISOU   70  O   GLY A  79     2272   1484   2310   -256    275    257       O  \\nATOM     71  N   VAL A  80      31.591   9.601  -5.877  1.00 10.40           N  \\nANISOU   71  N   VAL A  80     1690    719   1543   -227    301    379       N  \\nATOM     72  CA  VAL A  80      32.536  10.628  -5.468  1.00 11.18           C  \\nANISOU   72  CA  VAL A  80     1745    819   1684    -83    338    312       C  \\nATOM     73  C   VAL A  80      32.013  11.384  -4.245  1.00 13.80           C  \\nANISOU   73  C   VAL A  80     1956   1172   2116     24    367    331       C  \\nATOM     74  O   VAL A  80      32.725  11.539  -3.271  1.00 10.65           O  \\nANISOU   74  O   VAL A  80     1511    805   1730     46    389    236       O  \\nATOM     75  CB  VAL A  80      32.830  11.609  -6.604  1.00 13.99           C  \\nANISOU   75  CB  VAL A  80     2206   1127   1982    -46    353    378       C  \\nATOM     76  CG1 VAL A  80      33.718  12.724  -6.111  1.00 17.30           C  \\nANISOU   76  CG1 VAL A  80     2580   1539   2454     21    389    320       C  \\nATOM     77  CG2 VAL A  80      33.500  10.886  -7.770  1.00 13.03           C  \\nANISOU   77  CG2 VAL A  80     2225    983   1741   -137    407    339       C  \\nATOM     78  N   ILE A  81      30.763  11.834  -4.294  1.00 13.20           N  \\nANISOU   78  N   ILE A  81     1795   1113   2106    113    358    485       N  \\nATOM     79  CA  ILE A  81      30.179  12.589  -3.190  1.00 11.57           C  \\nANISOU   79  CA  ILE A  81     1445    961   1989    277    400    478       C  \\nATOM     80  C   ILE A  81      30.108  11.708  -1.940  1.00 13.01           C  \\nANISOU   80  C   ILE A  81     1533   1227   2182    176    459    408       C  \\nATOM     81  O   ILE A  81      30.520  12.124  -0.855  1.00 16.50           O  \\nANISOU   81  O   ILE A  81     1995   1608   2667    245    548    337       O  \\nATOM     82  CB  ILE A  81      28.752  13.144  -3.549  1.00 17.96           C  \\nANISOU   82  CB  ILE A  81     2062   1970   2793    435    299    580       C  \\nATOM     83  CG1 ILE A  81      28.846  14.286  -4.572  1.00 19.73           C  \\nANISOU   83  CG1 ILE A  81     2429   2060   3007    602    234    685       C  \\nATOM     84  CG2 ILE A  81      28.016  13.617  -2.294  1.00 17.99           C  \\nANISOU   84  CG2 ILE A  81     1869   2090   2875    608    390    536       C  \\nATOM     85  CD1 ILE A  81      27.489  14.757  -5.143  1.00 20.93           C  \\nANISOU   85  CD1 ILE A  81     2389   2417   3146    790     73    823       C  \\nATOM     86  N   ALA A  82      29.615  10.479  -2.093  1.00 14.62           N  \\nANISOU   86  N   ALA A  82     1675   1555   2325    -16    399    431       N  \\nATOM     87  CA  ALA A  82      29.518   9.568  -0.954  1.00 15.60           C  \\nANISOU   87  CA  ALA A  82     1756   1734   2436   -146    451    410       C  \\nATOM     88  C   ALA A  82      30.904   9.253  -0.397  1.00 17.38           C  \\nANISOU   88  C   ALA A  82     2169   1796   2638   -135    450    304       C  \\nATOM     89  O   ALA A  82      31.093   9.224   0.822  1.00 14.20           O  \\nANISOU   89  O   ALA A  82     1762   1408   2225   -118    497    282       O  \\nATOM     90  CB  ALA A  82      28.785   8.280  -1.339  1.00 12.86           C  \\nANISOU   90  CB  ALA A  82     1365   1496   2025   -401    364    459       C  \\nATOM     91  N   GLY A  83      31.866   9.017  -1.287  1.00  9.24           N  \\nANISOU   91  N   GLY A  83     1253    707   1549   -128    365    233       N  \\nATOM     92  CA  GLY A  83      33.240   8.762  -0.875  1.00 13.08           C  \\nANISOU   92  CA  GLY A  83     1783   1188   1997    -85    327    150       C  \\nATOM     93  C   GLY A  83      33.832   9.924  -0.086  1.00 12.02           C  \\nANISOU   93  C   GLY A  83     1606   1064   1896     -5    381    116       C  \\nATOM     94  O   GLY A  83      34.436   9.734   0.966  1.00 14.41           O  \\nANISOU   94  O   GLY A  83     1913   1363   2200     -6    377     98       O  \\nATOM     95  N   VAL A  84      33.648  11.136  -0.584  1.00 11.13           N  \\nANISOU   95  N   VAL A  84     1484    936   1808     43    418    124       N  \\nATOM     96  CA  VAL A  84      34.146  12.312   0.104  1.00  9.86           C  \\nANISOU   96  CA  VAL A  84     1332    751   1663     74    438     92       C  \\nATOM     97  C   VAL A  84      33.478  12.454   1.478  1.00 23.28           C  \\nANISOU   97  C   VAL A  84     3029   2426   3392    111    507     85       C  \\nATOM     98  O   VAL A  84      34.162  12.648   2.494  1.00 13.72           O  \\nANISOU   98  O   VAL A  84     1857   1190   2165     98    492     31       O  \\nATOM     99  CB  VAL A  84      33.919  13.579  -0.731  1.00 11.25           C  \\nANISOU   99  CB  VAL A  84     1553    869   1854    123    442    115       C  \\nATOM    100  CG1 VAL A  84      34.115  14.841   0.110  1.00 10.57           C  \\nANISOU  100  CG1 VAL A  84     1521    705   1792    167    448     70       C  \\nATOM    101  CG2 VAL A  84      34.850  13.573  -1.937  1.00 17.63           C  \\nANISOU  101  CG2 VAL A  84     2388   1661   2650     80    458    124       C  \\nATOM    102  N   ILE A  85      32.154  12.329   1.520  1.00 14.94           N  \\nANISOU  102  N   ILE A  85     1923   1361   2394    179    603    145       N  \\nATOM    103  CA  ILE A  85      31.434  12.529   2.766  1.00  9.86           C  \\nANISOU  103  CA  ILE A  85     1243    764   1739    273    714    109       C  \\nATOM    104  C   ILE A  85      31.759  11.396   3.733  1.00 13.41           C  \\nANISOU  104  C   ILE A  85     1720   1264   2112    139    730    122       C  \\nATOM    105  O   ILE A  85      32.010  11.623   4.920  1.00 12.84           O  \\nANISOU  105  O   ILE A  85     1713   1218   1946    152    765     51       O  \\nATOM    106  CB  ILE A  85      29.908  12.608   2.539  1.00 11.50           C  \\nANISOU  106  CB  ILE A  85     1248   1175   1948    376    772    165       C  \\nATOM    107  CG1 ILE A  85      29.540  13.861   1.742  1.00 12.43           C  \\nANISOU  107  CG1 ILE A  85     1381   1240   2101    537    698    162       C  \\nATOM    108  CG2 ILE A  85      29.158  12.574   3.871  1.00 13.13           C  \\nANISOU  108  CG2 ILE A  85     1364   1588   2037    399    874    114       C  \\nATOM    109  CD1 ILE A  85      28.073  13.892   1.307  1.00 21.23           C  \\nANISOU  109  CD1 ILE A  85     2243   2580   3242    666    702    240       C  \\nATOM    110  N   GLY A  86      31.759  10.176   3.214  1.00  9.78           N  \\nANISOU  110  N   GLY A  86     1252    811   1654     -9    668    216       N  \\nATOM    111  CA  GLY A  86      32.067   9.016   4.018  1.00 10.45           C  \\nANISOU  111  CA  GLY A  86     1408    923   1640   -133    602    253       C  \\nATOM    112  C   GLY A  86      33.429   9.138   4.676  1.00 13.84           C  \\nANISOU  112  C   GLY A  86     1950   1238   2069    -97    542    201       C  \\nATOM    113  O   GLY A  86      33.573   8.900   5.878  1.00 11.38           O  \\nANISOU  113  O   GLY A  86     1720    951   1652   -142    558    232       O  \\nATOM    114  N   THR A  87      34.433   9.509   3.889  1.00 12.99           N  \\nANISOU  114  N   THR A  87     1814   1108   2012    -35    437    126       N  \\nATOM    115  CA  THR A  87      35.804   9.618   4.400  1.00 13.69           C  \\nANISOU  115  CA  THR A  87     1935   1150   2118    -27    347     91       C  \\nATOM    116  C   THR A  87      35.930  10.647   5.525  1.00 12.38           C  \\nANISOU  116  C   THR A  87     1850    902   1951    -16    409     19       C  \\nATOM    117  O   THR A  87      36.549  10.379   6.559  1.00 12.81           O  \\nANISOU  117  O   THR A  87     1982    937   1949    -48    338     17       O  \\nATOM    118  CB  THR A  87      36.779   9.989   3.278  1.00 17.27           C  \\nANISOU  118  CB  THR A  87     2321   1640   2602     -6    290     50       C  \\nATOM    119  OG1 THR A  87      36.788   8.937   2.313  1.00 18.68           O  \\nANISOU  119  OG1 THR A  87     2482   1837   2777     -8    289     77       O  \\nATOM    120  CG2 THR A  87      38.187  10.186   3.824  1.00 16.57           C  \\nANISOU  120  CG2 THR A  87     2227   1480   2587    -10    237     21       C  \\nATOM    121  N   ILE A  88      35.341  11.822   5.302  1.00 17.44           N  \\nANISOU  121  N   ILE A  88     2481   1553   2593     40    488    -55       N  \\nATOM    122  CA  ILE A  88      35.267  12.888   6.299  1.00 19.16           C  \\nANISOU  122  CA  ILE A  88     2814   1716   2751     84    545   -196       C  \\nATOM    123  C   ILE A  88      34.589  12.416   7.594  1.00 19.87           C  \\nANISOU  123  C   ILE A  88     2959   1969   2623     75    598   -200       C  \\nATOM    124  O   ILE A  88      35.102  12.650   8.689  1.00 15.46           O  \\nANISOU  124  O   ILE A  88     2516   1441   1917     35    536   -287       O  \\nATOM    125  CB  ILE A  88      34.504  14.104   5.735  1.00 13.93           C  \\nANISOU  125  CB  ILE A  88     2113   1077   2102    181    576   -223       C  \\nATOM    126  CG1 ILE A  88      35.327  14.779   4.646  1.00 13.96           C  \\nANISOU  126  CG1 ILE A  88     2082   1046   2176    134    455   -189       C  \\nATOM    127  CG2 ILE A  88      34.123  15.087   6.842  1.00 16.44           C  \\nANISOU  127  CG2 ILE A  88     2550   1378   2318    268    642   -381       C  \\nATOM    128  CD1 ILE A  88      34.637  15.963   4.066  1.00 18.02           C  \\nANISOU  128  CD1 ILE A  88     2625   1515   2706    226    466   -198       C  \\nATOM    129  N   LEU A  89      33.448  11.740   7.460  1.00 11.97           N  \\nANISOU  129  N   LEU A  89     1866   1110   1573     77    699    -94       N  \\nATOM    130  CA  LEU A  89      32.719  11.270   8.622  1.00 15.64           C  \\nANISOU  130  CA  LEU A  89     2357   1779   1807     30    786    -64       C  \\nATOM    131  C   LEU A  89      33.496  10.176   9.350  1.00 17.03           C  \\nANISOU  131  C   LEU A  89     2650   1960   1862   -109    644     50       C  \\nATOM    132  O   LEU A  89      33.421  10.062  10.574  1.00 22.83           O  \\nANISOU  132  O   LEU A  89     3503   2822   2348   -157    660     47       O  \\nATOM    133  CB  LEU A  89      31.337  10.757   8.223  1.00 15.16           C  \\nANISOU  133  CB  LEU A  89     2118   1893   1750      8    923     44       C  \\nATOM    134  CG  LEU A  89      30.305  11.772   7.741  1.00 18.76           C  \\nANISOU  134  CG  LEU A  89     2413   2428   2288    194   1065    -39       C  \\nATOM    135  CD1 LEU A  89      28.953  11.097   7.500  1.00 18.26           C  \\nANISOU  135  CD1 LEU A  89     2108   2619   2212    118   1127     82       C  \\nATOM    136  CD2 LEU A  89      30.145  12.864   8.755  1.00 17.08           C  \\nANISOU  136  CD2 LEU A  89     2287   2258   1946    347   1167   -223       C  \\nATOM    137  N   LEU A  90      34.249   9.376   8.604  1.00 17.01           N  \\nANISOU  137  N   LEU A  90     2634   1814   2016   -149    506    153       N  \\nATOM    138  CA  LEU A  90      35.074   8.327   9.215  1.00 21.28           C  \\nANISOU  138  CA  LEU A  90     3288   2315   2484   -214    341    279       C  \\nATOM    139  C   LEU A  90      36.266   8.919   9.981  1.00 19.34           C  \\nANISOU  139  C   LEU A  90     3105   2064   2180   -180    177    183       C  \\nATOM    140  O   LEU A  90      36.646   8.435  11.049  1.00 18.95           O  \\nANISOU  140  O   LEU A  90     3181   2085   1935   -222     50    260       O  \\nATOM    141  CB  LEU A  90      35.559   7.351   8.143  1.00 14.32           C  \\nANISOU  141  CB  LEU A  90     2375   1259   1808   -208    262    380       C  \\nATOM    142  CG  LEU A  90      35.982   5.938   8.518  1.00 26.43           C  \\nANISOU  142  CG  LEU A  90     4044   2695   3303   -244    131    560       C  \\nATOM    143  CD1 LEU A  90      34.949   5.281   9.413  1.00 31.06           C  \\nANISOU  143  CD1 LEU A  90     4753   3391   3656   -397    202    703       C  \\nATOM    144  CD2 LEU A  90      36.156   5.148   7.244  1.00 21.46           C  \\nANISOU  144  CD2 LEU A  90     3291   1963   2900   -193    106    550       C  \\nATOM    145  N   ILE A  91      36.861   9.967   9.434  1.00 16.80           N  \\nANISOU  145  N   ILE A  91     2705   1662   2015   -132    162     29       N  \\nATOM    146  CA  ILE A  91      37.923  10.669  10.141  1.00 16.69           C  \\nANISOU  146  CA  ILE A  91     2731   1659   1952   -161      0    -90       C  \\nATOM    147  C   ILE A  91      37.387  11.282  11.438  1.00 21.02           C  \\nANISOU  147  C   ILE A  91     3458   2337   2192   -197     42   -207       C  \\nATOM    148  O   ILE A  91      37.971  11.111  12.510  1.00 23.14           O  \\nANISOU  148  O   ILE A  91     3838   2702   2251   -260   -131   -205       O  \\nATOM    149  CB  ILE A  91      38.548  11.761   9.261  1.00 16.48           C  \\nANISOU  149  CB  ILE A  91     2608   1501   2152   -163     12   -230       C  \\nATOM    150  CG1 ILE A  91      39.209  11.120   8.036  1.00 14.90           C  \\nANISOU  150  CG1 ILE A  91     2238   1215   2208   -130     -8   -127       C  \\nATOM    151  CG2 ILE A  91      39.553  12.591  10.062  1.00 20.71           C  \\nANISOU  151  CG2 ILE A  91     3190   2054   2625   -264   -162   -378       C  \\nATOM    152  CD1 ILE A  91      39.872  12.107   7.107  1.00 15.47           C  \\nANISOU  152  CD1 ILE A  91     2214   1180   2484   -172     32   -222       C  \\nATOM    153  N   SER A  92      36.265  11.984  11.320  1.00 21.14           N  \\nANISOU  153  N   SER A  92     3497   2366   2168   -137    273   -310       N  \\nATOM    154  CA  SER A  92      35.596  12.619  12.448  1.00 19.47           C  \\nANISOU  154  CA  SER A  92     3452   2280   1664   -123    395   -459       C  \\nATOM    155  C   SER A  92      35.332  11.605  13.560  1.00 21.28           C  \\nANISOU  155  C   SER A  92     3792   2717   1575   -208    375   -314       C  \\nATOM    156  O   SER A  92      35.657  11.825  14.733  1.00 23.88           O  \\nANISOU  156  O   SER A  92     4318   3152   1603   -269    293   -398       O  \\nATOM    157  CB  SER A  92      34.292  13.264  11.965  1.00 29.49           C  \\nANISOU  157  CB  SER A  92     4650   3558   2997     23    673   -538       C  \\nATOM    158  OG  SER A  92      33.609  13.945  12.998  1.00 39.92           O  \\nANISOU  158  OG  SER A  92     6117   5000   4052     92    844   -718       O  \\nATOM    159  N   TYR A  93      34.755  10.473  13.174  1.00 26.54           N  \\nANISOU  159  N   TYR A  93     4366   3429   2290   -241    439    -86       N  \\nATOM    160  CA  TYR A  93      34.474   9.408  14.121  1.00 27.68           C  \\nANISOU  160  CA  TYR A  93     4642   3725   2149   -357    429    110       C  \\nATOM    161  C   TYR A  93      35.794   8.928  14.720  1.00 27.37           C  \\nANISOU  161  C   TYR A  93     4738   3639   2021   -399    102    200       C  \\nATOM    162  O   TYR A  93      35.933   8.818  15.933  1.00 28.88           O  \\nANISOU  162  O   TYR A  93     5135   3977   1861   -470     24    228       O  \\nATOM    163  CB  TYR A  93      33.712   8.267  13.436  1.00 21.26           C  \\nANISOU  163  CB  TYR A  93     3721   2895   1460   -429    526    339       C  \\nATOM    164  CG  TYR A  93      33.489   7.046  14.302  1.00 33.29           C  \\nANISOU  164  CG  TYR A  93     5422   4503   2724   -592    501    595       C  \\nATOM    165  CD1 TYR A  93      32.502   7.034  15.280  1.00 28.66           C  \\nANISOU  165  CD1 TYR A  93     4837   4158   1893   -651    694    604       C  \\nATOM    166  CD2 TYR A  93      34.254   5.906  14.132  1.00 23.51           C  \\nANISOU  166  CD2 TYR A  93     4283   3078   1573   -625    269    812       C  \\nATOM    167  CE1 TYR A  93      32.295   5.937  16.063  1.00 29.00           C  \\nANISOU  167  CE1 TYR A  93     4966   4240   1811   -750    652    817       C  \\nATOM    168  CE2 TYR A  93      34.049   4.806  14.910  1.00 26.12           C  \\nANISOU  168  CE2 TYR A  93     4692   3432   1802   -692    224    999       C  \\nATOM    169  CZ  TYR A  93      33.070   4.827  15.874  1.00 32.94           C  \\nANISOU  169  CZ  TYR A  93     5569   4524   2424   -777    417   1012       C  \\nATOM    170  OH  TYR A  93      32.866   3.724  16.653  1.00 37.68           O  \\nANISOU  170  OH  TYR A  93     6289   5122   2905   -863    388   1217       O  \\nATOM    171  N   GLY A  94      36.775   8.688  13.859  1.00 27.56           N  \\nANISOU  171  N   GLY A  94     4631   3486   2355   -341    -90    240       N  \\nATOM    172  CA  GLY A  94      38.087   8.260  14.302  1.00 26.35           C  \\nANISOU  172  CA  GLY A  94     4513   3313   2186   -331   -418    327       C  \\nATOM    173  C   GLY A  94      38.758   9.167  15.325  1.00 30.85           C  \\nANISOU  173  C   GLY A  94     5189   4010   2522   -387   -592    152       C  \\nATOM    174  O   GLY A  94      39.384   8.678  16.259  1.00 34.40           O  \\nANISOU  174  O   GLY A  94     5764   4561   2745   -420   -848    270       O  \\nATOM    175  N   ILE A  95      38.646  10.484  15.172  1.00 30.72           N  \\nANISOU  175  N   ILE A  95     5156   3973   2545   -403   -483   -126       N  \\nATOM    176  CA  ILE A  95      39.352  11.367  16.102  1.00 36.00           C  \\nANISOU  176  CA  ILE A  95     5959   4723   2995   -499   -676   -329       C  \\nATOM    177  C   ILE A  95      38.577  11.567  17.405  1.00 41.88           C  \\nANISOU  177  C   ILE A  95     7009   5651   3252   -556   -564   -411       C  \\nATOM    178  O   ILE A  95      39.175  11.835  18.445  1.00 44.79           O  \\nANISOU  178  O   ILE A  95     7564   6144   3311   -660   -790   -498       O  \\nATOM    179  CB  ILE A  95      39.666  12.730  15.469  1.00 31.59           C  \\nANISOU  179  CB  ILE A  95     5328   4008   2665   -527   -628   -604       C  \\nATOM    180  CG1 ILE A  95      38.392  13.480  15.107  1.00 44.59           C  \\nANISOU  180  CG1 ILE A  95     7048   5569   4326   -434   -264   -754       C  \\nATOM    181  CG2 ILE A  95      40.500  12.536  14.236  1.00 28.34           C  \\nANISOU  181  CG2 ILE A  95     4621   3465   2683   -501   -716   -515       C  \\nATOM    182  CD1 ILE A  95      38.625  14.614  14.129  1.00 56.29           C  \\nANISOU  182  CD1 ILE A  95     8453   6814   6122   -422   -200   -928       C  \\nATOM    183  N   ARG A  96      37.255  11.416  17.350  1.00 44.07           N  \\nANISOU  183  N   ARG A  96     7323   5977   3443   -500   -216   -383       N  \\nATOM    184  CA  ARG A  96      36.445  11.402  18.559  1.00 48.82           C  \\nANISOU  184  CA  ARG A  96     8152   6800   3599   -539    -44   -413       C  \\nATOM    185  C   ARG A  96      36.929  10.279  19.462  1.00 76.51           C  \\nANISOU  185  C   ARG A  96    11748  10437   6886   -616   -281   -135       C  \\nATOM    186  O   ARG A  96      37.215  10.485  20.641  1.00 72.86           O  \\nANISOU  186  O   ARG A  96    11427  10122   6136   -653   -391   -197       O  \\nATOM    187  CB  ARG A  96      34.965  11.213  18.224  1.00 37.76           C  \\nANISOU  187  CB  ARG A  96     6623   5470   2255   -460    359   -360       C  \\nATOM    188  N   ARG A  97      37.055   9.096  18.870  1.00109.45           N  \\nANISOU  188  N   ARG A  97    15842  14523  11220   -617   -370    171       N  \\nATOM    189  CA  ARG A  97      37.355   7.874  19.604  1.00126.28           C  \\nANISOU  189  CA  ARG A  97    18046  16706  13230   -638   -546    477       C  \\nATOM    190  C   ARG A  97      38.833   7.466  19.569  1.00135.45           C  \\nANISOU  190  C   ARG A  97    19165  17788  14510   -587   -990    594       C  \\nATOM    191  O   ARG A  97      39.151   6.276  19.611  1.00149.94           O  \\nANISOU  191  O   ARG A  97    20993  19546  16431   -527  -1126    878       O  \\nATOM    192  CB  ARG A  97      36.497   6.726  19.058  1.00126.38           C  \\nANISOU  192  CB  ARG A  97    17986  16628  13406   -649   -349    735       C  \\nATOM    193  N   LEU A  98      39.734   8.442  19.490  1.00 91.41           N  \\nANISOU  193  N   LEU A  98    13524  12225   8981   -599  -1204    365       N  \\nATOM    194  CA  LEU A  98      41.164   8.149  19.561  1.00 61.19           C  \\nANISOU  194  CA  LEU A  98     9556   8401   5292   -545  -1627    456       C  \\nATOM    195  C   LEU A  98      41.950   9.379  19.988  1.00 66.32           C  \\nANISOU  195  C   LEU A  98    10163   9158   5878   -647  -1813    155       C  \\nATOM    196  O   LEU A  98      43.006   9.262  20.609  1.00 71.98           O  \\nANISOU  196  O   LEU A  98    10796  10000   6555   -646  -2141    198       O  \\nATOM    197  CB  LEU A  98      41.686   7.629  18.218  1.00 51.35           C  \\nANISOU  197  CB  LEU A  98     8064   6955   4491   -419  -1724    582       C  \\nTER     198      LEU A  98                                                      \\nMASTER      286    0    0    1    0    0    0    6  197    1    0    3          \\nEND                                                                             \\n\",\"pdb\");\n","\tviewer_17294480429883013.addModel(\"HEADER                                            20-OCT-24   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 20-OCT-24                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2024.42+release.3366cf78a3                                  \\nATOM      1  N   ASP A   1       7.436  21.211 -12.237  1.00  0.00           N  \\nATOM      2  CA  ASP A   1       6.048  21.655 -12.263  1.00  0.00           C  \\nATOM      3  C   ASP A   1       5.092  20.480 -12.423  1.00  0.00           C  \\nATOM      4  O   ASP A   1       4.104  20.565 -13.153  1.00  0.00           O  \\nATOM      5  CB  ASP A   1       5.827  22.657 -13.399  1.00  0.00           C  \\nATOM      6  CG  ASP A   1       6.087  22.058 -14.775  1.00  0.00           C  \\nATOM      7  OD1 ASP A   1       6.466  20.913 -14.842  1.00  0.00           O  \\nATOM      8  OD2 ASP A   1       5.905  22.752 -15.747  1.00  0.00           O  \\nATOM      9 1H   ASP A   1       8.044  22.011 -12.130  1.00  0.00           H  \\nATOM     10 2H   ASP A   1       7.573  20.579 -11.461  1.00  0.00           H  \\nATOM     11 3H   ASP A   1       7.655  20.736 -13.101  1.00  0.00           H  \\nATOM     12  HA  ASP A   1       5.823  22.147 -11.316  1.00  0.00           H  \\nATOM     13 1HB  ASP A   1       4.802  23.024 -13.366  1.00  0.00           H  \\nATOM     14 2HB  ASP A   1       6.487  23.514 -13.261  1.00  0.00           H  \\nATOM     15  N   ASN A   2       5.419  19.372 -11.775  1.00  0.00           N  \\nATOM     16  CA  ASN A   2       4.593  18.172 -11.839  1.00  0.00           C  \\nATOM     17  C   ASN A   2       3.919  17.836 -10.525  1.00  0.00           C  \\nATOM     18  O   ASN A   2       3.470  16.709 -10.316  1.00  0.00           O  \\nATOM     19  CB  ASN A   2       5.422  16.995 -12.311  1.00  0.00           C  \\nATOM     20  CG  ASN A   2       5.858  17.133 -13.743  1.00  0.00           C  \\nATOM     21  OD1 ASN A   2       5.072  16.901 -14.669  1.00  0.00           O  \\nATOM     22  ND2 ASN A   2       7.096  17.506 -13.943  1.00  0.00           N  \\nATOM     23  H   ASN A   2       6.247  19.366 -11.196  1.00  0.00           H  \\nATOM     24  HA  ASN A   2       3.786  18.345 -12.552  1.00  0.00           H  \\nATOM     25 1HB  ASN A   2       6.308  16.898 -11.681  1.00  0.00           H  \\nATOM     26 2HB  ASN A   2       4.844  16.077 -12.207  1.00  0.00           H  \\nATOM     27 1HD2 ASN A   2       7.441  17.614 -14.876  1.00  0.00           H  \\nATOM     28 2HD2 ASN A   2       7.697  17.682 -13.164  1.00  0.00           H  \\nATOM     29  N   GLY A   3       3.876  18.809  -9.629  1.00  0.00           N  \\nATOM     30  CA  GLY A   3       3.270  18.656  -8.321  1.00  0.00           C  \\nATOM     31  C   GLY A   3       4.017  19.439  -7.269  1.00  0.00           C  \\nATOM     32  O   GLY A   3       5.118  19.934  -7.513  1.00  0.00           O  \\nATOM     33  H   GLY A   3       4.266  19.708  -9.875  1.00  0.00           H  \\nATOM     34 1HA  GLY A   3       2.234  18.992  -8.359  1.00  0.00           H  \\nATOM     35 2HA  GLY A   3       3.256  17.601  -8.049  1.00  0.00           H  \\nATOM     36  N   GLU A   4       3.418  19.550  -6.100  1.00  0.00           N  \\nATOM     37  CA  GLU A   4       3.951  20.271  -4.966  1.00  0.00           C  \\nATOM     38  C   GLU A   4       5.064  19.514  -4.308  1.00  0.00           C  \\nATOM     39  O   GLU A   4       5.212  18.318  -4.533  1.00  0.00           O  \\nATOM     40  CB  GLU A   4       2.855  20.552  -3.947  1.00  0.00           C  \\nATOM     41  CG  GLU A   4       1.754  21.468  -4.446  1.00  0.00           C  \\nATOM     42  CD  GLU A   4       2.231  22.869  -4.704  1.00  0.00           C  \\nATOM     43  OE1 GLU A   4       2.854  23.433  -3.837  1.00  0.00           O  \\nATOM     44  OE2 GLU A   4       1.973  23.378  -5.771  1.00  0.00           O  \\nATOM     45  H   GLU A   4       2.521  19.098  -5.988  1.00  0.00           H  \\nATOM     46  HA  GLU A   4       4.351  21.222  -5.318  1.00  0.00           H  \\nATOM     47 1HB  GLU A   4       2.398  19.619  -3.641  1.00  0.00           H  \\nATOM     48 2HB  GLU A   4       3.293  21.009  -3.058  1.00  0.00           H  \\nATOM     49 1HG  GLU A   4       1.348  21.060  -5.371  1.00  0.00           H  \\nATOM     50 2HG  GLU A   4       0.953  21.492  -3.709  1.00  0.00           H  \\nATOM     51  N   TRP A   5       5.858  20.154  -3.467  1.00  0.00           N  \\nATOM     52  CA  TRP A   5       6.909  19.411  -2.794  1.00  0.00           C  \\nATOM     53  C   TRP A   5       6.366  18.204  -2.044  1.00  0.00           C  \\nATOM     54  O   TRP A   5       7.076  17.215  -1.861  1.00  0.00           O  \\nATOM     55  CB  TRP A   5       7.655  20.320  -1.821  1.00  0.00           C  \\nATOM     56  CG  TRP A   5       6.816  20.776  -0.665  1.00  0.00           C  \\nATOM     57  CD1 TRP A   5       6.077  21.916  -0.595  1.00  0.00           C  \\nATOM     58  CD2 TRP A   5       6.625  20.095   0.604  1.00  0.00           C  \\nATOM     59  NE1 TRP A   5       5.443  21.996   0.621  1.00  0.00           N  \\nATOM     60  CE2 TRP A   5       5.767  20.888   1.365  1.00  0.00           C  \\nATOM     61  CE3 TRP A   5       7.107  18.895   1.140  1.00  0.00           C  \\nATOM     62  CZ2 TRP A   5       5.376  20.525   2.645  1.00  0.00           C  \\nATOM     63  CZ3 TRP A   5       6.715  18.530   2.421  1.00  0.00           C  \\nATOM     64  CH2 TRP A   5       5.871  19.324   3.153  1.00  0.00           C  \\nATOM     65  H   TRP A   5       5.738  21.140  -3.288  1.00  0.00           H  \\nATOM     66  HA  TRP A   5       7.610  19.047  -3.545  1.00  0.00           H  \\nATOM     67 1HB  TRP A   5       8.525  19.795  -1.428  1.00  0.00           H  \\nATOM     68 2HB  TRP A   5       8.016  21.201  -2.351  1.00  0.00           H  \\nATOM     69  HD1 TRP A   5       5.999  22.658  -1.387  1.00  0.00           H  \\nATOM     70  HE1 TRP A   5       4.837  22.746   0.921  1.00  0.00           H  \\nATOM     71  HE3 TRP A   5       7.776  18.259   0.561  1.00  0.00           H  \\nATOM     72  HZ2 TRP A   5       4.706  21.143   3.243  1.00  0.00           H  \\nATOM     73  HZ3 TRP A   5       7.096  17.595   2.831  1.00  0.00           H  \\nATOM     74  HH2 TRP A   5       5.583  19.008   4.156  1.00  0.00           H  \\nATOM     75  N   ILE A   6       5.119  18.281  -1.571  1.00  0.00           N  \\nATOM     76  CA  ILE A   6       4.508  17.222  -0.803  1.00  0.00           C  \\nATOM     77  C   ILE A   6       3.888  16.122  -1.645  1.00  0.00           C  \\nATOM     78  O   ILE A   6       3.406  15.131  -1.097  1.00  0.00           O  \\nATOM     79  CB  ILE A   6       3.429  17.801   0.126  1.00  0.00           C  \\nATOM     80  CG1 ILE A   6       3.132  16.832   1.265  1.00  0.00           C  \\nATOM     81  CG2 ILE A   6       2.168  18.110  -0.654  1.00  0.00           C  \\nATOM     82  CD1 ILE A   6       2.310  17.434   2.380  1.00  0.00           C  \\nATOM     83  H   ILE A   6       4.585  19.121  -1.744  1.00  0.00           H  \\nATOM     84  HA  ILE A   6       5.277  16.746  -0.198  1.00  0.00           H  \\nATOM     85  HB  ILE A   6       3.796  18.718   0.580  1.00  0.00           H  \\nATOM     86 1HG1 ILE A   6       2.598  15.967   0.876  1.00  0.00           H  \\nATOM     87 2HG1 ILE A   6       4.070  16.472   1.690  1.00  0.00           H  \\nATOM     88 1HG2 ILE A   6       1.416  18.518   0.018  1.00  0.00           H  \\nATOM     89 2HG2 ILE A   6       2.391  18.835  -1.431  1.00  0.00           H  \\nATOM     90 3HG2 ILE A   6       1.790  17.196  -1.110  1.00  0.00           H  \\nATOM     91 1HD1 ILE A   6       2.141  16.685   3.153  1.00  0.00           H  \\nATOM     92 2HD1 ILE A   6       2.845  18.284   2.808  1.00  0.00           H  \\nATOM     93 3HD1 ILE A   6       1.353  17.770   1.985  1.00  0.00           H  \\nATOM     94  N   GLN A   7       3.785  16.339  -2.958  1.00  0.00           N  \\nATOM     95  CA  GLN A   7       3.173  15.423  -3.904  1.00  0.00           C  \\nATOM     96  C   GLN A   7       4.251  14.682  -4.647  1.00  0.00           C  \\nATOM     97  O   GLN A   7       3.970  13.839  -5.500  1.00  0.00           O  \\nATOM     98  CB  GLN A   7       2.269  16.166  -4.885  1.00  0.00           C  \\nATOM     99  CG  GLN A   7       1.077  16.826  -4.246  1.00  0.00           C  \\nATOM    100  CD  GLN A   7       0.226  17.581  -5.250  1.00  0.00           C  \\nATOM    101  OE1 GLN A   7       0.711  18.477  -5.945  1.00  0.00           O  \\nATOM    102  NE2 GLN A   7      -1.049  17.222  -5.331  1.00  0.00           N  \\nATOM    103  H   GLN A   7       4.240  17.108  -3.418  1.00  0.00           H  \\nATOM    104  HA  GLN A   7       2.567  14.708  -3.350  1.00  0.00           H  \\nATOM    105 1HB  GLN A   7       2.842  16.933  -5.398  1.00  0.00           H  \\nATOM    106 2HB  GLN A   7       1.903  15.471  -5.640  1.00  0.00           H  \\nATOM    107 1HG  GLN A   7       0.457  16.062  -3.780  1.00  0.00           H  \\nATOM    108 2HG  GLN A   7       1.424  17.529  -3.498  1.00  0.00           H  \\nATOM    109 1HE2 GLN A   7      -1.659  17.685  -5.975  1.00  0.00           H  \\nATOM    110 2HE2 GLN A   7      -1.401  16.490  -4.748  1.00  0.00           H  \\nATOM    111  N   LEU A   8       5.489  15.050  -4.366  1.00  0.00           N  \\nATOM    112  CA  LEU A   8       6.617  14.441  -5.027  1.00  0.00           C  \\nATOM    113  C   LEU A   8       7.456  13.679  -4.026  1.00  0.00           C  \\nATOM    114  O   LEU A   8       7.820  14.204  -2.973  1.00  0.00           O  \\nATOM    115  CB  LEU A   8       7.463  15.509  -5.723  1.00  0.00           C  \\nATOM    116  CG  LEU A   8       6.733  16.340  -6.822  1.00  0.00           C  \\nATOM    117  CD1 LEU A   8       7.660  17.419  -7.339  1.00  0.00           C  \\nATOM    118  CD2 LEU A   8       6.289  15.419  -7.943  1.00  0.00           C  \\nATOM    119  H   LEU A   8       5.656  15.746  -3.653  1.00  0.00           H  \\nATOM    120  HA  LEU A   8       6.244  13.748  -5.779  1.00  0.00           H  \\nATOM    121 1HB  LEU A   8       7.832  16.206  -4.970  1.00  0.00           H  \\nATOM    122 2HB  LEU A   8       8.320  15.024  -6.190  1.00  0.00           H  \\nATOM    123  HG  LEU A   8       5.864  16.829  -6.389  1.00  0.00           H  \\nATOM    124 1HD1 LEU A   8       7.149  18.002  -8.108  1.00  0.00           H  \\nATOM    125 2HD1 LEU A   8       7.948  18.077  -6.517  1.00  0.00           H  \\nATOM    126 3HD1 LEU A   8       8.551  16.960  -7.765  1.00  0.00           H  \\nATOM    127 1HD2 LEU A   8       5.777  16.001  -8.711  1.00  0.00           H  \\nATOM    128 2HD2 LEU A   8       7.160  14.930  -8.378  1.00  0.00           H  \\nATOM    129 3HD2 LEU A   8       5.611  14.665  -7.547  1.00  0.00           H  \\nATOM    130  N   VAL A   9       7.808  12.453  -4.383  1.00  0.00           N  \\nATOM    131  CA  VAL A   9       8.595  11.589  -3.523  1.00  0.00           C  \\nATOM    132  C   VAL A   9       9.915  11.283  -4.227  1.00  0.00           C  \\nATOM    133  O   VAL A   9       9.960  11.156  -5.453  1.00  0.00           O  \\nATOM    134  CB  VAL A   9       7.847  10.285  -3.216  1.00  0.00           C  \\nATOM    135  CG1 VAL A   9       6.562  10.575  -2.457  1.00  0.00           C  \\nATOM    136  CG2 VAL A   9       7.553   9.540  -4.510  1.00  0.00           C  \\nATOM    137  H   VAL A   9       7.482  12.086  -5.266  1.00  0.00           H  \\nATOM    138  HA  VAL A   9       8.774  12.104  -2.593  1.00  0.00           H  \\nATOM    139  HB  VAL A   9       8.468   9.664  -2.572  1.00  0.00           H  \\nATOM    140 1HG1 VAL A   9       6.045   9.639  -2.248  1.00  0.00           H  \\nATOM    141 2HG1 VAL A   9       6.799  11.075  -1.519  1.00  0.00           H  \\nATOM    142 3HG1 VAL A   9       5.920  11.217  -3.061  1.00  0.00           H  \\nATOM    143 1HG2 VAL A   9       7.024   8.616  -4.285  1.00  0.00           H  \\nATOM    144 2HG2 VAL A   9       6.936  10.165  -5.158  1.00  0.00           H  \\nATOM    145 3HG2 VAL A   9       8.493   9.306  -5.018  1.00  0.00           H  \\nATOM    146  N   PRO A  10      10.993  11.132  -3.438  1.00  0.00           N  \\nATOM    147  CA  PRO A  10      12.249  10.849  -4.075  1.00  0.00           C  \\nATOM    148  C   PRO A  10      12.238   9.632  -4.980  1.00  0.00           C  \\nATOM    149  O   PRO A  10      11.698   8.572  -4.665  1.00  0.00           O  \\nATOM    150  CB  PRO A  10      13.166  10.636  -2.869  1.00  0.00           C  \\nATOM    151  CG  PRO A  10      12.527  11.465  -1.762  1.00  0.00           C  \\nATOM    152  CD  PRO A  10      11.049  11.294  -1.973  1.00  0.00           C  \\nATOM    153  HA  PRO A  10      12.560  11.722  -4.667  1.00  0.00           H  \\nATOM    154 1HB  PRO A  10      13.222   9.566  -2.623  1.00  0.00           H  \\nATOM    155 2HB  PRO A  10      14.188  10.963  -3.112  1.00  0.00           H  \\nATOM    156 1HG  PRO A  10      12.859  11.105  -0.778  1.00  0.00           H  \\nATOM    157 2HG  PRO A  10      12.847  12.515  -1.841  1.00  0.00           H  \\nATOM    158 1HD  PRO A  10      10.707  10.387  -1.455  1.00  0.00           H  \\nATOM    159 2HD  PRO A  10      10.522  12.175  -1.598  1.00  0.00           H  \\nATOM    160  N   ARG A  11      12.886   9.835  -6.115  1.00  0.00           N  \\nATOM    161  CA  ARG A  11      13.026   8.873  -7.192  1.00  0.00           C  \\nATOM    162  C   ARG A  11      14.441   8.327  -7.158  1.00  0.00           C  \\nATOM    163  O   ARG A  11      15.347   8.950  -6.606  1.00  0.00           O  \\nATOM    164  CB  ARG A  11      12.743   9.507  -8.539  1.00  0.00           C  \\nATOM    165  CG  ARG A  11      11.357  10.116  -8.682  1.00  0.00           C  \\nATOM    166  CD  ARG A  11      10.303   9.074  -8.726  1.00  0.00           C  \\nATOM    167  NE  ARG A  11       8.984   9.646  -8.959  1.00  0.00           N  \\nATOM    168  CZ  ARG A  11       7.844   8.932  -9.038  1.00  0.00           C  \\nATOM    169  NH1 ARG A  11       7.876   7.625  -8.901  1.00  0.00           N  \\nATOM    170  NH2 ARG A  11       6.693   9.546  -9.252  1.00  0.00           N  \\nATOM    171  H   ARG A  11      13.279  10.753  -6.262  1.00  0.00           H  \\nATOM    172  HA  ARG A  11      12.305   8.071  -7.036  1.00  0.00           H  \\nATOM    173 1HB  ARG A  11      13.470  10.295  -8.730  1.00  0.00           H  \\nATOM    174 2HB  ARG A  11      12.858   8.759  -9.324  1.00  0.00           H  \\nATOM    175 1HG  ARG A  11      11.158  10.769  -7.832  1.00  0.00           H  \\nATOM    176 2HG  ARG A  11      11.307  10.694  -9.604  1.00  0.00           H  \\nATOM    177 1HD  ARG A  11      10.518   8.374  -9.532  1.00  0.00           H  \\nATOM    178 2HD  ARG A  11      10.280   8.539  -7.776  1.00  0.00           H  \\nATOM    179  HE  ARG A  11       8.919  10.649  -9.069  1.00  0.00           H  \\nATOM    180 1HH1 ARG A  11       8.755   7.156  -8.737  1.00  0.00           H  \\nATOM    181 2HH1 ARG A  11       7.021   7.091  -8.960  1.00  0.00           H  \\nATOM    182 1HH2 ARG A  11       6.668  10.552  -9.357  1.00  0.00           H  \\nATOM    183 2HH2 ARG A  11       5.839   9.012  -9.311  1.00  0.00           H  \\nATOM    184  N   PHE A  12      14.617   7.150  -7.742  1.00  0.00           N  \\nATOM    185  CA  PHE A  12      15.949   6.579  -7.838  1.00  0.00           C  \\nATOM    186  C   PHE A  12      16.653   7.364  -8.919  1.00  0.00           C  \\nATOM    187  O   PHE A  12      16.189   7.397 -10.059  1.00  0.00           O  \\nATOM    188  CB  PHE A  12      15.917   5.099  -8.180  1.00  0.00           C  \\nATOM    189  CG  PHE A  12      17.272   4.464  -8.241  1.00  0.00           C  \\nATOM    190  CD1 PHE A  12      17.956   4.149  -7.081  1.00  0.00           C  \\nATOM    191  CD2 PHE A  12      17.866   4.180  -9.464  1.00  0.00           C  \\nATOM    192  CE1 PHE A  12      19.206   3.563  -7.136  1.00  0.00           C  \\nATOM    193  CE2 PHE A  12      19.114   3.594  -9.522  1.00  0.00           C  \\nATOM    194  CZ  PHE A  12      19.785   3.285  -8.357  1.00  0.00           C  \\nATOM    195  H   PHE A  12      13.837   6.673  -8.171  1.00  0.00           H  \\nATOM    196  HA  PHE A  12      16.444   6.691  -6.872  1.00  0.00           H  \\nATOM    197 1HB  PHE A  12      15.325   4.567  -7.437  1.00  0.00           H  \\nATOM    198 2HB  PHE A  12      15.433   4.959  -9.145  1.00  0.00           H  \\nATOM    199  HD1 PHE A  12      17.498   4.367  -6.117  1.00  0.00           H  \\nATOM    200  HD2 PHE A  12      17.336   4.425 -10.385  1.00  0.00           H  \\nATOM    201  HE1 PHE A  12      19.734   3.320  -6.215  1.00  0.00           H  \\nATOM    202  HE2 PHE A  12      19.570   3.375 -10.487  1.00  0.00           H  \\nATOM    203  HZ  PHE A  12      20.771   2.824  -8.401  1.00  0.00           H  \\nATOM    204  N   SER A  13      17.780   7.979  -8.597  1.00  0.00           N  \\nATOM    205  CA  SER A  13      18.446   8.760  -9.621  1.00  0.00           C  \\nATOM    206  C   SER A  13      19.538   7.917 -10.248  1.00  0.00           C  \\nATOM    207  O   SER A  13      20.607   7.744  -9.664  1.00  0.00           O  \\nATOM    208  CB  SER A  13      19.029  10.029  -9.037  1.00  0.00           C  \\nATOM    209  OG  SER A  13      19.680  10.785 -10.025  1.00  0.00           O  \\nATOM    210  H   SER A  13      18.153   7.914  -7.661  1.00  0.00           H  \\nATOM    211  HA  SER A  13      17.711   9.035 -10.379  1.00  0.00           H  \\nATOM    212 1HB  SER A  13      18.233  10.622  -8.587  1.00  0.00           H  \\nATOM    213 2HB  SER A  13      19.734   9.774  -8.247  1.00  0.00           H  \\nATOM    214  HG  SER A  13      20.378  10.223 -10.372  1.00  0.00           H  \\nATOM    215  N   GLY A  14      19.243   7.333 -11.402  1.00  0.00           N  \\nATOM    216  CA  GLY A  14      20.182   6.436 -12.025  1.00  0.00           C  \\nATOM    217  C   GLY A  14      20.902   6.893 -13.341  1.00  0.00           C  \\nATOM    218  O   GLY A  14      21.690   6.124 -13.896  1.00  0.00           O  \\nATOM    219  H   GLY A  14      18.355   7.515 -11.846  1.00  0.00           H  \\nATOM    220 1HA  GLY A  14      20.979   6.198 -11.321  1.00  0.00           H  \\nATOM    221 2HA  GLY A  14      19.682   5.501 -12.272  1.00  0.00           H  \\nATOM    222  N   PRO A  15      20.592   8.133 -13.888  1.00  0.00           N  \\nATOM    223  CA  PRO A  15      21.215   8.505 -15.121  1.00  0.00           C  \\nATOM    224  C   PRO A  15      22.676   8.545 -14.705  1.00  0.00           C  \\nATOM    225  O   PRO A  15      22.814   9.134 -13.639  1.00  0.00           O  \\nATOM    226  CB  PRO A  15      20.639   9.866 -15.467  1.00  0.00           C  \\nATOM    227  CG  PRO A  15      19.312   9.919 -14.702  1.00  0.00           C  \\nATOM    228  CD  PRO A  15      19.588   9.160 -13.412  1.00  0.00           C  \\nATOM    229  HA  PRO A  15      20.937   7.781 -15.900  1.00  0.00           H  \\nATOM    230 1HB  PRO A  15      21.340  10.661 -15.162  1.00  0.00           H  \\nATOM    231 2HB  PRO A  15      20.508   9.954 -16.555  1.00  0.00           H  \\nATOM    232 1HG  PRO A  15      19.017  10.963 -14.525  1.00  0.00           H  \\nATOM    233 2HG  PRO A  15      18.511   9.460 -15.300  1.00  0.00           H  \\nATOM    234 1HD  PRO A  15      20.010   9.832 -12.684  1.00  0.00           H  \\nATOM    235 2HD  PRO A  15      18.684   8.734 -13.051  1.00  0.00           H  \\nATOM    236  N   GLU A  16      23.636   8.080 -15.492  1.00  0.00           N  \\nATOM    237  CA  GLU A  16      25.064   8.066 -15.192  1.00  0.00           C  \\nATOM    238  C   GLU A  16      25.671   9.436 -14.897  1.00  0.00           C  \\nATOM    239  O   GLU A  16      26.438   9.738 -14.008  1.00  0.00           O  \\nATOM    240  CB  GLU A  16      25.825   7.437 -16.351  1.00  0.00           C  \\nATOM    241  CG  GLU A  16      27.310   7.248 -16.098  1.00  0.00           C  \\nATOM    242  CD  GLU A  16      28.009   6.535 -17.220  1.00  0.00           C  \\nATOM    243  OE1 GLU A  16      27.355   6.180 -18.173  1.00  0.00           O  \\nATOM    244  OE2 GLU A  16      29.198   6.343 -17.127  1.00  0.00           O  \\nATOM    245  H   GLU A  16      23.353   7.671 -16.371  1.00  0.00           H  \\nATOM    246  HA  GLU A  16      25.218   7.487 -14.322  1.00  0.00           H  \\nATOM    247 1HB  GLU A  16      25.398   6.460 -16.580  1.00  0.00           H  \\nATOM    248 2HB  GLU A  16      25.714   8.059 -17.239  1.00  0.00           H  \\nATOM    249 1HG  GLU A  16      27.772   8.225 -15.960  1.00  0.00           H  \\nATOM    250 2HG  GLU A  16      27.442   6.681 -15.175  1.00  0.00           H  \\nATOM    251  N   ILE A  17      25.122  10.530 -15.525  1.00  0.00           N  \\nATOM    252  CA  ILE A  17      25.956  11.732 -15.075  1.00  0.00           C  \\nATOM    253  C   ILE A  17      25.809  11.855 -13.502  1.00  0.00           C  \\nATOM    254  O   ILE A  17      26.844  11.929 -12.830  1.00  0.00           O  \\nATOM    255  CB  ILE A  17      25.506  13.031 -15.753  1.00  0.00           C  \\nATOM    256  CG1 ILE A  17      25.813  12.986 -17.248  1.00  0.00           C  \\nATOM    257  CG2 ILE A  17      26.177  14.225 -15.107  1.00  0.00           C  \\nATOM    258  CD1 ILE A  17      25.150  14.088 -18.041  1.00  0.00           C  \\nATOM    259  H   ILE A  17      24.375  10.536 -16.204  1.00  0.00           H  \\nATOM    260  HA  ILE A  17      26.997  11.552 -15.346  1.00  0.00           H  \\nATOM    261  HB  ILE A  17      24.425  13.137 -15.656  1.00  0.00           H  \\nATOM    262 1HG1 ILE A  17      26.889  13.055 -17.401  1.00  0.00           H  \\nATOM    263 2HG1 ILE A  17      25.486  12.029 -17.658  1.00  0.00           H  \\nATOM    264 1HG2 ILE A  17      25.847  15.138 -15.600  1.00  0.00           H  \\nATOM    265 2HG2 ILE A  17      25.911  14.266 -14.058  1.00  0.00           H  \\nATOM    266 3HG2 ILE A  17      27.258  14.131 -15.204  1.00  0.00           H  \\nATOM    267 1HD1 ILE A  17      25.415  13.989 -19.094  1.00  0.00           H  \\nATOM    268 2HD1 ILE A  17      24.067  14.016 -17.930  1.00  0.00           H  \\nATOM    269 3HD1 ILE A  17      25.489  15.055 -17.672  1.00  0.00           H  \\nATOM    270  N   THR A  18      24.557  11.813 -13.058  1.00  0.00           N  \\nATOM    271  CA  THR A  18      24.114  11.918 -11.659  1.00  0.00           C  \\nATOM    272  C   THR A  18      24.577  10.716 -10.863  1.00  0.00           C  \\nATOM    273  O   THR A  18      24.989  10.733  -9.704  1.00  0.00           O  \\nATOM    274  CB  THR A  18      22.604  12.040 -11.558  1.00  0.00           C  \\nATOM    275  OG1 THR A  18      22.000  10.913 -12.145  1.00  0.00           O  \\nATOM    276  CG2 THR A  18      22.124  13.288 -12.263  1.00  0.00           C  \\nATOM    277  H   THR A  18      23.830  11.750 -13.756  1.00  0.00           H  \\nATOM    278  HA  THR A  18      24.552  12.797 -11.229  1.00  0.00           H  \\nATOM    279  HB  THR A  18      22.312  12.087 -10.510  1.00  0.00           H  \\nATOM    280  HG1 THR A  18      22.255  10.858 -13.070  1.00  0.00           H  \\nATOM    281 1HG2 THR A  18      21.040  13.359 -12.180  1.00  0.00           H  \\nATOM    282 2HG2 THR A  18      22.582  14.164 -11.803  1.00  0.00           H  \\nATOM    283 3HG2 THR A  18      22.406  13.242 -13.316  1.00  0.00           H  \\nATOM    284  N   LEU A  19      24.686   9.580 -11.529  1.00  0.00           N  \\nATOM    285  CA  LEU A  19      25.117   8.419 -10.754  1.00  0.00           C  \\nATOM    286  C   LEU A  19      26.450   8.758 -10.136  1.00  0.00           C  \\nATOM    287  O   LEU A  19      26.491   8.670  -8.905  1.00  0.00           O  \\nATOM    288  CB  LEU A  19      25.236   7.190 -11.617  1.00  0.00           C  \\nATOM    289  CG  LEU A  19      25.741   5.945 -10.920  1.00  0.00           C  \\nATOM    290  CD1 LEU A  19      24.764   5.547  -9.821  1.00  0.00           C  \\nATOM    291  CD2 LEU A  19      25.905   4.828 -11.935  1.00  0.00           C  \\nATOM    292  H   LEU A  19      24.255   9.469 -12.430  1.00  0.00           H  \\nATOM    293  HA  LEU A  19      24.372   8.222  -9.983  1.00  0.00           H  \\nATOM    294 1HB  LEU A  19      24.255   6.961 -12.035  1.00  0.00           H  \\nATOM    295 2HB  LEU A  19      25.898   7.402 -12.417  1.00  0.00           H  \\nATOM    296  HG  LEU A  19      26.703   6.155 -10.451  1.00  0.00           H  \\nATOM    297 1HD1 LEU A  19      25.128   4.651  -9.319  1.00  0.00           H  \\nATOM    298 2HD1 LEU A  19      24.680   6.359  -9.097  1.00  0.00           H  \\nATOM    299 3HD1 LEU A  19      23.785   5.347 -10.258  1.00  0.00           H  \\nATOM    300 1HD2 LEU A  19      26.270   3.931 -11.433  1.00  0.00           H  \\nATOM    301 2HD2 LEU A  19      24.943   4.616 -12.403  1.00  0.00           H  \\nATOM    302 3HD2 LEU A  19      26.622   5.132 -12.700  1.00  0.00           H  \\nATOM    303  N   ILE A  20      27.352   9.217 -11.012  1.00  0.00           N  \\nATOM    304  CA  ILE A  20      28.706   9.560 -10.665  1.00  0.00           C  \\nATOM    305  C   ILE A  20      28.760  10.681  -9.651  1.00  0.00           C  \\nATOM    306  O   ILE A  20      29.514  10.412  -8.725  1.00  0.00           O  \\nATOM    307  CB  ILE A  20      29.498   9.962 -11.912  1.00  0.00           C  \\nATOM    308  CG1 ILE A  20      29.704   8.755 -12.823  1.00  0.00           C  \\nATOM    309  CG2 ILE A  20      30.835  10.572 -11.519  1.00  0.00           C  \\nATOM    310  CD1 ILE A  20      30.228   9.109 -14.191  1.00  0.00           C  \\nATOM    311  H   ILE A  20      27.078   9.279 -11.986  1.00  0.00           H  \\nATOM    312  HA  ILE A  20      29.182   8.685 -10.224  1.00  0.00           H  \\nATOM    313  HB  ILE A  20      28.932  10.690 -12.479  1.00  0.00           H  \\nATOM    314 1HG1 ILE A  20      30.404   8.064 -12.358  1.00  0.00           H  \\nATOM    315 2HG1 ILE A  20      28.755   8.226 -12.948  1.00  0.00           H  \\nATOM    316 1HG2 ILE A  20      31.385  10.851 -12.416  1.00  0.00           H  \\nATOM    317 2HG2 ILE A  20      30.666  11.458 -10.906  1.00  0.00           H  \\nATOM    318 3HG2 ILE A  20      31.415   9.844 -10.951  1.00  0.00           H  \\nATOM    319 1HD1 ILE A  20      30.349   8.199 -14.781  1.00  0.00           H  \\nATOM    320 2HD1 ILE A  20      29.522   9.776 -14.691  1.00  0.00           H  \\nATOM    321 3HD1 ILE A  20      31.191   9.606 -14.093  1.00  0.00           H  \\nATOM    322  N   ILE A  21      27.917  11.710  -9.831  1.00  0.00           N  \\nATOM    323  CA  ILE A  21      27.917  12.857  -8.929  1.00  0.00           C  \\nATOM    324  C   ILE A  21      27.664  12.462  -7.518  1.00  0.00           C  \\nATOM    325  O   ILE A  21      28.520  12.873  -6.735  1.00  0.00           O  \\nATOM    326  CB  ILE A  21      26.858  13.895  -9.348  1.00  0.00           C  \\nATOM    327  CG1 ILE A  21      27.247  14.551 -10.687  1.00  0.00           C  \\nATOM    328  CG2 ILE A  21      26.689  14.951  -8.263  1.00  0.00           C  \\nATOM    329  CD1 ILE A  21      26.132  15.364 -11.320  1.00  0.00           C  \\nATOM    330  H   ILE A  21      27.437  11.785 -10.722  1.00  0.00           H  \\nATOM    331  HA  ILE A  21      28.897  13.331  -8.973  1.00  0.00           H  \\nATOM    332  HB  ILE A  21      25.908  13.399  -9.506  1.00  0.00           H  \\nATOM    333 1HG1 ILE A  21      28.103  15.207 -10.535  1.00  0.00           H  \\nATOM    334 2HG1 ILE A  21      27.547  13.785 -11.391  1.00  0.00           H  \\nATOM    335 1HG2 ILE A  21      25.938  15.675  -8.575  1.00  0.00           H  \\nATOM    336 2HG2 ILE A  21      26.370  14.474  -7.339  1.00  0.00           H  \\nATOM    337 3HG2 ILE A  21      27.638  15.460  -8.100  1.00  0.00           H  \\nATOM    338 1HD1 ILE A  21      26.481  15.792 -12.257  1.00  0.00           H  \\nATOM    339 2HD1 ILE A  21      25.292  14.729 -11.510  1.00  0.00           H  \\nATOM    340 3HD1 ILE A  21      25.837  16.163 -10.644  1.00  0.00           H  \\nATOM    341  N   PHE A  22      26.628  11.680  -7.320  1.00  0.00           N  \\nATOM    342  CA  PHE A  22      26.147  11.174  -6.080  1.00  0.00           C  \\nATOM    343  C   PHE A  22      27.240  10.378  -5.431  1.00  0.00           C  \\nATOM    344  O   PHE A  22      27.427  10.762  -4.274  1.00  0.00           O  \\nATOM    345  CB  PHE A  22      24.911  10.307  -6.283  1.00  0.00           C  \\nATOM    346  CG  PHE A  22      24.377   9.711  -5.017  1.00  0.00           C  \\nATOM    347  CD1 PHE A  22      23.622  10.473  -4.140  1.00  0.00           C  \\nATOM    348  CD2 PHE A  22      24.630   8.384  -4.699  1.00  0.00           C  \\nATOM    349  CE1 PHE A  22      23.129   9.923  -2.971  1.00  0.00           C  \\nATOM    350  CE2 PHE A  22      24.138   7.832  -3.532  1.00  0.00           C  \\nATOM    351  CZ  PHE A  22      23.386   8.603  -2.667  1.00  0.00           C  \\nATOM    352  H   PHE A  22      26.082  11.447  -8.137  1.00  0.00           H  \\nATOM    353  HA  PHE A  22      25.864  12.017  -5.448  1.00  0.00           H  \\nATOM    354 1HB  PHE A  22      24.120  10.902  -6.738  1.00  0.00           H  \\nATOM    355 2HB  PHE A  22      25.144   9.497  -6.967  1.00  0.00           H  \\nATOM    356  HD1 PHE A  22      23.418  11.516  -4.381  1.00  0.00           H  \\nATOM    357  HD2 PHE A  22      25.224   7.775  -5.382  1.00  0.00           H  \\nATOM    358  HE1 PHE A  22      22.537  10.534  -2.290  1.00  0.00           H  \\nATOM    359  HE2 PHE A  22      24.342   6.788  -3.293  1.00  0.00           H  \\nATOM    360  HZ  PHE A  22      22.999   8.170  -1.746  1.00  0.00           H  \\nATOM    361  N   GLY A  23      27.886   9.490  -6.217  1.00  0.00           N  \\nATOM    362  CA  GLY A  23      28.910   8.610  -5.709  1.00  0.00           C  \\nATOM    363  C   GLY A  23      30.005   9.394  -5.050  1.00  0.00           C  \\nATOM    364  O   GLY A  23      30.211   9.001  -3.905  1.00  0.00           O  \\nATOM    365  H   GLY A  23      27.570   9.283  -7.158  1.00  0.00           H  \\nATOM    366 1HA  GLY A  23      28.470   7.914  -4.996  1.00  0.00           H  \\nATOM    367 2HA  GLY A  23      29.317   8.017  -6.527  1.00  0.00           H  \\nATOM    368  N   VAL A  24      30.441  10.466  -5.722  1.00  0.00           N  \\nATOM    369  CA  VAL A  24      31.529  11.313  -5.306  1.00  0.00           C  \\nATOM    370  C   VAL A  24      31.255  12.009  -4.026  1.00  0.00           C  \\nATOM    371  O   VAL A  24      32.165  11.881  -3.200  1.00  0.00           O  \\nATOM    372  CB  VAL A  24      31.825  12.370  -6.389  1.00  0.00           C  \\nATOM    373  CG1 VAL A  24      32.838  13.386  -5.879  1.00  0.00           C  \\nATOM    374  CG2 VAL A  24      32.331  11.692  -7.648  1.00  0.00           C  \\nATOM    375  H   VAL A  24      30.100  10.590  -6.669  1.00  0.00           H  \\nATOM    376  HA  VAL A  24      32.416  10.694  -5.168  1.00  0.00           H  \\nATOM    377  HB  VAL A  24      30.917  12.909  -6.610  1.00  0.00           H  \\nATOM    378 1HG1 VAL A  24      33.035  14.125  -6.656  1.00  0.00           H  \\nATOM    379 2HG1 VAL A  24      32.439  13.886  -4.996  1.00  0.00           H  \\nATOM    380 3HG1 VAL A  24      33.766  12.877  -5.621  1.00  0.00           H  \\nATOM    381 1HG2 VAL A  24      32.537  12.444  -8.408  1.00  0.00           H  \\nATOM    382 2HG2 VAL A  24      33.244  11.143  -7.423  1.00  0.00           H  \\nATOM    383 3HG2 VAL A  24      31.594  11.020  -8.008  1.00  0.00           H  \\nATOM    384  N   MET A  25      30.038  12.572  -3.929  1.00  0.00           N  \\nATOM    385  CA  MET A  25      29.677  13.309  -2.760  1.00  0.00           C  \\nATOM    386  C   MET A  25      29.721  12.432  -1.553  1.00  0.00           C  \\nATOM    387  O   MET A  25      30.374  12.882  -0.610  1.00  0.00           O  \\nATOM    388  CB  MET A  25      28.296  13.923  -2.928  1.00  0.00           C  \\nATOM    389  CG  MET A  25      28.226  15.041  -3.951  1.00  0.00           C  \\nATOM    390  SD  MET A  25      26.547  15.619  -4.234  1.00  0.00           S  \\nATOM    391  CE  MET A  25      26.202  16.433  -2.677  1.00  0.00           C  \\nATOM    392  H   MET A  25      29.443  12.590  -4.753  1.00  0.00           H  \\nATOM    393  HA  MET A  25      30.400  14.113  -2.622  1.00  0.00           H  \\nATOM    394 1HB  MET A  25      27.589  13.152  -3.228  1.00  0.00           H  \\nATOM    395 2HB  MET A  25      27.957  14.323  -1.972  1.00  0.00           H  \\nATOM    396 1HG  MET A  25      28.827  15.883  -3.610  1.00  0.00           H  \\nATOM    397 2HG  MET A  25      28.634  14.693  -4.898  1.00  0.00           H  \\nATOM    398 1HE  MET A  25      25.193  16.845  -2.697  1.00  0.00           H  \\nATOM    399 2HE  MET A  25      26.285  15.712  -1.862  1.00  0.00           H  \\nATOM    400 3HE  MET A  25      26.920  17.240  -2.522  1.00  0.00           H  \\nATOM    401  N   ALA A  26      29.170  11.222  -1.765  1.00  0.00           N  \\nATOM    402  CA  ALA A  26      29.021  10.244  -0.728  1.00  0.00           C  \\nATOM    403  C   ALA A  26      30.351   9.850  -0.208  1.00  0.00           C  \\nATOM    404  O   ALA A  26      30.373   9.930   1.010  1.00  0.00           O  \\nATOM    405  CB  ALA A  26      28.269   9.019  -1.231  1.00  0.00           C  \\nATOM    406  H   ALA A  26      28.716  11.000  -2.642  1.00  0.00           H  \\nATOM    407  HA  ALA A  26      28.447  10.694   0.081  1.00  0.00           H  \\nATOM    408 1HB  ALA A  26      28.192   8.285  -0.430  1.00  0.00           H  \\nATOM    409 2HB  ALA A  26      27.268   9.312  -1.553  1.00  0.00           H  \\nATOM    410 3HB  ALA A  26      28.799   8.589  -2.061  1.00  0.00           H  \\nATOM    411  N   GLY A  27      31.293   9.645  -1.121  1.00  0.00           N  \\nATOM    412  CA  GLY A  27      32.620   9.207  -0.861  1.00  0.00           C  \\nATOM    413  C   GLY A  27      33.309  10.147   0.077  1.00  0.00           C  \\nATOM    414  O   GLY A  27      33.800   9.561   1.047  1.00  0.00           O  \\nATOM    415  H   GLY A  27      31.019   9.588  -2.094  1.00  0.00           H  \\nATOM    416 1HA  GLY A  27      32.596   8.205  -0.434  1.00  0.00           H  \\nATOM    417 2HA  GLY A  27      33.172   9.144  -1.798  1.00  0.00           H  \\nATOM    418  N   ILE A  28      33.176  11.459  -0.212  1.00  0.00           N  \\nATOM    419  CA  ILE A  28      33.843  12.464   0.572  1.00  0.00           C  \\nATOM    420  C   ILE A  28      33.336  12.426   1.980  1.00  0.00           C  \\nATOM    421  O   ILE A  28      34.206  12.378   2.847  1.00  0.00           O  \\nATOM    422  CB  ILE A  28      33.633  13.863  -0.021  1.00  0.00           C  \\nATOM    423  CG1 ILE A  28      34.351  13.986  -1.363  1.00  0.00           C  \\nATOM    424  CG2 ILE A  28      34.121  14.930   0.948  1.00  0.00           C  \\nATOM    425  CD1 ILE A  28      33.976  15.220  -2.143  1.00  0.00           C  \\nATOM    426  H   ILE A  28      32.733  11.705  -1.093  1.00  0.00           H  \\nATOM    427  HA  ILE A  28      34.912  12.253   0.568  1.00  0.00           H  \\nATOM    428  HB  ILE A  28      32.576  14.019  -0.215  1.00  0.00           H  \\nATOM    429 1HG1 ILE A  28      35.427  13.999  -1.199  1.00  0.00           H  \\nATOM    430 2HG1 ILE A  28      34.126  13.114  -1.977  1.00  0.00           H  \\nATOM    431 1HG2 ILE A  28      33.965  15.915   0.512  1.00  0.00           H  \\nATOM    432 2HG2 ILE A  28      33.565  14.854   1.882  1.00  0.00           H  \\nATOM    433 3HG2 ILE A  28      35.183  14.784   1.145  1.00  0.00           H  \\nATOM    434 1HD1 ILE A  28      34.527  15.237  -3.084  1.00  0.00           H  \\nATOM    435 2HD1 ILE A  28      32.903  15.209  -2.350  1.00  0.00           H  \\nATOM    436 3HD1 ILE A  28      34.225  16.107  -1.563  1.00  0.00           H  \\nATOM    437  N   ILE A  29      32.005  12.308   2.083  1.00  0.00           N  \\nATOM    438  CA  ILE A  29      31.361  12.299   3.364  1.00  0.00           C  \\nATOM    439  C   ILE A  29      31.839  11.173   4.210  1.00  0.00           C  \\nATOM    440  O   ILE A  29      32.136  11.521   5.343  1.00  0.00           O  \\nATOM    441  CB  ILE A  29      29.834  12.206   3.212  1.00  0.00           C  \\nATOM    442  CG1 ILE A  29      29.281  13.489   2.600  1.00  0.00           C  \\nATOM    443  CG2 ILE A  29      29.182  11.930   4.556  1.00  0.00           C  \\nATOM    444  CD1 ILE A  29      27.842  13.382   2.157  1.00  0.00           C  \\nATOM    445  H   ILE A  29      31.446  12.391   1.238  1.00  0.00           H  \\nATOM    446  HA  ILE A  29      31.597  13.230   3.875  1.00  0.00           H  \\nATOM    447  HB  ILE A  29      29.593  11.419   2.544  1.00  0.00           H  \\nATOM    448 1HG1 ILE A  29      29.356  14.299   3.324  1.00  0.00           H  \\nATOM    449 2HG1 ILE A  29      29.882  13.768   1.736  1.00  0.00           H  \\nATOM    450 1HG2 ILE A  29      28.102  11.867   4.430  1.00  0.00           H  \\nATOM    451 2HG2 ILE A  29      29.555  10.989   4.954  1.00  0.00           H  \\nATOM    452 3HG2 ILE A  29      29.420  12.737   5.248  1.00  0.00           H  \\nATOM    453 1HD1 ILE A  29      27.521  14.333   1.732  1.00  0.00           H  \\nATOM    454 2HD1 ILE A  29      27.751  12.599   1.404  1.00  0.00           H  \\nATOM    455 3HD1 ILE A  29      27.216  13.138   3.012  1.00  0.00           H  \\nATOM    456  N   GLY A  30      31.948   9.995   3.597  1.00  0.00           N  \\nATOM    457  CA  GLY A  30      32.340   8.755   4.186  1.00  0.00           C  \\nATOM    458  C   GLY A  30      33.686   8.897   4.817  1.00  0.00           C  \\nATOM    459  O   GLY A  30      33.691   8.501   5.973  1.00  0.00           O  \\nATOM    460  H   GLY A  30      31.571   9.953   2.660  1.00  0.00           H  \\nATOM    461 1HA  GLY A  30      31.605   8.457   4.926  1.00  0.00           H  \\nATOM    462 2HA  GLY A  30      32.359   7.979   3.422  1.00  0.00           H  \\nATOM    463  N   THR A  31      34.588   9.538   4.072  1.00  0.00           N  \\nATOM    464  CA  THR A  31      35.941   9.714   4.510  1.00  0.00           C  \\nATOM    465  C   THR A  31      35.955  10.493   5.800  1.00  0.00           C  \\nATOM    466  O   THR A  31      36.556   9.928   6.708  1.00  0.00           O  \\nATOM    467  CB  THR A  31      36.786  10.431   3.447  1.00  0.00           C  \\nATOM    468  OG1 THR A  31      36.832   9.637   2.254  1.00  0.00           O  \\nATOM    469  CG2 THR A  31      38.197  10.658   3.954  1.00  0.00           C  \\nATOM    470  H   THR A  31      34.354   9.734   3.106  1.00  0.00           H  \\nATOM    471  HA  THR A  31      36.380   8.732   4.680  1.00  0.00           H  \\nATOM    472  HB  THR A  31      36.337  11.384   3.212  1.00  0.00           H  \\nATOM    473  HG1 THR A  31      35.940   9.509   1.921  1.00  0.00           H  \\nATOM    474 1HG2 THR A  31      38.781  11.167   3.188  1.00  0.00           H  \\nATOM    475 2HG2 THR A  31      38.166  11.270   4.854  1.00  0.00           H  \\nATOM    476 3HG2 THR A  31      38.659   9.699   4.183  1.00  0.00           H  \\nATOM    477  N   ILE A  32      35.139  11.563   5.810  1.00  0.00           N  \\nATOM    478  CA  ILE A  32      35.107  12.441   6.960  1.00  0.00           C  \\nATOM    479  C   ILE A  32      34.666  11.706   8.197  1.00  0.00           C  \\nATOM    480  O   ILE A  32      35.385  11.892   9.171  1.00  0.00           O  \\nATOM    481  CB  ILE A  32      34.170  13.634   6.714  1.00  0.00           C  \\nATOM    482  CG1 ILE A  32      34.747  14.551   5.638  1.00  0.00           C  \\nATOM    483  CG2 ILE A  32      33.938  14.402   8.003  1.00  0.00           C  \\nATOM    484  CD1 ILE A  32      33.774  15.589   5.136  1.00  0.00           C  \\nATOM    485  H   ILE A  32      34.710  11.835   4.931  1.00  0.00           H  \\nATOM    486  HA  ILE A  32      36.107  12.823   7.129  1.00  0.00           H  \\nATOM    487  HB  ILE A  32      33.233  13.281   6.348  1.00  0.00           H  \\nATOM    488 1HG1 ILE A  32      35.622  15.066   6.031  1.00  0.00           H  \\nATOM    489 2HG1 ILE A  32      35.075  13.953   4.787  1.00  0.00           H  \\nATOM    490 1HG2 ILE A  32      33.274  15.243   7.811  1.00  0.00           H  \\nATOM    491 2HG2 ILE A  32      33.484  13.742   8.743  1.00  0.00           H  \\nATOM    492 3HG2 ILE A  32      34.891  14.771   8.383  1.00  0.00           H  \\nATOM    493 1HD1 ILE A  32      34.256  16.202   4.374  1.00  0.00           H  \\nATOM    494 2HD1 ILE A  32      32.903  15.093   4.705  1.00  0.00           H  \\nATOM    495 3HD1 ILE A  32      33.459  16.222   5.963  1.00  0.00           H  \\nATOM    496  N   LEU A  33      33.626  10.891   8.025  1.00  0.00           N  \\nATOM    497  CA  LEU A  33      33.014  10.127   9.080  1.00  0.00           C  \\nATOM    498  C   LEU A  33      34.017   9.207   9.723  1.00  0.00           C  \\nATOM    499  O   LEU A  33      34.043   9.318  10.945  1.00  0.00           O  \\nATOM    500  CB  LEU A  33      31.836   9.312   8.533  1.00  0.00           C  \\nATOM    501  CG  LEU A  33      30.606  10.120   8.121  1.00  0.00           C  \\nATOM    502  CD1 LEU A  33      29.613   9.209   7.422  1.00  0.00           C  \\nATOM    503  CD2 LEU A  33      29.987  10.762   9.348  1.00  0.00           C  \\nATOM    504  H   LEU A  33      33.158  10.960   7.128  1.00  0.00           H  \\nATOM    505  HA  LEU A  33      32.638  10.818   9.834  1.00  0.00           H  \\nATOM    506 1HB  LEU A  33      32.166   8.769   7.683  1.00  0.00           H  \\nATOM    507 2HB  LEU A  33      31.523   8.599   9.294  1.00  0.00           H  \\nATOM    508  HG  LEU A  33      30.900  10.892   7.418  1.00  0.00           H  \\nATOM    509 1HD1 LEU A  33      28.736   9.785   7.127  1.00  0.00           H  \\nATOM    510 2HD1 LEU A  33      30.074   8.780   6.541  1.00  0.00           H  \\nATOM    511 3HD1 LEU A  33      29.312   8.411   8.099  1.00  0.00           H  \\nATOM    512 1HD2 LEU A  33      29.110  11.339   9.053  1.00  0.00           H  \\nATOM    513 2HD2 LEU A  33      29.691   9.986  10.054  1.00  0.00           H  \\nATOM    514 3HD2 LEU A  33      30.716  11.423   9.819  1.00  0.00           H  \\nATOM    515  N   LEU A  34      34.803   8.508   8.885  1.00  0.00           N  \\nATOM    516  CA  LEU A  34      35.787   7.543   9.321  1.00  0.00           C  \\nATOM    517  C   LEU A  34      36.803   8.218  10.213  1.00  0.00           C  \\nATOM    518  O   LEU A  34      37.027   7.622  11.265  1.00  0.00           O  \\nATOM    519  CB  LEU A  34      36.484   6.902   8.116  1.00  0.00           C  \\nATOM    520  CG  LEU A  34      35.623   5.970   7.271  1.00  0.00           C  \\nATOM    521  CD1 LEU A  34      36.388   5.565   6.023  1.00  0.00           C  \\nATOM    522  CD2 LEU A  34      35.236   4.754   8.094  1.00  0.00           C  \\nATOM    523  H   LEU A  34      34.589   8.560   7.896  1.00  0.00           H  \\nATOM    524  HA  LEU A  34      35.276   6.757   9.880  1.00  0.00           H  \\nATOM    525 1HB  LEU A  34      36.846   7.686   7.473  1.00  0.00           H  \\nATOM    526 2HB  LEU A  34      37.339   6.330   8.473  1.00  0.00           H  \\nATOM    527  HG  LEU A  34      34.728   6.492   6.956  1.00  0.00           H  \\nATOM    528 1HD1 LEU A  34      35.772   4.899   5.420  1.00  0.00           H  \\nATOM    529 2HD1 LEU A  34      36.636   6.453   5.444  1.00  0.00           H  \\nATOM    530 3HD1 LEU A  34      37.304   5.051   6.310  1.00  0.00           H  \\nATOM    531 1HD2 LEU A  34      34.619   4.087   7.490  1.00  0.00           H  \\nATOM    532 2HD2 LEU A  34      36.136   4.227   8.411  1.00  0.00           H  \\nATOM    533 3HD2 LEU A  34      34.673   5.072   8.973  1.00  0.00           H  \\nATOM    534  N   ILE A  35      37.195   9.432   9.779  1.00  0.00           N  \\nATOM    535  CA  ILE A  35      38.213  10.180  10.482  1.00  0.00           C  \\nATOM    536  C   ILE A  35      37.736  10.460  11.868  1.00  0.00           C  \\nATOM    537  O   ILE A  35      38.540  10.095  12.727  1.00  0.00           O  \\nATOM    538  CB  ILE A  35      38.546  11.501   9.765  1.00  0.00           C  \\nATOM    539  CG1 ILE A  35      39.239  11.224   8.430  1.00  0.00           C  \\nATOM    540  CG2 ILE A  35      39.416  12.380  10.648  1.00  0.00           C  \\nATOM    541  CD1 ILE A  35      39.349  12.435   7.538  1.00  0.00           C  \\nATOM    542  H   ILE A  35      36.932   9.723   8.843  1.00  0.00           H  \\nATOM    543  HA  ILE A  35      39.121   9.580  10.517  1.00  0.00           H  \\nATOM    544  HB  ILE A  35      37.647  12.019   9.542  1.00  0.00           H  \\nATOM    545 1HG1 ILE A  35      40.242  10.843   8.614  1.00  0.00           H  \\nATOM    546 2HG1 ILE A  35      38.694  10.455   7.892  1.00  0.00           H  \\nATOM    547 1HG2 ILE A  35      39.642  13.309  10.126  1.00  0.00           H  \\nATOM    548 2HG2 ILE A  35      38.886  12.604  11.573  1.00  0.00           H  \\nATOM    549 3HG2 ILE A  35      40.344  11.858  10.878  1.00  0.00           H  \\nATOM    550 1HD1 ILE A  35      39.851  12.160   6.611  1.00  0.00           H  \\nATOM    551 2HD1 ILE A  35      38.357  12.811   7.312  1.00  0.00           H  \\nATOM    552 3HD1 ILE A  35      39.923  13.209   8.044  1.00  0.00           H  \\nATOM    553  N   SER A  36      36.462  10.885  11.962  1.00  0.00           N  \\nATOM    554  CA  SER A  36      35.904  11.232  13.238  1.00  0.00           C  \\nATOM    555  C   SER A  36      35.873  10.058  14.160  1.00  0.00           C  \\nATOM    556  O   SER A  36      36.269  10.363  15.284  1.00  0.00           O  \\nATOM    557  CB  SER A  36      34.502  11.780  13.063  1.00  0.00           C  \\nATOM    558  OG  SER A  36      34.522  12.998  12.371  1.00  0.00           O  \\nATOM    559  H   SER A  36      35.991  11.122  11.096  1.00  0.00           H  \\nATOM    560  HA  SER A  36      36.528  12.003  13.690  1.00  0.00           H  \\nATOM    561 1HB  SER A  36      33.900  11.065  12.522  1.00  0.00           H  \\nATOM    562 2HB  SER A  36      34.042  11.922  14.039  1.00  0.00           H  \\nATOM    563  HG  SER A  36      34.919  12.810  11.516  1.00  0.00           H  \\nATOM    564  N   TYR A  37      35.561   8.876  13.600  1.00  0.00           N  \\nATOM    565  CA  TYR A  37      35.453   7.701  14.418  1.00  0.00           C  \\nATOM    566  C   TYR A  37      36.709   7.421  15.154  1.00  0.00           C  \\nATOM    567  O   TYR A  37      36.483   7.279  16.353  1.00  0.00           O  \\nATOM    568  CB  TYR A  37      35.071   6.494  13.571  1.00  0.00           C  \\nATOM    569  CG  TYR A  37      34.960   5.209  14.357  1.00  0.00           C  \\nATOM    570  CD1 TYR A  37      33.825   4.954  15.110  1.00  0.00           C  \\nATOM    571  CD2 TYR A  37      35.994   4.284  14.324  1.00  0.00           C  \\nATOM    572  CE1 TYR A  37      33.723   3.778  15.829  1.00  0.00           C  \\nATOM    573  CE2 TYR A  37      35.892   3.110  15.042  1.00  0.00           C  \\nATOM    574  CZ  TYR A  37      34.763   2.855  15.792  1.00  0.00           C  \\nATOM    575  OH  TYR A  37      34.662   1.684  16.508  1.00  0.00           O  \\nATOM    576  H   TYR A  37      35.199   8.914  12.656  1.00  0.00           H  \\nATOM    577  HA  TYR A  37      34.667   7.863  15.155  1.00  0.00           H  \\nATOM    578 1HB  TYR A  37      34.113   6.679  13.084  1.00  0.00           H  \\nATOM    579 2HB  TYR A  37      35.808   6.351  12.794  1.00  0.00           H  \\nATOM    580  HD1 TYR A  37      33.013   5.680  15.135  1.00  0.00           H  \\nATOM    581  HD2 TYR A  37      36.888   4.486  13.732  1.00  0.00           H  \\nATOM    582  HE1 TYR A  37      32.831   3.577  16.421  1.00  0.00           H  \\nATOM    583  HE2 TYR A  37      36.705   2.383  15.018  1.00  0.00           H  \\nATOM    584  HH  TYR A  37      35.456   1.161  16.377  1.00  0.00           H  \\nATOM    585  N   SER A  38      37.834   7.499  14.439  1.00  0.00           N  \\nATOM    586  CA  SER A  38      39.096   7.223  15.069  1.00  0.00           C  \\nATOM    587  C   SER A  38      39.422   8.338  16.019  1.00  0.00           C  \\nATOM    588  O   SER A  38      39.796   8.248  17.182  1.00  0.00           O  \\nATOM    589  CB  SER A  38      40.183   7.071  14.032  1.00  0.00           C  \\nATOM    590  OG  SER A  38      40.401   8.280  13.352  1.00  0.00           O  \\nATOM    591  H   SER A  38      37.722   7.670  13.443  1.00  0.00           H  \\nATOM    592  HA  SER A  38      39.009   6.286  15.622  1.00  0.00           H  \\nATOM    593 1HB  SER A  38      41.104   6.752  14.514  1.00  0.00           H  \\nATOM    594 2HB  SER A  38      39.901   6.297  13.320  1.00  0.00           H  \\nATOM    595  HG  SER A  38      39.566   8.503  12.934  1.00  0.00           H  \\nATOM    596  N   ILE A  39      38.994   9.590  15.672  1.00  0.00           N  \\nATOM    597  CA  ILE A  39      39.422  10.550  16.696  1.00  0.00           C  \\nATOM    598  C   ILE A  39      38.775  10.186  18.030  1.00  0.00           C  \\nATOM    599  O   ILE A  39      39.406  10.245  19.086  1.00  0.00           O  \\nATOM    600  CB  ILE A  39      39.052  11.983  16.296  1.00  0.00           C  \\nATOM    601  CG1 ILE A  39      39.878  12.429  15.093  1.00  0.00           C  \\nATOM    602  CG2 ILE A  39      39.255  12.929  17.463  1.00  0.00           C  \\nATOM    603  CD1 ILE A  39      39.399  13.710  14.469  1.00  0.00           C  \\nATOM    604  H   ILE A  39      38.657   9.815  14.746  1.00  0.00           H  \\nATOM    605  HA  ILE A  39      40.506  10.490  16.796  1.00  0.00           H  \\nATOM    606  HB  ILE A  39      38.011  12.017  15.992  1.00  0.00           H  \\nATOM    607 1HG1 ILE A  39      40.915  12.563  15.394  1.00  0.00           H  \\nATOM    608 2HG1 ILE A  39      39.858  11.651  14.330  1.00  0.00           H  \\nATOM    609 1HG2 ILE A  39      38.989  13.941  17.162  1.00  0.00           H  \\nATOM    610 2HG2 ILE A  39      38.623  12.620  18.296  1.00  0.00           H  \\nATOM    611 3HG2 ILE A  39      40.300  12.906  17.773  1.00  0.00           H  \\nATOM    612 1HD1 ILE A  39      40.035  13.962  13.621  1.00  0.00           H  \\nATOM    613 2HD1 ILE A  39      38.370  13.586  14.128  1.00  0.00           H  \\nATOM    614 3HD1 ILE A  39      39.443  14.511  15.205  1.00  0.00           H  \\nATOM    615  N   ARG A  40      37.510   9.765  17.984  1.00  0.00           N  \\nATOM    616  CA  ARG A  40      36.805   9.426  19.195  1.00  0.00           C  \\nATOM    617  C   ARG A  40      37.229   8.135  19.858  1.00  0.00           C  \\nATOM    618  O   ARG A  40      37.352   8.064  21.081  1.00  0.00           O  \\nATOM    619  CB  ARG A  40      35.318   9.349  18.903  1.00  0.00           C  \\nATOM    620  CG  ARG A  40      34.451   9.011  20.098  1.00  0.00           C  \\nATOM    621  CD  ARG A  40      34.548  10.046  21.156  1.00  0.00           C  \\nATOM    622  NE  ARG A  40      33.683   9.749  22.287  1.00  0.00           N  \\nATOM    623  CZ  ARG A  40      33.775  10.336  23.496  1.00  0.00           C  \\nATOM    624  NH1 ARG A  40      34.696  11.248  23.715  1.00  0.00           N  \\nATOM    625  NH2 ARG A  40      32.939   9.995  24.462  1.00  0.00           N  \\nATOM    626  H   ARG A  40      37.036   9.805  17.091  1.00  0.00           H  \\nATOM    627  HA  ARG A  40      36.988  10.208  19.932  1.00  0.00           H  \\nATOM    628 1HB  ARG A  40      34.975  10.303  18.507  1.00  0.00           H  \\nATOM    629 2HB  ARG A  40      35.136   8.593  18.140  1.00  0.00           H  \\nATOM    630 1HG  ARG A  40      33.410   8.939  19.783  1.00  0.00           H  \\nATOM    631 2HG  ARG A  40      34.769   8.057  20.521  1.00  0.00           H  \\nATOM    632 1HD  ARG A  40      35.574  10.102  21.517  1.00  0.00           H  \\nATOM    633 2HD  ARG A  40      34.254  11.012  20.747  1.00  0.00           H  \\nATOM    634  HE  ARG A  40      32.961   9.052  22.156  1.00  0.00           H  \\nATOM    635 1HH1 ARG A  40      35.335  11.508  22.977  1.00  0.00           H  \\nATOM    636 2HH1 ARG A  40      34.766  11.688  24.622  1.00  0.00           H  \\nATOM    637 1HH2 ARG A  40      32.231   9.294  24.293  1.00  0.00           H  \\nATOM    638 2HH2 ARG A  40      33.008  10.435  25.367  1.00  0.00           H  \\nATOM    639  N   ARG A  41      37.469   7.104  19.052  1.00  0.00           N  \\nATOM    640  CA  ARG A  41      37.814   5.797  19.566  1.00  0.00           C  \\nATOM    641  C   ARG A  41      39.212   5.753  20.138  1.00  0.00           C  \\nATOM    642  O   ARG A  41      39.448   5.114  21.163  1.00  0.00           O  \\nATOM    643  CB  ARG A  41      37.692   4.751  18.468  1.00  0.00           C  \\nATOM    644  CG  ARG A  41      38.019   3.332  18.899  1.00  0.00           C  \\nATOM    645  CD  ARG A  41      37.086   2.848  19.949  1.00  0.00           C  \\nATOM    646  NE  ARG A  41      37.369   1.472  20.333  1.00  0.00           N  \\nATOM    647  CZ  ARG A  41      36.891   0.873  21.445  1.00  0.00           C  \\nATOM    648  NH1 ARG A  41      36.113   1.540  22.268  1.00  0.00           N  \\nATOM    649  NH2 ARG A  41      37.207  -0.384  21.706  1.00  0.00           N  \\nATOM    650  H   ARG A  41      37.298   7.189  18.064  1.00  0.00           H  \\nATOM    651  HA  ARG A  41      37.117   5.545  20.366  1.00  0.00           H  \\nATOM    652 1HB  ARG A  41      36.674   4.750  18.078  1.00  0.00           H  \\nATOM    653 2HB  ARG A  41      38.358   5.008  17.645  1.00  0.00           H  \\nATOM    654 1HG  ARG A  41      37.945   2.665  18.040  1.00  0.00           H  \\nATOM    655 2HG  ARG A  41      39.034   3.298  19.299  1.00  0.00           H  \\nATOM    656 1HD  ARG A  41      37.179   3.475  20.835  1.00  0.00           H  \\nATOM    657 2HD  ARG A  41      36.063   2.894  19.577  1.00  0.00           H  \\nATOM    658  HE  ARG A  41      37.965   0.926  19.725  1.00  0.00           H  \\nATOM    659 1HH1 ARG A  41      35.872   2.501  22.068  1.00  0.00           H  \\nATOM    660 2HH1 ARG A  41      35.755   1.092  23.099  1.00  0.00           H  \\nATOM    661 1HH2 ARG A  41      37.805  -0.897  21.073  1.00  0.00           H  \\nATOM    662 2HH2 ARG A  41      36.849  -0.831  22.537  1.00  0.00           H  \\nATOM    663  N   LEU A  42      40.126   6.446  19.478  1.00  0.00           N  \\nATOM    664  CA  LEU A  42      41.504   6.475  19.919  1.00  0.00           C  \\nATOM    665  C   LEU A  42      41.711   7.665  20.846  1.00  0.00           C  \\nATOM    666  O   LEU A  42      42.176   7.518  21.976  1.00  0.00           O  \\nATOM    667  OXT LEU A  42      41.413   8.796  20.469  1.00  0.00           O  \\nATOM    668  CB  LEU A  42      42.444   6.559  18.718  1.00  0.00           C  \\nATOM    669  CG  LEU A  42      42.380   5.384  17.744  1.00  0.00           C  \\nATOM    670  CD1 LEU A  42      43.298   5.654  16.563  1.00  0.00           C  \\nATOM    671  CD2 LEU A  42      42.778   4.108  18.463  1.00  0.00           C  \\nATOM    672  H   LEU A  42      39.948   6.959  18.629  1.00  0.00           H  \\nATOM    673  HA  LEU A  42      41.713   5.554  20.462  1.00  0.00           H  \\nATOM    674 1HB  LEU A  42      42.213   7.466  18.159  1.00  0.00           H  \\nATOM    675 2HB  LEU A  42      43.468   6.633  19.082  1.00  0.00           H  \\nATOM    676  HG  LEU A  42      41.361   5.283  17.359  1.00  0.00           H  \\nATOM    677 1HD1 LEU A  42      43.253   4.816  15.868  1.00  0.00           H  \\nATOM    678 2HD1 LEU A  42      42.979   6.564  16.055  1.00  0.00           H  \\nATOM    679 3HD1 LEU A  42      44.320   5.776  16.918  1.00  0.00           H  \\nATOM    680 1HD2 LEU A  42      42.732   3.269  17.768  1.00  0.00           H  \\nATOM    681 2HD2 LEU A  42      43.793   4.207  18.846  1.00  0.00           H  \\nATOM    682 3HD2 LEU A  42      42.092   3.929  19.294  1.00  0.00           H  \\nTER                                                                             \\n##Begin comments##\\nMembraneInfo1 MembraneCenter   31.246   8.804  -0.232\\nMembraneInfo2 MembraneNormal   -0.323   0.059  -0.944\\nMembraneInfo3 ATOM   9999  X   MEM A 999      31.246   8.804  -0.232\\nMembraneInfo4 ATOM   9999  Y   MEM A 999      26.396   9.689 -14.398\\nMembraneInfo5 ATOM   9999  Z   MEM A 999      36.097   7.919  13.935\\n##End comments##\\n\\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\nlabel fa_atr fa_rep fa_intra_rep fa_mbenv fa_mbsolv fa_pair hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_ss_dst dslf_cs_ang dslf_ss_dih dslf_ca_dih atom_pair_constraint rama omega fa_dun p_aa_pp ref cart_bonded Menv_smooth total\\nweights 0.8 0.44 0.004 0.3 0.35 0.49 1.17 1.17 2.34 2.2 0.5 2 5 5 0.5 0.2 0.5 0.56 0.32 1 0.5 0.5 NA\\npose -133.204 18.5293 0.51391 13.033 19.6014 -0.35959 -46.0647 0 -1.32037 0 0 0 0 0 3.14625 -4.40959 -1.85384 25.4832 -2.73453 -4.81 83.8735 -4.86075 -35.4365\\nASP:NtermProteinFull_1 -0.8812 0.03743 0.00881 0.20092 0.60858 -0.06886 0 0 0 0 0 0 0 0 0 0 -0.00444 1.68137 0 -0.67 0.01721 -0.47447 0.45534\\nASN_2 -1.19997 0.04721 0.00545 0.09386 0.61421 -0.06886 0 0 0 0 0 0 0 0 0 -0.20887 -0.06538 1.27439 -0.29817 -0.89 0.06512 -0.24228 -0.8733\\nGLY_3 -1.81568 0.03233 0.0001 0.21393 0.32191 0 0 0 0 0 0 0 0 0 0 0.20176 -0.13284 0 -0.80782 -0.17 0.10881 1.13347 -0.91403\\nGLU_4 -3.67394 0.45303 0.0061 1.66872 0.61829 -0.04331 0 0 -0.66018 0 0 0 0 0 0 -0.01485 -0.02709 2.10747 0.09166 -0.81 0.21631 0.5479 0.48009\\nTRP_5 -3.0895 0.25889 0.01904 0.71348 0.33786 0 0 0 0 0 0 0 0 0 0 -0.34807 -0.13453 0.72036 0.03411 0.91 0.19796 -1.07633 -1.45672\\nILE_6 -3.41931 0.51227 0.02918 -0.24913 0.34045 0 0 0 0 0 0 0 0 0 0 -0.02471 -0.0413 1.27233 0.29943 0.24 0.45945 -0.85696 -1.4383\\nGLN_7 -3.69205 0.22846 0.00567 1.01614 0.54269 -0.04331 0 0 -0.66018 0 0 0 0 0 0 -0.16153 -0.04536 2.03188 -0.03889 -0.97 0.64349 -0.13084 -1.27385\\nLEU_8 -3.54818 0.17229 0.01229 0.22718 0.36021 0 0 0 0 0 0 0 0 0 0 -0.01281 -0.03945 0.36171 -0.09974 -0.1 0.2947 -0.23339 -2.60518\\nVAL_9 -1.84956 0.39364 0.01636 -0.04219 0.3013 0 0 0 0 0 0 0 0 0 0 -0.10662 -0.04099 0.30634 -0.25392 0.29 1.32864 -0.26398 0.07902\\nPRO_10 -1.26129 0.36307 0.0021 -0.34493 0.18842 0 0 0 0 0 0 0 0 0 0 -0.22533 -0.10212 0.04599 -0.47478 0.02 1.72771 1.68253 1.62137\\nARG_11 -1.63291 0.03172 0.01243 1.44491 0.25504 0 0 0 0 0 0 0 0 0 0 -0.02125 0.00078 1.58813 -0.02273 -0.98 0.58324 -1.12468 0.13469\\nPHE_12 -1.53114 0.05915 0.01765 -0.13696 0.18359 0 0 0 0 0 0 0 0 0 0 -0.10723 -0.07209 1.45649 0.02143 0.63 0.40197 -0.89708 0.02581\\nSER_13 -1.91931 0.12437 0.00223 1.38431 0.26082 -0.04604 0 0 0 0 0 0 0 0 0 0.35871 -0.05476 0.3125 0.26376 -0.37 0.37033 0.22997 0.91688\\nGLY_14 -1.56416 1.19084 7e-05 0.43443 0.31006 0 0 0 0 0 0 0 0 0 0 0.08099 -0.08511 0 0.35659 -0.17 2.26647 0.84949 3.66968\\nPRO_15 -3.01901 1.42109 0.00625 0.03984 0.58601 0 0 0 0 0 0 0 0 0 0 -0.26335 -0.06729 0.24461 -0.38492 0.02 4.07814 -0.0863 2.57508\\nGLU_16 -3.16976 1.44294 0.00699 0.75059 0.53041 0 0 0 0 0 0 0 0 0 0 -0.3423 -0.10944 2.49511 -0.17141 -0.81 4.6598 0.54816 5.83109\\nILE_17 -2.96934 1.31148 0.01914 0.24509 0.4145 0 0 0 0 0 0 0 0 0 0 -0.07495 -0.0514 0.05137 -0.22534 0.24 3.78576 0.76207 3.50838\\nTHR_18 -4.67974 0.8269 0.00505 1.00136 0.73201 -0.04604 0 0 0 0 0 0 0 0 0 -0.19807 -0.04565 0.23534 0.0787 -0.27 2.23611 0.42631 0.30227\\nLEU_19 -4.18292 0.76821 0.01588 -0.20728 0.52403 0 0 0 0 0 0 0 0 0 0.3127 -0.0776 -0.00204 0.22853 -0.09066 -0.1 2.19306 -0.75613 -1.37422\\nILE_20 -4.2199 0.40024 0.02514 -0.11028 0.47671 0 0 0 0 0 0 0 0 0 0 -0.17508 -0.05449 0.08958 -0.17222 0.24 2.50203 -0.81275 -1.81101\\nILE_21 -4.75324 0.53115 0.02405 -0.17224 0.45335 0 0 0 0 0 0 0 0 0 0 -0.0583 -0.01291 0.06376 -0.18209 0.24 2.57585 -0.7762 -2.06683\\nPHE_22 -3.56332 0.20102 0.01847 0.00897 0.35512 0 0 0 0 0 0 0 0 0 0.29334 -0.15042 -0.00532 1.11516 -0.18203 0.63 2.78826 -0.85394 0.6553\\nGLY_23 -2.64764 0.19947 0.00048 0.90403 0.27185 0 0 0 0 0 0 0 0 0 0.32058 0.07334 -0.0175 0 0.28323 -0.17 3.35163 1.1475 3.71697\\nVAL_24 -3.71703 0.45312 0.01828 -0.00285 0.36828 0 0 0 0 0 0 0 0 0 0 -0.14387 -0.05187 -0.01962 -0.16555 0.29 2.84638 -0.3403 -0.46505\\nMET_25 -4.11805 0.2132 0.00705 0.20459 0.40197 0 0 0 0 0 0 0 0 0 0 -0.17829 0.00093 0.8954 0.02295 -0.34 2.41801 -0.08931 -0.56153\\nALA_26 -3.33029 0.47514 0.00181 0.4956 0.34027 0 0 0 0 0 0 0 0 0 0.30151 -0.22741 -0.03388 0 -0.07522 0.16 3.28653 0.61198 2.00602\\nGLY_27 -2.50118 0.13833 0.00043 0.9042 0.2563 0 0 0 0 0 0 0 0 0 0.3127 0.08575 -0.02999 0 0.27193 -0.17 3.54377 1.10311 3.91535\\nILE_28 -4.02139 0.29903 0.02254 -0.2403 0.39894 0 0 0 0 0 0 0 0 0 0 -0.15584 -0.0529 0.06409 -0.20478 0.24 2.39861 -0.81275 -2.06475\\nILE_29 -4.72961 0.63369 0.02528 -0.24032 0.45649 0 0 0 0 0 0 0 0 0 0 -0.15346 -0.03193 0.07165 -0.18618 0.24 2.29648 -0.7762 -2.39411\\nGLY_30 -3.27799 0.41572 0.00045 0.90405 0.33037 0 0 0 0 0 0 0 0 0 0.63833 0.05919 -0.04993 0 0.28854 -0.17 2.95836 1.09951 3.19661\\nTHR_31 -3.91161 0.39671 0.00811 1.39446 0.40126 0 0 0 0 0 0 0 0 0 0.32058 -0.06251 -0.01783 0.00657 0.00825 -0.27 2.98322 0.65874 1.91596\\nILE_32 -4.48193 0.55526 0.02429 -0.24789 0.44959 0 0 0 0 0 0 0 0 0 0 -0.15662 -0.04132 0.06643 -0.19496 0.24 2.47212 -0.79691 -2.11194\\nLEU_33 -4.21751 0.52847 0.01478 -0.2695 0.4453 0 0 0 0 0 0 0 0 0 0 -0.17762 -0.02661 0.4769 -0.11139 -0.1 2.26238 -0.7971 -1.9719\\nLEU_34 -3.70315 0.42801 0.01498 -0.32112 0.45417 0 0 0 0 0 0 0 0 0 0.30151 -0.22678 -0.03241 0.29478 -0.12759 -0.1 2.3894 -0.82365 -1.45186\\nILE_35 -4.50459 0.46524 0.02165 -0.38204 0.63195 0 0 0 0 0 0 0 0 0 0 -0.14298 -0.03537 0.05611 -0.20746 0.24 2.6883 -0.79369 -1.96287\\nSER_36 -3.60748 0.32969 0.00177 1.1487 0.64266 -0.02158 0 0 0 0 0 0 0 0 0 -0.17502 -0.05642 0.38896 0.14356 -0.37 3.15475 0.21548 1.79507\\nTYR_37 -3.89391 0.32948 0.01903 0.16036 0.72315 0 0 0 0 0 0 0 0 0 0 -0.1376 -0.02297 1.15229 -0.13909 0.51 3.36784 -0.05565 2.01294\\nSER_38 -4.04219 0.80239 0.00129 0.48447 0.99577 -0.02158 0 0 0 0 0 0 0 0 0.34499 -0.22894 -0.03858 0.35898 0.12092 -0.37 3.92083 0.27404 2.60239\\nILE_39 -3.43427 0.749 0.02201 -0.1292 0.81194 0 0 0 0 0 0 0 0 0 0 -0.21125 -0.07176 0.19036 -0.10577 0.24 2.6186 0.52484 1.20451\\nARG_40 -2.38993 0.20285 0.01922 0.04752 0.63795 0 0 0 0 0 0 0 0 0 0 -0.13801 -0.04016 1.63828 -0.03883 -0.98 0.65317 -2.50356 -2.89148\\nARG_41 -2.73166 0.04311 0.01737 0.04145 0.54951 0 0 0 0 0 0 0 0 0 0 -0.18183 -0.01013 1.71744 -0.05806 -0.98 0.41158 -2.14879 -3.33\\nLEU:CtermProteinFull_42 -2.30674 0.06368 0.01464 -0.00396 0.71812 0 0 0 0 0 0 0 0 0 0 0 0 0.44217 0 -0.1 0.35107 1.84739 1.02637\\n#END_POSE_ENERGIES_TABLE GYPA_gorilla__0.pdb\\n\\n\",\"pdb\");\n","\tviewer_17294480429883013.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n","\tviewer_17294480429883013.zoomTo();\n","viewer_17294480429883013.render();\n","});\n","</script>"]},"metadata":{}}]},{"cell_type":"markdown","metadata":{"id":"JFpJRYPwJ8w6"},"source":["**üìöHOMEWORK:** As a challenge, using what you have learned in the previous tutorials to superimpose structures and determine their similarity using RMSD, take your best structures and compare them to answer the following question: **Is the TM region different between orthologs?**"]},{"cell_type":"markdown","metadata":{"id":"7CV7AsCQIFrB"},"source":["5. To finalize, you can download your output files as indicated in the code cell below"]},{"cell_type":"code","metadata":{"id":"oy4qyfAGMDFp"},"source":["#download your output\n","%%bash\n","zip -r example.zip ./outs\n","mkdir outs\n","cp *.pdb ./outs/"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"lJpeTpYPIKfB"},"source":["**This is the end of the fifth tutorial. Good Science!**"]},{"cell_type":"markdown","metadata":{"id":"s9giUnnEAast"},"source":["#Appendix A"]},{"cell_type":"markdown","metadata":{"id":"n1NoW-mIX3Fp"},"source":["## Generating the input files using the standalone command-line version of Rosetta"]},{"cell_type":"markdown","metadata":{"id":"FF5pgAKmh14Y"},"source":["First, you need to get a Rosetta license [HERE](https://els2.comotion.uw.edu/product/rosetta), as you did for PyRosetta"]},{"cell_type":"markdown","metadata":{"id":"pZ8OMgudYJi9"},"source":["### Download and extract Rosetta in Google Colab (ETA ~15min)"]},{"cell_type":"code","metadata":{"id":"9eLMeKl_vyuf"},"source":["!wget --user=Academic_User --password=Xry3x4 https://www.rosettacommons.org/downloads/academic/3.12/rosetta_bin_linux_3.12_bundle.tgz\n","!tar xvf /content/rosetta_bin_linux_3.12_bundle.tgz"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"5XUiyaTK6nxy"},"source":["!unzip input_files.zip"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"oovXJfZUl5g7"},"source":["### Grishin File Generation"]},{"cell_type":"markdown","metadata":{"id":"ZaQNZszVl8bf"},"source":["The Grishin alignment file format is documented [here](https://www.rosettacommons.org/docs/latest/rosetta_basics/file_types/Grishan-format-alignment). The following is an example of its structure\n","```\n","## target_name template_name.pdb\n","#\n","scores from program: 0\n","0 HERE_GOES_YOUR_TARGET_SEQUENCE_ALIGNMENT_IN_ONE_LINE\n","0 HERE_GOES_YOUR_TEMPLATE_SEQUENCE_ALIGNMENT_IN_ONE_LINE\n","```\n","This is one of the real Grishin files used in this tutorial\n","```\n","## gorillaGYPA 5eh6_A.pdb\n","#\n","scores from program: 0\n","0 DNGEWIQLVPRFSGPEITLIIFGVMAGIIGTILLISYSIRRL\n","0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n","```\n","\n"]},{"cell_type":"code","metadata":{"id":"RmZi9ejLmvYF"},"source":["#Modify and run this cell to create grishin from clustal alignment\n","%%bash\n","echo \\\n","\"\"\"\\\n","## gorillaGYPA 5eh6_A.pdb\n","#\n","scores from program: 0\n","0 DNGEWIQLVPRFSGPEITLIIFGVMAGIIGTILLISYSIRRL\n","0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n","\"\"\" > /content/input_files/grishin/5eh6_on_gorillaGYPA.grishin\n","\n","echo \\\n","\"\"\"\\\n","## GYPA 5eh6_A.pdb\n","#\n","scores from program: 0\n","0 ETGERVQLAHHFSEPEITLIIFGVMAGVIGTILLISYGIRRL\n","0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n","\"\"\" > /content/input_files/grishin/5eh6_on_humanGYPA.grishin\n","\n","echo \\\n","\"\"\"\\\n","## marmotaGYPA 5eh6_A.pdb\n","#\n","scores from program: 0\n","0 -KQGRNQIIHPFSEPVIILIIFAVMFGIIGTILLISFCVRRL\n","0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n","\"\"\" > /content/input_files/grishin/5eh6_on_marmotaGYPA.grishin\n","\n","\n","echo \\\n","\"\"\"\\\n","## OrnithorhynchusGYPA 5eh6_A.pdb\n","#\n","scores from program: 0\n","0 ---IDPEIEHIFTGPVAAVIIYAVVCGVIGTILFIALVIKVV\n","0 --------------PEITLIIFGVIAGVIGTILLISYGIRRL\n","\"\"\" > /content/input_files/grishin/5eh6_on_ornithorhynchusGYPA.grishin"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"iAfdLyUYYV40"},"source":["### Partial Threading"]},{"cell_type":"code","metadata":{"id":"COXQteMfi0Ku"},"source":["import urllib.request\n","urllib.request.urlretrieve('http://files.rcsb.org/download/5eh6.pdb', '/content/input_files/pdbs/5eh6.pdb')"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"z6Z-k7TDigtJ"},"source":["%%bash\n","cd /content/input_files/pdbs\n","/content/rosetta_bin_linux_2020.08.61146_bundle/tools/protein_tools/scripts/clean_pdb.py 5eh6.pdb A\n","mv 5eh6_A.* ../cleaned"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"Yw46R_v1560X"},"source":["!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/GYPA.fasta -in:file:alignment ./grishin/5eh6.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n","!mv 5eh6_A.pdb.pdb 5eh6_A.human.pdb\n","!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/ornithorhynchusGYPA.fasta -in:file:alignment ./grishin/5eh6_on_ornithorhynchusGYPA.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n","!mv 5eh6_A.pdb.pdb 5eh6_A.ornithorhynchus.pdb\n","!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/marmotaGYPA.fasta -in:file:alignment ./grishin/5eh6_on_marmotaGYPA.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n","!mv 5eh6_A.pdb.pdb 5eh6_A.gorilla.pdb\n","!/content/rosetta_bin_linux_2020.08.61146_bundle/main/source/bin/partial_thread.static.linuxgccrelease -in:file:fasta ./fasta/gorillaGYPA.fasta -in:file:alignment ./grishin/5eh6_on_gorillaGYPA.grishin -in:file:template_pdb ./cleaned/5eh6_A.pdb\n","!mv 5eh6_A.pdb.pdb 5eh6_A.marmota.pdb\n","!mv 5eh6_A.*pdb ./input_files/threaded_pdbs/"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"2Wn-5LuU44Wm"},"source":["### Getting the fragments from oldRobetta server\n"]},{"cell_type":"markdown","metadata":{"id":"LmPktANT5uEe"},"source":["As stated in the Rosetta documentation fragments are:\n","\n","\n","> (...) used in the assembly of proteins whether for structure prediction or design, to cut down on the size of the protein-folding search space. They are a core part of the Rosetta design.\n","\n","For the purpose of this tutorial, we skipped this part. However, in the following figure you can see a screenshot of the [Robetta-server](http://old.robetta.org/fragmentsubmit.jsp) where you can obtain the fragments. For this, you have to register and submit a FASTA sequence of your target proteins. Do keep in mind that the length of your target will impact in the time needed for the fragment picking process."]},{"cell_type":"markdown","metadata":{"id":"Neo5VCZVJoBl"},"source":["<figure>\n","<center>\n","<img src='https://raw.githubusercontent.com/pb3lab/ibm3202/master/images/mpmodelling_04.png'/>\n","</figcaption></center>\n","</figure>"]},{"cell_type":"markdown","metadata":{"id":"xvXqKZZdZNjJ"},"source":["When your job is finished you will get an email with the corresponding fragments. You have to download them and place them inside the frags folder and make sure that their filename matches the one in XML RosettaScripts cell"]}]}